{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Generate dataset"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 20,
   "metadata": {},
   "outputs": [],
   "source": [
    "#This is a script that loads a pandas dataframe from csv, converts the string formatted as a list into a list of lists. The list of lists is then flattened into a single list.\n",
    "\n",
    "import pandas as pd\n",
    "import random\n",
    "import json\n",
    "df = pd.read_csv('./gpt_final_outs/o3mini_maxclust_80_subset_random_1000_nocutoff.csv', index_col=None)\n",
    "\n",
    "#rename the column \"Entity Name\" to \"Entity\" and \"Group Items\" to \"output\"\n",
    "df = df.rename(columns={\"Group Items\": \"Output\"})\n",
    "\n",
    "# Convert the string formatted as a list into a list of lists\n",
    "df['Output'] = df['Output'].apply(lambda x: eval(x))\n",
    "\n",
    "# Flatten the list of lists into a single list\n",
    "df['Input'] = df['Output'].apply(lambda x: [item.strip(\"*\") for sublist in x for item in sublist])\n",
    "\n",
    "#randomize the order of the single list\n",
    "df['Input'] = df['Input'].apply(lambda x: random.sample(x, len(x)))\n",
    "#df = df[df['Input'].apply(len) < 30]\n",
    "# make a new column, where internal lists of length 1 are removed \n",
    "df['Output>1'] = df['Output'].apply(lambda x: [item for item in x if len(item) > 1])\n",
    "#create a flag column by name one_item. For each row in Output column, If length of the list in output column is less than 2, put True otherwise False.\n",
    "df['one_item'] = df['Output>1'].apply(lambda x: True if len(x) == 0 else False)\n",
    "\n",
    "#zip the Group_Items_no_singles with the Entity Name \n",
    "df[\"Output\"] = df.apply(lambda row: {row[\"Entity Name\"]: row['Output']}, axis=1)\n",
    "df[\"Input\"] = df.apply(lambda row: {row[\"Entity Name\"]: row['Input']}, axis=1)\n",
    "df['Output>1'] = df.apply(lambda row: {row[\"Entity Name\"]: row['Output>1']}, axis=1)\n",
    "\n",
    "\n",
    "\n",
    "\n",
    "\n",
    "#convert the columns to json strings\n",
    "df[\"Output\"] = df[\"Output\"].apply(lambda x: json.dumps(x) if isinstance(x, dict) else x)\n",
    "df[\"Input\"] = df[\"Input\"].apply(lambda x: json.dumps(x) if isinstance(x, dict) else x)\n",
    "df[\"Output>1\"] = df[\"Output>1\"].apply(lambda x: json.dumps(x) if isinstance(x, dict) else x)\n",
    "\n",
    "#Change order of columns to have input, then output, then output>1, then entity name\n",
    "df = df[['Entity Name', 'Input', 'Output', 'Output>1', 'one_item']]\n",
    "\n",
    "#filter by Output>1 length of the list > 1\n",
    "#use column Output, but use only the ones where it clusteres multiple items together"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 21,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Entity Name</th>\n",
       "      <th>Input</th>\n",
       "      <th>Output</th>\n",
       "      <th>Output&gt;1</th>\n",
       "      <th>one_item</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>58963</td>\n",
       "      <td>{\"58963\": [\"Barrena et al. [researcher]\", \"Gar...</td>\n",
       "      <td>{\"58963\": [[\"Gardea-Torresdey et al. [research...</td>\n",
       "      <td>{\"58963\": []}</td>\n",
       "      <td>True</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>22775</td>\n",
       "      <td>{\"22775\": [\"physiological transport of macro-m...</td>\n",
       "      <td>{\"22775\": [[\"substance that could be transloca...</td>\n",
       "      <td>{\"22775\": [[\"**transportation [phenotype]**\", ...</td>\n",
       "      <td>False</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>22996</td>\n",
       "      <td>{\"22996\": [\"number of open stomata and amount ...</td>\n",
       "      <td>{\"22996\": [[\"**transpiration and stomatal cond...</td>\n",
       "      <td>{\"22996\": [[\"**transpiration and stomatal cond...</td>\n",
       "      <td>False</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>100116</td>\n",
       "      <td>{\"100116\": [\"Clade SH [clade]\", \"Clade M [clad...</td>\n",
       "      <td>{\"100116\": [[\"Clade M [clade]\"], [\"Clade SL [c...</td>\n",
       "      <td>{\"100116\": []}</td>\n",
       "      <td>True</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>133255</td>\n",
       "      <td>{\"133255\": [\"PSPTO3648 [gene]\", \"LPP\\u10b2 [ge...</td>\n",
       "      <td>{\"133255\": [[\"**PLSP2A [gene]**\", \"PLSP2A gene...</td>\n",
       "      <td>{\"133255\": [[\"**PLSP2A [gene]**\", \"PLSP2A gene...</td>\n",
       "      <td>False</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>995</th>\n",
       "      <td>31615</td>\n",
       "      <td>{\"31615\": [\"auxin-mediated modulation of the m...</td>\n",
       "      <td>{\"31615\": [[\"auxin-mediated modulation of the ...</td>\n",
       "      <td>{\"31615\": [[\"auxin-dependent inhibition of end...</td>\n",
       "      <td>False</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>996</th>\n",
       "      <td>157227</td>\n",
       "      <td>{\"157227\": [\"GmSAUR [gene]\", \"ZmSAUR1 (Zea may...</td>\n",
       "      <td>{\"157227\": [[\"GmSAUR [gene]\"], [\"StSAUR [gene]...</td>\n",
       "      <td>{\"157227\": [[\"ZmSAUR1 (Zea mays SAURs) [gene]\"...</td>\n",
       "      <td>False</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>997</th>\n",
       "      <td>131559</td>\n",
       "      <td>{\"131559\": [\"BtuC gene(s) [gene]\", \"cytochrome...</td>\n",
       "      <td>{\"131559\": [[\"BtuB gene(s) [gene]\"], [\"BtuC ge...</td>\n",
       "      <td>{\"131559\": []}</td>\n",
       "      <td>True</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>998</th>\n",
       "      <td>99091</td>\n",
       "      <td>{\"99091\": [\"Genome sequencing of O. fragrans '...</td>\n",
       "      <td>{\"99091\": [[\"Genome sequencing of O. fragrans ...</td>\n",
       "      <td>{\"99091\": [[\"**O. fragrans 'Liuyejingui' (OFL)...</td>\n",
       "      <td>False</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>999</th>\n",
       "      <td>65306</td>\n",
       "      <td>{\"65306\": [\"NF-TOC1 [protein complex]\", \"Hd1/N...</td>\n",
       "      <td>{\"65306\": [[\"**CO/NF-Y complex [protein comple...</td>\n",
       "      <td>{\"65306\": [[\"**CO/NF-Y complex [protein comple...</td>\n",
       "      <td>False</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>1000 rows × 5 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "     Entity Name                                              Input  \\\n",
       "0          58963  {\"58963\": [\"Barrena et al. [researcher]\", \"Gar...   \n",
       "1          22775  {\"22775\": [\"physiological transport of macro-m...   \n",
       "2          22996  {\"22996\": [\"number of open stomata and amount ...   \n",
       "3         100116  {\"100116\": [\"Clade SH [clade]\", \"Clade M [clad...   \n",
       "4         133255  {\"133255\": [\"PSPTO3648 [gene]\", \"LPP\\u10b2 [ge...   \n",
       "..           ...                                                ...   \n",
       "995        31615  {\"31615\": [\"auxin-mediated modulation of the m...   \n",
       "996       157227  {\"157227\": [\"GmSAUR [gene]\", \"ZmSAUR1 (Zea may...   \n",
       "997       131559  {\"131559\": [\"BtuC gene(s) [gene]\", \"cytochrome...   \n",
       "998        99091  {\"99091\": [\"Genome sequencing of O. fragrans '...   \n",
       "999        65306  {\"65306\": [\"NF-TOC1 [protein complex]\", \"Hd1/N...   \n",
       "\n",
       "                                                Output  \\\n",
       "0    {\"58963\": [[\"Gardea-Torresdey et al. [research...   \n",
       "1    {\"22775\": [[\"substance that could be transloca...   \n",
       "2    {\"22996\": [[\"**transpiration and stomatal cond...   \n",
       "3    {\"100116\": [[\"Clade M [clade]\"], [\"Clade SL [c...   \n",
       "4    {\"133255\": [[\"**PLSP2A [gene]**\", \"PLSP2A gene...   \n",
       "..                                                 ...   \n",
       "995  {\"31615\": [[\"auxin-mediated modulation of the ...   \n",
       "996  {\"157227\": [[\"GmSAUR [gene]\"], [\"StSAUR [gene]...   \n",
       "997  {\"131559\": [[\"BtuB gene(s) [gene]\"], [\"BtuC ge...   \n",
       "998  {\"99091\": [[\"Genome sequencing of O. fragrans ...   \n",
       "999  {\"65306\": [[\"**CO/NF-Y complex [protein comple...   \n",
       "\n",
       "                                              Output>1  one_item  \n",
       "0                                        {\"58963\": []}      True  \n",
       "1    {\"22775\": [[\"**transportation [phenotype]**\", ...     False  \n",
       "2    {\"22996\": [[\"**transpiration and stomatal cond...     False  \n",
       "3                                       {\"100116\": []}      True  \n",
       "4    {\"133255\": [[\"**PLSP2A [gene]**\", \"PLSP2A gene...     False  \n",
       "..                                                 ...       ...  \n",
       "995  {\"31615\": [[\"auxin-dependent inhibition of end...     False  \n",
       "996  {\"157227\": [[\"ZmSAUR1 (Zea mays SAURs) [gene]\"...     False  \n",
       "997                                     {\"131559\": []}      True  \n",
       "998  {\"99091\": [[\"**O. fragrans 'Liuyejingui' (OFL)...     False  \n",
       "999  {\"65306\": [[\"**CO/NF-Y complex [protein comple...     False  \n",
       "\n",
       "[1000 rows x 5 columns]"
      ]
     },
     "execution_count": 21,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df\n",
    "#note i don't really use Outputs>1 in this workflow."
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## System message prompts for making fine-tuning dataset (v1,v2,v3,v4)\n",
    "\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 23,
   "metadata": {},
   "outputs": [],
   "source": [
    "system_message = \"\"\"\n",
    "You are a data scientist specializing in grouping plant biological entities. Your task is to cluster similar entities while strictly adhering to the following guidelines:\n",
    "\t1.\tExact Phrase Matching Matters: \n",
    "1.1 Consider the Entire Phrase: Treat each entity as a single, whole phrase. This includes all key biological terms and any bracketed text\n",
    "1.2 Ignore Minor Surface Differences: Minor variations such as letter casing (uppercase vs. lowercase), spacing, punctuation, standard abbreviations, or singular vs. plural forms do not create new or separate entities.\n",
    "\t2.\tStrict (100%) Key Term Separation: If an entity has a different key biological term, it MUST GO into a separate cluster.\n",
    "3. Sub-identifier separation: If an entity differs by any numeric value, sub-identifier, or qualifier, they MUST BE placed in separate clusters.\n",
    "\t4.\tAvoid False Similarity: DO NOT cluster two entities together simply because they share a common word or term if their overall key term or concept is different.\n",
    "5. Extra Descriptor Differentiation: If one entity has an extra descriptor that changes its meaning, do not group them together.\n",
    "\t6.\tStrict Synonym/Near-Synonym Grouping: Only group entities together if they refer to the exact same biological structure, process, or concept.\n",
    "\t7.\tMaintain 100% Precision: If there is any doubt about whether two entities are the same, MUST place them in separate clusters.\n",
    "\t8.\tPreserve Original Data: DO NOT introduce new items, create duplicates, or omit any entity from your final output.\n",
    "\t9.\tOutput Format: Always return results in valid JSON format. You MUST USE GIVEN KEY.\n",
    "10. Discard Single-element Clusters: Remove any cluster that ends up with only one entity.\n",
    "11. Choose cluster representative: YOU MUST pickup most appropriate and easy-to-understand cluster representative and enclose it with '**', if there is more than one entity in that particular cluster. For example, pick the full term instead of an abbreviation.\n",
    "\n",
    "Read the input list, and return a clustered output list.\n",
    "\"\"\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## System message prompts for finetuning (v5))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 24,
   "metadata": {},
   "outputs": [],
   "source": [
    "system_message = \"\"\"You are senior scientist in plant biology. You NEED to meticulously do reasoning and then cluster given input, and provide the reasoning followed by clusters based on the reasoning performed.\n",
    "\n",
    "The following are the entity clustering guidelines:\n",
    "\t1.\tExact Phrase Matching Matters: Always consider the full phrase, including key biological terms, bracketed text (ignoring minor differences such as spacing, punctuation, correct abbreviations, plurality).\n",
    "\t2.\tStrict (100%) Key Term Separation: Entities with different biological terms MUST be placed in separate clusters.\n",
    "3. Sub-identifier separation: Separate Entities with numeric differences, sub-identifiers, or qualifiers into different groups.\n",
    "\t4.\tAvoid False Similarity: Do NOT cluster two items together in same group just because they share a common word or term.\n",
    "\t5.\tStrict Synonym/Near-Synonym Grouping: Only group entities that refer to the same biological structure, process, meaning or concept.\n",
    "\t6.\tMaintain 100% Precision: When in even small doubt, MUST place entities in separate clusters.\n",
    "\t7.\tPreserve Original Data: No new items should be introduced, no duplicates should be introduced, and no entities should be omitted.\n",
    "8. YOU MUST pickup most appropriate and easy-to-understand cluster representative and enclose it with '**', if there is more than one entity in that particular cluster. For example, pick the full term instead of an abbreviation. MUST mention this in reasoning statement.\n",
    "\t9.\tOutput Format: Always return results in valid JSON format. MUST USE GIVEN KEY.\n",
    "10. Discard Single-element Clusters: Remove any cluster that ends up with only one entity.\n",
    "\n",
    "Your main task is to read the input, do reasoning and output the reasoning and the correct clusters in a valid JSON object format. Input is a json object with given key and the value is the list of biological entities. The output is a json object with given key and the values are two json objects: reasoning and corresponding clustered entities (list of lists where each inner list is one cluster).\n",
    "\n",
    "\n",
    "\"\"\"\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 25,
   "metadata": {},
   "outputs": [],
   "source": [
    "#read output csv file from the o3-mini validation\n",
    "output_csv = pd.read_csv(\"./o3mini_val_outs/o3mini_1k.csv\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 26,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>cluster_id</th>\n",
       "      <th>is_correct</th>\n",
       "      <th>user_prompt</th>\n",
       "      <th>clusters</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>58963</td>\n",
       "      <td>True</td>\n",
       "      <td>Great job, that looks correct because each res...</td>\n",
       "      <td>[]</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>22775</td>\n",
       "      <td>True</td>\n",
       "      <td>Great job, the clusters appear to be correctly...</td>\n",
       "      <td>[[\"**transportation [phenotype]**\", \"transport...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>22996</td>\n",
       "      <td>False</td>\n",
       "      <td>I think you are wrong. In the current clusteri...</td>\n",
       "      <td>[[\"**transpiration and stomatal conductance va...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>100116</td>\n",
       "      <td>True</td>\n",
       "      <td>This looks correct to me, as the grouping foll...</td>\n",
       "      <td>[]</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>133255</td>\n",
       "      <td>True</td>\n",
       "      <td>Great job, the clustering is correct. The gene...</td>\n",
       "      <td>[[\"**PLSP2A [gene]**\", \"PLSP2A gene(s) [gene]\"...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>995</th>\n",
       "      <td>31615</td>\n",
       "      <td>False</td>\n",
       "      <td>I think you are wrong. The grouping of the end...</td>\n",
       "      <td>[[\"**auxin-regulated endocytosis [process]**\",...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>996</th>\n",
       "      <td>157227</td>\n",
       "      <td>True</td>\n",
       "      <td>This looks correct to me. Each gene entity is ...</td>\n",
       "      <td>[[\"ZmSAUR1 (Zea mays SAURs) [gene]\", \"ZmSAUR1 ...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>997</th>\n",
       "      <td>131559</td>\n",
       "      <td>True</td>\n",
       "      <td>Great job, that looks correct to me. Each gene...</td>\n",
       "      <td>[]</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>998</th>\n",
       "      <td>99091</td>\n",
       "      <td>True</td>\n",
       "      <td>Great job, the clustering is correct. You have...</td>\n",
       "      <td>[[\"**O. fragrans 'Liuyejingui' (OFL) [organism...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>999</th>\n",
       "      <td>65306</td>\n",
       "      <td>True</td>\n",
       "      <td>This looks correct to me, output the clusters ...</td>\n",
       "      <td>[[\"**CO/NF-Y complex [protein complex]**\", \"CO...</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>1000 rows × 4 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "     cluster_id  is_correct  \\\n",
       "0         58963        True   \n",
       "1         22775        True   \n",
       "2         22996       False   \n",
       "3        100116        True   \n",
       "4        133255        True   \n",
       "..          ...         ...   \n",
       "995       31615       False   \n",
       "996      157227        True   \n",
       "997      131559        True   \n",
       "998       99091        True   \n",
       "999       65306        True   \n",
       "\n",
       "                                           user_prompt  \\\n",
       "0    Great job, that looks correct because each res...   \n",
       "1    Great job, the clusters appear to be correctly...   \n",
       "2    I think you are wrong. In the current clusteri...   \n",
       "3    This looks correct to me, as the grouping foll...   \n",
       "4    Great job, the clustering is correct. The gene...   \n",
       "..                                                 ...   \n",
       "995  I think you are wrong. The grouping of the end...   \n",
       "996  This looks correct to me. Each gene entity is ...   \n",
       "997  Great job, that looks correct to me. Each gene...   \n",
       "998  Great job, the clustering is correct. You have...   \n",
       "999  This looks correct to me, output the clusters ...   \n",
       "\n",
       "                                              clusters  \n",
       "0                                                   []  \n",
       "1    [[\"**transportation [phenotype]**\", \"transport...  \n",
       "2    [[\"**transpiration and stomatal conductance va...  \n",
       "3                                                   []  \n",
       "4    [[\"**PLSP2A [gene]**\", \"PLSP2A gene(s) [gene]\"...  \n",
       "..                                                 ...  \n",
       "995  [[\"**auxin-regulated endocytosis [process]**\",...  \n",
       "996  [[\"ZmSAUR1 (Zea mays SAURs) [gene]\", \"ZmSAUR1 ...  \n",
       "997                                                 []  \n",
       "998  [[\"**O. fragrans 'Liuyejingui' (OFL) [organism...  \n",
       "999  [[\"**CO/NF-Y complex [protein complex]**\", \"CO...  \n",
       "\n",
       "[1000 rows x 4 columns]"
      ]
     },
     "execution_count": 26,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "output_csv"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [],
   "source": [
    "# RUN THIS CELL ONLY for V5 \n",
    "#filter rows in df such that entity name is not in the empty keys (JUST REMOVING THE rows with EMPTY CORRECTED CLUSTERS)\n",
    "empty_keys = [58963, 100116, 2276, 82106, 169694, 89650, 46224, 74528, 59649, 88515, 42251, 138189, 121395, 66209, 109497, 51350, 125145, 83273, 31266, 20901, 155284, 69128, 149825, 54598, 155550, 76415, 10829, 29912, 147673, 13676, 43068, 154151, 158579, 154079, 121796, 64226, 148026, 30540, 13722, 38783, 141611, 152629, 158331, 164114, 61159, 30892, 11432, 118549, 170116, 8815, 23115, 135141, 130704, 13447, 82891, 79708, 49031, 137362, 108939, 30902, 1979, 161173, 10932, 49361, 6893, 47841, 6353, 53420, 134561, 141969, 163157, 170003, 5317, 45644, 100049, 116121, 96226, 80941, 104768, 54324, 153755, 63041, 49218, 83032, 74072, 95491, 136879, 128564, 3984, 149304, 108934, 111625, 40897, 110509, 150263, 20671, 54278, 150596, 107203, 154934, 159575, 151766, 151906, 126710, 96658, 61283, 68647, 140937, 52882, 58544, 69542, 51757, 95962, 45446, 55035, 25174, 146950, 151274, 149112, 159628, 55001, 141304, 77432, 112493, 90683, 101253, 154023, 19303, 141728, 62513, 117070, 130555, 103177, 88707, 24195, 150152, 22385, 115627, 152158, 146433, 120792, 153959, 49211, 115818, 80040, 60288, 76699, 156169, 123227, 24942, 50885, 100895, 167231, 51356, 42848, 55951, 123371, 121683, 93067, 91142, 76685, 141591, 55077, 112506, 396, 114365, 130281, 86746, 147191, 9338, 22536, 121491, 47170, 165937, 150418, 105086, 81729, 75904, 33899, 66190, 51024, 151910, 42267, 71717, 42070, 104675, 63463, 42619, 106019, 93865, 143138, 6596, 153123, 68999, 91023, 123962, 133452, 90992, 59529, 148751, 153190, 124484, 16960, 153122, 124186, 99387, 151388, 61033, 129608, 52510, 172622, 63985, 125670, 79230, 72583, 81435, 70356, 62742, 135336, 150457, 66974, 166178, 157848, 87401, 14243, 121095, 2505, 126507, 147170, 64516, 141136, 78873, 160119, 41639, 90709, 103854, 158312, 148245, 154282, 75046, 49728, 8072, 421, 136198, 103253, 82563, 85713, 155406, 87406, 58568, 131897, 79792, 112158, 148520, 162083, 40475, 130199, 68083, 23032, 77116, 106386, 35567, 115768, 60811, 72695, 68689, 170670, 29340, 160703, 10963, 38965, 4789, 99646, 59223, 93167, 108297, 42453, 87710, 55327, 88040, 81258, 91177, 45197, 57719, 12113, 136929, 16321, 76485, 160180, 102136, 46329, 138085, 153343, 31361, 20951, 159175, 59286, 31879, 116062, 134504, 36974, 130268, 154497, 103242, 115840, 45677, 60775, 47189, 43411, 20497, 152378, 126195, 76278, 20565, 174379, 151225, 152992, 60554, 125654, 15562, 65518, 106938, 31479, 8680, 84113, 55344, 115283, 37690, 165081, 107511, 135491, 102013, 153636, 50605, 32062, 151681, 50986, 55214, 2112, 56183, 75941, 160016, 30687, 61853, 24749, 161408, 150525, 163081, 86850, 167068, 88886, 65805, 131559]\n",
    "\n",
    "df = df[~df['Entity Name'].isin(empty_keys)]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 27,
   "metadata": {},
   "outputs": [],
   "source": [
    "o3_user_prompt_dict = {}\n",
    "#create a dictionary with key as cluster_id and value as the user_prompt \n",
    "for index, row in output_csv.iterrows():\n",
    "    o3_user_prompt_dict[row['cluster_id']] = row['user_prompt']\n",
    "\n",
    "#create a new column named user_prompt in df\n",
    "df['user_prompt'] = df['Entity Name'].map(o3_user_prompt_dict)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 28,
   "metadata": {},
   "outputs": [],
   "source": [
    "#create a dictionary with key as cluster_id and value as the clusters\n",
    "o3_correct_cluster_dict = {}\n",
    "for index, row in output_csv.iterrows():\n",
    "    o3_correct_cluster_dict[row['cluster_id']] = row['clusters']\n",
    "\n",
    "#create a new column named o3_corrected_cluster in df\n",
    "df['o3_corrected_cluster'] = df['Entity Name'].map(o3_correct_cluster_dict)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 29,
   "metadata": {},
   "outputs": [],
   "source": [
    "# create a dictionary with key as cluster_id and value as the is_correct\n",
    "o3_is_correct_dict = {}\n",
    "for index, row in output_csv.iterrows():\n",
    "    o3_is_correct_dict[row['cluster_id']] = row['is_correct']\n",
    "\n",
    "#create a new column named is_correct in df\n",
    "df['is_correct'] = df['Entity Name'].map(o3_is_correct_dict)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 30,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Great job, the clusters appear to be correctly separated based on the guidelines. Each cluster groups only those entities that are 100% synonyms or near-synonyms without mixing distinct biological terms. For example, the cluster grouping '**transportation [phenotype]**' with 'transportion [phenotype]' is correctly grouped because they differ only in minor typographical aspects, and the clusters for 'transport in planta', 'secondary metabolite transport', 'photoassimilate transport', and 'enzyme transport' are all appropriately formed. Now output only the clusters that have more than 1 member in the correct JSON format.\n"
     ]
    }
   ],
   "source": [
    "variable_o3_user_prompt = df.iloc[1]['user_prompt']\n",
    "print(variable_o3_user_prompt)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 31,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[[\"**transportation [phenotype]**\", \"transportion [phenotype]\"], [\"transport within the plant(s) [phenotype]\", \"transport through the plant(s) [phenotype]\", \"**transport in planta [phenotype]**\"], [\"secondary metabolites transportation [phenotype]\", \"**secondary metabolite transport [phenotype]**\"], [\"**photoassimilate transport [phenotype]**\", \"transport of photo-assimilates [phenotype]\"], [\"**enzyme transport [phenotype]**\", \"transport of enzymes [phenotype]\"]]\n"
     ]
    }
   ],
   "source": [
    "variable_o3mini_corrected_cluster = df.iloc[1]['o3_corrected_cluster']\n",
    "print(variable_o3mini_corrected_cluster)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 32,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Entity Name</th>\n",
       "      <th>Input</th>\n",
       "      <th>Output</th>\n",
       "      <th>Output&gt;1</th>\n",
       "      <th>one_item</th>\n",
       "      <th>user_prompt</th>\n",
       "      <th>o3_corrected_cluster</th>\n",
       "      <th>is_correct</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>58963</td>\n",
       "      <td>{\"58963\": [\"Barrena et al. [researcher]\", \"Gar...</td>\n",
       "      <td>{\"58963\": [[\"Gardea-Torresdey et al. [research...</td>\n",
       "      <td>{\"58963\": []}</td>\n",
       "      <td>True</td>\n",
       "      <td>Great job, that looks correct because each res...</td>\n",
       "      <td>[]</td>\n",
       "      <td>True</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>22775</td>\n",
       "      <td>{\"22775\": [\"physiological transport of macro-m...</td>\n",
       "      <td>{\"22775\": [[\"substance that could be transloca...</td>\n",
       "      <td>{\"22775\": [[\"**transportation [phenotype]**\", ...</td>\n",
       "      <td>False</td>\n",
       "      <td>Great job, the clusters appear to be correctly...</td>\n",
       "      <td>[[\"**transportation [phenotype]**\", \"transport...</td>\n",
       "      <td>True</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>22996</td>\n",
       "      <td>{\"22996\": [\"number of open stomata and amount ...</td>\n",
       "      <td>{\"22996\": [[\"**transpiration and stomatal cond...</td>\n",
       "      <td>{\"22996\": [[\"**transpiration and stomatal cond...</td>\n",
       "      <td>False</td>\n",
       "      <td>I think you are wrong. In the current clusteri...</td>\n",
       "      <td>[[\"**transpiration and stomatal conductance va...</td>\n",
       "      <td>False</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>100116</td>\n",
       "      <td>{\"100116\": [\"Clade SH [clade]\", \"Clade M [clad...</td>\n",
       "      <td>{\"100116\": [[\"Clade M [clade]\"], [\"Clade SL [c...</td>\n",
       "      <td>{\"100116\": []}</td>\n",
       "      <td>True</td>\n",
       "      <td>This looks correct to me, as the grouping foll...</td>\n",
       "      <td>[]</td>\n",
       "      <td>True</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>133255</td>\n",
       "      <td>{\"133255\": [\"PSPTO3648 [gene]\", \"LPP\\u10b2 [ge...</td>\n",
       "      <td>{\"133255\": [[\"**PLSP2A [gene]**\", \"PLSP2A gene...</td>\n",
       "      <td>{\"133255\": [[\"**PLSP2A [gene]**\", \"PLSP2A gene...</td>\n",
       "      <td>False</td>\n",
       "      <td>Great job, the clustering is correct. The gene...</td>\n",
       "      <td>[[\"**PLSP2A [gene]**\", \"PLSP2A gene(s) [gene]\"...</td>\n",
       "      <td>True</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>995</th>\n",
       "      <td>31615</td>\n",
       "      <td>{\"31615\": [\"auxin-mediated modulation of the m...</td>\n",
       "      <td>{\"31615\": [[\"auxin-mediated modulation of the ...</td>\n",
       "      <td>{\"31615\": [[\"auxin-dependent inhibition of end...</td>\n",
       "      <td>False</td>\n",
       "      <td>I think you are wrong. The grouping of the end...</td>\n",
       "      <td>[[\"**auxin-regulated endocytosis [process]**\",...</td>\n",
       "      <td>False</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>996</th>\n",
       "      <td>157227</td>\n",
       "      <td>{\"157227\": [\"GmSAUR [gene]\", \"ZmSAUR1 (Zea may...</td>\n",
       "      <td>{\"157227\": [[\"GmSAUR [gene]\"], [\"StSAUR [gene]...</td>\n",
       "      <td>{\"157227\": [[\"ZmSAUR1 (Zea mays SAURs) [gene]\"...</td>\n",
       "      <td>False</td>\n",
       "      <td>This looks correct to me. Each gene entity is ...</td>\n",
       "      <td>[[\"ZmSAUR1 (Zea mays SAURs) [gene]\", \"ZmSAUR1 ...</td>\n",
       "      <td>True</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>997</th>\n",
       "      <td>131559</td>\n",
       "      <td>{\"131559\": [\"BtuC gene(s) [gene]\", \"cytochrome...</td>\n",
       "      <td>{\"131559\": [[\"BtuB gene(s) [gene]\"], [\"BtuC ge...</td>\n",
       "      <td>{\"131559\": []}</td>\n",
       "      <td>True</td>\n",
       "      <td>Great job, that looks correct to me. Each gene...</td>\n",
       "      <td>[]</td>\n",
       "      <td>True</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>998</th>\n",
       "      <td>99091</td>\n",
       "      <td>{\"99091\": [\"Genome sequencing of O. fragrans '...</td>\n",
       "      <td>{\"99091\": [[\"Genome sequencing of O. fragrans ...</td>\n",
       "      <td>{\"99091\": [[\"**O. fragrans 'Liuyejingui' (OFL)...</td>\n",
       "      <td>False</td>\n",
       "      <td>Great job, the clustering is correct. You have...</td>\n",
       "      <td>[[\"**O. fragrans 'Liuyejingui' (OFL) [organism...</td>\n",
       "      <td>True</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>999</th>\n",
       "      <td>65306</td>\n",
       "      <td>{\"65306\": [\"NF-TOC1 [protein complex]\", \"Hd1/N...</td>\n",
       "      <td>{\"65306\": [[\"**CO/NF-Y complex [protein comple...</td>\n",
       "      <td>{\"65306\": [[\"**CO/NF-Y complex [protein comple...</td>\n",
       "      <td>False</td>\n",
       "      <td>This looks correct to me, output the clusters ...</td>\n",
       "      <td>[[\"**CO/NF-Y complex [protein complex]**\", \"CO...</td>\n",
       "      <td>True</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>1000 rows × 8 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "     Entity Name                                              Input  \\\n",
       "0          58963  {\"58963\": [\"Barrena et al. [researcher]\", \"Gar...   \n",
       "1          22775  {\"22775\": [\"physiological transport of macro-m...   \n",
       "2          22996  {\"22996\": [\"number of open stomata and amount ...   \n",
       "3         100116  {\"100116\": [\"Clade SH [clade]\", \"Clade M [clad...   \n",
       "4         133255  {\"133255\": [\"PSPTO3648 [gene]\", \"LPP\\u10b2 [ge...   \n",
       "..           ...                                                ...   \n",
       "995        31615  {\"31615\": [\"auxin-mediated modulation of the m...   \n",
       "996       157227  {\"157227\": [\"GmSAUR [gene]\", \"ZmSAUR1 (Zea may...   \n",
       "997       131559  {\"131559\": [\"BtuC gene(s) [gene]\", \"cytochrome...   \n",
       "998        99091  {\"99091\": [\"Genome sequencing of O. fragrans '...   \n",
       "999        65306  {\"65306\": [\"NF-TOC1 [protein complex]\", \"Hd1/N...   \n",
       "\n",
       "                                                Output  \\\n",
       "0    {\"58963\": [[\"Gardea-Torresdey et al. [research...   \n",
       "1    {\"22775\": [[\"substance that could be transloca...   \n",
       "2    {\"22996\": [[\"**transpiration and stomatal cond...   \n",
       "3    {\"100116\": [[\"Clade M [clade]\"], [\"Clade SL [c...   \n",
       "4    {\"133255\": [[\"**PLSP2A [gene]**\", \"PLSP2A gene...   \n",
       "..                                                 ...   \n",
       "995  {\"31615\": [[\"auxin-mediated modulation of the ...   \n",
       "996  {\"157227\": [[\"GmSAUR [gene]\"], [\"StSAUR [gene]...   \n",
       "997  {\"131559\": [[\"BtuB gene(s) [gene]\"], [\"BtuC ge...   \n",
       "998  {\"99091\": [[\"Genome sequencing of O. fragrans ...   \n",
       "999  {\"65306\": [[\"**CO/NF-Y complex [protein comple...   \n",
       "\n",
       "                                              Output>1  one_item  \\\n",
       "0                                        {\"58963\": []}      True   \n",
       "1    {\"22775\": [[\"**transportation [phenotype]**\", ...     False   \n",
       "2    {\"22996\": [[\"**transpiration and stomatal cond...     False   \n",
       "3                                       {\"100116\": []}      True   \n",
       "4    {\"133255\": [[\"**PLSP2A [gene]**\", \"PLSP2A gene...     False   \n",
       "..                                                 ...       ...   \n",
       "995  {\"31615\": [[\"auxin-dependent inhibition of end...     False   \n",
       "996  {\"157227\": [[\"ZmSAUR1 (Zea mays SAURs) [gene]\"...     False   \n",
       "997                                     {\"131559\": []}      True   \n",
       "998  {\"99091\": [[\"**O. fragrans 'Liuyejingui' (OFL)...     False   \n",
       "999  {\"65306\": [[\"**CO/NF-Y complex [protein comple...     False   \n",
       "\n",
       "                                           user_prompt  \\\n",
       "0    Great job, that looks correct because each res...   \n",
       "1    Great job, the clusters appear to be correctly...   \n",
       "2    I think you are wrong. In the current clusteri...   \n",
       "3    This looks correct to me, as the grouping foll...   \n",
       "4    Great job, the clustering is correct. The gene...   \n",
       "..                                                 ...   \n",
       "995  I think you are wrong. The grouping of the end...   \n",
       "996  This looks correct to me. Each gene entity is ...   \n",
       "997  Great job, that looks correct to me. Each gene...   \n",
       "998  Great job, the clustering is correct. You have...   \n",
       "999  This looks correct to me, output the clusters ...   \n",
       "\n",
       "                                  o3_corrected_cluster  is_correct  \n",
       "0                                                   []        True  \n",
       "1    [[\"**transportation [phenotype]**\", \"transport...        True  \n",
       "2    [[\"**transpiration and stomatal conductance va...       False  \n",
       "3                                                   []        True  \n",
       "4    [[\"**PLSP2A [gene]**\", \"PLSP2A gene(s) [gene]\"...        True  \n",
       "..                                                 ...         ...  \n",
       "995  [[\"**auxin-regulated endocytosis [process]**\",...       False  \n",
       "996  [[\"ZmSAUR1 (Zea mays SAURs) [gene]\", \"ZmSAUR1 ...        True  \n",
       "997                                                 []        True  \n",
       "998  [[\"**O. fragrans 'Liuyejingui' (OFL) [organism...        True  \n",
       "999  [[\"**CO/NF-Y complex [protein complex]**\", \"CO...        True  \n",
       "\n",
       "[1000 rows x 8 columns]"
      ]
     },
     "execution_count": 32,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Doing the analysis\n",
    "\n",
    "#### Non empty: \n",
    "1. initial and corrected clusters - checking if lengths are equal\n",
    "2. initial and corrected clusters - checking if corrected cluster are shorter than initials\n",
    "3. initial and corrected clusters - checking if corrected cluster are longer than initials\n",
    "#### Empty:\n",
    "1. Check if both initial and corrected clusters are empty."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 33,
   "metadata": {},
   "outputs": [],
   "source": [
    "\n",
    "\n",
    "import ast\n",
    "\n",
    "def get_list_length(output):\n",
    "    # If the output is a string representation of a dictionary, convert it on the fly.\n",
    "    if isinstance(output, str):\n",
    "        output = ast.literal_eval(output)\n",
    "    # Get the length of the list in the first value of the dictionary.\n",
    "    return len(list(output.values())[0])\n",
    "\n",
    "df_nonempty_init_correct_same_length = df[df.apply(lambda row: get_list_length(row['Output']) == len(ast.literal_eval(row['o3_corrected_cluster'])), axis=1)]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 34,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "18"
      ]
     },
     "execution_count": 34,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "len(df_nonempty_init_correct_same_length)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 35,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "982"
      ]
     },
     "execution_count": 35,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "#get the rows where the Input is non empty list and o3 corrected cluster length is fewer than Input length\n",
    "df_nonempty_init_correct_fewer_length = df[df.apply(lambda row: get_list_length(row['Output']) > len(ast.literal_eval(row['o3_corrected_cluster'])), axis=1)]\n",
    "len(df_nonempty_init_correct_fewer_length)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 36,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "0"
      ]
     },
     "execution_count": 36,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "#get the rows where the Input is non empty list and o3 corrected cluster length is greater than Input length\n",
    "df_nonempty_init_correct_more_length = df[df.apply(lambda row: get_list_length(row['Output']) < len(ast.literal_eval(row['o3_corrected_cluster'])), axis=1)]\n",
    "len(df_nonempty_init_correct_more_length)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 37,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "0"
      ]
     },
     "execution_count": 37,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "#get the rows where the Input is empty list and o3 corrected cluster length is also empty\n",
    "df_empty_init_correct_also_empty = df[df.apply(lambda row: get_list_length(row['Output']) == 0 and len(ast.literal_eval(row['o3_corrected_cluster'])) == 0, axis=1)]\n",
    "len(df_empty_init_correct_also_empty)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 38,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Entity Name</th>\n",
       "      <th>Input</th>\n",
       "      <th>Output</th>\n",
       "      <th>Output&gt;1</th>\n",
       "      <th>one_item</th>\n",
       "      <th>user_prompt</th>\n",
       "      <th>o3_corrected_cluster</th>\n",
       "      <th>is_correct</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>58963</td>\n",
       "      <td>{\"58963\": [\"Barrena et al. [researcher]\", \"Gar...</td>\n",
       "      <td>{\"58963\": [[\"Gardea-Torresdey et al. [research...</td>\n",
       "      <td>{\"58963\": []}</td>\n",
       "      <td>True</td>\n",
       "      <td>Great job, that looks correct because each res...</td>\n",
       "      <td>[]</td>\n",
       "      <td>True</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>22775</td>\n",
       "      <td>{\"22775\": [\"physiological transport of macro-m...</td>\n",
       "      <td>{\"22775\": [[\"substance that could be transloca...</td>\n",
       "      <td>{\"22775\": [[\"**transportation [phenotype]**\", ...</td>\n",
       "      <td>False</td>\n",
       "      <td>Great job, the clusters appear to be correctly...</td>\n",
       "      <td>[[\"**transportation [phenotype]**\", \"transport...</td>\n",
       "      <td>True</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>22996</td>\n",
       "      <td>{\"22996\": [\"number of open stomata and amount ...</td>\n",
       "      <td>{\"22996\": [[\"**transpiration and stomatal cond...</td>\n",
       "      <td>{\"22996\": [[\"**transpiration and stomatal cond...</td>\n",
       "      <td>False</td>\n",
       "      <td>I think you are wrong. In the current clusteri...</td>\n",
       "      <td>[[\"**transpiration and stomatal conductance va...</td>\n",
       "      <td>False</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>100116</td>\n",
       "      <td>{\"100116\": [\"Clade SH [clade]\", \"Clade M [clad...</td>\n",
       "      <td>{\"100116\": [[\"Clade M [clade]\"], [\"Clade SL [c...</td>\n",
       "      <td>{\"100116\": []}</td>\n",
       "      <td>True</td>\n",
       "      <td>This looks correct to me, as the grouping foll...</td>\n",
       "      <td>[]</td>\n",
       "      <td>True</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>133255</td>\n",
       "      <td>{\"133255\": [\"PSPTO3648 [gene]\", \"LPP\\u10b2 [ge...</td>\n",
       "      <td>{\"133255\": [[\"**PLSP2A [gene]**\", \"PLSP2A gene...</td>\n",
       "      <td>{\"133255\": [[\"**PLSP2A [gene]**\", \"PLSP2A gene...</td>\n",
       "      <td>False</td>\n",
       "      <td>Great job, the clustering is correct. The gene...</td>\n",
       "      <td>[[\"**PLSP2A [gene]**\", \"PLSP2A gene(s) [gene]\"...</td>\n",
       "      <td>True</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>995</th>\n",
       "      <td>31615</td>\n",
       "      <td>{\"31615\": [\"auxin-mediated modulation of the m...</td>\n",
       "      <td>{\"31615\": [[\"auxin-mediated modulation of the ...</td>\n",
       "      <td>{\"31615\": [[\"auxin-dependent inhibition of end...</td>\n",
       "      <td>False</td>\n",
       "      <td>I think you are wrong. The grouping of the end...</td>\n",
       "      <td>[[\"**auxin-regulated endocytosis [process]**\",...</td>\n",
       "      <td>False</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>996</th>\n",
       "      <td>157227</td>\n",
       "      <td>{\"157227\": [\"GmSAUR [gene]\", \"ZmSAUR1 (Zea may...</td>\n",
       "      <td>{\"157227\": [[\"GmSAUR [gene]\"], [\"StSAUR [gene]...</td>\n",
       "      <td>{\"157227\": [[\"ZmSAUR1 (Zea mays SAURs) [gene]\"...</td>\n",
       "      <td>False</td>\n",
       "      <td>This looks correct to me. Each gene entity is ...</td>\n",
       "      <td>[[\"ZmSAUR1 (Zea mays SAURs) [gene]\", \"ZmSAUR1 ...</td>\n",
       "      <td>True</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>997</th>\n",
       "      <td>131559</td>\n",
       "      <td>{\"131559\": [\"BtuC gene(s) [gene]\", \"cytochrome...</td>\n",
       "      <td>{\"131559\": [[\"BtuB gene(s) [gene]\"], [\"BtuC ge...</td>\n",
       "      <td>{\"131559\": []}</td>\n",
       "      <td>True</td>\n",
       "      <td>Great job, that looks correct to me. Each gene...</td>\n",
       "      <td>[]</td>\n",
       "      <td>True</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>998</th>\n",
       "      <td>99091</td>\n",
       "      <td>{\"99091\": [\"Genome sequencing of O. fragrans '...</td>\n",
       "      <td>{\"99091\": [[\"Genome sequencing of O. fragrans ...</td>\n",
       "      <td>{\"99091\": [[\"**O. fragrans 'Liuyejingui' (OFL)...</td>\n",
       "      <td>False</td>\n",
       "      <td>Great job, the clustering is correct. You have...</td>\n",
       "      <td>[[\"**O. fragrans 'Liuyejingui' (OFL) [organism...</td>\n",
       "      <td>True</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>999</th>\n",
       "      <td>65306</td>\n",
       "      <td>{\"65306\": [\"NF-TOC1 [protein complex]\", \"Hd1/N...</td>\n",
       "      <td>{\"65306\": [[\"**CO/NF-Y complex [protein comple...</td>\n",
       "      <td>{\"65306\": [[\"**CO/NF-Y complex [protein comple...</td>\n",
       "      <td>False</td>\n",
       "      <td>This looks correct to me, output the clusters ...</td>\n",
       "      <td>[[\"**CO/NF-Y complex [protein complex]**\", \"CO...</td>\n",
       "      <td>True</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>1000 rows × 8 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "     Entity Name                                              Input  \\\n",
       "0          58963  {\"58963\": [\"Barrena et al. [researcher]\", \"Gar...   \n",
       "1          22775  {\"22775\": [\"physiological transport of macro-m...   \n",
       "2          22996  {\"22996\": [\"number of open stomata and amount ...   \n",
       "3         100116  {\"100116\": [\"Clade SH [clade]\", \"Clade M [clad...   \n",
       "4         133255  {\"133255\": [\"PSPTO3648 [gene]\", \"LPP\\u10b2 [ge...   \n",
       "..           ...                                                ...   \n",
       "995        31615  {\"31615\": [\"auxin-mediated modulation of the m...   \n",
       "996       157227  {\"157227\": [\"GmSAUR [gene]\", \"ZmSAUR1 (Zea may...   \n",
       "997       131559  {\"131559\": [\"BtuC gene(s) [gene]\", \"cytochrome...   \n",
       "998        99091  {\"99091\": [\"Genome sequencing of O. fragrans '...   \n",
       "999        65306  {\"65306\": [\"NF-TOC1 [protein complex]\", \"Hd1/N...   \n",
       "\n",
       "                                                Output  \\\n",
       "0    {\"58963\": [[\"Gardea-Torresdey et al. [research...   \n",
       "1    {\"22775\": [[\"substance that could be transloca...   \n",
       "2    {\"22996\": [[\"**transpiration and stomatal cond...   \n",
       "3    {\"100116\": [[\"Clade M [clade]\"], [\"Clade SL [c...   \n",
       "4    {\"133255\": [[\"**PLSP2A [gene]**\", \"PLSP2A gene...   \n",
       "..                                                 ...   \n",
       "995  {\"31615\": [[\"auxin-mediated modulation of the ...   \n",
       "996  {\"157227\": [[\"GmSAUR [gene]\"], [\"StSAUR [gene]...   \n",
       "997  {\"131559\": [[\"BtuB gene(s) [gene]\"], [\"BtuC ge...   \n",
       "998  {\"99091\": [[\"Genome sequencing of O. fragrans ...   \n",
       "999  {\"65306\": [[\"**CO/NF-Y complex [protein comple...   \n",
       "\n",
       "                                              Output>1  one_item  \\\n",
       "0                                        {\"58963\": []}      True   \n",
       "1    {\"22775\": [[\"**transportation [phenotype]**\", ...     False   \n",
       "2    {\"22996\": [[\"**transpiration and stomatal cond...     False   \n",
       "3                                       {\"100116\": []}      True   \n",
       "4    {\"133255\": [[\"**PLSP2A [gene]**\", \"PLSP2A gene...     False   \n",
       "..                                                 ...       ...   \n",
       "995  {\"31615\": [[\"auxin-dependent inhibition of end...     False   \n",
       "996  {\"157227\": [[\"ZmSAUR1 (Zea mays SAURs) [gene]\"...     False   \n",
       "997                                     {\"131559\": []}      True   \n",
       "998  {\"99091\": [[\"**O. fragrans 'Liuyejingui' (OFL)...     False   \n",
       "999  {\"65306\": [[\"**CO/NF-Y complex [protein comple...     False   \n",
       "\n",
       "                                           user_prompt  \\\n",
       "0    Great job, that looks correct because each res...   \n",
       "1    Great job, the clusters appear to be correctly...   \n",
       "2    I think you are wrong. In the current clusteri...   \n",
       "3    This looks correct to me, as the grouping foll...   \n",
       "4    Great job, the clustering is correct. The gene...   \n",
       "..                                                 ...   \n",
       "995  I think you are wrong. The grouping of the end...   \n",
       "996  This looks correct to me. Each gene entity is ...   \n",
       "997  Great job, that looks correct to me. Each gene...   \n",
       "998  Great job, the clustering is correct. You have...   \n",
       "999  This looks correct to me, output the clusters ...   \n",
       "\n",
       "                                  o3_corrected_cluster  is_correct  \n",
       "0                                                   []        True  \n",
       "1    [[\"**transportation [phenotype]**\", \"transport...        True  \n",
       "2    [[\"**transpiration and stomatal conductance va...       False  \n",
       "3                                                   []        True  \n",
       "4    [[\"**PLSP2A [gene]**\", \"PLSP2A gene(s) [gene]\"...        True  \n",
       "..                                                 ...         ...  \n",
       "995  [[\"**auxin-regulated endocytosis [process]**\",...       False  \n",
       "996  [[\"ZmSAUR1 (Zea mays SAURs) [gene]\", \"ZmSAUR1 ...        True  \n",
       "997                                                 []        True  \n",
       "998  [[\"**O. fragrans 'Liuyejingui' (OFL) [organism...        True  \n",
       "999  [[\"**CO/NF-Y complex [protein complex]**\", \"CO...        True  \n",
       "\n",
       "[1000 rows x 8 columns]"
      ]
     },
     "execution_count": 38,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 39,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "is_correct\n",
       "True     809\n",
       "False    191\n",
       "Name: count, dtype: int64"
      ]
     },
     "execution_count": 39,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "output_csv['is_correct'].value_counts()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 40,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Entity Name</th>\n",
       "      <th>Input</th>\n",
       "      <th>Output</th>\n",
       "      <th>Output&gt;1</th>\n",
       "      <th>one_item</th>\n",
       "      <th>user_prompt</th>\n",
       "      <th>o3_corrected_cluster</th>\n",
       "      <th>is_correct</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>58963</td>\n",
       "      <td>{\"58963\": [\"Barrena et al. [researcher]\", \"Gar...</td>\n",
       "      <td>{\"58963\": [[\"Gardea-Torresdey et al. [research...</td>\n",
       "      <td>{\"58963\": []}</td>\n",
       "      <td>True</td>\n",
       "      <td>Great job, that looks correct because each res...</td>\n",
       "      <td>[]</td>\n",
       "      <td>True</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>22775</td>\n",
       "      <td>{\"22775\": [\"physiological transport of macro-m...</td>\n",
       "      <td>{\"22775\": [[\"substance that could be transloca...</td>\n",
       "      <td>{\"22775\": [[\"**transportation [phenotype]**\", ...</td>\n",
       "      <td>False</td>\n",
       "      <td>Great job, the clusters appear to be correctly...</td>\n",
       "      <td>[[\"**transportation [phenotype]**\", \"transport...</td>\n",
       "      <td>True</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>22996</td>\n",
       "      <td>{\"22996\": [\"number of open stomata and amount ...</td>\n",
       "      <td>{\"22996\": [[\"**transpiration and stomatal cond...</td>\n",
       "      <td>{\"22996\": [[\"**transpiration and stomatal cond...</td>\n",
       "      <td>False</td>\n",
       "      <td>I think you are wrong. In the current clusteri...</td>\n",
       "      <td>[[\"**transpiration and stomatal conductance va...</td>\n",
       "      <td>False</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>100116</td>\n",
       "      <td>{\"100116\": [\"Clade SH [clade]\", \"Clade M [clad...</td>\n",
       "      <td>{\"100116\": [[\"Clade M [clade]\"], [\"Clade SL [c...</td>\n",
       "      <td>{\"100116\": []}</td>\n",
       "      <td>True</td>\n",
       "      <td>This looks correct to me, as the grouping foll...</td>\n",
       "      <td>[]</td>\n",
       "      <td>True</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>133255</td>\n",
       "      <td>{\"133255\": [\"PSPTO3648 [gene]\", \"LPP\\u10b2 [ge...</td>\n",
       "      <td>{\"133255\": [[\"**PLSP2A [gene]**\", \"PLSP2A gene...</td>\n",
       "      <td>{\"133255\": [[\"**PLSP2A [gene]**\", \"PLSP2A gene...</td>\n",
       "      <td>False</td>\n",
       "      <td>Great job, the clustering is correct. The gene...</td>\n",
       "      <td>[[\"**PLSP2A [gene]**\", \"PLSP2A gene(s) [gene]\"...</td>\n",
       "      <td>True</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>995</th>\n",
       "      <td>31615</td>\n",
       "      <td>{\"31615\": [\"auxin-mediated modulation of the m...</td>\n",
       "      <td>{\"31615\": [[\"auxin-mediated modulation of the ...</td>\n",
       "      <td>{\"31615\": [[\"auxin-dependent inhibition of end...</td>\n",
       "      <td>False</td>\n",
       "      <td>I think you are wrong. The grouping of the end...</td>\n",
       "      <td>[[\"**auxin-regulated endocytosis [process]**\",...</td>\n",
       "      <td>False</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>996</th>\n",
       "      <td>157227</td>\n",
       "      <td>{\"157227\": [\"GmSAUR [gene]\", \"ZmSAUR1 (Zea may...</td>\n",
       "      <td>{\"157227\": [[\"GmSAUR [gene]\"], [\"StSAUR [gene]...</td>\n",
       "      <td>{\"157227\": [[\"ZmSAUR1 (Zea mays SAURs) [gene]\"...</td>\n",
       "      <td>False</td>\n",
       "      <td>This looks correct to me. Each gene entity is ...</td>\n",
       "      <td>[[\"ZmSAUR1 (Zea mays SAURs) [gene]\", \"ZmSAUR1 ...</td>\n",
       "      <td>True</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>997</th>\n",
       "      <td>131559</td>\n",
       "      <td>{\"131559\": [\"BtuC gene(s) [gene]\", \"cytochrome...</td>\n",
       "      <td>{\"131559\": [[\"BtuB gene(s) [gene]\"], [\"BtuC ge...</td>\n",
       "      <td>{\"131559\": []}</td>\n",
       "      <td>True</td>\n",
       "      <td>Great job, that looks correct to me. Each gene...</td>\n",
       "      <td>[]</td>\n",
       "      <td>True</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>998</th>\n",
       "      <td>99091</td>\n",
       "      <td>{\"99091\": [\"Genome sequencing of O. fragrans '...</td>\n",
       "      <td>{\"99091\": [[\"Genome sequencing of O. fragrans ...</td>\n",
       "      <td>{\"99091\": [[\"**O. fragrans 'Liuyejingui' (OFL)...</td>\n",
       "      <td>False</td>\n",
       "      <td>Great job, the clustering is correct. You have...</td>\n",
       "      <td>[[\"**O. fragrans 'Liuyejingui' (OFL) [organism...</td>\n",
       "      <td>True</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>999</th>\n",
       "      <td>65306</td>\n",
       "      <td>{\"65306\": [\"NF-TOC1 [protein complex]\", \"Hd1/N...</td>\n",
       "      <td>{\"65306\": [[\"**CO/NF-Y complex [protein comple...</td>\n",
       "      <td>{\"65306\": [[\"**CO/NF-Y complex [protein comple...</td>\n",
       "      <td>False</td>\n",
       "      <td>This looks correct to me, output the clusters ...</td>\n",
       "      <td>[[\"**CO/NF-Y complex [protein complex]**\", \"CO...</td>\n",
       "      <td>True</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>996 rows × 8 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "     Entity Name                                              Input  \\\n",
       "0          58963  {\"58963\": [\"Barrena et al. [researcher]\", \"Gar...   \n",
       "1          22775  {\"22775\": [\"physiological transport of macro-m...   \n",
       "2          22996  {\"22996\": [\"number of open stomata and amount ...   \n",
       "3         100116  {\"100116\": [\"Clade SH [clade]\", \"Clade M [clad...   \n",
       "4         133255  {\"133255\": [\"PSPTO3648 [gene]\", \"LPP\\u10b2 [ge...   \n",
       "..           ...                                                ...   \n",
       "995        31615  {\"31615\": [\"auxin-mediated modulation of the m...   \n",
       "996       157227  {\"157227\": [\"GmSAUR [gene]\", \"ZmSAUR1 (Zea may...   \n",
       "997       131559  {\"131559\": [\"BtuC gene(s) [gene]\", \"cytochrome...   \n",
       "998        99091  {\"99091\": [\"Genome sequencing of O. fragrans '...   \n",
       "999        65306  {\"65306\": [\"NF-TOC1 [protein complex]\", \"Hd1/N...   \n",
       "\n",
       "                                                Output  \\\n",
       "0    {\"58963\": [[\"Gardea-Torresdey et al. [research...   \n",
       "1    {\"22775\": [[\"substance that could be transloca...   \n",
       "2    {\"22996\": [[\"**transpiration and stomatal cond...   \n",
       "3    {\"100116\": [[\"Clade M [clade]\"], [\"Clade SL [c...   \n",
       "4    {\"133255\": [[\"**PLSP2A [gene]**\", \"PLSP2A gene...   \n",
       "..                                                 ...   \n",
       "995  {\"31615\": [[\"auxin-mediated modulation of the ...   \n",
       "996  {\"157227\": [[\"GmSAUR [gene]\"], [\"StSAUR [gene]...   \n",
       "997  {\"131559\": [[\"BtuB gene(s) [gene]\"], [\"BtuC ge...   \n",
       "998  {\"99091\": [[\"Genome sequencing of O. fragrans ...   \n",
       "999  {\"65306\": [[\"**CO/NF-Y complex [protein comple...   \n",
       "\n",
       "                                              Output>1  one_item  \\\n",
       "0                                        {\"58963\": []}      True   \n",
       "1    {\"22775\": [[\"**transportation [phenotype]**\", ...     False   \n",
       "2    {\"22996\": [[\"**transpiration and stomatal cond...     False   \n",
       "3                                       {\"100116\": []}      True   \n",
       "4    {\"133255\": [[\"**PLSP2A [gene]**\", \"PLSP2A gene...     False   \n",
       "..                                                 ...       ...   \n",
       "995  {\"31615\": [[\"auxin-dependent inhibition of end...     False   \n",
       "996  {\"157227\": [[\"ZmSAUR1 (Zea mays SAURs) [gene]\"...     False   \n",
       "997                                     {\"131559\": []}      True   \n",
       "998  {\"99091\": [[\"**O. fragrans 'Liuyejingui' (OFL)...     False   \n",
       "999  {\"65306\": [[\"**CO/NF-Y complex [protein comple...     False   \n",
       "\n",
       "                                           user_prompt  \\\n",
       "0    Great job, that looks correct because each res...   \n",
       "1    Great job, the clusters appear to be correctly...   \n",
       "2    I think you are wrong. In the current clusteri...   \n",
       "3    This looks correct to me, as the grouping foll...   \n",
       "4    Great job, the clustering is correct. The gene...   \n",
       "..                                                 ...   \n",
       "995  I think you are wrong. The grouping of the end...   \n",
       "996  This looks correct to me. Each gene entity is ...   \n",
       "997  Great job, that looks correct to me. Each gene...   \n",
       "998  Great job, the clustering is correct. You have...   \n",
       "999  This looks correct to me, output the clusters ...   \n",
       "\n",
       "                                  o3_corrected_cluster  is_correct  \n",
       "0                                                   []        True  \n",
       "1    [[\"**transportation [phenotype]**\", \"transport...        True  \n",
       "2    [[\"**transpiration and stomatal conductance va...       False  \n",
       "3                                                   []        True  \n",
       "4    [[\"**PLSP2A [gene]**\", \"PLSP2A gene(s) [gene]\"...        True  \n",
       "..                                                 ...         ...  \n",
       "995  [[\"**auxin-regulated endocytosis [process]**\",...       False  \n",
       "996  [[\"ZmSAUR1 (Zea mays SAURs) [gene]\", \"ZmSAUR1 ...        True  \n",
       "997                                                 []        True  \n",
       "998  [[\"**O. fragrans 'Liuyejingui' (OFL) [organism...        True  \n",
       "999  [[\"**CO/NF-Y complex [protein complex]**\", \"CO...        True  \n",
       "\n",
       "[996 rows x 8 columns]"
      ]
     },
     "execution_count": 40,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# remove the rows where the Entity Name is 73713, 97119, 159677,664. I found some issues with the correctness of the clusters for these rows.\n",
    "\n",
    "df = df[~df['Entity Name'].isin([73713, 97119, 159677,664])]\n",
    "df"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 42,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>cluster_id</th>\n",
       "      <th>is_correct</th>\n",
       "      <th>user_prompt</th>\n",
       "      <th>clusters</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>58963</td>\n",
       "      <td>True</td>\n",
       "      <td>Great job, that looks correct because each res...</td>\n",
       "      <td>[]</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>22775</td>\n",
       "      <td>True</td>\n",
       "      <td>Great job, the clusters appear to be correctly...</td>\n",
       "      <td>[[\"**transportation [phenotype]**\", \"transport...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>22996</td>\n",
       "      <td>False</td>\n",
       "      <td>I think you are wrong. In the current clusteri...</td>\n",
       "      <td>[[\"**transpiration and stomatal conductance va...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>100116</td>\n",
       "      <td>True</td>\n",
       "      <td>This looks correct to me, as the grouping foll...</td>\n",
       "      <td>[]</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>133255</td>\n",
       "      <td>True</td>\n",
       "      <td>Great job, the clustering is correct. The gene...</td>\n",
       "      <td>[[\"**PLSP2A [gene]**\", \"PLSP2A gene(s) [gene]\"...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>995</th>\n",
       "      <td>31615</td>\n",
       "      <td>False</td>\n",
       "      <td>I think you are wrong. The grouping of the end...</td>\n",
       "      <td>[[\"**auxin-regulated endocytosis [process]**\",...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>996</th>\n",
       "      <td>157227</td>\n",
       "      <td>True</td>\n",
       "      <td>This looks correct to me. Each gene entity is ...</td>\n",
       "      <td>[[\"ZmSAUR1 (Zea mays SAURs) [gene]\", \"ZmSAUR1 ...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>997</th>\n",
       "      <td>131559</td>\n",
       "      <td>True</td>\n",
       "      <td>Great job, that looks correct to me. Each gene...</td>\n",
       "      <td>[]</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>998</th>\n",
       "      <td>99091</td>\n",
       "      <td>True</td>\n",
       "      <td>Great job, the clustering is correct. You have...</td>\n",
       "      <td>[[\"**O. fragrans 'Liuyejingui' (OFL) [organism...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>999</th>\n",
       "      <td>65306</td>\n",
       "      <td>True</td>\n",
       "      <td>This looks correct to me, output the clusters ...</td>\n",
       "      <td>[[\"**CO/NF-Y complex [protein complex]**\", \"CO...</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>1000 rows × 4 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "     cluster_id  is_correct  \\\n",
       "0         58963        True   \n",
       "1         22775        True   \n",
       "2         22996       False   \n",
       "3        100116        True   \n",
       "4        133255        True   \n",
       "..          ...         ...   \n",
       "995       31615       False   \n",
       "996      157227        True   \n",
       "997      131559        True   \n",
       "998       99091        True   \n",
       "999       65306        True   \n",
       "\n",
       "                                           user_prompt  \\\n",
       "0    Great job, that looks correct because each res...   \n",
       "1    Great job, the clusters appear to be correctly...   \n",
       "2    I think you are wrong. In the current clusteri...   \n",
       "3    This looks correct to me, as the grouping foll...   \n",
       "4    Great job, the clustering is correct. The gene...   \n",
       "..                                                 ...   \n",
       "995  I think you are wrong. The grouping of the end...   \n",
       "996  This looks correct to me. Each gene entity is ...   \n",
       "997  Great job, that looks correct to me. Each gene...   \n",
       "998  Great job, the clustering is correct. You have...   \n",
       "999  This looks correct to me, output the clusters ...   \n",
       "\n",
       "                                              clusters  \n",
       "0                                                   []  \n",
       "1    [[\"**transportation [phenotype]**\", \"transport...  \n",
       "2    [[\"**transpiration and stomatal conductance va...  \n",
       "3                                                   []  \n",
       "4    [[\"**PLSP2A [gene]**\", \"PLSP2A gene(s) [gene]\"...  \n",
       "..                                                 ...  \n",
       "995  [[\"**auxin-regulated endocytosis [process]**\",...  \n",
       "996  [[\"ZmSAUR1 (Zea mays SAURs) [gene]\", \"ZmSAUR1 ...  \n",
       "997                                                 []  \n",
       "998  [[\"**O. fragrans 'Liuyejingui' (OFL) [organism...  \n",
       "999  [[\"**CO/NF-Y complex [protein complex]**\", \"CO...  \n",
       "\n",
       "[1000 rows x 4 columns]"
      ]
     },
     "execution_count": 42,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "output_csv"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 46,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/var/folders/28/b8d1h3952xj9bg__2x97mt6r0000gn/T/ipykernel_64506/3995214724.py:11: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame.\n",
      "Try using .loc[row_indexer,col_indexer] = value instead\n",
      "\n",
      "See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  df['reasoning'] = df['Entity Name'].map(o3_reasoning_dict)\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Entity Name</th>\n",
       "      <th>Input</th>\n",
       "      <th>Output</th>\n",
       "      <th>Output&gt;1</th>\n",
       "      <th>one_item</th>\n",
       "      <th>user_prompt</th>\n",
       "      <th>o3_corrected_cluster</th>\n",
       "      <th>is_correct</th>\n",
       "      <th>reasoning</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>58963</td>\n",
       "      <td>{\"58963\": [\"Barrena et al. [researcher]\", \"Gar...</td>\n",
       "      <td>{\"58963\": [[\"Gardea-Torresdey et al. [research...</td>\n",
       "      <td>{\"58963\": []}</td>\n",
       "      <td>True</td>\n",
       "      <td>Great job, that looks correct because each res...</td>\n",
       "      <td>[]</td>\n",
       "      <td>True</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>22775</td>\n",
       "      <td>{\"22775\": [\"physiological transport of macro-m...</td>\n",
       "      <td>{\"22775\": [[\"substance that could be transloca...</td>\n",
       "      <td>{\"22775\": [[\"**transportation [phenotype]**\", ...</td>\n",
       "      <td>False</td>\n",
       "      <td>Great job, the clusters appear to be correctly...</td>\n",
       "      <td>[[\"**transportation [phenotype]**\", \"transport...</td>\n",
       "      <td>True</td>\n",
       "      <td>We formed clusters only when two or more entit...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>22996</td>\n",
       "      <td>{\"22996\": [\"number of open stomata and amount ...</td>\n",
       "      <td>{\"22996\": [[\"**transpiration and stomatal cond...</td>\n",
       "      <td>{\"22996\": [[\"**transpiration and stomatal cond...</td>\n",
       "      <td>False</td>\n",
       "      <td>I think you are wrong. In the current clusteri...</td>\n",
       "      <td>[[\"**transpiration and stomatal conductance va...</td>\n",
       "      <td>False</td>\n",
       "      <td>Our analysis of the stomatal conductance–relat...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>100116</td>\n",
       "      <td>{\"100116\": [\"Clade SH [clade]\", \"Clade M [clad...</td>\n",
       "      <td>{\"100116\": [[\"Clade M [clade]\"], [\"Clade SL [c...</td>\n",
       "      <td>{\"100116\": []}</td>\n",
       "      <td>True</td>\n",
       "      <td>This looks correct to me, as the grouping foll...</td>\n",
       "      <td>[]</td>\n",
       "      <td>True</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>133255</td>\n",
       "      <td>{\"133255\": [\"PSPTO3648 [gene]\", \"LPP\\u10b2 [ge...</td>\n",
       "      <td>{\"133255\": [[\"**PLSP2A [gene]**\", \"PLSP2A gene...</td>\n",
       "      <td>{\"133255\": [[\"**PLSP2A [gene]**\", \"PLSP2A gene...</td>\n",
       "      <td>False</td>\n",
       "      <td>Great job, the clustering is correct. The gene...</td>\n",
       "      <td>[[\"**PLSP2A [gene]**\", \"PLSP2A gene(s) [gene]\"...</td>\n",
       "      <td>True</td>\n",
       "      <td>In this gene list, only one multi‐entity clust...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>991</th>\n",
       "      <td>31615</td>\n",
       "      <td>{\"31615\": [\"auxin-mediated modulation of the m...</td>\n",
       "      <td>{\"31615\": [[\"auxin-mediated modulation of the ...</td>\n",
       "      <td>{\"31615\": [[\"auxin-dependent inhibition of end...</td>\n",
       "      <td>False</td>\n",
       "      <td>I think you are wrong. The grouping of the end...</td>\n",
       "      <td>[[\"**auxin-regulated endocytosis [process]**\",...</td>\n",
       "      <td>False</td>\n",
       "      <td>The three process-typed items 'auxin-dependent...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>992</th>\n",
       "      <td>157227</td>\n",
       "      <td>{\"157227\": [\"GmSAUR [gene]\", \"ZmSAUR1 (Zea may...</td>\n",
       "      <td>{\"157227\": [[\"GmSAUR [gene]\"], [\"StSAUR [gene]...</td>\n",
       "      <td>{\"157227\": [[\"ZmSAUR1 (Zea mays SAURs) [gene]\"...</td>\n",
       "      <td>False</td>\n",
       "      <td>This looks correct to me. Each gene entity is ...</td>\n",
       "      <td>[[\"ZmSAUR1 (Zea mays SAURs) [gene]\", \"ZmSAUR1 ...</td>\n",
       "      <td>True</td>\n",
       "      <td>The clustering was performed by strictly adher...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>993</th>\n",
       "      <td>131559</td>\n",
       "      <td>{\"131559\": [\"BtuC gene(s) [gene]\", \"cytochrome...</td>\n",
       "      <td>{\"131559\": [[\"BtuB gene(s) [gene]\"], [\"BtuC ge...</td>\n",
       "      <td>{\"131559\": []}</td>\n",
       "      <td>True</td>\n",
       "      <td>Great job, that looks correct to me. Each gene...</td>\n",
       "      <td>[]</td>\n",
       "      <td>True</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>994</th>\n",
       "      <td>99091</td>\n",
       "      <td>{\"99091\": [\"Genome sequencing of O. fragrans '...</td>\n",
       "      <td>{\"99091\": [[\"Genome sequencing of O. fragrans ...</td>\n",
       "      <td>{\"99091\": [[\"**O. fragrans 'Liuyejingui' (OFL)...</td>\n",
       "      <td>False</td>\n",
       "      <td>Great job, the clustering is correct. You have...</td>\n",
       "      <td>[[\"**O. fragrans 'Liuyejingui' (OFL) [organism...</td>\n",
       "      <td>True</td>\n",
       "      <td>We first examined the full phrases and noted t...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>995</th>\n",
       "      <td>65306</td>\n",
       "      <td>{\"65306\": [\"NF-TOC1 [protein complex]\", \"Hd1/N...</td>\n",
       "      <td>{\"65306\": [[\"**CO/NF-Y complex [protein comple...</td>\n",
       "      <td>{\"65306\": [[\"**CO/NF-Y complex [protein comple...</td>\n",
       "      <td>False</td>\n",
       "      <td>This looks correct to me, output the clusters ...</td>\n",
       "      <td>[[\"**CO/NF-Y complex [protein complex]**\", \"CO...</td>\n",
       "      <td>True</td>\n",
       "      <td>Two multi‐element clusters were identified bas...</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>996 rows × 9 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "     Entity Name                                              Input  \\\n",
       "0          58963  {\"58963\": [\"Barrena et al. [researcher]\", \"Gar...   \n",
       "1          22775  {\"22775\": [\"physiological transport of macro-m...   \n",
       "2          22996  {\"22996\": [\"number of open stomata and amount ...   \n",
       "3         100116  {\"100116\": [\"Clade SH [clade]\", \"Clade M [clad...   \n",
       "4         133255  {\"133255\": [\"PSPTO3648 [gene]\", \"LPP\\u10b2 [ge...   \n",
       "..           ...                                                ...   \n",
       "991        31615  {\"31615\": [\"auxin-mediated modulation of the m...   \n",
       "992       157227  {\"157227\": [\"GmSAUR [gene]\", \"ZmSAUR1 (Zea may...   \n",
       "993       131559  {\"131559\": [\"BtuC gene(s) [gene]\", \"cytochrome...   \n",
       "994        99091  {\"99091\": [\"Genome sequencing of O. fragrans '...   \n",
       "995        65306  {\"65306\": [\"NF-TOC1 [protein complex]\", \"Hd1/N...   \n",
       "\n",
       "                                                Output  \\\n",
       "0    {\"58963\": [[\"Gardea-Torresdey et al. [research...   \n",
       "1    {\"22775\": [[\"substance that could be transloca...   \n",
       "2    {\"22996\": [[\"**transpiration and stomatal cond...   \n",
       "3    {\"100116\": [[\"Clade M [clade]\"], [\"Clade SL [c...   \n",
       "4    {\"133255\": [[\"**PLSP2A [gene]**\", \"PLSP2A gene...   \n",
       "..                                                 ...   \n",
       "991  {\"31615\": [[\"auxin-mediated modulation of the ...   \n",
       "992  {\"157227\": [[\"GmSAUR [gene]\"], [\"StSAUR [gene]...   \n",
       "993  {\"131559\": [[\"BtuB gene(s) [gene]\"], [\"BtuC ge...   \n",
       "994  {\"99091\": [[\"Genome sequencing of O. fragrans ...   \n",
       "995  {\"65306\": [[\"**CO/NF-Y complex [protein comple...   \n",
       "\n",
       "                                              Output>1  one_item  \\\n",
       "0                                        {\"58963\": []}      True   \n",
       "1    {\"22775\": [[\"**transportation [phenotype]**\", ...     False   \n",
       "2    {\"22996\": [[\"**transpiration and stomatal cond...     False   \n",
       "3                                       {\"100116\": []}      True   \n",
       "4    {\"133255\": [[\"**PLSP2A [gene]**\", \"PLSP2A gene...     False   \n",
       "..                                                 ...       ...   \n",
       "991  {\"31615\": [[\"auxin-dependent inhibition of end...     False   \n",
       "992  {\"157227\": [[\"ZmSAUR1 (Zea mays SAURs) [gene]\"...     False   \n",
       "993                                     {\"131559\": []}      True   \n",
       "994  {\"99091\": [[\"**O. fragrans 'Liuyejingui' (OFL)...     False   \n",
       "995  {\"65306\": [[\"**CO/NF-Y complex [protein comple...     False   \n",
       "\n",
       "                                           user_prompt  \\\n",
       "0    Great job, that looks correct because each res...   \n",
       "1    Great job, the clusters appear to be correctly...   \n",
       "2    I think you are wrong. In the current clusteri...   \n",
       "3    This looks correct to me, as the grouping foll...   \n",
       "4    Great job, the clustering is correct. The gene...   \n",
       "..                                                 ...   \n",
       "991  I think you are wrong. The grouping of the end...   \n",
       "992  This looks correct to me. Each gene entity is ...   \n",
       "993  Great job, that looks correct to me. Each gene...   \n",
       "994  Great job, the clustering is correct. You have...   \n",
       "995  This looks correct to me, output the clusters ...   \n",
       "\n",
       "                                  o3_corrected_cluster  is_correct  \\\n",
       "0                                                   []        True   \n",
       "1    [[\"**transportation [phenotype]**\", \"transport...        True   \n",
       "2    [[\"**transpiration and stomatal conductance va...       False   \n",
       "3                                                   []        True   \n",
       "4    [[\"**PLSP2A [gene]**\", \"PLSP2A gene(s) [gene]\"...        True   \n",
       "..                                                 ...         ...   \n",
       "991  [[\"**auxin-regulated endocytosis [process]**\",...       False   \n",
       "992  [[\"ZmSAUR1 (Zea mays SAURs) [gene]\", \"ZmSAUR1 ...        True   \n",
       "993                                                 []        True   \n",
       "994  [[\"**O. fragrans 'Liuyejingui' (OFL) [organism...        True   \n",
       "995  [[\"**CO/NF-Y complex [protein complex]**\", \"CO...        True   \n",
       "\n",
       "                                             reasoning  \n",
       "0                                                  NaN  \n",
       "1    We formed clusters only when two or more entit...  \n",
       "2    Our analysis of the stomatal conductance–relat...  \n",
       "3                                                  NaN  \n",
       "4    In this gene list, only one multi‐entity clust...  \n",
       "..                                                 ...  \n",
       "991  The three process-typed items 'auxin-dependent...  \n",
       "992  The clustering was performed by strictly adher...  \n",
       "993                                                NaN  \n",
       "994  We first examined the full phrases and noted t...  \n",
       "995  Two multi‐element clusters were identified bas...  \n",
       "\n",
       "[996 rows x 9 columns]"
      ]
     },
     "execution_count": 46,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# RUN THIS CELL for V5 only\n",
    "\n",
    "#mapping cluster_id from output_csv_v2 with Entity Name add the reasoning column to df\n",
    "output_csv_v2 = pd.read_csv(\"./o3mini_val_outs/o3mini_1k_v2_again.csv\")\n",
    "#create a dictionary with key as cluster_id and value as the reasoning\n",
    "o3_reasoning_dict = {}\n",
    "for index, row in output_csv_v2.iterrows():\n",
    "    o3_reasoning_dict[row['cluster_id']] = row['reasoning']\n",
    "\n",
    "#create a new column named reasoning in df\n",
    "df['reasoning'] = df['Entity Name'].map(o3_reasoning_dict)\n",
    "df = df.reset_index(drop=True)\n",
    "df "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 47,
   "metadata": {},
   "outputs": [],
   "source": [
    "must_present = df_nonempty_init_correct_same_length['Entity Name'].to_list()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 48,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "18"
      ]
     },
     "execution_count": 48,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "len(must_present)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Entity Name</th>\n",
       "      <th>Input</th>\n",
       "      <th>Output</th>\n",
       "      <th>Output&gt;1</th>\n",
       "      <th>one_item</th>\n",
       "      <th>user_prompt</th>\n",
       "      <th>o3_corrected_cluster</th>\n",
       "      <th>is_correct</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>31465</td>\n",
       "      <td>{\"31465\": [\"stomatal number establishment [pro...</td>\n",
       "      <td>{\"31465\": [[\"**stomatal patterning [process]**...</td>\n",
       "      <td>{\"31465\": [[\"**stomatal patterning [process]**...</td>\n",
       "      <td>False</td>\n",
       "      <td>I believe the clustering is not entirely corre...</td>\n",
       "      <td>[[\"**stomatal patterning [process]**\", \"Stomat...</td>\n",
       "      <td>False</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>10291</td>\n",
       "      <td>{\"10291\": [\"phase of post-mitotic expansion [p...</td>\n",
       "      <td>{\"10291\": [[\"Extensive cell expansion [phenoty...</td>\n",
       "      <td>{\"10291\": [[\"**enhanced post-mitotic cell expa...</td>\n",
       "      <td>False</td>\n",
       "      <td>I think there is an issue with the grouping of...</td>\n",
       "      <td>[[\"**post-mitotic cell expansion [phenotype]**...</td>\n",
       "      <td>False</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>134031</td>\n",
       "      <td>{\"134031\": [\"CARDIOLIPIN SYNTHASE (CLS) [gene]...</td>\n",
       "      <td>{\"134031\": [[\"CLS [gene]\", \"**CARDIOLIPIN SYNT...</td>\n",
       "      <td>{\"134031\": [[\"CLS [gene]\", \"**CARDIOLIPIN SYNT...</td>\n",
       "      <td>False</td>\n",
       "      <td>This looks incorrect to me. The entry 'CL synt...</td>\n",
       "      <td>[[\"**CARDIOLIPIN SYNTHASE (CLS) [gene]**\", \"CL...</td>\n",
       "      <td>False</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>13588</td>\n",
       "      <td>{\"13588\": [\"shoot apical meristem size [phenot...</td>\n",
       "      <td>{\"13588\": [[\"**IFM size [phenotype]**\", \"size ...</td>\n",
       "      <td>{\"13588\": [[\"**IFM size [phenotype]**\", \"size ...</td>\n",
       "      <td>False</td>\n",
       "      <td>Great job, that looks correct to me. Each clus...</td>\n",
       "      <td>[[\"**IFM size [phenotype]**\", \"size of the inf...</td>\n",
       "      <td>True</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>152916</td>\n",
       "      <td>{\"152916\": [\"Haspin candidate gene(s) [gene]\",...</td>\n",
       "      <td>{\"152916\": [[\"GID1 homologs [gene]\"], [\"Omp85 ...</td>\n",
       "      <td>{\"152916\": [[\"**HEIP1 homologs [gene]**\", \"hom...</td>\n",
       "      <td>False</td>\n",
       "      <td>Great job, that looks correct because each gen...</td>\n",
       "      <td>[[\"**HEIP1 homologs [gene]**\", \"homologs of HE...</td>\n",
       "      <td>True</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>374</th>\n",
       "      <td>124777</td>\n",
       "      <td>{\"124777\": [\"additional lncRNAs [metabolite]\",...</td>\n",
       "      <td>{\"124777\": [[\"**long noncoding RNAs (lncRNAs) ...</td>\n",
       "      <td>{\"124777\": [[\"**long noncoding RNAs (lncRNAs) ...</td>\n",
       "      <td>False</td>\n",
       "      <td>Great job, that looks correct to me. The gener...</td>\n",
       "      <td>[[\"**long noncoding RNAs (lncRNAs) [metabolite...</td>\n",
       "      <td>True</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>375</th>\n",
       "      <td>8926</td>\n",
       "      <td>{\"8926\": [\"differences between the three genot...</td>\n",
       "      <td>{\"8926\": [[\"**morphological differences [pheno...</td>\n",
       "      <td>{\"8926\": [[\"**morphological differences [pheno...</td>\n",
       "      <td>False</td>\n",
       "      <td>I think you are wrong because the grouping of ...</td>\n",
       "      <td>[[\"**morphological differences [phenotype]**\",...</td>\n",
       "      <td>False</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>376</th>\n",
       "      <td>74789</td>\n",
       "      <td>{\"74789\": [\"Non-Race-specific Disease Resistan...</td>\n",
       "      <td>{\"74789\": [[\"**non-race specific disease resis...</td>\n",
       "      <td>{\"74789\": [[\"**non-race specific disease resis...</td>\n",
       "      <td>False</td>\n",
       "      <td>This looks incorrect to me because the groupin...</td>\n",
       "      <td>[[\"**non-race specific disease resistance (NDR...</td>\n",
       "      <td>False</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>377</th>\n",
       "      <td>49218</td>\n",
       "      <td>{\"49218\": [\"deletions on chromosome 6 [genetic...</td>\n",
       "      <td>{\"49218\": [[\"Mutants defective in chromosomal ...</td>\n",
       "      <td>{\"49218\": []}</td>\n",
       "      <td>True</td>\n",
       "      <td>Great job, that looks correct because each ent...</td>\n",
       "      <td>[]</td>\n",
       "      <td>True</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>378</th>\n",
       "      <td>69128</td>\n",
       "      <td>{\"69128\": [\"Residues 35 and 36 on the central ...</td>\n",
       "      <td>{\"69128\": [[\"Residues 35 and 36 on the central...</td>\n",
       "      <td>{\"69128\": []}</td>\n",
       "      <td>True</td>\n",
       "      <td>Great job, the items have been correctly separ...</td>\n",
       "      <td>[]</td>\n",
       "      <td>True</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>379 rows × 8 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "     Entity Name                                              Input  \\\n",
       "0          31465  {\"31465\": [\"stomatal number establishment [pro...   \n",
       "1          10291  {\"10291\": [\"phase of post-mitotic expansion [p...   \n",
       "2         134031  {\"134031\": [\"CARDIOLIPIN SYNTHASE (CLS) [gene]...   \n",
       "3          13588  {\"13588\": [\"shoot apical meristem size [phenot...   \n",
       "4         152916  {\"152916\": [\"Haspin candidate gene(s) [gene]\",...   \n",
       "..           ...                                                ...   \n",
       "374       124777  {\"124777\": [\"additional lncRNAs [metabolite]\",...   \n",
       "375         8926  {\"8926\": [\"differences between the three genot...   \n",
       "376        74789  {\"74789\": [\"Non-Race-specific Disease Resistan...   \n",
       "377        49218  {\"49218\": [\"deletions on chromosome 6 [genetic...   \n",
       "378        69128  {\"69128\": [\"Residues 35 and 36 on the central ...   \n",
       "\n",
       "                                                Output  \\\n",
       "0    {\"31465\": [[\"**stomatal patterning [process]**...   \n",
       "1    {\"10291\": [[\"Extensive cell expansion [phenoty...   \n",
       "2    {\"134031\": [[\"CLS [gene]\", \"**CARDIOLIPIN SYNT...   \n",
       "3    {\"13588\": [[\"**IFM size [phenotype]**\", \"size ...   \n",
       "4    {\"152916\": [[\"GID1 homologs [gene]\"], [\"Omp85 ...   \n",
       "..                                                 ...   \n",
       "374  {\"124777\": [[\"**long noncoding RNAs (lncRNAs) ...   \n",
       "375  {\"8926\": [[\"**morphological differences [pheno...   \n",
       "376  {\"74789\": [[\"**non-race specific disease resis...   \n",
       "377  {\"49218\": [[\"Mutants defective in chromosomal ...   \n",
       "378  {\"69128\": [[\"Residues 35 and 36 on the central...   \n",
       "\n",
       "                                              Output>1  one_item  \\\n",
       "0    {\"31465\": [[\"**stomatal patterning [process]**...     False   \n",
       "1    {\"10291\": [[\"**enhanced post-mitotic cell expa...     False   \n",
       "2    {\"134031\": [[\"CLS [gene]\", \"**CARDIOLIPIN SYNT...     False   \n",
       "3    {\"13588\": [[\"**IFM size [phenotype]**\", \"size ...     False   \n",
       "4    {\"152916\": [[\"**HEIP1 homologs [gene]**\", \"hom...     False   \n",
       "..                                                 ...       ...   \n",
       "374  {\"124777\": [[\"**long noncoding RNAs (lncRNAs) ...     False   \n",
       "375  {\"8926\": [[\"**morphological differences [pheno...     False   \n",
       "376  {\"74789\": [[\"**non-race specific disease resis...     False   \n",
       "377                                      {\"49218\": []}      True   \n",
       "378                                      {\"69128\": []}      True   \n",
       "\n",
       "                                           user_prompt  \\\n",
       "0    I believe the clustering is not entirely corre...   \n",
       "1    I think there is an issue with the grouping of...   \n",
       "2    This looks incorrect to me. The entry 'CL synt...   \n",
       "3    Great job, that looks correct to me. Each clus...   \n",
       "4    Great job, that looks correct because each gen...   \n",
       "..                                                 ...   \n",
       "374  Great job, that looks correct to me. The gener...   \n",
       "375  I think you are wrong because the grouping of ...   \n",
       "376  This looks incorrect to me because the groupin...   \n",
       "377  Great job, that looks correct because each ent...   \n",
       "378  Great job, the items have been correctly separ...   \n",
       "\n",
       "                                  o3_corrected_cluster  is_correct  \n",
       "0    [[\"**stomatal patterning [process]**\", \"Stomat...       False  \n",
       "1    [[\"**post-mitotic cell expansion [phenotype]**...       False  \n",
       "2    [[\"**CARDIOLIPIN SYNTHASE (CLS) [gene]**\", \"CL...       False  \n",
       "3    [[\"**IFM size [phenotype]**\", \"size of the inf...        True  \n",
       "4    [[\"**HEIP1 homologs [gene]**\", \"homologs of HE...        True  \n",
       "..                                                 ...         ...  \n",
       "374  [[\"**long noncoding RNAs (lncRNAs) [metabolite...        True  \n",
       "375  [[\"**morphological differences [phenotype]**\",...       False  \n",
       "376  [[\"**non-race specific disease resistance (NDR...       False  \n",
       "377                                                 []        True  \n",
       "378                                                 []        True  \n",
       "\n",
       "[379 rows x 8 columns]"
      ]
     },
     "execution_count": 102,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "#randomly sample rows from the dataframe such that number of rows where is_correct column is True is equal in number to the rows where is_correct column is False (V1, V2, V3)\n",
    "#also make sure the Entity Name must contain the must_present list\n",
    "df_correct = df[(df['is_correct'] == True)]\n",
    "df_must_present = df[df['Entity Name'].isin(must_present)]\n",
    "df_incorrect = df[df['is_correct'] == False]\n",
    "\n",
    "\n",
    "df_correct_sample = df_correct.sample(n=181, random_state=42) #change n to 50 for V1, V2\n",
    "df_incorrect_sample = df_incorrect.sample(n=180, random_state=42) #change n to 50 for V1, V2\n",
    "\n",
    "\n",
    "df_sample = pd.concat([df_correct_sample, df_incorrect_sample, df_must_present])\n",
    "df_sample = df_sample.sample(frac=1).reset_index(drop=True)\n",
    "\n",
    "df_sample\n",
    "\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 22,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Entity Name</th>\n",
       "      <th>Input</th>\n",
       "      <th>Output</th>\n",
       "      <th>Output&gt;1</th>\n",
       "      <th>one_item</th>\n",
       "      <th>user_prompt</th>\n",
       "      <th>o3_corrected_cluster</th>\n",
       "      <th>is_correct</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>58963</td>\n",
       "      <td>{\"58963\": [\"Gardea-Torresdey et al. [researche...</td>\n",
       "      <td>{\"58963\": [[\"Gardea-Torresdey et al. [research...</td>\n",
       "      <td>{\"58963\": []}</td>\n",
       "      <td>True</td>\n",
       "      <td>Great job, that looks correct because each res...</td>\n",
       "      <td>[]</td>\n",
       "      <td>True</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>22775</td>\n",
       "      <td>{\"22775\": [\"transport in planta [phenotype]\", ...</td>\n",
       "      <td>{\"22775\": [[\"substance that could be transloca...</td>\n",
       "      <td>{\"22775\": [[\"**transportation [phenotype]**\", ...</td>\n",
       "      <td>False</td>\n",
       "      <td>Great job, the clusters appear to be correctly...</td>\n",
       "      <td>[[\"**transportation [phenotype]**\", \"transport...</td>\n",
       "      <td>True</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>22996</td>\n",
       "      <td>{\"22996\": [\"stomatal conductance, transpiratio...</td>\n",
       "      <td>{\"22996\": [[\"**transpiration and stomatal cond...</td>\n",
       "      <td>{\"22996\": [[\"**transpiration and stomatal cond...</td>\n",
       "      <td>False</td>\n",
       "      <td>I think you are wrong. In the current clusteri...</td>\n",
       "      <td>[[\"**transpiration and stomatal conductance va...</td>\n",
       "      <td>False</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>100116</td>\n",
       "      <td>{\"100116\": [\"Clade M [clade]\", \"Clade U [clade...</td>\n",
       "      <td>{\"100116\": [[\"Clade M [clade]\"], [\"Clade SL [c...</td>\n",
       "      <td>{\"100116\": []}</td>\n",
       "      <td>True</td>\n",
       "      <td>This looks correct to me, as the grouping foll...</td>\n",
       "      <td>[]</td>\n",
       "      <td>True</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>133255</td>\n",
       "      <td>{\"133255\": [\"LPE1 [gene]\", \"SUPPRESSOR OF LLP1...</td>\n",
       "      <td>{\"133255\": [[\"**PLSP2A [gene]**\", \"PLSP2A gene...</td>\n",
       "      <td>{\"133255\": [[\"**PLSP2A [gene]**\", \"PLSP2A gene...</td>\n",
       "      <td>False</td>\n",
       "      <td>Great job, the clustering is correct. The gene...</td>\n",
       "      <td>[[\"**PLSP2A [gene]**\", \"PLSP2A gene(s) [gene]\"...</td>\n",
       "      <td>True</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>995</th>\n",
       "      <td>31615</td>\n",
       "      <td>{\"31615\": [\"auxin-regulated endocytosis [proce...</td>\n",
       "      <td>{\"31615\": [[\"auxin-mediated modulation of the ...</td>\n",
       "      <td>{\"31615\": [[\"auxin-dependent inhibition of end...</td>\n",
       "      <td>False</td>\n",
       "      <td>I think you are wrong. The grouping of the end...</td>\n",
       "      <td>[[\"**auxin-regulated endocytosis [process]**\",...</td>\n",
       "      <td>False</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>996</th>\n",
       "      <td>157227</td>\n",
       "      <td>{\"157227\": [\"GmSAUR [gene]\", \"SlSAUR1-like [ge...</td>\n",
       "      <td>{\"157227\": [[\"GmSAUR [gene]\"], [\"StSAUR [gene]...</td>\n",
       "      <td>{\"157227\": [[\"ZmSAUR1 (Zea mays SAURs) [gene]\"...</td>\n",
       "      <td>False</td>\n",
       "      <td>This looks correct to me. Each gene entity is ...</td>\n",
       "      <td>[[\"ZmSAUR1 (Zea mays SAURs) [gene]\", \"ZmSAUR1 ...</td>\n",
       "      <td>True</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>997</th>\n",
       "      <td>131559</td>\n",
       "      <td>{\"131559\": [\"BtuC gene(s) [gene]\", \"cytochrome...</td>\n",
       "      <td>{\"131559\": [[\"BtuB gene(s) [gene]\"], [\"BtuC ge...</td>\n",
       "      <td>{\"131559\": []}</td>\n",
       "      <td>True</td>\n",
       "      <td>Great job, that looks correct to me. Each gene...</td>\n",
       "      <td>[]</td>\n",
       "      <td>True</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>998</th>\n",
       "      <td>99091</td>\n",
       "      <td>{\"99091\": [\"OFL [organism]\", \"D. oleifera geno...</td>\n",
       "      <td>{\"99091\": [[\"Genome sequencing of O. fragrans ...</td>\n",
       "      <td>{\"99091\": [[\"**O. fragrans 'Liuyejingui' (OFL)...</td>\n",
       "      <td>False</td>\n",
       "      <td>Great job, the clustering is correct. You have...</td>\n",
       "      <td>[[\"**O. fragrans 'Liuyejingui' (OFL) [organism...</td>\n",
       "      <td>True</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>999</th>\n",
       "      <td>65306</td>\n",
       "      <td>{\"65306\": [\"Hd1/NF-Y complexes [protein comple...</td>\n",
       "      <td>{\"65306\": [[\"**CO/NF-Y complex [protein comple...</td>\n",
       "      <td>{\"65306\": [[\"**CO/NF-Y complex [protein comple...</td>\n",
       "      <td>False</td>\n",
       "      <td>This looks correct to me, output the clusters ...</td>\n",
       "      <td>[[\"**CO/NF-Y complex [protein complex]**\", \"CO...</td>\n",
       "      <td>True</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>996 rows × 8 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "     Entity Name                                              Input  \\\n",
       "0          58963  {\"58963\": [\"Gardea-Torresdey et al. [researche...   \n",
       "1          22775  {\"22775\": [\"transport in planta [phenotype]\", ...   \n",
       "2          22996  {\"22996\": [\"stomatal conductance, transpiratio...   \n",
       "3         100116  {\"100116\": [\"Clade M [clade]\", \"Clade U [clade...   \n",
       "4         133255  {\"133255\": [\"LPE1 [gene]\", \"SUPPRESSOR OF LLP1...   \n",
       "..           ...                                                ...   \n",
       "995        31615  {\"31615\": [\"auxin-regulated endocytosis [proce...   \n",
       "996       157227  {\"157227\": [\"GmSAUR [gene]\", \"SlSAUR1-like [ge...   \n",
       "997       131559  {\"131559\": [\"BtuC gene(s) [gene]\", \"cytochrome...   \n",
       "998        99091  {\"99091\": [\"OFL [organism]\", \"D. oleifera geno...   \n",
       "999        65306  {\"65306\": [\"Hd1/NF-Y complexes [protein comple...   \n",
       "\n",
       "                                                Output  \\\n",
       "0    {\"58963\": [[\"Gardea-Torresdey et al. [research...   \n",
       "1    {\"22775\": [[\"substance that could be transloca...   \n",
       "2    {\"22996\": [[\"**transpiration and stomatal cond...   \n",
       "3    {\"100116\": [[\"Clade M [clade]\"], [\"Clade SL [c...   \n",
       "4    {\"133255\": [[\"**PLSP2A [gene]**\", \"PLSP2A gene...   \n",
       "..                                                 ...   \n",
       "995  {\"31615\": [[\"auxin-mediated modulation of the ...   \n",
       "996  {\"157227\": [[\"GmSAUR [gene]\"], [\"StSAUR [gene]...   \n",
       "997  {\"131559\": [[\"BtuB gene(s) [gene]\"], [\"BtuC ge...   \n",
       "998  {\"99091\": [[\"Genome sequencing of O. fragrans ...   \n",
       "999  {\"65306\": [[\"**CO/NF-Y complex [protein comple...   \n",
       "\n",
       "                                              Output>1  one_item  \\\n",
       "0                                        {\"58963\": []}      True   \n",
       "1    {\"22775\": [[\"**transportation [phenotype]**\", ...     False   \n",
       "2    {\"22996\": [[\"**transpiration and stomatal cond...     False   \n",
       "3                                       {\"100116\": []}      True   \n",
       "4    {\"133255\": [[\"**PLSP2A [gene]**\", \"PLSP2A gene...     False   \n",
       "..                                                 ...       ...   \n",
       "995  {\"31615\": [[\"auxin-dependent inhibition of end...     False   \n",
       "996  {\"157227\": [[\"ZmSAUR1 (Zea mays SAURs) [gene]\"...     False   \n",
       "997                                     {\"131559\": []}      True   \n",
       "998  {\"99091\": [[\"**O. fragrans 'Liuyejingui' (OFL)...     False   \n",
       "999  {\"65306\": [[\"**CO/NF-Y complex [protein comple...     False   \n",
       "\n",
       "                                           user_prompt  \\\n",
       "0    Great job, that looks correct because each res...   \n",
       "1    Great job, the clusters appear to be correctly...   \n",
       "2    I think you are wrong. In the current clusteri...   \n",
       "3    This looks correct to me, as the grouping foll...   \n",
       "4    Great job, the clustering is correct. The gene...   \n",
       "..                                                 ...   \n",
       "995  I think you are wrong. The grouping of the end...   \n",
       "996  This looks correct to me. Each gene entity is ...   \n",
       "997  Great job, that looks correct to me. Each gene...   \n",
       "998  Great job, the clustering is correct. You have...   \n",
       "999  This looks correct to me, output the clusters ...   \n",
       "\n",
       "                                  o3_corrected_cluster  is_correct  \n",
       "0                                                   []        True  \n",
       "1    [[\"**transportation [phenotype]**\", \"transport...        True  \n",
       "2    [[\"**transpiration and stomatal conductance va...       False  \n",
       "3                                                   []        True  \n",
       "4    [[\"**PLSP2A [gene]**\", \"PLSP2A gene(s) [gene]\"...        True  \n",
       "..                                                 ...         ...  \n",
       "995  [[\"**auxin-regulated endocytosis [process]**\",...       False  \n",
       "996  [[\"ZmSAUR1 (Zea mays SAURs) [gene]\", \"ZmSAUR1 ...        True  \n",
       "997                                                 []        True  \n",
       "998  [[\"**O. fragrans 'Liuyejingui' (OFL) [organism...        True  \n",
       "999  [[\"**CO/NF-Y complex [protein complex]**\", \"CO...        True  \n",
       "\n",
       "[996 rows x 8 columns]"
      ]
     },
     "execution_count": 22,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "#Sample 100 for training and 25 for testing with random state 42 (V3)\n",
    "df_train = df_sample.sample(n=350, random_state=42)\n",
    "df_test = df_sample[~df_sample.index.isin(df_train.index)]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 49,
   "metadata": {},
   "outputs": [],
   "source": [
    "df = df.reset_index(drop=True)\n",
    "#take 960 rows randomly from df (For V4)\n",
    "df_train = df.sample(n=960, random_state=42)\n",
    "df_test = df[~df.index.isin(df_train.index)]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 50,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "(960, 36)"
      ]
     },
     "execution_count": 50,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "len(df_train), len(df_test)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 51,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "996"
      ]
     },
     "execution_count": 51,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "len(df)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 52,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "is_correct\n",
       "True     779\n",
       "False    181\n",
       "Name: count, dtype: int64"
      ]
     },
     "execution_count": 52,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df_train['is_correct'].value_counts()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 53,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "is_correct\n",
       "True     29\n",
       "False     7\n",
       "Name: count, dtype: int64"
      ]
     },
     "execution_count": 53,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df_test['is_correct'].value_counts()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "# For V1,V2,V3,V4\n",
    "\n",
    "detailed_description = \"\"\"Here are two example inputs, reasoning for clustering, and the corresponding outputs to help you understand the task better:\n",
    "Input-1\n",
    "{\n",
    "  \"0\": '['Meiotic block [phenotype]', 'early arrest [phenotype]', 'arrest of meiotic progression in anaphase II [phenotype]', 'arrested zygotic divisions [phenotype]', 'meiotic arrest [phenotype]', 'delayed/arrested meiosis [phenotype]', 'pachytene arrest [phenotype]', 'block in meiosis prophase I [phenotype]', 'male meiosis until the end of the first division [phenotype]', 'meiotic prophase arrest [phenotype]', 'arrest in late prophase I [phenotype]', 'arrest at the end of meiosis I [phenotype]', 'leptotene arrest [phenotype]', 'meiosis I arrest [phenotype]', 'arresting the first mitosis during gametogenesis [phenotype]', 'absence of meiotic arrest [phenotype]', 'meiotic arrest phenotype [phenotype]', 'meiotic arrest at telophase I [phenotype]', 'termination of meiosis after anaphase I [phenotype]', 'premature termination of meiosis after anaphase I [phenotype]', 'mitotic arrest during female gametogenesis [phenotype]', 'arrest after meiosis I [phenotype]', 'meiotic arrest at pachytene [phenotype]', 'arrested endosperm nuclear divisions [phenotype]', 'meiotic arrest in anaphase II [phenotype]', 'meiotic division stop [phenotype]', 'arrest of the first mitotic division in gametogenesis [phenotype]', 'FNM half-stop [phenotype]', 'males arresting in the middle of prophase I [phenotype]', 'arresting prior to the first mitotic division [phenotype]']'\n",
    "}\n",
    "Input-1 → Output-1 [REASONING]\n",
    "\n",
    "1) Generic Meiotic Arrest\n",
    "\n",
    "Cluster:\n",
    "[ \"Meiotic block [phenotype]\", \"**meiotic arrest [phenotype]**\", \"meiotic arrest phenotype [phenotype]\", \"meiotic division stop [phenotype]\" ]\n",
    "• Incorrect: Grouping these separately as if they refer to different stages would obscure their identical meaning.\n",
    "• Correct: Recognize they all broadly indicate a total halt in meiosis at an unspecified stage.\n",
    "• Important: These terms are functionally synonymous, capturing any generic failure of meiotic progression.\n",
    "• Show in the output?: Yes, because the length of cluster is greater than 1.\n",
    "• Cluster representative: I selected 'meiotic arrest [phenotype]' as the representative because it is the most concise and commonly used term to describe a complete halt in the meiotic process. Unlike the other phrases which include additional descriptors or less formal wording, 'meiotic arrest' clearly and unambiguously captures the essential biological event, making it the best exemplar for the cluster.\n",
    "\n",
    "\n",
    "2) Delayed / Arrested Meiosis\n",
    "\n",
    "Cluster:\n",
    "[ \"delayed/arrested meiosis [phenotype]\" ]\n",
    "• Incorrect: Merging it with generic meiotic arrest overlooks the “delayed” aspect, which implies partial progression.\n",
    "• Correct: Keep it separate because it specifies a slow or partial block before a final stall.\n",
    "• Important: “Delayed” suggests some chromosomes or cells proceed further than in an outright immediate block.\n",
    "• Show in the output?: No, because the length of cluster is equal to 1.\n",
    "\n",
    "3) Absence of Meiotic Arrest\n",
    "\n",
    "Cluster:\n",
    "[ \"absence of meiotic arrest [phenotype]\" ]\n",
    "• Incorrect: Combining with any arrest group would contradict its meaning.\n",
    "• Correct: Maintain it as the negative counterpart, meaning no block occurs.\n",
    "• Important: This phenotype is crucial for comparisons, showing normal meiotic completion instead of a stoppage.\n",
    "• Show in the output?: No, because the length of cluster is equal to 1.\n",
    "\n",
    "4) Prophase I Arrest (Broad)\n",
    "\n",
    "Cluster:\n",
    "[ \"block in meiosis prophase I [phenotype]\", \"meiotic prophase arrest [phenotype]\" ]\n",
    "• Incorrect: Splitting them into sub-sub-stages if the label doesn’t specify.\n",
    "• Correct: They both emphasize an arrest somewhere in prophase I, without detailing the exact sub-stage (leptotene, pachytene, etc.).\n",
    "• Important: This captures a prophase I blockade in general, distinct from specific sub-stage arrests.\n",
    "• Show in the output?: Yes, because the length of cluster is greater than 1.\n",
    "• Cluster representative: I selected 'meiotic prophase arrest [phenotype]' as the representative because it succinctly captures the core biological event of an arrest occurring during prophase, without the additional wording found in the alternative. This clarity makes it the most direct and precise exemplar for the cluster.\n",
    "\n",
    "\n",
    "5) Male Arrest in Mid–Prophase I\n",
    "\n",
    "Cluster:\n",
    "[ \"males arresting in the middle of prophase I [phenotype]\" ]\n",
    "• Incorrect: Mixing with general prophase I arrest loses the male-specific nature and midpoint detail.\n",
    "• Correct: Keep it unique because it adds a sex specification (male) and timing (mid-prophase I).\n",
    "• Important: This addresses sex-specific contexts where the XY body or other male meiotic events fail around zygotene/pachytene.\n",
    "• Show in the output?: No, because the length of cluster is equal to 1.\n",
    "\n",
    "6) Early Arrest (Undefined Sub-Stage)\n",
    "\n",
    "Cluster:\n",
    "[ \"early arrest [phenotype]\" ]\n",
    "• Incorrect: Equating “early” to a named stage like leptotene or zygotene.\n",
    "• Correct: It must stand alone since it lacks a formal sub-stage but implies an initial block.\n",
    "• Important: The label indicates an arrest that happens before mid- or late-stage phenomena but is otherwise unspecified.\n",
    "• Show in the output?: No, because the length of cluster is equal to 1.\n",
    "\n",
    "7) Leptotene Arrest\n",
    "\n",
    "Cluster:\n",
    "[ \"leptotene arrest [phenotype]\" ]\n",
    "• Incorrect: Merging it with “early arrest” would lose the sub-stage clarity.\n",
    "• Correct: “Leptotene” is a well-defined earliest sub-stage of prophase I, deserving its own node.\n",
    "• Important: This precisely pinpoints where chromosomes start to condense yet fail to progress.\n",
    "• Show in the output?: No, because the length of cluster is equal to 1.\n",
    "\n",
    "8) Pachytene Arrest (Synonyms)\n",
    "\n",
    "Cluster:\n",
    "[ \"**pachytene arrest [phenotype]**\", \"meiotic arrest at pachytene [phenotype]\" ]\n",
    "• Incorrect: Splitting these two would ignore that they describe the exact same block.\n",
    "• Correct: They both name the pachytene sub-stage, so they cluster together.\n",
    "• Important: Pachytene is when homologs are fully synapsed, so arrest here is distinct from earlier or later prophase I phases.\n",
    "• Show in the output?: Yes, because the length of cluster greater than 1.\n",
    "• Cluster representative: I selected 'pachytene arrest [phenotype]' as the representative because it is more concise and directly highlights the specific stage of meiotic arrest without additional qualifiers. This succinct phrasing clearly captures the biological event at the pachytene stage, making it the best exemplar for the cluster.\n",
    "\n",
    "9) Late Prophase I Arrest\n",
    "\n",
    "Cluster:\n",
    "[ \"arrest in late prophase I [phenotype]\" ]\n",
    "• Incorrect: Grouping with general prophase I arrests would lose the “late” distinction.\n",
    "• Correct: Keep it separate because it suggests diplotene or diakinesis sub-stages.\n",
    "• Important: Identifies that the block occurs after chromosome synapsis (pachytene) but before metaphase I.\n",
    "• Show in the output?: No, because the length of cluster is equal to 1.\n",
    "\n",
    "10) Meiosis I Arrest (Broad)\n",
    "\n",
    "Cluster:\n",
    "[ \"meiosis I arrest [phenotype]\" ]\n",
    "• Incorrect: Conflating with prophase I or end-of-meiosis-I arrests.\n",
    "• Correct: This label is intentionally broad for a block anywhere in the entire first meiotic division.\n",
    "• Important: Distinct from narrower arrests at anaphase I or telophase I.\n",
    "• Show in the output?: No, because the length of cluster is equal to 1.\n",
    "\n",
    "11) End-of-Meiosis-I Arrest\n",
    "\n",
    "Cluster:\n",
    "[ \"arrest at the end of meiosis I [phenotype]\", \"arrest after meiosis I [phenotype]\", \"**meiotic arrest at telophase I [phenotype]**\" ]\n",
    "• Incorrect: Mixing with generic “meiosis I arrest” might obscure that these specifically reach telophase I or just beyond.\n",
    "• Correct: All describe an arrest that specifically coincides with or follows telophase I.\n",
    "• Important: They finish prophase–anaphase I but fail to transition into or complete meiosis II.\n",
    "• Show in the output?: Yes, because the length of cluster is greater than 1.\n",
    "• Cluster representative: I selected 'meiotic arrest at telophase I [phenotype]' as the representative because it explicitly specifies the stage of arrest, leaving no ambiguity about the timing within meiosis. By clearly indicating telophase I, it provides a precise and biologically accurate descriptor compared to the more ambiguous alternatives present in the cluster.\n",
    "\n",
    "12) Male Meiosis Until End of First Division\n",
    "\n",
    "Cluster:\n",
    "[ \"male meiosis until the end of the first division [phenotype]\" ]\n",
    "• Incorrect: Merging with “arrest at the end of meiosis I” would ignore the explicit mention of male gametogenesis.\n",
    "• Correct: It parallels an end-of-meiosis-I block but is sex-specific.\n",
    "• Important: Reflects male-specific phenotypes where meiosis I completes in a partial sense but doesn’t proceed to meiosis II.\n",
    "• Show in the output?: No, because the length of cluster is equal to 1.\n",
    "\n",
    "13) Post–Anaphase I Termination\n",
    "\n",
    "Cluster:\n",
    "[ \"**termination of meiosis after anaphase I [phenotype]**\", \"premature termination of meiosis after anaphase I [phenotype]\" ]\n",
    "• Incorrect: Combining with end-of-meiosis-I arrests (telophase I) might overlook the specific time point (right after anaphase I).\n",
    "• Correct: These highlight that meiosis halts immediately following homolog separation in anaphase I.\n",
    "• Important: “Premature termination” still implies the same staging (post-anaphase I), so they cluster together.\n",
    "• Show in the output?: Yes, because the length of cluster is greater than 1.\n",
    "• Cluster representative: I selected 'termination of meiosis after anaphase I [phenotype]' as the representative because it provides a clear and concise description of the cessation of meiosis immediately following anaphase I. The absence of the qualifier 'premature' avoids additional nuance regarding timing, making it a more universally applicable and straightforward term to represent the cluster.\n",
    "\n",
    "14) Anaphase II Arrest (Synonyms)\n",
    "\n",
    "Cluster:\n",
    "[ \"arrest of meiotic progression in anaphase II [phenotype]\", \"**meiotic arrest in anaphase II [phenotype]**\" ]\n",
    "• Incorrect: Grouping with anaphase I or telophase I arrests would misrepresent the division stage.\n",
    "• Correct: Both pinpoint the second meiotic anaphase, so they are true synonyms.\n",
    "• Important: This arrest means meiosis I completed successfully, but the cell fails during separation of sister chromatids.\n",
    "• Show in the output?: Yes, because the length of cluster is greater than 1.\n",
    "• Cluster representative: I selected 'meiotic arrest in anaphase II [phenotype]' as the representative because it succinctly and directly identifies the specific phase (anaphase II) at which the arrest occurs. Its concise phrasing avoids unnecessary complexity, making it the clearest descriptor of the biological event in this cluster.\n",
    "\n",
    "15) Arrested Zygotic Divisions\n",
    "\n",
    "Cluster:\n",
    "[ \"arrested zygotic divisions [phenotype]\" ]\n",
    "• Incorrect: Folding into meiotic blocks misses that zygotic divisions are post-fertilization mitoses.\n",
    "• Correct: Keep separate, as this block is in the earliest embryo after fertilization.\n",
    "• Important: Distinguishing embryonic arrests from gametogenic or meiotic ones is crucial in developmental contexts.\n",
    "• Show in the output?: No, because the length of cluster is equal to 1.\n",
    "\n",
    "16) Arrested Endosperm Nuclear Divisions\n",
    "\n",
    "Cluster:\n",
    "[ \"arrested endosperm nuclear divisions [phenotype]\" ]\n",
    "• Incorrect: Combining with zygotic divisions lumps distinct post-fertilization tissues (embryo vs. endosperm).\n",
    "• Correct: Endosperm is a separate tissue formed post-fertilization (often triploid), so it merits its own category.\n",
    "• Important: In many plants, endosperm divides separately, so an arrest here is unique from zygotic embryonic arrest.\n",
    "• Show in the output: No, because the length of cluster is equal to 1.\n",
    "\n",
    "17) First Mitotic Division in Gametogenesis (Synonyms)\n",
    "\n",
    "Cluster:\n",
    "[ \"arresting the first mitosis during gametogenesis [phenotype]\", \"**arrest of the first mitotic division in gametogenesis [phenotype]**\", \"FNM half-stop [phenotype]\" ]\n",
    "• Incorrect: Splitting these fails to see that all reference halting the very first post-meiotic mitosis.\n",
    "• Correct: They describe the same stage (first mitosis in gametogenesis), so they are synonyms.\n",
    "• Important: “FNM half-stop” is shorthand for the same phenomenon, not a different event.\n",
    "• Show in the output: Yes, because the length of cluster is greater than 1.\n",
    "• Cluster representative: I selected 'arrest of the first mitotic division in gametogenesis [phenotype]' as the representative because it is the most precise and descriptive term. It clearly specifies the process (mitotic division) and context (gametogenesis), avoiding the informal shorthand of 'FNM half-stop' and the less formal phrasing of 'arresting the first mitosis during gametogenesis.' This precision makes it the best exemplar for the cluster.\n",
    "\n",
    "18) Arrest Prior to First Mitotic Division\n",
    "\n",
    "Cluster:\n",
    "[ \"arresting prior to the first mitotic division [phenotype]\" ]\n",
    "• Incorrect: Assuming it is the same as “arresting the first mitosis” would confuse the actual onset of that mitosis.\n",
    "• Correct: This indicates cells never even enter mitosis.\n",
    "• Important: Distinguishing “before it starts” from “during the division” can be crucial for understanding gametogenesis defects.\n",
    "• Show in the output: No, because the length of cluster is equal to 1.\n",
    "\n",
    "19) Mitotic Arrest During Female Gametogenesis\n",
    "\n",
    "Cluster:\n",
    "[ \"mitotic arrest during female gametogenesis [phenotype]\" ]\n",
    "• Incorrect: Merging with the “first mitosis” cluster might ignore that multiple mitotic divisions can occur in female lines.\n",
    "• Correct: A female-specific block in some mitotic division (not necessarily the first).\n",
    "• Important: Sex specificity and indefinite mitotic stage set it apart from a clearly labeled “first mitosis” arrest.\n",
    "• Show in the output: No, because the length of cluster is equal to 1.\n",
    "Output-1\n",
    "{\n",
    "\"0\": [\n",
    "[\n",
    "\"Meiotic block [phenotype]\",\n",
    "\"**meiotic arrest [phenotype]**\",\n",
    "\"meiotic arrest phenotype [phenotype]\",\n",
    "\"meiotic division stop [phenotype]\"\n",
    "],\n",
    "[\n",
    "\"block in meiosis prophase I [phenotype]\",\n",
    "\"**meiotic prophase arrest [phenotype]**\"\n",
    "],\n",
    "[\n",
    "\"**pachytene arrest [phenotype]**\",\n",
    "\"meiotic arrest at pachytene [phenotype]\"\n",
    "],\n",
    "[\n",
    "\"arrest at the end of meiosis I [phenotype]\",\n",
    "\"arrest after meiosis I [phenotype]\",\n",
    "\"**meiotic arrest at telophase I [phenotype]**\"\n",
    "],\n",
    "[\n",
    "\"**termination of meiosis after anaphase I [phenotype]**\",\n",
    "\"premature termination of meiosis after anaphase I [phenotype]\"\n",
    "],\n",
    "[\n",
    "\"arrest of meiotic progression in anaphase II [phenotype]\",\n",
    "\"**meiotic arrest in anaphase II [phenotype]**\"\n",
    "],\n",
    "[\n",
    "\"arresting the first mitosis during gametogenesis [phenotype]\",\n",
    "\"**arrest of the first mitotic division in gametogenesis [phenotype]**\",\n",
    "\"FNM half-stop [phenotype]\"\n",
    "]\n",
    "]\n",
    "}\n",
    "Input-2\n",
    "{\n",
    "  \"1\": '[\n",
    "    \"Salt-stress severity [treatment]\",\n",
    "    \"high NaCl stress [treatment]\",\n",
    "    \"potassium deprivation stress [treatment]\",\n",
    "    \"salt-stress response [treatment]\",\n",
    "    \"salt stress tolerance [treatment]\",\n",
    "    \"heat and salt stress conditions [treatment]\",\n",
    "    \"prolonged levels of salt stress [treatment]\",\n",
    "    \"recovery from salt stress [treatment]\",\n",
    "    \"salt stress assay [treatment]\",\n",
    "    \"salt stress signaling pathways [treatment]\",\n",
    "    \"gradual salt stress treatments [treatment]\",\n",
    "    \"salt and low temperature stresses [treatment]\",\n",
    "    \"salt and silicon stresses [treatment]\"\n",
    "  ]'\n",
    "}\n",
    "Input-2 → Output-2 [REASONING]\n",
    "\n",
    "1) Salt-Stress Core\n",
    "\n",
    "Clustered Terms\n",
    "• “Salt-stress severity [treatment]”\n",
    "• “**high NaCl stress [treatment]**”\n",
    "• “salt-stress response [treatment]”\n",
    "• “salt stress tolerance [treatment]”\n",
    "• “prolonged levels of salt stress [treatment]”\n",
    "• “recovery from salt stress [treatment]”\n",
    "• “salt stress assay [treatment]”\n",
    "• “salt stress signaling pathways [treatment]”\n",
    "• “gradual salt stress treatments [treatment]”\n",
    "\n",
    "Incorrect: Splitting “NaCl” from “salt” would be biologically misleading since NaCl is the chemical basis of most salt stress.\n",
    "Correct: Recognize all are purely salt-based conditions; “NaCl” is the explicit form, but it is still “salt.”\n",
    "Important: These labels measure or manipulate salt-stress conditions alone (no other stress factor).\n",
    "• Show in the output?: Yes, because the length of cluster is greater than 1.\n",
    "Cluster representative: I selected 'high NaCl stress [treatment]' as the representative because it directly encapsulates the core concept of salt stress by explicitly naming the chemical agent (NaCl) responsible for inducing the stress condition. This term is both succinct and unambiguous, avoiding additional qualifiers (like severity, response, or tolerance) that could shift the focus away from the primary salt stress condition.\n",
    "\n",
    "2) Unique Stress: Potassium Deprivation\n",
    "\n",
    "Clustered Term\n",
    "• “potassium deprivation stress [treatment]”\n",
    "\n",
    "Incorrect: Combining this with salt-based treatments implies overlapping ionic stress without specificity.\n",
    "Correct: Keep it separate because it focuses on K+ deficiency rather than NaCl excess.\n",
    "Important: Potassium starvation is a distinct abiotic stress requiring separate interpretation and management from salt stress.\n",
    "• Show in the output?: No, because the length of cluster is equal to 1.\n",
    "\n",
    "3) Heat + Salt Stress\n",
    "\n",
    "Clustered Term\n",
    "• “heat and salt stress conditions [treatment]”\n",
    "\n",
    "Incorrect: Merging with the salt core group would lose the additional heat component.\n",
    "Correct: Keep it in its own cluster because it involves two distinct stressors (heat + salt).\n",
    "Important: Many experiments examine combined stresses differently than single-stress treatments.\n",
    "• Show in the output?: No, because the length of cluster is equal to 1.\n",
    "\n",
    "4) Salt + Low Temperature\n",
    "\n",
    "Clustered Term\n",
    "• “salt and low temperature stresses [treatment]”\n",
    "\n",
    "Incorrect: Folding it into a single “salt” cluster disregards the cold factor.\n",
    "Correct: Identify that it specifically tests tolerance or response to dual stress: salt and cold.\n",
    "Important: Understanding multi-stress interactions is crucial for breeding or experimental design.\n",
    "• Show in the output?: No, because the length of cluster is equal to 1.\n",
    "\n",
    "5) Salt + Silicon\n",
    "\n",
    "Clustered Term\n",
    "• “salt and silicon stresses [treatment]”\n",
    "\n",
    "Incorrect: Grouping with plain salt stress lumps unique “silicon” involvement into generic salt.\n",
    "Correct: Keep separate because it’s salt + another factor (silicon) that could mitigate or alter salt stress.\n",
    "Important: Silicon is sometimes used to ameliorate salt stress, so it forms a distinct combined treatment.\n",
    "• Show in the output: No, because the length of cluster is equal to 1.\n",
    "\n",
    "output-2\n",
    "{\n",
    "  \"1\": [\n",
    "    [\n",
    "      \"Salt-stress severity [treatment]\",\n",
    "      \"**high NaCl stress [treatment]**\",\n",
    "      \"salt-stress response [treatment]\",\n",
    "      \"salt stress tolerance [treatment]\",\n",
    "      \"prolonged levels of salt stress [treatment]\",\n",
    "      \"recovery from salt stress [treatment]\",\n",
    "      \"salt stress assay [treatment]\",\n",
    "      \"salt stress signaling pathways [treatment]\",\n",
    "      \"gradual salt stress treatments [treatment]\"]\n",
    "  ]\n",
    "}\n",
    "\"\"\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 58,
   "metadata": {},
   "outputs": [],
   "source": [
    "import ast\n",
    "\n",
    "def split_dict(d, chunk_size=100):\n",
    "    keys = list(d.keys())\n",
    "    chunks = []\n",
    "    for i in range(0, len(keys), chunk_size):\n",
    "        chunk_keys = keys[i:i+chunk_size]\n",
    "        chunk_dict = {k: d[k] for k in chunk_keys}\n",
    "        chunks.append(chunk_dict)\n",
    "    return chunks\n",
    "\n",
    "def flatten_bracketed_strings(value_list):\n",
    "    \"\"\"\n",
    "    Takes a list. For each item:\n",
    "      - If item is a string that looks like '[...]', parse it and extend the list.\n",
    "      - Otherwise, keep as is.\n",
    "    Returns a new flattened list.\n",
    "    \"\"\"\n",
    "    new_list = []\n",
    "    for val in value_list:\n",
    "        if (\n",
    "            isinstance(val, str) \n",
    "            and val.strip().startswith(\"[\") \n",
    "            and val.strip().endswith(\"]\")\n",
    "        ):\n",
    "            # Attempt to parse the bracketed string\n",
    "            try:\n",
    "                parsed = ast.literal_eval(val)  # convert string -> Python list\n",
    "                if isinstance(parsed, list):\n",
    "                    new_list.extend(parsed)  # flatten\n",
    "                else:\n",
    "                    # If it's not a list, just append as-is\n",
    "                    new_list.append(val)\n",
    "            except (SyntaxError, ValueError):\n",
    "                # If parsing fails, keep original\n",
    "                new_list.append(val)\n",
    "        else:\n",
    "            new_list.append(val)\n",
    "    return new_list\n",
    "\n",
    "\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 31,
   "metadata": {},
   "outputs": [],
   "source": [
    "#For training \n",
    "df = df_train.copy()\n",
    "#For testing\n",
    "#df = df_test.copy()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 32,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "Index(['Entity Name', 'Input', 'Output', 'Output>1', 'one_item', 'user_prompt',\n",
       "       'o3_corrected_cluster', 'is_correct'],\n",
       "      dtype='object')"
      ]
     },
     "execution_count": 32,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df.columns  "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 211,
   "metadata": {},
   "outputs": [],
   "source": [
    "entity_dict = {}\n",
    "for item in df[\"Output\"]:\n",
    "    try:\n",
    "        parsed_dict = json.loads(item)\n",
    "        entity_dict.update(parsed_dict)\n",
    "    except json.JSONDecodeError:\n",
    "        print(f\"Skipping invalid JSON: {item}\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 41,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Entity Name</th>\n",
       "      <th>Input</th>\n",
       "      <th>Output</th>\n",
       "      <th>Output&gt;1</th>\n",
       "      <th>one_item</th>\n",
       "      <th>user_prompt</th>\n",
       "      <th>o3_corrected_cluster</th>\n",
       "      <th>is_correct</th>\n",
       "      <th>reasoning</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>630</th>\n",
       "      <td>23162</td>\n",
       "      <td>{\"23162\": [\"Cross talk [phenotype]\", \"node for...</td>\n",
       "      <td>{\"23162\": [[\"signal transduction cross-talk [p...</td>\n",
       "      <td>{\"23162\": [[\"**Cross talk [phenotype]**\", \"cro...</td>\n",
       "      <td>False</td>\n",
       "      <td>This looks correct to me because each cluster ...</td>\n",
       "      <td>[[\"**Cross talk [phenotype]**\", \"cross-talks [...</td>\n",
       "      <td>True</td>\n",
       "      <td>We first identified a large group of entities ...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>367</th>\n",
       "      <td>135617</td>\n",
       "      <td>{\"135617\": [\"gene(s) related to respiratory el...</td>\n",
       "      <td>{\"135617\": [[\"coordinately regulated gene(s) f...</td>\n",
       "      <td>{\"135617\": [[\"coordinately regulated gene(s) f...</td>\n",
       "      <td>False</td>\n",
       "      <td>This looks incorrect to me. Two issues stand o...</td>\n",
       "      <td>[[\"**gene(s) involved in the alternative respi...</td>\n",
       "      <td>False</td>\n",
       "      <td>After careful evaluation following the cluster...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>485</th>\n",
       "      <td>122763</td>\n",
       "      <td>{\"122763\": [\"overlapping gene(s) in apple chro...</td>\n",
       "      <td>{\"122763\": [[\"**orthologous regions [phenotype...</td>\n",
       "      <td>{\"122763\": [[\"**orthologous regions [phenotype...</td>\n",
       "      <td>False</td>\n",
       "      <td>This looks incorrect to me, because some clust...</td>\n",
       "      <td>[[\"**Homologous regions [phenotype]**\", \"Homol...</td>\n",
       "      <td>False</td>\n",
       "      <td>The clustering was performed by first consider...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>290</th>\n",
       "      <td>151814</td>\n",
       "      <td>{\"151814\": [\"locus tms5 [gene]\", \"tms5 locus [...</td>\n",
       "      <td>{\"151814\": [[\"orf79 [gene]\"], [\"SLG1indica [ge...</td>\n",
       "      <td>{\"151814\": [[\"**asp-lsl [gene]**\", \"ASP-LSL lo...</td>\n",
       "      <td>False</td>\n",
       "      <td>Great job, the clusters are correctly formed. ...</td>\n",
       "      <td>[[\"**asp-lsl [gene]**\", \"ASP-LSL locus [gene]\"...</td>\n",
       "      <td>True</td>\n",
       "      <td>The clustering was performed by strictly group...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>72</th>\n",
       "      <td>131773</td>\n",
       "      <td>{\"131773\": [\"VvNPF3.2 [gene]\", \"VviNPF2.1 [gen...</td>\n",
       "      <td>{\"131773\": [[\"VviNPF2.1 [gene]\"], [\"**VvNPF3.2...</td>\n",
       "      <td>{\"131773\": [[\"**VvNPF3.2 [gene]**\", \"grapevine...</td>\n",
       "      <td>False</td>\n",
       "      <td>Great job, that looks correct because the enti...</td>\n",
       "      <td>[[\"**VvNPF3.2 [gene]**\", \"grapevine NFP3.2 [ge...</td>\n",
       "      <td>True</td>\n",
       "      <td>The gene entities were clustered based on exac...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>345</th>\n",
       "      <td>130937</td>\n",
       "      <td>{\"130937\": [\"AtEDEM1 homolog [gene]\", \"AtEDEM1...</td>\n",
       "      <td>{\"130937\": [[\"AtEDEM1/AtEDEM2 [gene]\", \"two ED...</td>\n",
       "      <td>{\"130937\": [[\"AtEDEM1/AtEDEM2 [gene]\", \"two ED...</td>\n",
       "      <td>False</td>\n",
       "      <td>Great job, the grouping is spot‐on. The cluste...</td>\n",
       "      <td>[[\"AtEDEM1/AtEDEM2 [gene]\", \"two EDEM homologs...</td>\n",
       "      <td>True</td>\n",
       "      <td>The gene list was first divided into two major...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>241</th>\n",
       "      <td>63849</td>\n",
       "      <td>{\"63849\": [\"catalytic manganese cluster [prote...</td>\n",
       "      <td>{\"63849\": [[\"DCH model [phenotype]\"], [\"**mang...</td>\n",
       "      <td>{\"63849\": [[\"**manganese cluster [protein comp...</td>\n",
       "      <td>False</td>\n",
       "      <td>Great job, that looks correct to me. You have ...</td>\n",
       "      <td>[[\"**manganese cluster [protein complex]**\", \"...</td>\n",
       "      <td>True</td>\n",
       "      <td>The clustering was executed by strictly applyi...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>13</th>\n",
       "      <td>81737</td>\n",
       "      <td>{\"81737\": [\"wt MP [protein]\", \"TMEM16A wild-ty...</td>\n",
       "      <td>{\"81737\": [[\"native ScHal3 [protein]\"], [\"nati...</td>\n",
       "      <td>{\"81737\": [[\"**wt HcPro [protein]**\", \"Wild-ty...</td>\n",
       "      <td>False</td>\n",
       "      <td>Great job, the clustering looks correct to me....</td>\n",
       "      <td>[[\"**wt HcPro [protein]**\", \"Wild-type (wt) Hc...</td>\n",
       "      <td>True</td>\n",
       "      <td>In this dataset, each entry refers to a specif...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>315</th>\n",
       "      <td>123050</td>\n",
       "      <td>{\"123050\": [\"poly(A)-ending species [rna]\", \"t...</td>\n",
       "      <td>{\"123050\": [[\"polyadenylated RNA 3' ends [geno...</td>\n",
       "      <td>{\"123050\": [[\"polyadenylated RNA 3' ends [geno...</td>\n",
       "      <td>False</td>\n",
       "      <td>Great job, the clustering appears to be correc...</td>\n",
       "      <td>[[\"polyadenylated RNA 3' ends [genomic region]...</td>\n",
       "      <td>True</td>\n",
       "      <td>We examined all 26 entities using their full p...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>387</th>\n",
       "      <td>46941</td>\n",
       "      <td>{\"46941\": [\"emb2750 seeds [mutant]\", \"essp8 se...</td>\n",
       "      <td>{\"46941\": [[\"**hes seeds [mutant]**\", \"hes see...</td>\n",
       "      <td>{\"46941\": [[\"**hes seeds [mutant]**\", \"hes see...</td>\n",
       "      <td>False</td>\n",
       "      <td>This looks correct to me. The clusters have be...</td>\n",
       "      <td>[[\"**hes seeds [mutant]**\", \"hes seed [mutant]...</td>\n",
       "      <td>True</td>\n",
       "      <td>The clustering was performed by examining the ...</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>600 rows × 9 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "     Entity Name                                              Input  \\\n",
       "630        23162  {\"23162\": [\"Cross talk [phenotype]\", \"node for...   \n",
       "367       135617  {\"135617\": [\"gene(s) related to respiratory el...   \n",
       "485       122763  {\"122763\": [\"overlapping gene(s) in apple chro...   \n",
       "290       151814  {\"151814\": [\"locus tms5 [gene]\", \"tms5 locus [...   \n",
       "72        131773  {\"131773\": [\"VvNPF3.2 [gene]\", \"VviNPF2.1 [gen...   \n",
       "..           ...                                                ...   \n",
       "345       130937  {\"130937\": [\"AtEDEM1 homolog [gene]\", \"AtEDEM1...   \n",
       "241        63849  {\"63849\": [\"catalytic manganese cluster [prote...   \n",
       "13         81737  {\"81737\": [\"wt MP [protein]\", \"TMEM16A wild-ty...   \n",
       "315       123050  {\"123050\": [\"poly(A)-ending species [rna]\", \"t...   \n",
       "387        46941  {\"46941\": [\"emb2750 seeds [mutant]\", \"essp8 se...   \n",
       "\n",
       "                                                Output  \\\n",
       "630  {\"23162\": [[\"signal transduction cross-talk [p...   \n",
       "367  {\"135617\": [[\"coordinately regulated gene(s) f...   \n",
       "485  {\"122763\": [[\"**orthologous regions [phenotype...   \n",
       "290  {\"151814\": [[\"orf79 [gene]\"], [\"SLG1indica [ge...   \n",
       "72   {\"131773\": [[\"VviNPF2.1 [gene]\"], [\"**VvNPF3.2...   \n",
       "..                                                 ...   \n",
       "345  {\"130937\": [[\"AtEDEM1/AtEDEM2 [gene]\", \"two ED...   \n",
       "241  {\"63849\": [[\"DCH model [phenotype]\"], [\"**mang...   \n",
       "13   {\"81737\": [[\"native ScHal3 [protein]\"], [\"nati...   \n",
       "315  {\"123050\": [[\"polyadenylated RNA 3' ends [geno...   \n",
       "387  {\"46941\": [[\"**hes seeds [mutant]**\", \"hes see...   \n",
       "\n",
       "                                              Output>1  one_item  \\\n",
       "630  {\"23162\": [[\"**Cross talk [phenotype]**\", \"cro...     False   \n",
       "367  {\"135617\": [[\"coordinately regulated gene(s) f...     False   \n",
       "485  {\"122763\": [[\"**orthologous regions [phenotype...     False   \n",
       "290  {\"151814\": [[\"**asp-lsl [gene]**\", \"ASP-LSL lo...     False   \n",
       "72   {\"131773\": [[\"**VvNPF3.2 [gene]**\", \"grapevine...     False   \n",
       "..                                                 ...       ...   \n",
       "345  {\"130937\": [[\"AtEDEM1/AtEDEM2 [gene]\", \"two ED...     False   \n",
       "241  {\"63849\": [[\"**manganese cluster [protein comp...     False   \n",
       "13   {\"81737\": [[\"**wt HcPro [protein]**\", \"Wild-ty...     False   \n",
       "315  {\"123050\": [[\"polyadenylated RNA 3' ends [geno...     False   \n",
       "387  {\"46941\": [[\"**hes seeds [mutant]**\", \"hes see...     False   \n",
       "\n",
       "                                           user_prompt  \\\n",
       "630  This looks correct to me because each cluster ...   \n",
       "367  This looks incorrect to me. Two issues stand o...   \n",
       "485  This looks incorrect to me, because some clust...   \n",
       "290  Great job, the clusters are correctly formed. ...   \n",
       "72   Great job, that looks correct because the enti...   \n",
       "..                                                 ...   \n",
       "345  Great job, the grouping is spot‐on. The cluste...   \n",
       "241  Great job, that looks correct to me. You have ...   \n",
       "13   Great job, the clustering looks correct to me....   \n",
       "315  Great job, the clustering appears to be correc...   \n",
       "387  This looks correct to me. The clusters have be...   \n",
       "\n",
       "                                  o3_corrected_cluster  is_correct  \\\n",
       "630  [[\"**Cross talk [phenotype]**\", \"cross-talks [...        True   \n",
       "367  [[\"**gene(s) involved in the alternative respi...       False   \n",
       "485  [[\"**Homologous regions [phenotype]**\", \"Homol...       False   \n",
       "290  [[\"**asp-lsl [gene]**\", \"ASP-LSL locus [gene]\"...        True   \n",
       "72   [[\"**VvNPF3.2 [gene]**\", \"grapevine NFP3.2 [ge...        True   \n",
       "..                                                 ...         ...   \n",
       "345  [[\"AtEDEM1/AtEDEM2 [gene]\", \"two EDEM homologs...        True   \n",
       "241  [[\"**manganese cluster [protein complex]**\", \"...        True   \n",
       "13   [[\"**wt HcPro [protein]**\", \"Wild-type (wt) Hc...        True   \n",
       "315  [[\"polyadenylated RNA 3' ends [genomic region]...        True   \n",
       "387  [[\"**hes seeds [mutant]**\", \"hes seed [mutant]...        True   \n",
       "\n",
       "                                             reasoning  \n",
       "630  We first identified a large group of entities ...  \n",
       "367  After careful evaluation following the cluster...  \n",
       "485  The clustering was performed by first consider...  \n",
       "290  The clustering was performed by strictly group...  \n",
       "72   The gene entities were clustered based on exac...  \n",
       "..                                                 ...  \n",
       "345  The gene list was first divided into two major...  \n",
       "241  The clustering was executed by strictly applyi...  \n",
       "13   In this dataset, each entry refers to a specif...  \n",
       "315  We examined all 26 entities using their full p...  \n",
       "387  The clustering was performed by examining the ...  \n",
       "\n",
       "[600 rows x 9 columns]"
      ]
     },
     "execution_count": 41,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 56,
   "metadata": {},
   "outputs": [],
   "source": [
    "variable_input_enitity_dict = {}\n",
    "for item in df[\"Input\"]:\n",
    "    try:\n",
    "        parsed_dict = json.loads(item)\n",
    "        variable_input_enitity_dict.update(parsed_dict)\n",
    "    except json.JSONDecodeError:\n",
    "        print(f\"Skipping invalid JSON: {item}\")\n",
    "\n",
    "variable_output_enitity_dict = {}\n",
    "for item in df[\"Output\"]:\n",
    "    try:\n",
    "        parsed_dict = json.loads(item)\n",
    "        variable_output_enitity_dict.update(parsed_dict)\n",
    "    except json.JSONDecodeError:\n",
    "        print(f\"Skipping invalid JSON: {item}\")\n",
    "\n",
    "variable_o3_user_prompt_enitity_dict =  df.set_index('Entity Name')['user_prompt'].to_dict()\n",
    "\n",
    "variable_o3mini_corrected_cluster_enitity_dict =  df.set_index('Entity Name')['o3_corrected_cluster'].to_dict()\n",
    "\n",
    "#variable_reasoning_enitity_dict =  df.set_index('Entity Name')['reasoning'].to_dict() # run this for V5 only\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 59,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "length of chunks_variable_input_entity: 996\n",
      "length of chunks_variable_output_entity: 996\n",
      "length of chunks_variable_o3_user_prompt: 996\n",
      "length of chunks_variable_o3mini_corrected_cluster: 996\n"
     ]
    }
   ],
   "source": [
    "#create chunks for each variable dictonary\n",
    "\n",
    "chunks_variable_input_entity = split_dict(variable_input_enitity_dict,1)\n",
    "chunks_variable_output_entity = split_dict(variable_output_enitity_dict,1)\n",
    "chunks_variable_o3_user_prompt = split_dict(variable_o3_user_prompt_enitity_dict,1)\n",
    "chunks_variable_o3mini_corrected_cluster = split_dict(variable_o3mini_corrected_cluster_enitity_dict,1)\n",
    "#chunks_variable_o3mini_reasoning = split_dict(variable_reasoning_enitity_dict,1) # run this for V5 only\n",
    "#print length of each chunk\n",
    "print('length of chunks_variable_input_entity:', len(chunks_variable_input_entity))\n",
    "print('length of chunks_variable_output_entity:', len(chunks_variable_output_entity))\n",
    "print('length of chunks_variable_o3_user_prompt:', len(chunks_variable_o3_user_prompt))\n",
    "print('length of chunks_variable_o3mini_corrected_cluster:', len(chunks_variable_o3mini_corrected_cluster))\n",
    "#print('length of chunks_variable_o3mini_reasoning:', len(chunks_variable_o3mini_reasoning)) # run this for V5 only"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 60,
   "metadata": {},
   "outputs": [],
   "source": [
    "variable_o3mini_corrected_cluster_enitity_dict =  df.set_index('Entity Name')['o3_corrected_cluster'].to_dict()\n",
    "chunks_variable_o3mini_corrected_cluster = split_dict(variable_o3mini_corrected_cluster_enitity_dict,1)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Creating jsonl file for V1"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 172,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Done writing 25 examples to /Users/manojitharajula/Documents/PhD/Connectome/Entity_disambiguation/kmeans/4omini_finetuning/input_data/4omini_testset_25.jsonl\n"
     ]
    }
   ],
   "source": [
    "import jsonlines\n",
    "output_path = '/'\n",
    "out_file = '/Users/manojitharajula/Documents/PhD/Connectome/Entity_disambiguation/kmeans/4omini_finetuning/input_data/4omini_testset_25.jsonl'\n",
    "count_examples = 0\n",
    "with jsonlines.open(out_file, mode='w') as writer:\n",
    "    # Loop through each different chhunk and create a line in jsonl file\n",
    "    for i in range(len(chunks_variable_input_entity)):\n",
    "        line = {\n",
    "                \"messages\": [\n",
    "\n",
    "                        {\n",
    "                            \"role\": \"system\",\n",
    "                            \"content\": system_message\n",
    "                        },\n",
    "\n",
    "                        {\n",
    "                            \"role\": \"user\",\n",
    "                            \"content\": json.dumps(chunks_variable_input_entity[i], indent=2)\n",
    "                       \n",
    "                        },\n",
    "\n",
    "                        {\n",
    "                            \"role\": \"assistant\",\n",
    "                            \"content\": json.dumps(chunks_variable_output_entity[i], indent=2)\n",
    "                        },\n",
    "\n",
    "                        {\n",
    "                            \"role\": \"user\",\n",
    "                            \"content\": json.dumps(chunks_variable_o3_user_prompt[i], indent=2)\n",
    "                        },\n",
    "\n",
    "                        {\n",
    "                            \"role\": \"assistant\",\n",
    "                            \"content\": json.dumps(chunks_variable_o3mini_corrected_cluster[i], indent=2)\n",
    "                        }\n",
    "                ]\n",
    "                }\n",
    "        # draw a float between 0 and 1\n",
    "        include_detailed_description = random.random() < 0.2\n",
    "       #insert detailed description at index 1 if include_detailed_description is true\n",
    "        if include_detailed_description:\n",
    "            line[\"messages\"].insert(1, {\"role\": \"user\", \"content\": detailed_description})\n",
    "        writer.write(line)\n",
    "        count_examples += 1\n",
    "print('Done writing {} examples to {}'.format(count_examples, out_file))\n",
    "\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": []
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Jsonl file creation for V2, V3, V4"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 218,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Done writing 36 examples to /Users/manojitharajula/Documents/PhD/Connectome/Entity_disambiguation/kmeans/4omini_finetuning/input_data/4omini_testset_v4_36.jsonl\n"
     ]
    }
   ],
   "source": [
    "import jsonlines\n",
    "output_path = '/'\n",
    "out_file = '/Users/manojitharajula/Documents/PhD/Connectome/Entity_disambiguation/kmeans/4omini_finetuning/input_data/4omini_testset_v4_36.jsonl'\n",
    "count_examples = 0\n",
    "with jsonlines.open(out_file, mode='w') as writer:\n",
    "    # Loop through each different chhunk and create a line in jsonl file\n",
    "    for i in range(len(chunks_variable_input_entity)):\n",
    "        line = {\n",
    "                \"messages\": [\n",
    "\n",
    "                        {\n",
    "                            \"role\": \"system\",\n",
    "                            \"content\": system_message\n",
    "                        },\n",
    "\n",
    "                        {\n",
    "                            \"role\": \"user\",\n",
    "                            \"content\": detailed_description\n",
    "                        },\n",
    "\n",
    "                        {\n",
    "                            \"role\": \"user\",\n",
    "                            \"content\": json.dumps(chunks_variable_input_entity[i], indent=2)\n",
    "                       \n",
    "                        }, \n",
    "\n",
    "                        {\n",
    "                            \"role\": \"assistant\",\n",
    "                            \"content\": json.dumps(chunks_variable_o3mini_corrected_cluster[i], indent=2)\n",
    "                        }\n",
    "                ]\n",
    "                }\n",
    "\n",
    "        writer.write(line)\n",
    "        count_examples += 1\n",
    "print('Done writing {} examples to {}'.format(count_examples, out_file))\n",
    "\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### jsonl file creation for V5 (this way we can avoid Backslahses)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 115,
   "metadata": {},
   "outputs": [],
   "source": [
    "# convert list of dictionaries, chunks_variable_input_entity into a single dictionary\n",
    "variable_input_entity_dict = {}\n",
    "for d in chunks_variable_input_entity:\n",
    "    variable_input_entity_dict.update(d)\n",
    "\n",
    "# same for reasoning \n",
    "variable_o3mini_reasoning_dict = {}\n",
    "for d in chunks_variable_o3mini_reasoning:\n",
    "    variable_o3mini_reasoning_dict.update(d)\n",
    "\n",
    "# same for corrected cluster\n",
    "variable_o3mini_corrected_cluster_dict = {}\n",
    "for d in chunks_variable_o3mini_corrected_cluster:\n",
    "    variable_o3mini_corrected_cluster_dict.update(d)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 122,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Done writing 600 examples to /Users/manojitharajula/Documents/PhD/Connectome/Entity_disambiguation/kmeans/4omini_finetuning/input_data/4omini_trainset_v5_600.jsonl\n"
     ]
    }
   ],
   "source": [
    "import jsonlines\n",
    "import json\n",
    "output_path = '/'\n",
    "out_file = '/Users/manojitharajula/Documents/PhD/Connectome/Entity_disambiguation/kmeans/4omini_finetuning/input_data/4omini_trainset_v5_600.jsonl'\n",
    "count_examples = 0\n",
    "with jsonlines.open(out_file, mode='w') as writer:\n",
    "    # Loop through each different chhunk and create a line in jsonl file\n",
    "    for i in range(len(chunks_variable_input_entity)):\n",
    "        cluster_id = list(chunks_variable_input_entity[i].keys())[0]\n",
    " \n",
    "        line = {\n",
    "                \"messages\": [\n",
    "\n",
    "                        {\n",
    "                            \"role\": \"system\",\n",
    "                            \"content\": system_message\n",
    "                        },\n",
    "\n",
    "                        {\n",
    "                        \"role\": \"user\",\n",
    "                        \"content\": f\"{cluster_id}: {variable_input_entity_dict[cluster_id]}\"\n",
    "                        }, \n",
    "\n",
    "                       {\n",
    "                        \"role\": \"assistant\",\n",
    "                        \"content\": (\n",
    "                            f\"{cluster_id}:\\n\"\n",
    "                            f\"reasoning: {variable_o3mini_reasoning_dict[int(cluster_id)]},\\n\"\n",
    "                            f\"clusters: {eval(variable_o3mini_corrected_cluster_dict[int(cluster_id)])}\"\n",
    "                        )\n",
    "                    }\n",
    "                ]\n",
    "                }\n",
    "\n",
    "        writer.write(line)\n",
    "        count_examples += 1\n",
    "print('Done writing {} examples to {}'.format(count_examples, out_file))\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "connectome",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.12.4"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
