{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Generating training data for fine tuning using o3-mini\n",
    "\n",
    "In this notebook i show my suggestion for making a fine-tuning dataset for 4o-mini, using the power of o3-mini.\n",
    "\n",
    "The first step is getting the data.\n",
    "\n",
    "We take the clustering made by Manoj with o3-mini, and then ask o3-mini with reasoning set to high to go through its own results for validation, make corrections if needed, and then explain why this is wrong as a user message.\n",
    "\n",
    "This message will be used to simulate a conversation with 4o-mini to teach it clustering patterns that we want to highlight. We are essentially trying to gaslight GPT into making better clusters. Making it doubt itself.  \n",
    "\n",
    "With fine tuing we therefore aim to make the predictions of 4o-mini be more similar to o3-mini, but at a lower cost. \n",
    "\n",
    "\n",
    "I think the bonus with this approach, is that we don't need annotate ourselves, just pick the ones we agree with the validation, which is much faster.\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Generate dataset"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 105,
   "metadata": {},
   "outputs": [],
   "source": [
    "#This is a script that loads a pandas dataframe from csv, converts the string formatted as a list into a list of lists. The list of lists is then flattened into a single list.\n",
    "\n",
    "import pandas as pd\n",
    "import random\n",
    "import json\n",
    "df = pd.read_csv('o3mini_maxclust_80_subset_random_1000_nocutoff.csv', index_col=None)\n",
    "\n",
    "#rename the column \"Entity Name\" to \"Entity\" and \"Group Items\" to \"output\"\n",
    "df = df.rename(columns={\"Group Items\": \"Output\"})\n",
    "\n",
    "# Convert the string formatted as a list into a list of lists\n",
    "df['Output'] = df['Output'].apply(lambda x: eval(x))\n",
    "\n",
    "# Flatten the list of lists into a single list\n",
    "df['Input'] = df['Output'].apply(lambda x: [item.strip(\"*\") for sublist in x for item in sublist])\n",
    "\n",
    "#randomize the order of the single list\n",
    "df['Input'] = df['Input'].apply(lambda x: random.sample(x, len(x)))\n",
    "df = df[df['Input'].apply(len) < 30]\n",
    "# make a new column, where internal lists of length 1 are removed \n",
    "df['Output>1'] = df['Output'].apply(lambda x: [item for item in x if len(item) > 1])\n",
    "\n",
    "\n",
    "#zip the Group_Items_no_singles with the Entity Name \n",
    "df[\"Output\"] = df.apply(lambda row: {row[\"Entity Name\"]: row['Output']}, axis=1)\n",
    "df[\"Input\"] = df.apply(lambda row: {row[\"Entity Name\"]: row['Input']}, axis=1)\n",
    "df['Output>1'] = df.apply(lambda row: {row[\"Entity Name\"]: row['Output>1']}, axis=1)\n",
    "\n",
    "\n",
    "\n",
    "#filter for single_list length < 30\n",
    "\n",
    "\n",
    "\n",
    "#convert the columns to json strings\n",
    "df[\"Output\"] = df[\"Output\"].apply(lambda x: json.dumps(x) if isinstance(x, dict) else x)\n",
    "df[\"Input\"] = df[\"Input\"].apply(lambda x: json.dumps(x) if isinstance(x, dict) else x)\n",
    "df[\"Output>1\"] = df[\"Output>1\"].apply(lambda x: json.dumps(x) if isinstance(x, dict) else x)\n",
    "\n",
    "#Change order of columns to have input, then output, then output>1, then entity name\n",
    "df = df[['Entity Name', 'Input', 'Output', 'Output>1']]\n",
    "\n",
    "#filter by Output>1 length of the list > 1\n",
    "#use column Output, but use only the ones where it clusteres multiple items together"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 106,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Entity Name</th>\n",
       "      <th>Input</th>\n",
       "      <th>Output</th>\n",
       "      <th>Output&gt;1</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>58963</td>\n",
       "      <td>{\"58963\": [\"Gardea-Torresdey et al. [researche...</td>\n",
       "      <td>{\"58963\": [[\"Gardea-Torresdey et al. [research...</td>\n",
       "      <td>{\"58963\": []}</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>22996</td>\n",
       "      <td>{\"22996\": [\"number of open stomata and amount ...</td>\n",
       "      <td>{\"22996\": [[\"**transpiration and stomatal cond...</td>\n",
       "      <td>{\"22996\": [[\"**transpiration and stomatal cond...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>100116</td>\n",
       "      <td>{\"100116\": [\"modern clade [gene]\", \"Clade SL [...</td>\n",
       "      <td>{\"100116\": [[\"Clade M [clade]\"], [\"Clade SL [c...</td>\n",
       "      <td>{\"100116\": []}</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>133255</td>\n",
       "      <td>{\"133255\": [\"LPP\\u10b2 [gene]\", \"SUPPRESSOR OF...</td>\n",
       "      <td>{\"133255\": [[\"**PLSP2A [gene]**\", \"PLSP2A gene...</td>\n",
       "      <td>{\"133255\": [[\"**PLSP2A [gene]**\", \"PLSP2A gene...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>2276</td>\n",
       "      <td>{\"2276\": [\"fc2 phenotypes [phenotype]\", \"100% ...</td>\n",
       "      <td>{\"2276\": [[\"F1 trichome phenotype [phenotype]\"...</td>\n",
       "      <td>{\"2276\": []}</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>995</th>\n",
       "      <td>31615</td>\n",
       "      <td>{\"31615\": [\"auxin-regulated endocytosis [proce...</td>\n",
       "      <td>{\"31615\": [[\"auxin-mediated modulation of the ...</td>\n",
       "      <td>{\"31615\": [[\"auxin-dependent inhibition of end...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>996</th>\n",
       "      <td>157227</td>\n",
       "      <td>{\"157227\": [\"ZmSAUR1 [gene]\", \"ZmSAUR1 (Zea ma...</td>\n",
       "      <td>{\"157227\": [[\"GmSAUR [gene]\"], [\"StSAUR [gene]...</td>\n",
       "      <td>{\"157227\": [[\"ZmSAUR1 (Zea mays SAURs) [gene]\"...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>997</th>\n",
       "      <td>131559</td>\n",
       "      <td>{\"131559\": [\"cytochrome b(5) [gene]\", \"BtuN ge...</td>\n",
       "      <td>{\"131559\": [[\"BtuB gene(s) [gene]\"], [\"BtuC ge...</td>\n",
       "      <td>{\"131559\": []}</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>998</th>\n",
       "      <td>99091</td>\n",
       "      <td>{\"99091\": [\"D. oleifera genome [organism]\", \"O...</td>\n",
       "      <td>{\"99091\": [[\"Genome sequencing of O. fragrans ...</td>\n",
       "      <td>{\"99091\": [[\"**O. fragrans 'Liuyejingui' (OFL)...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>999</th>\n",
       "      <td>65306</td>\n",
       "      <td>{\"65306\": [\"NF-YB/NF-YC dimer [protein complex...</td>\n",
       "      <td>{\"65306\": [[\"**CO/NF-Y complex [protein comple...</td>\n",
       "      <td>{\"65306\": [[\"**CO/NF-Y complex [protein comple...</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>802 rows Ã— 4 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "     Entity Name                                              Input  \\\n",
       "0          58963  {\"58963\": [\"Gardea-Torresdey et al. [researche...   \n",
       "2          22996  {\"22996\": [\"number of open stomata and amount ...   \n",
       "3         100116  {\"100116\": [\"modern clade [gene]\", \"Clade SL [...   \n",
       "4         133255  {\"133255\": [\"LPP\\u10b2 [gene]\", \"SUPPRESSOR OF...   \n",
       "5           2276  {\"2276\": [\"fc2 phenotypes [phenotype]\", \"100% ...   \n",
       "..           ...                                                ...   \n",
       "995        31615  {\"31615\": [\"auxin-regulated endocytosis [proce...   \n",
       "996       157227  {\"157227\": [\"ZmSAUR1 [gene]\", \"ZmSAUR1 (Zea ma...   \n",
       "997       131559  {\"131559\": [\"cytochrome b(5) [gene]\", \"BtuN ge...   \n",
       "998        99091  {\"99091\": [\"D. oleifera genome [organism]\", \"O...   \n",
       "999        65306  {\"65306\": [\"NF-YB/NF-YC dimer [protein complex...   \n",
       "\n",
       "                                                Output  \\\n",
       "0    {\"58963\": [[\"Gardea-Torresdey et al. [research...   \n",
       "2    {\"22996\": [[\"**transpiration and stomatal cond...   \n",
       "3    {\"100116\": [[\"Clade M [clade]\"], [\"Clade SL [c...   \n",
       "4    {\"133255\": [[\"**PLSP2A [gene]**\", \"PLSP2A gene...   \n",
       "5    {\"2276\": [[\"F1 trichome phenotype [phenotype]\"...   \n",
       "..                                                 ...   \n",
       "995  {\"31615\": [[\"auxin-mediated modulation of the ...   \n",
       "996  {\"157227\": [[\"GmSAUR [gene]\"], [\"StSAUR [gene]...   \n",
       "997  {\"131559\": [[\"BtuB gene(s) [gene]\"], [\"BtuC ge...   \n",
       "998  {\"99091\": [[\"Genome sequencing of O. fragrans ...   \n",
       "999  {\"65306\": [[\"**CO/NF-Y complex [protein comple...   \n",
       "\n",
       "                                              Output>1  \n",
       "0                                        {\"58963\": []}  \n",
       "2    {\"22996\": [[\"**transpiration and stomatal cond...  \n",
       "3                                       {\"100116\": []}  \n",
       "4    {\"133255\": [[\"**PLSP2A [gene]**\", \"PLSP2A gene...  \n",
       "5                                         {\"2276\": []}  \n",
       "..                                                 ...  \n",
       "995  {\"31615\": [[\"auxin-dependent inhibition of end...  \n",
       "996  {\"157227\": [[\"ZmSAUR1 (Zea mays SAURs) [gene]\"...  \n",
       "997                                     {\"131559\": []}  \n",
       "998  {\"99091\": [[\"**O. fragrans 'Liuyejingui' (OFL)...  \n",
       "999  {\"65306\": [[\"**CO/NF-Y complex [protein comple...  \n",
       "\n",
       "[802 rows x 4 columns]"
      ]
     },
     "execution_count": 106,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df\n",
    "#note i don't really use Outputs>1 in this workflow."
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Prompts for making fine-tuning dataset\n",
    "\n",
    "- constant_system_message is unchanged\n",
    "- constant_user_message is only slightly modified from Manoj's promts, in the beginning and the end.\n",
    "- variable_user_message is \"Output\" for each entry in the dataframe\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 63,
   "metadata": {},
   "outputs": [],
   "source": [
    "constant_system_message = \"\"\"\n",
    "You are senior scientist in plant biology. You NEED to meticulously VALIDATE given clustering output, and provide a corrected clustering if needed. Explain which groupings are wrong and provide final output. You MUST write response like a senior human plant scientist.\n",
    "\n",
    "The following are the entity clustering guidelines:\n",
    "\t1.\tExact Phrase Matching Matters: Always consider the full phrase, including key biological terms, bracketed text (ignoring minor differences such as spacing, punctuation, correct abbreviations, plurality).\n",
    "\t2.\tStrict (100%) Key Term Separation: Entities with different biological terms MUST be placed in separate clusters.\n",
    "3. Sub-identifier separation: Separate Entities with numeric differences, sub-identifiers, or qualifiers into different groups.\n",
    "\t4.\tAvoid False Similarity: Do NOT cluster two items together in same group just because they share a common word or term.\n",
    "\t5.\tStrict Synonym/Near-Synonym Grouping: Only group entities that refer to the same biological structure, process, meaning or concept.\n",
    "\t6.\tMaintain 100% Precision: When in even small doubt, MUST place entities in separate clusters.\n",
    "\t7.\tPreserve Original Data: No new items should be introduced, no duplicates should be introduced, and no entities should be omitted.\n",
    "8. YOU MUST pickup most appropriate and easy-to-understand cluster representative and enclose it with '**', if there is more than one entity in that particular cluster. For example, pick the full term instead of an abbreviation.\n",
    "\t9.\tOutput Format: Always return results in valid JSON format. MUST USE GIVEN KEY.\n",
    "\n",
    "Your main task is to read the clustered entities and output the validation in the below mentioned format.\n",
    "\n",
    "The goal:\n",
    "These results came from a smaller LLM, and i want you to take a look at it.\n",
    "We  your expertise in plant biology, to generate prompts for fine tuning the smaller LLM and mimick your patterns for this specific task.\n",
    "I want you to validate the clustering of this smaller model, and provide a corrected clustering if needed.\n",
    "Then you should imagine that you are a user that want to explain to the smaller LLM what it is doing wrong, to give it the final output. You should never mention the smaller LLM, but instead just pretend you are a user. \n",
    "An example of this would be:\n",
    "'I think you are wrong, cluster X is actually different from cluster Y because of reasons. Cluster X and Y should be clustered seperately'.\n",
    "If the smaller model is correct however, you should write something like:\n",
    "'Great job, that looks correct. Now output the correct format, but only output clusters with more than one member.'\n",
    "The expected input format:\n",
    "\n",
    "{\n",
    "  \"submission_id_1\": {\n",
    "    \"clusters\": [\n",
    "      [\n",
    "        \"**cluster_item1**\",\n",
    "        \"cluster_item2\"\n",
    "      ],\n",
    "      [\n",
    "        \"cluster_item3\"\n",
    "      ]\n",
    "    ]\n",
    "  }\n",
    "}\n",
    "\n",
    "Return format:\n",
    "\n",
    "If the clustering is correct:\n",
    "{\n",
    "  \"submission_id_1\": {\n",
    "    \"is_correct\": \"True\",\n",
    "    \"user_prompt\": \"This looks correct to me, output the clusters that have more than 1 member in the correct json format.\",\n",
    "    \"clusters\": [\n",
    "      [\n",
    "        \"cluster_item1\",\n",
    "        \"cluster_item2\"\n",
    "      ]\n",
    "    ]\n",
    "  }\n",
    "}\n",
    "\n",
    "If the clustering is incorrect:\n",
    "\n",
    "{\n",
    "  \"submission_id_1\": {\n",
    "    \"is_correct\": \"False\",\n",
    "    \"user_prompt\": \"This looks incorrect to me, <insert reason here>. Output the clusters that have more than 1 member in the correct json format.\",\n",
    "    \"clusters\": [\n",
    "      [\n",
    "        \"cluster_item2\",\n",
    "        \"cluster_item3\"\n",
    "      ]\n",
    "    ]\n",
    "  }\n",
    "}\n",
    "\n",
    "Warnings:\n",
    "\n",
    "Be completely sure to not forget any nodes from the input list.\n",
    "Remember not to base corrections on higher domain level knowledge, that the smaller model might not have, but rather the semantic meaning of the terms.\n",
    "is_correct should be returned as a string \"True\" or \"False\", and only one of these two.\n",
    "If there is any doubt if two groups should cluster together, cluster them separately.\n",
    "Remember to select group representative by enclosing entity in \"**\"\n",
    "\n",
    "\"\"\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 64,
   "metadata": {},
   "outputs": [],
   "source": [
    "constant_user_message = \"\"\"  \n",
    "Here are 2 examples of validation behavior:\n",
    "\n",
    "Example 1, correct behavior:\n",
    "\n",
    "input:\n",
    "{120406: ['difficulty in detecting mutations [phenotype]', 'hereditary breast cancers [phenotype]', 'mutational status of parental tumor cells [phenotype]', 'certain mutation types [phenotype]', 'Well-characterized mutations [phenotype]', 'alternative classifications [phenotype]', 'unique molecular signature [phenotype]', 'oncological diseases [phenotype]', 'independent mutation data [phenotype]', 'inherited disease mutation databases [database]', 'mutational signatures [mutation]', 'mutation types [phenotype]', 'More than half of human cancers [phenotype]', 'somatic and germline mutations [phenotype]', 'Identified mutations [phenotype]', 'lung cancer patients [phenotype]', 'germline transmissible [phenotype]', 'mutations in the germline [phenotype]', 'sporadic breast cancers [phenotype]', 'segregating mutations [phenotype]', 'breast/ovarian cancers [phenotype]', 'non-small-cell lung carcinoma [phenotype]', 'inherited breast cancer cases [phenotype]', 'tumour-specific aberrations [phenotype]', 'list of likely candidate mutations [phenotype]', 'germline-transmitted mutations [phenotype]', 'systematic asymmetries in heritable mutations [genetic feature]', 'targeted heritable mutations [phenotype]', 'fixed mutations in ancestral line [phenotype]', 'various types of mutation [phenotype]', 'several types of mutation [phenotype]', 'screening of these exons [phenotype]', 'many human cancers [phenotype]', 'difficulties to differentiate mutations [phenotype]', 'base substitutions, deletions, insertions, and translocations [phenotype]', 'specific mutational classes [phenotype]', 'germline mutations [phenotype]', 'other types of mutation [phenotype]', 'unmasking of deleterious mutations [phenotype]']}\n",
    "\n",
    "Explanation of correct clustering:\n",
    "\n",
    "[\"**germline mutations [phenotype]**\", \"germline-transmitted mutations [phenotype]\", \"mutations in the germline [phenotype]\", \"germline transmissible [phenotype]\"] all describe inheritable genetic alterations passed from one generation to the next.\n",
    "[\"More than half of human cancers [phenotype]\", \"**oncological diseases [phenotype]**\", \"many human cancers [phenotype]\"] reference broad groups of malignant diseases occurring in humans.\n",
    "[\"**mutation types [phenotype]**\", \"base substitutions, deletions, insertions, and translocations [phenotype]\", \"various types of mutation [phenotype]\", \"several types of mutation [phenotype]\", \"other types of mutation [phenotype]\", \"certain mutation types [phenotype]\", \"specific mutational classes [phenotype]\"] collectively describe or list categories of genetic changes.\n",
    "[\"difficulties to differentiate mutations [phenotype]\", \"**difficulty in detecting mutations [phenotype]**\"] denote challenges in identifying or distinguishing specific genetic variants.\n",
    "[\"inherited breast cancer cases [phenotype]\", \"**hereditary breast cancers [phenotype]**\"] explicitly point to breast cancer instances where genetic risk is inherited.\n",
    "Notes on Separation Certain single-element entries might share partial similarity, such as \"somatic and germline mutations [phenotype]\" or \"targeted heritable mutations [phenotype],\" but remain separate because of additional qualifiers or exact term differences\n",
    "\n",
    "Example 2, incorrect clustering:\n",
    "input:\n",
    "{\"165162\": [[\"**internalized cargo [metabolite]**\", \"endocytosed material [metabolite]\", \"Internalized cargoes [metabolite]\"], [\"**Internalization of endocytic tracer FM4-64 [metabolite]**\", \"internalisation of an endocytic tracer dye [metabolite]\"], [\"endocytosis of transferrin [metabolite]\"], [\"cellular uptake of 14 chemicals [metabolite]\"]]}\n",
    "\n",
    "Explanation of incorrect behavior:\n",
    "\n",
    "[\"**internalized cargo [metabolite]**\", \"endocytosed material [metabolite]\", \"Internalized cargoes [metabolite]\"] all describe general endocytic cargo without specifying particular molecules or numeric differences.\n",
    "[\"**Internalization of endocytic tracer FM4-64 [metabolite]**\", \"internalisation of an endocytic tracer dye [metabolite]\"] are clustered wrong, one specifically mentions FM4-64, while the other refers generally to any tracer dye, and they should therefore be separated following strict separation rules.\n",
    "\n",
    "Now read the cluster, validate and output the desired format:\n",
    "\"\"\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 65,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "{\"22996\": [\"Conductance measurements [phenotype]\", \"stomatal conductance, transpiration rate, hydraulic conductivity and photosynthetic efficiency [phenotype]\", \"E and g s [phenotype]\", \"regulation of stomatal conductance through FER [phenotype]\", \"engineering stomatal conductance [phenotype]\", \"stomatal conductance, transpiration rate, relative-water content of plant(s) leaf [phenotype]\", \"stomatal conductance measurements [phenotype]\", \"measuring stomatal conductance [phenotype]\", \"K(leaf) values [phenotype]\", \"direct measurements of g s [phenotype]\", \"AtPIP2;5 regulating gm [phenotype]\", \"conductance levels [phenotype]\", \"number of open stomata and amount of water loss through transpiration [phenotype]\", \"methods to quickly assess the stomatal conductance (g\\u209b) [phenotype]\", \"time-resolved measurements of stomatal conductance [phenotype]\", \"time-resolved stomatal conductance analyses [phenotype]\", \"stomatal conductance analyses [phenotype]\", \"transpiration and stomatal conductance values [phenotype]\", \"kinetic stomatal conductance analyses [phenotype]\", \"derived model [phenotype]\", \"measured stomatal conductance [phenotype]\", \"WUE and Stomatal conductance [phenotype]\", \"stomatal conductances [phenotype]\", \"both SD and photosynthetic gas exchange traits [phenotype]\", \"consideration of stomatal conductance [phenotype]\"]}\n"
     ]
    }
   ],
   "source": [
    "#make user_message_2 which should be Input 2 from the df\n",
    "variable_user_message = df.iloc[1]['Output']\n",
    "print(variable_user_message)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [],
   "source": [
    "from openai import OpenAI\n",
    "\n",
    "\n",
    "run = False\n",
    "\n",
    "if run:\n",
    "    client = OpenAI()\n",
    "\n",
    "    response = client.chat.completions.create(\n",
    "      model=\"o3-mini\",\n",
    "      messages=[\n",
    "        {\n",
    "          \"role\": \"developer\",\n",
    "          \"content\": [\n",
    "            {\n",
    "              \"type\": \"text\",\n",
    "              \"text\": constant_system_message\n",
    "            }\n",
    "          ]\n",
    "        },\n",
    "        {\n",
    "          \"role\": \"user\",\n",
    "          \"content\": [\n",
    "            {\n",
    "              \"type\": \"text\",\n",
    "              \"text\": constant_user_message\n",
    "            }\n",
    "          ]\n",
    "        },\n",
    "        {\n",
    "          \"role\": \"user\",\n",
    "          \"content\": [\n",
    "            {\n",
    "              \"type\": \"text\",\n",
    "              \"text\": variable_user_message\n",
    "            }\n",
    "          ]\n",
    "        }\n",
    "      ],\n",
    "      response_format={\n",
    "        \"type\": \"json_object\"\n",
    "      },\n",
    "      reasoning_effort=\"high\"\n",
    "    )"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Redo: Cost estimate for o3-mini"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 104,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      " \n",
      "o3-mini\n",
      "individual requests:  802\n",
      "Total input tokens: 3642512\n",
      "Total output tokens: 116118\n",
      "Estimated input cost for o3-mini: $4.01\n",
      "Estimated output cost for o3-mini: $1.53\n",
      "Total estimated cost for o3-mini: $5.54\n",
      " \n",
      "[3575, 553, 261, 1238, 57204, 70116, 306, 97243, 6804, 36598, 26291, 13, 4886, 5296, 382, 316, 19723, 6771, 26291, 2049, 38971, 137858, 316, 290, 3992, 20959, 734, 197, 16, 13, 197, 50196, 133290, 114005, 93032, 25, 30141, 3331, 290, 3149, 27179, 11, 3463, 2140, 36598, 5941, 11, 59422, 295, 2201, 350, 733, 7443, 15054, 19504, 2238, 472, 47236, 11, 107300, 11, 6145, 111046, 929, 11, 103951, 6294, 197, 17, 13, 197, 64810, 350, 1353, 18929, 7926, 12167, 172201, 25, 115198, 483, 2647, 36598, 5941, 52178, 413, 12989, 306, 13574, 51310, 558, 18, 13, 5934, 12, 32535, 39612, 25, 121693, 115198, 483, 52077, 19504, 11, 1543, 165192, 19873, 11, 503, 192156, 1511, 2647, 8896, 558, 197, 19, 13, 49448, 1876, 7983, 50901, 536, 25, 3756, 7116, 19723, 1920, 4732, 4717, 306, 2684, 3566, 1327, 2236, 1023, 5143, 261, 5355, 2195, 503, 2914, 558, 197, 20, 13, 197, 64810, 40762, 11105, 33677, 830, 6968, 1276, 11105, 7849, 289, 25, 12817, 3566, 26291, 484, 6716, 316, 290, 2684, 36598, 8866, 11, 2273, 11, 10915, 503, 8496, 558, 197, 21, 13, 24747, 4649, 524, 220, 1353, 4, 87969, 25, 4296, 306, 1952, 3291, 16928, 11, 52178, 2475, 26291, 306, 13574, 51310, 558, 197, 22, 13, 22447, 109918, 21193, 4833, 25, 3004, 620, 4732, 1757, 413, 17882, 11, 860, 91135, 1757, 413, 17882, 11, 326, 860, 26291, 1757, 413, 92895, 558, 23, 13, 19461, 52178, 40151, 1645, 11081, 19723, 26958, 326, 469, 8722, 480, 483, 188301, 787, 538, 1354, 382, 945, 1572, 1001, 9481, 306, 484, 5024, 19723, 558, 197, 24, 13, 145342, 26193, 25, 30141, 622, 4376, 306, 4529, 8205, 6011, 13, 52178, 20180, 192663, 24657, 558]\n",
      " \n",
      "o3-mini-batch\n",
      "individual requests:  802\n",
      "Total input tokens: 3642512\n",
      "Total output tokens: 116118\n",
      "Estimated input cost for o3-mini-batch: $2.00\n",
      "Estimated output cost for o3-mini-batch: $0.77\n",
      "Total estimated cost for o3-mini-batch: $2.77\n",
      " \n",
      "[3575, 553, 261, 1238, 57204, 70116, 306, 97243, 6804, 36598, 26291, 13, 4886, 5296, 382, 316, 19723, 6771, 26291, 2049, 38971, 137858, 316, 290, 3992, 20959, 734, 197, 16, 13, 197, 50196, 133290, 114005, 93032, 25, 30141, 3331, 290, 3149, 27179, 11, 3463, 2140, 36598, 5941, 11, 59422, 295, 2201, 350, 733, 7443, 15054, 19504, 2238, 472, 47236, 11, 107300, 11, 6145, 111046, 929, 11, 103951, 6294, 197, 17, 13, 197, 64810, 350, 1353, 18929, 7926, 12167, 172201, 25, 115198, 483, 2647, 36598, 5941, 52178, 413, 12989, 306, 13574, 51310, 558, 18, 13, 5934, 12, 32535, 39612, 25, 121693, 115198, 483, 52077, 19504, 11, 1543, 165192, 19873, 11, 503, 192156, 1511, 2647, 8896, 558, 197, 19, 13, 49448, 1876, 7983, 50901, 536, 25, 3756, 7116, 19723, 1920, 4732, 4717, 306, 2684, 3566, 1327, 2236, 1023, 5143, 261, 5355, 2195, 503, 2914, 558, 197, 20, 13, 197, 64810, 40762, 11105, 33677, 830, 6968, 1276, 11105, 7849, 289, 25, 12817, 3566, 26291, 484, 6716, 316, 290, 2684, 36598, 8866, 11, 2273, 11, 10915, 503, 8496, 558, 197, 21, 13, 24747, 4649, 524, 220, 1353, 4, 87969, 25, 4296, 306, 1952, 3291, 16928, 11, 52178, 2475, 26291, 306, 13574, 51310, 558, 197, 22, 13, 22447, 109918, 21193, 4833, 25, 3004, 620, 4732, 1757, 413, 17882, 11, 860, 91135, 1757, 413, 17882, 11, 326, 860, 26291, 1757, 413, 92895, 558, 23, 13, 19461, 52178, 40151, 1645, 11081, 19723, 26958, 326, 469, 8722, 480, 483, 188301, 787, 538, 1354, 382, 945, 1572, 1001, 9481, 306, 484, 5024, 19723, 558, 197, 24, 13, 145342, 26193, 25, 30141, 622, 4376, 306, 4529, 8205, 6011, 13, 52178, 20180, 192663, 24657, 558]\n",
      " \n",
      "o1\n",
      "individual requests:  802\n",
      "Total input tokens: 3642512\n",
      "Total output tokens: 116118\n",
      "Estimated input cost for o1: $55.00\n",
      "Estimated output cost for o1: $20.90\n",
      "Total estimated cost for o1: $75.90\n",
      " \n",
      "[3575, 553, 261, 1238, 57204, 70116, 306, 97243, 6804, 36598, 26291, 13, 4886, 5296, 382, 316, 19723, 6771, 26291, 2049, 38971, 137858, 316, 290, 3992, 20959, 734, 197, 16, 13, 197, 50196, 133290, 114005, 93032, 25, 30141, 3331, 290, 3149, 27179, 11, 3463, 2140, 36598, 5941, 11, 59422, 295, 2201, 350, 733, 7443, 15054, 19504, 2238, 472, 47236, 11, 107300, 11, 6145, 111046, 929, 11, 103951, 6294, 197, 17, 13, 197, 64810, 350, 1353, 18929, 7926, 12167, 172201, 25, 115198, 483, 2647, 36598, 5941, 52178, 413, 12989, 306, 13574, 51310, 558, 18, 13, 5934, 12, 32535, 39612, 25, 121693, 115198, 483, 52077, 19504, 11, 1543, 165192, 19873, 11, 503, 192156, 1511, 2647, 8896, 558, 197, 19, 13, 49448, 1876, 7983, 50901, 536, 25, 3756, 7116, 19723, 1920, 4732, 4717, 306, 2684, 3566, 1327, 2236, 1023, 5143, 261, 5355, 2195, 503, 2914, 558, 197, 20, 13, 197, 64810, 40762, 11105, 33677, 830, 6968, 1276, 11105, 7849, 289, 25, 12817, 3566, 26291, 484, 6716, 316, 290, 2684, 36598, 8866, 11, 2273, 11, 10915, 503, 8496, 558, 197, 21, 13, 24747, 4649, 524, 220, 1353, 4, 87969, 25, 4296, 306, 1952, 3291, 16928, 11, 52178, 2475, 26291, 306, 13574, 51310, 558, 197, 22, 13, 22447, 109918, 21193, 4833, 25, 3004, 620, 4732, 1757, 413, 17882, 11, 860, 91135, 1757, 413, 17882, 11, 326, 860, 26291, 1757, 413, 92895, 558, 23, 13, 19461, 52178, 40151, 1645, 11081, 19723, 26958, 326, 469, 8722, 480, 483, 188301, 787, 538, 1354, 382, 945, 1572, 1001, 9481, 306, 484, 5024, 19723, 558, 197, 24, 13, 145342, 26193, 25, 30141, 622, 4376, 306, 4529, 8205, 6011, 13, 52178, 20180, 192663, 24657, 558]\n",
      " \n",
      "o1-batch\n",
      "individual requests:  802\n",
      "Total input tokens: 3642512\n",
      "Total output tokens: 116118\n",
      "Estimated input cost for o1-batch: $27.50\n",
      "Estimated output cost for o1-batch: $10.45\n",
      "Total estimated cost for o1-batch: $37.95\n",
      " \n",
      "[3575, 553, 261, 1238, 57204, 70116, 306, 97243, 6804, 36598, 26291, 13, 4886, 5296, 382, 316, 19723, 6771, 26291, 2049, 38971, 137858, 316, 290, 3992, 20959, 734, 197, 16, 13, 197, 50196, 133290, 114005, 93032, 25, 30141, 3331, 290, 3149, 27179, 11, 3463, 2140, 36598, 5941, 11, 59422, 295, 2201, 350, 733, 7443, 15054, 19504, 2238, 472, 47236, 11, 107300, 11, 6145, 111046, 929, 11, 103951, 6294, 197, 17, 13, 197, 64810, 350, 1353, 18929, 7926, 12167, 172201, 25, 115198, 483, 2647, 36598, 5941, 52178, 413, 12989, 306, 13574, 51310, 558, 18, 13, 5934, 12, 32535, 39612, 25, 121693, 115198, 483, 52077, 19504, 11, 1543, 165192, 19873, 11, 503, 192156, 1511, 2647, 8896, 558, 197, 19, 13, 49448, 1876, 7983, 50901, 536, 25, 3756, 7116, 19723, 1920, 4732, 4717, 306, 2684, 3566, 1327, 2236, 1023, 5143, 261, 5355, 2195, 503, 2914, 558, 197, 20, 13, 197, 64810, 40762, 11105, 33677, 830, 6968, 1276, 11105, 7849, 289, 25, 12817, 3566, 26291, 484, 6716, 316, 290, 2684, 36598, 8866, 11, 2273, 11, 10915, 503, 8496, 558, 197, 21, 13, 24747, 4649, 524, 220, 1353, 4, 87969, 25, 4296, 306, 1952, 3291, 16928, 11, 52178, 2475, 26291, 306, 13574, 51310, 558, 197, 22, 13, 22447, 109918, 21193, 4833, 25, 3004, 620, 4732, 1757, 413, 17882, 11, 860, 91135, 1757, 413, 17882, 11, 326, 860, 26291, 1757, 413, 92895, 558, 23, 13, 19461, 52178, 40151, 1645, 11081, 19723, 26958, 326, 469, 8722, 480, 483, 188301, 787, 538, 1354, 382, 945, 1572, 1001, 9481, 306, 484, 5024, 19723, 558, 197, 24, 13, 145342, 26193, 25, 30141, 622, 4376, 306, 4529, 8205, 6011, 13, 52178, 20180, 192663, 24657, 558]\n"
     ]
    }
   ],
   "source": [
    "import tiktoken\n",
    "\n",
    "# with open(\"data/api_key.txt\", \"r\") as f:\n",
    "#     api_key = f.read()\n",
    "\n",
    "def prepare_embedding_data(df, system_message, user_message, assistant_response, model_name, input_cost_per_token, output_cost_per_token):\n",
    "    \"\"\"Estimate the cost of embedding the data.\"\"\"\n",
    "    print(\" \")\n",
    "    print(model_name)\n",
    "    encoding = tiktoken.get_encoding(\"o200k_base\")\n",
    "    embedding_data = []\n",
    "    total_input_tokens = 0\n",
    "    total_output_tokens = 0\n",
    "    print(\"individual requests: \", len(df))\n",
    "    for sub_list in zip(df[\"Input\"], df[\"Output\"]):\n",
    "        \n",
    "        for item in sub_list:\n",
    "            n_tokens = len(encoding.encode(item))\n",
    "            total_input_tokens += n_tokens\n",
    "        total_input_tokens += len(encoding.encode(system_message))\n",
    "        total_input_tokens += len(encoding.encode(user_message))\n",
    "        total_input_tokens += len(encoding.encode(assistant_response))\n",
    "        total_output_tokens += len(encoding.encode(sub_list[1])) # the output format of my prompt is pretty short, so i multiply by 3 to overestimate\n",
    "        \n",
    "    print(f\"Total input tokens: {total_input_tokens}\")\n",
    "    print(f\"Total output tokens: {total_output_tokens}\")\n",
    "\n",
    "    # Calculate costs\n",
    "    input_cost = (total_input_tokens / 1_000_000) * input_cost_per_token  \n",
    "    output_cost = ((total_output_tokens*3) / 1_000_000) * output_cost_per_token \n",
    "    total_cost = input_cost + output_cost\n",
    "\n",
    "    print(f\"Estimated input cost for {model_name}: ${input_cost:.2f}\")\n",
    "    print(f\"Estimated output cost for {model_name}: ${output_cost:.2f}\")\n",
    "    print(f\"Total estimated cost for {model_name}: ${total_cost:.2f}\")\n",
    "    print(\" \")\n",
    "    print(encoding.encode(constant_system_message))\n",
    "    return None\n",
    "# Example usage\n",
    "\n",
    "constant_fine_tuning_format_promt = \"\"\n",
    "o3_mini_cost = prepare_embedding_data(df, constant_system_message, constant_user_message, constant_fine_tuning_format_promt, \"o3-mini\", 1.10, 4.40)\n",
    "o3_mini_batch_cost = prepare_embedding_data(df, constant_system_message, constant_user_message, constant_fine_tuning_format_promt, \"o3-mini-batch\", 0.55, 2.20)\n",
    "o1_cost = prepare_embedding_data(df, constant_system_message, constant_user_message, constant_fine_tuning_format_promt, \"o1\", 15.10, 60)\n",
    "o1_batch_cost = prepare_embedding_data(df, constant_system_message, constant_user_message, constant_fine_tuning_format_promt, \"o1-batch\", 7.55, 30)"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "PhD",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.12.2"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
