{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Generating training data for fine tuning using o3-mini\n",
    "\n",
    "In this notebook i show my suggestion for making a fine-tuning dataset for 4o-mini, using the power of o3-mini.\n",
    "\n",
    "The first step is getting the data.\n",
    "\n",
    "We take the clustering made by Manoj with o3-mini, and then ask o3-mini with reasoning set to high to go through its own results for validation, make corrections if needed, and then explain why this is wrong as a user message.\n",
    "\n",
    "This message will be used to simulate a conversation with 4o-mini to teach it clustering patterns that we want to highlight. We are essentially trying to gaslight GPT into making better clusters. Making it doubt itself.  \n",
    "\n",
    "With fine tuing we therefore aim to make the predictions of 4o-mini be more similar to o3-mini, but at a lower cost. \n",
    "\n",
    "\n",
    "I think the bonus with this approach, is that we don't need annotate ourselves, just pick the ones we agree with the validation, which is much faster.\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Generate dataset"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 105,
   "metadata": {},
   "outputs": [],
   "source": [
    "#This is a script that loads a pandas dataframe from csv, converts the string formatted as a list into a list of lists. The list of lists is then flattened into a single list.\n",
    "\n",
    "import pandas as pd\n",
    "import random\n",
    "import json\n",
    "df = pd.read_csv('o3mini_maxclust_80_subset_random_1000_nocutoff.csv', index_col=None)\n",
    "\n",
    "#rename the column \"Entity Name\" to \"Entity\" and \"Group Items\" to \"output\"\n",
    "df = df.rename(columns={\"Group Items\": \"Output\"})\n",
    "\n",
    "# Convert the string formatted as a list into a list of lists\n",
    "df['Output'] = df['Output'].apply(lambda x: eval(x))\n",
    "\n",
    "# Flatten the list of lists into a single list\n",
    "df['Input'] = df['Output'].apply(lambda x: [item.strip(\"*\") for sublist in x for item in sublist])\n",
    "\n",
    "#randomize the order of the single list\n",
    "df['Input'] = df['Input'].apply(lambda x: random.sample(x, len(x)))\n",
    "df = df[df['Input'].apply(len) < 30]\n",
    "# make a new column, where internal lists of length 1 are removed \n",
    "df['Output>1'] = df['Output'].apply(lambda x: [item for item in x if len(item) > 1])\n",
    "\n",
    "\n",
    "#zip the Group_Items_no_singles with the Entity Name \n",
    "df[\"Output\"] = df.apply(lambda row: {row[\"Entity Name\"]: row['Output']}, axis=1)\n",
    "df[\"Input\"] = df.apply(lambda row: {row[\"Entity Name\"]: row['Input']}, axis=1)\n",
    "df['Output>1'] = df.apply(lambda row: {row[\"Entity Name\"]: row['Output>1']}, axis=1)\n",
    "\n",
    "\n",
    "\n",
    "#filter for single_list length < 30\n",
    "\n",
    "\n",
    "\n",
    "#convert the columns to json strings\n",
    "df[\"Output\"] = df[\"Output\"].apply(lambda x: json.dumps(x) if isinstance(x, dict) else x)\n",
    "df[\"Input\"] = df[\"Input\"].apply(lambda x: json.dumps(x) if isinstance(x, dict) else x)\n",
    "df[\"Output>1\"] = df[\"Output>1\"].apply(lambda x: json.dumps(x) if isinstance(x, dict) else x)\n",
    "\n",
    "#Change order of columns to have input, then output, then output>1, then entity name\n",
    "df = df[['Entity Name', 'Input', 'Output', 'Output>1']]\n",
    "\n",
    "#filter by Output>1 length of the list > 1\n",
    "#use column Output, but use only the ones where it clusteres multiple items together"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 106,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Entity Name</th>\n",
       "      <th>Input</th>\n",
       "      <th>Output</th>\n",
       "      <th>Output&gt;1</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>58963</td>\n",
       "      <td>{\"58963\": [\"Gardea-Torresdey et al. [researche...</td>\n",
       "      <td>{\"58963\": [[\"Gardea-Torresdey et al. [research...</td>\n",
       "      <td>{\"58963\": []}</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>22996</td>\n",
       "      <td>{\"22996\": [\"number of open stomata and amount ...</td>\n",
       "      <td>{\"22996\": [[\"**transpiration and stomatal cond...</td>\n",
       "      <td>{\"22996\": [[\"**transpiration and stomatal cond...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>100116</td>\n",
       "      <td>{\"100116\": [\"modern clade [gene]\", \"Clade SL [...</td>\n",
       "      <td>{\"100116\": [[\"Clade M [clade]\"], [\"Clade SL [c...</td>\n",
       "      <td>{\"100116\": []}</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>133255</td>\n",
       "      <td>{\"133255\": [\"LPP\\u10b2 [gene]\", \"SUPPRESSOR OF...</td>\n",
       "      <td>{\"133255\": [[\"**PLSP2A [gene]**\", \"PLSP2A gene...</td>\n",
       "      <td>{\"133255\": [[\"**PLSP2A [gene]**\", \"PLSP2A gene...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>2276</td>\n",
       "      <td>{\"2276\": [\"fc2 phenotypes [phenotype]\", \"100% ...</td>\n",
       "      <td>{\"2276\": [[\"F1 trichome phenotype [phenotype]\"...</td>\n",
       "      <td>{\"2276\": []}</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>995</th>\n",
       "      <td>31615</td>\n",
       "      <td>{\"31615\": [\"auxin-regulated endocytosis [proce...</td>\n",
       "      <td>{\"31615\": [[\"auxin-mediated modulation of the ...</td>\n",
       "      <td>{\"31615\": [[\"auxin-dependent inhibition of end...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>996</th>\n",
       "      <td>157227</td>\n",
       "      <td>{\"157227\": [\"ZmSAUR1 [gene]\", \"ZmSAUR1 (Zea ma...</td>\n",
       "      <td>{\"157227\": [[\"GmSAUR [gene]\"], [\"StSAUR [gene]...</td>\n",
       "      <td>{\"157227\": [[\"ZmSAUR1 (Zea mays SAURs) [gene]\"...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>997</th>\n",
       "      <td>131559</td>\n",
       "      <td>{\"131559\": [\"cytochrome b(5) [gene]\", \"BtuN ge...</td>\n",
       "      <td>{\"131559\": [[\"BtuB gene(s) [gene]\"], [\"BtuC ge...</td>\n",
       "      <td>{\"131559\": []}</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>998</th>\n",
       "      <td>99091</td>\n",
       "      <td>{\"99091\": [\"D. oleifera genome [organism]\", \"O...</td>\n",
       "      <td>{\"99091\": [[\"Genome sequencing of O. fragrans ...</td>\n",
       "      <td>{\"99091\": [[\"**O. fragrans 'Liuyejingui' (OFL)...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>999</th>\n",
       "      <td>65306</td>\n",
       "      <td>{\"65306\": [\"NF-YB/NF-YC dimer [protein complex...</td>\n",
       "      <td>{\"65306\": [[\"**CO/NF-Y complex [protein comple...</td>\n",
       "      <td>{\"65306\": [[\"**CO/NF-Y complex [protein comple...</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>802 rows × 4 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "     Entity Name                                              Input  \\\n",
       "0          58963  {\"58963\": [\"Gardea-Torresdey et al. [researche...   \n",
       "2          22996  {\"22996\": [\"number of open stomata and amount ...   \n",
       "3         100116  {\"100116\": [\"modern clade [gene]\", \"Clade SL [...   \n",
       "4         133255  {\"133255\": [\"LPP\\u10b2 [gene]\", \"SUPPRESSOR OF...   \n",
       "5           2276  {\"2276\": [\"fc2 phenotypes [phenotype]\", \"100% ...   \n",
       "..           ...                                                ...   \n",
       "995        31615  {\"31615\": [\"auxin-regulated endocytosis [proce...   \n",
       "996       157227  {\"157227\": [\"ZmSAUR1 [gene]\", \"ZmSAUR1 (Zea ma...   \n",
       "997       131559  {\"131559\": [\"cytochrome b(5) [gene]\", \"BtuN ge...   \n",
       "998        99091  {\"99091\": [\"D. oleifera genome [organism]\", \"O...   \n",
       "999        65306  {\"65306\": [\"NF-YB/NF-YC dimer [protein complex...   \n",
       "\n",
       "                                                Output  \\\n",
       "0    {\"58963\": [[\"Gardea-Torresdey et al. [research...   \n",
       "2    {\"22996\": [[\"**transpiration and stomatal cond...   \n",
       "3    {\"100116\": [[\"Clade M [clade]\"], [\"Clade SL [c...   \n",
       "4    {\"133255\": [[\"**PLSP2A [gene]**\", \"PLSP2A gene...   \n",
       "5    {\"2276\": [[\"F1 trichome phenotype [phenotype]\"...   \n",
       "..                                                 ...   \n",
       "995  {\"31615\": [[\"auxin-mediated modulation of the ...   \n",
       "996  {\"157227\": [[\"GmSAUR [gene]\"], [\"StSAUR [gene]...   \n",
       "997  {\"131559\": [[\"BtuB gene(s) [gene]\"], [\"BtuC ge...   \n",
       "998  {\"99091\": [[\"Genome sequencing of O. fragrans ...   \n",
       "999  {\"65306\": [[\"**CO/NF-Y complex [protein comple...   \n",
       "\n",
       "                                              Output>1  \n",
       "0                                        {\"58963\": []}  \n",
       "2    {\"22996\": [[\"**transpiration and stomatal cond...  \n",
       "3                                       {\"100116\": []}  \n",
       "4    {\"133255\": [[\"**PLSP2A [gene]**\", \"PLSP2A gene...  \n",
       "5                                         {\"2276\": []}  \n",
       "..                                                 ...  \n",
       "995  {\"31615\": [[\"auxin-dependent inhibition of end...  \n",
       "996  {\"157227\": [[\"ZmSAUR1 (Zea mays SAURs) [gene]\"...  \n",
       "997                                     {\"131559\": []}  \n",
       "998  {\"99091\": [[\"**O. fragrans 'Liuyejingui' (OFL)...  \n",
       "999  {\"65306\": [[\"**CO/NF-Y complex [protein comple...  \n",
       "\n",
       "[802 rows x 4 columns]"
      ]
     },
     "execution_count": 106,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df\n",
    "#note i don't really use Outputs>1 in this workflow."
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Prompts for making fine-tuning dataset\n",
    "\n",
    "- constant_system_message is unchanged\n",
    "- constant_user_message is only slightly modified from Manoj's promts, in the beginning and the end.\n",
    "- variable_user_message is \"Output\" for each entry in the dataframe\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 63,
   "metadata": {},
   "outputs": [],
   "source": [
    "constant_system_message = \"\"\"\n",
    "You are senior scientist in plant biology. You NEED to meticulously VALIDATE given clustering output, and provide a corrected clustering if needed. Explain which groupings are wrong and provide final output. You MUST write response like a senior human plant scientist.\n",
    "\n",
    "The following are the entity clustering guidelines:\n",
    "\t1.\tExact Phrase Matching Matters: Always consider the full phrase, including key biological terms, bracketed text (ignoring minor differences such as spacing, punctuation, correct abbreviations, plurality).\n",
    "\t2.\tStrict (100%) Key Term Separation: Entities with different biological terms MUST be placed in separate clusters.\n",
    "3. Sub-identifier separation: Separate Entities with numeric differences, sub-identifiers, or qualifiers into different groups.\n",
    "\t4.\tAvoid False Similarity: Do NOT cluster two items together in same group just because they share a common word or term.\n",
    "\t5.\tStrict Synonym/Near-Synonym Grouping: Only group entities that refer to the same biological structure, process, meaning or concept.\n",
    "\t6.\tMaintain 100% Precision: When in even small doubt, MUST place entities in separate clusters.\n",
    "\t7.\tPreserve Original Data: No new items should be introduced, no duplicates should be introduced, and no entities should be omitted.\n",
    "8. YOU MUST pickup most appropriate and easy-to-understand cluster representative and enclose it with '**', if there is more than one entity in that particular cluster. For example, pick the full term instead of an abbreviation.\n",
    "\t9.\tOutput Format: Always return results in valid JSON format. MUST USE GIVEN KEY.\n",
    "\n",
    "Your main task is to read the clustered entities and output the validation in the below mentioned format.\n",
    "\n",
    "The goal:\n",
    "These results came from a smaller LLM, and i want you to take a look at it.\n",
    "We  your expertise in plant biology, to generate prompts for fine tuning the smaller LLM and mimick your patterns for this specific task.\n",
    "I want you to validate the clustering of this smaller model, and provide a corrected clustering if needed.\n",
    "Then you should imagine that you are a user that want to explain to the smaller LLM what it is doing wrong, to give it the final output. You should never mention the smaller LLM, but instead just pretend you are a user. \n",
    "An example of this would be:\n",
    "'I think you are wrong, cluster X is actually different from cluster Y because of reasons. Cluster X and Y should be clustered seperately'.\n",
    "If the smaller model is correct however, you should write something like:\n",
    "'Great job, that looks correct. Now output the correct format, but only output clusters with more than one member.'\n",
    "The expected input format:\n",
    "\n",
    "{\n",
    "  \"submission_id_1\": {\n",
    "    \"clusters\": [\n",
    "      [\n",
    "        \"**cluster_item1**\",\n",
    "        \"cluster_item2\"\n",
    "      ],\n",
    "      [\n",
    "        \"cluster_item3\"\n",
    "      ]\n",
    "    ]\n",
    "  }\n",
    "}\n",
    "\n",
    "Return format:\n",
    "\n",
    "If the clustering is correct:\n",
    "{\n",
    "  \"submission_id_1\": {\n",
    "    \"is_correct\": \"True\",\n",
    "    \"user_prompt\": \"This looks correct to me, output the clusters that have more than 1 member in the correct json format.\",\n",
    "    \"clusters\": [\n",
    "      [\n",
    "        \"cluster_item1\",\n",
    "        \"cluster_item2\"\n",
    "      ]\n",
    "    ]\n",
    "  }\n",
    "}\n",
    "\n",
    "If the clustering is incorrect:\n",
    "\n",
    "{\n",
    "  \"submission_id_1\": {\n",
    "    \"is_correct\": \"False\",\n",
    "    \"user_prompt\": \"This looks incorrect to me, <insert reason here>. Output the clusters that have more than 1 member in the correct json format.\",\n",
    "    \"clusters\": [\n",
    "      [\n",
    "        \"cluster_item2\",\n",
    "        \"cluster_item3\"\n",
    "      ]\n",
    "    ]\n",
    "  }\n",
    "}\n",
    "\n",
    "Warnings:\n",
    "\n",
    "Be completely sure to not forget any nodes from the input list.\n",
    "Remember not to base corrections on higher domain level knowledge, that the smaller model might not have, but rather the semantic meaning of the terms.\n",
    "is_correct should be returned as a string \"True\" or \"False\", and only one of these two.\n",
    "If there is any doubt if two groups should cluster together, cluster them separately.\n",
    "Remember to select group representative by enclosing entity in \"**\"\n",
    "\n",
    "\"\"\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 64,
   "metadata": {},
   "outputs": [],
   "source": [
    "constant_user_message = \"\"\"  \n",
    "Here are 2 examples of validation behavior:\n",
    "\n",
    "Example 1, correct behavior:\n",
    "\n",
    "input:\n",
    "{120406: ['difficulty in detecting mutations [phenotype]', 'hereditary breast cancers [phenotype]', 'mutational status of parental tumor cells [phenotype]', 'certain mutation types [phenotype]', 'Well-characterized mutations [phenotype]', 'alternative classifications [phenotype]', 'unique molecular signature [phenotype]', 'oncological diseases [phenotype]', 'independent mutation data [phenotype]', 'inherited disease mutation databases [database]', 'mutational signatures [mutation]', 'mutation types [phenotype]', 'More than half of human cancers [phenotype]', 'somatic and germline mutations [phenotype]', 'Identified mutations [phenotype]', 'lung cancer patients [phenotype]', 'germline transmissible [phenotype]', 'mutations in the germline [phenotype]', 'sporadic breast cancers [phenotype]', 'segregating mutations [phenotype]', 'breast/ovarian cancers [phenotype]', 'non-small-cell lung carcinoma [phenotype]', 'inherited breast cancer cases [phenotype]', 'tumour-specific aberrations [phenotype]', 'list of likely candidate mutations [phenotype]', 'germline-transmitted mutations [phenotype]', 'systematic asymmetries in heritable mutations [genetic feature]', 'targeted heritable mutations [phenotype]', 'fixed mutations in ancestral line [phenotype]', 'various types of mutation [phenotype]', 'several types of mutation [phenotype]', 'screening of these exons [phenotype]', 'many human cancers [phenotype]', 'difficulties to differentiate mutations [phenotype]', 'base substitutions, deletions, insertions, and translocations [phenotype]', 'specific mutational classes [phenotype]', 'germline mutations [phenotype]', 'other types of mutation [phenotype]', 'unmasking of deleterious mutations [phenotype]']}\n",
    "\n",
    "Explanation of correct clustering:\n",
    "\n",
    "[\"**germline mutations [phenotype]**\", \"germline-transmitted mutations [phenotype]\", \"mutations in the germline [phenotype]\", \"germline transmissible [phenotype]\"] all describe inheritable genetic alterations passed from one generation to the next.\n",
    "[\"More than half of human cancers [phenotype]\", \"**oncological diseases [phenotype]**\", \"many human cancers [phenotype]\"] reference broad groups of malignant diseases occurring in humans.\n",
    "[\"**mutation types [phenotype]**\", \"base substitutions, deletions, insertions, and translocations [phenotype]\", \"various types of mutation [phenotype]\", \"several types of mutation [phenotype]\", \"other types of mutation [phenotype]\", \"certain mutation types [phenotype]\", \"specific mutational classes [phenotype]\"] collectively describe or list categories of genetic changes.\n",
    "[\"difficulties to differentiate mutations [phenotype]\", \"**difficulty in detecting mutations [phenotype]**\"] denote challenges in identifying or distinguishing specific genetic variants.\n",
    "[\"inherited breast cancer cases [phenotype]\", \"**hereditary breast cancers [phenotype]**\"] explicitly point to breast cancer instances where genetic risk is inherited.\n",
    "Notes on Separation Certain single-element entries might share partial similarity, such as \"somatic and germline mutations [phenotype]\" or \"targeted heritable mutations [phenotype],\" but remain separate because of additional qualifiers or exact term differences\n",
    "\n",
    "Example 2, incorrect clustering:\n",
    "input:\n",
    "{\"165162\": [[\"**internalized cargo [metabolite]**\", \"endocytosed material [metabolite]\", \"Internalized cargoes [metabolite]\"], [\"**Internalization of endocytic tracer FM4-64 [metabolite]**\", \"internalisation of an endocytic tracer dye [metabolite]\"], [\"endocytosis of transferrin [metabolite]\"], [\"cellular uptake of 14 chemicals [metabolite]\"]]}\n",
    "\n",
    "Explanation of incorrect behavior:\n",
    "\n",
    "[\"**internalized cargo [metabolite]**\", \"endocytosed material [metabolite]\", \"Internalized cargoes [metabolite]\"] all describe general endocytic cargo without specifying particular molecules or numeric differences.\n",
    "[\"**Internalization of endocytic tracer FM4-64 [metabolite]**\", \"internalisation of an endocytic tracer dye [metabolite]\"] are clustered wrong, one specifically mentions FM4-64, while the other refers generally to any tracer dye, and they should therefore be separated following strict separation rules.\n",
    "\n",
    "Now read the cluster, validate and output the desired format:\n",
    "\"\"\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 65,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "{\"22996\": [\"Conductance measurements [phenotype]\", \"stomatal conductance, transpiration rate, hydraulic conductivity and photosynthetic efficiency [phenotype]\", \"E and g s [phenotype]\", \"regulation of stomatal conductance through FER [phenotype]\", \"engineering stomatal conductance [phenotype]\", \"stomatal conductance, transpiration rate, relative-water content of plant(s) leaf [phenotype]\", \"stomatal conductance measurements [phenotype]\", \"measuring stomatal conductance [phenotype]\", \"K(leaf) values [phenotype]\", \"direct measurements of g s [phenotype]\", \"AtPIP2;5 regulating gm [phenotype]\", \"conductance levels [phenotype]\", \"number of open stomata and amount of water loss through transpiration [phenotype]\", \"methods to quickly assess the stomatal conductance (g\\u209b) [phenotype]\", \"time-resolved measurements of stomatal conductance [phenotype]\", \"time-resolved stomatal conductance analyses [phenotype]\", \"stomatal conductance analyses [phenotype]\", \"transpiration and stomatal conductance values [phenotype]\", \"kinetic stomatal conductance analyses [phenotype]\", \"derived model [phenotype]\", \"measured stomatal conductance [phenotype]\", \"WUE and Stomatal conductance [phenotype]\", \"stomatal conductances [phenotype]\", \"both SD and photosynthetic gas exchange traits [phenotype]\", \"consideration of stomatal conductance [phenotype]\"]}\n"
     ]
    }
   ],
   "source": [
    "#make user_message_2 which should be Input 2 from the df\n",
    "variable_user_message = df.iloc[1]['Output']\n",
    "print(variable_user_message)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [],
   "source": [
    "from openai import OpenAI\n",
    "\n",
    "\n",
    "run = False\n",
    "\n",
    "if run:\n",
    "    client = OpenAI()\n",
    "\n",
    "    response = client.chat.completions.create(\n",
    "      model=\"o3-mini\",\n",
    "      messages=[\n",
    "        {\n",
    "          \"role\": \"developer\",\n",
    "          \"content\": [\n",
    "            {\n",
    "              \"type\": \"text\",\n",
    "              \"text\": constant_system_message\n",
    "            }\n",
    "          ]\n",
    "        },\n",
    "        {\n",
    "          \"role\": \"user\",\n",
    "          \"content\": [\n",
    "            {\n",
    "              \"type\": \"text\",\n",
    "              \"text\": constant_user_message\n",
    "            }\n",
    "          ]\n",
    "        },\n",
    "        {\n",
    "          \"role\": \"user\",\n",
    "          \"content\": [\n",
    "            {\n",
    "              \"type\": \"text\",\n",
    "              \"text\": variable_user_message\n",
    "            }\n",
    "          ]\n",
    "        }\n",
    "      ],\n",
    "      response_format={\n",
    "        \"type\": \"json_object\"\n",
    "      },\n",
    "      reasoning_effort=\"high\"\n",
    "    )"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Redo: Cost estimate for o3-mini"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 104,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      " \n",
      "o3-mini\n",
      "individual requests:  802\n",
      "Total input tokens: 3642512\n",
      "Total output tokens: 116118\n",
      "Estimated input cost for o3-mini: $4.01\n",
      "Estimated output cost for o3-mini: $1.53\n",
      "Total estimated cost for o3-mini: $5.54\n",
      " \n",
      "[3575, 553, 261, 1238, 57204, 70116, 306, 97243, 6804, 36598, 26291, 13, 4886, 5296, 382, 316, 19723, 6771, 26291, 2049, 38971, 137858, 316, 290, 3992, 20959, 734, 197, 16, 13, 197, 50196, 133290, 114005, 93032, 25, 30141, 3331, 290, 3149, 27179, 11, 3463, 2140, 36598, 5941, 11, 59422, 295, 2201, 350, 733, 7443, 15054, 19504, 2238, 472, 47236, 11, 107300, 11, 6145, 111046, 929, 11, 103951, 6294, 197, 17, 13, 197, 64810, 350, 1353, 18929, 7926, 12167, 172201, 25, 115198, 483, 2647, 36598, 5941, 52178, 413, 12989, 306, 13574, 51310, 558, 18, 13, 5934, 12, 32535, 39612, 25, 121693, 115198, 483, 52077, 19504, 11, 1543, 165192, 19873, 11, 503, 192156, 1511, 2647, 8896, 558, 197, 19, 13, 49448, 1876, 7983, 50901, 536, 25, 3756, 7116, 19723, 1920, 4732, 4717, 306, 2684, 3566, 1327, 2236, 1023, 5143, 261, 5355, 2195, 503, 2914, 558, 197, 20, 13, 197, 64810, 40762, 11105, 33677, 830, 6968, 1276, 11105, 7849, 289, 25, 12817, 3566, 26291, 484, 6716, 316, 290, 2684, 36598, 8866, 11, 2273, 11, 10915, 503, 8496, 558, 197, 21, 13, 24747, 4649, 524, 220, 1353, 4, 87969, 25, 4296, 306, 1952, 3291, 16928, 11, 52178, 2475, 26291, 306, 13574, 51310, 558, 197, 22, 13, 22447, 109918, 21193, 4833, 25, 3004, 620, 4732, 1757, 413, 17882, 11, 860, 91135, 1757, 413, 17882, 11, 326, 860, 26291, 1757, 413, 92895, 558, 23, 13, 19461, 52178, 40151, 1645, 11081, 19723, 26958, 326, 469, 8722, 480, 483, 188301, 787, 538, 1354, 382, 945, 1572, 1001, 9481, 306, 484, 5024, 19723, 558, 197, 24, 13, 145342, 26193, 25, 30141, 622, 4376, 306, 4529, 8205, 6011, 13, 52178, 20180, 192663, 24657, 558]\n",
      " \n",
      "o3-mini-batch\n",
      "individual requests:  802\n",
      "Total input tokens: 3642512\n",
      "Total output tokens: 116118\n",
      "Estimated input cost for o3-mini-batch: $2.00\n",
      "Estimated output cost for o3-mini-batch: $0.77\n",
      "Total estimated cost for o3-mini-batch: $2.77\n",
      " \n",
      "[3575, 553, 261, 1238, 57204, 70116, 306, 97243, 6804, 36598, 26291, 13, 4886, 5296, 382, 316, 19723, 6771, 26291, 2049, 38971, 137858, 316, 290, 3992, 20959, 734, 197, 16, 13, 197, 50196, 133290, 114005, 93032, 25, 30141, 3331, 290, 3149, 27179, 11, 3463, 2140, 36598, 5941, 11, 59422, 295, 2201, 350, 733, 7443, 15054, 19504, 2238, 472, 47236, 11, 107300, 11, 6145, 111046, 929, 11, 103951, 6294, 197, 17, 13, 197, 64810, 350, 1353, 18929, 7926, 12167, 172201, 25, 115198, 483, 2647, 36598, 5941, 52178, 413, 12989, 306, 13574, 51310, 558, 18, 13, 5934, 12, 32535, 39612, 25, 121693, 115198, 483, 52077, 19504, 11, 1543, 165192, 19873, 11, 503, 192156, 1511, 2647, 8896, 558, 197, 19, 13, 49448, 1876, 7983, 50901, 536, 25, 3756, 7116, 19723, 1920, 4732, 4717, 306, 2684, 3566, 1327, 2236, 1023, 5143, 261, 5355, 2195, 503, 2914, 558, 197, 20, 13, 197, 64810, 40762, 11105, 33677, 830, 6968, 1276, 11105, 7849, 289, 25, 12817, 3566, 26291, 484, 6716, 316, 290, 2684, 36598, 8866, 11, 2273, 11, 10915, 503, 8496, 558, 197, 21, 13, 24747, 4649, 524, 220, 1353, 4, 87969, 25, 4296, 306, 1952, 3291, 16928, 11, 52178, 2475, 26291, 306, 13574, 51310, 558, 197, 22, 13, 22447, 109918, 21193, 4833, 25, 3004, 620, 4732, 1757, 413, 17882, 11, 860, 91135, 1757, 413, 17882, 11, 326, 860, 26291, 1757, 413, 92895, 558, 23, 13, 19461, 52178, 40151, 1645, 11081, 19723, 26958, 326, 469, 8722, 480, 483, 188301, 787, 538, 1354, 382, 945, 1572, 1001, 9481, 306, 484, 5024, 19723, 558, 197, 24, 13, 145342, 26193, 25, 30141, 622, 4376, 306, 4529, 8205, 6011, 13, 52178, 20180, 192663, 24657, 558]\n",
      " \n",
      "o1\n",
      "individual requests:  802\n",
      "Total input tokens: 3642512\n",
      "Total output tokens: 116118\n",
      "Estimated input cost for o1: $55.00\n",
      "Estimated output cost for o1: $20.90\n",
      "Total estimated cost for o1: $75.90\n",
      " \n",
      "[3575, 553, 261, 1238, 57204, 70116, 306, 97243, 6804, 36598, 26291, 13, 4886, 5296, 382, 316, 19723, 6771, 26291, 2049, 38971, 137858, 316, 290, 3992, 20959, 734, 197, 16, 13, 197, 50196, 133290, 114005, 93032, 25, 30141, 3331, 290, 3149, 27179, 11, 3463, 2140, 36598, 5941, 11, 59422, 295, 2201, 350, 733, 7443, 15054, 19504, 2238, 472, 47236, 11, 107300, 11, 6145, 111046, 929, 11, 103951, 6294, 197, 17, 13, 197, 64810, 350, 1353, 18929, 7926, 12167, 172201, 25, 115198, 483, 2647, 36598, 5941, 52178, 413, 12989, 306, 13574, 51310, 558, 18, 13, 5934, 12, 32535, 39612, 25, 121693, 115198, 483, 52077, 19504, 11, 1543, 165192, 19873, 11, 503, 192156, 1511, 2647, 8896, 558, 197, 19, 13, 49448, 1876, 7983, 50901, 536, 25, 3756, 7116, 19723, 1920, 4732, 4717, 306, 2684, 3566, 1327, 2236, 1023, 5143, 261, 5355, 2195, 503, 2914, 558, 197, 20, 13, 197, 64810, 40762, 11105, 33677, 830, 6968, 1276, 11105, 7849, 289, 25, 12817, 3566, 26291, 484, 6716, 316, 290, 2684, 36598, 8866, 11, 2273, 11, 10915, 503, 8496, 558, 197, 21, 13, 24747, 4649, 524, 220, 1353, 4, 87969, 25, 4296, 306, 1952, 3291, 16928, 11, 52178, 2475, 26291, 306, 13574, 51310, 558, 197, 22, 13, 22447, 109918, 21193, 4833, 25, 3004, 620, 4732, 1757, 413, 17882, 11, 860, 91135, 1757, 413, 17882, 11, 326, 860, 26291, 1757, 413, 92895, 558, 23, 13, 19461, 52178, 40151, 1645, 11081, 19723, 26958, 326, 469, 8722, 480, 483, 188301, 787, 538, 1354, 382, 945, 1572, 1001, 9481, 306, 484, 5024, 19723, 558, 197, 24, 13, 145342, 26193, 25, 30141, 622, 4376, 306, 4529, 8205, 6011, 13, 52178, 20180, 192663, 24657, 558]\n",
      " \n",
      "o1-batch\n",
      "individual requests:  802\n",
      "Total input tokens: 3642512\n",
      "Total output tokens: 116118\n",
      "Estimated input cost for o1-batch: $27.50\n",
      "Estimated output cost for o1-batch: $10.45\n",
      "Total estimated cost for o1-batch: $37.95\n",
      " \n",
      "[3575, 553, 261, 1238, 57204, 70116, 306, 97243, 6804, 36598, 26291, 13, 4886, 5296, 382, 316, 19723, 6771, 26291, 2049, 38971, 137858, 316, 290, 3992, 20959, 734, 197, 16, 13, 197, 50196, 133290, 114005, 93032, 25, 30141, 3331, 290, 3149, 27179, 11, 3463, 2140, 36598, 5941, 11, 59422, 295, 2201, 350, 733, 7443, 15054, 19504, 2238, 472, 47236, 11, 107300, 11, 6145, 111046, 929, 11, 103951, 6294, 197, 17, 13, 197, 64810, 350, 1353, 18929, 7926, 12167, 172201, 25, 115198, 483, 2647, 36598, 5941, 52178, 413, 12989, 306, 13574, 51310, 558, 18, 13, 5934, 12, 32535, 39612, 25, 121693, 115198, 483, 52077, 19504, 11, 1543, 165192, 19873, 11, 503, 192156, 1511, 2647, 8896, 558, 197, 19, 13, 49448, 1876, 7983, 50901, 536, 25, 3756, 7116, 19723, 1920, 4732, 4717, 306, 2684, 3566, 1327, 2236, 1023, 5143, 261, 5355, 2195, 503, 2914, 558, 197, 20, 13, 197, 64810, 40762, 11105, 33677, 830, 6968, 1276, 11105, 7849, 289, 25, 12817, 3566, 26291, 484, 6716, 316, 290, 2684, 36598, 8866, 11, 2273, 11, 10915, 503, 8496, 558, 197, 21, 13, 24747, 4649, 524, 220, 1353, 4, 87969, 25, 4296, 306, 1952, 3291, 16928, 11, 52178, 2475, 26291, 306, 13574, 51310, 558, 197, 22, 13, 22447, 109918, 21193, 4833, 25, 3004, 620, 4732, 1757, 413, 17882, 11, 860, 91135, 1757, 413, 17882, 11, 326, 860, 26291, 1757, 413, 92895, 558, 23, 13, 19461, 52178, 40151, 1645, 11081, 19723, 26958, 326, 469, 8722, 480, 483, 188301, 787, 538, 1354, 382, 945, 1572, 1001, 9481, 306, 484, 5024, 19723, 558, 197, 24, 13, 145342, 26193, 25, 30141, 622, 4376, 306, 4529, 8205, 6011, 13, 52178, 20180, 192663, 24657, 558]\n"
     ]
    }
   ],
   "source": [
    "import tiktoken\n",
    "\n",
    "# with open(\"data/api_key.txt\", \"r\") as f:\n",
    "#     api_key = f.read()\n",
    "\n",
    "def prepare_embedding_data(df, system_message, user_message, assistant_response, model_name, input_cost_per_token, output_cost_per_token):\n",
    "    \"\"\"Estimate the cost of embedding the data.\"\"\"\n",
    "    print(\" \")\n",
    "    print(model_name)\n",
    "    encoding = tiktoken.get_encoding(\"o200k_base\")\n",
    "    embedding_data = []\n",
    "    total_input_tokens = 0\n",
    "    total_output_tokens = 0\n",
    "    print(\"individual requests: \", len(df))\n",
    "    for sub_list in zip(df[\"Input\"], df[\"Output\"]):\n",
    "        \n",
    "        for item in sub_list:\n",
    "            n_tokens = len(encoding.encode(item))\n",
    "            total_input_tokens += n_tokens\n",
    "        total_input_tokens += len(encoding.encode(system_message))\n",
    "        total_input_tokens += len(encoding.encode(user_message))\n",
    "        total_input_tokens += len(encoding.encode(assistant_response))\n",
    "        total_output_tokens += len(encoding.encode(sub_list[1])) # the output format of my prompt is pretty short, so i multiply by 3 to overestimate\n",
    "        \n",
    "    print(f\"Total input tokens: {total_input_tokens}\")\n",
    "    print(f\"Total output tokens: {total_output_tokens}\")\n",
    "\n",
    "    # Calculate costs\n",
    "    input_cost = (total_input_tokens / 1_000_000) * input_cost_per_token  \n",
    "    output_cost = ((total_output_tokens*3) / 1_000_000) * output_cost_per_token \n",
    "    total_cost = input_cost + output_cost\n",
    "\n",
    "    print(f\"Estimated input cost for {model_name}: ${input_cost:.2f}\")\n",
    "    print(f\"Estimated output cost for {model_name}: ${output_cost:.2f}\")\n",
    "    print(f\"Total estimated cost for {model_name}: ${total_cost:.2f}\")\n",
    "    print(\" \")\n",
    "    print(encoding.encode(constant_system_message))\n",
    "    return None\n",
    "# Example usage\n",
    "\n",
    "constant_fine_tuning_format_promt = \"\"\n",
    "o3_mini_cost = prepare_embedding_data(df, constant_system_message, constant_user_message, constant_fine_tuning_format_promt, \"o3-mini\", 1.10, 4.40)\n",
    "o3_mini_batch_cost = prepare_embedding_data(df, constant_system_message, constant_user_message, constant_fine_tuning_format_promt, \"o3-mini-batch\", 0.55, 2.20)\n",
    "o1_cost = prepare_embedding_data(df, constant_system_message, constant_user_message, constant_fine_tuning_format_promt, \"o1\", 15.10, 60)\n",
    "o1_batch_cost = prepare_embedding_data(df, constant_system_message, constant_user_message, constant_fine_tuning_format_promt, \"o1-batch\", 7.55, 30)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Distillation/Fine tuning\n",
    "\n",
    "Now that we have generated the a dataset of 802 simulated conversations, we can set up the fine tuning.\n",
    "\n",
    "This is the order of prompts:\n",
    "\n",
    "- system_prompt: Same as original\n",
    "- user_prompt_1: Same as original, but only include sometimes (maybe 25% of the time), to prevent overfitting to the examples and to make the fine-tuning data look more like the data that we will produce.\n",
    "- user_prompt_2: Input from df\n",
    "- assistant_output_1 : precomputed output from df\n",
    "- **user_prompt_3**: the o3-mini generated user_prompt from validation explaining whether or not it was correct.\n",
    "- **assistant_output_2**: the o3-mini generated corrections, excluding clusters with only 1 member.\n",
    "\n",
    "When we then make predictions using the fine-tuned model, we slightly modify the system_prompt to say that it needs to exclude clusters with only 1 member, and bam wham bamalam, we hopefully have a model. "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "detailed_description = \"\"\"Input-1\n",
    "{\n",
    "  \"0\": '['Meiotic block [phenotype]', 'early arrest [phenotype]', 'arrest of meiotic progression in anaphase II [phenotype]', 'arrested zygotic divisions [phenotype]', 'meiotic arrest [phenotype]', 'delayed/arrested meiosis [phenotype]', 'pachytene arrest [phenotype]', 'block in meiosis prophase I [phenotype]', 'male meiosis until the end of the first division [phenotype]', 'meiotic prophase arrest [phenotype]', 'arrest in late prophase I [phenotype]', 'arrest at the end of meiosis I [phenotype]', 'leptotene arrest [phenotype]', 'meiosis I arrest [phenotype]', 'arresting the first mitosis during gametogenesis [phenotype]', 'absence of meiotic arrest [phenotype]', 'meiotic arrest phenotype [phenotype]', 'meiotic arrest at telophase I [phenotype]', 'termination of meiosis after anaphase I [phenotype]', 'premature termination of meiosis after anaphase I [phenotype]', 'mitotic arrest during female gametogenesis [phenotype]', 'arrest after meiosis I [phenotype]', 'meiotic arrest at pachytene [phenotype]', 'arrested endosperm nuclear divisions [phenotype]', 'meiotic arrest in anaphase II [phenotype]', 'meiotic division stop [phenotype]', 'arrest of the first mitotic division in gametogenesis [phenotype]', 'FNM half-stop [phenotype]', 'males arresting in the middle of prophase I [phenotype]', 'arresting prior to the first mitotic division [phenotype]']'\n",
    "}\n",
    "Input-1 → Output-1 [REASONING]\n",
    "\n",
    "1) Generic Meiotic Arrest\n",
    "\n",
    "Cluster:\n",
    "[ \"Meiotic block [phenotype]\", \"**meiotic arrest [phenotype]**\", \"meiotic arrest phenotype [phenotype]\", \"meiotic division stop [phenotype]\" ]\n",
    "• Incorrect: Grouping these separately as if they refer to different stages would obscure their identical meaning.\n",
    "• Correct: Recognize they all broadly indicate a total halt in meiosis at an unspecified stage.\n",
    "• Important: These terms are functionally synonymous, capturing any generic failure of meiotic progression.\n",
    "• Cluster representative: I selected 'meiotic arrest [phenotype]' as the representative because it is the most concise and commonly used term to describe a complete halt in the meiotic process. Unlike the other phrases which include additional descriptors or less formal wording, 'meiotic arrest' clearly and unambiguously captures the essential biological event, making it the best exemplar for the cluster.\n",
    "\n",
    "2) Delayed / Arrested Meiosis\n",
    "\n",
    "Cluster:\n",
    "[ \"delayed/arrested meiosis [phenotype]\" ]\n",
    "• Incorrect: Merging it with generic meiotic arrest overlooks the “delayed” aspect, which implies partial progression.\n",
    "• Correct: Keep it separate because it specifies a slow or partial block before a final stall.\n",
    "• Important: “Delayed” suggests some chromosomes or cells proceed further than in an outright immediate block.\n",
    "\n",
    "3) Absence of Meiotic Arrest\n",
    "\n",
    "Cluster:\n",
    "[ \"absence of meiotic arrest [phenotype]\" ]\n",
    "• Incorrect: Combining with any arrest group would contradict its meaning.\n",
    "• Correct: Maintain it as the negative counterpart, meaning no block occurs.\n",
    "• Important: This phenotype is crucial for comparisons, showing normal meiotic completion instead of a stoppage.\n",
    "\n",
    "4) Prophase I Arrest (Broad)\n",
    "\n",
    "Cluster:\n",
    "[ \"block in meiosis prophase I [phenotype]\", \"meiotic prophase arrest [phenotype]\" ]\n",
    "• Incorrect: Splitting them into sub-sub-stages if the label doesn’t specify.\n",
    "• Correct: They both emphasize an arrest somewhere in prophase I, without detailing the exact sub-stage (leptotene, pachytene, etc.).\n",
    "• Important: This captures a prophase I blockade in general, distinct from specific sub-stage arrests.\n",
    "• Cluster representative: I selected 'meiotic prophase arrest [phenotype]' as the representative because it succinctly captures the core biological event of an arrest occurring during prophase, without the additional wording found in the alternative. This clarity makes it the most direct and precise exemplar for the cluster.\n",
    "\n",
    "5) Male Arrest in Mid–Prophase I\n",
    "\n",
    "Cluster:\n",
    "[ \"males arresting in the middle of prophase I [phenotype]\" ]\n",
    "• Incorrect: Mixing with general prophase I arrest loses the male-specific nature and midpoint detail.\n",
    "• Correct: Keep it unique because it adds a sex specification (male) and timing (mid-prophase I).\n",
    "• Important: This addresses sex-specific contexts where the XY body or other male meiotic events fail around zygotene/pachytene.\n",
    "\n",
    "6) Early Arrest (Undefined Sub-Stage)\n",
    "\n",
    "Cluster:\n",
    "[ \"early arrest [phenotype]\" ]\n",
    "• Incorrect: Equating “early” to a named stage like leptotene or zygotene.\n",
    "• Correct: It must stand alone since it lacks a formal sub-stage but implies an initial block.\n",
    "• Important: The label indicates an arrest that happens before mid- or late-stage phenomena but is otherwise unspecified.\n",
    "\n",
    "7) Leptotene Arrest\n",
    "\n",
    "Cluster:\n",
    "[ \"leptotene arrest [phenotype]\" ]\n",
    "• Incorrect: Merging it with “early arrest” would lose the sub-stage clarity.\n",
    "• Correct: “Leptotene” is a well-defined earliest sub-stage of prophase I, deserving its own node.\n",
    "• Important: This precisely pinpoints where chromosomes start to condense yet fail to progress.\n",
    "\n",
    "8) Pachytene Arrest (Synonyms)\n",
    "\n",
    "Cluster:\n",
    "[ \"**pachytene arrest [phenotype]**\", \"meiotic arrest at pachytene [phenotype]\" ]\n",
    "• Incorrect: Splitting these two would ignore that they describe the exact same block.\n",
    "• Correct: They both name the pachytene sub-stage, so they cluster together.\n",
    "• Important: Pachytene is when homologs are fully synapsed, so arrest here is distinct from earlier or later prophase I phases.\n",
    "• Cluster representative: I selected 'pachytene arrest [phenotype]' as the representative because it is more concise and directly highlights the specific stage of meiotic arrest without additional qualifiers. This succinct phrasing clearly captures the biological event at the pachytene stage, making it the best exemplar for the cluster.\n",
    "\n",
    "9) Late Prophase I Arrest\n",
    "\n",
    "Cluster:\n",
    "[ \"arrest in late prophase I [phenotype]\" ]\n",
    "• Incorrect: Grouping with general prophase I arrests would lose the “late” distinction.\n",
    "• Correct: Keep it separate because it suggests diplotene or diakinesis sub-stages.\n",
    "• Important: Identifies that the block occurs after chromosome synapsis (pachytene) but before metaphase I.\n",
    "\n",
    "10) Meiosis I Arrest (Broad)\n",
    "\n",
    "Cluster:\n",
    "[ \"meiosis I arrest [phenotype]\" ]\n",
    "• Incorrect: Conflating with prophase I or end-of-meiosis-I arrests.\n",
    "• Correct: This label is intentionally broad for a block anywhere in the entire first meiotic division.\n",
    "• Important: Distinct from narrower arrests at anaphase I or telophase I.\n",
    "\n",
    "11) End-of-Meiosis-I Arrest\n",
    "\n",
    "Cluster:\n",
    "[ \"arrest at the end of meiosis I [phenotype]\", \"arrest after meiosis I [phenotype]\", \"**meiotic arrest at telophase I [phenotype]**\" ]\n",
    "• Incorrect: Mixing with generic “meiosis I arrest” might obscure that these specifically reach telophase I or just beyond.\n",
    "• Correct: All describe an arrest that specifically coincides with or follows telophase I.\n",
    "• Important: They finish prophase–anaphase I but fail to transition into or complete meiosis II.\n",
    "• Cluster representative: I selected 'meiotic arrest at telophase I [phenotype]' as the representative because it explicitly specifies the stage of arrest, leaving no ambiguity about the timing within meiosis. By clearly indicating telophase I, it provides a precise and biologically accurate descriptor compared to the more ambiguous alternatives present in the cluster.\n",
    "\n",
    "12) Male Meiosis Until End of First Division\n",
    "\n",
    "Cluster:\n",
    "[ \"male meiosis until the end of the first division [phenotype]\" ]\n",
    "• Incorrect: Merging with “arrest at the end of meiosis I” would ignore the explicit mention of male gametogenesis.\n",
    "• Correct: It parallels an end-of-meiosis-I block but is sex-specific.\n",
    "• Important: Reflects male-specific phenotypes where meiosis I completes in a partial sense but doesn’t proceed to meiosis II.\n",
    "\n",
    "13) Post–Anaphase I Termination\n",
    "\n",
    "Cluster:\n",
    "[ \"**termination of meiosis after anaphase I [phenotype]**\", \"premature termination of meiosis after anaphase I [phenotype]\" ]\n",
    "• Incorrect: Combining with end-of-meiosis-I arrests (telophase I) might overlook the specific time point (right after anaphase I).\n",
    "• Correct: These highlight that meiosis halts immediately following homolog separation in anaphase I.\n",
    "• Important: “Premature termination” still implies the same staging (post-anaphase I), so they cluster together.\n",
    "• Cluster representative: I selected 'termination of meiosis after anaphase I [phenotype]' as the representative because it provides a clear and concise description of the cessation of meiosis immediately following anaphase I. The absence of the qualifier 'premature' avoids additional nuance regarding timing, making it a more universally applicable and straightforward term to represent the cluster.\n",
    "\n",
    "14) Anaphase II Arrest (Synonyms)\n",
    "\n",
    "Cluster:\n",
    "[ \"arrest of meiotic progression in anaphase II [phenotype]\", \"**meiotic arrest in anaphase II [phenotype]**\" ]\n",
    "• Incorrect: Grouping with anaphase I or telophase I arrests would misrepresent the division stage.\n",
    "• Correct: Both pinpoint the second meiotic anaphase, so they are true synonyms.\n",
    "• Important: This arrest means meiosis I completed successfully, but the cell fails during separation of sister chromatids.\n",
    "• Cluster representative: I selected 'meiotic arrest in anaphase II [phenotype]' as the representative because it succinctly and directly identifies the specific phase (anaphase II) at which the arrest occurs. Its concise phrasing avoids unnecessary complexity, making it the clearest descriptor of the biological event in this cluster.\n",
    "\n",
    "15) Arrested Zygotic Divisions\n",
    "\n",
    "Cluster:\n",
    "[ \"arrested zygotic divisions [phenotype]\" ]\n",
    "• Incorrect: Folding into meiotic blocks misses that zygotic divisions are post-fertilization mitoses.\n",
    "• Correct: Keep separate, as this block is in the earliest embryo after fertilization.\n",
    "• Important: Distinguishing embryonic arrests from gametogenic or meiotic ones is crucial in developmental contexts.\n",
    "\n",
    "16) Arrested Endosperm Nuclear Divisions\n",
    "\n",
    "Cluster:\n",
    "[ \"arrested endosperm nuclear divisions [phenotype]\" ]\n",
    "• Incorrect: Combining with zygotic divisions lumps distinct post-fertilization tissues (embryo vs. endosperm).\n",
    "• Correct: Endosperm is a separate tissue formed post-fertilization (often triploid), so it merits its own category.\n",
    "• Important: In many plants, endosperm divides separately, so an arrest here is unique from zygotic embryonic arrest.\n",
    "\n",
    "17) First Mitotic Division in Gametogenesis (Synonyms)\n",
    "\n",
    "Cluster:\n",
    "[ \"arresting the first mitosis during gametogenesis [phenotype]\", \"**arrest of the first mitotic division in gametogenesis [phenotype]**\", \"FNM half-stop [phenotype]\" ]\n",
    "• Incorrect: Splitting these fails to see that all reference halting the very first post-meiotic mitosis.\n",
    "• Correct: They describe the same stage (first mitosis in gametogenesis), so they are synonyms.\n",
    "• Important: “FNM half-stop” is shorthand for the same phenomenon, not a different event.\n",
    "• Cluster representative: I selected 'arrest of the first mitotic division in gametogenesis [phenotype]' as the representative because it is the most precise and descriptive term. It clearly specifies the process (mitotic division) and context (gametogenesis), avoiding the informal shorthand of 'FNM half-stop' and the less formal phrasing of 'arresting the first mitosis during gametogenesis.' This precision makes it the best exemplar for the cluster.\n",
    "\n",
    "18) Arrest Prior to First Mitotic Division\n",
    "\n",
    "Cluster:\n",
    "[ \"arresting prior to the first mitotic division [phenotype]\" ]\n",
    "• Incorrect: Assuming it is the same as “arresting the first mitosis” would confuse the actual onset of that mitosis.\n",
    "• Correct: This indicates cells never even enter mitosis.\n",
    "• Important: Distinguishing “before it starts” from “during the division” can be crucial for understanding gametogenesis defects.\n",
    "\n",
    "19) Mitotic Arrest During Female Gametogenesis\n",
    "\n",
    "Cluster:\n",
    "[ \"mitotic arrest during female gametogenesis [phenotype]\" ]\n",
    "• Incorrect: Merging with the “first mitosis” cluster might ignore that multiple mitotic divisions can occur in female lines.\n",
    "• Correct: A female-specific block in some mitotic division (not necessarily the first).\n",
    "• Important: Sex specificity and indefinite mitotic stage set it apart from a clearly labeled “first mitosis” arrest.\n",
    "Output-1\n",
    "{\n",
    "\"0\": [\n",
    "[\n",
    "\"Meiotic block [phenotype]\",\n",
    "\"**meiotic arrest [phenotype]**\",\n",
    "\"meiotic arrest phenotype [phenotype]\",\n",
    "\"meiotic division stop [phenotype]\"\n",
    "],\n",
    "[\n",
    "\"delayed/arrested meiosis [phenotype]\"\n",
    "],\n",
    "[\n",
    "\"absence of meiotic arrest [phenotype]\"\n",
    "],\n",
    "[\n",
    "\"block in meiosis prophase I [phenotype]\",\n",
    "\"**meiotic prophase arrest [phenotype]**\"\n",
    "],\n",
    "[\n",
    "\"males arresting in the middle of prophase I [phenotype]\"\n",
    "],\n",
    "[\n",
    "\"early arrest [phenotype]\"\n",
    "],\n",
    "[\n",
    "\"leptotene arrest [phenotype]\"\n",
    "],\n",
    "[\n",
    "\"**pachytene arrest [phenotype]**\",\n",
    "\"meiotic arrest at pachytene [phenotype]\"\n",
    "],\n",
    "[\n",
    "\"arrest in late prophase I [phenotype]\"\n",
    "],\n",
    "[\n",
    "\"meiosis I arrest [phenotype]\"\n",
    "],\n",
    "[\n",
    "\"arrest at the end of meiosis I [phenotype]\",\n",
    "\"arrest after meiosis I [phenotype]\",\n",
    "\"**meiotic arrest at telophase I [phenotype]**\"\n",
    "],\n",
    "[\n",
    "\"male meiosis until the end of the first division [phenotype]\"\n",
    "],\n",
    "[\n",
    "\"**termination of meiosis after anaphase I [phenotype]**\",\n",
    "\"premature termination of meiosis after anaphase I [phenotype]\"\n",
    "],\n",
    "[\n",
    "\"arrest of meiotic progression in anaphase II [phenotype]\",\n",
    "\"**meiotic arrest in anaphase II [phenotype]**\"\n",
    "],\n",
    "[\n",
    "\"arrested zygotic divisions [phenotype]\"\n",
    "],\n",
    "[\n",
    "\"arrested endosperm nuclear divisions [phenotype]\"\n",
    "],\n",
    "[\n",
    "\"arresting the first mitosis during gametogenesis [phenotype]\",\n",
    "\"**arrest of the first mitotic division in gametogenesis [phenotype]**\",\n",
    "\"FNM half-stop [phenotype]\"\n",
    "],\n",
    "[\n",
    "\"arresting prior to the first mitotic division [phenotype]\"\n",
    "],\n",
    "[\n",
    "\"mitotic arrest during female gametogenesis [phenotype]\"\n",
    "]\n",
    "]\n",
    "}\n",
    "Input-2\n",
    "{\n",
    "  \"1\": '[\n",
    "    \"Salt-stress severity [treatment]\",\n",
    "    \"high NaCl stress [treatment]\",\n",
    "    \"potassium deprivation stress [treatment]\",\n",
    "    \"salt-stress response [treatment]\",\n",
    "    \"salt stress tolerance [treatment]\",\n",
    "    \"heat and salt stress conditions [treatment]\",\n",
    "    \"prolonged levels of salt stress [treatment]\",\n",
    "    \"recovery from salt stress [treatment]\",\n",
    "    \"salt stress assay [treatment]\",\n",
    "    \"salt stress signaling pathways [treatment]\",\n",
    "    \"gradual salt stress treatments [treatment]\",\n",
    "    \"salt and low temperature stresses [treatment]\",\n",
    "    \"salt and silicon stresses [treatment]\"\n",
    "  ]'\n",
    "}\n",
    "Input-2 → Output-2 [REASONING]\n",
    "\n",
    "1) Salt-Stress Core\n",
    "\n",
    "Clustered Terms\n",
    "• “Salt-stress severity [treatment]”\n",
    "• “**high NaCl stress [treatment]**”\n",
    "• “salt-stress response [treatment]”\n",
    "• “salt stress tolerance [treatment]”\n",
    "• “prolonged levels of salt stress [treatment]”\n",
    "• “recovery from salt stress [treatment]”\n",
    "• “salt stress assay [treatment]”\n",
    "• “salt stress signaling pathways [treatment]”\n",
    "• “gradual salt stress treatments [treatment]”\n",
    "\n",
    "Incorrect: Splitting “NaCl” from “salt” would be biologically misleading since NaCl is the chemical basis of most salt stress.\n",
    "Correct: Recognize all are purely salt-based conditions; “NaCl” is the explicit form, but it is still “salt.”\n",
    "Important: These labels measure or manipulate salt-stress conditions alone (no other stress factor).\n",
    "Cluster representative: I selected 'high NaCl stress [treatment]' as the representative because it directly encapsulates the core concept of salt stress by explicitly naming the chemical agent (NaCl) responsible for inducing the stress condition. This term is both succinct and unambiguous, avoiding additional qualifiers (like severity, response, or tolerance) that could shift the focus away from the primary salt stress condition.\n",
    "\n",
    "2) Unique Stress: Potassium Deprivation\n",
    "\n",
    "Clustered Term\n",
    "• “potassium deprivation stress [treatment]”\n",
    "\n",
    "Incorrect: Combining this with salt-based treatments implies overlapping ionic stress without specificity.\n",
    "Correct: Keep it separate because it focuses on K+ deficiency rather than NaCl excess.\n",
    "Important: Potassium starvation is a distinct abiotic stress requiring separate interpretation and management from salt stress.\n",
    "\n",
    "3) Heat + Salt Stress\n",
    "\n",
    "Clustered Term\n",
    "• “heat and salt stress conditions [treatment]”\n",
    "\n",
    "Incorrect: Merging with the salt core group would lose the additional heat component.\n",
    "Correct: Keep it in its own cluster because it involves two distinct stressors (heat + salt).\n",
    "Important: Many experiments examine combined stresses differently than single-stress treatments.\n",
    "\n",
    "4) Salt + Low Temperature\n",
    "\n",
    "Clustered Term\n",
    "• “salt and low temperature stresses [treatment]”\n",
    "\n",
    "Incorrect: Folding it into a single “salt” cluster disregards the cold factor.\n",
    "Correct: Identify that it specifically tests tolerance or response to dual stress: salt and cold.\n",
    "Important: Understanding multi-stress interactions is crucial for breeding or experimental design.\n",
    "\n",
    "5) Salt + Silicon\n",
    "\n",
    "Clustered Term\n",
    "• “salt and silicon stresses [treatment]”\n",
    "\n",
    "Incorrect: Grouping with plain salt stress lumps unique “silicon” involvement into generic salt.\n",
    "Correct: Keep separate because it’s salt + another factor (silicon) that could mitigate or alter salt stress.\n",
    "Important: Silicon is sometimes used to ameliorate salt stress, so it forms a distinct combined treatment.\n",
    "\n",
    "output-2\n",
    "{\n",
    "  \"1\": [\n",
    "    [\n",
    "      \"Salt-stress severity [treatment]\",\n",
    "      \"**high NaCl stress [treatment]**\",\n",
    "      \"salt-stress response [treatment]\",\n",
    "      \"salt stress tolerance [treatment]\",\n",
    "      \"prolonged levels of salt stress [treatment]\",\n",
    "      \"recovery from salt stress [treatment]\",\n",
    "      \"salt stress assay [treatment]\",\n",
    "      \"salt stress signaling pathways [treatment]\",\n",
    "      \"gradual salt stress treatments [treatment]\"\n",
    "    ],\n",
    "    [\"potassium deprivation stress [treatment]\"],\n",
    "    [\"heat and salt stress conditions [treatment]\"],\n",
    "    [\"salt and low temperature stresses [treatment]\"],\n",
    "    [\"salt and silicon stresses [treatment]\"]\n",
    "  ]\n",
    "}\n",
    "\"\"\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'messages': [{'role': 'system',\n",
       "   'content': \"You are a data scientist specializing in grouping plant biological entities. Your task is to cluster similar entities while strictly adhering to the following guidelines:\\n\\t1.\\tExact Phrase Matching Matters: Always consider the full phrase, including key biological terms, bracketed text (ignoring minor differences such as spacing, punctuation, correct abbreviations, plurality).\\n\\t2.\\tStrict (100%) Key Term Separation: Entities with different biological terms MUST be placed in separate clusters.\\n3. Sub-identifier separation: Separate Entities with numeric differences, sub-identifiers, or qualifiers into different groups.\\n\\t4.\\tAvoid False Similarity: Do NOT cluster two items together in same group just because they share a common word or term.\\n\\t5.\\tStrict Synonym/Near-Synonym Grouping: Only group entities that refer to the same biological structure, process, meaning or concept.\\n\\t6.\\tMaintain 100% Precision: When in even small doubt, MUST place entities in separate clusters.\\n\\t7.\\tPreserve Original Data: No new items should be introduced, no duplicates should be introduced, and no entities should be omitted.\\n8. YOU MUST pickup most appropriate cluster representative and enclose it with '**', if there is more than one entity in that particular cluster.\\n\\t9.\\tOutput Format: Always return results in valid JSON format. MUST USE GIVEN KEY.\\n\"},\n",
       "  {'role': 'user',\n",
       "   'content': '  \\nTwo examples of the reasoning you should use:\\n\\nInput-1\\n{\\n\\xa0 \"0\": \\'[\\'Meiotic block [phenotype]\\', \\'early arrest [phenotype]\\', \\'arrest of meiotic progression in anaphase II [phenotype]\\', \\'arrested zygotic divisions [phenotype]\\', \\'meiotic arrest [phenotype]\\', \\'delayed/arrested meiosis [phenotype]\\', \\'pachytene arrest [phenotype]\\', \\'block in meiosis prophase I [phenotype]\\', \\'male meiosis until the end of the first division [phenotype]\\', \\'meiotic prophase arrest [phenotype]\\', \\'arrest in late prophase I [phenotype]\\', \\'arrest at the end of meiosis I [phenotype]\\', \\'leptotene arrest [phenotype]\\', \\'meiosis I arrest [phenotype]\\', \\'arresting the first mitosis during gametogenesis [phenotype]\\', \\'absence of meiotic arrest [phenotype]\\', \\'meiotic arrest phenotype [phenotype]\\', \\'meiotic arrest at telophase I [phenotype]\\', \\'termination of meiosis after anaphase I [phenotype]\\', \\'premature termination of meiosis after anaphase I [phenotype]\\', \\'mitotic arrest during female gametogenesis [phenotype]\\', \\'arrest after meiosis I [phenotype]\\', \\'meiotic arrest at pachytene [phenotype]\\', \\'arrested endosperm nuclear divisions [phenotype]\\', \\'meiotic arrest in anaphase II [phenotype]\\', \\'meiotic division stop [phenotype]\\', \\'arrest of the first mitotic division in gametogenesis [phenotype]\\', \\'FNM half-stop [phenotype]\\', \\'males arresting in the middle of prophase I [phenotype]\\', \\'arresting prior to the first mitotic division [phenotype]\\']\\'\\n}\\nInput-1 → Output-1 [REASONING]\\n\\n1) Generic Meiotic Arrest\\n\\nCluster:\\n[ \"Meiotic block [phenotype]\", \"**meiotic arrest [phenotype]**\", \"meiotic arrest phenotype [phenotype]\", \"meiotic division stop [phenotype]\" ]\\n• Incorrect: Grouping these separately as if they refer to different stages would obscure their identical meaning.\\n• Correct: Recognize they all broadly indicate a total halt in meiosis at an unspecified stage.\\n• Important: These terms are functionally synonymous, capturing any generic failure of meiotic progression.\\n• Cluster representative: I selected \\'meiotic arrest [phenotype]\\' as the representative because it is the most concise and commonly used term to describe a complete halt in the meiotic process. Unlike the other phrases which include additional descriptors or less formal wording, \\'meiotic arrest\\' clearly and unambiguously captures the essential biological event, making it the best exemplar for the cluster.\\n\\n2) Delayed / Arrested Meiosis\\n\\nCluster:\\n[ \"delayed/arrested meiosis [phenotype]\" ]\\n• Incorrect: Merging it with generic meiotic arrest overlooks the “delayed” aspect, which implies partial progression.\\n• Correct: Keep it separate because it specifies a slow or partial block before a final stall.\\n• Important: “Delayed” suggests some chromosomes or cells proceed further than in an outright immediate block.\\n\\n3) Absence of Meiotic Arrest\\n\\nCluster:\\n[ \"absence of meiotic arrest [phenotype]\" ]\\n• Incorrect: Combining with any arrest group would contradict its meaning.\\n• Correct: Maintain it as the negative counterpart, meaning no block occurs.\\n• Important: This phenotype is crucial for comparisons, showing normal meiotic completion instead of a stoppage.\\n\\n4) Prophase I Arrest (Broad)\\n\\nCluster:\\n[ \"block in meiosis prophase I [phenotype]\", \"meiotic prophase arrest [phenotype]\" ]\\n• Incorrect: Splitting them into sub-sub-stages if the label doesn’t specify.\\n• Correct: They both emphasize an arrest somewhere in prophase I, without detailing the exact sub-stage (leptotene, pachytene, etc.).\\n• Important: This captures a prophase I blockade in general, distinct from specific sub-stage arrests.\\n• Cluster representative: I selected \\'meiotic prophase arrest [phenotype]\\' as the representative because it succinctly captures the core biological event of an arrest occurring during prophase, without the additional wording found in the alternative. This clarity makes it the most direct and precise exemplar for the cluster.\\n\\n5) Male Arrest in Mid–Prophase I\\n\\nCluster:\\n[ \"males arresting in the middle of prophase I [phenotype]\" ]\\n• Incorrect: Mixing with general prophase I arrest loses the male-specific nature and midpoint detail.\\n• Correct: Keep it unique because it adds a sex specification (male) and timing (mid-prophase I).\\n• Important: This addresses sex-specific contexts where the XY body or other male meiotic events fail around zygotene/pachytene.\\n\\n6) Early Arrest (Undefined Sub-Stage)\\n\\nCluster:\\n[ \"early arrest [phenotype]\" ]\\n• Incorrect: Equating “early” to a named stage like leptotene or zygotene.\\n• Correct: It must stand alone since it lacks a formal sub-stage but implies an initial block.\\n• Important: The label indicates an arrest that happens before mid- or late-stage phenomena but is otherwise unspecified.\\n\\n7) Leptotene Arrest\\n\\nCluster:\\n[ \"leptotene arrest [phenotype]\" ]\\n• Incorrect: Merging it with “early arrest” would lose the sub-stage clarity.\\n• Correct: “Leptotene” is a well-defined earliest sub-stage of prophase I, deserving its own node.\\n• Important: This precisely pinpoints where chromosomes start to condense yet fail to progress.\\n\\n8) Pachytene Arrest (Synonyms)\\n\\nCluster:\\n[ \"**pachytene arrest [phenotype]**\", \"meiotic arrest at pachytene [phenotype]\" ]\\n• Incorrect: Splitting these two would ignore that they describe the exact same block.\\n• Correct: They both name the pachytene sub-stage, so they cluster together.\\n• Important: Pachytene is when homologs are fully synapsed, so arrest here is distinct from earlier or later prophase I phases.\\n• Cluster representative: I selected \\'pachytene arrest [phenotype]\\' as the representative because it is more concise and directly highlights the specific stage of meiotic arrest without additional qualifiers. This succinct phrasing clearly captures the biological event at the pachytene stage, making it the best exemplar for the cluster.\\n\\n9) Late Prophase I Arrest\\n\\nCluster:\\n[ \"arrest in late prophase I [phenotype]\" ]\\n• Incorrect: Grouping with general prophase I arrests would lose the “late” distinction.\\n• Correct: Keep it separate because it suggests diplotene or diakinesis sub-stages.\\n• Important: Identifies that the block occurs after chromosome synapsis (pachytene) but before metaphase I.\\n\\n10) Meiosis I Arrest (Broad)\\n\\nCluster:\\n[ \"meiosis I arrest [phenotype]\" ]\\n• Incorrect: Conflating with prophase I or end-of-meiosis-I arrests.\\n• Correct: This label is intentionally broad for a block anywhere in the entire first meiotic division.\\n• Important: Distinct from narrower arrests at anaphase I or telophase I.\\n\\n11) End-of-Meiosis-I Arrest\\n\\nCluster:\\n[ \"arrest at the end of meiosis I [phenotype]\", \"arrest after meiosis I [phenotype]\", \"**meiotic arrest at telophase I [phenotype]**\" ]\\n• Incorrect: Mixing with generic “meiosis I arrest” might obscure that these specifically reach telophase I or just beyond.\\n• Correct: All describe an arrest that specifically coincides with or follows telophase I.\\n• Important: They finish prophase–anaphase I but fail to transition into or complete meiosis II.\\n• Cluster representative: I selected \\'meiotic arrest at telophase I [phenotype]\\' as the representative because it explicitly specifies the stage of arrest, leaving no ambiguity about the timing within meiosis. By clearly indicating telophase I, it provides a precise and biologically accurate descriptor compared to the more ambiguous alternatives present in the cluster.\\n\\n12) Male Meiosis Until End of First Division\\n\\nCluster:\\n[ \"male meiosis until the end of the first division [phenotype]\" ]\\n• Incorrect: Merging with “arrest at the end of meiosis I” would ignore the explicit mention of male gametogenesis.\\n• Correct: It parallels an end-of-meiosis-I block but is sex-specific.\\n• Important: Reflects male-specific phenotypes where meiosis I completes in a partial sense but doesn’t proceed to meiosis II.\\n\\n13) Post–Anaphase I Termination\\n\\nCluster:\\n[ \"**termination of meiosis after anaphase I [phenotype]**\", \"premature termination of meiosis after anaphase I [phenotype]\" ]\\n• Incorrect: Combining with end-of-meiosis-I arrests (telophase I) might overlook the specific time point (right after anaphase I).\\n• Correct: These highlight that meiosis halts immediately following homolog separation in anaphase I.\\n• Important: “Premature termination” still implies the same staging (post-anaphase I), so they cluster together.\\n• Cluster representative: I selected \\'termination of meiosis after anaphase I [phenotype]\\' as the representative because it provides a clear and concise description of the cessation of meiosis immediately following anaphase I. The absence of the qualifier \\'premature\\' avoids additional nuance regarding timing, making it a more universally applicable and straightforward term to represent the cluster.\\n\\n14) Anaphase II Arrest (Synonyms)\\n\\nCluster:\\n[ \"arrest of meiotic progression in anaphase II [phenotype]\", \"**meiotic arrest in anaphase II [phenotype]**\" ]\\n• Incorrect: Grouping with anaphase I or telophase I arrests would misrepresent the division stage.\\n• Correct: Both pinpoint the second meiotic anaphase, so they are true synonyms.\\n• Important: This arrest means meiosis I completed successfully, but the cell fails during separation of sister chromatids.\\n• Cluster representative: I selected \\'meiotic arrest in anaphase II [phenotype]\\' as the representative because it succinctly and directly identifies the specific phase (anaphase II) at which the arrest occurs. Its concise phrasing avoids unnecessary complexity, making it the clearest descriptor of the biological event in this cluster.\\n\\n15) Arrested Zygotic Divisions\\n\\nCluster:\\n[ \"arrested zygotic divisions [phenotype]\" ]\\n• Incorrect: Folding into meiotic blocks misses that zygotic divisions are post-fertilization mitoses.\\n• Correct: Keep separate, as this block is in the earliest embryo after fertilization.\\n• Important: Distinguishing embryonic arrests from gametogenic or meiotic ones is crucial in developmental contexts.\\n\\n16) Arrested Endosperm Nuclear Divisions\\n\\nCluster:\\n[ \"arrested endosperm nuclear divisions [phenotype]\" ]\\n• Incorrect: Combining with zygotic divisions lumps distinct post-fertilization tissues (embryo vs. endosperm).\\n• Correct: Endosperm is a separate tissue formed post-fertilization (often triploid), so it merits its own category.\\n• Important: In many plants, endosperm divides separately, so an arrest here is unique from zygotic embryonic arrest.\\n\\n17) First Mitotic Division in Gametogenesis (Synonyms)\\n\\nCluster:\\n[ \"arresting the first mitosis during gametogenesis [phenotype]\", \"**arrest of the first mitotic division in gametogenesis [phenotype]**\", \"FNM half-stop [phenotype]\" ]\\n• Incorrect: Splitting these fails to see that all reference halting the very first post-meiotic mitosis.\\n• Correct: They describe the same stage (first mitosis in gametogenesis), so they are synonyms.\\n• Important: “FNM half-stop” is shorthand for the same phenomenon, not a different event.\\n• Cluster representative: I selected \\'arrest of the first mitotic division in gametogenesis [phenotype]\\' as the representative because it is the most precise and descriptive term. It clearly specifies the process (mitotic division) and context (gametogenesis), avoiding the informal shorthand of \\'FNM half-stop\\' and the less formal phrasing of \\'arresting the first mitosis during gametogenesis.\\' This precision makes it the best exemplar for the cluster.\\n\\n18) Arrest Prior to First Mitotic Division\\n\\nCluster:\\n[ \"arresting prior to the first mitotic division [phenotype]\" ]\\n• Incorrect: Assuming it is the same as “arresting the first mitosis” would confuse the actual onset of that mitosis.\\n• Correct: This indicates cells never even enter mitosis.\\n• Important: Distinguishing “before it starts” from “during the division” can be crucial for understanding gametogenesis defects.\\n\\n19) Mitotic Arrest During Female Gametogenesis\\n\\nCluster:\\n[ \"mitotic arrest during female gametogenesis [phenotype]\" ]\\n• Incorrect: Merging with the “first mitosis” cluster might ignore that multiple mitotic divisions can occur in female lines.\\n• Correct: A female-specific block in some mitotic division (not necessarily the first).\\n• Important: Sex specificity and indefinite mitotic stage set it apart from a clearly labeled “first mitosis” arrest.\\nOutput-1\\n{\\n\"0\": [\\n[\\n\"Meiotic block [phenotype]\",\\n\"**meiotic arrest [phenotype]**\",\\n\"meiotic arrest phenotype [phenotype]\",\\n\"meiotic division stop [phenotype]\"\\n],\\n[\\n\"delayed/arrested meiosis [phenotype]\"\\n],\\n[\\n\"absence of meiotic arrest [phenotype]\"\\n],\\n[\\n\"block in meiosis prophase I [phenotype]\",\\n\"**meiotic prophase arrest [phenotype]**\"\\n],\\n[\\n\"males arresting in the middle of prophase I [phenotype]\"\\n],\\n[\\n\"early arrest [phenotype]\"\\n],\\n[\\n\"leptotene arrest [phenotype]\"\\n],\\n[\\n\"**pachytene arrest [phenotype]**\",\\n\"meiotic arrest at pachytene [phenotype]\"\\n],\\n[\\n\"arrest in late prophase I [phenotype]\"\\n],\\n[\\n\"meiosis I arrest [phenotype]\"\\n],\\n[\\n\"arrest at the end of meiosis I [phenotype]\",\\n\"arrest after meiosis I [phenotype]\",\\n\"**meiotic arrest at telophase I [phenotype]**\"\\n],\\n[\\n\"male meiosis until the end of the first division [phenotype]\"\\n],\\n[\\n\"**termination of meiosis after anaphase I [phenotype]**\",\\n\"premature termination of meiosis after anaphase I [phenotype]\"\\n],\\n[\\n\"arrest of meiotic progression in anaphase II [phenotype]\",\\n\"**meiotic arrest in anaphase II [phenotype]**\"\\n],\\n[\\n\"arrested zygotic divisions [phenotype]\"\\n],\\n[\\n\"arrested endosperm nuclear divisions [phenotype]\"\\n],\\n[\\n\"arresting the first mitosis during gametogenesis [phenotype]\",\\n\"**arrest of the first mitotic division in gametogenesis [phenotype]**\",\\n\"FNM half-stop [phenotype]\"\\n],\\n[\\n\"arresting prior to the first mitotic division [phenotype]\"\\n],\\n[\\n\"mitotic arrest during female gametogenesis [phenotype]\"\\n]\\n]\\n}\\nInput-2\\n{\\n\\xa0 \"1\": \\'[\\n\\xa0 \\xa0 \"Salt-stress severity [treatment]\",\\n\\xa0 \\xa0 \"high NaCl stress [treatment]\",\\n\\xa0 \\xa0 \"potassium deprivation stress [treatment]\",\\n\\xa0 \\xa0 \"salt-stress response [treatment]\",\\n\\xa0 \\xa0 \"salt stress tolerance [treatment]\",\\n\\xa0 \\xa0 \"heat and salt stress conditions [treatment]\",\\n\\xa0 \\xa0 \"prolonged levels of salt stress [treatment]\",\\n\\xa0 \\xa0 \"recovery from salt stress [treatment]\",\\n\\xa0 \\xa0 \"salt stress assay [treatment]\",\\n\\xa0 \\xa0 \"salt stress signaling pathways [treatment]\",\\n\\xa0 \\xa0 \"gradual salt stress treatments [treatment]\",\\n\\xa0 \\xa0 \"salt and low temperature stresses [treatment]\",\\n\\xa0 \\xa0 \"salt and silicon stresses [treatment]\"\\n\\xa0 ]\\'\\n}\\nInput-2 → Output-2 [REASONING]\\n\\n1) Salt-Stress Core\\n\\nClustered Terms\\n• “Salt-stress severity [treatment]”\\n• “**high NaCl stress [treatment]**”\\n• “salt-stress response [treatment]”\\n• “salt stress tolerance [treatment]”\\n• “prolonged levels of salt stress [treatment]”\\n• “recovery from salt stress [treatment]”\\n• “salt stress assay [treatment]”\\n• “salt stress signaling pathways [treatment]”\\n• “gradual salt stress treatments [treatment]”\\n\\nIncorrect: Splitting “NaCl” from “salt” would be biologically misleading since NaCl is the chemical basis of most salt stress.\\nCorrect: Recognize all are purely salt-based conditions; “NaCl” is the explicit form, but it is still “salt.”\\nImportant: These labels measure or manipulate salt-stress conditions alone (no other stress factor).\\nCluster representative: I selected \\'high NaCl stress [treatment]\\' as the representative because it directly encapsulates the core concept of salt stress by explicitly naming the chemical agent (NaCl) responsible for inducing the stress condition. This term is both succinct and unambiguous, avoiding additional qualifiers (like severity, response, or tolerance) that could shift the focus away from the primary salt stress condition.\\n\\n2) Unique Stress: Potassium Deprivation\\n\\nClustered Term\\n• “potassium deprivation stress [treatment]”\\n\\nIncorrect: Combining this with salt-based treatments implies overlapping ionic stress without specificity.\\nCorrect: Keep it separate because it focuses on K+ deficiency rather than NaCl excess.\\nImportant: Potassium starvation is a distinct abiotic stress requiring separate interpretation and management from salt stress.\\n\\n3) Heat + Salt Stress\\n\\nClustered Term\\n• “heat and salt stress conditions [treatment]”\\n\\nIncorrect: Merging with the salt core group would lose the additional heat component.\\nCorrect: Keep it in its own cluster because it involves two distinct stressors (heat + salt).\\nImportant: Many experiments examine combined stresses differently than single-stress treatments.\\n\\n4) Salt + Low Temperature\\n\\nClustered Term\\n• “salt and low temperature stresses [treatment]”\\n\\nIncorrect: Folding it into a single “salt” cluster disregards the cold factor.\\nCorrect: Identify that it specifically tests tolerance or response to dual stress: salt and cold.\\nImportant: Understanding multi-stress interactions is crucial for breeding or experimental design.\\n\\n5) Salt + Silicon\\n\\nClustered Term\\n• “salt and silicon stresses [treatment]”\\n\\nIncorrect: Grouping with plain salt stress lumps unique “silicon” involvement into generic salt.\\nCorrect: Keep separate because it’s salt + another factor (silicon) that could mitigate or alter salt stress.\\nImportant: Silicon is sometimes used to ameliorate salt stress, so it forms a distinct combined treatment.\\n\\noutput-2\\n{\\n\\xa0 \"1\": [\\n\\xa0 \\xa0 [\\n\\xa0 \\xa0 \\xa0 \"Salt-stress severity [treatment]\",\\n\\xa0 \\xa0 \\xa0 \"**high NaCl stress [treatment]**\",\\n\\xa0 \\xa0 \\xa0 \"salt-stress response [treatment]\",\\n\\xa0 \\xa0 \\xa0 \"salt stress tolerance [treatment]\",\\n\\xa0 \\xa0 \\xa0 \"prolonged levels of salt stress [treatment]\",\\n\\xa0 \\xa0 \\xa0 \"recovery from salt stress [treatment]\",\\n\\xa0 \\xa0 \\xa0 \"salt stress assay [treatment]\",\\n\\xa0 \\xa0 \\xa0 \"salt stress signaling pathways [treatment]\",\\n\\xa0 \\xa0 \\xa0 \"gradual salt stress treatments [treatment]\"\\n\\xa0 \\xa0 ],\\n\\xa0 \\xa0 [\"potassium deprivation stress [treatment]\"],\\n\\xa0 \\xa0 [\"heat and salt stress conditions [treatment]\"],\\n\\xa0 \\xa0 [\"salt and low temperature stresses [treatment]\"],\\n\\xa0 \\xa0 [\"salt and silicon stresses [treatment]\"]\\n\\xa0 ]\\n}\\n'},\n",
       "  {'role': 'user',\n",
       "   'content': '{\"22996\": [\"Conductance measurements [phenotype]\", \"stomatal conductance, transpiration rate, hydraulic conductivity and photosynthetic efficiency [phenotype]\", \"E and g s [phenotype]\", \"regulation of stomatal conductance through FER [phenotype]\", \"engineering stomatal conductance [phenotype]\", \"stomatal conductance, transpiration rate, relative-water content of plant(s) leaf [phenotype]\", \"stomatal conductance measurements [phenotype]\", \"measuring stomatal conductance [phenotype]\", \"K(leaf) values [phenotype]\", \"direct measurements of g s [phenotype]\", \"AtPIP2;5 regulating gm [phenotype]\", \"conductance levels [phenotype]\", \"number of open stomata and amount of water loss through transpiration [phenotype]\", \"methods to quickly assess the stomatal conductance (g\\\\u209b) [phenotype]\", \"time-resolved measurements of stomatal conductance [phenotype]\", \"time-resolved stomatal conductance analyses [phenotype]\", \"stomatal conductance analyses [phenotype]\", \"transpiration and stomatal conductance values [phenotype]\", \"kinetic stomatal conductance analyses [phenotype]\", \"derived model [phenotype]\", \"measured stomatal conductance [phenotype]\", \"WUE and Stomatal conductance [phenotype]\", \"stomatal conductances [phenotype]\", \"both SD and photosynthetic gas exchange traits [phenotype]\", \"consideration of stomatal conductance [phenotype]\"]}'},\n",
       "  {'role': 'assistant',\n",
       "   'content': '{\"22996\": [[\"**transpiration and stomatal conductance values [phenotype]**\", \"E and g s [phenotype]\"], [\"consideration of stomatal conductance [phenotype]\", \"**stomatal conductance measurements [phenotype]**\", \"measured stomatal conductance [phenotype]\", \"Conductance measurements [phenotype]\", \"stomatal conductances [phenotype]\", \"direct measurements of g s [phenotype]\", \"stomatal conductance analyses [phenotype]\", \"conductance levels [phenotype]\", \"measuring stomatal conductance [phenotype]\"], [\"derived model [phenotype]\"], [\"engineering stomatal conductance [phenotype]\"], [\"K(leaf) values [phenotype]\"], [\"stomatal conductance, transpiration rate, hydraulic conductivity and photosynthetic efficiency [phenotype]\"], [\"AtPIP2;5 regulating gm [phenotype]\"], [\"kinetic stomatal conductance analyses [phenotype]\", \"**time-resolved stomatal conductance analyses [phenotype]**\", \"time-resolved measurements of stomatal conductance [phenotype]\"], [\"number of open stomata and amount of water loss through transpiration [phenotype]\"], [\"stomatal conductance, transpiration rate, relative-water content of plant(s) leaf [phenotype]\"], [\"WUE and Stomatal conductance [phenotype]\"], [\"both SD and photosynthetic gas exchange traits [phenotype]\"], [\"regulation of stomatal conductance through FER [phenotype]\"], [\"methods to quickly assess the stomatal conductance (g\\\\u209b) [phenotype]\"]]}'},\n",
       "  {'role': 'user',\n",
       "   'content': 'This looks correct to me, output the clusters that have more than 1 member in the correct json format.'},\n",
       "  {'role': 'assistant',\n",
       "   'content': [['**CO/NF-Y complex [protein complex]**',\n",
       "     'CO/NF-Y heterotrimers [protein complex]',\n",
       "     'CO/NF-Y structure [protein complex]'],\n",
       "    ['**CO/NF-YB/NF-YC trimer complex [protein complex]**',\n",
       "     'trimer structure of NF-YB-YC-CO [protein complex]']]}]}"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "system_prompt = \"\"\"You are a data scientist specializing in grouping plant biological entities. Your task is to cluster similar entities while strictly adhering to the following guidelines:\n",
    "\n",
    "\t1.\tExact Phrase Matching Matters: Always consider the full phrase, including key biological terms, bracketed text (ignoring minor differences such as spacing, punctuation, correct abbreviations, plurality).\n",
    "\t2.\tStrict (100%) Key Term Separation: Entities with different biological terms MUST be placed in separate clusters.\n",
    "    3. Sub-identifier separation: Separate Entities with numeric differences, sub-identifiers, or qualifiers into different groups.\n",
    "\t4.\tAvoid False Similarity: Do NOT cluster two items together in same group just because they share a common word or term.\n",
    "\t5.\tStrict Synonym/Near-Synonym Grouping: Only group entities that refer to the same biological structure, process, meaning or concept.\n",
    "\t6.\tMaintain 100% Precision: When in even small doubt, MUST place entities in separate clusters.\n",
    "\t7.\tPreserve Original Data: No new items should be introduced, no duplicates should be introduced, and no entities should be omitted.\n",
    "    8. YOU MUST pickup most appropriate and easy-to-understand cluster representative and enclose it with '**', if there is more than one entity in that particular cluster. For example, pick the full term instead of an abbreviation.\n",
    "\t9.\tOutput Format: Always return results in valid JSON format. MUST USE GIVEN KEY.\n",
    "\n",
    "Read the input list, and return a clustered output list.\n",
    "\"\"\"\n",
    "\n",
    "\n",
    "\n",
    "\n",
    "input_list = \"\" #the Input list, taken from original K-meansdataframe\n",
    "\n",
    "initial_output = \"\" # the first round of o3-mini predictions from the dataframe\n",
    "\n",
    "o3_user_prompt = \"\" # the userprompt that o3-mini came up with from the second round o3-mini output\n",
    "\n",
    "corrected_output = \"\" # the corrected and shortened list.\n",
    "\n",
    "#for previous example\n",
    "fine_tune_dict = {\n",
    "    \"messages\": [\n",
    "        {\"role\": \"system\",\"content\": system_prompt}, \n",
    "        {\"role\": \"user\", \"content\": input_list}, \n",
    "        {\"role\": \"assistant\", \"content\": initial_output},\n",
    "        {\"role\": \"user\", \"content\": o3_user_prompt},\n",
    "        {\"role\": \"assistant\", \"content\": corrected_output},\n",
    "        ]}\n",
    "\n",
    "# draw a float between 0 and 1\n",
    "include_detailed_description = random.random() < 0.2\n",
    "if include_detailed_description:\n",
    "    # assign user_prompt_1 at index 1\n",
    "    fine_tune_dict[\"messages\"].insert(1, {\"role\": \"user\", \"content\": detailed_description})\n",
    "    \n",
    "fine_tune_dict\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [],
   "source": [
    "import pandas as pd\n",
    "df = pd.read_csv (\"o3mini_rand_50.csv\",index_col=None)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "\"This looks incorrect to me. The problem lies with the fourth cluster. It currently groups together four items: 'a biological control agent [treatment]', 'biological control agent (BCA) [treatment]', 'potent biological control agent [treatment]', and 'potential biological control agent [treatment]'. According to our strict grouping rules, items with qualifiers that change the meaning (i.e. 'potent' and 'potential') must be separated from the generic form. In this case, 'a biological control agent [treatment]' and 'biological control agent (BCA) [treatment]' clearly refer to the general concept and should be clustered together (with the latter as the representative), while the other two should be in separate clusters. Since we are only outputting clusters with more than one member, only the generic biological control agent group should be included. Also, the first cluster (biopesticides) is correctly grouped. Please output only the clusters with more than one entity in the correct JSON format.\""
      ]
     },
     "execution_count": 10,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df[\"user_len\"] = df[\"user_prompt\"].str.len()\n",
    "df[\"user_len\"].max()\n",
    "df[df[\"user_len\"] == df[\"user_len\"].max()][\"user_prompt\"].values[0]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "#"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "PhD",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.12.2"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
