{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Generate dataset"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 122,
   "metadata": {},
   "outputs": [],
   "source": [
    "#This is a script that loads a pandas dataframe from csv, converts the string formatted as a list into a list of lists. The list of lists is then flattened into a single list.\n",
    "\n",
    "import pandas as pd\n",
    "import random\n",
    "import json\n",
    "df = pd.read_csv('./gpt_final_outs/o3mini_maxclust_80_subset_random_1000_nocutoff.csv', index_col=None)\n",
    "\n",
    "#rename the column \"Entity Name\" to \"Entity\" and \"Group Items\" to \"output\"\n",
    "df = df.rename(columns={\"Group Items\": \"Output\"})\n",
    "\n",
    "# Convert the string formatted as a list into a list of lists\n",
    "df['Output'] = df['Output'].apply(lambda x: eval(x))\n",
    "\n",
    "# Flatten the list of lists into a single list\n",
    "df['Input'] = df['Output'].apply(lambda x: [item.strip(\"*\") for sublist in x for item in sublist])\n",
    "\n",
    "#randomize the order of the single list\n",
    "df['Input'] = df['Input'].apply(lambda x: random.sample(x, len(x)))\n",
    "#df = df[df['Input'].apply(len) < 30]\n",
    "# make a new column, where internal lists of length 1 are removed \n",
    "df['Output>1'] = df['Output'].apply(lambda x: [item for item in x if len(item) > 1])\n",
    "#create a flag column by name one_item. For each row in Output column, If length of the list in output column is less than 2, put True otherwise False.\n",
    "df['one_item'] = df['Output>1'].apply(lambda x: True if len(x) == 0 else False)\n",
    "\n",
    "#zip the Group_Items_no_singles with the Entity Name \n",
    "df[\"Output\"] = df.apply(lambda row: {row[\"Entity Name\"]: row['Output']}, axis=1)\n",
    "df[\"Input\"] = df.apply(lambda row: {row[\"Entity Name\"]: row['Input']}, axis=1)\n",
    "df['Output>1'] = df.apply(lambda row: {row[\"Entity Name\"]: row['Output>1']}, axis=1)\n",
    "\n",
    "\n",
    "\n",
    "\n",
    "\n",
    "#convert the columns to json strings\n",
    "df[\"Output\"] = df[\"Output\"].apply(lambda x: json.dumps(x) if isinstance(x, dict) else x)\n",
    "df[\"Input\"] = df[\"Input\"].apply(lambda x: json.dumps(x) if isinstance(x, dict) else x)\n",
    "df[\"Output>1\"] = df[\"Output>1\"].apply(lambda x: json.dumps(x) if isinstance(x, dict) else x)\n",
    "\n",
    "#Change order of columns to have input, then output, then output>1, then entity name\n",
    "df = df[['Entity Name', 'Input', 'Output', 'Output>1', 'one_item']]\n",
    "\n",
    "#filter by Output>1 length of the list > 1\n",
    "#use column Output, but use only the ones where it clusteres multiple items together"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 123,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Entity Name</th>\n",
       "      <th>Input</th>\n",
       "      <th>Output</th>\n",
       "      <th>Output&gt;1</th>\n",
       "      <th>one_item</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>58963</td>\n",
       "      <td>{\"58963\": [\"Gardea-Torresdey et al. [researche...</td>\n",
       "      <td>{\"58963\": [[\"Gardea-Torresdey et al. [research...</td>\n",
       "      <td>{\"58963\": []}</td>\n",
       "      <td>True</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>22775</td>\n",
       "      <td>{\"22775\": [\"physiological transport of macro-m...</td>\n",
       "      <td>{\"22775\": [[\"substance that could be transloca...</td>\n",
       "      <td>{\"22775\": [[\"**transportation [phenotype]**\", ...</td>\n",
       "      <td>False</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>22996</td>\n",
       "      <td>{\"22996\": [\"stomatal conductance analyses [phe...</td>\n",
       "      <td>{\"22996\": [[\"**transpiration and stomatal cond...</td>\n",
       "      <td>{\"22996\": [[\"**transpiration and stomatal cond...</td>\n",
       "      <td>False</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>100116</td>\n",
       "      <td>{\"100116\": [\"modern clade [gene]\", \"modern cla...</td>\n",
       "      <td>{\"100116\": [[\"Clade M [clade]\"], [\"Clade SL [c...</td>\n",
       "      <td>{\"100116\": []}</td>\n",
       "      <td>True</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>133255</td>\n",
       "      <td>{\"133255\": [\"Plastidic LPPs [gene]\", \"PLSP2B [...</td>\n",
       "      <td>{\"133255\": [[\"**PLSP2A [gene]**\", \"PLSP2A gene...</td>\n",
       "      <td>{\"133255\": [[\"**PLSP2A [gene]**\", \"PLSP2A gene...</td>\n",
       "      <td>False</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>995</th>\n",
       "      <td>31615</td>\n",
       "      <td>{\"31615\": [\"auxin-regulated endocytosis [proce...</td>\n",
       "      <td>{\"31615\": [[\"auxin-mediated modulation of the ...</td>\n",
       "      <td>{\"31615\": [[\"auxin-dependent inhibition of end...</td>\n",
       "      <td>False</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>996</th>\n",
       "      <td>157227</td>\n",
       "      <td>{\"157227\": [\"SlSAUR1-like [gene]\", \"ZmSAUR1 [g...</td>\n",
       "      <td>{\"157227\": [[\"GmSAUR [gene]\"], [\"StSAUR [gene]...</td>\n",
       "      <td>{\"157227\": [[\"ZmSAUR1 (Zea mays SAURs) [gene]\"...</td>\n",
       "      <td>False</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>997</th>\n",
       "      <td>131559</td>\n",
       "      <td>{\"131559\": [\"BtuC gene(s) [gene]\", \"BtuN gene(...</td>\n",
       "      <td>{\"131559\": [[\"BtuB gene(s) [gene]\"], [\"BtuC ge...</td>\n",
       "      <td>{\"131559\": []}</td>\n",
       "      <td>True</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>998</th>\n",
       "      <td>99091</td>\n",
       "      <td>{\"99091\": [\"O. fragrans 'Liuyejingui' (OFL) [o...</td>\n",
       "      <td>{\"99091\": [[\"Genome sequencing of O. fragrans ...</td>\n",
       "      <td>{\"99091\": [[\"**O. fragrans 'Liuyejingui' (OFL)...</td>\n",
       "      <td>False</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>999</th>\n",
       "      <td>65306</td>\n",
       "      <td>{\"65306\": [\"CO/NF-YB/NF-YC trimer complex [pro...</td>\n",
       "      <td>{\"65306\": [[\"**CO/NF-Y complex [protein comple...</td>\n",
       "      <td>{\"65306\": [[\"**CO/NF-Y complex [protein comple...</td>\n",
       "      <td>False</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>1000 rows × 5 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "     Entity Name                                              Input  \\\n",
       "0          58963  {\"58963\": [\"Gardea-Torresdey et al. [researche...   \n",
       "1          22775  {\"22775\": [\"physiological transport of macro-m...   \n",
       "2          22996  {\"22996\": [\"stomatal conductance analyses [phe...   \n",
       "3         100116  {\"100116\": [\"modern clade [gene]\", \"modern cla...   \n",
       "4         133255  {\"133255\": [\"Plastidic LPPs [gene]\", \"PLSP2B [...   \n",
       "..           ...                                                ...   \n",
       "995        31615  {\"31615\": [\"auxin-regulated endocytosis [proce...   \n",
       "996       157227  {\"157227\": [\"SlSAUR1-like [gene]\", \"ZmSAUR1 [g...   \n",
       "997       131559  {\"131559\": [\"BtuC gene(s) [gene]\", \"BtuN gene(...   \n",
       "998        99091  {\"99091\": [\"O. fragrans 'Liuyejingui' (OFL) [o...   \n",
       "999        65306  {\"65306\": [\"CO/NF-YB/NF-YC trimer complex [pro...   \n",
       "\n",
       "                                                Output  \\\n",
       "0    {\"58963\": [[\"Gardea-Torresdey et al. [research...   \n",
       "1    {\"22775\": [[\"substance that could be transloca...   \n",
       "2    {\"22996\": [[\"**transpiration and stomatal cond...   \n",
       "3    {\"100116\": [[\"Clade M [clade]\"], [\"Clade SL [c...   \n",
       "4    {\"133255\": [[\"**PLSP2A [gene]**\", \"PLSP2A gene...   \n",
       "..                                                 ...   \n",
       "995  {\"31615\": [[\"auxin-mediated modulation of the ...   \n",
       "996  {\"157227\": [[\"GmSAUR [gene]\"], [\"StSAUR [gene]...   \n",
       "997  {\"131559\": [[\"BtuB gene(s) [gene]\"], [\"BtuC ge...   \n",
       "998  {\"99091\": [[\"Genome sequencing of O. fragrans ...   \n",
       "999  {\"65306\": [[\"**CO/NF-Y complex [protein comple...   \n",
       "\n",
       "                                              Output>1  one_item  \n",
       "0                                        {\"58963\": []}      True  \n",
       "1    {\"22775\": [[\"**transportation [phenotype]**\", ...     False  \n",
       "2    {\"22996\": [[\"**transpiration and stomatal cond...     False  \n",
       "3                                       {\"100116\": []}      True  \n",
       "4    {\"133255\": [[\"**PLSP2A [gene]**\", \"PLSP2A gene...     False  \n",
       "..                                                 ...       ...  \n",
       "995  {\"31615\": [[\"auxin-dependent inhibition of end...     False  \n",
       "996  {\"157227\": [[\"ZmSAUR1 (Zea mays SAURs) [gene]\"...     False  \n",
       "997                                     {\"131559\": []}      True  \n",
       "998  {\"99091\": [[\"**O. fragrans 'Liuyejingui' (OFL)...     False  \n",
       "999  {\"65306\": [[\"**CO/NF-Y complex [protein comple...     False  \n",
       "\n",
       "[1000 rows x 5 columns]"
      ]
     },
     "execution_count": 123,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df\n",
    "#note i don't really use Outputs>1 in this workflow."
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Prompts for making fine-tuning dataset (LLM as a Judge)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 124,
   "metadata": {},
   "outputs": [],
   "source": [
    "constant_system_message = \"\"\"\n",
    "You are senior scientist in plant biology. You NEED to meticulously VALIDATE given clustering output, and provide a corrected clustering if needed. Explain which groupings are wrong and provide final output. You MUST write response like a senior human plant scientist.\n",
    "\n",
    "The following are the entity clustering guidelines:\n",
    "\t1.\tExact Phrase Matching Matters: Always consider the full phrase, including key biological terms, bracketed text (ignoring minor differences such as spacing, punctuation, correct abbreviations, plurality).\n",
    "\t2.\tStrict (100%) Key Term Separation: Entities with different biological terms MUST be placed in separate clusters.\n",
    "3. Sub-identifier separation: Separate Entities with numeric differences, sub-identifiers, or qualifiers into different groups.\n",
    "\t4.\tAvoid False Similarity: Do NOT cluster two items together in same group just because they share a common word or term.\n",
    "\t5.\tStrict Synonym/Near-Synonym Grouping: Only group entities that refer to the same biological structure, process, meaning or concept.\n",
    "\t6.\tMaintain 100% Precision: When in even small doubt, MUST place entities in separate clusters.\n",
    "\t7.\tPreserve Original Data: No new items should be introduced, no duplicates should be introduced, and no entities should be omitted.\n",
    "8. YOU MUST pickup most appropriate and easy-to-understand cluster representative and enclose it with '**', if there is more than one entity in that particular cluster. For example, pick the full term instead of an abbreviation.\n",
    "\t9.\tOutput Format: Always return results in valid JSON format. MUST USE GIVEN KEY.\n",
    "\n",
    "Your main task is to read the clustered entities and output the validation in the below mentioned format.\n",
    "\n",
    "The goal:\n",
    "These results came from a smaller LLM, and i want you to take a look at it.\n",
    "With your expertise in plant biology, generate prompts for fine-tuning the smaller LLM and mimick your patterns for this specific task.\n",
    "I want you to validate the clustering of this smaller model, and provide a corrected clustering if needed.\n",
    "Then you should imagine that you are a human plant biology senior scientist who explains to the smaller LLM what it is doing wrong, to give it the final output. You should never mention the smaller LLM, but instead just pretend you are a human user. \n",
    "An example of this would be:\n",
    "'I think you are wrong, cluster X is actually different from cluster Y because of so and so reasons. Cluster X and Y should be clustered seperately'.\n",
    "If the smaller model is correct however, you should write something like:\n",
    "'Great job, that looks correct because of so and so reasons. Now output the correct format, but only output clusters with more than one member.'\n",
    "The expected input format:\n",
    "\n",
    "{\n",
    "  \"submission_id_1\": {\n",
    "    \"clusters\": [\n",
    "      [\n",
    "        \"**cluster_item1**\",\n",
    "        \"cluster_item2\"\n",
    "      ],\n",
    "      [\n",
    "        \"cluster_item3\"\n",
    "      ]\n",
    "    ]\n",
    "  }\n",
    "}\n",
    "\n",
    "Return format:\n",
    "\n",
    "If the clustering is correct:\n",
    "{\n",
    "  \"submission_id_1\": {\n",
    "    \"is_correct\": \"True\",\n",
    "    \"user_prompt\": \"This looks correct to me, output the clusters that have more than 1 member in the correct json format.\",\n",
    "    \"clusters\": [\n",
    "      [\n",
    "        \"cluster_item1\",\n",
    "        \"cluster_item2\"\n",
    "      ]\n",
    "    ]\n",
    "  }\n",
    "}\n",
    "\n",
    "If the clustering is incorrect:\n",
    "\n",
    "{\n",
    "  \"submission_id_1\": {\n",
    "    \"is_correct\": \"False\",\n",
    "    \"user_prompt\": \"This looks incorrect to me, <insert reason here>. Output the clusters that have more than 1 member in the correct json format.\",\n",
    "    \"clusters\": [\n",
    "      [\n",
    "        \"cluster_item2\",\n",
    "        \"cluster_item3\"\n",
    "      ]\n",
    "    ]\n",
    "  }\n",
    "}\n",
    "\n",
    "Warnings:\n",
    "\n",
    "Be completely sure to not forget any nodes from the input list.\n",
    "Remember not to base corrections on higher domain level knowledge, that the smaller model might not have, but rather the semantic meaning of the terms.\n",
    "is_correct should be returned as a string \"True\" or \"False\", and only one of these two.\n",
    "If there is any doubt if two groups should cluster together, cluster them separately.\n",
    "Remember to select group representative by enclosing entity in \"**\"\n",
    "\n",
    "\"\"\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 125,
   "metadata": {},
   "outputs": [],
   "source": [
    "constant_user_message = \"\"\"  \n",
    "Here are 2 examples of validation behavior:\n",
    "\n",
    "Example 1, correct behavior:\n",
    "\n",
    "input:\n",
    "{120406: ['difficulty in detecting mutations [phenotype]', 'hereditary breast cancers [phenotype]', 'mutational status of parental tumor cells [phenotype]', 'certain mutation types [phenotype]', 'Well-characterized mutations [phenotype]', 'alternative classifications [phenotype]', 'unique molecular signature [phenotype]', 'oncological diseases [phenotype]', 'independent mutation data [phenotype]', 'inherited disease mutation databases [database]', 'mutational signatures [mutation]', 'mutation types [phenotype]', 'More than half of human cancers [phenotype]', 'somatic and germline mutations [phenotype]', 'Identified mutations [phenotype]', 'lung cancer patients [phenotype]', 'germline transmissible [phenotype]', 'mutations in the germline [phenotype]', 'sporadic breast cancers [phenotype]', 'segregating mutations [phenotype]', 'breast/ovarian cancers [phenotype]', 'non-small-cell lung carcinoma [phenotype]', 'inherited breast cancer cases [phenotype]', 'tumour-specific aberrations [phenotype]', 'list of likely candidate mutations [phenotype]', 'germline-transmitted mutations [phenotype]', 'systematic asymmetries in heritable mutations [genetic feature]', 'targeted heritable mutations [phenotype]', 'fixed mutations in ancestral line [phenotype]', 'various types of mutation [phenotype]', 'several types of mutation [phenotype]', 'screening of these exons [phenotype]', 'many human cancers [phenotype]', 'difficulties to differentiate mutations [phenotype]', 'base substitutions, deletions, insertions, and translocations [phenotype]', 'specific mutational classes [phenotype]', 'germline mutations [phenotype]', 'other types of mutation [phenotype]', 'unmasking of deleterious mutations [phenotype]']}\n",
    "\n",
    "Explanation of correct clustering:\n",
    "\n",
    "[\"**germline mutations [phenotype]**\", \"germline-transmitted mutations [phenotype]\", \"mutations in the germline [phenotype]\", \"germline transmissible [phenotype]\"] all describe inheritable genetic alterations passed from one generation to the next.\n",
    "[\"More than half of human cancers [phenotype]\", \"**oncological diseases [phenotype]**\", \"many human cancers [phenotype]\"] reference broad groups of malignant diseases occurring in humans.\n",
    "[\"**mutation types [phenotype]**\", \"base substitutions, deletions, insertions, and translocations [phenotype]\", \"various types of mutation [phenotype]\", \"several types of mutation [phenotype]\", \"other types of mutation [phenotype]\", \"certain mutation types [phenotype]\", \"specific mutational classes [phenotype]\"] collectively describe or list categories of genetic changes.\n",
    "[\"difficulties to differentiate mutations [phenotype]\", \"**difficulty in detecting mutations [phenotype]**\"] denote challenges in identifying or distinguishing specific genetic variants.\n",
    "[\"inherited breast cancer cases [phenotype]\", \"**hereditary breast cancers [phenotype]**\"] explicitly point to breast cancer instances where genetic risk is inherited.\n",
    "Notes on Separation Certain single-element entries might share partial similarity, such as \"somatic and germline mutations [phenotype]\" or \"targeted heritable mutations [phenotype],\" but remain separate because of additional qualifiers or exact term differences\n",
    "\n",
    "Example 2, incorrect clustering:\n",
    "input:\n",
    "{\"165162\": [[\"**internalized cargo [metabolite]**\", \"endocytosed material [metabolite]\", \"Internalized cargoes [metabolite]\"], [\"**Internalization of endocytic tracer FM4-64 [metabolite]**\", \"internalisation of an endocytic tracer dye [metabolite]\"], [\"endocytosis of transferrin [metabolite]\"], [\"cellular uptake of 14 chemicals [metabolite]\"]]}\n",
    "\n",
    "Explanation of incorrect behavior:\n",
    "\n",
    "[\"**internalized cargo [metabolite]**\", \"endocytosed material [metabolite]\", \"Internalized cargoes [metabolite]\"] all describe general endocytic cargo without specifying particular molecules or numeric differences.\n",
    "[\"**Internalization of endocytic tracer FM4-64 [metabolite]**\", \"internalisation of an endocytic tracer dye [metabolite]\"] are clustered wrong, one specifically mentions FM4-64, while the other refers generally to any tracer dye, and they should therefore be separated following strict separation rules.\n",
    "\n",
    "Now read the cluster, validate and output the desired format:\n",
    "\"\"\""
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Prompts for finetuning dataset (v5)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "system_message = \"\"\"You are senior scientist in plant biology. You NEED to meticulously do reasoning and then cluster given input, and provide the reasoning followed by clusters based on the reasoning performed.\n",
    "\n",
    "The following are the entity clustering guidelines:\n",
    "\t1.\tExact Phrase Matching Matters: Always consider the full phrase, including key biological terms, bracketed text (ignoring minor differences such as spacing, punctuation, correct abbreviations, plurality).\n",
    "\t2.\tStrict (100%) Key Term Separation: Entities with different biological terms MUST be placed in separate clusters.\n",
    "3. Sub-identifier separation: Separate Entities with numeric differences, sub-identifiers, or qualifiers into different groups.\n",
    "\t4.\tAvoid False Similarity: Do NOT cluster two items together in same group just because they share a common word or term.\n",
    "\t5.\tStrict Synonym/Near-Synonym Grouping: Only group entities that refer to the same biological structure, process, meaning or concept.\n",
    "\t6.\tMaintain 100% Precision: When in even small doubt, MUST place entities in separate clusters.\n",
    "\t7.\tPreserve Original Data: No new items should be introduced, no duplicates should be introduced, and no entities should be omitted.\n",
    "8. YOU MUST pickup most appropriate and easy-to-understand cluster representative and enclose it with '**', if there is more than one entity in that particular cluster. For example, pick the full term instead of an abbreviation. MUST mention this in reasoning statement.\n",
    "\t9.\tOutput Format: Always return results in valid JSON format. MUST USE GIVEN KEY.\n",
    "10. Discard Single-element Clusters: Remove any cluster that ends up with only one entity.\n",
    "\n",
    "Your main task is to read the input, do reasoning and output the reasoning and the correct clusters in a valid JSON object format. Input is a json object with given key and the value is the list of biological entities. The output is a json object with given key and the values are two json objects: reasoning and corresponding clustered entities (list of lists where each inner list is one cluster).\n",
    "\n",
    "\"\"\"\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "detailed_description = \"\"\"\n",
    "Input-1\n",
    "\n",
    "{\n",
    "  \"0\": [\n",
    "    \"meiotic arrest [phenotype]\",\n",
    "    \"delayed/arrested meiosis [phenotype]\",\n",
    "    \"absence of meiotic arrest [phenotype]\",\n",
    "    \"meiotic prophase arrest [phenotype]\",\n",
    "    \"males arresting in the middle of prophase I [phenotype]\",\n",
    "    \"early arrest [phenotype]\",\n",
    "    \"leptotene arrest [phenotype]\",\n",
    "    \"pachytene arrest [phenotype]\",\n",
    "    \"arrest in late prophase I [phenotype]\",\n",
    "    \"meiosis I arrest [phenotype]\",\n",
    "    \"meiotic arrest at telophase I [phenotype]\",\n",
    "    \"male meiosis until the end of the first division [phenotype]\",\n",
    "    \"termination of meiosis after anaphase I [phenotype]\",\n",
    "    \"meiotic arrest in anaphase II [phenotype]\",\n",
    "    \"arrested zygotic divisions [phenotype]\",\n",
    "    \"arrested endosperm nuclear divisions [phenotype]\",\n",
    "    \"arrest of the first mitotic division in gametogenesis [phenotype]\",\n",
    "    \"arresting prior to the first mitotic division [phenotype]\",\n",
    "    \"mitotic arrest during female gametogenesis [phenotype]\"\n",
    "  ]\n",
    "}\n",
    "\n",
    "Clustered entities given input-1:\n",
    "\n",
    "{\n",
    "  \"0\": [\n",
    "    [\n",
    "      \"Meiotic block [phenotype]\",\n",
    "      \"**meiotic arrest [phenotype]**\",\n",
    "      \"meiotic arrest phenotype [phenotype]\",\n",
    "      \"meiotic division stop [phenotype]\"\n",
    "    ],\n",
    "    [\n",
    "      \"block in meiosis prophase I [phenotype]\",\n",
    "      \"**meiotic prophase arrest [phenotype]**\"\n",
    "    ],\n",
    "    [\n",
    "      \"**pachytene arrest [phenotype]**\",\n",
    "      \"meiotic arrest at pachytene [phenotype]\"\n",
    "    ],\n",
    "    [\n",
    "      \"arrest at the end of meiosis I [phenotype]\",\n",
    "      \"arrest after meiosis I [phenotype]\",\n",
    "      \"**meiotic arrest at telophase I [phenotype]**\"\n",
    "    ],\n",
    "    [\n",
    "      \"**termination of meiosis after anaphase I [phenotype]**\",\n",
    "      \"premature termination of meiosis after anaphase I [phenotype]\"\n",
    "    ],\n",
    "    [\n",
    "      \"arrest of meiotic progression in anaphase II [phenotype]\",\n",
    "      \"**meiotic arrest in anaphase II [phenotype]**\"\n",
    "    ],\n",
    "    [\n",
    "      \"arresting the first mitosis during gametogenesis [phenotype]\",\n",
    "      \"**arrest of the first mitotic division in gametogenesis [phenotype]**\",\n",
    "      \"FNM half-stop [phenotype]\"\n",
    "    ]\n",
    "  ]\n",
    "}\n",
    "\n",
    "Output-1:\n",
    "{\n",
    "  \"0\": {\n",
    "    \"reasoning\": \"All synonyms describing a complete, generic meiotic halt have to be clustered under **'meiotic arrest [phenotype]'** because they share the same meaning for an unspecified block. Broader prophase I arrests must merge into **'meiotic prophase arrest [phenotype]'** to group all general prophase I blocks without substage detail. Any pachytene-specific terms must fall under **'pachytene arrest [phenotype]'** to capture the unique arrest at the fully synapsed stage. Arrests at the end of meiosis I must be collected under **'meiotic arrest at telophase I [phenotype]'** so that the precise transition at telophase I is clearly indicated. Post–anaphase I failures must be grouped as **'termination of meiosis after anaphase I [phenotype]'** to highlight that the process ends immediately following homolog separation. Anaphase II stalls have to be assigned to **'meiotic arrest in anaphase II [phenotype]'** in order to specify a distinct block during sister chromatid separation. Finally, blocks in the first post–meiotic mitosis must form the cluster **'arrest of the first mitotic division in gametogenesis [phenotype]'** because they occur right after meiosis is completed. Any single-term categories remain excluded from the final output, as each must contain at least two synonymous labels to be shown.\"\n",
    "  }\n",
    "}\n",
    "\n",
    "\n",
    "Input-2\n",
    "{\n",
    "  \"2\": '[\n",
    "    \"Salt-stress severity [treatment]\",\n",
    "    \"high NaCl stress [treatment]\",\n",
    "    \"potassium deprivation stress [treatment]\",\n",
    "    \"salt-stress response [treatment]\",\n",
    "    \"salt stress tolerance [treatment]\",\n",
    "    \"heat and salt stress conditions [treatment]\",\n",
    "    \"prolonged levels of salt stress [treatment]\",\n",
    "    \"recovery from salt stress [treatment]\",\n",
    "    \"salt stress assay [treatment]\",\n",
    "    \"salt stress signaling pathways [treatment]\",\n",
    "    \"gradual salt stress treatments [treatment]\",\n",
    "    \"salt and low temperature stresses [treatment]\",\n",
    "    \"salt and silicon stresses [treatment]\"\n",
    "  ]'\n",
    "}\n",
    "\n",
    "clustered entities for given input-2\n",
    "{\n",
    "  \"2\": [\n",
    "    [\n",
    "      \"Salt-stress severity [treatment]\",\n",
    "      \"**high NaCl stress [treatment]**\",\n",
    "      \"salt-stress response [treatment]\",\n",
    "      \"salt stress tolerance [treatment]\",\n",
    "      \"prolonged levels of salt stress [treatment]\",\n",
    "      \"recovery from salt stress [treatment]\",\n",
    "      \"salt stress assay [treatment]\",\n",
    "      \"salt stress signaling pathways [treatment]\",\n",
    "      \"gradual salt stress treatments [treatment]\"\n",
    "    ]\n",
    "  ]\n",
    "}\n",
    "\n",
    "Output-2:\n",
    "{\n",
    "  \"2\": {\n",
    "    \"reasoning\": \"All treatments that involve only salt (NaCl) must be clustered under **'high NaCl stress [treatment]'** because they focus on the same abiotic stress factor (NaCl) without additional conditions. Each label—such as 'Salt-stress severity,' 'salt-stress response,' or 'salt stress tolerance'—addresses NaCl-induced stress exclusively, which justifies grouping them together. Single-term items mixing salt with other factors (e.g., heat, low temperature, or silicon) or an entirely different stress (e.g., potassium deprivation) must remain separate because their conditions differ fundamentally from pure NaCl stress. Consequently, any single-term clusters are omitted from the final output unless they contain multiple synonymous labels.\"\n",
    "  }\n",
    "}\n",
    "\"\"\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 70,
   "metadata": {},
   "outputs": [],
   "source": [
    "#sample 25 rows from the dataframe where one_item is False and also sample another 25 rows where one_item is True\n",
    "df_one_item = df[df['one_item'] == True].sample(25)\n",
    "df_not_one_item = df[df['one_item'] == False].sample(25)\n",
    "\n",
    "#concatenate the two dataframes\n",
    "df_final = pd.concat([df_one_item, df_not_one_item])\n",
    "#randomize the order of the rows\n",
    "df_final = df_final.sample(frac=1)\n",
    "df_final = df_final.reset_index(drop=True)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 71,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Entity Name</th>\n",
       "      <th>Input</th>\n",
       "      <th>Output</th>\n",
       "      <th>Output&gt;1</th>\n",
       "      <th>one_item</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>109139</td>\n",
       "      <td>{\"109139\": [\"satC core promoter for (-)-strand...</td>\n",
       "      <td>{\"109139\": [[\"119 bp minimal promoter region [...</td>\n",
       "      <td>{\"109139\": [[\"pTRY-B fragment [genomic region]...</td>\n",
       "      <td>False</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>87406</td>\n",
       "      <td>{\"87406\": [\"little to no training [treatment]\"...</td>\n",
       "      <td>{\"87406\": [[\"favorite textbook [education]\"], ...</td>\n",
       "      <td>{\"87406\": []}</td>\n",
       "      <td>True</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>100596</td>\n",
       "      <td>{\"100596\": [\"woody flowering plant(s) [organis...</td>\n",
       "      <td>{\"100596\": [[\"**woody flowering plant(s) [orga...</td>\n",
       "      <td>{\"100596\": [[\"**woody flowering plant(s) [orga...</td>\n",
       "      <td>False</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>78873</td>\n",
       "      <td>{\"78873\": [\"chaperonin 60 subunit beta 2 [prot...</td>\n",
       "      <td>{\"78873\": [[\"chaperonin 60 subunit beta [prote...</td>\n",
       "      <td>{\"78873\": []}</td>\n",
       "      <td>True</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>29577</td>\n",
       "      <td>{\"29577\": [\"pre-ribosome export [process]\", \"p...</td>\n",
       "      <td>{\"29577\": [[\"ribosome export [process]\", \"**pr...</td>\n",
       "      <td>{\"29577\": [[\"ribosome export [process]\", \"**pr...</td>\n",
       "      <td>False</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>122504</td>\n",
       "      <td>{\"122504\": [\"same role [phenotype]\", \"similar ...</td>\n",
       "      <td>{\"122504\": [[\"same role [phenotype]\", \"similar...</td>\n",
       "      <td>{\"122504\": [[\"same role [phenotype]\", \"similar...</td>\n",
       "      <td>False</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>101327</td>\n",
       "      <td>{\"101327\": [\"volatiles emitted by A. thaliana ...</td>\n",
       "      <td>{\"101327\": [[\"**endophyte-inoculated A. thalia...</td>\n",
       "      <td>{\"101327\": [[\"**endophyte-inoculated A. thalia...</td>\n",
       "      <td>False</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>63504</td>\n",
       "      <td>{\"63504\": [\"PsaB-PsaI-PsaH side of PSI [protei...</td>\n",
       "      <td>{\"63504\": [[\"anti-PsaA antibodies [protein]\"],...</td>\n",
       "      <td>{\"63504\": [[\"PsaL protein(s) levels [protein]\"...</td>\n",
       "      <td>False</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>63849</td>\n",
       "      <td>{\"63849\": [\"manganese-binding polypeptides [pr...</td>\n",
       "      <td>{\"63849\": [[\"DCH model [phenotype]\"], [\"**mang...</td>\n",
       "      <td>{\"63849\": [[\"**manganese cluster [protein comp...</td>\n",
       "      <td>False</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9</th>\n",
       "      <td>664</td>\n",
       "      <td>{\"664\": [\"last stages of mite development [phe...</td>\n",
       "      <td>{\"664\": [[\"mature females that emerged [phenot...</td>\n",
       "      <td>{\"664\": [[\"mature females that emerged [phenot...</td>\n",
       "      <td>False</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>10</th>\n",
       "      <td>143138</td>\n",
       "      <td>{\"143138\": [\"VDOF2 [protein]\", \"VDOP1 [protein...</td>\n",
       "      <td>{\"143138\": [[\"VDOP1 [protein]\"], [\"VDOF2 [prot...</td>\n",
       "      <td>{\"143138\": []}</td>\n",
       "      <td>True</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>11</th>\n",
       "      <td>32886</td>\n",
       "      <td>{\"32886\": [\"future position of the new cell wa...</td>\n",
       "      <td>{\"32886\": [[\"site of cell wall fusion [subcell...</td>\n",
       "      <td>{\"32886\": [[\"site of cell wall fusion [subcell...</td>\n",
       "      <td>False</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>12</th>\n",
       "      <td>111625</td>\n",
       "      <td>{\"111625\": [\"sequences between -3604 and -3563...</td>\n",
       "      <td>{\"111625\": [[\"sequences between -3604 and -356...</td>\n",
       "      <td>{\"111625\": []}</td>\n",
       "      <td>True</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>13</th>\n",
       "      <td>30540</td>\n",
       "      <td>{\"30540\": [\"engineers [person]\", \"expert knowl...</td>\n",
       "      <td>{\"30540\": [[\"engineers [person]\"], [\"optics [d...</td>\n",
       "      <td>{\"30540\": []}</td>\n",
       "      <td>True</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>14</th>\n",
       "      <td>149112</td>\n",
       "      <td>{\"149112\": [\"BnaMYB21 [gene]\", \"cnu_m244a [gen...</td>\n",
       "      <td>{\"149112\": [[\"cnu_m244a [gene]\"], [\"BnGPT1-A2 ...</td>\n",
       "      <td>{\"149112\": []}</td>\n",
       "      <td>True</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>15</th>\n",
       "      <td>108934</td>\n",
       "      <td>{\"108934\": [\"hyper-up gene(s) [gene]\", \"hyperm...</td>\n",
       "      <td>{\"108934\": [[\"hyper-up gene(s) [gene]\"], [\"hyp...</td>\n",
       "      <td>{\"108934\": []}</td>\n",
       "      <td>True</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>16</th>\n",
       "      <td>140128</td>\n",
       "      <td>{\"140128\": [\"CAMTA1 dependent positively regul...</td>\n",
       "      <td>{\"140128\": [[\"CAMTA1 dependent positively regu...</td>\n",
       "      <td>{\"140128\": [[\"**upregulated CKX gene(s) [gene]...</td>\n",
       "      <td>False</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>17</th>\n",
       "      <td>102196</td>\n",
       "      <td>{\"102196\": [\"vacuolar-enriched membranes of A....</td>\n",
       "      <td>{\"102196\": [[\"A. thaliana root(s) endodermis [...</td>\n",
       "      <td>{\"102196\": [[\"**A. thaliana root(s) plasma mem...</td>\n",
       "      <td>False</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>18</th>\n",
       "      <td>141136</td>\n",
       "      <td>{\"141136\": [\"pAsr1491 [gene]\", \"pAsr1755 [gene...</td>\n",
       "      <td>{\"141136\": [[\"pAsr11779 [promoter]\"], [\"pAsr14...</td>\n",
       "      <td>{\"141136\": []}</td>\n",
       "      <td>True</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>19</th>\n",
       "      <td>135617</td>\n",
       "      <td>{\"135617\": [\"mitochondrial electron transport ...</td>\n",
       "      <td>{\"135617\": [[\"coordinately regulated gene(s) f...</td>\n",
       "      <td>{\"135617\": [[\"coordinately regulated gene(s) f...</td>\n",
       "      <td>False</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>20</th>\n",
       "      <td>168604</td>\n",
       "      <td>{\"168604\": [\"elevated levels of endogenous 9-N...</td>\n",
       "      <td>{\"168604\": [[\"elevated levels of 9-NAE-HOD [me...</td>\n",
       "      <td>{\"168604\": [[\"elevated levels of 9-NAE-HOD [me...</td>\n",
       "      <td>False</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>21</th>\n",
       "      <td>1979</td>\n",
       "      <td>{\"1979\": [\"embryos at pre-globular and globula...</td>\n",
       "      <td>{\"1979\": [[\"Preglobular embryos [phenotype]\"],...</td>\n",
       "      <td>{\"1979\": []}</td>\n",
       "      <td>True</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>22</th>\n",
       "      <td>126710</td>\n",
       "      <td>{\"126710\": [\"transition of plant(s) chromosoma...</td>\n",
       "      <td>{\"126710\": [[\"transition of plant(s) chromosom...</td>\n",
       "      <td>{\"126710\": []}</td>\n",
       "      <td>True</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>23</th>\n",
       "      <td>10829</td>\n",
       "      <td>{\"10829\": [\"direct antagonism [phenotype]\", \"d...</td>\n",
       "      <td>{\"10829\": [[\"direct antagonism [phenotype]\"], ...</td>\n",
       "      <td>{\"10829\": []}</td>\n",
       "      <td>True</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>24</th>\n",
       "      <td>28668</td>\n",
       "      <td>{\"28668\": [\"regions of active transport [tissu...</td>\n",
       "      <td>{\"28668\": [[\"transport zones [organ]\"], [\"**re...</td>\n",
       "      <td>{\"28668\": [[\"**regions of active transport [ti...</td>\n",
       "      <td>False</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>25</th>\n",
       "      <td>101253</td>\n",
       "      <td>{\"101253\": [\"200 Gy-treated A. thaliana [organ...</td>\n",
       "      <td>{\"101253\": [[\"irradiated seeds of A. thaliana ...</td>\n",
       "      <td>{\"101253\": []}</td>\n",
       "      <td>True</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>26</th>\n",
       "      <td>88364</td>\n",
       "      <td>{\"88364\": [\"improved algorithms [treatment]\", ...</td>\n",
       "      <td>{\"88364\": [[\"low-cost methodology [treatment]\"...</td>\n",
       "      <td>{\"88364\": [[\"low-cost methodology [treatment]\"...</td>\n",
       "      <td>False</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>27</th>\n",
       "      <td>72373</td>\n",
       "      <td>{\"72373\": [\"UMAMIT uniporters [protein]\", \"Amm...</td>\n",
       "      <td>{\"72373\": [[\"**Ammonium transporters [protein]...</td>\n",
       "      <td>{\"72373\": [[\"**Ammonium transporters [protein]...</td>\n",
       "      <td>False</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>28</th>\n",
       "      <td>103231</td>\n",
       "      <td>{\"103231\": [\"yeast strain YK44 [organism]\", \"y...</td>\n",
       "      <td>{\"103231\": [[\"**yeast strain 9.3 [organism]**\"...</td>\n",
       "      <td>{\"103231\": [[\"**yeast strain 9.3 [organism]**\"...</td>\n",
       "      <td>False</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>29</th>\n",
       "      <td>3539</td>\n",
       "      <td>{\"3539\": [\"effective survival strategies [phen...</td>\n",
       "      <td>{\"3539\": [[\"pro-survival strategy [phenotype]\"...</td>\n",
       "      <td>{\"3539\": [[\"pro-survival strategy [phenotype]\"...</td>\n",
       "      <td>False</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>30</th>\n",
       "      <td>114243</td>\n",
       "      <td>{\"114243\": [\"expression in rice [process]\", \"i...</td>\n",
       "      <td>{\"114243\": [[\"overexpression analyses [process...</td>\n",
       "      <td>{\"114243\": [[\"blueberry expression analyses [p...</td>\n",
       "      <td>False</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>31</th>\n",
       "      <td>155976</td>\n",
       "      <td>{\"155976\": [\"only SIP1 homologue [gene]\", \"SiP...</td>\n",
       "      <td>{\"155976\": [[\"SIP [gene]\"], [\"SIP1 [gene]\"], [...</td>\n",
       "      <td>{\"155976\": [[\"**AtDIR1-like [gene]**\", \"DIR1-l...</td>\n",
       "      <td>False</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>32</th>\n",
       "      <td>7573</td>\n",
       "      <td>{\"7573\": [\"Increased human population [phenoty...</td>\n",
       "      <td>{\"7573\": [[\"swelling population [phenotype]\", ...</td>\n",
       "      <td>{\"7573\": [[\"swelling population [phenotype]\", ...</td>\n",
       "      <td>False</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>33</th>\n",
       "      <td>121095</td>\n",
       "      <td>{\"121095\": [\"transformation with NLPPya or NLP...</td>\n",
       "      <td>{\"121095\": [[\"modifying FIL [gene]\"], [\"In pla...</td>\n",
       "      <td>{\"121095\": []}</td>\n",
       "      <td>True</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>34</th>\n",
       "      <td>130281</td>\n",
       "      <td>{\"130281\": [\"A. thaliana pre-tRNA(Tyr) [gene]\"...</td>\n",
       "      <td>{\"130281\": [[\"A. thaliana PRORP2 [gene]\"], [\"t...</td>\n",
       "      <td>{\"130281\": []}</td>\n",
       "      <td>True</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>35</th>\n",
       "      <td>101782</td>\n",
       "      <td>{\"101782\": [\"Goettingen-7 [organism]\", \"access...</td>\n",
       "      <td>{\"101782\": [[\"Goettingen-7 [organism]\"], [\"Goe...</td>\n",
       "      <td>{\"101782\": [[\"accession Dijon-G (Di-G) [organi...</td>\n",
       "      <td>False</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>36</th>\n",
       "      <td>92876</td>\n",
       "      <td>{\"92876\": [\"biopesticide [treatment]\", \"potent...</td>\n",
       "      <td>{\"92876\": [[\"**biopesticides [treatment]**\", \"...</td>\n",
       "      <td>{\"92876\": [[\"**biopesticides [treatment]**\", \"...</td>\n",
       "      <td>False</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>37</th>\n",
       "      <td>52510</td>\n",
       "      <td>{\"52510\": [\"21,570 gene(s) [gene]\", \"21,359 ge...</td>\n",
       "      <td>{\"52510\": [[\"21 gene(s) [gene]\"], [\"21,912 gen...</td>\n",
       "      <td>{\"52510\": []}</td>\n",
       "      <td>True</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>38</th>\n",
       "      <td>42267</td>\n",
       "      <td>{\"42267\": [\"Abstinence by mutual consent (amc)...</td>\n",
       "      <td>{\"42267\": [[\"Abstinence by mutual consent (amc...</td>\n",
       "      <td>{\"42267\": []}</td>\n",
       "      <td>True</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>39</th>\n",
       "      <td>82013</td>\n",
       "      <td>{\"82013\": [\"unique mitochondrial protein(s) [p...</td>\n",
       "      <td>{\"82013\": [[\"protein(s) upregulated in the mit...</td>\n",
       "      <td>{\"82013\": [[\"**protein(s) localized to mitocho...</td>\n",
       "      <td>False</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>40</th>\n",
       "      <td>128564</td>\n",
       "      <td>{\"128564\": [\"43 AaMADS putative gene(s) [gene]...</td>\n",
       "      <td>{\"128564\": [[\"34 MFS gene(s) [gene]\"], [\"57 MA...</td>\n",
       "      <td>{\"128564\": []}</td>\n",
       "      <td>True</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>41</th>\n",
       "      <td>133452</td>\n",
       "      <td>{\"133452\": [\"tannin deficient seed 4 [gene]\", ...</td>\n",
       "      <td>{\"133452\": [[\"Thalianol gene(s) cluster [genom...</td>\n",
       "      <td>{\"133452\": []}</td>\n",
       "      <td>True</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>42</th>\n",
       "      <td>41639</td>\n",
       "      <td>{\"41639\": [\"eleven M&lt;sub&gt;3&lt;/sub&gt; A. thaliana l...</td>\n",
       "      <td>{\"41639\": [[\"20 A. thaliana lines [mutant]\"], ...</td>\n",
       "      <td>{\"41639\": []}</td>\n",
       "      <td>True</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>43</th>\n",
       "      <td>38783</td>\n",
       "      <td>{\"38783\": [\"HVA1-P line [mutant]\", \"Chl accumu...</td>\n",
       "      <td>{\"38783\": [[\"COM lines [mutant]\"], [\"AEQsig6 s...</td>\n",
       "      <td>{\"38783\": []}</td>\n",
       "      <td>True</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>44</th>\n",
       "      <td>52882</td>\n",
       "      <td>{\"52882\": [\"BR receptor complex [gene]\", \"seco...</td>\n",
       "      <td>{\"52882\": [[\"BRCT-containing gene(s) [gene]\"],...</td>\n",
       "      <td>{\"52882\": []}</td>\n",
       "      <td>True</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>45</th>\n",
       "      <td>75945</td>\n",
       "      <td>{\"75945\": [\"domain I of IAA17 [protein domain]...</td>\n",
       "      <td>{\"75945\": [[\"EBD [protein domain]\"], [\"ES doma...</td>\n",
       "      <td>{\"75945\": [[\"**ETR1 receiver domain [protein d...</td>\n",
       "      <td>False</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>46</th>\n",
       "      <td>96226</td>\n",
       "      <td>{\"96226\": [\"leaf blades, nodules, and buds [or...</td>\n",
       "      <td>{\"96226\": [[\"D. nobile leaf [organ]\"], [\"leaf ...</td>\n",
       "      <td>{\"96226\": []}</td>\n",
       "      <td>True</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>47</th>\n",
       "      <td>114365</td>\n",
       "      <td>{\"114365\": [\"3 s every 3 min [phenotype]\", \"th...</td>\n",
       "      <td>{\"114365\": [[\"CBF3 phase and waveform [phenoty...</td>\n",
       "      <td>{\"114365\": []}</td>\n",
       "      <td>True</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>48</th>\n",
       "      <td>2276</td>\n",
       "      <td>{\"2276\": [\"F1 trichome phenotype [phenotype]\",...</td>\n",
       "      <td>{\"2276\": [[\"F1 trichome phenotype [phenotype]\"...</td>\n",
       "      <td>{\"2276\": []}</td>\n",
       "      <td>True</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>49</th>\n",
       "      <td>14243</td>\n",
       "      <td>{\"14243\": [\"abnormally flattened siliques [phe...</td>\n",
       "      <td>{\"14243\": [[\"short and curled siliques [phenot...</td>\n",
       "      <td>{\"14243\": []}</td>\n",
       "      <td>True</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "    Entity Name                                              Input  \\\n",
       "0        109139  {\"109139\": [\"satC core promoter for (-)-strand...   \n",
       "1         87406  {\"87406\": [\"little to no training [treatment]\"...   \n",
       "2        100596  {\"100596\": [\"woody flowering plant(s) [organis...   \n",
       "3         78873  {\"78873\": [\"chaperonin 60 subunit beta 2 [prot...   \n",
       "4         29577  {\"29577\": [\"pre-ribosome export [process]\", \"p...   \n",
       "5        122504  {\"122504\": [\"same role [phenotype]\", \"similar ...   \n",
       "6        101327  {\"101327\": [\"volatiles emitted by A. thaliana ...   \n",
       "7         63504  {\"63504\": [\"PsaB-PsaI-PsaH side of PSI [protei...   \n",
       "8         63849  {\"63849\": [\"manganese-binding polypeptides [pr...   \n",
       "9           664  {\"664\": [\"last stages of mite development [phe...   \n",
       "10       143138  {\"143138\": [\"VDOF2 [protein]\", \"VDOP1 [protein...   \n",
       "11        32886  {\"32886\": [\"future position of the new cell wa...   \n",
       "12       111625  {\"111625\": [\"sequences between -3604 and -3563...   \n",
       "13        30540  {\"30540\": [\"engineers [person]\", \"expert knowl...   \n",
       "14       149112  {\"149112\": [\"BnaMYB21 [gene]\", \"cnu_m244a [gen...   \n",
       "15       108934  {\"108934\": [\"hyper-up gene(s) [gene]\", \"hyperm...   \n",
       "16       140128  {\"140128\": [\"CAMTA1 dependent positively regul...   \n",
       "17       102196  {\"102196\": [\"vacuolar-enriched membranes of A....   \n",
       "18       141136  {\"141136\": [\"pAsr1491 [gene]\", \"pAsr1755 [gene...   \n",
       "19       135617  {\"135617\": [\"mitochondrial electron transport ...   \n",
       "20       168604  {\"168604\": [\"elevated levels of endogenous 9-N...   \n",
       "21         1979  {\"1979\": [\"embryos at pre-globular and globula...   \n",
       "22       126710  {\"126710\": [\"transition of plant(s) chromosoma...   \n",
       "23        10829  {\"10829\": [\"direct antagonism [phenotype]\", \"d...   \n",
       "24        28668  {\"28668\": [\"regions of active transport [tissu...   \n",
       "25       101253  {\"101253\": [\"200 Gy-treated A. thaliana [organ...   \n",
       "26        88364  {\"88364\": [\"improved algorithms [treatment]\", ...   \n",
       "27        72373  {\"72373\": [\"UMAMIT uniporters [protein]\", \"Amm...   \n",
       "28       103231  {\"103231\": [\"yeast strain YK44 [organism]\", \"y...   \n",
       "29         3539  {\"3539\": [\"effective survival strategies [phen...   \n",
       "30       114243  {\"114243\": [\"expression in rice [process]\", \"i...   \n",
       "31       155976  {\"155976\": [\"only SIP1 homologue [gene]\", \"SiP...   \n",
       "32         7573  {\"7573\": [\"Increased human population [phenoty...   \n",
       "33       121095  {\"121095\": [\"transformation with NLPPya or NLP...   \n",
       "34       130281  {\"130281\": [\"A. thaliana pre-tRNA(Tyr) [gene]\"...   \n",
       "35       101782  {\"101782\": [\"Goettingen-7 [organism]\", \"access...   \n",
       "36        92876  {\"92876\": [\"biopesticide [treatment]\", \"potent...   \n",
       "37        52510  {\"52510\": [\"21,570 gene(s) [gene]\", \"21,359 ge...   \n",
       "38        42267  {\"42267\": [\"Abstinence by mutual consent (amc)...   \n",
       "39        82013  {\"82013\": [\"unique mitochondrial protein(s) [p...   \n",
       "40       128564  {\"128564\": [\"43 AaMADS putative gene(s) [gene]...   \n",
       "41       133452  {\"133452\": [\"tannin deficient seed 4 [gene]\", ...   \n",
       "42        41639  {\"41639\": [\"eleven M<sub>3</sub> A. thaliana l...   \n",
       "43        38783  {\"38783\": [\"HVA1-P line [mutant]\", \"Chl accumu...   \n",
       "44        52882  {\"52882\": [\"BR receptor complex [gene]\", \"seco...   \n",
       "45        75945  {\"75945\": [\"domain I of IAA17 [protein domain]...   \n",
       "46        96226  {\"96226\": [\"leaf blades, nodules, and buds [or...   \n",
       "47       114365  {\"114365\": [\"3 s every 3 min [phenotype]\", \"th...   \n",
       "48         2276  {\"2276\": [\"F1 trichome phenotype [phenotype]\",...   \n",
       "49        14243  {\"14243\": [\"abnormally flattened siliques [phe...   \n",
       "\n",
       "                                               Output  \\\n",
       "0   {\"109139\": [[\"119 bp minimal promoter region [...   \n",
       "1   {\"87406\": [[\"favorite textbook [education]\"], ...   \n",
       "2   {\"100596\": [[\"**woody flowering plant(s) [orga...   \n",
       "3   {\"78873\": [[\"chaperonin 60 subunit beta [prote...   \n",
       "4   {\"29577\": [[\"ribosome export [process]\", \"**pr...   \n",
       "5   {\"122504\": [[\"same role [phenotype]\", \"similar...   \n",
       "6   {\"101327\": [[\"**endophyte-inoculated A. thalia...   \n",
       "7   {\"63504\": [[\"anti-PsaA antibodies [protein]\"],...   \n",
       "8   {\"63849\": [[\"DCH model [phenotype]\"], [\"**mang...   \n",
       "9   {\"664\": [[\"mature females that emerged [phenot...   \n",
       "10  {\"143138\": [[\"VDOP1 [protein]\"], [\"VDOF2 [prot...   \n",
       "11  {\"32886\": [[\"site of cell wall fusion [subcell...   \n",
       "12  {\"111625\": [[\"sequences between -3604 and -356...   \n",
       "13  {\"30540\": [[\"engineers [person]\"], [\"optics [d...   \n",
       "14  {\"149112\": [[\"cnu_m244a [gene]\"], [\"BnGPT1-A2 ...   \n",
       "15  {\"108934\": [[\"hyper-up gene(s) [gene]\"], [\"hyp...   \n",
       "16  {\"140128\": [[\"CAMTA1 dependent positively regu...   \n",
       "17  {\"102196\": [[\"A. thaliana root(s) endodermis [...   \n",
       "18  {\"141136\": [[\"pAsr11779 [promoter]\"], [\"pAsr14...   \n",
       "19  {\"135617\": [[\"coordinately regulated gene(s) f...   \n",
       "20  {\"168604\": [[\"elevated levels of 9-NAE-HOD [me...   \n",
       "21  {\"1979\": [[\"Preglobular embryos [phenotype]\"],...   \n",
       "22  {\"126710\": [[\"transition of plant(s) chromosom...   \n",
       "23  {\"10829\": [[\"direct antagonism [phenotype]\"], ...   \n",
       "24  {\"28668\": [[\"transport zones [organ]\"], [\"**re...   \n",
       "25  {\"101253\": [[\"irradiated seeds of A. thaliana ...   \n",
       "26  {\"88364\": [[\"low-cost methodology [treatment]\"...   \n",
       "27  {\"72373\": [[\"**Ammonium transporters [protein]...   \n",
       "28  {\"103231\": [[\"**yeast strain 9.3 [organism]**\"...   \n",
       "29  {\"3539\": [[\"pro-survival strategy [phenotype]\"...   \n",
       "30  {\"114243\": [[\"overexpression analyses [process...   \n",
       "31  {\"155976\": [[\"SIP [gene]\"], [\"SIP1 [gene]\"], [...   \n",
       "32  {\"7573\": [[\"swelling population [phenotype]\", ...   \n",
       "33  {\"121095\": [[\"modifying FIL [gene]\"], [\"In pla...   \n",
       "34  {\"130281\": [[\"A. thaliana PRORP2 [gene]\"], [\"t...   \n",
       "35  {\"101782\": [[\"Goettingen-7 [organism]\"], [\"Goe...   \n",
       "36  {\"92876\": [[\"**biopesticides [treatment]**\", \"...   \n",
       "37  {\"52510\": [[\"21 gene(s) [gene]\"], [\"21,912 gen...   \n",
       "38  {\"42267\": [[\"Abstinence by mutual consent (amc...   \n",
       "39  {\"82013\": [[\"protein(s) upregulated in the mit...   \n",
       "40  {\"128564\": [[\"34 MFS gene(s) [gene]\"], [\"57 MA...   \n",
       "41  {\"133452\": [[\"Thalianol gene(s) cluster [genom...   \n",
       "42  {\"41639\": [[\"20 A. thaliana lines [mutant]\"], ...   \n",
       "43  {\"38783\": [[\"COM lines [mutant]\"], [\"AEQsig6 s...   \n",
       "44  {\"52882\": [[\"BRCT-containing gene(s) [gene]\"],...   \n",
       "45  {\"75945\": [[\"EBD [protein domain]\"], [\"ES doma...   \n",
       "46  {\"96226\": [[\"D. nobile leaf [organ]\"], [\"leaf ...   \n",
       "47  {\"114365\": [[\"CBF3 phase and waveform [phenoty...   \n",
       "48  {\"2276\": [[\"F1 trichome phenotype [phenotype]\"...   \n",
       "49  {\"14243\": [[\"short and curled siliques [phenot...   \n",
       "\n",
       "                                             Output>1  one_item  \n",
       "0   {\"109139\": [[\"pTRY-B fragment [genomic region]...     False  \n",
       "1                                       {\"87406\": []}      True  \n",
       "2   {\"100596\": [[\"**woody flowering plant(s) [orga...     False  \n",
       "3                                       {\"78873\": []}      True  \n",
       "4   {\"29577\": [[\"ribosome export [process]\", \"**pr...     False  \n",
       "5   {\"122504\": [[\"same role [phenotype]\", \"similar...     False  \n",
       "6   {\"101327\": [[\"**endophyte-inoculated A. thalia...     False  \n",
       "7   {\"63504\": [[\"PsaL protein(s) levels [protein]\"...     False  \n",
       "8   {\"63849\": [[\"**manganese cluster [protein comp...     False  \n",
       "9   {\"664\": [[\"mature females that emerged [phenot...     False  \n",
       "10                                     {\"143138\": []}      True  \n",
       "11  {\"32886\": [[\"site of cell wall fusion [subcell...     False  \n",
       "12                                     {\"111625\": []}      True  \n",
       "13                                      {\"30540\": []}      True  \n",
       "14                                     {\"149112\": []}      True  \n",
       "15                                     {\"108934\": []}      True  \n",
       "16  {\"140128\": [[\"**upregulated CKX gene(s) [gene]...     False  \n",
       "17  {\"102196\": [[\"**A. thaliana root(s) plasma mem...     False  \n",
       "18                                     {\"141136\": []}      True  \n",
       "19  {\"135617\": [[\"coordinately regulated gene(s) f...     False  \n",
       "20  {\"168604\": [[\"elevated levels of 9-NAE-HOD [me...     False  \n",
       "21                                       {\"1979\": []}      True  \n",
       "22                                     {\"126710\": []}      True  \n",
       "23                                      {\"10829\": []}      True  \n",
       "24  {\"28668\": [[\"**regions of active transport [ti...     False  \n",
       "25                                     {\"101253\": []}      True  \n",
       "26  {\"88364\": [[\"low-cost methodology [treatment]\"...     False  \n",
       "27  {\"72373\": [[\"**Ammonium transporters [protein]...     False  \n",
       "28  {\"103231\": [[\"**yeast strain 9.3 [organism]**\"...     False  \n",
       "29  {\"3539\": [[\"pro-survival strategy [phenotype]\"...     False  \n",
       "30  {\"114243\": [[\"blueberry expression analyses [p...     False  \n",
       "31  {\"155976\": [[\"**AtDIR1-like [gene]**\", \"DIR1-l...     False  \n",
       "32  {\"7573\": [[\"swelling population [phenotype]\", ...     False  \n",
       "33                                     {\"121095\": []}      True  \n",
       "34                                     {\"130281\": []}      True  \n",
       "35  {\"101782\": [[\"accession Dijon-G (Di-G) [organi...     False  \n",
       "36  {\"92876\": [[\"**biopesticides [treatment]**\", \"...     False  \n",
       "37                                      {\"52510\": []}      True  \n",
       "38                                      {\"42267\": []}      True  \n",
       "39  {\"82013\": [[\"**protein(s) localized to mitocho...     False  \n",
       "40                                     {\"128564\": []}      True  \n",
       "41                                     {\"133452\": []}      True  \n",
       "42                                      {\"41639\": []}      True  \n",
       "43                                      {\"38783\": []}      True  \n",
       "44                                      {\"52882\": []}      True  \n",
       "45  {\"75945\": [[\"**ETR1 receiver domain [protein d...     False  \n",
       "46                                      {\"96226\": []}      True  \n",
       "47                                     {\"114365\": []}      True  \n",
       "48                                       {\"2276\": []}      True  \n",
       "49                                      {\"14243\": []}      True  "
      ]
     },
     "execution_count": 71,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df_final"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 72,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "one_item\n",
       "False    25\n",
       "True     25\n",
       "Name: count, dtype: int64"
      ]
     },
     "execution_count": 72,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df_final['one_item'].value_counts()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 128,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Entity Name</th>\n",
       "      <th>Input</th>\n",
       "      <th>Output</th>\n",
       "      <th>Output&gt;1</th>\n",
       "      <th>one_item</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>58963</td>\n",
       "      <td>{\"58963\": [\"Gardea-Torresdey et al. [researche...</td>\n",
       "      <td>{\"58963\": [[\"Gardea-Torresdey et al. [research...</td>\n",
       "      <td>{\"58963\": []}</td>\n",
       "      <td>True</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>22775</td>\n",
       "      <td>{\"22775\": [\"physiological transport of macro-m...</td>\n",
       "      <td>{\"22775\": [[\"substance that could be transloca...</td>\n",
       "      <td>{\"22775\": [[\"**transportation [phenotype]**\", ...</td>\n",
       "      <td>False</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>22996</td>\n",
       "      <td>{\"22996\": [\"stomatal conductance analyses [phe...</td>\n",
       "      <td>{\"22996\": [[\"**transpiration and stomatal cond...</td>\n",
       "      <td>{\"22996\": [[\"**transpiration and stomatal cond...</td>\n",
       "      <td>False</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>100116</td>\n",
       "      <td>{\"100116\": [\"modern clade [gene]\", \"modern cla...</td>\n",
       "      <td>{\"100116\": [[\"Clade M [clade]\"], [\"Clade SL [c...</td>\n",
       "      <td>{\"100116\": []}</td>\n",
       "      <td>True</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>133255</td>\n",
       "      <td>{\"133255\": [\"Plastidic LPPs [gene]\", \"PLSP2B [...</td>\n",
       "      <td>{\"133255\": [[\"**PLSP2A [gene]**\", \"PLSP2A gene...</td>\n",
       "      <td>{\"133255\": [[\"**PLSP2A [gene]**\", \"PLSP2A gene...</td>\n",
       "      <td>False</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>995</th>\n",
       "      <td>31615</td>\n",
       "      <td>{\"31615\": [\"auxin-regulated endocytosis [proce...</td>\n",
       "      <td>{\"31615\": [[\"auxin-mediated modulation of the ...</td>\n",
       "      <td>{\"31615\": [[\"auxin-dependent inhibition of end...</td>\n",
       "      <td>False</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>996</th>\n",
       "      <td>157227</td>\n",
       "      <td>{\"157227\": [\"SlSAUR1-like [gene]\", \"ZmSAUR1 [g...</td>\n",
       "      <td>{\"157227\": [[\"GmSAUR [gene]\"], [\"StSAUR [gene]...</td>\n",
       "      <td>{\"157227\": [[\"ZmSAUR1 (Zea mays SAURs) [gene]\"...</td>\n",
       "      <td>False</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>997</th>\n",
       "      <td>131559</td>\n",
       "      <td>{\"131559\": [\"BtuC gene(s) [gene]\", \"BtuN gene(...</td>\n",
       "      <td>{\"131559\": [[\"BtuB gene(s) [gene]\"], [\"BtuC ge...</td>\n",
       "      <td>{\"131559\": []}</td>\n",
       "      <td>True</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>998</th>\n",
       "      <td>99091</td>\n",
       "      <td>{\"99091\": [\"O. fragrans 'Liuyejingui' (OFL) [o...</td>\n",
       "      <td>{\"99091\": [[\"Genome sequencing of O. fragrans ...</td>\n",
       "      <td>{\"99091\": [[\"**O. fragrans 'Liuyejingui' (OFL)...</td>\n",
       "      <td>False</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>999</th>\n",
       "      <td>65306</td>\n",
       "      <td>{\"65306\": [\"CO/NF-YB/NF-YC trimer complex [pro...</td>\n",
       "      <td>{\"65306\": [[\"**CO/NF-Y complex [protein comple...</td>\n",
       "      <td>{\"65306\": [[\"**CO/NF-Y complex [protein comple...</td>\n",
       "      <td>False</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>1000 rows × 5 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "     Entity Name                                              Input  \\\n",
       "0          58963  {\"58963\": [\"Gardea-Torresdey et al. [researche...   \n",
       "1          22775  {\"22775\": [\"physiological transport of macro-m...   \n",
       "2          22996  {\"22996\": [\"stomatal conductance analyses [phe...   \n",
       "3         100116  {\"100116\": [\"modern clade [gene]\", \"modern cla...   \n",
       "4         133255  {\"133255\": [\"Plastidic LPPs [gene]\", \"PLSP2B [...   \n",
       "..           ...                                                ...   \n",
       "995        31615  {\"31615\": [\"auxin-regulated endocytosis [proce...   \n",
       "996       157227  {\"157227\": [\"SlSAUR1-like [gene]\", \"ZmSAUR1 [g...   \n",
       "997       131559  {\"131559\": [\"BtuC gene(s) [gene]\", \"BtuN gene(...   \n",
       "998        99091  {\"99091\": [\"O. fragrans 'Liuyejingui' (OFL) [o...   \n",
       "999        65306  {\"65306\": [\"CO/NF-YB/NF-YC trimer complex [pro...   \n",
       "\n",
       "                                                Output  \\\n",
       "0    {\"58963\": [[\"Gardea-Torresdey et al. [research...   \n",
       "1    {\"22775\": [[\"substance that could be transloca...   \n",
       "2    {\"22996\": [[\"**transpiration and stomatal cond...   \n",
       "3    {\"100116\": [[\"Clade M [clade]\"], [\"Clade SL [c...   \n",
       "4    {\"133255\": [[\"**PLSP2A [gene]**\", \"PLSP2A gene...   \n",
       "..                                                 ...   \n",
       "995  {\"31615\": [[\"auxin-mediated modulation of the ...   \n",
       "996  {\"157227\": [[\"GmSAUR [gene]\"], [\"StSAUR [gene]...   \n",
       "997  {\"131559\": [[\"BtuB gene(s) [gene]\"], [\"BtuC ge...   \n",
       "998  {\"99091\": [[\"Genome sequencing of O. fragrans ...   \n",
       "999  {\"65306\": [[\"**CO/NF-Y complex [protein comple...   \n",
       "\n",
       "                                              Output>1  one_item  \n",
       "0                                        {\"58963\": []}      True  \n",
       "1    {\"22775\": [[\"**transportation [phenotype]**\", ...     False  \n",
       "2    {\"22996\": [[\"**transpiration and stomatal cond...     False  \n",
       "3                                       {\"100116\": []}      True  \n",
       "4    {\"133255\": [[\"**PLSP2A [gene]**\", \"PLSP2A gene...     False  \n",
       "..                                                 ...       ...  \n",
       "995  {\"31615\": [[\"auxin-dependent inhibition of end...     False  \n",
       "996  {\"157227\": [[\"ZmSAUR1 (Zea mays SAURs) [gene]\"...     False  \n",
       "997                                     {\"131559\": []}      True  \n",
       "998  {\"99091\": [[\"**O. fragrans 'Liuyejingui' (OFL)...     False  \n",
       "999  {\"65306\": [[\"**CO/NF-Y complex [protein comple...     False  \n",
       "\n",
       "[1000 rows x 5 columns]"
      ]
     },
     "execution_count": 128,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "entity_dict = {}\n",
    "for item in df[\"Output\"]:\n",
    "    try:\n",
    "        parsed_dict = json.loads(item)\n",
    "        entity_dict.update(parsed_dict)\n",
    "    except json.JSONDecodeError:\n",
    "        print(f\"Skipping invalid JSON: {item}\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 130,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Number of chunks: 1000\n"
     ]
    }
   ],
   "source": [
    "import ast\n",
    "\n",
    "def split_dict(d, chunk_size=100):\n",
    "    keys = list(d.keys())\n",
    "    chunks = []\n",
    "    for i in range(0, len(keys), chunk_size):\n",
    "        chunk_keys = keys[i:i+chunk_size]\n",
    "        chunk_dict = {k: d[k] for k in chunk_keys}\n",
    "        chunks.append(chunk_dict)\n",
    "    return chunks\n",
    "\n",
    "def flatten_bracketed_strings(value_list):\n",
    "    \"\"\"\n",
    "    Takes a list. For each item:\n",
    "      - If item is a string that looks like '[...]', parse it and extend the list.\n",
    "      - Otherwise, keep as is.\n",
    "    Returns a new flattened list.\n",
    "    \"\"\"\n",
    "    new_list = []\n",
    "    for val in value_list:\n",
    "        if (\n",
    "            isinstance(val, str) \n",
    "            and val.strip().startswith(\"[\") \n",
    "            and val.strip().endswith(\"]\")\n",
    "        ):\n",
    "            # Attempt to parse the bracketed string\n",
    "            try:\n",
    "                parsed = ast.literal_eval(val)  # convert string -> Python list\n",
    "                if isinstance(parsed, list):\n",
    "                    new_list.extend(parsed)  # flatten\n",
    "                else:\n",
    "                    # If it's not a list, just append as-is\n",
    "                    new_list.append(val)\n",
    "            except (SyntaxError, ValueError):\n",
    "                # If parsing fails, keep original\n",
    "                new_list.append(val)\n",
    "        else:\n",
    "            new_list.append(val)\n",
    "    return new_list\n",
    "\n",
    "\n",
    "\n",
    "chunks = split_dict(entity_dict, 1) # Use batch size of 1\n",
    "print('Number of chunks:',len(chunks))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 131,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "{'58963': [['Gardea-Torresdey et al. [researcher]'], ['Barrena et al. [researcher]']]}\n",
      "-------------------\n",
      "{'22775': [['substance that could be translocated [phenotype]'], ['transport of metabolites [phenotype]'], ['**transportation [phenotype]**', 'transportion [phenotype]'], ['transport of intermediates [phenotype]'], ['dye transport [phenotype]'], ['transport issues [phenotype]'], ['As transport [phenotype]'], ['transport and distribution throughout the plant(s) [phenotype]'], ['traffic [phenotype]'], ['transport within the plant(s) [phenotype]', 'transport through the plant(s) [phenotype]', '**transport in planta [phenotype]**'], ['secondary metabolites transportation [phenotype]', '**secondary metabolite transport [phenotype]**'], ['systemic movement via the phloem [phenotype]'], ['transport of compounds [phenotype]'], ['changes in transport rates [phenotype]'], ['chemical transport [phenotype]'], ['glucosinolates transport [phenotype]'], ['**photoassimilate transport [phenotype]**', 'transport of photo-assimilates [phenotype]'], ['transport of matter [phenotype]'], ['transportation of terpenoids [phenotype]'], ['transport phenomena [phenotype]'], ['transport and accumulation of metabolites [phenotype]'], ['**enzyme transport [phenotype]**', 'transport of enzymes [phenotype]'], ['transport events [phenotype]'], ['transportation of lignin [phenotype]'], ['spatiotemporal transport in intact plant(s) [phenotype]'], ['physiological transport of macro-molecules [phenotype]'], ['transport of chlorophyll metabolites [phenotype]'], ['transport among different tissues [phenotype]'], ['biochemical transport [phenotype]'], ['transport changes [phenotype]'], ['uptake and translocation within plant(s) [phenotype]']]}\n",
      "-------------------\n",
      "{'22996': [['**transpiration and stomatal conductance values [phenotype]**', 'E and g s [phenotype]'], ['consideration of stomatal conductance [phenotype]', '**stomatal conductance measurements [phenotype]**', 'measured stomatal conductance [phenotype]', 'Conductance measurements [phenotype]', 'stomatal conductances [phenotype]', 'direct measurements of g s [phenotype]', 'stomatal conductance analyses [phenotype]', 'conductance levels [phenotype]', 'measuring stomatal conductance [phenotype]'], ['derived model [phenotype]'], ['engineering stomatal conductance [phenotype]'], ['K(leaf) values [phenotype]'], ['stomatal conductance, transpiration rate, hydraulic conductivity and photosynthetic efficiency [phenotype]'], ['AtPIP2;5 regulating gm [phenotype]'], ['kinetic stomatal conductance analyses [phenotype]', '**time-resolved stomatal conductance analyses [phenotype]**', 'time-resolved measurements of stomatal conductance [phenotype]'], ['number of open stomata and amount of water loss through transpiration [phenotype]'], ['stomatal conductance, transpiration rate, relative-water content of plant(s) leaf [phenotype]'], ['WUE and Stomatal conductance [phenotype]'], ['both SD and photosynthetic gas exchange traits [phenotype]'], ['regulation of stomatal conductance through FER [phenotype]'], ['methods to quickly assess the stomatal conductance (gₛ) [phenotype]']]}\n",
      "-------------------\n",
      "{'100116': [['Clade M [clade]'], ['Clade SL [clade]'], ['Clade SH [clade]'], ['Clade U [clade]'], ['modern clade [gene]'], ['modern clade WOX members [gene]']]}\n",
      "-------------------\n",
      "{'133255': [['**PLSP2A [gene]**', 'PLSP2A gene(s) [gene]'], ['PLSP2 [gene]'], ['PLSP2B [gene]'], ['LPP2 [gene]'], ['LPE1 [gene]'], ['**LPP gamma [gene]**', 'LPPႰ [gene]'], ['LPPႱ [gene]'], ['LPPႲ [gene]'], ['Plastidic LPPs [gene]'], ['ptLPD2 [gene]'], ['SUPPRESSOR OF LLP1 1 (SOL1) [gene]'], ['HvLPP1 [gene]'], ['HvLPP2 [gene]'], ['PSPTO3648 [gene]'], ['cyanobacterial LPP [gene]']]}\n",
      "-------------------\n",
      "{'2276': [['F1 trichome phenotype [phenotype]'], ['near-full hybrid NFH3 [phenotype]'], ['F25H stressed progeny [phenotype]'], ['R1 progenies [phenotype]'], ['Mendelian segregation in the F1 generation [phenotype]'], ['fc2 phenotypes [phenotype]'], ['production of normal F2 seeds [phenotype]'], ['dwarf F2 plant(s) [phenotype]'], ['phenotypically normal F2 progeny [phenotype]'], ['wild-type phenotype F1 plant(s) [phenotype]'], ['100% F1 progeny [phenotype]'], ['wt progeny from F3 progeny pools [phenotype]'], ['white F2 progeny [phenotype]'], ['white panicle phenotype in F1 progeny [phenotype]'], ['most of the S1 progeny [phenotype]']]}\n",
      "-------------------\n",
      "{'65264': [['MYC2-JAZ-NINJA-TPL complex [protein complex]'], ['**JAZ/NINJA/TPL repressor complex [protein complex]**', 'TPL-NINJIA-JAZ complex [protein complex]'], ['Myc-TPL [protein]'], ['Carbon Catabolite Repression 4-Negative on TATA-less (NOT) complex subunit NOT2 [protein]'], ['TCP Topless EAR repressor complexes [protein complex]'], ['TPR1-TPR2 complex [protein complex]'], ['Tup1 [protein complex]'], ['TOPLESS-RELATED (TPR1-4) [protein complex]'], ['TPL family [protein complex]'], ['WOX-TPL repressor complex [protein complex]'], ['CDF-TPL transcriptional complex [protein complex]'], ['LUH-SLK2 complex [protein complex]'], ['BAH-PHD-CPL2 [protein complex]'], ['BAH-PHD-CPL2 pathway [process]'], ['Suppressor of Ty (SPT) module [protein complex]'], ['subnetwork with double-negative feedback via TF monomers [protein complex]'], ['TPL-related protein(s) complex [protein]'], ['**TPL/TPR [protein complex]**', 'TPL-TPR complexes [protein complex]', 'TPL/TPRs [protein complex]'], ['TPL oligomer [protein complex]'], ['NINJA/TOPLESS transcriptional co-repressor complex [protein complex]'], ['NINJA-TPL [protein]'], ['binding to NINJA/TPL repressor complex [protein complex]'], ['recruitment of corepressor NINJA [protein]'], ['TIE1 and TPL/TPRs [protein complex]'], ['TPL/TPR repressor complex [protein complex]'], ['TPL/TPR trimeric complex [protein complex]'], ['TPL-AtMed21 interaction [protein complex]'], ['TOPLESS (TPL) co-repressor complex [protein complex]'], ['docking point for TPL-HDAC complex [protein complex]'], ['five members of the TPL/Groucho/TLE co-repressor family [protein complex]'], ['TOPLESS repressive complex [protein complex]'], ['TPL/TPR1 [protein complex]'], ['TPL/TPR co-repressor activity [protein complex]'], ['TPL/TPR co-repressors [protein complex]'], ['TOPLESS complex [protein complex]']]}\n",
      "-------------------\n",
      "{'82106': [['mammalian isoforms [organism]'], ['multiple isoforms in metazoans [organism]'], ['yeast isoforms [organism]'], ['copies of eIF(iso)4E [protein]'], ['multiple forms in mammals [protein]'], ['isoforms and organisms [organism]'], ['more than one eIF4E protein(s) forms [protein]'], ['complex isoform distribution [protein]'], ['isoforms of many species [organism]'], ['many isoforms [protein]'], ['isoforms from animals and yeast [organism]']]}\n",
      "-------------------\n",
      "{'169694': [['reduced accumulation of fumarate [metabolite]'], ['small loss of non-photoactive plastoquinone-9 [metabolite]'], ['zeatin accumulation relieved [metabolite]'], ['reduced production of volatiles [metabolite]'], ['small increase in pyrimidine intermediates [metabolite]'], ['decreased production of phloretic acid [metabolite]'], ['slight decrease in vernolic acid accumulation [metabolite]']]}\n",
      "-------------------\n",
      "{'33354': [['**mitochondrial redox status [subcellular compartment]**', 'redox status of mitochondria [subcellular compartment]', 'Redox state in mitochondria [subcellular compartment]', 'redox status of the mitochondria [subcellular compartment]', 'redox status in the mitochondria [subcellular compartment]', 'mitochondrial redox homeostasis [subcellular compartment]', 'mitochondria redox states [subcellular compartment]'], ['oxidation in mitochondria [subcellular compartment]'], ['**Fewer mitochondria [subcellular compartment]**', 'decrease in number of mitochondria [subcellular compartment]', 'decrease in mitochondria [subcellular compartment]', 'reduced number of mitochondria [subcellular compartment]', 'number of less efficient mitochondria [subcellular compartment]'], ['ROS homeostasis in mitochondria [subcellular compartment]']]}\n",
      "-------------------\n",
      "{'89650': [['Elevated auxin response [metabolite]'], ['Al stress-induced auxin synthesis [metabolite]'], ['Al-induced auxin accumulation [metabolite]'], ['increases in auxin sensitivity [metabolite]'], ['altered sensitivities to auxin [plant hormone]'], ['Al-induced auxin accumulation in root(s) TZ [metabolite]'], ['increased auxin content in root(s) apical meristem zone [tissue]'], ['extreme sensitivities to exogenous auxin [metabolite]']]}\n",
      "-------------------\n",
      "{'156043': [['FLIP [gene]'], ['**HEI10 gene(s) [gene]**', 'E3 ligase HEI10 [gene]', 'Pro-crossover E3 ligase HEI10 [gene]']]}\n",
      "-------------------\n",
      "{'116946': [['OTU157779 abundance [organism]'], ['SynCom size [phenotype]'], ['UV fractograms [phenotype]'], ['number of clones [phenotype]'], ['total UMI counts [phenotype]'], ['number of nuclear DNA-containing clones on each macroarray [phenotype]'], ['**quantification results [phenotype]**', 'actual numbers obtained in the assays [phenotype]'], ['H2Bub levels [metabolite]'], ['**7634 colony-forming units [phenotype]**', '520 colonies [phenotype]'], ['CT values of 72 samples [phenotype]'], ['3UI-Ts densities [phenotype]'], ['OM samples [phenotype]'], ['number and relative abundance of transcripts [phenotype]']]}\n",
      "-------------------\n",
      "{'93624': [['haploid microspores [cell type]'], ['**microspore mother cells [cell type]**', 'microspore mother cells from undergoing meiosis [cell type]', 'microspore mother cells (MMC) [cell type]'], ['**microspores [organism]**', 'Microspore [organism]', 'microspores/pollen [organism]'], ['2n Microspores [gamete]'], ['Micropylar megaspore [structure]'], ['Microspores derived from imaged meiocytes [organ]'], ['Microspore nucleus [organ]'], ['Sister microspores [cell type]'], ['Developing microspores [cell type]'], ['pollen mitosis I [process]'], ['Haploid microspore [organ]'], ['Microspore [organ]'], ['Micropylar end of the megagametophyte [organ]'], ['microsporocyte [organ]'], ['microspore mother cells [organ]'], ['microspore stage [cell type]'], ['orderly microspore exine structure [phenotype]'], ['larch microspore nucleus [organ]'], ['unicellular microspore [organ]'], ['microgametophyte (pollen) [organ]'], ['Microsporocytes [tissue]'], ['wild-type microsporocytes [organism]'], ['**Microspores and pollen grains [cell type]**', 'microspores/pollen [cell type]'], ['unitegmic micropyles [organ]'], ['tomato primary microsporocytes [cell type]'], ['one to eight microspores [cell type]']]}\n",
      "-------------------\n",
      "{'122687': [['**putative orthologs [gene]**', 'potential orthologues [gene]', 'possible orthologues [gene]', 'putative ortholog gene(s) [gene]'], ['two putative orthologs [gene]']]}\n",
      "-------------------\n",
      "{'165162': [['**internalized cargo [metabolite]**', 'endocytosed material [metabolite]', 'Internalized cargoes [metabolite]'], ['**Internalization of endocytic tracer FM4-64 [metabolite]**', 'internalisation of an endocytic tracer dye [metabolite]'], ['endocytosis of transferrin [metabolite]'], ['cellular uptake of 14 chemicals [metabolite]']]}\n",
      "-------------------\n",
      "{'46224': [['1O2-stressed fc2 seedlings [mutant]'], ['lac12 mutant seedlings [mutant]'], ['pap1-D/fls1ko seedlings [mutant]'], ['drb2/Cd seedlings [mutant]'], ['drb1/Cd seedlings [mutant]'], ['tt4 seedlings [mutant]'], ['flg22 in DD seedlings pretreated with DEX [mutant]'], ['fructose in fins1 seedlings [mutant]'], ['fructose insensitivity of fins1 seedlings [mutant]'], ['pap1 seedlings [mutant]'], ['pap1-D seedlings [mutant]'], ['dfc seedlings [mutant]'], ['stressed fc2 seedlings [mutant]'], ['stressed fc2 mutant seedlings [mutant]'], ['flavonoid-deficient tt4 seedlings [mutant]'], ['tt4(2YY6) seedlings [mutant]'], ['vte1 seedlings [mutant]']]}\n",
      "-------------------\n",
      "{'140180': [['**heat response transcripts [gene]**', 'mRNAs of some heat-inducible gene(s) [gene]']]}\n",
      "-------------------\n",
      "{'28964': [['CMT orientation [cellular process]'], ['**CMT/CMF mechanism [cell structure]**', 'CMF/CMT-based mechanism [cell structure]'], ['CMT interactions [cell type]'], ['interactions between endomembrane systems and CMTs [subcellular compartment]'], ['main factors linking CMTs and cell wall [subcellular compartment]'], ['interacting factor between CSCs and CMTs [subcellular compartment]'], ['effect of cellulose microfibrils on MT array alignment at the plasma membrane [process]'], ['cellulose synthase machinery and cortical MTs [process]'], ['generating a parallel arrangement of CMTs [protein complex]'], ['mechanisms critical for generating transverse CMT arrays in WT [protein complex]'], ['**CMT-mediated orientation of cellulose microfibrils [subcellular compartment]**', 'CMT-guided cellulose synthesis [subcellular compartment]', 'cortical microtubule (CMT)-mediated orientation of cellulose microfibrils [subcellular compartment]'], ['encountering sharp cell edges [process]'], ['interaction of CSCs with microtubules [process]']]}\n",
      "-------------------\n",
      "{'74528': [['LORELEI [protein]'], ['LORELEI-like [protein]'], ['CLE41/44 [protein]'], ['JR1 protein(s) [protein]'], ['LOC112798274 14-3-3-like protein(s) A [protein]'], ['OSJ_11117 [protein]'], ['small, secreted cysteine-rich ligands [protein]'], ['twelve grapevine protein(s) [protein]']]}\n",
      "-------------------\n",
      "{'121083': [['Continuous editing [phenotype]'], ['gene(s) editing [phenotype]', '**Genome Editing [phenotype]**', 'Genome-editing [phenotype]'], ['anticipated base editing [phenotype]'], ['genome editing target gene(s) [phenotype]'], ['editing those gene(s) [phenotype]'], ['genome editing experiments [phenotype]'], ['**gene-editing events [phenotype]**', 'different editing events [phenotype]'], ['editing outcomes [phenotype]', '**genome editing outcomes [phenotype]**', 'genome edits [phenotype]'], ['gene(s) editing activities [phenotype]', '**gene-editing activity [phenotype]**'], ['more sophisticated genome engineering [phenotype]'], ['gene(s) edited progeny [phenotype]'], ['gene(s) editing in germline tissues [phenotype]'], ['**successful gene(s) editing [phenotype]**', 'effective genome editing [phenotype]'], ['genome editing function [phenotype]'], ['endogenous gene(s) editing [phenotype]'], ['effective gene(s) modification [phenotype]'], ['the field of genome editing [phenotype]'], ['precise gene-editing tool [phenotype]'], ['genotype-independent gene(s) editing [process]'], ['marker gene-independent genome engineering [phenotype]'], ['identifying delivery-mediated gene(s) editing [phenotype]'], ['genome-editing purposes [phenotype]'], ['genome editing research [phenotype]'], ['efficient genome engineering [phenotype]']]}\n",
      "-------------------\n",
      "{'59649': [['cell junction (GO:0030054) [cell type]'], ['the wall in fibre cells [cell type]'], ['defects in cortical microtubule organization in fiber cells [cell type]']]}\n",
      "-------------------\n",
      "{'81737': [['native ScHal3 [protein]'], ['native Hal3 [protein]'], ['Wildtype RACB [protein]'], ['wild-type AS2D protein(s) [gene]'], ['TMEM16A wild-type protein(s) [protein]'], ['wt coilin [protein]'], ['wild-type GUS protein(s) [protein]'], ['wild-type P3 protein(s) [gene]'], ['**wt HcPro [protein]**', 'Wild-type (wt) HcPro [protein]'], ['wild-type OsPP2C50 [protein]'], ['Native Sco protein(s) [protein]'], ['wild-type Ngb [protein]'], ['wild-type CrUb [protein]'], ['unmodified and modified forms of HTA9 [protein]'], ['original SEU protein(s) [protein]'], ['wild-type CHLM [protein]'], ['wt MP [protein]'], ['wild-type SR protein(s) [protein]']]}\n",
      "-------------------\n",
      "{'88515': [['split root(s) agar plate assays [treatment]'], ['s1 seeds [treatment]'], ['both systems [treatment]'], ['colonization assays [treatment]'], ['surface of a 45° inclined agar plate [treatment]'], ['surface of a horizontal impenetrable plate [treatment]'], ['split agar plates [environment]'], ['early germination tests on the ground [treatment]'], ['bottom of Petri dish [treatment]'], ['lines with smaller diameters [treatment]'], ['plane pl1 [treatment]'], ['plane pl2 [treatment]'], ['miniaturized agar and soil system [treatment]'], ['seedlings and assay buffer [treatment]'], ['beds of glass spheres [treatment]'], ['soil to compare root(s) architecture in Pi-limiting soil conditions with that observed on agar plates [treatment]'], ['agar plates containing NaCl [treatment]'], ['plated on agar plates containing test compounds [treatment]']]}\n",
      "-------------------\n",
      "{'167266': [['AS derivatives [metabolite]'], ['endogenous TAA [metabolite]'], ['feeding of L-sorbosone [metabolite]'], ['metabolites such as ascorbic acid [metabolite]', 'Plant-derived ascorbate [metabolite]', '**endogenous ascorbic acid [metabolite]**'], ['production of ascorbate [metabolite]', 'synthesis of Asc [metabolite]', '**synthesis of ascorbic acid [metabolite]**'], ['metabolites of ascorbate synthesis [metabolite]'], ['production of L-ascorbic acid [metabolite]'], ['production of dehydroascorbate [metabolite]'], ['AsA precursor [metabolite]'], ['minor sugars involved in ascorbate synthesis [metabolite]'], ['tuber ascorbate [metabolite]']]}\n",
      "-------------------\n",
      "{'42251': [['PDR11 [mutant]'], ['plant(s) depleted of PDAT1 [mutant]'], ['atdrp1a single mutants [mutant]'], ['atdrp2b single mutants [mutant]'], ['atdrp1e null mutant plant(s) [mutant]'], ['NRPD1 position mutated in A. thaliana nrpd1-47 [gene]']]}\n",
      "-------------------\n",
      "{'19963': [['enhanced resistance to BXW disease [phenotype]'], ['enhanced resistance to tomato bacterial speck [phenotype]'], ['enhanced disease resistance to bacterial leaf streak (BLS) [phenotype]'], ['enhanced disease resistance to bacterial blight (BB) [phenotype]'], ['increased late blight resistance [phenotype]', 'greater resistance to late blight [phenotype]', '**enhanced resistance to late blight [phenotype]**', 'increased resistance to late blight [phenotype]', 'reduced susceptibility to late blight [phenotype]', 'enhanced late blight resistance [phenotype]'], ['enhanced blast disease-resistance [phenotype]'], ['**enhanced bolting resistance [phenotype]**', 'improved bolting resistance [phenotype]'], ['enhanced clubroot resistance [phenotype]'], ['increased resistance to citrus canker [phenotype]', '**enhanced citrus canker resistance [phenotype]**'], ['enhanced resistance to seedling blight [phenotype]'], ['increased tolerance to blast disease [phenotype]'], ['increased bacterial wilt (BW) tolerance [phenotype]'], ['increased resistance to bacterial wilt [phenotype]', 'significant levels of enhanced resistance to bacterial wilt (BW) [phenotype]', '**enhanced resistance to bacterial wilt (BW) [phenotype]**']]}\n",
      "-------------------\n",
      "{'138189': [['Constitutive expression of Class I KNOX gene(s) [gene]'], ['constitutive high level expression of CLV3 [gene]'], ['constitutive expression of CRK6 [gene]'], ['constitutive expression of CRK10 [gene]'], ['constitutive expression of CRK11 [gene]'], ['constitutive activation of LjCLE gene(s) [gene]'], ['Constitutive expression of AtTSPO [gene]'], ['Constitutive expression of MpKAN [gene]'], ['Constitutive expression of KNOX2 [gene]'], ['constitutive expression of SlIPT3 [gene]'], ['Constitutive expression of CaPUB1 [gene]'], ['constitutive expression of CnTCP4 [gene]'], ['constitutive expression of CnTCP2 [gene]'], ['Constitutive expression of CKX1 [gene]'], ['Constitutive expression of NLRs [protein]'], ['Constitutive expression in leaf of SKS13 [gene]'], ['constitutive expression of CRF2 [gene]'], ['Steady state levels of C4 protein(s) [gene]'], ['CRL constitutive expression [phenotype]'], ['Constitutive expression of FCP1 [gene]'], ['constitutive expression of mcSKD1 [gene]'], ['Constitutive expression of CIR1 [gene]'], ['constitutive expression of AdCIPK5 [gene]'], ['the stability of Keep on Going (KEG) [gene]'], ['constitutive expression of RAP2.12 [gene]'], ['Rhythmical expression of CO [phenotype]'], ['CO diurnal expression [process]']]}\n",
      "-------------------\n",
      "{'73713': [['ROC1 [protein]'], ['ROS sensors [protein]'], ['aCO2 [protein]'], ['plant(s) RSHs [protein]'], ['OXS2 [protein]'], ['oxi1 [protein]'], ['PER1 protein(s) [protein]'], ['TOP1 and TOP2 [protein]'], ['osmotin-like (Ol) [protein]'], ['CPR [phenotype]'], ['SAPX [protein]'], ['ERFVII8 [protein]'], ['Ripening-induced protein(s) (ASR) [protein]'], ['**OSIP108 [protein]**', 'oxidative stress-induced bioreactive peptide, OSIP108 [protein]'], ['ROS [protein]'], ['OC-I-inhibited cysteine proteinases [protein]'], ['The PPR-Domain protein(s) SOAR1 [protein]'], ['A novel protein, OZONE-RESPONSIVE APOPLASTIC PROTEIN1 (OsORAP1) [protein]', '**OsORAP1 [protein]**'], ['plant(s) ZCr [protein]'], ['stromal-cell-derived factor 2-like protein(s) [protein]'], ['hydrogen peroxide-responsive protein(s) [protein]'], ['Enhanced stability of ARR10 protein(s) [gene]'], ['SIMILAR TO RCD-ONE (SROs) [protein]'], ['CaOSM1 protein(s) [protein]'], ['Stromal cell-derived factor2 (SDF2)-type protein(s) [protein]'], ['ROS-related protein(s) [protein]'], ['JAZ stability [protein]'], ['oxidation-related zinc finger one protein(s) [protein]'], ['ROS response protein(s) in root(s) [protein]'], ['ROS-associated protein(s) [protein]'], ['aging responsive protein(s) [phenotype]'], ['oxidant-sensitive protein(s) in plant(s) [organism]'], ['oxidant receptor protein(s) 1 (Orp1) [protein]'], ['bZIP60, both isoforms of IRE1, S2P, NAC062, J-domain protein(s) ERdj2a, and AERO1 (ER oxidoreductin 1) [protein]'], ['POX2 [protein]'], ['ROS-responsive protein(s) [protein]'], ['stability of RAP2.12 [protein]'], ['groups including pathogenesis-related proteins, metallothioneins, severe drought-stressed proteins, Hsp90, thaumatin-like protein, catalase (CAT1), WRKY family, and oxalate oxidase [phenotype]'], ['Cys-rich repeat secretory protein(s) 55 precursor [protein]'], ['ARR5 stability [phenotype]']]}\n",
      "-------------------\n",
      "{'126982': [[\"absence of 5' UTR [regulatory sequence]\"], ['lack of Pol IV [gene]', '**Loss of Pol IV [gene]**'], ['loss of DRB2 activity [gene]'], ['absence of natural TE insertions in RPI2 [gene]'], ['Lack of pol IV RNAs [gene]'], ['Loss of maternal Pol IV [gene]'], ['Loss of RNA Pol IV [gene]'], [\"Loss of one parent's Pol IV [gene]\"], ['**Loss of XRN4 [gene]**', 'loss of XRN4 activity [gene]'], ['absence of HEXO3-RNAi [gene]'], ['absence of MORF protein(s) [gene]'], ['absence of AtNUC-L1 protein(s) [gene]'], ['absence of DRB1 activity [gene]'], ['loss of RDR2 [gene]'], ['loss of ELP6 [gene]'], ['lack of DCL2 [gene]'], ['lack of DCL3 [gene]'], ['absence of Cas13 [gene]'], ['absence of RDRs [gene]'], ['deletion of yeast nuclear RNase P RNA [gene]'], ['lack of asymmetric distribution of PIN3 [gene]'], ['lack of phasing for NBS-LRR-derived siRNAs [rna]'], ['The absence of stem(s) loops #1 and #2 [subcellular compartment]'], ['absence of a functional MRE [subcellular compartment]'], ['loss of Pol V [protein]'], ['lack of Shine-Dalgarno sequence [genetic element]'], ['absence of functional Pol IV [treatment]'], ['absence of respective uORF [gene element]'], ['absence of RNA triggers [epigenotype]'], ['D-arm absent from tRNA-like structure [genetic feature]']]}\n",
      "-------------------\n",
      "{'121395': [['repeats [phenotype]'], ['a variable number of tandem repeats [phenotype]'], ['variant repeat tracts [phenotype]'], ['Highly divergent tandem repeats [phenotype]']]}\n",
      "-------------------\n",
      "{'138084': [['GL2 expression [phenotype]'], ['**IKU2 expression [gene]**', 'expression of IKU2 [gene]'], ['KAT2 expression [gene expression]'], ['CTPS2 expression [gene]'], ['**FC2 expression [gene]**', 'Expression of FC2 [gene]'], ['FC2 expression [gene expression]'], ['ASIL2 expression [gene]'], ['ETR2 expression [phenotype]'], ['Expression levels of EIN2 [gene]'], ['DLK2 gene(s) expression [gene]'], ['**ACT2 expression [gene]**', 'ACT2 gene(s) expression [gene]'], ['expression of JAZ2 [gene]'], ['Expression levels of GTG2 [gene]'], ['Expression levels of DREB2A [gene]'], ['Expression of VOZ2 [gene]'], ['GAS2 expression [gene]'], ['Expression of 2-ODD [gene]'], ['endogenous levels of TCMP-2 [gene]'], ['ZmAPR2 expression [gene]'], ['IREG2 expression [gene]'], ['TEM2 levels [gene]'], ['GL2 activity [phenotype]'], ['**GL2 expression [gene]**', 'expression levels of GL2 [gene]'], ['Expression of RTM2 [gene]'], ['TOE1/2 levels [gene]'], ['GSTF2 expression [gene]'], ['**GR2 protein(s) levels [gene expression]**', 'GR2 protein(s) level [gene expression]'], ['MtDCL2b expression [gene]'], ['TGG2 expression [gene expression]'], ['OPT3 expression [gene expression]'], ['Expression of MAX2 [gene]'], ['**GDEF2 expression [gene]**', 'Expression of GDEF2 [gene]'], ['Vlaro2 expression [gene]'], ['abundance of Tim23-2 [gene]'], ['comparable level of Thi2.1 expression [phenotype]'], ['NDB2 expression [gene]'], ['GSH2 expression [phenotype]'], ['expression of endogenous MAF2 gene(s) [gene]'], ['expression of its paralogues [gene]'], ['full ECT2 function [gene]'], ['TCU2 expression [gene expression]'], ['progression of TT2 expression [gene expression]'], ['endogenous LOG2 gene(s) activity [gene]'], ['LOG2 expression [gene]'], ['TaTIP2;2 expression [gene expression]'], ['accumulation of Thi2.4 protein(s) [gene]'], ['Ivr2 expression [gene]'], ['AtZIP2 expression [gene]'], ['TED2 expression [gene]'], ['ECT2-FLAG-ADAR expression levels [gene]'], ['Dof2.1 expression [gene]'], ['basal expression levels of KIN2 [gene]'], ['ADC2 expression [gene expression]'], ['expression of ACTIN2 [gene]']]}\n",
      "-------------------\n",
      "{'66209': [['FH1 protein(s) [protein complex]'], ['FH EXTs [protein]'], ['FH EXTs [gene]'], ['dis2-1S protein(s) [protein]'], ['Actin-associated formin family protein(s) [gene]'], ['FtsZ homologues [gene]'], ['Profilin families [protein family]'], ['metazoan formins [gene]'], ['Class II formins [gene]'], ['expression of profilin gene(s) (PRF4 and PRF5) [gene]'], ['ADF/cofilin (AC) family [protein family]'], ['flagellar basal-body rod modification protein(s) FlgD [protein]'], ['PRF1 and PRF2 protein(s) [protein]'], ['Class I formin FH1 [gene]'], ['actin-binding formin homology FH2 protein(s) [protein]'], ['numerous group 2 formins [gene]'], ['Cdc12p formin [gene]'], ['AtACT2-containing filaments [protein]'], ['general properties of the formin family [gene]']]}\n",
      "-------------------\n",
      "{'109497': [['single A [genetic element]'], [\"DUE's [genetic element]\"], ['mini3a [genetic element]'], ['helitron 11 [genetic element]'], ['A+C (Imino, M) [genetic element]'], ['all purine progenote [genetic element]'], ['all pyrimidine progenote [genetic element]'], ['C-C pair [genetic element]'], ['only one of eight pairs [genetic element]'], ['binary elements [component]']]}\n",
      "-------------------\n",
      "{'24': [['**pollen functions [phenotype]**', 'pollen grain function [phenotype]'], ['pollen feature [phenotype]'], ['pollen nutrition [phenotype]'], ['pollen phenotypes [phenotype]'], ['pollen physiology [phenotype]'], ['normal pollen function [phenotype]'], ['phosphor in seeds and pollen grains [phenotype]']]}\n",
      "-------------------\n",
      "{'51350': [['panicle characteristics similar to EMS-induced frizzy panicle mutant fzp [mutant]'], ['panicle characteristics similar to gamma-ray induced frizzy panicle mutant fzp2 [mutant]']]}\n",
      "-------------------\n",
      "{'44730': [['cesa5-2 fei2-3 double mutant [mutant]'], ['**cesa5 fei2 double mutant [mutant]**', 'fei2 cesa5 double mutant [mutant]'], ['hy5 csu2 double mutants [mutant]'], ['seuss cyp85A2 double mutants [mutant]'], ['seu-3 cyp85A2-1 double mutant [mutant]'], ['seu-3 cyp85A2-2 double mutant [mutant]'], ['seu-1 cyp85A2-4 double mutant [mutant]'], ['sco3-1phyB-9 [mutant]'], ['swn7clf28 double mutant [mutant]'], ['flu/ex1 double mutant [mutant]']]}\n",
      "-------------------\n",
      "{'125145': [['msh1 rearrangements [genomic feature]'], ['abrupt changes in coverage of the At7 genome [gene]'], ['increase in CG content [genetic feature]'], ['aCGH-detected mutation [mutation]'], ['one-rearrangement reads [phenotype]'], ['Chr4 alterations [genotype]'], ['Chr5 alterations [genotype]'], ['shift in base composition [genetic feature]'], ['sequence anomalies [gene]'], ['suggesting presence of genomic rearrangement [gene]'], ['identification of complex changes [process]'], ['changes in primary sequence [genomic feature]'], ['aCGH identified mutation [phenotype]'], ['indicate rearrangement [mutant]'], ['5-3 copy number change [phenotype]']]}\n",
      "-------------------\n",
      "{'83273': [['circular dichroism [phenotype]'], ['solvable crystal structure [phenotype]'], ['outcome of crystallization trial [phenotype]'], ['crystallization of the protein(s) [phenotype]'], ['quality of protein(s) crystals [phenotype]'], ['**resolution for computing atomic structures [phenotype]**', 'resolutions up to 1.45 A [phenotype]'], ['resolution of the gels [phenotype]']]}\n",
      "-------------------\n",
      "{'31266': [['ethylene-mediated cell expansion of the epidermis [phenotype]'], ['Epidermal growth factor [signal molecule]'], ['ER-regulated cell elongation in hypocotyl [phenotype]'], ['many growth and cell fate decisions [process]']]}\n",
      "-------------------\n",
      "{'149512': [['SEGS-1 sequences [gene]'], ['SEGS-1 related sequences [gene]'], ['full-length copy of SEGS-1 [gene]'], ['SEGS-2 episomes [gene]', '**SEGS-2 episome [gene]**'], ['SEGS-2 partial copies [gene]'], ['SEGS-2-related sequences [gene]'], ['Sequences Enhancing Geminivirus Symptoms (SEGS1 and SEGS2) [gene]'], ['SEGS-1 related sequences [genetic material]'], ['SEGS-1 sequence [genetic material]'], ['SEGS-1 and SEGS-2 [genetic material]']]}\n",
      "-------------------\n",
      "{'19384': [['Acclimation efficiency [phenotype]'], ['metabolic flexibility [phenotype]'], ['Aphid fitness [phenotype]'], ['fitness of plant(s) [organism]'], ['survival of DPF plant(s) [phenotype]'], ['intraspecific recognition [phenotype]'], ['interspecific recognition [phenotype]'], ['ability of plant(s) to forage their environment [phenotype]'], ['intrinsic metabolic fitness [phenotype]'], ['host-plant productivity [phenotype]'], ['**key factors for high acclimation capacity [phenotype]**', 'key determinant for optimal acclimation capacity [phenotype]'], ['environmental performance [phenotype]'], ['epiphytic fitness [phenotype]'], ['leaf adapted obligate biotroph [phenotype]'], ['**competitive ability of plant(s) [phenotype]**', 'competitive power of plant(s) [phenotype]'], ['host fitness [phenomenon]'], ['environmental fitness of plant(s) [process]']]}\n",
      "-------------------\n",
      "{'138656': [['expression levels of AtFRO2 [gene]', '**expression of AtFRO2 [gene]**'], ['AtFer1 expression [gene]'], ['ASMT expression [gene]'], ['AtFIT expression [gene]'], ['expression of At-NEET [gene]', '**At-NEET expression [gene]**'], ['expression of AtIRT2 [gene]'], ['**expression of IRON-REGULATED TRANSPORTER1 (IRT1) [gene]**', 'did not influence the expression of AtIRT1 [gene]', 'expression of Fe-uptake-associated gene(s) IRT1 [gene]'], ['IRT1 [phenotype]'], ['AtSHMT4 expression [gene expression]'], ['expression of AtNRAMP3 [gene]'], ['AtNRAMP3 expression [gene expression]']]}\n",
      "-------------------\n",
      "{'20901': [['5.8% of WT [organism]'], ['AF>0.2 in WT samples [phenotype]'], ['10% in WT (WT) [phenotype]'], ['50% of WT [phenotype]'], ['>10% WT [phenotype]'], ['50-60% of WT levels [phenotype]'], ['20% of the WT expression [phenotype]'], ['grain weight 51.5% and 55.2% of WT [phenotype]'], ['three to five times the nodules of WT [phenotype]'], ['50% sterol level of WT [phenotype]'], ['55% of the wild-type value [phenotype]']]}\n",
      "-------------------\n",
      "{'160258': [['Promoters of homologous FMI gene(s) [gene]', '**promoters of the floral meristem identity gene(s) [gene]**'], ['**promoters of floral meristem identity gene(s) SQUAMOSA [gene]**', 'floral meristem identity gene(s) SQUAMOSA promoter [gene]'], ['conserved motif in the promoter region of the floral meristem identity gene(s) SQUAMOSA [gene]']]}\n",
      "-------------------\n",
      "{'156790': [['GNOM-like 1 [gene]'], ['ANAC017 homologs [gene]'], ['NSP1 gene(s) [gene]'], ['plant(s) SNF1 [gene]'], ['plant(s) synaptotagmins (SYTs) [gene]'], ['plant(s) GSK3s [gene]'], ['CNI-1 gene(s) [gene]'], ['SORTING NEXIN2 (SNX2a) [gene]'], ['Sgn3 [protein]'], ['Functions of Acorus sequences [gene]'], ['plant(s) Rac-like GTPases [gene]'], ['CARD1 homologues [gene]'], ['**synaptotagmin 1 [gene]**', 'synaptotagmin 1 (SYT1) [gene]'], ['combinations of transporters, receptors, and kinases [gene]'], ['flg22-induced phosphorylation of PBL1 [phenotype]'], ['GNOM pathway [gene]'], ['CORYNE function [gene]'], ['AtNIGT1/HRS1 motif [gene]'], ['signal from SGN3 [gene]'], ['snc1 signaling [gene]'], ['signaling mediated by AGCVIII protein(s) kinase UCN [gene]'], ['NODULE INCEPTION (NIN)-LIKE protein(s) (NLP) gene(s) family [gene]']]}\n",
      "-------------------\n",
      "{'19225': [['CRR1-dependent Cu accumulation [phenotype]'], ['High concentrations of Zinc [trait]'], ['Zn accumulation capacity [phenotype]'], ['variation in Zn accumulation capacity [phenotype]'], ['highest Zn accumulation capacity [phenotype]'], ['altered Zn accumulation capacity [phenotype]'], ['**Zn accumulation [phenotype]**', 'accumulation of Zn [phenotype]', 'Zn accumulation phenotype [phenotype]', 'plant(s) zinc accumulation [phenotype]', 'zinc (Zn) accumulation [phenotype]'], ['**high Zn accumulation [phenotype]**', 'higher accumulation of Zn [phenotype]', 'relatively high Zn accumulation [phenotype]', 'Increased Zn accumulation [phenotype]', 'higher Zn accumulation [phenotype]'], ['increased Zn concentrations [phenotype]'], ['**shoot Zn accumulation [phenotype]**', 'accumulation of Zn in shoots [phenotype]', 'enhanced shoot accumulation of Zn [phenotype]'], ['increased Zn accumulation in rosette leaf and root(s) [phenotype]'], ['increased Zn accumulation in plant(s) tissues [phenotype]'], ['high Zn accumulation ability in leaf [phenotype]'], ['significant increase in plant(s) Zn accumulation [phenotype]'], ['proportional increases in Zn accumulation with increasing Zn concentrations [phenotype]'], ['strong enrichment effect on Zn [phenotype]'], ['zinc accumulation levels [phenotype]'], ['Zn tolerance and accumulation [phenotype]'], ['**storage of excess Zn [phenotype]**', 'significant impact on Zn storage [phenotype]', 'zinc deposition [phenotype]'], ['**Zn/Fe storage [phenotype]**', 'Fe/Zn storage [phenotype]'], ['higher Zn, Cd, and Ni accumulation [phenotype]'], ['Zn and Cd accumulation [phenotype]'], ['quantitative trait loci (QTL) for Zn accumulation [genetic locu]'], ['zinc accumulation quantitative trait locus [phenotype]'], ['**Zn sequestration [phenotype]**', 'vacuolar immobilization of Zn [phenotype]', 'sequestration of Zn [phenotype]', 'enhanced Zn compartmentalization in the vacuole [phenotype]', 'ability to accumulate Zn in the vacuole [phenotype]']]}\n",
      "-------------------\n",
      "{'155284': [['U2AF65A transcripts [gene]'], ['UFO RNA [gene]']]}\n",
      "-------------------\n",
      "{'97357': [['**outer layers of seed coat [organ]**', 'shell of plant(s) seed [organ]', 'outermost layer of the seed [organ]', 'outer layer of seed coat [organ]', 'surface of seeds [organ]', 'covering structures [organ]', 'stiff envelope [organ]', 'outermost layers of seeds [organ]', 'extremely reduced seed epidermis [organ]', 'outer layers [organ]', 'outer seed coat wall [organ]', 'testa of seeds [organ]', 'hard, impermeable seed coat [organ]', 'outer seed coat layers [organ]', 'outer layer of the seed coat [organ]', 'entire seed surface [organ]', 'structure of seed surface [organ]', 'seed surface structure [organ]', 'outer wall of the seed coat [organ]'], ['outer integument of seeds [organ]', 'outer seed integument [organ]', 'outer integuments of the seed coat [organ]', '**outer integument of the seed coat [organ]**', 'outer integument of seedcoat [organ]', 'external integument [organ]'], ['**inner integument of seed coat [organ]**', 'inner integument of the black seed coat [organ]', 'internal integument [organ]'], ['shorter outer integument [organ]'], ['longer outer integument [organ]'], ['longer inner integument [organ]'], ['kernel aleurone [organ]'], ['seed cutin [organ]'], ['seed integument [organ]'], ['inner part of seed coat [organism]']]}\n",
      "-------------------\n",
      "{'34328': [['**Xenopus oocytes [cell type]**', 'Xenopus laevis oocytes [cell type]', 'X. laevis oocytes [cell type]', 'Xenopus laevis (African clawed frog) oocytes [cell type]', 'Xenopus oocyte cells [cell type]'], ['frog spermatozoa [cell type]'], ['water-injected control oocytes [cell type]'], ['EcHKT-expressing oocytes [cell type]'], ['HKT1 expressing oocytes [cell type]'], ['Oocytes [metabolite]'], ['AIT3/NPF4.1-expressing oocytes [cell type]'], ['NPF2.5 injected oocytes [cell type]'], ['detectable egg cell [cell type]'], ['Oocyte [organism]'], ['Xenopus leaf oocytes [cell type]'], ['isolated egg cells [cell type]']]}\n",
      "-------------------\n",
      "{'67751': [['PERKs [protein clas]'], ['**PERKs [protein]**', 'Proline rich extensin like receptor kinases (PERKs) [protein]', 'proline-rich, extensin-like receptor kinases (PERKs) [protein]'], ['PROLINE-RICH RECEPTOR-LIKE protein(s) KINASE 4 (PERK4) [gene]'], ['proline rich extension-like receptor kinases (PERKs) [gene]'], ['ERK element [phenotype]'], ['PERK protein(s) family [protein family]'], ['pERK species [protein]'], ['proline-rich extensin-like receptor kinase (PERK) family [gene family]']]}\n",
      "-------------------\n",
      "{'69128': [['Residues 35 and 36 on the central alpha-helix [protein motif]'], ['Residues 91 and 111 on the beta-sheet [protein motif]'], ['His-318, Thr-323, and Arg-566 [amino acid]'], ['catalytic triad of MtAIH [protein structure]'], ['Arg116, Ser166, and Ser167 [protein fragment]'], ['Gly376 and Ser243 [tissue]'], ['Phe148/Leu135 [amino acid]'], ['Binding of Cs<sup>+</sup> to 749th residue of AtBE1 [gene]'], ['residues His-216 and Tyr-212 [protein complex]'], ['Modifications of residues AtcTPI-Cys218 [gene]'], ['residues Phe166, Tyr169, Phe170, His276, and Phe277 [amino acid]'], ['AtCXE20 catalytic triad [protein complex]'], ['residues F366, N415, Y418 [amino acid]'], ['51 residues [protein structure]'], ['D351 and E577 [protein]'], ['apex of nucleophile elbow [organ]'], ['catalytic triad of soluble Type II thioesterases [protein]'], ['R340, D336, and D503 [amino acid residue]'], ['catalytic triad residues [gene]']]}\n",
      "-------------------\n",
      "{'14381': [['**Shortened anther filament [phenotype]**', 'filament elongation defect [phenotype]', 'short filament [phenotype]', 'shorter filament length in anthers [phenotype]'], ['elongated weak hypocotyls [phenotype]'], ['severe cracking in flowering stem(s) [phenotype]'], ['lodging stem(s) [phenotype]'], ['SCW defects in inflorescence fiber cells [phenotype]'], ['**shortened and downward-pointing pedicels [phenotype]**', 'occasional downward bending at the distal pedicel [phenotype]'], ['**fragile or brittle apex phenotype [phenotype]**', 'investigating brittle apex phenotype [phenotype]'], ['tip rupturing of moss protonemal filaments [phenotype]'], ['dwarfed, largely sterile, premature leaf senescence, and novel filamentous structures at the base of the flower [phenotype]'], ['lack of sturdiness in inflorescence stem(s) [phenotype]', '**limp floral stem(s) [phenotype]**'], ['extremely short suspensors [phenotype]'], ['**weak stem(s) [phenotype]**', 'loss of stem(s) rigidity [phenotype]', 'stem(s) weakness [phenotype]'], ['cracks in the stem(s) [phenotype]'], ['brittle stem(s) [phenotype]'], ['brittle stem(s) nodes [phenotype]'], ['collapsed interfascicular fibers [phenotype]'], ['short and occasionally fasciated stem(s) [phenotype]'], ['fragile culms [phenotype]']]}\n",
      "-------------------\n",
      "{'149825': [['A9 [protein]'], ['WT HAP1 gene(s) [gene]'], ['AtVIT1 gene(s) [gene]'], ['NPGR2 [protein]'], ['At5g11420-encoded protein(s) [protein]'], ['HaSSA protein(s) [protein]'], ['HAIKU1 [protein]'], ['TT8 protein(s) [gene]'], ['NtAn1b protein(s) [protein]'], ['Y-type TUA6 [gene]'], ['Y-type TUA8 [gene]'], ['Maternal allele of TTG1 [gene]'], ['At1g10310 transcript [gene]'], ['TaAFP-B protein(s) [phenotype]'], ['Triticum aestivum RGP (TaRGP) [protein]'], ['T1170 [transcript]'], ['AtEm1 mRNAs [gene]'], ['timothy grass profilin [gene]']]}\n",
      "-------------------\n",
      "{'151611': [['OsWAK [gene]'], ['OsWAK14 [gene]'], ['OsWAK91 [gene]'], ['OsWAK1 [gene]'], ['**OsWAK90 [gene]**', 'OsWAK90 gene(s) [gene]'], ['OsWAK112d gene(s) [gene]'], ['OsWAF1 [gene]'], ['OsWAK/Xa4 [gene]'], ['OsWAK112 [gene]'], ['OsWAK14, OsWAK91, OsWAK92, and OsWAK112 [gene]'], ['125 OsWAK gene(s) family members [gene]'], ['OsWAK gene(s) family [gene]'], ['OsiWAK1 [gene]'], ['OsWNK [gene family]'], ['OsWAK91 in rice [gene]'], ['rice WAKs [gene]'], ['OsWAK112 activity [phenotype]'], ['OsWAK gene(s) family expansion [gene]'], ['OsWAK gene(s) in indica and japonica subspecies [gene]']]}\n",
      "-------------------\n",
      "{'10291': [['Extensive cell expansion [phenotype]'], ['**enhanced post-mitotic cell expansion [phenotype]**', 'induction of enhanced post-mitotic cell expansion [phenotype]'], ['minor cell expansion [phenotype]'], ['progressive enlargement of stem(s) cell reservoirs [phenotype]'], ['Cortical cell expansion [phenotype]'], ['**post-mitotic cell expansion [phenotype]**', 'phase of post-mitotic expansion [phenotype]', 'post-mitotic cell expansion activity [phenotype]', 'postmitotic cell expansion [phenotype]'], ['excess mitotic expansion [phenotype]'], ['mitotic expansion [phenotype]'], ['**onset of cell expansion [phenotype]**', 'timing of cell expansion onset [phenotype]', 'onset of expansion [phenotype]', 'onset of the cell expansion phase [phenotype]'], ['**post-mitotic elongation [phenotype]**', 'postmitotic cell elongation [phenotype]'], ['nucleus expansion [phenotype]'], ['cell-expansion kinetics [phenotype]'], ['**Cellular expansion [phenotype]**', 'cell expansion itself [phenotype]'], ['nuclear and cellular expansion [phenotype]'], ['cell proliferation and/or cell expansion [phenotype]'], ['**expansion stage [phenotype]**', 'expansive stage [phenotype]'], ['involvement in cell expansion [phenotype]'], ['continued cell growth [phenotype]'], ['**extended cell expansion period [phenotype]**', 'periods of cell expansion [phenotype]', 'increased period of cell expansion [phenotype]'], ['earlier onset of cell expansion [phenotype]'], ['**endosperm cellular expansion [phenotype]**', 'endospermic cellular expansion [phenotype]'], ['intense and long period of cellular growth [phenotype]'], ['transition to cell expansion [phenotype]'], ['redundant roles in cell expansion [phenotype]'], ['mitotic cell plate expansion [phenotype]'], ['expansion ends [phenotype]'], ['expansion phases [phenotype]'], ['extensive post-mitotic cell growth [phenotype]'], ['increase in the rate of cell expansion [phenotype]'], ['phase of cell expansion and storage reserve accumulation [phenotype]']]}\n",
      "-------------------\n",
      "{'33542': [['**Chromatin remodeling [subcellular compartment]**', 'regulated structural transitions of chromatin [subcellular compartment]'], ['**sustained changes to chromatin structure [subcellular compartment]**', 'persistent chromatin changes [subcellular compartment]'], ['looser structure [subcellular compartment]', '**decondensation of the chromatin [subcellular compartment]**', 'local and temporal opening of chromatin [subcellular compartment]', 'formation of open chromatin structure [subcellular compartment]'], ['**stress-induced chromatin changes [subcellular compartment]**', 'stress-associated reorganized chromatin states [subcellular compartment]'], ['chromatin stability [subcellular compartment]'], ['SWI/SNF-dependent accessibility [subcellular compartment]'], ['reprogramming of accessible chromatin [subcellular compartment]'], ['minimal changes in chromatin accessibility [subcellular compartment]'], ['loss of chromatin accessibility [subcellular compartment]'], ['restriction of chromatin accessibility [subcellular compartment]'], ['stable association of cohesin with chromatin [subcellular compartment]'], ['problem of chromatin status [subcellular compartment]'], ['conserved and ubiquitous structural components of eukaryotic chromatin [subcellular compartment]'], ['structural properties of chromatin [subcellular compartment]'], ['analysing chromatin dynamics [subcellular compartment]'], ['changing allocation of chromatin states [subcellular compartment]'], ['chromatin compaction dynamics [subcellular compartment]'], ['dynamic changes of chromatin structure [subcellular compartment]'], ['sequence determinants of heterochromatin condensation [subcellular compartment]'], ['controlling chromatin compaction [subcellular compartment]'], ['context of chromatin structures [subcellular compartment]'], ['**highly condensed chromatin [subcellular compartment]**', 'highest level of chromatin compaction [subcellular compartment]'], ['access to diverse chromatin states [subcellular compartment]'], ['high chromatin stability [subcellular compartment]'], ['chromatin destabilization [subcellular compartment]']]}\n",
      "-------------------\n",
      "{'77861': [['SBP-box DNA-binding domain [protein domain]', '**SBP-domain [protein domain]**', 'SBP motif [protein domain]', 'plant-specific DNA-binding domain named SBP [protein domain]'], ['SBP domains [protein]'], ['protein(s) sequences of the DNA-binding domain of SBP-box gene(s) [gene]']]}\n",
      "-------------------\n",
      "{'54598': [['35S::SEC-overexpressing transgenic plant(s) [mutant]'], ['35S:SEPs transgenic plant(s) [mutant]']]}\n",
      "-------------------\n",
      "{'122940': [['**fast evolution of lineage-specific gene(s) [phenotype]**', 'acceleration of LS gene(s) evolution [phenotype]'], ['emergence of proto-genes [phenotype]'], ['faster evolution in young primate gene(s) [phenomenon]'], ['faster than their paralogs [process]'], ['de novo gene(s) evolution [phenomenon]'], ['maternal-offspring co-adaptation theory [phenomenon]'], ['intron gains [phenotype]']]}\n",
      "-------------------\n",
      "{'140773': [['promoters of ABCB11 [gene]'], ['promoters of ABCB14 [gene]'], ['promoters of ABCB15 [gene]'], ['promoters of ABCG33 [gene]'], ['ABC transporter gene(s) promoters [gene]'], [\"epidermal ABC transporters' promoters [gene]\"], ['**promoters of ABA-responsive gene(s) [gene]**', 'promoter of ABA-responsive gene(s) [gene]', 'promoters of ABA responsive gene(s) [gene]', 'promoter regions of ABA-responsive gene(s) [gene]'], ['promoters of AtBAG gene(s) [gene]'], ['Promoter regions of GI/ABA-regulated transcripts [genomic region]'], ['promoter of Ni-inducible gene(s) (AHB1) [gene]'], ['four alternative alcA promoter-driven reporter gene(s) [gene]'], ['promoters of AtABA1, AtABA3, AtNCED3, AtAAO3, and AtCYP707A3 [gene]'], ['promoters of ABA-inducible gene(s) [gene]'], ['promoters of ABA-repressed gene(s) [gene]'], ['promoters of ABA-induced gene(s) [gene]'], ['promoters of ABA-related gene(s) [gene]'], ['promoters of ABA-dependent gene(s) [gene]'], ['promoter regions of gene(s) critical for ABA biosynthesis [process]'], ['promoter regions of AHG3 [gene]'], ['promoter regions of HAB1 [gene]'], ['promoter regions of HAB2 [gene]'], ['promoter regions of ABA-related group-A PP2C-encoding gene(s) [gene]'], ['promoters of downstream gene(s) of the AREB family [promoter]'], ['promoters of ABA response gene(s) [gene]'], ['promoters of gene(s) involved in ABA signalling [phenotype]'], ['promoters of ABA-regulated gene(s) [gene]'], ['promoter of gene(s) encoding ABA anabolic enzymes [gene]'], ['promoters of ABA biosynthetic gene(s) [gene]'], ['regulatory region of ABA-induced gene(s) [gene]'], ['promoters of ABA-responsive ABF/AREB TF gene(s) [gene]'], ['promoter analyses of ABA-regulated gene(s) [gene]'], ['promoters of A-ARR gene(s) [gene]'], ['promoter regions of ABA signaling factors [dna sequence]'], ['ABA3 promoters [gene]'], ['promoters of CYP707A3 [gene]'], ['promoters of AAO3 [gene]']]}\n",
      "-------------------\n",
      "{'155550': [['HAG subfamily [gene]'], ['multiple HAC and HAG gene(s) [gene]'], ['six HATs [gene]'], ['32 HATs [gene]'], ['HAG1-HAG3 [gene]'], ['4 HAGs [gene]'], ['HPA2-like HAGs [gene]']]}\n",
      "-------------------\n",
      "{'98394': [['Stipa purpurea (S. purpurea) [organism]'], ['Rumex [organism]'], ['**Rorippa sylvestris [organism]**', 'Ro. sylvestris [organism]'], ['Rhytidium rugosum [organism]'], ['Vitex negundo [organism]'], ['Alhagi sparsifolia [organism]'], ['Alhagi sparsifolia Shap. [organism]'], ['**Grimmia pilifera [organism]**', 'G. pilifera [organism]'], ['R P. fugax plant(s) [organism]'], ['Rumex palustris [organism]'], ['Bittersweet [organism]'], ['S. pennellii (wild species, LA0716) [organism]'], ['**Salix psammophila [organism]**', 'S. psammophila [organism]'], ['Salix dasyclados x S. viminalis hybrid [organism]'], ['Rumex species [organism]'], ['Waterlogging-tolerant M. arvensis [organism]'], ['Ceriops tagal [organism]'], ['Salix variegata [organism]'], ['R. soongorica [organism]'], ['Hippophae rhamnoides [organism]'], ['Fertile Salvia miltiorrhiza [organism]'], ['R. stricta [organism]'], ['Ramando myconi [organism]'], ['Reaumuria trigyna [organism]'], ['R. palustris plant(s) [organism]'], ['Ipomoea campanulata [organism]'], ['Jacquemontia pentantha [organism]'], ['Rorippa × anceps [organism]'], ['Several species of the genus Selaginella [organism]'], ['H. ammodendron [organism]'], ['Sesbania herbacea [organism]'], ['Eclipta prostrata [organism]'], ['Rorippa species [organism]'], ['H. rhamnoides ssp. mongolica [organism]'], ['damp rock [organ]'], ['Eremosparton songoricum [organism]'], ['Rumex spp. species [organism]'], ['**Zygophyllum simplex [organism]**', 'Z. simplex [organism]'], ['Carragana microphylla [organism]'], [\"R. stricta's adaptation [organism]\"], ['S. rexii [organism]']]}\n",
      "-------------------\n",
      "{'76415': [['methyltransferase (MET) domain [protein domain]'], ['metabolite sensing domain [protein function]']]}\n",
      "-------------------\n",
      "{'10829': [['direct antagonism [phenotype]'], ['direct inhibitor [phenotype]']]}\n",
      "-------------------\n",
      "{'159017': [['CER gene(s) [gene]'], ['cuticle biosynthesis-related gene(s) [gene]', 'cuticle formation gene(s) [gene]', 'cuticular biosynthetic gene(s) [gene]', '**cuticle biosynthetic gene(s) [gene]**'], ['putative cuticle synthesis gene(s) [gene]'], ['waxy cuticle synthesis gene(s) [gene]'], ['leaf stratum corneum biosynthesis gene(s) [gene]'], ['Cuticle-related gene(s) [gene]'], ['cuticle-associated gene(s) families [gene]'], ['seed coat mucilage biosynthesis gene(s) [gene]'], ['SHINE [gene]'], ['CER2 gene(s) and glossy2 gene(s) [gene]'], ['common groups of gene(s) [gene]']]}\n",
      "-------------------\n",
      "{'24079': [['**GA homeostasis [phenotype]**', 'GA homeostasis regulation [phenotype]'], ['GA negative feed-back regulation [phenotype]', '**GA feedback regulation [phenotype]**', 'negative feedback regulation of gibberellin (GA) [phenotype]'], ['feed-back regulation [phenotype]'], ['feedback inhibition mechanism [phenotype]']]}\n",
      "-------------------\n",
      "{'84957': [['**plant(s) treated with cold treatment [treatment]**', 'growth under low temperatures [treatment]', 'plant(s) growing at decreased temperature [treatment]', 'Exposing temperate plant(s) to low temperatures [treatment]', 'transfer to cold conditions [treatment]', 'plant(s) are exposed to low-temperature stress [treatment]', 'cold treated [treatment]', 'plant(s) exposed to cold [treatment]', 'in plant(s) grown at low temperatures [treatment]', 'cold-stressed plant(s) [treatment]', 'plant(s) are transferred to lower temperature 4 hours after dawn [treatment]', 'plant(s) grown at lower temperatures [treatment]', 'plant(s) at low temperature [treatment]', 'plant(s) exposed to low temperature [treatment]', 'plant(s) exposed to low temperatures [treatment]', 'plant(s) subjected to cold [treatment]', 'plant(s) grown in the cold [treatment]'], ['Cold storage [treatment]'], ['C-LL stressed plant(s) [treatment]'], ['Cold-treated tomatoes [treatment]'], ['cold-treated seedlings [treatment]'], ['aerial parts under cold treatment [treatment]'], ['root(s) under cold treatment [treatment]'], ['root(s) and aerial tissues under cold treatment [treatment]'], ['tobacco under cold [treatment]'], ['cold treated pea plant(s) [treatment]'], ['cold-treated leaf [tissue]'], ['cold-treated adult leaf [tissue]'], ['cold-treated wheat aerial tissues [tissue]'], ['**Cold treated leaf [organ]**', 'leaf under low temperature treatment [organ]'], ['**low-temperature-treated plant(s) [organism]**', 'low temperature treated plant(s) [organism]'], ['plant(s) subjected to low temperatures [condition]']]}\n",
      "-------------------\n",
      "{'81337': [['circular dichroism spectroscopy [tool]'], ['CD spectroscopy [treatment]'], ['far-UV CD spectroscopy [treatment]'], ['circular dichroism [treatment]'], ['**circular dichroism (CD) spectrum [treatment]**', 'Circular Dichroism spectrum [treatment]'], ['circular dichroism and infrared spectroscopy [treatment]'], ['**CD spectroscopy [technique]**', 'CD-spectroscopy [technique]'], ['circular dichroism spectrum analysis [technique]'], ['FT-IR and CD analysis [technique]'], ['far UV-circular dichroism spectra [technique]'], ['circular dichroism (CD) spectrometry [technique]'], ['far UV-CD spectroscopy [technique]'], ['CD spectroscopy [experiment]'], ['circular dichroism spectroscopy [method]'], ['circular dichroism [method]'], ['CD spectroscopic and thermal stability analyses [method]']]}\n",
      "-------------------\n",
      "{'29912': [['after cell division [process]'], ['third division [process]']]}\n",
      "-------------------\n",
      "{'147673': [['Poplar RING-H2 Zinc Finger [gene]'], ['AtAIRP2 (RING-HC type) [gene]'], ['CaDSR1 [gene]'], ['ABA-Insensitive RING protein(s) 1 gene(s) (CaAIR1) [gene]']]}\n",
      "-------------------\n",
      "{'161970': [['Rboh-derived O2- [metabolite]'], ['**O2- [molecule]**', 'Superoxide radicals (O2-) [molecule]'], ['**superoxide ions [metabolite]**', 'Superoxide anion free radical [metabolite]', 'Superoxide anions (O2 -) [metabolite]', 'O-2 [metabolite]', 'superoxo anion radical [metabolite]', 'O2- free radicals [metabolite]', 'O2- ions [metabolite]', 'O.2- [metabolite]', 'O2*- [metabolite]', 'superoxide radicals (O2(-)) [metabolite]'], ['1 O2 [metabolite]'], ['KO(2) [metabolite]'], ['cellular O2- [metabolite]'], ['superoxide radical content [metabolite]'], ['cytosolic O2 - [metabolite]'], ['mitochondrial oxygen superoxides [metabolite]'], ['antisuperoxide anion [metabolite]'], ['oxygen free radical (O2(-)) production rate [metabolite]']]}\n",
      "-------------------\n",
      "{'81157': [['proteasome-dependent degradation of DREB2A [gene]', 'degradation of DREB2A [gene]', '**DREB2A degradation [gene]**'], ['proteasome-dependent degradation of CER3 [process]'], ['cleavage of BAK1 [protein]'], [\"SLY1-mediated A vs A' differential proteasome-dependent DELLA destruction [protein]\"], ['Processive degradation of D1 [process]'], ['proteolytic cleavage of PpRALF1 [gene]'], ['Deg1-mediated degradation [process]'], ['ARR2 proteolysis [mechanism]'], ['VPS23A-mediated degradation of ABA receptor [protein]'], ['proteasomal degradation by BRUTUS (BTS) [gene]'], ['ABA-mediated protein(s) degradation [process]'], ['proteasome-dependent ABI5 degradation [protein]'], ['proteasome-dependent degradation of MIN7 [gene]'], ['DUBm proteolytic degradation [process]'], ['proteolytic degradation of GAPC [process]'], ['proteasomal degradation of CRY2 [gene]'], ['DREB2 proteolysis [process]'], ['DREB2 post-translational deactivation [process]'], ['DRM2 degradation [protein]'], ['**proteasome-dependent degradation of CER3 [gene]**', 'proteasome-dependent degradation of ECERIFERUM3 (CER3) [gene]'], ['SH3P2 degradation [process]'], ['proteasome-mediated degradation of RGL2 [process]'], ['CRL4-dependent degradation of DDB2 [process]'], ['degradation machinery for Rep-MYBs [protein]'], ['**Rep-MYB degradation [process]**', 'degradation of Rep-MYBs by proteasomes [process]'], ['proteasomal degradation of GI [gene]'], ['autolytic degradation of XCP2 [protein]'], ['proteasome-independent degradation of BZR1 [gene]'], ['autolytically cleaved from the full length DEK1 protein(s) [protein]'], ['degradation of RAP2.3 [process]'], ['proteasomal degradation of VRN2 [protein]'], ['ubiquitin-proteasome-dependent degradation of LMO2 [process]'], ['**proteolysis of DREB2A [process]**', 'degradation of DREB2A [process]'], ['synthetic degradation assay in yeast [process]'], ['cleavage of DA1 substrates [protein]'], ['proteasome degradation of JAI3 [protein]'], ['proteasome-mediated destruction of RGL2 [gene]'], ['proteasome degradation of RCAR3 [protein]'], ['DRIP degradation [process]'], ['Degradation of MUSE13 [protein]'], ['proteasomal degradation of ABI5 [process]'], ['proteolytic degradation of PsbP [process]'], ['proteolytic degradation of PsbQ [process]']]}\n",
      "-------------------\n",
      "{'114243': [['overexpression analyses [process]'], ['blueberry expression analyses [process]', 'Wider analyses of floral gene(s) expression [process]', 'Cross-species expression analyses [process]', 'expression in rice [process]', 'investigate gene(s) expression in many plant(s) [process]', '**analysis of plant(s) gene(s) expression [process]**', 'sporophytic expression profiling [process]'], ['positional effects of T-DNA insertion [process]'], ['pea study [process]'], ['Gene-expression analysis in transgenic plant(s) [mutant]'], ['**Studies of gene(s) expression in plant(s) [organism]**', 'expression analysis in roses [organism]'], ['stoichiometric expression balance [process]'], ['transcription analysis of cotton gene(s) [phenotype]'], ['plant(s) AS events [process]']]}\n",
      "-------------------\n",
      "{'118804': [['Fractions [treatment]'], ['sampling time [environmental factor]'], ['oversampling [measurement]'], ['**sampling time points [treatment]**', 'sampling times [treatment]'], ['each time point [technique]', '**each sampling time point [technique]**'], ['larger samples [parameter]'], ['sampling patterns [treatment]'], ['uniformity in sequencing output per sample [method]'], ['sampling time [time]'], ['Correlation between repeated samples [treatment]'], ['sample sets [treatment]'], ['meticulous representative sample selection [method]'], ['Sample size [treatment]'], ['sample stages [treatment]'], ['pooled sampling [treatment]', '**sample pooling [treatment]**'], ['sample fractions [method]'], ['bootstrap samples [treatment]'], ['sampling time point [experiment]'], ['Sample size [description]'], ['sample sizes [variable]'], ['varying sample sizes [treatment]'], ['sample type [variable]'], ['a single reconstruction of the sample [treatment]'], ['multiple time points [treatment]'], ['two time points [technique]'], ['sample weights from data [treatment]'], ['a subset of the samples [treatment]'], ['serial sampling approach [treatment]'], ['different times [process]'], ['pool-split experiment [technique]'], ['random subsamples of the data [treatment]'], ['at least one sample [sample]'], ['increasing the number of samples [method]'], ['multiple pairs of time points [treatment]'], ['sample complexity [treatment]'], ['consecutive time points [treatment]'], ['actual sampling time [time]'], ['sample stability [protocol]'], ['various times [treatment]']]}\n",
      "-------------------\n",
      "{'13676': [['growth side [phenotype]'], ['growth points [phenotype]'], ['growth space [phenotype]'], ['growing part [phenotype]']]}\n",
      "-------------------\n",
      "{'43068': [['The absence of PsbS [protein]'], ['loss of atToc33 [protein]'], ['absence of Hsa32 [protein]'], ['absence of HsfA2 [gene]']]}\n",
      "-------------------\n",
      "{'29913': [['somatic doubling [process]'], ['further mitotic cell divisions [process]', 'several rounds of mitosis [process]', 'one more round of mitosis [process]', '**additional mitoses [process]**', 'repeated mitosis [process]', 'progression of subsequent mitoses [process]', 'successive rounds of mitosis [process]'], ['**Abnormal mitotic cell cycle [process]**', 'disorder of mitosis [process]'], ['numerous cell divisions [cellular process]'], ['a series of cell divisions [cell type]'], ['progression through mitosis [process]'], ['later stages of plant(s) cell mitosis [process]', '**later stages of mitosis [process]**'], ['next mitosis [process]'], ['mitotic stage of endosperm development [developmental stage]'], ['cellularized mitotic proliferation [process]'], ['reduction in mitotic cell area [cell type]'], ['a cell that continues undergoing several mitotic cycles [cell type]'], ['to follow a subpopulation of cells through successive mitoses [cell type]'], ['mitotic tissues [tissue]', '**mitotic tissue [tissue]**'], ['mitosis after return from cold to warm temperatures [process]']]}\n",
      "-------------------\n",
      "{'154151': [['DAR1 [gene]'], ['DAR2 [gene]'], ['TCTP in mammals [gene]'], ['EXORDIUM gene(s) [gene]'], ['TRANSTHYRETIN-LIKE [gene]'], ['TCP4 protein(s) [gene]'], ['TCP15-likes [gene]'], ['tcp5-likes [gene]'], ['Cotton homologs of RAPTOR [gene]'], ['GT1-like gene(s) [gene]'], ['Mob1/Mats [gene]']]}\n",
      "-------------------\n",
      "{'46762': [['SlAMS-silenced plant(s) [mutant]'], ['GE#3-10 [mutant]'], ['GE#2 sugi trees [mutant]'], ['Regenerated sugi trees with biallelic CjACOS5 deletions [mutant]', '**Biallelic CjACOS5-mutated sugi trees [mutant]**'], ['CjACOS5-mutated sugi trees [mutant]'], ['CjTKPR1-WM-introduced plant(s) [mutant]'], ['Somatic variant of cv. Mourvèdre named carpel-less (CLS) [mutant]'], ['SGC [mutant]']]}\n",
      "-------------------\n",
      "{'90570': [['H3K9ac [treatment]'], ['**H3K9K14ac [treatment]**', 'H3K9K14 acetylation [treatment]'], ['deacetylation of H3K9ac [treatment]'], ['**trimethylating lysine-27 on histone 3 (H3K27me3) [treatment]**', 'trimethylation of histone 3 lysine 27 (H3K27) [treatment]'], ['H4K5ac [treatment]'], ['H4 acetylation [treatment]'], ['Acetylation of H3K56 [treatment]'], ['H4K20me2 [treatment]'], ['HAT activity specific to H4K5 [treatment]']]}\n",
      "-------------------\n",
      "{'108873': [['second generation progeny of methylated grandparents [organism]', '**transgenerationally methylated progeny [organism]**'], ['Atypical members of de novo DNA methylation pathway [phenotype]'], ['de novo epiallelism [phenotype]'], ['subset of de novo DNA methylation [phenotype]'], ['two epialleles (A1/A1) [genomic region]']]}\n",
      "-------------------\n",
      "{'158579': [['SHADE1 QTL [gene]'], ['shade-related direct downstream targets gene(s) [gene]'], ['11 shade-regulated BBX family members [gene]'], ['Shade-induced gene(s) pair [gene]'], ['shade-related gene(s) [gene]'], ['shade-induced PAR1/2 [gene]'], ['Shade Avoidance3 (SAV3) gene(s) [gene]'], ['Key signaling gene(s) [gene]'], ['Shade-induced WRKYs [gene]'], ['Shade signaling components [gene]'], ['shade-responsive gene(s) [gene]'], ['shade-induced PAR1 [gene]'], ['both shade- and FR-responsive gene(s) sets [phenotype]'], ['gene(s) highly induced by shade [phenotype]'], ['3 shade-induced gene(s) [gene]'], ['shade-induced marker gene(s) [phenotype]'], ['expression of shade responsive gene(s) [process]'], ['levels of negative regulators of shade avoidance [gene]'], ['SAV3-dependent shade-induced gene(s) [gene]'], ['these interactors [gene]'], ['649 TF gene(s) [gene]']]}\n",
      "-------------------\n",
      "{'137310': [['**Overexpression of AAD2 [gene]**', 'Overexpressing AAD2 [gene]'], ['Overexpression of AAD3 [gene]'], ['overexpression of ALDH3H1 protein(s) [gene]'], ['overexpression of GAT-ADCS [gene]'], ['Overexpression of VvSNAT2 [gene]'], ['Overexpression of ADH2 gene(s) [gene]'], ['AtHb1 overexpression [gene]'], ['DGAT-overexpression [gene]'], ['overexpressed CA [gene]'], ['Overexpressing ASN1 [gene]'], ['Overexpression of AtRH9 [gene]'], ['overexpressed A1B [gene]'], ['Overexpressing BnDGAT1s [gene]'], ['overexpression of CsHCAR [gene]'], ['ADH1 overexpression [gene]'], ['CaATHB-12 overexpression [treatment]'], ['constitutively overexpressed alternative HYH [gene]']]}\n",
      "-------------------\n",
      "{'77944': [['**PAZ domain [domain]**', 'PAZ domain [protein]', 'PAZ [protein domain]', 'PAZ domain [gene]', 'P-element-induced wimpy testis (PIWI)-Argonaute-Zwille (PAZ) domain [protein domain]', 'PAZ [domain]', 'PAZ [protein]', 'AGO1 PAZ domain [protein domain]', 'PAZ domain of AGO1 [protein]', 'AGO PAZ [protein domain]', 'AGO10 PAZ domain [protein domain]', 'Piwi/Argonaute/Zwiklle (PAZ) domain [domain]', 'PAZ domain of AGO1 [gene]', 'PAZ domain of AGO4 [gene]', 'protein(s) with a PAZ domain [protein]', 'PAZ domain protein(s) [protein]'], ['Gly-rich Ago1 domain [protein domain]'], ['**Mid domain [protein domain]**', 'MID [domain]', 'Argonaute Mid [protein domain]'], ['PAZ, MID, and PIWI domains [protein domain]'], ['DUF1785 domain [gene]'], ['N [domain]'], ['PIWI domain of AGO2 [gene]'], ['PIWI [protein domain]'], ['MID domain of AGO6 [gene]'], ['MID domain of AGO4 [gene]'], ['P-element-induced wimpy-testis domain [subcellular compartment]'], ['C-terminal PAZ (Piwi-Argonaute-Zwille) domain [subcellular compartment]'], ['domains DUF1785, PAZ, MID and PIWI [protein domain]'], ['DUF1785 [protein domain]'], ['AGO-binding domain [phenotype]'], ['middle (MID) domain of AtAGO2 [protein domain]'], ['middle (MID) domain of AtAGO5 [protein domain]'], ['Argonaute Linker1 [protein domain]'], ['DUF1785-1 domain [gene]'], ['PAZ-1 domain [gene]'], ['B-1 domain [gene]'], ['PIWI-1 domain [gene]'], ['B-2 domain [protein domain]'], ['DUF1785-3 domain [protein domain]'], ['B-3 domain [protein domain]'], ['block 39 [protein domain]'], ['B-4 domain [protein domain]'], ['DUF1785 domain [domain]'], ['MID domain nucleotide specificity loop [protein domain]'], ['non-conserved PAZ domain [protein domain]'], ['RNA-binding PAZ domain [protein domain]'], [\"the 5' end-binding pocket [protein domain]\"]]}\n",
      "-------------------\n",
      "{'154079': [['Loop A [phenotype]'], ['additional receptors [gene]'], ['different pathway from SOS gene(s) [gene]'], ['same genetic pathway as MAX2 [gene]'], ['subclass d1 [gene]'], ['subclass d2 [gene]'], ['subgroup g1 [gene]'], ['subgroup g2 [gene]'], ['group H [gene]'], ['single pathway or overlapping pathways with AS2 [gene]'], ['signal transduction [gene group]'], ['all AtSRs [gene]'], ['new components in as-1-based signal transduction pathways [gene element]'], ['cross-talk with gene(s) in cytosolic pathways [gene]']]}\n",
      "-------------------\n",
      "{'121796': [['acquosome frequencies [phenotype]'], ['site frequency spectra [phenotype]'], ['realistic frequency distributions of cell types [phenotype]']]}\n",
      "-------------------\n",
      "{'64226': [['Redox site of Trxs [protein]'], ['Functions of cytoplasmic TRXs [gene]'], ['cytosolic TRX [protein]'], ['participation of thioredoxins (Trxs) [protein]'], ['redox-active cysteins in TrxR and Trx domain in NTRC [protein]']]}\n",
      "-------------------\n",
      "{'148026': [['At4 [gene]'], ['PHE [gene]'], ['AtPHL1 [gene]'], ['PtPHR1 [gene]'], ['TaZAT8 [gene]'], ['Phosphorus-Starvation Tolerance 1 (PSTOL1) [gene]'], ['sweet potato SPF1 [gene]'], ['PpDHNC [gene]'], ['StSP6A [gene]'], ['Phe253 [gene]'], ['PsNSP1 [gene]'], ['PsNSP2 [gene]'], ['PsDWARF27 [gene]'], ['PGSC0003DMG400035001 [gene]'], ['PLDepsilon-OE [gene]'], ['Pp3c7_24750 [gene]'], ['Pp3c8_2040 [gene]'], ['COST1 [gene]'], ['Pp3c22_130 [gene]'], ['LOPH1 gene(s) [gene]'], ['SpMDP1 [gene]'], ['ZPT2-2 [gene]'], ['PS-IAA6 [gene]'], ['GmPTF1 [gene]'], ['NgGPP1 [gene]'], ['PNP-A (At2g18660) [gene]'], ['PAYT036935.1 [gene]'], ['PRCE1 [gene]'], ['prce3 [gene]'], ['PagEPFL4 [gene]']]}\n",
      "-------------------\n",
      "{'83360': [['recognition through main-chain atoms [phenotype]'], ['**DNA binding properties [phenotype]**', 'DBPs [phenotype]', 'DBP annotation [phenotype]', 'common structural framework of DNA binding [phenotype]'], ['canonical GTPases [phenotype]'], ['conserved mode of chromophore binding [phenotype]'], ['coupled folding and binding [phenotype]'], ['properties of the bait [phenotype]'], ['major determinant of binding branched DNA junctions [phenotype]'], ['different binding surfaces [phenotype]'], ['its partner on the egg [phenotype]'], ['**non DNA binders [phenotype]**', 'non-DNA-binding protein(s) [phenotype]'], ['protein(s) related to heat shock protein(s) binding [phenotype]'], ['protein(s) binding from information about proximity and orientation [phenotype]'], ['a binding structure [phenotype]'], ['client binding properties [phenotype]'], ['sequence-independent DNA-binding protein(s) [phenotype]'], ['overlaps in binding specificity [phenotype]'], ['constitutive binding to DNA [phenotype]'], ['reliable binding modes [phenotype]']]}\n",
      "-------------------\n",
      "{'169962': [['PAL isozymes [protein]'], ['P. patens PGDH isozymes [enzyme]'], ['PpPGDH2 [enzyme]'], ['N. benthamiana P4H candidates [enzyme]'], ['**P4H enzyme [enzyme]**', 'other P4H(s) [enzyme]'], ['PHYTYL ESTERASE 1 and 2 (PES1, 2) [enzyme]'], ['Function of TpPCS1 [enzyme]'], ['class P [enzyme]'], ['cellulose dehydrogenase [protein domain]'], ['PDH1/2 [enzyme]'], ['two isoforms of ProDH and P5CS [enzyme]'], ['endogenous PP2CA [protein]']]}\n",
      "-------------------\n",
      "{'30540': [['engineers [person]'], ['optics [domain]'], ['expert knowledge of fine mechanics [concept]']]}\n",
      "-------------------\n",
      "{'128302': [['Csy4 derived system [treatment]', 'CSY4 [treatment]', '**CRISPR/Cas9 with RNA endoribonuclease Csy4 processing system [treatment]**', 'Csy4 RNase system [treatment]'], ['**ribozyme gRNA processing system [treatment]**', 'HDV ribozyme-based RNA processing systems [treatment]'], ['single guide with ubiquitin promoter-driven Cas9 [treatment]'], ['OsBiP1-Cas9-tRNA [treatment]'], ['intron-optimized Cas9 [treatment]'], ['virus-inducible CRISPR/Cas9 system [treatment]'], ['transient expression of Cas9 and sgRNA complexes [treatment]'], ['RNA-guided Cas9 system [treatment]'], ['increased activity of Cas9p nuclease at 37°C [treatment]'], ['efficiency of RNA-guided Cas9 system [treatment]'], ['in vivo recognition and cleavage of 21-nt target sequence of GUS-sensors [treatment]'], ['**CRISPRa technology [treatment]**', 'CRISPR/dCas9-based transcriptional activation [treatment]', 'dCas9-VP64 system [treatment]', 'nuclease-deactivated (dCas9) CRISPR systems [treatment]', 'tethering transcriptional activation domains to a nuclease-dead Cas9 (dCas9) [treatment]'], ['research on RNA editing [treatment]'], ['RNA editing efficiency [treatment]'], ['processing system [treatment]'], ['Cas13 systems [technique]'], [\"construct harnessing Csy4 bacterial endoribonuclease's RNA processing ability [method]\"], ['catalytically inactive Cas9 (dCas9) [tool]'], ['editing in vitro [process]'], ['RNA endoribonuclease Csy4-processing system [molecule]'], [\"O'Connell et al. [author]\"]]}\n",
      "-------------------\n",
      "{'102887': [['Fusarium oxysporum [organism]'], ['**Fusarium oxysporum f. sp. vasinfectum [organism]**', 'F. oxysporum f. sp. vasinfectum [organism]', 'cotton pathogen Fusarium oxysporum f. sp. vasinfectum [organism]', 'Fusarium oxysporum f. sp. vasinfectum (FOV) [organism]'], ['F. oxysporum f. sp. Vesinfectum [organism]'], ['Cotton root(s) infected by Verticillium dahliae [organism]'], ['Fusarium wilt of cotton [organism]'], ['BP2 [organism]']]}\n",
      "-------------------\n",
      "{'104766': [['**microorganisms [organism]**', 'Microbial species [organism]', 'Microbial cells [organism]', 'micro-organisms [organism]', 'Core microbes [organism]', 'micro-organism [organism]', 'True microbes [organism]'], ['microorganisms [cell type]'], ['Rare microbes [microbiotum]']]}\n",
      "-------------------\n",
      "{'13416': [['**wavy pattern [phenotype]**', 'wavy or skewing phenotypes [phenotype]', 'regular, loose waves [phenotype]', 'waving phenotypes [phenotype]', 'wavy growth trajectory [phenotype]', 'wavy pattern of growth [phenotype]'], ['wavy stem(s) [phenotype]'], ['waving effect in other parts of plant(s) [phenotype]']]}\n",
      "-------------------\n",
      "{'13722': [['altered epidermal cell elongation [phenotype]'], ['rosette growth by epidermal CK [phenotype]'], ['marginal contribution of epidermal cell length [phenotype]'], ['pronounced contribution of epidermal cell length [phenotype]']]}\n",
      "-------------------\n",
      "{'124777': [['**long noncoding RNAs (lncRNAs) [metabolite]**', 'long ncRNAs (lncRNAs) [metabolite]'], ['Cytoplasmic lncRNAs [metabolite]'], ['extracellular lncRNAs [metabolite]'], ['additional lncRNAs [metabolite]']]}\n",
      "-------------------\n",
      "{'21475': [['**water deficit tolerance [phenotype]**', 'tolerance to water deficits [phenotype]', 'tolerance of water deficit [phenotype]', 'tolerance to water deficit [phenotype]', 'water-deficit tolerance [phenotype]', 'tolerance of water deficiency [phenotype]', 'tolerance of low water potential [phenotype]', 'low water potential tolerance [phenotype]', 'tolerance of plant(s) to water-deficit [phenotype]', 'tolerance to water shortage [phenotype]', 'tolerance to water deficiency [phenotype]', 'plant(s) water deficit tolerance [phenotype]', 'plant(s) tolerance to water deficit [phenotype]', 'water deficiency tolerance [phenotype]', 'tolerance to water limiting conditions [phenotype]', 'tolerance against water deficiency [phenotype]', 'tolerance to water-limiting environments [phenotype]', 'tolerance to water deficit stress [phenotype]'], ['plant(s) tolerance to transient water deficit [phenotype]'], ['tolerate periodic water-limited conditions [phenotype]'], ['tolerance to water deficit in tomato seedlings [phenotype]', '**tomato tolerance to water deficit [phenotype]**'], ['tolerance to long-term water deficit [phenotype]'], ['gain of tolerance during water-deficit conditions [phenotype]'], ['plant(s) tolerance to severe water deficit [phenotype]'], ['tolerance to water deficit in soybeans [phenotype]']]}\n",
      "-------------------\n",
      "{'35129': [['pericycle-like cells [cell type]', 'pericycle or pericycle-like cells [cell type]', '**pericycle-like [cell type]**'], ['dividing pericyclic cells [tissue]'], ['auxin-responsive pericycle-like cells [cell type]'], ['**pericycle derivatives [cell type]**', 'cells derived from the pericycle [cell type]'], ['expanding pericycle cells [cell type]'], ['pericycle cells in G1 phase [cell type]'], ['pericycle-derived target cells [cell type]'], ['quiescent pericycle cells [cell type]'], ['dividing pericycle and C5 cells [cell type]'], ['dividing pericycle-derived cells [cell type]'], ['rapidly proliferating pericycle-derived cells [cell type]'], ['pericycle cells in mitotically-active state [cell type]'], ['stem-cell-like pericycle cells [cell type]'], ['**living pericycle cells [cell type]**', 'cells of the root(s) pericycle [cell type]'], ['cell division capacity of pericycle [cell type]']]}\n",
      "-------------------\n",
      "{'129826': [['**AtABCB1 [gene]**', 'AtABCB-1 [gene]'], ['ABAP1 [gene]'], ['AtBELL1 [gene]'], ['plant(s) AtRanBPM [gene]'], ['At1B (AKB) [gene]'], ['VARB [gene]'], ['ATBES1 gene(s) [gene]', '**ATBES1 [gene]**'], ['AtBES1 gene(s) in Sub V [gene]'], ['rbcS2B transcripts [gene]'], ['Myosin XI-B (Myo11B1) [gene]'], ['AtBAT1 transgenic plant(s) [organism]'], ['BOBBER1 (BOB1) [gene]'], ['WT AtBOR1 [gene]'], ['AtMBF1s [gene]'], ['TBL10 mRNA [gene]'], ['AtBARD1 [gene]'], ['AtBOR1 mRNA [gene]'], ['AtTSBtype2 [gene]'], ['BAT1 mRNA [molecule]'], ['AtB alpha mRNA [gene]'], [\"AtB'epsilon mRNAs [gene]\"], ['AtBI1 mRNA [gene]'], ['AtB beta mRNA [mrna]'], ['ATP 1b [gene]'], ['Expression of AtCAD B1 [gene]'], ['Expression of AtCAD B2 [gene]'], ['AKINbeta1 mRNA [gene]'], ['AKINbeta2 mRNA [gene]'], ['AtBAN pro and AtFUL pro [gene]'], ['plastidic AtBT1 [gene]'], ['B-class (AtARF1/2) [gene]'], ['AtBOR1 levels in tomato [organism]'], ['AtBl1 expression [gene]'], ['AtBAN expression [gene]'], ['gain-of-function substitute of AtBES1 [gene]'], ['**AtBax inhibitor 1 [gene]**', 'AtBI1 gene(s) [gene]']]}\n",
      "-------------------\n",
      "{'144869': [['SND1 [protein]'], ['Transcripts of SnRK2.3 [gene]', '**SnRK2.3 expression [gene]**'], ['SAP and MIZ1 domain-containing ligase1 (SIZ1) [gene]'], ['SLAC1/protein kinase/ABA-dependent pathways [gene]'], ['group 3 SnRK2s [gene]'], ['NAC072 protein(s) [gene]'], ['FsPK1 [gene]'], ['PP2C hub [gene]'], ['Acidic domain [gene]'], ['ZmOST1 [gene]'], ['AtSnRK2.2, AtSnRK2.3, and AtSnRK2.6 [gene]'], ['Subclass III SnRK2s [gene family]'], ['SUCROSE NONFERMENTING1-related protein(s) kinase2s (SnRK2s) [gene]'], ['VOLTAGE-DEPENDENT ANION CHANNEL 3 [gene]'], ['ABA-SnRK2 signalling module [treatment]'], ['AtSnRK2.2 transcription [gene]'], ['ZmPYL and ZmSnRK2 [protein]'], ['ZmSnRKs [protein]'], ['NAC domain protein(s) (PvNAC1) [gene]'], ['SnRK2.6/OST1 activity [gene]']]}\n",
      "-------------------\n",
      "{'38783': [['COM lines [mutant]'], ['AEQsig6 seedlings [mutant]'], ['Atcesa6 prc1 Atcesa1 rsw1 YFP-CESA6 control line [mutant]'], ['R211E lines [mutant]'], ['R211A and R211E lines [mutant]'], ['C1, C3 and A5 [mutant]'], ['PSR2-5 [mutant]'], ['Col-0 mutant lines cpr5 rps2 and coil rps2 [mutant]'], ['HVA1-P line [mutant]'], ['Cas9-PPO lines [mutant]'], ['Cas9-CRU transformed mutant lines [mutant]'], ['Cas9-PPO transformed mutant lines [mutant]'], ['cao and R211E lines [mutant]'], ['Chl accumulation in R211E lines [phenotype]'], ['parental FAH12 controls [mutant]']]}\n",
      "-------------------\n",
      "{'171389': [['methyltransferase 1 (MET1) [enzyme]', '**DNA Methyltransferase 1 (MET1) [enzyme]**', 'DNA methyltransferase1 (MET1) [enzyme]', 'DNA Methyltransferase MET1 [enzyme]'], ['INCREASE in BONSAI METHYLATION 1 (IBM1) [enzyme]'], ['STEROL METHYLTRANSFERASE 1 (SMT1) [enzyme]'], ['sterol methyltransferase1 (smt1) [protein]'], ['DNA methyltransferase CMT3 [enzyme]'], ['methionine synthase 1 (AtMS1) [protein]'], ['MT1 [enzyme]'], ['DNA methylases METHYLTRANSFERASE 1 (MET1) [protein]'], ['the only demethyltransferase in callus formation [process]'], ['plant(s) sterol C-methyltransferase [enzyme]']]}\n",
      "-------------------\n",
      "{'124651': [['RNAi-mediated KD of HvMORC1 [gene]'], ['RNAi knock-down of LjLbs gene(s) [gene]'], ['RNAi knockdown of MdCCD7 [gene]'], ['RNA dilution study [treatment]'], ['RNAi silencing of HFIE [gene]'], ['OsFIE2 RNAi construct [treatment]'], ['CHLH RNAi experiments [method]'], ['moderate RNAi of MSH2 gene(s) [gene]'], ['RNAi of MSH7 gene(s) [gene]'], ['RNA interference of OpERF2 [gene]'], ['siRNA knock-down of SMG1 [gene]'], ['TMBP200 RNA interference (RNAi) [treatment]', '**TMBP200 RNAi [treatment]**'], ['VIGS-based silencing of lncRNA1459 and lncRNA1840 [treatment]'], ['RNA interference with StCYP86A1 [gene]'], ['Nb-P4H10-RNAi sequence [gene]'], ['RNAi-mediated knockdown of TOP1beta [gene]'], ['RLK902 RNAi and amiRNA approaches [treatment]'], ['RPS5 RNAi [gene]'], ['small RNA interference of HsfA1 [gene]'], ['RNA interference with PGSC0003DMG400035001 [gene]'], ['RNAi-silencing of CDC5 [gene]'], ['RNAi experiments [treatment]'], ['57 RNAi experiments [process]'], ['micro-RNA-mediated knock down of TcNPR3 expression [gene]'], ['RNAi-induced knockdown of NtMYC2 [gene]'], ['RNA interference of OsTDL1A [treatment]'], ['RNAi silencing of aact2 [treatment]'], ['suppressing homeologous GT43 gene(s) by RNAi [gene]'], ['RNAi-fragment based on GmFTL1 coding-sequence [gene]'], ['Host-derived RNAi of MiMsp40 [gene]'], ['RNAi of met1 [treatment]'], ['RNAi inhibition of AtNDB1 [gene]'], ['RNAi-mediated knockout of BRI1 [gene]'], ['RNAi evidence [technique]'], ['Knock down of NAT398b [gene]'], ['Knock down of NAT398c [gene]'], ['RNAi of Os07g37920 [gene]'], ['RNAi of LBD14 [gene]'], ['RNAi knockdown of TOB2 [gene]'], ['RNAi knockdown of TOB3 [gene]'], ['PDCT single-gene RNAi [treatment]'], ['RNA interference of WSL3 [gene]'], ['**RNA interference (RNAi) of MeNPF4.5 [treatment]**', 'MeNPF4.5-RNAi expression [treatment]'], ['BSC-specific RNAi [treatment]'], ['AtVOCRNAiLS [treatment]'], ['AtVOCRNAiLL [treatment]'], ['AtVOCRNAiSS [treatment]'], ['AtVOCRNAiSL [treatment]'], ['RNAi targeting HaAK [treatment]'], ['host-delivered hpRNAi-mediated silencing of EcR, HR3, AK, CHI, and NPF gene(s) [gene]'], ['VvMSA-RNAi cells [cell type]'], ['co-silencing on CYP51 gene(s) [gene]'], ['mRNAi studies [treatment]']]}\n",
      "-------------------\n",
      "{'141611': [['gSTEP [gene]'], ['GLDTFt insertion [gene]'], ['IMPL2-GST [gene]'], ['WT DGAT1 amplicon [gene]'], ['GST mock [gene]'], ['GST-DOFT [gene]']]}\n",
      "-------------------\n",
      "{'152629': [['AKTs [gene]'], ['ShKT domain [gene]'], ['AKT gene(s) (Spipo8G0031700 and Spipo23G0039700) [gene]'], ['AKT gene(s) (Spipo8G0031700) [gene]'], ['two gene(s) encoding AtKCO1-like K+ channels [gene]']]}\n",
      "-------------------\n",
      "{'152756': [['SNF1-RELATED protein(s) KINASE2s (SnRK2s) [gene]', '**Snf1-Related protein(s) Kinase 2 (SnRK2) [gene]**', 'SNF1-related protein(s) kinases 2 [gene]', 'Snf1-related protein(s) kinase2 (SnRK2) [gene]', 'SNF1-RELATED KINASE 2 (SnRK2s) [gene]', 'SNF1-RELATED protein(s) KINASE 2s (SnRK2s) [gene]', 'SNF1-related kinase (SnRK2) encoding gene(s) [gene]'], ['SRK2I [gene]'], ['SNF1-activated protein(s) kinase (SNF1) [gene]'], ['SNF1 [gene]'], ['AT3G01090/KIN10 [gene]', 'sucrose-non-fermentation1 (SNF1)-related kinase KIN10 [gene]', '**KIN10 (SNF1 kinase homolog 10)/SnRK1.1 [gene]**'], ['**AT3G29160/KIN11 [gene]**', 'AtKIN11 [gene]', 'KIN11 (SNF1 kinase homolog 11)/SnRK1.2 [gene]'], ['SUCROSE NON-FERMENTING1 RELATED KINASES1 (SnRK1) [gene]', 'SUCROSE NON-FERMENTING-1 (SNF1)-RELATED KINASE 1 (SnRK1) [gene]', 'Snf1 RELATED KINASE 1 (SnRK1) [gene]', '**SNF1-related kinase 1 (SnRK1) [gene]**', 'FvSnRK1 [gene]', 'Snf1-Related protein(s) Kinase 1 (SnRK1) [gene]', 'SNF1-related kinase (SnRK1) [gene]', 'SNF1-RELATED protein(s) KINASE1s (SnRK1s) [gene]', 'Snf-related kinase 1 (SnRK1) [gene]', 'Sucrose-non-fermenting-1-Related protein(s) Kinase 1 (SnRK1) [gene]', 'SnRK1 kinase(s) [gene]', 'energy sensor SUCROSE NON-FERMENTING1 (SNF1)-RELATED protein(s) KINASE1 (SnRK1) [gene]', 'SNF1-related protein(s) kinase-1 (SnRK1) [gene]', 'Snf1-related protein(s) kinase1 (SnRK1) [gene]', 'Sucrose nonfermenting related kinase1 (SnRK1) [gene]', 'SNF1-related kinase 1 [gene]', 'plant(s) SNF1-related protein(s) kinase (SnRK1) [gene]', 'Snf1-RELATED-KINASE1 (SnRK1) [gene]', 'SNF1-RELATED KINASE1 [gene]', 'SNF1-Related Kinase 1 (SnRK1) family [gene]'], ['AKIN12 [gene]'], ['SNF1-related protein(s) kinase2.2 (SnRK2.2) [gene]', '**SNF1-related protein(s) kinase 2.2 (SnRK2.2) [gene]**'], ['SnRK2-interacting calcium sensor (SCS) [gene]'], ['Snf1T210 [gene]'], ['SNF1 kinase [gene]'], ['SNF1-Related protein(s) Kinases SnRK2.10 [gene]'], ['SNF1-serine/threonine-protein kinase TaSnRK2.4 (SnRK2.4) [gene]'], ['FvSnRK3s [gene]'], ['SNF-1 RELATED protein(s) KINASE 2.8 (SnRK2.8) [gene]'], ['SNF1/AMPK/SnRK1 [gene]'], ['**SNF1-related kinases [gene]**', 'SNF1-related protein(s) kinases (SnRK) [gene]', 'SNF1-related kinase [gene]'], ['SNF1-related protein(s) kinase 2.4 [gene]'], ['ApSNRK1.1-1/2 [gene]'], ['gene(s) involved in SNF1-related kinases (SnRKs) pathway [gene]'], ['expression of SNF1-related protein(s) kinases (SnrK 2.1 and 2.4) [gene]'], ['SnAK [gene]'], ['SNF1/AMPK/SnRK1 kinases [gene]'], ['SNF1-related protein(s) kinases 2E (SRK2E) [gene]'], ['SNF1-related protein(s) kinase 2.6 (SnRK2.6) [gene]'], ['Snf-1 [gene]'], ['SNF1-related protein(s) kinase 2.6-2C [gene]'], ['SNF-1 related kinase (SnRK1) [enzyme]'], ['presence of SnAK [gene]'], ['activity of SNF1 kinases [gene]'], ['SNF1-related protein(s) kinase 2 (SnRK2) family [gene family]'], ['SNF1-related protein(s) kinase SOS2 [gene]']]}\n",
      "-------------------\n",
      "{'158331': [['BLIND [gene]'], ['BIM1 hybridization signal [gene]'], ['BME3 [gene]'], ['CaBLIND (CaBL) [gene]'], ['BA1/LAX1 clade [gene]'], ['bim2-1 and bim3-1 [gene]']]}\n",
      "-------------------\n",
      "{'40149': [['bp blr rev triple mutants [mutant]'], ['root(s) of pgm2/3 transgenic plant(s) [organ]'], ['plant(s) with CEN180 insertion [mutant]'], ['csn3-3 plant(s) [mutant]'], ['L3 plant(s) [mutant]'], ['intergenotype populations with CBF3 plant(s) [mutant]'], ['TET3c tomato plant(s) [mutant]'], ['sfr3 transformant plant(s) [mutant]'], ['other triple mutants [mutant]'], ['Triple mutant ovules [mutant]'], ['crn-10 bam1 bam2 bam3 plant(s) [mutant]'], ['**PfHd3a gene-edited mutant perilla [mutant]**', 'PfHd3a GE-mutant perillas [mutant]'], ['rbr triplex plant(s) [mutant]'], ['tmt1/tmt2/tmt3 plant(s) [mutant]'], ['**msl10-3G plant(s) [mutant]**', 'msl10-3G mutants [mutant]'], ['plant(s) carrying cd3 D2213 allele [mutant]'], ['cenh3-4 mutant plant(s) [mutant]'], ['camta3-D plant(s) [mutant]'], ['sterile edited BnCYP704B1a plant(s) [mutant]'], ['sterile edited BnCYP704B1b plant(s) [mutant]'], ['cef3-c1 plant(s) [mutant]'], ['cef3 plant(s) [mutant]'], ['sec3a/SEC3A plant(s) [mutant]'], ['MB3-treated plant(s) [mutant]'], ['B3-Q plant(s) [mutant]'], [\"**c3'h/med plant(s) [mutant]**\", \"mutant c3'h/med plant(s) [mutant]\"], ['brl3 plant(s) [mutant]'], ['1 of the 102 triple-edited gl1 plant(s) [organism]'], ['cenh3 mutant plant(s) [mutant]'], ['genome of cenh3 mutant plant(s) [organism]'], ['cenh3 mutant parent [mutant]'], ['plant(s) expressing variant CENH3s [mutant]'], ['CENH3 tailswap plant(s) [mutant]'], ['pCambia1301 control plant(s) [mutant]'], ['cpm plant(s) [mutant]'], ['plant(s) with mutated vet3-1 allele [mutant]'], ['T3 MQ1v plant(s) [mutant]'], ['plant(s) expressing S369AS376A-ZmPHOT1 [mutant]'], ['opr3-3 plant(s) [mutant]'], ['cdr3-1D plant(s) [mutant]'], ['plant(s) with ΔTaELF-B3 allele [mutant]'], ['plant(s) expressing CdDHN4-S [mutant]'], ['tir3 plant(s) [mutant]'], ['three auxin mutants [mutant]'], ['plant(s) bearing MET1, DRM2, CMT3, and MOM1 mutated alleles [mutant]'], ['barley plant(s) carrying the exotic ELF3 allele [mutant]'], ['plant(s) engineered to carry new PCNA-ECO1 modules [organism]'], ['dml3 plant(s) [mutant]'], ['aboveground biomass of shv3 mutant plant(s) [mutant]'], ['mass1;2;3-crispr#7 plant(s) [mutant]'], ['plant(s) binary vector pBI121 [phenotype]'], ['CBF3-ox background [mutant]'], ['cmt3 hybrids [mutant]'], ['bam1 bam2 bam3 mutant plant(s) [mutant]'], ['plant(s) carrying all three independent Cas9-gRNA combinations [mutant]'], ['plant(s) carrying the glr3.3 mutation [mutant]'], ['plant(s) genome of eif3h mutant plant(s) [genome]'], ['constructs C1, C2, C3, C4, C5, or C0 [construct]'], ['plant(s) harboring ScZRC1 [mutant]'], ['35S-P0(Col) plant(s) [organism]'], [\"c3'h/med background [mutant]\"], ['CENH3/cenh3-1 plant(s) [organism]'], ['plant(s) with CPSF30-3 mutation [mutant]'], ['chromosomes of parent plant(s) with mutant CENH3 [mutant]'], ['plant(s) transformed with MmGA3ox2 [mutant]'], ['plant(s) transformed with MmGA3ox1 [mutant]'], ['plant(s) transformed with MmGA3ox1 + MmGA3ox2 [mutant]'], ['clock-altered T3 plant(s) [mutant]'], ['buds of CMS plant(s) [mutant]']]}\n",
      "-------------------\n",
      "{'164114': [['different extraction methods [technique]', '**metabolite extractions [technique]**'], ['metabolite identification [technique]'], ['Successful implementation of workflow [process]'], ['developed methods and technologies [process]'], ['automated elucidation of metabolite structures [treatment]'], ['direct detection of plant(s) proteins, lipids, carbohydrates and many other biomolecules [metabolite]']]}\n",
      "-------------------\n",
      "{'42296': [['atrga1 mutant [mutant]'], ['tga256 [mutant]'], ['agg1-2 [mutant]'], ['agg2-2 [mutant]'], ['agg3-2 [mutant]'], ['agg1 mutant [mutant]'], ['point-mutation in A. thaliana GnTI (AtGnTI) [gene]'], ['atagl62-2 [mutant]'], ['atbg1 single mutant [mutant]'], ['atglr3.7-2 mutant [mutant]'], ['Atrgga [mutant]'], ['AtTTG1 mutants [mutant]'], ['atago1-27 [mutant]'], ['atago1-46 [mutant]'], ['AtAgo1 [mutant]'], ['aattg1 mutants [mutant]'], ['atgcn1 [mutant]'], ['Aattg1 alleles [gene]'], ['allele Aattg1-2 [mutant]', '**Aattg1-2 [mutant]**'], ['Aattg1-1 [mutant]'], ['gns1/Atelp2 [mutant]'], ['AtGNC mutant gnc [mutant]'], ['**aggie2 mutant [mutant]**', 'aggie2 mutation [mutant]'], ['mto1-1 mutant of A. thaliana [mutant]'], ['atgullo2-2 [mutant]'], ['AtGulLO2 [mutant]'], ['atg2-2 [mutant]'], ['atgsl6 [mutant]'], ['AGO1 perturbation [mutant]'], ['atgpi8-1 [mutant]'], ['expression of an inhibitory mutant of A. thaliana Rab 1b (AtRab1b(N121I)) [mutant]'], ['mto1 mutants of A. thaliana [mutant]'], ['atgl3.4-1 seedlings [mutant]'], ['atuggt1-1 [mutant]'], ['atuggt1-2 [mutant]'], ['atggt atpcs1 [mutant]']]}\n",
      "-------------------\n",
      "{'76769': [['B-box [gene]'], ['B-box [protein motif]'], ['**NLS motif [protein]**', 'NLS motifs [protein]'], ['monopartite motifs [protein domain]'], ['nuclear localization sequences (NLSs) [sequence motif]', '**Nuclear localization signal (NLS) [sequence motif]**', 'predicted NLSs [sequence motif]', 'functional NLS [sequence motif]', 'Nuclear Localization Sequence [sequence motif]'], ['SAHIQ motif [protein domain]'], ['**NLS motif [protein domain]**', 'NLS motifs [protein domain]', 'nuclear localization motifs [protein domain]', 'nuclear localization signal (NLS) motifs [protein domain]', 'nuclear localization motif [protein domain]'], ['RRKK motif [protein sequence]'], ['**NLS motif [protein motif]**', 'Nuclear localization signal [protein motif]', 'nuclear localization signal sequence [protein motif]', 'nuclear location signal [protein motif]', 'nuclear localization sequences [protein motif]', 'Nuclear localization signal (NLS) [protein motif]'], ['nuclear localization signal sequences [genetic element]'], ['nuclear localization (AP2-associated motif) [sequence]'], ['Two putative nuclear localization signals [sequence feature]'], ['potential nuclear localization signal [treatment]'], ['**potential NLS motif [phenotype]**', 'potential nuclear localization sequence [phenotype]'], ['**putative NLS motif [protein motif]**', 'putative nuclear localization signals (NLS) [protein motif]'], ['functional NLS [protein motif]'], ['alternative bipartite NLS motif [protein domain]'], ['mitochondria/nuclear localization signal (MLS/NLS) motifs [protein]'], ['bipartite nuclear localization signal motif [phenotype]'], ['full NLS motif [protein motif]'], ['putative nuclear localization signal motifs [protein domain]'], ['nuclear localisation signal (NLS) peptide motif [protein motif]'], ['uncharacterised NLS motif [protein motif]'], ['NLS signatures [protein domain]'], ['**nuclear localization signal motifs [phenotype]**', 'nuclear localization motif [phenotype]'], ['canonical NLS motifs [protein]'], ['basic nuclear localization signals [phenotype]'], ['cryptic NLS motifs [protein domain]'], ['cNLS motifs [protein domain]'], ['nuclear localization signal sequences [treatment]'], ['two NLS motifs [sequence feature]'], ['one NLS motif [sequence feature]'], ['two potential NLS motifs [protein domain]'], ['nuclear export signal (NES) motif [protein domain]', 'NES motifs [protein domain]'], ['three putative nuclear localization signals [phenotype]'], ['two nuclear localization signals [phenotype]'], ['nuclear localization sequences [protein sequence]'], ['nuclear localization signal sequence [sequence feature]'], ['two NLS motifs [protein]'], ['one nuclear localization signal motif [protein domain]'], ['bipartite nuclear localization signal motif [protein]'], ['a nuclear localization signal [phenotype]'], ['nucleus localizing sequence [sequence]'], ['nuclear localization signal (NLS) motif [peptide]'], ['N-X7-R/K [gene]'], ['conserved and functional Nuclear Localization Signal [protein]'], ['NLS located in motif 6 [gene]'], ['**Nuclear localization signals [sequence]**', 'nuclear localization sequences [sequence]'], ['predicted NLS motifs [protein domain]'], ['a putative nuclear localisation signal [sequence feature]'], ['atypical nuclear localization signal (NLS) [protein motif]'], ['NLSs and novel motifs [protein domain]']]}\n",
      "-------------------\n",
      "{'101782': [['Goettingen-7 [organism]'], ['Goettingen-22 [organism]'], ['Gu-1 [organism]'], ['accession Dijon-G (Di-G) [organism]', 'Di-G [organism]', '**Dijon-G [organism]**'], ['15-11 [organism]'], ['Di-I [organism]']]}\n",
      "-------------------\n",
      "{'5857': [['**plant(s) resistance [phenotype]**', 'resistance of plant(s) [phenotype]', 'Resistance in plant(s) [phenotype]', \"plant's resistance [phenotype]\", \"plants' resistance [phenotype]\"], ['**Effective resistance [phenotype]**', 'plant(s) with effective resistance [phenotype]'], ['vascular resistance [phenotype]'], ['This resistance [phenotype]'], ['plant(s) resistance strategies [phenotype]'], ['**plant(s) resistance responses [phenotype]**', \"plants' resistance responses [phenotype]\"], ['plant(s) resistance [process]'], ['plant(s) resistance to harmful organisms [phenotype]'], ['**plant(s) resistance to insects and pathogens [phenotype]**', 'plant(s) resistance to pathogens and herbivorous insects [phenotype]', 'plant(s) resistance against pathogens and insects [phenotype]', 'plant(s) resistance against insects and pathogens [phenotype]'], ['plant(s) resistance-related functions [phenotype]'], ['plant(s) resistance against various foliar pathogens [phenotype]'], ['local and/or systemic plant(s) resistance [phenotype]'], ['a component of resistance responses in plant(s) [phenotype]'], ['plant(s) resistance to leaf microbes [phenotype]'], ['plant(s) basal resistance to pathogens [phenotype]'], ['plant(s) resistance to pathogenesis [phenotype]'], ['plant(s) resistance against herbivores [phenotype]'], ['plant(s) resistance to phytopathogens [phenotype]'], ['plant(s) resistance to pests and pathogens [phenotype]'], ['**plant(s) resistance to pathogen [phenotype]**', 'plant(s) resistance against pathogen [phenotype]'], ['plant(s) resistance toward pests [phenotype]'], ['plant(s) resistance by suppressing cell death [phenotype]'], ['effectivity of resistance [phenotype]'], ['post-stomatal resistance [phenotype]'], ['resistance against pathogens and/or herbivores [phenotype]'], ['resistance against plant(s) pathogens [phenotype]'], ['control plant(s) resistance [phenotype]'], ['plant(s) resistance to viruses, fungi and bacteria [phenotype]'], ['resistant context [phenotype]'], ['resistance against local defense response [phenotype]'], ['conferring resistance towards pathogens [phenotype]'], ['plant(s) resistance to toxins and pathogens [phenotype]'], ['resisting pathogens [phenotype]'], ['plant(s) resistance system to pathogens [phenotype]'], ['disease resistance activities [phenotype]'], ['plant(s) disease resistance responses [phenotype]'], ['establishment of resistance [phenotype]'], [\"plant's defensive capacity [phenotype]\"], ['development of a resistance response [phenotype]'], ['systemic resistance of plant(s) to pathogens [phenotype]']]}\n",
      "-------------------\n",
      "{'61159': [['IR-regulated hallmark functions [phenotype]'], ['Regulatory mechanisms [phenotype]'], ['Complex regulatory networks [phenotype]'], ['expression of upstream regulators [phenotype]'], ['cis regulation [phenotype]'], ['aberrant regulatory process [phenotype]'], ['regulatory cores [phenotype]'], ['cis-regulatory-elements [phenotype]'], ['Positive regulatory outcome of CDPKs [phenotype]'], ['negative regulator of SnRK2 [phenotype]'], ['negative regulator of FLS2 signaling [phenotype]'], ['key negative regulator of G1/S transition [phenotype]'], ['regulatory mechanisms of HMGR [phenotype]'], ['key negative regulator of BR signaling [phenotype]'], ['SAR-negative regulators [phenotype]'], ['binding of TFs to regulatory elements [phenotype]'], ['regulatory specificity [phenotype]'], ['co-regulation network [phenotype]'], ['regulatory scaffolds [phenotype]'], ['correlative expression with regulatory targets [phenotype]'], ['cis-regulators [phenotype]'], ['positive regulatory signals [phenotype]'], ['channel regulator activity [phenotype]']]}\n",
      "-------------------\n",
      "{'30892': [['external structural frame [device part]'], ['major components [device]']]}\n",
      "-------------------\n",
      "{'11432': [['dynamic injury [phenotype]'], ['cell wall damage syndrome (CWDS) [phenotype]'], ['Cr-induced derangement [phenotype]'], ['index of injury [phenotype]'], ['Accumulated damage [phenotype]'], ['too much pressure [phenotype]'], ['lesions formation [phenotype]'], ['Exogenous damage [phenotype]'], ['Massive tissue damage in P-type [phenotype]'], ['growth-dependent damage [phenotype]'], ['extensive tissue damage [phenotype]'], ['severe damage [phenotype]'], ['Induced total damage [phenotype]'], ['unnecessary tissue damage [phenotype]'], ['posthypoxic injury [phenotype]'], ['permanent structural damage in the tissue [phenotype]'], ['PSI FeS cluster damage [phenotype]'], ['damage accumulated during space travel [phenotype]'], ['enzymatic damage [phenotype]'], ['moderate tissue damage [phenotype]'], ['large tissue damage [phenotype]'], ['physical damages to antenna complex [phenotype]'], ['severe damage on cultures [phenotype]'], ['significant injuries on shoot physiology [phenotype]'], ['increased cellular damage [phenotype]'], ['intensive tissue damage [phenotype]'], ['constitutive damage-associated molecular pattern (DAMP) [phenotype]'], ['damage to phytoplankton [phenotype]'], ['damage to PSI and PSII [phenotype]'], ['severe cellular damage under As(V) stress [phenotype]'], ['mite-induced damage [phenotype]'], ['permanent lesions in WT [phenotype]'], ['lesions in A. thaliana [phenotype]'], ['bark breakdown [phenotype]'], ['previous herbivory [phenotype]'], ['damage to enzyme protein(s) [phenotype]'], ['SSB in shoots [phenotype]'], ['damage related to metabolic imbalance [phenotype]'], ['bleach effect [phenotype]'], ['significant injury [phenotype]'], ['gross cellular damage [phenotype]'], ['spontaneous production of lesions [phenotype]'], ['biological damage [phenotype]'], ['damage to tissue above point of application [phenotype]'], ['severe damage on crops [phenotype]']]}\n",
      "-------------------\n",
      "{'118549': [['markers in region of ~23.38 cM [marker]'], ['2.1 cM interval [genomic region]'], ['Fine map of 4.74 cM [genomic region]'], ['6 cM region [genetic interval]'], ['4 cM interval [genetic interval]'], ['15.9, 8.4, and 4.3 cM [genomic region]'], ['3.6 cM [genomic region]'], ['13.5 cM [genomic region]'], ['6-28 cM on Chr10 [chromosome]'], ['1.9 cM [statistic]'], ['1.1 cM window [statistic]'], ['1.3-cM interval [genetic location]'], ['2.9 cM interval [genomic region]'], ['a 22.8 cM interval on chromosome 8 [genomic location]'], ['3.6 cM interval [genomic location]'], ['2.7 cM [region]'], ['5.4 cM region [genomic location]'], ['12 cM on chromosome II [organism]'], ['42 cM on chromosome II [organism]'], ['5.5 cM interval of chromosome V [genomic location]'], ['2.7 cM interval within MRC-J [genomic location]'], ['12 cM interval on chromosome II [genomic location]'], ['region of 0.4 cM [genomic region]'], ['< or =3.2 cM (centiMorgan) [genomic region]'], ['0.09 cM interval [genomic region]'], ['approximately 0.5-cM interval [genomic location]'], ['>0.7-cM region [genomic location]'], ['less than 0.1-cm region [phenotype]']]}\n",
      "-------------------\n",
      "{'170116': [['NTT activity [process]'], ['TST1 activity [process]'], ['NAD+ affinity in isolated mitochondria of potato tubers [subcellular compartment]']]}\n",
      "-------------------\n",
      "{'60611': [['**modest increase at low temperature [phenotype]**', 'improvement in low-temperature performance [phenotype]'], ['less growth at low pH [phenotype]'], ['decrease in bacterial growth [phenotype]'], ['downregulated upon temperature increase [phenotype]'], ['exacerbated growth reductions [phenotype]'], ['reduced growth of fungus [phenotype]']]}\n",
      "-------------------\n",
      "{'8815': [['North American lineage [phenotype]'], ['developing lineages [phenotype]'], ['derived lineages [phenotype]'], ['Distinct phylogenetic differences [phenotype]'], ['distinct evolutionary relationships [phenotype]'], ['different lineages [phenotype]'], ['lineage differentiation [phenotype]'], ['nine lineages [phenotype]'], ['specific lineages [phenotype]'], ['highly differentiated lineage [phenotype]'], ['lineage-specific heterogeneity in rates of divergence [phenotype]'], ['C. chinense-specific lineages [phenotype]'], ['lineage III [phenotype]'], ['relictual lineage [phenotype]'], ['morphological lineages [phenotype]']]}\n",
      "-------------------\n",
      "{'23115': [['cargo group [phenotype]'], ['strigolactone-dependent interaction [phenotype]'], ['non-binding SL-receptor interactions [phenotype]'], ['strigolactone specificity [phenotype]'], ['strigolactone-independent function [phenotype]']]}\n",
      "-------------------\n",
      "{'85602': [['reduced iron conditions [treatment]', 'reduced iron availability [treatment]', 'reduced iron [treatment]', 'Low iron availability [treatment]', 'lack of sufficient iron [treatment]', '**iron starvation [treatment]**', 'excluding iron [treatment]', 'low iron supply [treatment]', 'conditions that restrict LODIS accumulation [treatment]', 'absence of iron [treatment]', 'iron-limiting stress [treatment]'], ['Prolonged iron deficiency [treatment]'], ['aplastic anemia [disease]']]}\n",
      "-------------------\n",
      "{'131773': [['VviNPF2.1 [gene]'], ['**VvNPF3.2 [gene]**', 'grapevine NFP3.2 [gene]'], ['**VvNPF6.5 [gene]**', 'CHL1 homologous gene(s) VvNPF6.5 [gene]', 'vNPF6.5 [gene]']]}\n",
      "-------------------\n",
      "{'135141': [['OsCHLH [gene]'], ['CmRSH4b [gene]']]}\n",
      "-------------------\n",
      "{'130704': [['homologous counterparts in A. thaliana chromosome 4 [genomic region]'], ['homologous location for MAM in A. thaliana [genomic location]']]}\n",
      "-------------------\n",
      "{'151821': [['GmPYLs [gene]'], ['GmPYL15-20 [gene]'], ['**GmYABBYs [gene]**', 'GmYABBY gene(s) [gene]'], ['GmbHLH gene(s) [gene]'], ['GmPIF4 I group transcription [gene]'], ['2 GmXIPs [gene]'], ['all subgroups of GmLRR-RLKs [gene]'], ['GmIDL3a and GmIDL3b [gene]'], ['GmIDL4a and GmIDL4b [gene]'], ['Four nucleotide sequences [gene]'], ['four Group Ia COL gene(s) (GmCOL1a/b, COL2a/b) [gene]'], ['other GmbHLHs [gene]'], ['four GmPIF4s [gene]'], ['two GmPIF4s [gene]'], ['twenty syntenic gene(s) pairs with GmPHY gene(s) [gene]']]}\n",
      "-------------------\n",
      "{'13447': [['plant-specific clade [phenotype]', '**green clade [phenotype]**'], ['plant(s) with other features [phenotype]'], ['any other plant(s) trait [phenotype]'], ['the source to derive plant(s) specific features [phenotype]'], ['origin of green plant(s) [phenotype]'], ['paleobotany [phenotype]'], ['modern plant(s) characteristics [phenotype]']]}\n",
      "-------------------\n",
      "{'57227': [['CsOLE6 [gene]'], ['CsOLE7 [gene]'], ['CsOLE8 [gene]'], ['CsOLE3 [gene]'], ['CsOLE4 [gene]'], ['Col-0 gene(s) ID [gene]'], ['control between StCOL1 [gene]'], ['Group Ia COLs of P. nil [gene]'], ['**CsCOL9 [gene]**', 'CONSTANS-like 9 (COL9) [gene]'], ['CsCOL6/7 [gene]'], ['CsCOL3/4 [gene]'], ['CsCOL7 [gene]'], ['CsCOL3/6 [gene]'], ['COL1 NAT [gene]'], ['CONSTANS (CO) homologs [gene]'], ['COL family [gene]'], ['8 ZmCOL gene(s) pairs [gene]'], ['Col-0 genotype [gene]'], ['CsOLE2 [gene]'], ['downstream targets of StCOL1 [gene]']]}\n",
      "-------------------\n",
      "{'8164': [['**Causal mutation [phenotype]**', 'Causative mutation [phenotype]', 'Causal mutations [phenotype]', 'causal mutation for phenotype [phenotype]', 'causative mutations [phenotype]', 'molecular lesion responsible for observed phenotypes [phenotype]', 'causal mutation of the phenotypes [phenotype]', 'mutation that causes the phenotype of interest [phenotype]', 'causative genetic defect [phenotype]'], ['causative mutations [mutation]', '**Causal mutation [mutation]**', 'causal lesions [mutation]'], ['causative mutations [mutant]'], ['causative variants [genotype]'], ['causative variant [genetic variation]', '**Causal mutation [genetic variation]**'], ['causative variants [genetic variant]'], ['phenotype-causing mutation [genetic alteration]']]}\n",
      "-------------------\n",
      "{'82891': [['hairpin structures [structure]'], ['helical breaking protein(s) [protein]'], ['non-helical breaking protein(s) [protein]'], ['small patches forming helices [protein structure]'], ['small helical insertions [protein feature]'], ['knot, hairpin and helical spine features [protein]']]}\n",
      "-------------------\n",
      "{'79708': [['Ubiquitin ligase ATL31 [gene]'], ['Ubiquitin ligase CNI1/ATL31 [gene]'], ['ubiquitin ligase SlATL31 [gene]']]}\n",
      "-------------------\n",
      "{'123995': [['21-nt and 22-nt miR168a/b [gene]'], ['pri-miRNA172d [gene]'], ['mir173 ssRNA [gene]'], ['pri-miR172 precursor [gene]'], ['pri-miR173 [metabolite]'], ['PhmiR172 precursors [gene]'], [\"miR173-5'A [gene]\"], ['primary transcripts of miR27319 [gene]'], ['21 nt forms of miR173 [gene]'], ['21 nt forms of miR472 [gene]'], ['miR173/miR173* sequences [gene]'], ['MIR173 foldback [gene]'], ['21 nt miR173 [metabolite]'], ['22 nt miR173 sources [gene]'], ['22 nt authentic miR173 [gene]'], ['miR172a backbone [gene]'], ['miR173 precursor [gene]'], ['polyadenylated regions of miR173ts_CHS transcripts [rna]'], ['pri-miR173 [gene]'], ['miR173-induced primary siRNAs [gene]'], ['Pri-miRNA172b [gene]'], ['pre-miR172d [gene]'], ['pre-MIR172b [gene]'], ['MIR172 C-C variant [gene]'], ['C-C mismatch in miRNA/miRNA* duplex of MIR172A [gene]'], ['miR171_2M decoy [gene]'], ['miR171_3B decoy [gene]'], ['**pri-miR172s [gene]**', 'pri-miRNAs 172 [gene]', 'pri-miR172 transcripts [gene]'], ['mature forms of miRNAs171b&c [gene]'], ['miR173-precursor sequence [genetic element]'], ['miR173CH3/miR173*CH3 [rna]'], ['pri-miR172c [gene]'], ['pri-miRNA172E [gene]'], ['single deoxyadenylate to miR173 [gene]'], [\"2'Ome MIR168 [metabolite]\"], ['putative miR172 recognition site [description]'], ['unique mature miR172 sequences [mirna]'], ['production of miR25 [rna]'], ['mature miR171 sequences [gene]'], ['pri-miR172b [mirna]'], ['spliced pri-miR172b [rna]'], ['pri/pre-miR173 [gene]'], ['transcription of miR172b [gene]'], ['22 nt miR173 [metabolite]'], ['21 and 22 nt forms of miR173 [gene]'], ['miR172a backbone [rna molecule]'], ['aberrant ~20, 23 and 24 nt miR173 species [metabolite]'], ['transcription of miR-17-92 [gene]'], ['mature sequence of miR172 [gene]'], ['mature miR172c sequence [gene]'], ['miR173ts_CHS transcripts [rna]'], ['mature sequence of gma-miR172a [gene]'], ['pri-miR171 [metabolite]'], ['transcription of MIR172 gene(s) [process]'], ['MIR827 and MIR172A exceptions to pairing rule [gene]'], ['C-G variant in MIR172A precursor [gene]'], ['generation of two small RNA variants from MIR172A precursor [phenotype]'], ['wild-type MIR172A precursor [gene]'], ['pre-miRNA172 [metabolite]'], ['Mtpri-miR171b [gene]'], ['five precursors [gene]'], ['five pri-miR172s [gene]'], ['four pri-miR172s [gene]'], ['nine pri-miR172s [gene]'], ['natural precursor structures of ath-miR172a [gene]'], ['pri-miRNA 172a (pri-miR172a) [gene]'], ['processing of primary microRNA172 transcripts (pri-miR172) [gene]'], ['tasiRNAs generated by the miR173 cascade [gene]'], ['mature form of miR1073 [gene]']]}\n",
      "-------------------\n",
      "{'49031': [['Promoter substitution [mutant]'], ['Negative regulation of double mutations of BRFs [gene]'], ['mutation causing cosuppression [mutation]'], ['multi-copy suppression in mal3Delta strains [mutant]'], ['epistatic relationship to ban [mutant]']]}\n",
      "-------------------\n",
      "{'103397': [['**Rhizobium species [organism]**', 'Rhizobium bacteria [organism]', 'rhizobacteria from the Rhizobium genus [organism]'], ['N2-fixing Rhizobia [organism]'], ['Legume nodulating rhizobia [bacterium]'], ['legume-nodulating rhizobia [organism]'], ['Rhizobiacea family [organism]'], ['NifH rhizobia [organism]'], ['Rhizobiaceae members [organism]'], ['rhizobia proteobacteria [organism]'], ['Rhizobium lineage of the Alphaproteobacteria [organism]'], ['Rhizobiaceae [community]']]}\n",
      "-------------------\n",
      "{'137362': [['overexpression of CO1 and CO2 [gene]'], ['overexpression of IbTCP11/17 [gene]'], ['Overexpressing CpMAX1a [gene]'], ['Overexpression of ScCOX11 [gene]'], ['Overexpression of COX11 variants [gene]'], ['overexpression of any allele of CG_2 [gene]'], ['Overexpression of CaOXR1/CaRAV1 [gene]']]}\n",
      "-------------------\n",
      "{'165859': [['peaks for carboxylate (COO-) asymmetric stretching vibrations [phenotype]'], ['peaks for carboxylate (COO-) symmetric stretching vibrations [phenotype]'], ['bands for degree of esterification of pectic polysaccharides [phenotype]'], ['band for pectin [metabolite]'], ['34% S4B intensity [phenotype]'], ['18% S4B levels [phenotype]'], ['~1745 cm-1 [phenotype]'], ['1405 cm-1 [phenotype]'], ['1455 cm-1 [phenotype]'], ['840 to 857 cm-1 [phenotype]'], ['842 cm-1 [phenotype]'], ['1745 cm-1 and 849 cm-1 [phenotype]'], ['1745 cm-1 [phenotype]'], ['1610 cm-1 [phenotype]'], ['849 to 853 cm-1 [phenotype]'], ['849/851 cm-1 [phenotype]'], ['1745 cm-1 peak [phenotype]'], ['852 cm-1 (20 DAI) and 853 cm-1 (26 DAI) [phenotype]'], ['857 cm-1 after infection [phenotype]'], ['853 cm-1 (20 DAI) and 850 cm-1 (26 DAI) [phenotype]']]}\n",
      "-------------------\n",
      "{'144071': [['**MYC recognition site responsive to drought, ABA, and cold signals [gene]**', 'drought-, ABA- and cold-responsive MYC recognition site [gene]']]}\n",
      "-------------------\n",
      "{'126527': [['point centromeres [phenotype]'], ['chromatids [phenotype]'], ['SC central region [phenotype]'], ['centromere architecture [phenotype]'], ['Monocentric chromosomes [phenotype]'], ['chromosome bodies with centromeres [phenotype]'], ['Chromocenter assembly [phenotype]'], ['homologs within SC central element [phenotype]'], ['a single site [phenotype]', '**centromere clustering [phenotype]**'], ['scattered distribution of centromeres [phenotype]'], ['functional centromeres [phenotype]'], ['complex megabase-size centromeres [phenotype]', '**complex centromeres [phenotype]**'], ['human centromeric chromatin [phenotype]'], ['nuclear chromocenters [phenotype]'], ['centromere-specific localization pattern [phenotype]'], ['extra centromere foci [phenotype]'], ['even distribution across noncentromeric region of nuclear genome [genomic feature]'], ['centromere sequences [phenotype]'], ['middle regions of chromosomes [phenotype]'], ['dicentric-like structure [phenotype]'], ['nuclear distributions of centromeres [phenotype]'], ['Holocentric chromosomes [phenotype]']]}\n",
      "-------------------\n",
      "{'108939': [['Hypermethylation of homologs [metabolite]'], ['hypomethylation of gene(s) [gene]'], ['hypomethylation of gene(s) fragments [gene region]'], ['hypomethylation of genic sequences [metabolite]'], ['lower methylation levels in maternal alleles [epigenetic mark]'], ['hypermethylation of gene(s) [metabolite]'], ['hypermethylation of specific gene(s) [gene]'], ['enhanced expression of the reporter [phenotype]']]}\n",
      "-------------------\n",
      "{'46154': [['old3-1 plant(s) [mutant]'], ['Adult abs3-1D plant(s) [mutant]'], ['dcl4-2/ago1-57 plant(s) [mutant]'], ['bob1-3 plant(s) [mutant]'], ['fvesep3 fruit [mutant]'], ['15-d-old rms1 plant(s) [mutant]'], ['Identification of one plant(s) (B15-1) [mutant]'], ['**genuine acl1-1 plant(s) [mutant]**', 'original acl1-1 plant(s) [mutant]'], ['3-week-old asil1-1 plant(s) [mutant]'], ['smo3-D plant(s) [mutant]'], ['mature prf1-1 plant(s) [mutant]'], ['old3-2/oas-a1.1 plant(s) [organism]'], ['cyp78a5 parents at adult stage [organism]']]}\n",
      "-------------------\n",
      "{'30902': [['Field phi [process]'], ['introducing a matrix [mathematical method]', '**Matrix [mathematical method]**'], ['Jacobians [mathematical concept]'], ['simple mathematical calculations [technique]'], ['inner product [method]'], ['8 x 8 order eight magic square [concept]'], ['complex numbers [mathematical concept]'], ['vector calculation [mathematical concept]'], ['vector calculation of complex numbers [process]'], ['constants [mathematical concept]'], ['Matrix [concept]'], ['integral [metric]'], ['above calculation [measurement]']]}\n",
      "-------------------\n",
      "{'12508': [['Leaf mechanical damage [phenotype]'], ['cauline leaf damage [phenotype]'], ['actual herbivory on leaf [phenotype]', '**Leaf herbivory [phenotype]**'], ['**leaf damage [phenotype]**', 'Foliar damage [phenotype]'], ['Damage on young leaf [phenotype]'], ['leaf damage levels [phenotype]'], ['herbivory on rockcress rosette leaf [phenotype]', '**rockcress rosette herbivory [phenotype]**'], ['total leaf damage [phenotype]'], ['evaluate plant(s) damage in detached leaf [phenotype]'], ['absolute leaf damage [phenotype]'], ['damage on mature leaf [phenotype]'], ['leaf holes [phenotype]'], ['~80% leaf damage [phenotype]'], ['~70% leaf damage [phenotype]'], ['~60% leaf damage [phenotype]'], ['~85% leaf damage [phenotype]'], ['~75% leaf damage [phenotype]'], ['~65% leaf damage [phenotype]'], ['~95% leaf damage [phenotype]'], ['~97% leaf damage [phenotype]'], ['leaf burning [phenotype]'], ['approximately 10% damage [phenotype]'], ['varied leaf damage [phenotype]'], ['foliar phytotoxicity [phenotype]'], ['leaf area damage [phenotype]']]}\n",
      "-------------------\n",
      "{'1979': [['Preglobular embryos [phenotype]'], ['embryos at pre-globular and globular stages [phenotype]']]}\n",
      "-------------------\n",
      "{'105892': [['Genvestigator database [database]', '**Genevestigator database [database]**'], ['PathoPlant database [database]'], ['SIGnAL [phenomenon]'], ['eFP browser [database]'], ['CSB.DB [tool]'], ['PLACE database [tool]'], ['PlantGSEA [database]'], ['PLANTPAN 2.0 database [database]'], ['MapMan classification data [database]'], ['PASTDB [datum resource]'], ['PaGenBase [treatment]'], ['Genevestigator transcriptome database [tool]'], ['DRASTIC [database]'], ['Platform for RIKEN Metabolomics (PRIMe) [database]'], ['PLEXdb database [treatment]'], ['PLANEX database [database]'], ['CORNET platform [tool]'], ['CoP database [database]'], ['PathoPlant database [tool]'], ['Diurnal Database v2.0 [database]'], ['PlaNet co-expression databases [database]'], ['PlantCARE analysis databases [phenotype]'], ['mining transcriptomic databases [method]'], ['MAGI and PlantGDB databases [treatment]'], ['P. edulis transcriptome database [treatment]'], ['PLEXdb [process]'], ['PlantPAN database [technique]'], ['Mapman4 [database]']]}\n",
      "-------------------\n",
      "{'161173': [['130-bp ndhA DNA segment [metabolite]'], ['pyrophosphate moiety of NAD [metabolite]'], ['adenine of NADP+ [metabolite]']]}\n",
      "-------------------\n",
      "{'149073': [['**BnaDofs [gene]**', 'BnaDof gene(s) [gene]', 'Dof gene(s) from B. napus [gene]', 'Dof gene(s) family in B. napus [gene]', 'all BnaDofs [gene]', 'B. napus Dof gene(s) family [gene]'], ['BnaDof protein(s) [protein]'], ['BnaDof117 [gene]'], ['BnaDof30, BnaDof31, BnaDof75, and BnaDof76 [gene]'], ['BnaDof33 [gene]'], ['BnaDof20 [gene]'], ['BnaA07g02590D [gene]'], ['17 BnaDofs [gene]'], ['BnaDof30-31 [gene]'], ['BnaDof75-76 [gene]'], ['BnaDof86-87 [gene]'], ['BnaDof gene(s) pairs [gene]'], ['BnaDof30 [gene]'], ['BnaDof13 [gene]'], ['BnaDof31 [gene]'], ['BnaDof75 [gene]'], ['BnaDof29 [gene]'], ['BnaDof74 [gene]'], ['BnaDof32 [gene]'], ['BnaDof72 [gene]'], ['59 BnaDofs [gene]'], ['BnaDof01 [gene]'], ['BnaDof34 [gene]'], ['BnaDof65 [gene]'], ['BnaDof101 [gene]'], ['BnaDof61 [gene]'], ['BnaDof62 [gene]'], ['BnaDof66 [gene]'], ['BnaDof116 [gene]'], ['BnaDof11 [gene]'], ['BnaDof12 [gene]'], ['BnaDof73 [gene]'], ['BnaDof96 [gene]'], ['117 BnaDofs [gene]'], ['BnaDof22 [gene]'], ['BnaDof44 [gene]'], ['BnaDof71 [gene]'], ['BnaDof85 [gene]'], ['BnaDof40 [gene]'], ['BnaDof54 [gene]'], ['BnaDof55 [gene]'], ['BnaDof [gene]'], ['BnaDof53 [gene]'], ['BnaDof102 [gene]'], ['BnaDof17 [gene]'], ['BnaDof47 [gene]'], ['BnaDof81 [gene]'], ['BnaDof113 [gene]'], ['11 BnaDofs [gene]'], ['10 BnaDofs [gene]'], ['35% of BnaDofs [gene]'], ['28% of BnaDofs [gene]'], ['~18% of BnaDofs [gene]'], ['the least number of BnaDofs [gene]'], ['BnaDof76 [gene]'], ['BnaDof111 [gene]']]}\n",
      "-------------------\n",
      "{'114975': [['**interaction data [method]**', 'information on gene(s) interactions [method]'], ['previously reported interactions [gene]'], ['entire interactome [gene]']]}\n",
      "-------------------\n",
      "{'10932': [['hyperacetylation to the maximum degree [phenotype]'], ['glutathione adduct [phenotype]'], ['acetylated [phenotype]'], ['a single methylene group [phenotype]'], ['succinylated [phenotype]'], ['acetylated and succinylated [phenotype]'], ['monoacetylated [phenotype]'], ['diacetylated [phenotype]'], ['a high degree [phenotype]'], ['disulfide bonded [phenotype]']]}\n",
      "-------------------\n",
      "{'105671': [['interaction mapping [technique]'], ['expression QTL mapping [method]', '**eQTL mapping [technique]**', 'QTL mapping and gene(s) expression analyses [technique]', 'eQTL analysis [tool]', 'QTL/microarray screen [method]'], ['FPKM method [method]'], ['cis-eQTL-mapping approach [method]', '**cis-eQTL mapping [method]**'], ['read number correlated with intergenic signal [technique]'], ['VBQTL model [method]'], ['meta-analysis of A. thaliana eQTL datasets [tool]']]}\n",
      "-------------------\n",
      "{'87588': [['sampling time [treatment]'], ['**duration [treatment]**', 'duration of the treatment [treatment]', 'time frame [treatment]'], ['**treatment time [treatment]**', 'treating time [treatment]'], ['FREQUENCY [treatment]'], ['phosphorylation period [treatment]'], ['timing [treatment]'], ['optimal treatment duration [treatment]'], ['time [experimental condition]'], ['crew time [treatment]'], ['**exposure time [treatment]**', 'exposure duration [treatment]'], ['time under stress [treatment]'], ['wild-type gamma-IR time course [treatment]'], ['intervening time [treatment]'], ['T > 100 [treatment]'], ['developmental window [treatment]'], ['timing and frequency of treatments [treatment]'], ['experimental period [treatment]', 'during the course of this work [treatment]', '**total experiment length [treatment]**', 'time period [treatment]', 'the time of study [treatment]'], ['time of their introduction [treatment]'], ['time [method]'], ['optimum time [treatment]'], ['**specific timing [treatment]**', 'specific times [treatment]'], ['time of induction [treatment]'], ['initial treatment times [treatment]'], ['drawdown times [treatment]'], ['minutes after stimulus application [treatment]'], ['timeframe relevant for the experimental procedure [treatment]'], ['1 h compared with the control [treatment]'], ['determine optimum dosage and timing of application [phenotype]'], ['whole development period [treatment]'], ['timely fashion [treatment]'], ['20 days of selection [treatment]'], ['whole period of mock treatment [treatment]'], ['the same time window [treatment]'], ['time-saving [treatment]'], ['temporal control of the application of the chemical [treatment]']]}\n",
      "-------------------\n",
      "{'89031': [['AGAR [treatment]'], ['**Phytoagar [treatment]**', 'Phyto agar [treatment]'], ['Type of gelling agent [treatment]'], ['Phytagel [treatment]'], ['0.5% phytagel [treatment]'], ['microbial fermentation of Sphingomonas elodea [organism]'], ['Phytagel [medium]'], ['Gellan [medium]'], ['AGAR [gelling agent]'], ['gelling agents [gelling agent]'], ['Proper gelling agents [gelling agent]'], ['Type A agar [gelling agent]'], ['Nacalai agar [gelling agent]'], ['Type E agar [gelling agent]'], ['other Type A agars [gelling agent]'], ['Nacalai agar plates [gelling agent]']]}\n",
      "-------------------\n",
      "{'6394': [['traits VI [phenotype]'], ['Quality attributes of grapes [phenotype]'], ['sour taste [phenotype]'], ['delicious taste [phenotype]'], ['**tea quality [phenotype]**', 'quality of tea [phenotype]'], ['Grape and wine flavour [phenotype]'], ['flavor of rice [phenotype]'], ['flavour compounds [phenotype]'], ['taste [phenotype]'], ['sweetness to bitterness [phenotype]'], ['sharp flavor [phenotype]'], ['wine quality aspects [phenotype]', 'wine characteristics [phenotype]', 'sensory properties within wine [phenotype]', '**wine quality [phenotype]**', 'wine features [phenotype]'], ['olfaction [phenotype]'], ['flavor and medicinal ingredient [phenotype]'], ['distinctive odorant properties [phenotype]'], ['unique flavors to mold-ripened cheeses [phenotype]'], ['unique flavors to Japanese koji [phenotype]'], ['unique flavors to mold-fermented food products [phenotype]'], ['bitterness [phenotype]'], ['most popular non-alcoholic beverage [phenotype]'], ['apple flavor [phenotype]'], ['juice quality [phenotype]'], ['soluble solids [phenotype]'], ['distinctive flavor and aroma [phenotype]'], ['flavoring [phenotype]'], ['berry characteristics [phenotype]'], ['taste quality [phenotype]'], ['flavor of many fruits [phenotype]'], ['eating quality of cooked rice [phenotype]'], ['strawberry fruit flavor [phenotype]'], ['**organoleptic characteristics [phenotype]**', 'organoleptic properties [phenotype]', 'organoleptic appeal [phenotype]', 'sensory parameters made by tasting [phenotype]', 'organoleptic quality [phenotype]', 'sensory qualities [phenotype]'], ['palatable [phenotype]'], ['sweet tasting property [phenotype]'], ['fruit taste [phenotype]'], ['variables related to sweetness perception [phenotype]'], ['grape and wine quality [phenotype]'], ['composition of coffee aroma [phenotype]'], [\"**fruit's sugar content [phenotype]**\", 'sugar contents of fruits [phenotype]'], ['pleasant flavors [phenotype]'], ['purchasing behaviour of consumers [phenotype]'], ['sweet and sour flavour [phenotype]'], ['smoothness and mouth feel of chocolate [phenotype]'], ['tea umami taste [phenotype]'], ['coffee beverage quality [phenotype]'], ['attractive appearance [phenotype]'], ['sweetness [phenotype]'], ['high-intensity sweetness [phenotype]'], ['high levels of sweetness [phenotype]'], ['acidity levels [phenotype]'], ['characteristic smell [phenotype]'], ['standard sour lemon [phenotype]'], ['delicious taste, exotic flavour and nutritional value [phenotype]'], ['distinctive flavor profile [phenotype]']]}\n",
      "-------------------\n",
      "{'49361': [['PIF3-A20 variant [gene]'], ['ADF1-S6phos [mutant]'], ['FIT phospho-mutants [mutant]'], ['EIN3 fragment 1-70 S35A T42A double mutation [mutant]'], ['FUS3S>D [mutant]'], ['At eIF4E-T67V/T91A [mutant]'], ['mEBP1-N3A [mutant]'], ['mEBP1-M3A [mutant]'], ['mEBP1-C4A [mutant]'], ['FER-CA-GFP phospho-mutants [mutant]'], ['AtAMT1;3 phospho-variants [mutant]'], ['AtAMT1;3 T494 phospho-variants [mutant]'], ['AtADF1 phosphorylation mutants [mutant]'], ['Dephosphomimetic mutants of ALMT4 S382 [mutant]'], ['Site-directed mutants of PIF3 [mutant]'], ['phospho-dead FITm mutations [mutant]'], ['EIN3 fragment 1-70 S35A mutation [mutant]'], ['EIN3 fragment 1-70 T42A mutation [mutant]'], ['FER-NA [mutant]'], ['FER-CA [mutant]'], ['3985 phosphopeptides [metabolite]']]}\n",
      "-------------------\n",
      "{'6893': [['Suspension cultures [phenotype]'], ['growing culture [phenotype]']]}\n",
      "-------------------\n",
      "{'47841': [['NopD-NDelta2-53 [gene]'], ['NopD-NDelta2-60 [gene]']]}\n",
      "-------------------\n",
      "{'122438': [['future functional studies [phenotype]', 'further functional investigation [phenotype]', 'future functional analyses [phenotype]', '**further functional studies [phenotype]**'], ['further functional studies [research field]'], ['further functional studies [research]'], ['further functional research [process]'], ['more detailed functional work [study]']]}\n",
      "-------------------\n",
      "{'96464': [['**young seedlings [organism]**', 'Juvenile seedlings [organism]', 'Young seedling [organism]', 'young plant(s) seedlings [organism]', 'Developing seedlings [organism]', 'young plantlets [organism]', 'juvenile seedling [organism]', 'immature seedlings [organism]'], ['developing offspring [organism]'], ['younger parents [organism]'], ['abnormal seedlings [organism]'], ['young seeds [organism]'], ['younger seedlings [organism]'], ['juvenile citrus seedlings [organism]'], ['juvenile poplar [organism]'], ['1-wk-old seedlings [organism]'], ['youngest seedlings [organism]'], ['very young seedlings [organism]'], ['in seedlings [organism]'], ['early seedlings [organism]'], ['weak seedlings [organism]'], ['growing sporophytic seedlings [organism]'], ['young seedling shoots [organism]'], ['entire seedling [organism]'], ['juvenile trees [organism]'], ['newly hatched larvae [organism]'], ['developing pea seedlings [organism]'], ['**young seedlings [organ]**', 'young plantlets [organ]', 'Young seedling [organ]', 'young developing seedlings [organ]'], ['whole seedlings [organ]'], ['remains of young seedlings [organ]'], ['miniature seedling [organ]'], ['early seedlings [organ]'], ['seedlings [developmental stage]']]}\n",
      "-------------------\n",
      "{'6918': [['duration of any given phase [phenotype]', '**phase length [phenotype]**'], ['length of cell cycles [phenotype]'], ['length of time that cells elongate [phenotype]'], ['Maximum cell length [phenotype]', 'average length of differentiated cells [phenotype]', '**cell lengths [phenotype]**', 'spatial gradient of cell lengths [phenotype]', 'cell length distribution data [phenotype]', 'comparatively longer [phenotype]', 'defined length [phenotype]', 'characteristic cell lengths [phenotype]', 'lengths of differentiated cells [phenotype]', 'relatively constant lengths in WT [phenotype]', 'cell length in TD [phenotype]', 'maximal cell length [phenotype]', 'linear functions of cell length [phenotype]'], ['Cell size after division [phenotype]', 'Larger cells [phenotype]', 'large cells [phenotype]', '**Final cell size [phenotype]**', 'more uniform in size [phenotype]', 'target size [phenotype]', 'relatively constant cell size [phenotype]', 'setting stereotyped sizes [phenotype]', 'driving force of cell size [phenotype]', 'characteristic sizes of cells [phenotype]', 'Homeostasis of cell size [phenotype]', 'cell size/growth [phenotype]', 'size homeostasis [phenotype]'], ['geometrical inputs [phenotype]', '**proper size and shape [phenotype]**', 'cell length, width, and surface area [phenotype]', 'final size, shape, and functions [phenotype]', 'preferential dimensions [phenotype]'], ['colony size [phenotype]'], ['nuclear size [phenotype]'], ['**telomere length homeostasis [phenotype]**', 'telomere length changes [phenotype]', 'stability of telomeres [phenotype]', 'display identical telomere length [phenotype]', 'its telomeres [phenotype]', 'telomeres in boundaries [phenotype]', 'stable telomeres [phenotype]', 'quantification of telomere length [phenotype]', 'differences in telomere length [phenotype]', 'longest telomeres [phenotype]', 'positive or negative changes in telomere length [phenotype]', 'maintaining telomere length [phenotype]'], ['number of cells per unit length [phenotype]'], ['shortest distance [phenotype]'], ['ploidy or ploidy-related cell size [phenotype]'], ['microtubule length [phenotype]']]}\n",
      "-------------------\n",
      "{'32180': [['plant(s) SGs [protein complex]'], ['plant-like splicing machinery [machinery]'], ['plant(s) TGN/EE [subcellular compartment]', '**plant(s) trans-Golgi network (TGN)/early endosome [subcellular compartment]**'], ['plant(s) lamina components [protein]', '**plant(s) lamins [protein]**'], ['**The Golgi apparatus in plant(s) cells [organelle]**', 'the plant(s) Golgi apparatus [organelle]'], ['plant(s) Golgi protein(s) [protein]'], ['plant(s) COPII subunits [protein complex]'], ['functional plant(s) COPII subunits [protein complex]'], ['plant(s) Golgi [organelle]'], ['plant(s) lipid rafts [subcellular compartment]'], ['Mobile transcripts [macromolecule]'], ['plant(s) ER [organ]'], ['plant(s) cortical ER network [organelle]'], ['plant(s) mitochondrial protein(s) [protein]'], ['**plant(s) nuclear lamina [protein complex]**', 'plant(s) lamina-like structure [protein complex]', 'plant(s) lamina constituents [protein complex]'], ['plant(s) mRNA splicing machinery [process]'], ['cis-Golgi plant(s) golgins [protein]'], ['plant(s) condensates [structure]'], ['plant(s) TGN [subcellular compartment]'], ['plant(s) trans-Golgi network [organ]'], ['plant(s) TGN components [subcellular compartment]'], ['Translational machinery in plant(s) mitochondria [machinery]'], ['plant(s) mitoribosomes [organelle]'], ['plant(s) cytoplasmic ribosome [protein complex]'], ['plant(s) ER [organelle]'], ['apoplastic sRNAs [rna]'], ['plant(s) mitochondrial transcripts [transcript]'], ['plant(s) RNA GQS [subcellular compartment]'], ['The plant(s) Golgi [organ]'], ['GA-TGNs (Golgi-associated TGNs) [organelle]'], ['plant(s) cellular S-acylation machinery [process]'], ['plant(s) secretory system [subcellular compartment]'], ['plant(s) mitochondrial mRNAs [genetic material]'], ['plant(s) endocytic processes [phenotype]'], ['**plant(s) secretory pathway [subcellular compartment]**', 'secretory pathway in plant(s) [subcellular compartment]', \"plant's secretory pathway [subcellular compartment]\"], ['plasma membrane SUTs [subcellular compartment]'], ['plant(s) Golgi [subcellular compartment]'], ['similar percentages of Golgi protein(s) among plant(s) species [process]'], ['plant(s) cis-Golgi [organelle]'], ['plant(s) nuclear import machinery [organism]'], ['plant(s) P-body complex [subcellular compartment]'], ['entry of VlsPLA2 into plant(s) nucleoplasmic reticulum [process]'], ['mechanisms of Golgi and plasmamembrane-bound glucosyltransferases [protein]'], ['nuclear envelope association in plant(s) [subcellular compartment]'], ['plant-specific envelope targeting mechanism [subcellular compartment]'], ['global plant(s) translation [process]'], ['plant(s) ER-Golgi interface [organ]'], ['plant(s) cells by clathrin-mediated endocytosis [cell type]'], ['trans Golgi of root(s) tip cells [organ]'], ['secretion in planta [process]'], ['plant(s) Golgi apparatus [subcellular compartment]'], ['plant(s) microsomal FAE complexes [protein complex]'], ['Tonoplast transporters [subcellular compartment]'], ['plant(s) mitochondrial BER pathway [subcellular compartment]'], ['plant(s) legumain catalytic domain [subcellular compartment]'], ['nucellar projection cells in barley [organ]'], ['plant(s) Golgi-localized protein(s) [protein]'], ['eukaryotic GluRs [gene]'], ['structure of plant(s) [process]'], ['plant(s) cell secretory pathway [subcellular compartment]'], ['plant(s) mtDNAs [genome]'], ['plant(s) Golgi proteomes [subcellular compartment]'], ['plant(s) P-body [subcellular compartment]'], ['protein(s) body matrix of tobacco endosperm and embryo [subcellular compartment]'], [\"host plant's translational machinery [organism]\"], ['plant(s) COPII coat components [protein]'], ['plastid transcriptome [tissue]'], ['plant(s) golgin [protein]'], ['SYNTAXIN OF PLANTS61 (SYP61) trans-Golgi network compartment [subcellular compartment]'], ['luminal synthesis of plant(s) cell surface components [process]'], ['endoplasmic reticulum markers [marker]']]}\n",
      "-------------------\n",
      "{'6353': [['dimeric nature [phenotype]'], ['distinct recognition specificities [phenotype]'], ['distinct downstream signaling activity [phenotype]'], ['differences in enzymatic output or receptor-protein interactions [phenotype]'], ['capacity to recognize corresponding receptors [phenotype]']]}\n",
      "-------------------\n",
      "{'53420': [['2 sequences [gene]'], ['Several sequences [gene]'], ['oat sequence [gene]'], ['specific sequences [gene]'], [\"intervening sequence and 3'R [gene]\"], ['sequences flanking the intervening sequence [gene]'], ['Highly sequenced gene(s) [gene]'], ['novel sequences [gene]'], ['Sequences from rice [gene]'], ['2743 core sequences [gene]'], ['343 core sequences [gene]'], ['5028 possible amino acid sequences [gene]'], ['incomplete sequenced genome [gene]'], ['four sequences [gene]'], ['Non-redundant sequences [gene]'], ['comparison of these sequences [gene]'], ['20 distinct sequences [gene]'], ['Seven sequence LOGOs [gene]'], ['sequence logos [gene]'], ['Choice of appropriate regulatory sequences [gene]'], ['SAD gene(s) sequences [gene]'], ['99 non-redundant sequences [gene]'], ['different nucleotide sequence lengths of StUSPs [gene]'], ['first caleosin sequence [gene]'], ['HECT domain sequences [gene]'], ['2181 EST sequences [gene]'], ['single genome sequence [gene]'], ['46 group 8-like sequences [gene]'], ['barley sequence [gene]'], ['the same sequence [gene]'], ['ancestral sequences [gene]'], ['intervening sequence [gene]'], ['intact intervening sequence [gene]'], ['more variable than downstream protein(s) sequence [gene]'], ['more single-molecule sequences [gene]'], ['primary sequence [gene]'], ['coding sequences similar to AMT3;3 [gene]'], [\"5'-GAGAG-3' sequence [gene]\"], ['coding sequence of NIC3 [gene]'], ['almost identical sequences in coding regions [gene]'], ['two types of RPA1 sequences [gene]'], ['252 MYB family sequences [gene]'], ['1 of 5 sequences [gene]'], ['3 sequences [gene]'], ['48% of ESL sequences [gene]'], ['entire viral genome sequences [gene]'], ['NRT sequences of rice [gene]'], ['full-length protein(s) sequences [gene]'], ['grapevine sequences [gene]'], ['other available rice sequences [gene]'], ['four monocot sequences [gene]'], ['conserved sequences of TA [gene]'], ['a single sequence [gene]'], ['EXP gene(s) sequences [gene]'], ['signature sequences of domains [gene]'], ['genome sequence databases [gene]'], ['genome sequence database [gene]'], ['five copies of ABRE sequences [gene]'], ['three copies of W-box sequences [gene]'], ['upstream flanking sequence [gene]'], ['1.5% of DeoC sequences [gene]'], ['13 nonexpressed sequences [gene]'], ['sequences containing SSRs [gene]'], ['any other protein(s) sequences in GenBank [gene]'], ['upstream sequences of AGL15 [gene]'], ['any translated sequence in the GenBank database [gene]'], ['lysine-rich coding sequences [gene]'], ['any other DNA or protein(s) sequences [gene]']]}\n",
      "-------------------\n",
      "{'161552': [['**Gln residue [metabolite]**', 'Q residues [metabolite]'], ['Glutamine (Q) [gene]'], ['glutamine [compound]'], ['Glutamine (Gln) [metabolite]', '**Glutamine (Q) [metabolite]**', 'Q (glutamine) [metabolite]'], ['q [amino acid]'], ['lysine (K) to glutamine/threonine (Q/T) shift [metabolite]'], ['Cys and glutamine (Gln) [metabolite]'], ['N-terminal glutamine [metabolite]']]}\n",
      "-------------------\n",
      "{'134561': [['TaBG1 [gene]'], ['AtBGLU30-His6 [gene]'], ['BG1/BGLU18 [gene]'], ['AtBGLU15 [gene]'], ['AT1G66270/BGLU21 [gene]'], ['At1g61820 (BGLU46) [gene]'], ['purified BGAL4 [gene]'], ['BdBGLC2 (Bd2g51280) [gene]'], ['AtBGLU28 tagged with His6 [gene]'], ['BGLU40 [gene]']]}\n",
      "-------------------\n",
      "{'52398': [['miRNA duplex (miRNA: miRNA* double-stranded dimer) [gene]', '**miRNA/miRNA* duplex [gene]**', 'miRNA:miRNA* duplexes [gene]', 'MiRNA:miRNA* duplex [gene]', 'mature miRNA:miRNA* duplex [gene]', 'miRNA:miRNA* double-stranded complex [gene]'], ['miRNA-sibling RNAs [mirna]'], ['miRNA variants [mirna]'], ['miRNA* variants [mirna]'], ['mature miRNA sequences [gene]'], ['amiRNA sequences [gene]'], ['**miRNA:mRNA hybrids [gene]**', 'natural miRNA:mRNA hybrid structures [gene]'], ['miRNA:miRNA* duplexes [rna]'], ['**miRNA:intron interactions [gene]**', 'miRNA:intron pairs [gene]'], ['1912 miRNA:intron pairs [gene]'], ['1912 miRNA:intron interactions [genomic feature]'], ['**MiRNA:miRNA* duplex [srna]**', 'duplex of microRNA (miRNA:miRNA*) [srna]'], ['39 non-redundant anticorrelated miRNA:mRNA target pairs [gene]'], ['miRNA319 [regulatory element]'], ['miRNA binding sites [molecule]'], ['One miRNA [genetic element]'], ['**Functional synergetic miRNA pairs [genetic element]**', 'Synergistic miRNA-miRNA pairs [genetic element]', 'miRNA-miRNA cooperation pairs [genetic element]', 'synergistic miRNA pairs [genetic element]'], ['**miRNA pairs [genetic element]**', 'miRNA-miRNA pairs [genetic element]', 'identified miRNA-miRNA pairs [genetic element]'], ['**miRNA expression data [genetic element]**', 'miRNA expression [genetic element]'], ['differentially expressed MIRNAs [genetic element]'], ['miRNA conservation [genetic element]'], ['miRNA homology [genetic element]'], ['miRNA-miRNA pairs [gene]', '**miRNA pairs [gene]**'], ['conserved miRNA pairs [gene]'], ['SNPs in multiple miRNAs [genetic variation]'], ['**different SNPs in same miRNA sequence [genetic variation]**', 'different SNPs within miRNA [genetic variation]'], ['**SNPs in miRNA sequence [genetic element]**', 'SNPs in miRNA sequences [genetic element]'], ['**SNPs in miRNA binding site [genetic element]**', 'SNPs in miRNA binding sites [genetic element]'], ['SNPs in miRNAs [genetic element]'], ['AGO1-loading 21-nt miRNAs [genetic element]'], ['miRNA/miRNA* [mirna]'], ['canonical miRNAs [genetic element]'], ['long miRNAs [genetic element]'], ['region coding for stem(s) loop of known miRNAs [sequence]'], ['Mature miRNA [sequence]'], ['miRNA sites [genetic element]'], ['5 different nucleotides from miR393 sequences [gene]'], ['cooperative miRNA pairs [gene]'], ['more than one miRNA [genetic element]'], ['coronary artery disease-related miRNA expression data [genetic element]'], ['pairs of differentially expressed miRNAs [genetic element]'], ['miRNA-target pairs [genetic element]'], ['miRNA pairs in response to stress [genetic element]'], ['intra-family miRNA pairs [gene]'], ['inter-family miRNA pairs [gene]'], ['homologous miRNA pairs [gene]'], ['cooperative homologous miRNA pairs [gene]'], ['**SNPs in mature miRNA sequences [genetic variation]**', 'SNPs in mature miRNAs [genetic variation]'], ['SNPs in miRNA binding sites [genetic variation]'], ['multiple miRNA pairs [regulatory motif]'], ['SNPs in miRNA sequences [genetic variation]'], ['27,586 target:miRNA pairs with inverse expression correlation [interaction]'], [\"5' end of miRNA [sequence]\"], ['predicted stability of the miRNA:miRNA(∗) duplex [gene]']]}\n",
      "-------------------\n",
      "{'23033': [['sensitivity to Glu-dependent stomatal closure [phenotype]', '**Glu-induced stomatal closure [phenotype]**', 'Glu-dependent stomatal movement [phenotype]', 'Glu-induced stomatal response [phenotype]'], ['cGMP-dependent stomatal movements [phenotype]'], ['**stomatal opening to 8-Br-cGMP [phenotype]**', '8-Br-cGMP response [phenotype]']]}\n",
      "-------------------\n",
      "{'24163': [['hormone-signaling regulation [phenotype]', '**hormone regulation [phenotype]**', 'hormone-mediated regulation [phenotype]'], ['hormonal regulation and adaptation [phenotype]']]}\n",
      "-------------------\n",
      "{'62676': [['**MBP fusion [treatment]**', 'MBP-fusion protein(s) [treatment]'], ['MBP-PAN fusion protein(s) [protein]', '**MBP-PAN [protein]**', 'MBP-PAN protein(s) [protein]'], ['SlHsfA3-MBP fusion protein(s) [protein]'], ['MBP-JMJ24 fusion protein(s) [protein]'], ['Fusion protein(s) between maltose binding protein(s) (MBP) [protein]', '**MBP fusion [protein]**', 'fusion protein(s) with maltose-binding protein(s) (MBP) [protein]', 'maltose-binding protein(s) fusion [protein]'], ['purified recombinant maltose-binding protein(s) (MBP)-OsPTOX [protein]'], ['Maltose Binding protein(s) (MBP) [protein]']]}\n",
      "-------------------\n",
      "{'28668': [['transport zones [organ]'], ['**regions of active transport [tissue]**', 'transport areas [tissue]', 'tissue locations involved in active transport [tissue]']]}\n",
      "-------------------\n",
      "{'141969': [['GF(EXT3)P reporter [gene]'], ['GFP-based reporters [gene]'], ['WOX8Delta-YFP expression [gene]'], ['ASY3-RFP reporter [gene]']]}\n",
      "-------------------\n",
      "{'38274': [['S45AB [mutant]'], ['117AB [mutant]'], ['9012AB [mutant]'], ['CN12AB [mutant]'], ['CN12A [mutant]'], ['Varuna [mutant]'], ['T1 lines BY1241 [mutant]'], [\"Line '1900264' [mutant]\", \"'1900264' line [mutant]\", \"**'1900264' [mutant]**\"], [\"**Line '1900262' [mutant]**\", \"'1900262' line [mutant]\"], ['V141 [mutant]'], ['wild-type R-o-18 [mutant]'], ['Express 617 [mutant]'], ['BnJAG-5 mutant line [mutant]'], ['BnJAG-33 line [mutant]'], ['O-3 [mutant]'], ['O-3-7 [mutant]'], ['O-line [mutant]'], ['L8 [mutant]'], ['132000B-3 [organism]'], [\"'Y689' [mutant]\"], ['wild-type varieties Aj, Lx, ez1, ey5 [organism]'], ['G53R/T146I [mutant]'], ['M70I/T146I [mutant]'], ['M70I/G53R [mutant]'], ['B. napus line M4 [mutant]'], ['M4 [mutant]'], ['low-GS line (LG) [mutant]'], ['J9709 [organism]'], ['WZ1 line [mutant]'], ['WZ2 line [mutant]'], ['WZ5 line [mutant]'], ['Jia9709 line [mutant]'], ['J9709 [mutant]'], ['BnCK [mutant]'], ['line 33 [mutant]'], ['C634 [mutant]'], ['Line T36 [mutant]'], ['Br I [mutant]'], ['CN12B [mutant]'], [\"golden inner leaf '1900264' line [mutant]\"], ['oil from V141 [metabolite]'], ['white-petal line 11-192 [mutant]'], ['K-5-8 [organism]'], ['O-3-7-5 [mutant]'], ['ultra-hairy K-5-8 and K-6-3 [mutant]'], ['green Rs306 line [mutant]'], ['166g-2 line [mutant]'], ['NI-9 [mutant]'], ['NY7 [mutant]'], ['wild-type line Quantum [mutant]'], ['CT line [mutant]'], ['33A [mutant]'], ['33B [mutant]'], ['33AB [mutant]'], ['Y177-12 [mutant]'], ['P-8 [mutant]'], ['B. napus yellow-seeded line L2 [mutant]'], ['B. napus near-isogenic two-type line 7365AB [mutant]']]}\n",
      "-------------------\n",
      "{'95565': [['The root(s) term plant(s) structure development stage [concept]'], ['**plant(s) root-architecture [organ]**', 'plant(s) root(s) architecture [organ]'], ['architecture and extension of plant(s) root(s) system [function]'], ['appearance of vascular plant(s) with a root(s) system [organism]']]}\n",
      "-------------------\n",
      "{'163157': [['Interfering with flavonoid biosynthesis in A. thaliana [process]'], ['A. thaliana phenylpropanoid pathway [pathway]'], ['FFA detoxification in A. thaliana [phenotype]'], ['mechanisms of small molecules on A. thaliana [process]']]}\n",
      "-------------------\n",
      "{'157066': [['GOA [gene]'], ['gene(s) controlling fruit size [gene]'], ['Large-fruited allele [gene]'], ['Fru [gene]'], ['fruit shape and size biosynthesis pathway gene(s) [gene]'], ['**gene(s) for fruit size/shape [phenotype]**', 'gene(s) related to fruit shape and size [phenotype]']]}\n",
      "-------------------\n",
      "{'170003': [['Golgi-localized xylosyltransferases [enzyme]'], ['Golgi-localized (1,3)-alpha-d-xylosyltransferases [enzyme]']]}\n",
      "-------------------\n",
      "{'5317': [['detrimental effect of stresses [phenotype]'], ['growth-retarding effect [phenotype]'], ['detrimental effects of defense/stress cascade [phenotype]']]}\n",
      "-------------------\n",
      "{'45644': [['SOC1 overexpression [mutant]'], ['Ectopic expression of IiSEP1 [gene]'], ['shb1-D mutation [mutant]']]}\n",
      "-------------------\n",
      "{'145995': [['**basal expression of defense gene(s) [gene]**', 'expression of basal defense gene(s) [gene]'], ['basal defence gene(s) expression [phenotype]'], ['defence gene(s) (AtFRK1) expression [phenotype]'], ['defence-responsive gene(s) expression [phenotype]']]}\n",
      "-------------------\n",
      "{'100049': [['Sunflower [organism]'], ['Superior citrus cultivars [organism]'], ['two O. sativa resistant varieties [organism]'], ['9 additional resistant accessions of O. glaberrima [organism]'], ['All resistant accessions of O. glaberrima [organism]'], ['Resistant Brassica cultivars [organism]'], ['economically important crops [organism]'], ['larvae feeding on resistant accessions [organism]'], ['Resistant tea accessions [organism]'], ['resistant cucumber cultivar [organism]'], ['resistant cultivars of radish [organism]'], ['Resistant sunflower cultivars [organism]'], ['most tolerant accessions [organism]'], ['Indian type [organism]'], ['Low-Cd or high Cd-resistance wheat cultivars [organism]'], ['Solanum demissum [organism]'], ['Major crop plant(s) [organism]'], ['resistant wheat cultivars [organism]'], ['resistant Chinese cabbage cultivars [organism]'], ['Whitefly-resistant lines of tomato [organism]'], ['Whitefly-resistant lines of cabbage [organism]'], ['resistant Cvi [organism]'], ['cacao genotypes [organism]'], ['resistant eggplants [organism]'], ['Resistant bean varieties [organism]'], ['tan spot resistant triticale varieties [organism]'], ['resistant ramie cultivar [organism]'], ['Ozone-resistant accession [organism]'], ['resistance breeding of D. catenatum [organism]'], ['Resistant olive varieties [organism]'], ['Resistant apple cultivars [organism]'], ['resistant/tolerant 11-stack seedling [organism]'], ['Wheat cultivar Xingmin318 (XM318) [organism]'], ['sorghum resistant cultivar SC748-5 [organism]'], ['Seven apricot varieties [organism]'], ['resistance cultivar Red Mexican [organism]'], ['resistant strawberry varieties [organism]'], ['resistant tomato cultivars [organism]'], ['Fusarium resistant cultivar Sumai3 [organism]'], ['maize S. frugiperda-resistant lines [organism]'], ['Indian cucumber accessions [organism]'], ['various field studies on several crop plant(s) [organism]'], ['wheat, rice, maize, and soybean [organism]'], ['organisms resistant to UV or ionizing radiation [organism]'], ['differential resistance to Fusarium oxysporum [organism]'], ['resistant wheat cultivars Hongyou and Chiyacaos [organism]'], ['resistant O. glaberrima accessions [organism]'], ['moderately resistant cultivars Wuhan 1 and Nyubai [organism]'], ['resistant variety Zhengzhi 13 [organism]'], ['resistant sorghum cultivars [organism]'], ['numerous crops [organism]'], ['aflatoxin-resistant peanut varieties [organism]'], ['high aflatoxin resistance peanut cultivar XHXL [organism]'], ['resistant cucumber lines [organism]'], ['barley, rice, tomato, lettuce, pea, pepper and melon [organism]'], ['crops such as cabbage, lettuce, tulip, melon, spinach and banana [organism]'], ['sugarcane genotype resistant to S. scitamineum infection [organism]'], ['HLB-tolerant citrus genotypes [organism]'], ['resistant to other diseases and pests [disease]']]}\n",
      "-------------------\n",
      "{'116121': [['507 unique C-to-U editing sites [phenotype]'], ['75 indels [gene]'], ['76,348 non-redundant insertions [genetic element]'], ['over 600 Cs [gene]'], ['115,000 independent T-DNA insertions [gene]']]}\n",
      "-------------------\n",
      "{'156965': [['Production of Anthocyanin Pigment1 (PAP1) [gene]', '**PAP1/MYB75 [gene]**', 'AtMYB75/PAP1 gene(s) [gene]', 'Defense gene(s) such as MYB-like transcription factor PRODUCTION OF ANTHOCYANIN PIGMENT 1 (PAP1) [gene]', 'PAP1/AtMYB75 [gene]', 'Production of Anthocyanin Pigment-1 (PAP-1) or MYB75 [gene]', 'MYB gene(s) Production of Anthocyanin Pigment 1 (PAP1) [gene]', 'PRODUCTION OF ANTHOCYANIN PIGMENT1 (PAP1/MYB75) [gene]', 'anthocyanin regulator MYB75/PAP1 [gene]'], ['**PAP2/MYB90 [gene]**', 'PAP2 (MYB90) [gene]', 'PAP2 gene(s) [gene]'], ['two alleles of PAP2 (MYB90) [gene]'], ['**PRODUCTION OF ANTHOCYANIN PIGMENT 1/2 (PAP1/2, MYB75/90) [gene]**', 'MYB gene(s) PAP1/PAP2 [gene]'], ['PAP1, PAP2, MYB113, and MYB114 [gene]'], ['C1 and Pl [protein]']]}\n",
      "-------------------\n",
      "{'90299': [['Response to hydrogen peroxide [process]', '**response to oxidative stress [process]**', 'responses to oxidative stress [process]', 'plant(s) responses to oxidative stress [process]', 'ROS stress response [process]', 'response to oxidation stress [process]', 'ROS-induced stress responses [process]', 'plant(s) response to oxidative stress [process]'], ['Antioxidants, heat shock protein(s) (HSPs), and enzymes [concept]'], ['**oxidative stress-mediated induction [process]**', 'activating antioxidation [process]'], ['H2O2 detoxification and stress responses [phenotype]'], ['oxidative stress response in A. thaliana [organism]'], ['GNSR response to oxidative stress [treatment]'], ['regulating resistance to oxidative stress [process]'], ['**genotoxic stress responses [process]**', 'DDR to genotoxic stress [process]'], ['stimulate endogenous oxidative stress [phenotype]'], ['ROS production under salt stress [process]'], ['plastid responses to oxidative stress [process]'], ['integrating metal and oxidative stress responses [process]'], ['plant(s) responses to oxidative challenges [process]'], ['plant(s) response to nitro-oxidative stress [process]'], ['Antioxidative responses [process]'], ['one of the earliest responses of plant(s) cells [process]']]}\n",
      "-------------------\n",
      "{'96226': [['D. nobile leaf [organ]'], ['leaf blades, nodules, and buds [organ]']]}\n",
      "-------------------\n",
      "{'123058': [['intergenic poly(A) sites [metabolite]'], ['canonical poly(A) signal AAUAAA [sequence]'], ['short polyA tail [gene]'], ['22-bp polyA [gene]'], ['6-bp polyA tail [gene]'], ['**15-bp polyA tail [gene]**', '15-bp polyA tract [gene]'], ['50-nt polyadenylation-inhibitory element RNA [gene]'], ['14 to 13-nucleotides polyA [sequence element]'], [\"5'-cap and 3'-polyA tail [gene]\"], ['first 10 nt of rbcS-E9 site 1 poly(A) signal [gene]'], ['poly (A) tail of 39 nt [gene]'], ['composition of mRNA poly(A) signals [gene]']]}\n",
      "-------------------\n",
      "{'173551': [['NDP-sugar pyrophosphorylase synthesis [process]'], ['**oxygenation of RuBP [process]**', 'Oxygenation of ribulose-1,5-bisphosphate [process]', 'Rubisco oxygenation reaction [process]'], ['Conversion of PN to pyridoxine glucosides [phenotype]'], ['PPCK synthesis/degradation [process]'], ['Conversion of phosphoribosylanthranilate [phenotype]'], ['formation of TMP [metabolite]'], ['APR activity [process]'], ['Conversion of erythrose 4-phosphate and phosphoenol pyruvate [process]'], ['phosphorolytic reaction [phenotype]'], ['**PLP formation [process]**', \"formation of pyridoxal 5'-phosphate [process]\", \"Pyridoxal 5'-phosphate (PLP) formation [process]\", 'PLP synthesis [process]'], ['Defects in R5P synthesis [metabolic process]'], ['Conversion of R5P [metabolite]'], ['Formation of pyrophosphate [process]'], ['Restoring Tre6P synthesis [phenotype]'], ['Pro-P5C cycle [process]'], ['reduction of 4-hydroxyphenylpyruvic acid (pHPP) [metabolite]'], ['transamination of pHPP [metabolite]'], ['phosphorylation of riboflavin [process]'], ['PK-catalyzed pyruvate synthesis [phenotype]'], ['PLP-dependent cyclopropanation [process]'], ['conversion of iPRMP [process]'], [\"reverse phosphorylation to 5'-monophosphate [process]\"], ['TTP synthesis [process]'], ['triose-P to RuBP [process]'], ['synthesis of 5-phosphoribosyl 1-pyrophosphate [process]'], ['phosphorylation of ribulose 5-phosphate [reaction]'], ['PEP-dependent reactions [process]'], ['VTE5/VTE6 phytol phosphorylation [phenotype]'], ['formation of P-hydroxy Phenyl Pyruvate (HPP) [chemical reaction]'], ['rapid anaplerotic PEP carboxylation [process]'], ['anaplerotic PEP carboxylation [process]'], ['conversion of PEP to pyruvate [metabolite]'], ['2-PG production [process]'], ['biosynthesis of PRPP [metabolite]'], ['**PMP and PNP into PLP [process]**', 'conversion of phosphorylated forms of PN and PM into PLP [process]'], ['synthesis of TPS and TPP [process]'], ['formation of 2-phosphoglycolate (2PG) [metabolite]'], ['ribose-5-phosphate formation [reaction]'], [\"conversion of adenosine to 5'AMP [process]\"], ['transfer of shikimate-3-phosphate [process]'], ['carboxylation of phosphoenolpyruvate [process]'], ['elongation of ADP-ribose polymers [process]'], [\"3'-phosphate termini to 3' hydroxyl [process]\"], ['poly(ADP-ribose) synthesis [process]'], ['ATP-dependent phosphorylation of ribulose 5-phosphate (Ru5P) [reaction]'], ['conversion of 3-PHP to 3-phosphoserine [process]'], ['SBP formation reaction [reaction]'], ['synthesis of PRPP from ribose 5-phosphate and ATP [metabolite]'], ['Rubisco oxygenase reaction [reaction]'], ['**conversion of pyruvate to phosphoenolpyruvate [metabolite]**', 'conversion of pyruvate to phosphoenolpyruvate (PEP) [metabolite]', 'formation of phosphoenolpyruvate [metabolite]'], ['production of pyruvic acid from phosphoenolpyruvate [metabolite]'], ['ping-pong bi-bi kinetic mechanism [mechanism]'], ['thiamin salvage activity [function]'], ['ThDP synthesis [metabolite]'], ['Phosphorylation to ribose-5-phosphate [metabolic process]'], ['the mechanism [phenotype]'], ['dephosphorylation to produce trehalose [process]'], ['reversible interconversion of ribulose-5-phosphate and ribose-5-phosphate [metabolite]'], ['coupling of pyrimidine and thiazole [metabolite]'], ['ATP-dependent conversion of ribulose-5-phosphate (Ru5P) to ribulose-1,5-bisphosphate [reaction]'], ['biosynthesis of pyridoxal-5-phosphate [metabolite]'], [\"3'\\t5' phosphorolysis [process]\"]]}\n",
      "-------------------\n",
      "{'80941': [['Ectopic expression of conjugation system components [process]'], ['Disruption of ATG12-ATG5 conjugation pathway [protein complex]']]}\n",
      "-------------------\n",
      "{'9281': [['relative intensity values [phenotype]'], ['relative noise in impedance [phenotype]'], ['relative noise in phase shift [phenotype]'], ['relative ETR [phenotype]'], ['ratio change [phenotype]'], ['ratio of peak areas [phenotype]'], ['RQ value [phenotype]'], ['rbe [phenotype]', '**relative biological effect (RBE-value) [phenotype]**'], ['relative rate [phenotype]'], ['relative ratio of abundance [phenotype]'], ['relative amplitudes [phenotype]'], ['log expression ratio [phenotype]'], ['ratio between the value of each magnitude at peak induction [phenotype]'], ['RARf [phenotype]'], ['ambient CO2 ratio [phenotype]'], ['baseline method [phenotype]'], ['proportion of duplicates with response retention and loss [phenotype]'], ['observed and predicted amounts of CO and FT [phenotype]'], ['minimum ratios [phenotype]'], ['Self-similarity ratio [phenotype]'], ['ratio of productive and abortive processes [phenotype]'], ['relative noise in measured impedance and phase shift values [phenotype]'], ['ratio between angular coefficient of calibration curve in matrix and in solvent [phenotype]'], ['lnRR [phenotype]'], ['relative expression strength [phenotype]'], ['comparison with normal stiffness [phenotype]'], ['fold-change of each term [phenotype]'], ['phase and RAE means [phenotype]'], ['that measured with AUROCs [phenotype]'], ['relative rate tests [phenotype]'], ['ratio of effective sizes [phenotype]'], ['baseline FI [phenotype]'], ['relative concentrations in sample [phenotype]'], ['average RFI of Variovorax [phenotype]'], ['excellent dynamic ratio range [phenotype]'], ['ratio of lifetime components [phenotype]'], ['left ventricular ejection fraction (LVEF) [phenotype]']]}\n",
      "-------------------\n",
      "{'104768': [['specialized microorganisms [organism]'], ['microniches for exploitation by diverse micro-organisms [organism]']]}\n",
      "-------------------\n",
      "{'54324': [['transgenic plant(s) with 35S:BcYAB3-GUS [mutant]'], ['transgenic pER8::cMYC-MKK7 plant(s) [mutant]'], ['Transgenic plant(s) expressing 35S::MycCIB2 [mutant]'], ['Transgenic plant(s) expressing 35S::MycCIB4 [mutant]'], ['Transgenic plant(s) expressing 35S::MycCIB5 [mutant]'], ['Transgenic plant(s) expressing 35S::MycCIL1 [mutant]'], ['Transgenic plant(s) expressing 35S::MycCIB3 [mutant]'], ['Transgenic plant(s) expressing 35S::MycCIL2 [mutant]'], ['transgenic plant(s) with pCLE19:CLE19 G6T:tCLE19 construct [mutant]'], ['Transgenic plant(s) (35S::Thi2.4) [mutant]'], ['p35S:mTCP24 [transgenic plant]'], ['transgenic plant(s) containing p35S::AtBMI1C [mutant]'], ['transgenic cef3pCEF3::CEF3 plant(s) [mutant]'], ['Super:CBF3-MYC transgenic plant(s) [mutant]'], ['transgenic pC4H::F5H plant(s) [mutant]'], ['DR5::LUC transgenic plant(s) [mutant]'], ['INOpro:MEE45 transgenic plant(s) [mutant]'], ['transgenic plant(s) expressing pRAD51::DMC1g [mutant]'], ['Transgenic plant(s) expressing pRAD51::DMC1-II3A [mutant]'], ['transgenic plant(s) of L1-4, S21, S3 and 35S::OAR1 [mutant]'], ['transgenic plant(s) (35S::spermine synthase SPMS) [mutant]'], ['plant(s) carrying pTIR1:tir1 D170E/M473L-Myc transgene [mutant]'], ['Transgenic CaMV 35S::RAN1 plant(s) [mutant]'], ['WT, myb61 loss-of-function mutants and 35S::MYB61 gain-of-function transgenic plant(s) [mutant]'], ['transgenic plant(s) carrying CCR2p::LUC and CCA1p::LUC [mutant]']]}\n",
      "-------------------\n",
      "{'153755': [['mammalian SNAP25 [gene]'], ['AtSNAP29 [gene]'], ['HsSNAP34 [gene]'], ['BnaSNAP31 [gene]'], ['BnaSNAP34a [gene]'], ['BnaSNAP34b [gene]'], ['AhSNAP33 [gene]'], ['BnaSNAP38 [gene]'], ['BnaSNAP30 [gene]']]}\n",
      "-------------------\n",
      "{'63041': [['LC/MS shotgun proteomic analysis [process]'], ['Label-free LC-MS/MS proteomics [technique]'], ['using both LC-MS/MS and combination of 2-D DIGE with ProQ Diamond staining [technique]']]}\n",
      "-------------------\n",
      "{'49218': [['Mutants defective in chromosomal break repair [mutant]'], ['Lesions in the recombination machinery [mutant]'], ['mutations in chromosome-segregation machinery [mutant]'], ['abnormal homologous recombination repair in Drosophila [process]'], ['deletions on chromosome 6 [genetic alteration]']]}\n",
      "-------------------\n",
      "{'29519': [['**Mitophagy [process]**', 'Selective mitochondria recycling [process]', 'autophagic degradation of mitochondria [process]', 'Senescence-induced breakdown of mitochondrial vesicles [process]', 'autophagic transport of mitochondria [process]', 'mitochondrial autophagy [process]', 'adaptor for autophagy of mitochondria [process]', 'Targeting of mitochondria for autophagic degradation [process]'], ['**partial autophagy of chloroplast components [process]**', 'distinct type of chloroplast-targeted autophagy [process]'], ['membrane-dependent autophagy pathway [process]'], ['ATG-dependent vesicles [process]'], ['LON2- and autophagy-dependent pathways [process]'], ['Selective targeting of mitochondria [subcellular compartment]'], ['autophagic engulfment [process]'], ['mitochondrial autophagic processes [phenotype]'], ['targeted energy-organellar autophagy [process]'], ['**selective autophagy of ER [process]**', 'selective autophagy of the endoplasmic reticulum (ER) [process]', 'selective autophagy of the ER [process]'], ['lysosomal-autophagic pathway [signaling pathway]'], ['SGS3 and granule autophagy [phenotype]'], ['granule autophagy [process]'], ['Parkin-dependent mitophagy processes [phenotype]'], ['**organelle-specific autophagy [process]**', 'organelle-targeting autophagy [process]'], ['both endosomal sorting and autophagic degradation [process]'], ['autophagic degradation of mitochondria [organ]'], ['autophagy-related membrane targeting [process]'], ['autophagy-dependent vesicular trafficking [process]'], ['autophagic body breakdown [process]'], ['induction of piecemeal autophagy [process]'], ['selective autophagy of damaged organelles [process]'], ['Bulk degradation of the cytosol [tissue]'], ['autophagic degradation pathways [process]'], ['phosphatidylinositol 3-phosphate-dependent manner [process]'], ['entire organelle autophagy [intracellular process]']]}\n",
      "-------------------\n",
      "{'87227': [['direct training [treatment]'], ['pretrained models [treatment]', 'supervised pre-training [treatment]', '**pretraining [treatment]**'], ['hyperparameter tuning [treatment]'], ['Deep Conditional GANs [treatment]'], ['fast incremental training [treatment]'], ['testing [treatment]']]}\n",
      "-------------------\n",
      "{'115288': [['**suppressors of transgene silencing [phenotype]**', 'anti-silencing factors [phenotype]'], ['obvious candidate factors involved in transgene silencing [phenotype]'], ['compounds that release LUC silencing [phenotype]'], ['screens using silenced transgene reporters [phenotype]'], ['marker of gene(s) silencing [phenotype]'], ['trigger of transgene silencing [phenotype]'], ['problems of potential transgene silencing and regulatory restrictions of GM wheat lines [phenotype]']]}\n",
      "-------------------\n",
      "{'28875': [['**Cellular redox state [process]**', 'redox state of the cell [process]', 'cellular redox condition [process]', 'cellular redox-state [process]', 'cellular redox [process]', 'cellular redox conditions [process]', 'cell redox state [process]'], ['**Cellular redox homeostasis [process]**', 'maintenance of cellular redox equilibrium [process]', 'maintenance and regulation of cellular redox state [process]', 'maintenance of intracellular redox homeostasis [process]', 'intracellular redox homeostasis [process]', 'maintaining cell redox homeostasis [process]', 'maintenance of redox homeostasis [process]', 'maintaining redox homeostasis [process]', 'maintaining cellular redox homeostasis [process]', 'intracellular redox equilibrium [process]', 'maintaining the redox state [process]'], ['**redox-status of the cellular environment [process]**', 'differing redox status [process]'], ['**Formation and removal of ROS [process]**', 'cellular ROS production and processing [process]', 'ROS-processing capacity [process]', 'ROS localization [process]', 'regulation of ROS-generating machinery [process]'], ['ER redox homeostasis [process]'], ['cell rescue and/or cell redox homeostasis [process]'], ['**Cellular redox changes [process]**', 'redox fluctuations [process]', 'alterations in cellular redox state [process]', 'restoration of cellular redox state [process]'], ['**cellular redox regulation [process]**', 'regulation of redox status [process]', 'regulating the intracellular redox environment [process]', 'regulating redox balance [process]'], ['modulation of redox environment [cell type]'], ['flexibility to compatible redox reactions [process]'], ['regulation of electron transport [process]'], ['adjustment of cellular redox potential [process]'], ['intercellular redox state [process]'], ['maintenance of redox ability of cells [process]'], ['redox state of organelles [process]'], ['intercellular redox homeostasis [process]'], ['redox homeostasis in the cell [subcellular compartment]'], ['maintaining redox homeostasis in plastids [process]'], ['redox status of different cellular compartments [process]']]}\n",
      "-------------------\n",
      "{'147280': [['**products of stress-responsive gene(s) [gene]**', 'Products of stress responsive gene(s) [gene]'], ['stress-responsive enzymes [gene]'], ['stress-responsive genes/proteins [gene]'], ['Stress-associated protein(s) [gene]'], ['**Stress responsive marker gene(s) [gene]**', 'stress-responsive marker gene(s) [gene]'], ['**stress-responsive protein(s) [gene]**', 'Stress response protein(s) [gene]'], ['stress-responsive protein(s) kinases [gene]'], ['differentially expressed extensin gene(s) [gene]'], ['stress-responsive gene(s) transcripts [gene]'], ['key stress-responsive proline gene(s) [gene]']]}\n",
      "-------------------\n",
      "{'121643': [['**Unstructured regions [phenotype]**', 'disorder distribution [phenotype]', 'variations in frequency of disordered segments [phenotype]', 'different degrees of disorder [phenotype]'], ['different positions [phenotype]'], ['five different scrambled versions [phenotype]'], ['scrambled versions [phenotype]'], ['diverse structures [genetic condition]'], ['variance of higher-order structure [phenotype]', 'various structures [phenotype]', 'different secondary and tertiary structures [phenotype]', 'different regions of structure space [phenotype]', '**different three-dimensional structures [phenotype]**'], ['DNA structural conformations [phenotype]'], ['alternative secondary structures [phenotype]'], ['large variance in protein(s) tertiary structures [gene]'], ['multiple integer sizes [phenotype]'], ['largest differences in shape [phenotype]'], ['greater proportion of alpha helical regions [phenotype]'], ['contradictions [description]'], ['**different conformation [phenotype]**', 'distinct conformations [phenotype]', 'alternative folding conformation [phenotype]'], ['discrete folding pathways [phenotype]'], ['variations in lengths of secondary structure elements [phenotype]'], ['slight variation in 3D structure [phenotype]'], ['various complex permutations [phenotype]'], ['different charge distribution [phenotype]'], ['noticeable differences at the level of predicted three-dimensional structures [phenotype]'], ['more diversity in topology [phenotype]'], ['different subunit composition [phenotype]'], ['random coil values [phenotype]'], ['three different multi-domain arrangements [phenotype]'], ['**different domain architectures [phenotype]**', 'different domain organizations [phenotype]'], ['substantially larger fluctuations around the mean geometry [phenotype]'], ['different topologies [phenotype]'], ['conformational differences in loop regions [phenotype]'], ['longer loops [phenotype]'], ['higher random ratios [phenotype]'], ['mixed populations [phenotype]']]}\n",
      "-------------------\n",
      "{'88615': [['Endophytic interaction [process]'], ['symbiotic interaction between soybeans and rhizobia [process]'], ['optimal mycorrhizae [process]'], ['beneficial interaction with Ct [process]'], ['beneficial interaction [process]'], ['closer interaction between plant(s) [organism]'], ['Legume-rhizobium symbiosis [treatment]', 'common bean-R. tropici symbiosis [treatment]', 'rhizobial symbiotic conditions [treatment]', '**symbiosis with rhizobia [treatment]**', 'interaction with rhizobia [treatment]', 'rhizobial symbioses [treatment]', 'Rhizobium tropici symbiotic conditions [treatment]', 'symbiotic interaction with rhizobia [treatment]'], ['establishment of S. indica [treatment]'], ['symbiotic condition [treatment]'], ['symbiotic access [treatment]'], ['known mycorrhizal symbioses [treatment]'], ['establishment of AM symbiosis [interaction]'], ['Rice-Bradyrhizobium (RB) [interaction]'], ['all plant-diazotroph combinations [interaction]']]}\n",
      "-------------------\n",
      "{'2018': [['regulation of endosperm formation [phenotype]'], ['**regulation of endosperm development [phenotype]**', 'regulating endosperm development [phenotype]'], ['regulation of early endosperm development [phenotype]'], ['rice endosperm development [phenotype]'], ['endosperm development in rice and maize [phenotype]'], ['rice seed and endosperm development [phenotype]'], ['**maize endosperm development [phenotype]**', 'endosperm development in maize [phenotype]'], ['barley endosperm development [phenotype]'], ['canola endosperm development [phenotype]'], ['grape endosperm development [phenotype]'], ['**starchy endosperm development [phenotype]**', 'development of starchy endosperm [phenotype]'], ['hydrolyzing endosperm starch [phenotype]'], ['coordination of endosperm [phenotype]'], ['maturation of endosperm [phenotype]']]}\n",
      "-------------------\n",
      "{'131699': [['ORYsa;Sultr1;1 [gene]'], ['ORYsa;Sultr4;1 [gene]'], ['OsaSULTR2;1 [gene]'], ['OsaSULTR2 [gene]'], ['OsaSULTR3;4 [gene]', '**OsSULTR3;4 [gene]**'], ['OsSULTR3;3 [gene]'], ['LjSULTR1;2 [gene]']]}\n",
      "-------------------\n",
      "{'83032': [['uneven distribution of charge [phenotype]'], ['non-ideal geometry [protein complex]']]}\n",
      "-------------------\n",
      "{'139101': [['Expression of PRR9 [gene]', '**PRR9 expression [gene]**'], ['RVE8 expression [gene]'], ['PRR9 and ELF4 expression [gene]'], ['PRR5 transcript levels [gene expression]'], ['PRR9 transcript levels [gene expression]', '**PRR9 expression [gene expression]**'], ['RVE8 expression [phenotype]'], ['PRR9pro-mediated gene(s) expression [phenotype]'], ['PRR9/7 expression [phenotype]'], ['PRR7 and PRR9 expression [gene]'], ['PRR9 transcription activity [process]'], ['modified pattern of PRR9 expression [gene]'], ['PRR7 expression levels [phenotype]']]}\n",
      "-------------------\n",
      "{'74072': [['41 AtRALF precursor protein(s) sequences [protein]'], ['81 darkness-responsive protein(s) [protein]'], ['65 AtWRKY protein(s) [protein]'], ['33 protein(s) records [record]'], ['17 RING protein(s) [protein]'], ['at least 19 SR protein(s) [protein]'], ['eight different chromosomal HMGB protein(s) [protein]'], ['21 different PKs [protein]'], ['16 candidate protein(s) [metabolite]'], ['176 protein(s) [protein]'], ['eight La or La-like protein(s) [protein]'], ['21 members [protein]'], ['11 out of 18 SR protein(s) [protein]'], ['7 WDR protein(s) [protein]'], ['36 peptide sequences [protein]'], ['17 members [protein]'], ['at least 12 PDI-related protein(s) [protein]'], ['18 subfamilies [protein family]'], ['21 HSF representatives [protein]'], ['nine predicted components [protein complex]'], ['18 SRs [protein]'], ['34 different isoforms [protein]'], ['three SEC24 isoforms [protein]'], ['at least nine members of the group I WRKY protein(s) [protein]'], ['at least 12 PDI members [protein]'], ['thirteen 14-3-3 isoforms [protein]'], ['four SDG protein(s) [protein]'], ['protein(s) sequences of 144 cotton ARFs [protein]'], ['16 DoIAAs [protein]'], ['nine functional AGO protein(s) [protein]'], ['more than 21 ARF protein(s) [protein]'], ['24 PATs [protein]'], ['seven VSRs [protein]'], ['15 consensus YSPTSPS repeats [protein domain]'], ['26 variant YSPTSPS repeats [protein domain]'], ['four to eight ARM repeats [protein]'], ['over 20 formin protein(s) [protein]'], ['at least 24 PATs [protein]'], ['four different forms of TOM20 [protein]']]}\n",
      "-------------------\n",
      "{'168322': [['over-accumulation of ACC [metabolite]'], ['Overaccumulation of vitamin B6 [metabolite]'], ['m-tyrosine toxicity [phenotype]'], ['xanthosine accumulation [metabolite]'], ['Lutein overaccumulation [metabolite]'], ['accumulation of unfolded protein(s) [metabolite]'], ['over or under accumulation of SUMO conjugates [metabolite]'], ['allantoin accumulation [metabolite]'], ['BCAA over-accumulation [metabolite]'], ['**methionine overaccumulation [metabolite]**', 'over-accumulation of Met [metabolite]', 'over-accumulation of methionine [metabolite]'], ['overaccumulation of several amino acids [metabolite]'], ['accumulation of unexpected compounds [metabolite]'], ['Accumulation of \\nβ-nootkatol [metabolite]'], ['Cys accumulation [metabolite]', '**over-accumulation of cysteine [metabolite]**', 'excessive L-Cys accumulation [metabolite]'], ['over-accumulation of L [metabolite]'], ['accumulation of detrimental products [metabolite]'], ['accumulation of methylglyoxal (MG) [metabolite]'], ['overaccumulation of sinapoylmalate [metabolite]'], ['over-accumulation of Gly [metabolite]'], ['over-accumulation of Ser [metabolite]'], ['hyperaccumulation of lipid droplets [metabolite]'], ['excess myo-inositol [metabolite]'], ['scytalone accumulation [metabolite]'], ['over-accumulation of thiols [metabolite]'], ['hyperaccumulation of JA-Ile [metabolite]'], ['hyper-accumulation of tyrosine [metabolite]'], ['shoot accumulation of S-methylcysteine (MeCys) [metabolite]']]}\n",
      "-------------------\n",
      "{'103165': [['obligate biotrophic fungi of the Erysiphales [organism]', '**biotrophic powdery mildew fungi [organism]**', 'biotrophic powdery mildew fungus [organism]', 'obligate biotrophic PM fungi [organism]', 'biotrophic powdery mildews [organism]', 'obligate biotrophic powdery mildew fungi [organism]'], ['obligate biotrophic fungal mildew Oidium neolycopersici [phenotype]']]}\n",
      "-------------------\n",
      "{'150131': [['Wheat specific-miRNA tae-miR5048 [gene]', '**tae-miR5048 [gene]**'], ['Tae-miRNA9780 [regulatory element]'], ['tae-miR9780 [gene]']]}\n",
      "-------------------\n",
      "{'95491': [['woody taproot [organ]'], ['Seedling root(s) [organism]'], ['taproots [organ]'], ['growth of the sink [organ]'], ['basal root(s) of potato [organ]'], ['seedling taproots [organ]'], ['newly developed primary root(s) [organ]'], ['seedling root(s) system [organ]']]}\n",
      "-------------------\n",
      "{'136879': [['Downregulation of PPD orthologs in soybean [gene]'], ['ROP4 downregulation [gene]'], ['down-regulation of PPDs orthologs [gene]'], ['downregulation of TCP5 [gene]'], ['downregulation of TCP expression [gene]'], ['downregulation of KNOX I family gene(s) [gene]'], ['down regulation of TaPMR5 [gene]'], ['Downregulation of PtKOR2 [gene]'], ['downregulation of microtubule-associated gene(s) [gene]'], ['Downregulation of TCP cell factor gene(s) [gene]'], ['Knock-downs of BPS-like gene(s) [gene]'], ['down-regulation of GI gene(s) [gene]'], ['Downregulation of CTPS2 [gene]'], ['down-regulation of root(s) CAP POLYGALACTURONASE28 [gene]'], ['down-regulation of the barley AP2-like gene(s) [gene]'], ['Downregulation of PLP [gene]'], ['downregulation of BnPLP1 [gene]'], ['down-regulation of PtoTCP20 [gene]'], ['down-regulation of LTPs [gene]'], ['Down-regulation of NRPB2 [gene]'], ['downregulation of ALC/SPT gene(s) [gene]'], ['GAPCp down-regulation [gene expression]'], ['downregulation of VcPHT1s gene(s) in root(s) system [gene]'], ['slight down regulation of PLC1 [gene]'], ['slight down regulation of PLC2 [gene]'], ['slight down regulation of PLC4 [gene]'], ['reductions in the dose of REV [gene]'], ['downregulation of EXO70B2 [gene]'], ['downregulation of transcription factor CPC (Cpc) [gene]'], ['down-regulation of 252 gene(s) [gene]'], ['down-regulation of CUC family gene(s) [gene]']]}\n",
      "-------------------\n",
      "{'137552': [['synergistic effect in activating PR1 [gene]', 'elevated basal PR1 expression [gene]', 'enhanced induction of PR-1 gene(s) [gene]', '**upregulated PR1 [gene]**', 'increases in PR1 [gene]'], ['increasing PR1 expression [gene expression]']]}\n",
      "-------------------\n",
      "{'144741': [['Induction of JA-regulated gene(s) [gene]', 'induction of JA pathway gene(s) [gene]', 'transcription of some JA responsive gene(s) [gene]', 'Induction of JA-dependent gene(s) [gene]', 'expression of JA response gene(s) [gene]', '**The induction of JA-responsive gene(s) [gene]**', 'induction of JA-related gene(s) [gene]', 'expression of JA-dependent response gene(s) [gene]', 'expression of jasmonic acid-responsive gene(s) [gene]', 'expression of gene(s) in the JA response pathway [gene]', 'expression of JA responsive gene(s) [gene]', 'activation of JA-responsive gene(s) [gene]', 'transcription activation of JA responsive gene(s) [gene]', 'stimulation of JA-related gene(s) [gene]', 'induction of gene(s) in jasmonate (JA) signaling pathways [gene]'], ['induction of JAZ gene(s) [gene]'], ['induction of JA-responsive gene(s) (PDF1.2) [gene]'], ['induction of early JA-responsive gene(s) [gene]'], ['induction of biosynthetic and signalling of JA gene(s) [gene]'], ['wound induction of JA-dependent gene(s) [gene]'], ['**transcriptional activation of JA-responsive gene(s) [process]**', 'activation of JA-responsive gene(s) [process]'], ['JA accumulation through transcriptional activation of key gene(s) in the JA biosynthesis pathway [process]'], ['**induction of JR gene(s) by jasmonic acid [process]**', 'JR gene(s) by jasmonic acid [process]'], ['wound-dependent induction of JR gene(s) [process]'], ['activation of JA response gene(s) [phenotype]', 'expression of jasmonic acid-responsive gene(s) [phenotype]', '**induction of JA-responsive gene(s) [phenotype]**', 'transcription of several gene(s) induced by JA [phenotype]'], ['Wound-induced activation of JA-responsive gene(s) [phenotype]'], ['induction of JA/ethylene-inducible gene(s) [gene family]'], ['inducibility under JA treatment [treatment]'], ['gene(s) induction by jasmonic acid (JA) [metabolite]']]}\n",
      "-------------------\n",
      "{'128564': [['34 MFS gene(s) [gene]'], ['57 MATE gene(s) [gene]'], ['56 MATE gene(s) [gene]'], ['56 MATE gene(s) [gene family]'], ['58 MATE paralogues [gene]'], ['43 AaMADS putative gene(s) [gene]'], ['66 AtMADS box gene(s) [gene]'], ['29 AtMADS gene(s) [gene]'], ['50 AtMADS gene(s) [gene]'], ['17 MADS box gene(s) [gene]'], ['37 nonfunctional type I gene(s) [gene]'], ['43 presumed functional type II gene(s) [gene]']]}\n",
      "-------------------\n",
      "{'95572': [['**modulation of root(s) development [organ]**', 'moderator in root(s) development [organ]', 'complex role in root(s) development [organ]'], ['root(s) organ growth [organ]'], ['root(s) growth apparatus [organ]'], ['regulating root(s) growth [organ]', '**root(s) growth control [organ]**']]}\n",
      "-------------------\n",
      "{'65557': [['CMT3- and KYP-based histone H3K9me2 methylation [process]'], ['maintenance of H3 mK9 [process]'], ['**Clr4 H3 lysine 9 methyltransferase [protein]**', 'H3 lysine 9-specific histone methyltransferase [protein]'], ['**H3 K9 methyltransferase [enzyme]**', 'H3 K9 MTase [enzyme]', 'histone H3 lysine 9 (H3K9) methyltransferase [enzyme]', 'histone H3K9-specific methyltransferase [enzyme]']]}\n",
      "-------------------\n",
      "{'154438': [['SOUL4 and SOUL5 [gene]'], ['GhZHD protein(s) [protein]'], ['37 GhZHD protein(s) [protein]'], ['Get3d partners [gene]'], ['**Me14-3-3 gene(s) [gene]**', 'gene(s) encoding Me14-3-3 protein(s) [gene]'], ['14-3-3 gene(s) [gene]'], ['PePRE1/3 protein(s) [gene]'], ['PRE1 and PRE3 protein(s) [gene]'], ['GH9B1 gene(s) [gene group]'], ['SAUR19-24 [protein]'], ['SMXL3/4/5 protein(s) [gene]'], ['OsGH9 family gene(s) [gene]'], ['GH9C gene(s) [gene]'], ['GH9B gene(s) [gene]'], ['seven DA1-related protein(s) [gene]'], ['SAUR76/78 protein(s) [gene]'], ['multiple Get3 gene(s) [gene]'], ['most Get3 protein(s) [gene]'], ['15 14-3-3 gene(s) [gene]'], ['other 14-3-3 members [gene]']]}\n",
      "-------------------\n",
      "{'3984': [['stress response and sucrose signaling [phenotype]'], ['submergence signaling [phenotype]']]}\n",
      "-------------------\n",
      "{'97119': [['Major potato production areas [geographical location]', 'over 90 countries and regions [geographical location]', 'widely grown in the northern hemisphere [geographical location]', '44% of artificial forests in Japan [geographical location]', 'tropical and subtropical countries of Africa and Asia [geographical location]', '93 countries [geographical location]', 'Southwest and Western Pacific region [geographical location]', '93 tropical countries [geographical location]', '36,000 hectares [geographical location]', 'over 80 countries [geographical location]', 'More than 100 countries [geographical location]', '**world [geographical location]**', '130 countries [geographical location]'], ['70% of peanut production areas [organism]'], ['major regions worldwide [organism]'], ['**almost all banana-growing regions [organism]**', 'many banana-growing regions [organism]'], ['major fruit-growing regions [organism]'], ['citrus producing areas [organism]'], ['domestic and international markets [location]', 'all major citrus-producing areas [location]', 'tropical and subtropical regions of the world [location]', 'more than 48 countries [location]', 'many tropical and subtropical countries [location]', 'Europe and North America [location]', 'northern and southern America [location]', 'tropical and sub-tropical areas [location]', '**around the world [location]**', 'more than 100 countries or regions [location]', 'most papaya growing regions [location]', \"about 40% of Russia's forested areas [location]\"], ['Japan [country]'], ['1.5 million ha in Europe [region]', '**worldwide [region]**'], ['tropical and subtropical regions of China and Southeast Asian countries [phenotype]'], ['at least 48 countries worldwide [process]'], ['more than 80 countries [environment]'], ['more than 20 countries [geographical region]'], ['**the world [geographic location]**', 'tomato-growing areas [geographic location]']]}\n",
      "-------------------\n",
      "{'82483': [['**Persulfidated protein(s) [protein]**', 'persulfidated proteome [protein]', 'S-sulfhydrated protein(s) [protein]', 'protein(s) persulfides [protein]'], ['endogenous persulfidated protein(s) [protein]'], ['adducted protein(s) [protein]'], ['Sulfenylated protein(s) [protein]'], ['specific protein(s) targets [protein]'], ['tyrosine-sulfated protein(s) [metabolite]'], ['glutathionylated protein(s) [protein]']]}\n",
      "-------------------\n",
      "{'149304': [['non-redundant functions of SINE-group KASH gene(s) [process]'], ['most CKX gene(s) [gene]'], ['other plant(s) DXR gene(s) [gene]'], ['KCS gene(s) in M. domestica [gene]'], ['three DXS gene(s) [gene]'], ['several KCS [gene]'], ['diversity of GSK gene(s) [gene]']]}\n",
      "-------------------\n",
      "{'108934': [['hyper-up gene(s) [gene]'], ['hypermethylated epialleles [epigenetic mark]'], ['hypermethylation counterparts [gene]']]}\n",
      "-------------------\n",
      "{'172645': [['Initial step in oxalate degradation pathway [phenotype]', '**first step in oxalate catabolism pathway [phenotype]**'], ['first step in oxalate degradation pathway [process]'], ['the first step in oxalate catabolism [pathway]'], ['first step in alternative pathway of oxalate catabolism [phenotype]'], ['the first step in an alternative pathway [process]'], ['pathway of oxalate degradation [phenotype]']]}\n",
      "-------------------\n",
      "{'111625': [['sequences between -3604 and -3563 bp [gene]'], ['-1286 to -1065 region [dna sequence]'], ['ANAC092 promoter sequence [gene]'], ['Oshsp17.3 promoter [dna sequence]'], ['YUC9 promoter [dna sequence]'], ['EXPA5 promoter [dna sequence]'], ['to ABI5 promoter [dna sequence]'], ['MIR390a promoter [regulatory element]'], ['MS188 promoter [dna sequence]'], ['SPX1 promoter [dna sequence]'], ['MsPIN7 promoter [dna sequence]'], ['miR399f promoter [dna sequence]'], ['MAN7 promoter region [gene]']]}\n",
      "-------------------\n",
      "{'82013': [['protein(s) upregulated in the mitochondrial fraction [protein]'], ['native mitochondrial protein(s) [protein]'], ['One of the protein(s) identified [protein]'], ['protein(s) composition of mitochondria [organelle]'], ['protein(s) make-up of mitochondria [organ]'], ['**protein(s) localized to mitochondria [protein]**', 'protein(s) localized in the mitochondrion [protein]'], ['six mitochondrial protein(s) [protein]'], ['protein(s) related to mitochondrial functions [process]'], ['many more mitochondrial protein(s) [protein]'], ['newly predicted protein(s) [protein]'], ['protein(s) on or near mitochondria [organ]'], ['protein(s) essential for mitochondrial function [protein]'], ['mitochondrial protein(s) studies [study]'], ['mitochondrial-related protein(s) [protein]'], ['mitochondrial type [organ]'], ['over 100 mitochondrial protein(s) [protein complex]'], ['data on mitochondrial protein(s) [protein]'], ['hundreds of discrete functional entities [structure]'], ['unique mitochondrial protein(s) [protein]'], ['protein(s) localized in mitochondria matrix [protein]']]}\n",
      "-------------------\n",
      "{'88082': [['**Rc exposure [treatment]**', 'Transfer to constant red light (Rc) [treatment]', 'continuous red light (Rc) [treatment]', 'Rc-grown seedlings [treatment]'], ['R20 plant(s) [treatment]'], ['Cold treatment under red light [treatment]'], ['**Red-light conditions [treatment]**', 'red light conditions [treatment]'], ['**continuous R light [treatment]**', 'Continuous red-light [treatment]', 'continuous monochromatic red light [treatment]', 'continuous R [treatment]', 'constant R light [treatment]', 'continuous red light (R) [treatment]', 'constant red light conditions [treatment]'], ['**red light-treated conditions [treatment]**', 'R light-treated seedlings [treatment]', 'red light-treated WT samples [treatment]'], ['red light input to the clock [treatment]'], ['presence of red light [treatment]'], ['**red-light only [treatment]**', 'ones cultivated under only red light [treatment]'], ['red LGR [treatment]'], ['red/dark cycles [treatment]'], ['R/B condition [treatment]'], ['**red photostimulation [treatment]**', 'red-light photostimulated sample [treatment]'], ['red light preirradiation [treatment]'], ['continuous red-enriched light [treatment]'], ['red night breaks [treatment]'], ['red-light input [treatment]'], ['red light short treatments [treatment]'], ['**red light illumination [treatment]**', 'red illumination [treatment]'], ['Red light cold-treated plant(s) [organism]'], ['R20 plant(s) [organism]'], ['prolonged red-light exposure [condition]'], ['red light photostimulation [environment]']]}\n",
      "-------------------\n",
      "{'23463': [['internal signals during development [phenotype]'], ['morphogenic signals [process]'], ['**Developmental signals [process]**', 'development signals [process]'], ['signaling for leaf development [phenotype]'], ['synchronizing cues [phenotype]'], ['multiple developmental signals [process]'], ['signaling cue that controls many aspects of plant(s) growth and development [process]'], ['Developmental signals [signal]'], ['signals involved in chloroplast development [phenotype]'], ['CLV3 signal [signal]'], ['critical signals for reproductive competence [process]'], ['a developmental and hormonal response signal [process]'], ['plant-specific signals and mechanisms [mechanism]'], ['development in response to hormonal signaling [process]'], ['a signal for developmental programming [process]'], ['signals in plant(s) developmental processes [phenotype]'], ['signals determining organ fate [phenotype]'], ['multiple developmental signals [phenotype]']]}\n",
      "-------------------\n",
      "{'113587': [['Wave P5 [phenotype]'], ['translational status of transcription factors [phenotype]'], ['transcription factor landscape [phenotype]'], ['accumulation of transcription factors [phenotype]'], ['transcription factor profiling [phenotype]'], ['transcription factor prioritization [phenotype]'], ['Transcription factor expression [phenotype]'], ['Transcription factor partitioning [phenotype]'], ['nuclear accumulation of transcription factors [phenotype]'], ['enriched transcription factors [phenotype]'], ['highest levels of transcription factor activity [phenotype]'], ['**nuclear localized transcription factor [phenotype]**', 'nuclear-localized transcription factors [phenotype]'], ['most abundant transcription factors [phenotype]'], ['timing of transcription factor expression [phenotype]'], ['pattern of transcription factor expression [phenotype]'], ['conserved translation initiation factor [phenotype]'], ['AS in transcripts of transcription factors [phenotype]'], ['localization patterns of transcription factors [phenotype]'], ['expression of lineage-specific transcription factors [phenotype]'], ['behavior of lineage-specific transcription factors [phenotype]'], ['expression of endogenous transcription factor [protein]']]}\n",
      "-------------------\n",
      "{'134031': [['CLS [gene]', '**CARDIOLIPIN SYNTHASE (CLS) [gene]**', 'final enzyme of CL biosynthesis [gene]', 'CL synthase (Crls1) gene(s) [gene]']]}\n",
      "-------------------\n",
      "{'118047': [['peanut genome database [gene]'], ['genome database [database]'], ['potato genome database [database]'], ['Moso Bamboo Genome Annotation database [database]'], ['Moso Bamboo Genome DNA database [database]'], ['genome database of R. chinensis [organism]'], ['Banana Genome database [database]'], ['Citrus Pan-genome to Breeding Database [database]'], ['Tea Genome [database]'], ['Grape Genome Database [database]'], ['pineapple genome database [database]'], ['Citrus clementina genome database [database]'], ['Citrus genome databases [database]'], ['bamboo genome database (BambooGDB) [phenotype]'], ['**Chlamydomonas reinhardtii genome database [database]**', 'Chlamydomonas genome database [database]'], ['Citrus genome database [gene]'], ['Walnut genome data [database]'], ['trifoliate orange genome database [database]'], ['**Nicotiana tabacum Genome Database [database]**', 'tobacco database [database]', 'tobacco genome database [database]'], ['genome-wide protein(s) database of melon [organism]'], ['PeanutBase [database]'], ['strawberry gene(s) bank [database]'], ['TM-1 genome database [database]'], ['**cotton genome databases [database]**', 'cotton Genome A Database [database]'], ['the Sugarcane Genome Hub database [phenotype]'], ['jujube genome database [organism]'], ['**Cucumber Genome DataBase [database]**', 'JGI Cucumis sativus Genome database [database]', 'Cucumber Genome Initiative (CuGI) database [database]'], ['MELONOMICS website [database]'], ['cassava genome database [database]'], ['banana transcriptome database [organism]'], ['areca genome database [phenotype]'], ['cucumber genome databases [genome]'], ['lettuce genome database [database]'], ['B. rapa genome database [database]'], ['Cucurbit Genomics Database [database]'], ['**MorusDB [database]**', 'Morus database [database]'], ['**Populus genome database [database]**', 'Populus genome database [database]', 'poplar genome database [database]', 'Populus databases [database]'], ['cacao genome database [genomic resource]'], ['high-quality genomic data of passion fruit [genomic datum]'], ['apple genome database [database]'], ['Eschscholzia Genome DataBase [database]'], ['**Genome Database for Rosaceae [database]**', 'GDR database [database]'], ['White Lupin Genome Database [database]'], ['genome DNA database [gene]'], ['Strawberry Genome Resources [database]'], ['Sesamum indicum genome database [database]'], ['pomegranate genome databases [genome]'], ['sweetpotato genome database [database]'], ['cannabis genome database [database]'], ['G. lamblia genome database [database]'], ['**Kiwifruit Genome Database [database]**', 'Genomic Database of Kiwifruit [database]'], ['Gold Delicious apple genome database [database]'], ['Malus genome database [database]'], ['JGI Poplar Genome Project database [database]'], ['Neurospora crassa genome (MNCDB) [database]']]}\n",
      "-------------------\n",
      "{'66521': [['**S1-bZIPs [protein complex]**', 'S1 bZIPs [protein complex]'], ['**C/S1 network dimers [protein complex]**', 'C/S1 network bZIPs [protein complex]'], ['G and H networks [protein complex]'], ['groups A, B, D, G, I, and S2 [protein complex]'], ['C/S1 heterodimerization network [phenotype]']]}\n",
      "-------------------\n",
      "{'40897': [['high oleate fad2-2 mutant seeds [mutant]'], ['fah1-2 seeds [mutant]'], ['seeds of WT, fad2-1, fad3-2, fad6-1, and fae1-1 acyl lipid mutants [mutant]']]}\n",
      "-------------------\n",
      "{'127431': [['Poly(A) sites [phenotype]', '**poly(A) site [phenotype]**', 'mRNA poly(A) site positions [phenotype]'], ['positioning of poly(A) sites [gene]'], ['alternative poly (A) [gene feature]'], ['position of poly(A) site [genomic feature]'], ['mapping potential poly(A) sites of miRNA gene(s) [gene]']]}\n",
      "-------------------\n",
      "{'143560': [['BELL transcription factors [gene family]'], ['LBD transcription factors [gene]'], ['TRB family transcription factors [gene]'], ['DELLA transcription factors [gene]'], ['**INDETERMINATE DOMAIN (IDD) transcription factors [gene]**', 'IDD transcription factors [gene]'], ['IDD transcription factors [protein]'], ['function of bHLHs, MYBs, HDs, and other transcription factors [phenotype]'], ['BELL transcription factors [protein]'], ['BELL transcription factors [gene]'], ['hub transcription factors [protein]'], ['DELLA-associated transcription factors [protein]']]}\n",
      "-------------------\n",
      "{'3586': [['Regulation of water deficit stress tolerance [phenotype]', 'regulation of drought resistance [phenotype]', 'regulating drought-stress responses [phenotype]', 'drought regulation [phenotype]', 'regulatory roles in plant(s) drought tolerance [phenotype]', '**regulation of drought tolerance [phenotype]**', 'drought regulator [phenotype]', 'regulating drought tolerance in plant(s) [phenotype]', 'modulating plant(s) response to drought [phenotype]', 'important roles in regulation of plant(s) drought resistance [phenotype]', 'water stress control [phenotype]', 'mediating drought [phenotype]', 'regulation of plant(s) resistance against water deficit [phenotype]'], ['process of stomatal closure [phenotype]'], ['regulation of water balance [phenotype]'], ['co-regulation of defence and drought tolerance traits [phenotype]'], ['investigating plant(s) drought resistance mechanisms [phenotype]']]}\n",
      "-------------------\n",
      "{'44855': [['alg3-deficient strain YG170 [mutant]'], ['erg3 yeast strain [organism]'], ['**yeast Deltaalg10 mutant [mutant]**', 'S. cerevisiae Deltaalg10 knockout strain [mutant]']]}\n",
      "-------------------\n",
      "{'97990': [['Scutellaria baicalensis Georgi [organism]', 'S. baicalensis [organism]', '**Scutellaria baicalensis [organism]**'], ['**Taiwanese Scutellaria [organism]**', 'Taiwanese Scutellaria species [organism]'], ['S. tashiroi [organism]'], ['S. playfairii [organism]'], ['S. taiwanensis [organism]', '**Scutellaria taiwanensis [organism]**'], ['S. taiwanense [organism]'], ['S. racemosa [organism]'], ['four Taiwanese Scutellaria species [organism]'], ['Taiwanese Scutellaria species are recently and rapidly divergent species [phenotype]']]}\n",
      "-------------------\n",
      "{'129762': [['AGO family members [gene]'], ['AtAGO9 [gene]'], ['AtAGO7 [gene]'], ['AtAGO5 [gene]'], ['At AGO1 [gene]'], ['host AGOs [gene]'], ['**AtAGOs [gene]**', 'AGOs (AtAGOs) [gene]'], ['AtAGO8 [gene]'], ['AaAGO1 [gene]']]}\n",
      "-------------------\n",
      "{'110509': [['leghemoglobin loci [gene]'], ['chromatin loci [genomic feature]'], ['5 of 8 loci [genetic element]'], ['L1 locus [genetic locu]'], ['Imprinted loci [gene]'], ['PEG loci [gene]'], ['mce loci [gene]'], ['breakpoint loci [genomic feature]'], ['five control loci [gene]'], ['nuclear loci [gene]'], ['organellar loci [gene]'], ['distribution pattern of Psi loci [gene]'], ['GENE2 loci [gene]'], ['LUX loci [gene]'], ['immunity loci [gene]'], ['PSL loci [gene]'], ['induced loci [gene]'], ['TERT loci [genomic region]'], ['accessory loci [gene]'], ['homeobox domain loci [gene]'], ['F-box loci [gene]'], ['flanking loci [gene]'], ['EST loci [gene]'], ['the activation of the three different silent loci [gene]'], ['mitochondrial loci [gene]'], ['differentially spliced loci [gene]'], ['certain heterochromatic loci [gene]'], ['are colocalized loci [gene]'], ['some imprinted loci [sequence]'], ['loci that give rise to these transcripts [locu]'], ['constitutively silenced loci [gene]'], ['loci of active gene(s) expression [genomic feature]'], ['PEP-bound loci [phenotype]'], ['locus-specific gene(s) activation [process]'], ['meiosis-specific loci [gene]'], ['memory loci [gene]'], ['discrete set of genomic loci [genomic locus]']]}\n",
      "-------------------\n",
      "{'150263': [['ZmFLZ25 [gene]'], ['FLZ6 and FLZ10 [gene]'], ['ZmFLZ gene(s) [gene]'], ['ZmFLZ12 [gene]'], ['ZmFLZ5 [gene]'], ['ZmFLZ16 [gene]'], ['ZmFLZ28 [gene]'], ['ZmFLZ10 [gene]'], ['ZmFLZ20 [gene]'], ['ZmFLZ27 [gene]'], ['ZmFLZ18 [gene]'], ['ZmFLZ36 [gene]'], ['ZmFLZ13 [gene]'], ['ZmFLZ14 [gene]'], ['ZmFLZ9 [gene]'], ['ZmFLZ23 [gene]'], ['ZmFLZ32 [gene]'], ['FLZ10 [gene]']]}\n",
      "-------------------\n",
      "{'20671': [['Bacterial virulence factor COR [phenotype]'], ['virulent interactions between plant(s) and Xanthomonas oryzae [phenotype]'], ['Bacterial pathogen [phenotype]'], ['host-V. dahliae interactions [phenotype]'], ['nonhost pathogens P. syringae pv. tabaci [phenotype]'], ['nonhost pathogens P. syringae pv. tomato T1 [phenotype]'], ['interaction between Salmonella and plant(s) host [phenotype]'], ['a bacterial vascular pathogen [phenotype]'], ['causative agent for dudding disease [phenotype]'], ['virulent bacterial pathogen infection [phenotype]']]}\n",
      "-------------------\n",
      "{'166810': [['CKNGs [chemical property]'], ['**CK N-glucosides [metabolite]**', 'CK N-glucoconjugates [metabolite]', 'CK-N-glucosides [metabolite]', 'CK N-glucoside [metabolite]', 'N-glucoside cytokinins [metabolite]'], ['CK O-glucosides [metabolite]'], ['**CK N7-glucosides [metabolite]**', 'N7-glycosylated CKs [metabolite]'], ['CK glucoside contents [metabolite]']]}\n",
      "-------------------\n",
      "{'63523': [['**Proximity to PSII [protein complex]**', 'co-localization with PSII [protein complex]', 'association with PSII [protein complex]'], ['**Mn binding in PSII complexes [protein complex]**', 'manganese binding to PSII [protein complex]'], ['High-affinity Mn-binding site [protein complex]'], ['Metal coordination to PSII reaction center protein(s) [protein complex]'], ['cross-linking of PSII protein(s) [protein complex]'], ['proximity of PSI and PSII [protein complex]'], ['binding of PSII extrinsic protein(s) [protein complex]'], ['access channels [process]'], ['lumenal side of PSII [protein complex]'], ['electron transfer from QA to QB [protein complex]'], ['common light-inducible quenching mechanism involving PSII core [protein complex]'], ['water splitting site [protein complex]'], ['dynamics of the photosystem II (PSII) complex [protein complex]'], ['protein-protein interactions between PSII units [protein complex]'], ['First site (Q1) [subcellular compartment]']]}\n",
      "-------------------\n",
      "{'47496': [['hkl1-1 plant(s) [mutant]'], ['ren1-1 plant(s) [mutant]'], ['esp1-1 mutant plant(s) [mutant]'], ['patrol1 plant(s) [mutant]'], ['zet-1 (Ler) plant(s) [mutant]'], ['pre1-1 plant(s) [mutant]'], ['Recessive ldl1 mutants [mutant]'], ['ctl1 mutants [mutant]'], ['del1-1 plant(s) [mutant]'], ['prors1-1 plant(s) [mutant]'], ['grs1-1 plant(s) [mutant]'], ['lrs1-1 mutant [mutant]'], ['**lec1 plant(s) [mutant]**', 'lec1 mutant plant(s) [mutant]', 'mutant phenotypes of lec1 [mutant]'], ['slk1-1 plant(s) [mutant]'], ['slk2-1 plant(s) [mutant]'], ['era1 plant(s) [mutant]'], ['lp1 mutant [mutant]'], ['the1-1 seedlings [mutant]'], ['lip1 plant(s) [mutant]'], ['qrt1/qrt1 plant(s) [mutant]'], ['polarity abnormalities in eal-1; as2 plant(s) [mutant]'], ['LOF the1-1 and mca1 plant(s) [mutant]'], ['wat1 seedlings [mutant]'], ['tip1 mutant plant(s) [mutant]'], ['cas1-1 plant(s) [mutant]'], ['len1 plant(s) [mutant]'], ['szl1 plant(s) [mutant]'], ['top1alpha plant(s) [mutant]'], ['gonst1 plant(s) [mutant]'], ['enl1 plant(s) [mutant]'], ['pr1 plant(s) [mutant]'], ['wat1 plant(s) [mutant]'], ['T-DNA insertional mutant ctl1-1 [mutant]'], ['clv1-1 plant(s) [mutant]'], ['gl1-1 plant(s) [gene]'], ['mod1 mutant plant(s) [mutant]'], ['sub-1 phenotype [mutant]'], ['dal1 mutant plant(s) [mutant]']]}\n",
      "-------------------\n",
      "{'54278': [['35S:ERF5 plant(s) [mutant]'], ['35S:ERF6 plant(s) [mutant]'], ['35S:ERF4-A plant(s) [mutant]'], ['35S:ERF4-A overexpressing plant(s) [mutant]'], ['35S:ERF4-R plant(s) [mutant]'], ['35S:CRF6 overexpressing plant(s) [mutant]'], ['35S: AREB1 plant(s) [mutant]']]}\n",
      "-------------------\n",
      "{'83699': [['physical interaction in nucleus [subcellular compartment]'], ['physical interaction between them [protein]', 'Direct interactions [protein]', '**physical interaction between protein(s) [protein]**'], ['physical interactions between protein(s) [interaction]'], ['interactions of yeast protein(s) [protein]']]}\n",
      "-------------------\n",
      "{'8664': [['WT-like or intermediate phenotypes [phenotype]'], ['distributions similar to wild-type [phenotype]'], ['**similar phenotype to wild-type [phenotype]**', 'similar phenotypes to wild-type [phenotype]'], ['phenotype [similar to gottwald et al., 2000]'], ['morphologically indistinguishable from wild-type control [phenotype]'], ['similar to Westar [phenotype]'], ['wild ecotype [phenotype]'], ['wild-type-like [phenotype]']]}\n",
      "-------------------\n",
      "{'146835': [['AT1G60420 (AtNRX1) [gene]'], ['AtP3B [gene]'], ['AtSRP3 (ArathZ1; At1g64030) [gene]', '**AtSRP3 (At1g64030) [gene]**'], ['AP2-EREBP family (At5g64750) [gene]'], ['AT4G23450 (AtAIRP1) [gene]'], ['AtAIRP3 [gene]'], ['AtGR-RBP2 [gene]'], ['AtGR-RBP7 [gene]'], ['AtGR-RBP8 [gene]'], ['AtSRP2 (At2g14540) [gene]'], ['At-Pirin1 [gene]'], ['AtRNP1 [gene]'], ['AT1G73390 (AtBro1) [gene]'], ['AtBAP31 [gene]']]}\n",
      "-------------------\n",
      "{'96929': [['Oil in soybean seeds [organism]'], ['plant(s) seed oils [metabolite]'], ['oil stored in cotyledons [organ]'], ['Oil seeds [organism]'], ['plant(s) oil bodies [subcellular compartment]'], ['Oil seeds [organ]'], ['**plant(s) oil body [organelle]**', 'plant(s) OBs [organelle]', 'plant(s) oil bodies [organelle]'], ['safflower oil body [organ]'], ['Seed oilbodies [organelle]'], ['unusually large oilbodies [organelle]'], ['plant(s) oleosins [protein]'], ['oilseed embryo [organ]'], ['plant(s) seed oil bodies [subcellular compartment]'], ['kernel oil accumulation [process]'], ['oil accumulation of seeds [organ]'], ['oil body membrane [organ]'], ['oil storage [process]'], ['oil-body of perilla seeds [organ]'], ['the most common storage lipids in seed [organ]'], ['large quantity of oil bodies [organ]'], ['oilseed LDs [organelle]'], ['oilseed LD biology [organelle]'], ['oil accumulating seeds and fruits [organ]'], ['**oil-bodies [organelle]**', 'the oil body [organelle]'], ['sesame seed lipid bodies [tissue]'], ['membranes from oil seeds [organ]'], ['oil biosynthesis in P. ostii developing endosperm [organ]'], ['lily seed oil bodies [organ]'], ['major storage reserve in oilseeds [organ]'], ['the main type of seed oil stored in plant(s) [phenotype]'], ['seeds oil bodies [organ]', '**Seed oil bodies [organ]**'], ['OB coat [organelle]'], ['energy content in non-seed organs [organ]'], ['sesame oil bodies [organ]'], ['purified oil bodies [organ]'], ['major seed storage compound in plant(s) [organism]'], ['major carbon and energy reserves in seeds [organ]']]}\n",
      "-------------------\n",
      "{'150596': [['SibZIPs [gene]'], ['SibZIP02 [gene]'], ['SibZIP04 [gene]'], ['SibZIP61 [gene]'], ['SibZIP62 [gene]'], ['SibZIP06 [gene]'], ['SibZIP07 [gene]'], ['SibZIP16 [gene]'], ['SibZIP24 [gene]'], ['SibZIP42 [gene]'], ['SibZIP48 [gene]'], ['SibZIP49 [gene]'], ['SibZIP53 [gene]'], ['SibZIP36 [gene]'], ['SibZIP54 [gene]'], ['SibZIP23 [gene]'], ['SibZIP26 [gene]'], ['SibZIP41 [gene]'], ['SibZIP55 [gene]'], ['SibZIP44 [gene]'], ['SibZIP19 [gene]'], ['SibZIP35 [gene]'], ['SibZIP17 [gene]'], ['SibZIP25 [gene]'], ['SibZIP29 [gene]'], ['SibZIP38 [gene]'], ['SibZIP08 [gene]'], ['SibZIP21 [gene]'], ['SibZIP33 [gene]'], ['SibZIP57 [gene]'], ['SibZIP10 [gene]'], ['SibZIP31 [gene]'], ['SibZIP01 [gene]'], ['SibZIP28 [gene]'], ['SibZIP56 [gene]'], ['SibZIP14 [gene]']]}\n",
      "-------------------\n",
      "{'159677': [['**CO/FT module [gene]**', 'FLOWERING LOCUS T/CONSTANS (FT/CO) module [gene]', 'CONSTANS/FLOWERING LOCUS T (CO/FT) module [gene]'], ['**CO/FT regulatory module [gene]**', 'CO/FT regulatory system [gene]', 'activation of FT by CO [gene]', 'CO-FT regulatory module [gene]', 'regulation of FT gene(s) by CO [gene]'], ['**CO/FT regulon [gene]**', 'CONSTANS/FLOWERING LOCUS T (CO/FT) regulon [gene]', 'CO-FT regulon [gene]', 'GI-CONSTANS (CO)-FLOWERING LOCUS T (FT) regulon [gene]', 'central regulon (CO-FT) [gene]'], ['**CO and FT homologs [gene]**', 'CO (DN52048-c0-g4) and FT (DN56531-c3-g1) homologs [gene]', 'CO and FT [gene]', 'CONSTANS (CO) and FT protein(s) [gene]', 'CO and FT protein(s) [gene]'], ['**CO-FT pathway [gene]**', 'GI-CONSTANS (CO)-FLOWERING LOCUS T (FT) pathway [gene]', 'CONSTANS (CO)-FLOWERING LOCUS T (FT) pathway [gene]'], ['**CO-mediated transcription activation of FT [gene]**', 'CO and FT transcription [gene]', 'CO promotion of FT [gene]', 'CO-mediated FT activation [gene]'], ['COREs at FT [gene]'], ['FLOWERING LOCUS T (FT) activator CONSTANS (CO) [gene]'], ['CO/FT mechanism [gene]'], ['CO to induce FT [gene]'], ['CO binding to FT promoter [gene]'], ['FT through CONSTANS (CO) [gene]'], ['binding of RcCO to promoter of FLOWERING LOCUS T (RcFT) [gene]'], ['**CO targeting [gene]**', 'targeting of CO to FT [gene]'], ['**CO/FT module [protein complex]**', 'CO-FT module [protein complex]', 'function of core CO-FT module [protein complex]', 'CO-FT regulatory module [protein complex]'], ['**CO-FT module [phenotype]**', 'CO/FT module [phenotype]'], ['**CO-mediated activation of FT [phenotype]**', 'CO activation of FT [phenotype]', 'CO activation of FT expression [phenotype]', 'modulation of CO/FT ortholog transcripts [phenotype]', 'mRNA expression of key flowering regulatory gene(s) CONSTANS (CO) and FLOWERING LUCUS T (FT) [phenotype]', 'CONSTANS-mediated regulation of flowering locus FT [phenotype]', \"CO's ability to modulate FT-gene expression [phenotype]\"], ['CONSTANS (CO) protein(s) [protein]'], ['**CO-FT pathway [process]**', 'CO-FT flowering pathway [process]'], ['CO activity for FT expression [process]'], ['CO-FT pathway [gene pathway]'], ['CO-FT module [gene pathway]'], [\"**CO's ability to control FT expression [gene expression]**\", 'CO-FT regulatory system [gene expression]']]}\n",
      "-------------------\n",
      "{'107203': [['SMRT-seq [treatment]'], ['Roche Nimblegen High Density (HD2) tiling array [treatment]'], ['SMRT-Seq data [technology]'], ['snmC-seq2 libraries [treatment]', 'snmC-seq2 method [treatment]', '**snmC-seq2 [treatment]**'], ['Combination of Pico-kit and snmC-seq2 methods [treatment]'], ['deep sequencing (MNase-seq) assay [treatment]'], ['SOLiD SNP calls [treatment]'], ['single-nucleus sequencing [treatment]'], ['MNase-seq tracks [treatment]'], ['m6A-seq [treatment]']]}\n",
      "-------------------\n",
      "{'664': [['mature females that emerged [phenotype]', '**the adult female stage [phenotype]**'], ['**fourth instar [phenotype]**', 'fourth-stage juveniles [phenotype]'], ['adult development stage of C. virgata [phenotype]'], ['juveniles developing into winged adult aphids [phenotype]'], ['adults [phenotype]'], ['last stages of mite development [phenotype]']]}\n",
      "-------------------\n",
      "{'154934': [['ARPF2 [gene]'], ['RING-finger peroxins (PEX2, PEX10, PEX12) [gene]', '**PEX2RF [gene]**']]}\n",
      "-------------------\n",
      "{'159575': [['AcBBX5 [gene]'], ['AtBBX4 [gene]'], ['AtCIB5 [gene]'], ['citrus B-box zinc finger protein(s) CsBBX28 [protein]']]}\n",
      "-------------------\n",
      "{'151766': [['OsSRP-LGC [gene]'], ['OsSRP-FRS [gene]'], ['OsSRP-FAS [gene]'], ['OsSRP-GMS [gene]'], ['OsSRP-LLS [gene]'], ['OsSRP-FLC [gene]'], ['OsSRP-QKG and -GRA [gene]'], ['Rice RSp29 [gene]'], ['Rice RSZp23 [gene]'], ['Arachis diogoi SR protein(s) RSZ21 [gene]'], ['OsRSZp23 [gene]'], ['OsSR45 [gene]'], ['OsSR40 [gene]'], ['OsRS29 [gene]'], ['OsRS2Z38 [gene]'], ['OsRS2Z21a [gene]'], ['OsSR45a [gene]'], ['OsRSZ23 [gene]'], ['Sc [gene]'], ['OsRS33 [gene]'], ['OsRSZ21a [gene]'], ['cpSRP54a [gene]'], ['cpSRP54b [gene]'], ['SRS8 [gene]'], ['SIZ1 protein(s) in rice and sorghum [gene]'], ['RS2Z [gene]'], ['RSZ [gene]'], ['OsSR33a [gene]'], ['OsRS2Z37 [gene]'], ['OsRS2Z39 [gene]'], ['OsRSZ21 [gene]'], ['OsSR32 [gene]'], ['OsSR33 [gene]'], ['OsRS2Z36 [gene]']]}\n",
      "-------------------\n",
      "{'78859': [['**gammaTuRC [protein complex]**', 'gammaTuRCs [protein complex]', 'gamma-TuRC [protein complex]', 'γ-tubulin ring complexes (γ-TuRCs) [protein complex]', 'gamma-tubulin ring complex (gammaTuRC) [protein complex]', 'gammaTuRC complexes [protein complex]', 'γ-tubulin-containing ring complex (γTuRC) [protein complex]'], ['**gamma-TuRC components [protein complex]**', 'gammaTuRC components [protein complex]'], ['Decrease of γ-tubulin ring complex (γTuRC) components [protein]'], ['Non-nucleating γTuRCs [protein complex]'], ['γTuSC [protein complex]'], ['γTuRC function [process]'], ['gamma-tubulin ring complex (gammaTuRC) [subcellular compartment]'], ['lattice-bound gammaTuRC complexes [protein complex]'], ['nucleated gammaTuRC [protein complex]'], ['motile γ-TuRCs [protein complex]']]}\n",
      "-------------------\n",
      "{'129774': [['**AtGLK1 [protein]**', 'transcription activator AtGLK1 [protein]'], ['AtGTL1 ortholog in poplar, PtaGTL1 [gene]', '**AtGTL1 [gene]**', 'AtGTL1 mRNA [gene]', 'AT-GTL1 [gene]'], ['AtRGTB2 mRNA [gene]'], ['AtGUN1 [gene]'], ['GLK1 (At2g20570) [gene]', '**AtGLK1 [gene]**', 'AtGLK1 gene(s) [gene]'], ['GL-1 gene(s) [gene]'], ['G-type lectin S-receptor-like serine/threonine-protein kinase At2g19130 [gene]'], ['AtPGL3 transcript levels [gene]'], ['AtCML4 [gene]'], ['AtGLR1.1 [gene]'], ['AtGln1;1-5 [gene]'], ['AtGSK1 [gene]'], ['AtGLK gene(s) [gene]'], ['OEGLK1-1 and OEGLK1-2 transgenic plant(s) [gene]'], ['transcripts of At1g49510 [gene]'], ['AtGLB1 [protein]'], ['AtSGN3 [gene]'], ['AtGS1 [gene]'], ['AtGL-rec gene(s) [gene]'], ['**AtGLN1.1 [gene]**', 'AtGLN1.1 gene(s) [gene]'], ['**AtGLN1.4 [gene]**', 'AtGLN1.4 gene(s) [gene]'], ['AtGLN1.1-AtGLN1.5 [gene]'], ['AtGLN1 gene(s) [gene]'], ['AtGLN1.2 gene(s) [gene]'], ['AtGLN1.3 gene(s) [gene]'], ['AtCAL2 [gene]'], ['AtCAL3 [gene]'], ['AtCAL6 [gene]'], ['ATDGK1 mRNA [gene]'], ['AT-GTL2 [gene]'], ['AT-GTL gene(s) [gene]'], ['ATR1-like clade gene(s) [gene]'], ['AtXLG1 [gene]'], ['AKINgamma mRNA [gene]'], ['AtCBL1 gene(s) [gene]'], ['AtGTL [gene]'], ['AtHTL/KAI2 [gene]'], ['AtGLK1 transcripts [gene]'], ['AtGLK2 transcripts [gene]'], ['AtGLN1 gene(s) family [gene]'], ['GLN1;4 [protein]']]}\n",
      "-------------------\n",
      "{'88186': [['higher light intensity [treatment]', '**high light intensities [treatment]**', 'high-light condition [treatment]', 'higher intensities [treatment]'], ['Changes in light intensity [treatment]'], ['Full-Light [treatment]'], ['combined heat and light treatments [treatment]'], ['varying shade intensity [treatment]'], ['all three treatments in wild-type [treatment]'], ['light received by photoreceptors [treatment]'], ['A = 0.99 [treatment]'], ['previous light conditions [treatment]'], ['continuous white-light conditions [treatment]'], ['light intensity range [treatment]'], ['**different wavelength lights [treatment]**', 'two light qualities [treatment]', 'illumination with different light qualities [treatment]'], ['several orders of magnitude of light input [treatment]'], ['several light intensities [treatment]'], ['light induced [treatment]'], ['light quality and quantity [treatment]'], ['mixed light condition [treatment]'], ['every light intensity [treatment]'], ['WL or WL + FR conditions [treatment]'], ['**light and carbon treatment [treatment]**', 'regulation by light and carbon [treatment]'], ['light treatments [process]'], ['each light condition [treatment]'], ['three different light intensities [treatment]'], ['low intensities [treatment]'], ['three light intensities [treatment]'], ['defined light treatments [treatment]'], ['illumination and magnetic field exposure conditions [treatment]'], ['different lighting settings [treatment]'], ['four different spectral conditions [treatment]'], ['three different illumination conditions [treatment]'], ['same light intensity [treatment]'], ['two light regimes [treatment]'], ['continuous low-level illumination [treatment]'], ['two light conditions [treatment]'], ['same light conditions [treatment]'], ['irradiance treatments [treatment]'], ['three different light regimes [treatment]']]}\n",
      "-------------------\n",
      "{'72711': [['A. thaliana ACBP family [protein]', '**A. thaliana ACBPs (AtACBPs) [protein]**'], ['A. thaliana ACBP1 [protein]'], ['**A. thaliana nuclear cap-binding protein(s) [protein]**', 'A. thaliana homolog of a nuclear mRNA cap binding protein(s) [protein]', 'A. thaliana nuclear mRNA cap-binding protein(s) [protein]'], ['A. thaliana PEBP-family protein(s) sequences [protein]'], [\"A. thaliana B' subunits [protein]\"], ['A. thaliana PAT protein(s) [protein complex]'], ['A. thaliana membrane-associated progesterone binding protein(s) [protein]'], ['A. thaliana BiP [protein]'], ['A. thaliana BiP promoters [protein]'], ['A. thaliana GBPL3 [protein]'], ['A. thaliana ACBP6 [protein]'], ['**A. thaliana Class IV ACBPs [protein]**', 'A. thaliana Class IV ACBPs [protein clas]'], ['A. thaliana ACBP family [protein family]', '**A. thaliana ACBPs [protein family]**'], ['A. thaliana ACBP4 [protein]'], ['A. thaliana ACBP5 [protein]'], ['A. thaliana ABI5 protein(s) [protein]'], ['A. thaliana auxin binding protein(s) [protein]'], ['subset of A. thaliana SBP domain protein(s) [protein family]'], ['A. thaliana PAD1/alpha4 [protein]'], ['A. thaliana polyadenylation factor subunit [protein]'], ['A. thaliana PABPs [protein]'], ['A. thaliana clade A PP2Cs [protein]'], ['A. thaliana nuclear poly(A) binding protein(s) [protein]'], ['A. thaliana protein(s) AthPBS1 [protein]'], ['A. thaliana PAB binding [protein complex]'], [\"A. thaliana PP2A B' protein(s) [protein]\"], ['A. thaliana homologs of PabN1 [protein]'], ['A. thaliana B-protein AP3 [protein]'], ['A. thaliana type A subunit of PP2A [protein]']]}\n",
      "-------------------\n",
      "{'151906': [['GmCYS14 [gene]'], ['GmCYS16 [gene]'], ['GmCYS17 [gene]'], ['GmCYS15 [gene]'], ['GmCYS18 [gene]']]}\n",
      "-------------------\n",
      "{'103979': [['wild-type strain VOT [organism]'], ['P. polaris NCPPB3395 WT [organism]'], ['WT FZB42 [organism]'], ['GI/WT culture [strain]'], ['D. dadantii WT strain [organism]'], ['WT bacterial strain 3937 [organism]'], ['WT D. dadantii strain 3937 [organism]'], ['**Erwinia amylovora bacterium WT strain [organism]**', 'wild-type strain of E. amylovora [organism]'], ['E. coli wild-type strain WM1074 [organism]'], ['S. liquefaciens WT [organism]'], ['WT GvW [organism]'], ['B. cinerea WT strain [organism]'], ['infection by WT bacterial strain [organism]']]}\n",
      "-------------------\n",
      "{'111579': [['**regulatory SNP [mutation]**', 'single-nucleotide regulatory mutations [mutation]'], ['single sites [gene]']]}\n",
      "-------------------\n",
      "{'132981': [['Cytosolic Pyrophosphatases [gene]', '**Inorganic pyrophosphatase [gene]**', 'soluble PPases [gene]', 'isoform of inorganic pyrophosphatase [gene]', 'Pyrophosphatases [gene]', 'cytosolic soluble pyrophosphatases [gene]', 'other pyrophosphatases [gene]', 'several PPases [gene]'], ['Inorganic pyrophosphatase 2 [gene]'], ['soluble pyrophosphatase from Saccharomyces cerevisiae [gene]'], ['Escherichia coli pyrophosphatase [gene]'], ['Pyrophosphorylase [gene]'], ['**bisphosphate nucleotidase [gene]**', 'endogenous bisphosphate nucleotidase activities [gene]'], ['exopolyphosphatase (ppx) [gene]'], ['phosphoglycerate mutase [gene]'], ['Starch phosphorylating/dephosphorylating enzymes [gene]'], ['nucleoside triphosphate phosphohydrolase II [gene]'], ['Enzyme encoded by low phosphate root(s) 1 (LPR1) [gene]'], ['phosphoglycolate phosphatase 1B [gene]'], ['specialized phosphohydrolases [gene]'], ['DNA coding inorganic pyrophosphatases [gene]'], ['Fungal counterparts [gene]'], ['Phosphorylases [gene]'], ['Bacterial phosphite oxidoreductase gene(s) [gene]'], ['PHOSPHOTYROSYL PHOSPHATASE ACTIVATOR [gene]']]}\n",
      "-------------------\n",
      "{'98032': [['**Musa acuminata ssp. malaccensis [organism]**', 'Musa acuminate ssp. malaccensis [organism]', 'Musa acuminata subsp. malaccensis [organism]'], ['Banana (Musa sp, Zingiberales) [organism]', 'Musa species [organism]', '**Musa [organism]**', 'banana (Musa spp.) [organism]', 'Musa spp. [organism]', 'Bananas (Musa) [organism]', 'Banana plant(s) [organism]', 'Musa (Zingiberales) [organism]', 'banana species [organism]', 'banana and plantain species [organism]'], ['**Musa acuminata [organism]**', 'Banana (Musa acuminata L.) [organism]', 'M. acuminata [organism]', 'Bananas (Musa acuminata) [organism]', 'Musa acuminata Colla [organism]', 'monocots Musa acuminata [organism]'], ['**Musa balbisiana [organism]**', 'M. balbisiana [organism]'], ['**Musa acuminate [organism]**', 'banana Musa acuminate [organism]', 'banana (Musa acuminate) [organism]'], ['Musa accuminata [organism]'], ['Musa (alpha) [organism]'], ['Musa acuminata subsp. Zebrina [organism]'], ['banana (Musa nana) [organism]'], ['**Musa paradisiaca [organism]**', 'Musa x paradisiaca [organism]'], ['banana (Musa acuminata and Musa balbisiana) [organism]'], ['Banana plant(s) [organ]'], ['mulberry [organism]', 'Mulberry varieties [organism]', 'Mulberry (Morus L.) [organism]', '**Morus [organism]**', 'Mulberry (Morus species) [organism]', 'Mulberry trees [organism]', 'Mulberry tree (Morus L.) [organism]', 'mulberry (Morus spp.) [organism]', 'mulberry tree [organism]', 'mulberries [organism]', 'mulberry plant(s) [organism]'], ['Morus atropurpurea [organism]'], ['Morus atropurpurea fruit [organ]'], ['**Mulberry (Morus alba L.) [organism]**', 'M. alba [organism]', 'Morus alba L. [organism]'], ['Morus alba. var. tortuosa [organism]'], ['**Morus notabilis [organism]**', 'mulberry (Morus notabilis) [organism]', 'mulberry, Morus notabilis [organism]'], ['Morus serrata [organism]'], ['Morus indica [organism]'], ['Morus multicaulis [organism]'], ['mulberry fruit [organ]'], ['mulberry fruit [organism]'], ['fruit of two mulberry varieties [organism]'], ['barks of mulberry [organism]']]}\n",
      "-------------------\n",
      "{'78262': [['cytoplasmic region [protein domain]'], ['Intracellular loop region of LIT1 [gene]'], ['short sequences in middle cytoplasmic regions [protein]'], ['central hydrophilic T-loop of PIN2 [subcellular compartment]'], ['cytoplasmic region [protein]'], ['Full-length tail [subcellular compartment]'], ['Histidine-rich cytoplasmic loop [subcellular compartment]'], ['central hydrophilic loops of PIN protein(s) [subcellular compartment]'], ['cytoplasmic TM8-TM9 loop [subcellular compartment]'], ['cytoplasmic loops [subcellular compartment]'], ['less conserved HL region [subcellular compartment]'], ['**central hydrophilic loop [subcellular compartment]**', 'central HL region [subcellular compartment]'], ['cytosolic positions [sequence]'], ['intracellularly positioned hydrophilic loops [subcellular compartment]'], ['intracellular side [protein region]'], ['cytoplasmic region [protein region]'], ['cytoplasmic hydrophilic loop [subcellular compartment]'], ['short intra-cellular tail [protein]'], ['fifth cytoplasmic hydrophilic loop [subcellular compartment]'], ['intracellular part [gene region]'], ['internal region of BASL [protein]'], ['cytoplasmic TM2-TM3 loop [subcellular compartment]'], ['cytoplasmic DE-loop of the D1 protein(s) [subcellular compartment]']]}\n",
      "-------------------\n",
      "{'142848': [['down-regulated TF [gene]'], ['mitochondrial transcription termination factor (mTERF) family [gene]'], ['forgetter (FGT) [chromatin regulator]'], ['mTERF [gene family]'], ['**type II TCPs [gene family]**', 'class II (TCP-C) [gene family]'], ['ULT1-repressed TF gene(s) in reproductive apices [gene]'], ['Passive repressors [gene]'], ['**ligand-responsive transcriptional dual regulator MetR [gene]**', 'MetE transcriptional regulator MetR [gene]'], ['PRR transcriptional repressors [gene]'], ['TPL gene(s) family [gene]', '**transcriptional corepressor TOPLESS [gene]**', 'transcriptional corepressor TOPLESS (TPL) [gene]', 'transcriptional co-repressor TOPLESS (TPL) [gene]'], ['general corepressor [description]'], ['transcriptional repression of TIFY8 [gene]'], ['RLFGI motif [gene]'], ['ATH1 repression function [function]'], ['tumor-suppressive transcription factor [phenotype]'], ['transcriptional repressor TEM1 [gene]'], ['TIF1beta/KAP1 [protein]'], ['negative regulator of transcriptional initiation [gene]'], ['transcriptional suppressor in interfascicular fibers [tissue]'], ['TLLLFR repression motif [gene]'], ['TFs repress large subset of G2/M-specific gene(s) [gene]'], ['transcriptional repressor (TRP) [gene]'], ['negative transcription factor [gene]'], ['FT transcriptional repressor [gene]'], ['potential transcriptional repressors [gene]'], ['TPL-related 1 [transcriptional co-repressor]'], ['C2 repressor domain [gene]'], ['transcriptional repression by LOF1.2 [gene]'], ['Gro/TLE family of transcriptional co-repressors [protein family]'], ['adjacent transcription unit [gene]']]}\n",
      "-------------------\n",
      "{'166006': [['**Lipopolysaccharides [metabolite]**', 'Bacteria lipopolysaccharide (LPS) [metabolite]', 'lipopolysaccharides (LPS) [metabolite]', 'lipopolysaccharide (LPS) [metabolite]', 'LPS molecule [metabolite]', 'Bacterial lipopolysaccharide (LPS) [metabolite]', 'LPSs [metabolite]', 'bacterial LPS [metabolite]', 'endotoxin [metabolite]', 'Bacteria-derived lipopolysaccharides [metabolite]', 'lipopolysaccharide (LPS) molecule [metabolite]', 'bacterial lipopolysaccharides (LPS) [metabolite]', 'bacterial lipopolysaccharides [metabolite]', 'Bacterial lipopolysaccharide [metabolite]', 'Endotoxins [metabolite]', 'bacterial-derived lipopolysaccharides [metabolite]'], ['**purified LPS [metabolite]**', 'pLPS [metabolite]'], ['**Lipopolysaccharide [molecule]**', 'lipopolysaccharides (LPS) [molecule]', 'Bacterial lipopolysaccharide (LPS) [molecule]'], ['B. cenocepacia LPS [metabolite]'], ['**LPS chemotypes [metabolite]**', 'LPS chemotype [metabolite]'], ['LPS chemotypes from Pst and Xcc [metabolite]'], ['**LPS from Burkholderia cepacia [molecule]**', 'LPS from B. cepacia [molecule]'], ['LPS from Escherichia coli [molecule]'], ['different moieties of LPS [metabolite]'], ['LPS treatment [metabolite]'], ['common polysaccharide antigen [metabolite]'], ['LPS [ligand]'], ['LPS fragments [metabolite]'], ['C. balustinum cell wall lipopolysaccharides [metabolite]'], ['LPS preparations [metabolite]'], ['LPS-containing cell walls [metabolite]'], ['Elicitor-active preparations of lipopolysaccharides (LPS) [metabolite]'], ['Herbaspirillum LPS [metabolite]'], ['LPSs from B. cepacia [metabolite]'], ['A. brasilense LPS [metabolite]'], ['LPS structures [metabolite]'], ['LPS from Xanthomonas campestris pv. campestris [metabolite]'], [\"hydrophobic 'rough' LPS [metabolite]\"], ['lipoteichoic acid [metabolite]'], ['LPS from Pseudomonas aeruginosa [metabolite]'], ['cell wall-derived LPS [metabolite]'], ['LPS of Pseudomonas species [molecule]'], ['LPS of X. campestris [molecule]'], ['**Xanthomonas LPS [metabolite]**', 'bacterial Xanthomonas LPS [metabolite]'], ['pro-inflammatory ligands lipopolysaccharides [metabolite]'], ['surface lipopolysaccharides [metabolite]'], ['movement and localization of LPS [metabolite]'], ['LPS from Escherichia coli [metabolite]']]}\n",
      "-------------------\n",
      "{'126710': [['transition of plant(s) chromosomal DNA [genomic region]'], ['3S breakpoint [genotype]'], ['translocation of sub-chromosomal region [gene]'], ['9A/9B translocation breakpoint [genome]'], ['localization of a viewpoint along the chromosome arm [genomic region]'], ['translocation breakpoint in Chr.2L and Chr.3S [genotype]'], ['long distance chromosomal interactions [genomic region]'], ['wrong chromosome [genomic location]'], ['TLs between chromosomal sites of interest [treatment]']]}\n",
      "-------------------\n",
      "{'96658': [['2 mo [time]'], ['day 20 [time]'], ['21 days [time]']]}\n",
      "-------------------\n",
      "{'61283': [['FAD2 [mutant]'], ['fry1 [mutant]'], ['fab2 mutant [mutant]'], ['fab2-2 [mutant]'], ['fab2-3 [mutant]'], ['F2:3 [mutant]'], ['fh1:CRISPR [mutant]'], ['arf3 mutant [mutant]'], ['ACHT2-TF2 [mutant]'], ['ft-10 tsf-1 mft-2 bft-2 [mutant]'], ['FIONA1:FIONA1/fiona1-1 [mutant]'], ['FIONA1:FIONA1m/fiona1-1 [mutant]'], ['fad6-3 [mutant]'], ['r(2997) yellow-flesh [mutant]'], ['Col-FRI [mutant]'], ['FRI-Col [mutant]'], ['fugu2 mutant [mutant]'], ['AlF4 [mutant]'], ['F1 individual [mutant]'], ['38 F2 individuals [mutant]'], ['12 F2 individuals [mutant]'], ['uf plant(s) [mutant]'], ['tfl1 [mutant]'], ['arf7 arf19 [mutant]'], ['fiz1 mutant [mutant]'], ['gfg1-1 [mutant]'], ['CRF4OX [mutant]'], ['F2:3 families [mutant]'], ['flail3 pFLAIL:gFLAIL [mutant]'], ['F2s [mutant]'], ['12% in FerIN-3 [mutant]'], ['up to 49% in FerIN-8 [mutant]'], ['50 F2 progeny [mutant]'], ['EFD [mutant]']]}\n",
      "-------------------\n",
      "{'68647': [['interactions with EFOP protein(s) [gene]'], ['binding of eIF2alpha-P [protein]'], ['interaction of MVK and eATP receptor P2K1 [protein]'], ['NET2A-PRK interactions [interaction]'], ['CK2-mediated increase in eIF5 interaction with eIF1 [protein]'], ['ease of interaction between CK2alpha and HMGB protein(s) [protein]']]}\n",
      "-------------------\n",
      "{'131322': [['A. thaliana RGS protein(s) (AtRGS1) [protein]', 'A. thaliana regulator of G-protein signaling 1 (AtRGS1) [protein]', '**A. thaliana RGS1 [protein]**', 'A. thaliana Regulator of G Signaling (AtRGS1) [protein]', 'A. thaliana Regulator of G Signaling protein(s) 1 (AtRGS1) [protein]'], ['A. thaliana GTS1 [protein]'], ['A. thaliana Regulator of G Signaling protein(s) 1 (AtRGS1) [gene]', '**regulator of G-protein signaling 1 (RGS1) in A. thaliana [gene]**'], ['A. thaliana RGS protein(s) [protein]'], ['small G protein(s) in A. thaliana [protein]']]}\n",
      "-------------------\n",
      "{'140937': [['Pi deficiency-responsive MGD3 promoter [gene]'], ['constitutive glyceraldehyde-3-phosphate dehydrogenase (GAP) promoter [gene]'], ['a thiamine-repressible promoter [treatment]'], ['thiamine-repressible nmt promoter [treatment]']]}\n",
      "-------------------\n",
      "{'155758': [['ePHD2 [gene]'], ['EZH1 [gene]'], ['**Ezh2 [gene]**', 'E(z)H2 [gene]', 'EZH2 histone methyltransferase [gene]', 'Enhancer of Zeste homolog 2 of humans [gene]', 'EZH2 homologs [gene]'], ['Drosophila E(Z) [gene]'], ['Class I E(z) homologs [gene]'], ['Enhancer-of-zeste [gene]'], ['ZmHM gene(s) [gene]'], ['histone demethylase PHF8 [gene]'], ['histone-lysine N-methyltransferase/zinc ion binding factor [gene]']]}\n",
      "-------------------\n",
      "{'106729': [['Clustered regularly interspaced short palindromic repeat-mediated genome editing [treatment]', 'CRISP/Cas gene(s) editing technique [treatment]', '**CRISPR-Cas genome editing [treatment]**'], ['type II clustered regularly interspaced short palindromic repeats (CRISPR) [treatment]'], ['clustered regularly interspaced short palindromic repeats (CRISPR) [treatment]', '**Clustered Regularly Interspaced Short Palindromic Repeats [treatment]**'], ['Clustered Regularly Interspaced Short Palindromic Repeats (CRISPR)-Systems [treatment]'], ['clustered regularly interspaced short palindromic repeats (CRISPR)/Cas technologies [treatment]'], ['Clustered regularly interspaced short palindromic repeat (CRISPR)-associated (Cas) (CRISPR-Cas) nucleases technology [treatment]'], ['**Clustered Regulatory Interspaced Short Palindromic Repeat associated protein(s) Cas9 (CRISPR-Cas9) [treatment]**', 'clustered, regularly interspaced, short palindromic repeats (CRISPR/Cas9) [treatment]'], ['**CRISPR technology [technique]**', 'CRISPR approach [technique]', 'Clustered Regularly Interspaced Short Palindromic Repeats (CRISPR)/Cas-based RNA-guided DNA endonuclease system [technique]'], ['CRISPR [immune system]'], ['CRISPR [system]'], ['**CRISPR/CAS technologies [tool]**', 'CRISPR/Cas-system [tool]'], ['CRISPR [technology]']]}\n",
      "-------------------\n",
      "{'52882': [['BRCT-containing gene(s) [gene]'], ['rice BBX gene(s) family [gene]'], ['BR receptor complex [gene]'], ['housekeeping RBR gene(s) [gene]'], ['four RBR subfamilies [gene]'], ['losses of Helicase RBR gene(s) [gene]'], ['many RBR gene(s) [gene]'], ['new RBR gene(s) [gene]'], ['older RBR gene(s) [gene]'], ['massive losses of RBR gene(s) [gene]'], ['helicase RBR gene(s) [gene]'], ['first set of RBR gene(s) [gene]'], ['second set of RBR gene(s) [gene]'], ['five RBR gene(s) [gene]'], ['essential RBR gene(s) [gene]'], ['six RBR gene(s) [gene]'], ['2-3 RBR gene(s) [gene]'], ['animal RBR gene(s) [gene]'], ['a set of RBR gene(s) [gene]'], ['large increases in RBR gene(s) [gene]'], ['reduced number of RBR gene(s) [gene]'], ['evolution of RBR gene(s) [gene]']]}\n",
      "-------------------\n",
      "{'1682': [['**Cotton oil [phenotype]**', 'Cotton seed oil [phenotype]'], ['well known for oil-rich seeds [phenotype]'], ['crude oil [phenotype]'], ['the most important byproduct [phenotype]'], ['**an oilseed plant(s) [phenotype]**', 'one of the major world oil crops [phenotype]', 'an important source of oil and livestock feed [phenotype]'], ['Edible oil [phenotype]']]}\n",
      "-------------------\n",
      "{'63376': [['light harvesting complex II [protein complex]', 'light-harvesting complex II protein(s) [protein]', 'light-harvesting complex of photosystem II (PSII) [protein complex]', '**Light-harvesting complex II [protein complex]**', 'Light-harvesting complex II subunit [protein]', 'PSII light harvesting complex [protein complex]', 'several light-harvesting complex II protein(s) [protein]', 'light-harvesting complexes II [protein complex]', 'apoproteins of the light-harvesting complex of photosystem II [protein complex]', 'photosystem II light harvesting complex [protein complex]', 'light-harvesting complex of PSII [protein complex]', 'PSII light-harvesting complexes [protein]', 'photosystem II light-harvesting complex protein(s) [protein]', 'Light-harvesting complex II [protein]', 'polypeptides of light-harvesting complex II [protein complex]', 'light harvesting complexes II [protein]', 'light harvesting complex of PSII [protein complex]', 'organization of the light-harvesting complex II [protein complex]', 'Photosystem II light harvesting complex protein(s) [protein]', 'light-harvesting PSII complex protein(s) [protein complex]', 'PSII light-harvesting complexes [protein complex]', 'light-harvesting complex of photosystem II [protein complex]', 'major polypeptides of the light-harvesting complex of photosystem II [protein complex]', 'light-harvesting complex-II structure and function [protein complex]', 'photosystem II light harvesting complex [protein]', 'organization of light harvesting complex II [protein complex]', 'light-harvesting complex II protein(s) [protein complex]', 'photosystem II light-harvesting complex [protein complex]', 'light-harvesting chlorophyll protein(s) associated with photosystem II [protein]'], ['photosystem II light harvesting complex protein(s) 2.1 [protein]'], ['Photosystem II light harvesting complex protein(s) 2.3 [protein]'], ['Light Harvesting Complex of Photosystem II 5 [protein]'], ['light harvesting complex II [phenotype]'], ['**Light Harvesting Complex I [protein complex]**', 'photosystem I-Light Harvesting Complex I [protein]', 'light-harvesting protein(s) complex I [protein complex]'], ['light-harvesting complex I-730 [protein complex]'], ['light-harvesting protein(s) complex I [gene]'], ['**light harvesting complexes for photosystem I and II [protein complex]**', 'photosystem I-light harvesting complex I and II [protein]', 'light harvesting protein(s) of photosystems I and II [protein complex]'], ['CP26 [protein complex]'], ['**CP24 [protein complex]**', 'CP24 light-harvesting complex [protein complex]'], ['interactions with photosystems [protein complex]'], ['less PSII-light-harvesting complex II supercomplex [protein complex]'], ['PSII-light-harvesting complex II supercomplex [protein complex]'], ['ordered arrays of light-harvesting complex II-photosystem II [protein complex]']]}\n",
      "-------------------\n",
      "{'163102': [['RNS signals [metabolite]'], ['recognition of elicitors [metabolite]'], ['**Recognition of internal effectors [metabolite]**', 'intracellular detection of effector molecules [metabolite]'], ['pathogen-derived ligands [metabolite]'], ['recognition of effectors [metabolite]', '**Recognition of effector molecules [metabolite]**'], ['**recognition of pathogen-derived molecules [metabolite]**', 'Recognition of pathogens/pathogen-derived molecules [metabolite]', 'perception of pathogen-derived ligands [metabolite]'], ['Patterns [metabolite]'], ['Other patterns [molecule]'], ['invasion pattern molecules [metabolite]'], ['non-self molecular patterns [metabolite]'], ['conserved structure components [metabolite]'], ['specific recognition of pathogen-encoded molecules [metabolite]'], ['extrinsic molecular patterns [metabolite]'], [\"surveillance of 'non-self' molecules [metabolite]\"], [\"surveillance of 'modified self' molecules [metabolite]\"], ['ligands for host immune receptors [phenotype]'], ['known signaling components [metabolite]'], ['non-self determinants [metabolite]'], ['non-self signaling molecules [metabolite]'], ['pathogen-associated signals [metabolite]']]}\n",
      "-------------------\n",
      "{'134876': [['dirigent protein(s) ESBL [gene]'], ['ALG5 [gene]'], ['Pp201625 [gene]'], ['Pp42620 [gene]'], ['expression of 3-dehydroshikimate dehydratase [gene]'], ['HCT transcripts [gene]'], ['switchgrass HCT enzymes [protein]'], ['**homologs of NST1 [gene]**', 'The homologs of NST1 [gene]'], ['JsiCOMT1 [gene]'], ['tra2 [gene]'], ['comt-1 [gene]'], ['comt-4 [gene]'], ['TRA2 and 4CL1 [gene]'], ['ligase VI [gene]'], ['NST gene(s) [gene]'], ['IRX12 (LAC4) [gene]'], ['PH02Gene45333 [gene]'], ['hydroxystyrene CoA [gene]'], ['HCT and C3H [gene]'], ['PbBGLU1 transcripts [gene]'], ['PbBGLU16 transcripts [gene]'], ['PtrBGLU6 [gene]'], ['IRX9L and IRX14L [gene]'], ['IRX9L gene(s) [gene]'], [\"c3'h gene(s) [gene]\"], ['PdWND3A [gene]'], ['clade NAC-s [phenotype]'], ['lignin-synthesis related module 86_1 [gene]'], ['Potri.016G078300.1 [gene]'], ['QsuB gene(s) [gene]'], ['HCT gene(s) candidates [gene]'], ['PvHCT1 and PvHCT2 [gene]'], ['AT1G77520 [gene]'], ['F5H transcript [gene]'], ['AtPrx64 active sites [structure]'], ['LG network [gene network]'], ['xxt2 locus [gene]'], ['P. ovata homolog [gene]'], ['uclacyanins [gene]'], ['**NAC transcription factor PgSND1-like [gene]**', 'PgSND1-like gene(s) [gene]'], ['KLCR1 [gene]'], ['CcCCoAOMT1 [gene]'], ['SvABCG17 [gene]'], ['TaF5H1 [gene]'], ['AtC3H [gene]'], ['AtCCOMT [gene]'], ['AtCCR [gene]'], ['AtF5H [gene]'], ['PbCCOMT [gene]'], ['Pp21725 [gene]'], ['Pp156311 [gene]'], ['coniferyl-alcohol glucosyltransferase [gene]'], ['GhSND2 [gene]'], ['LACK OF SYMBIONT ACCOMMODATION (LjLAN) [gene]'], ['C3H3 [gene]'], ['HCT-like gene(s) [gene]'], ['MsHCT [gene]'], ['enzyme gene(s) HCT [gene]'], ['enzyme gene(s) CCR1 [gene]'], ['enzyme gene(s) COMT [gene]'], ['AtUXS3 orthologs [gene]'], ['PanC3H1 [gene]'], ['HCT isoform PvHCT2 [enzyme]'], ['LAC15_1 [gene]'], ['LGB network [gene network]'], ['OsLAC15 [gene]'], ['10 CesAs in Pinus taeda [gene]'], ['80% identity to aspen bispecific lignin OMT [gene]'], ['lignin synthesis gene(s) HCT [gene]']]}\n",
      "-------------------\n",
      "{'96456': [['seed-holders [device]', '**seed-holder [device]**'], ['warm soils [organ]'], ['seed holders [treatment]'], ['new device designed as seed-holder [organ]'], ['seed-holder [material]'], ['Germination environment [environment]'], ['optimum climate space for seedling emergence [environment]'], ['germinating in matching environments [organ]'], ['immediate germination environment [environment]'], ['June in Kentucky [environment]'], ['June in Rhode Island [environment]'], ['November in Rhode Island [environment]']]}\n",
      "-------------------\n",
      "{'25495': [['Dwarf phenotypes [phenotype]', '**Dwarf phenotype [phenotype]**', 'dwarf stature [phenotype]', 'display dwarf phenotypes [phenotype]', 'dwarf forms [phenotype]'], ['dwarfed genotypes [phenotype]'], ['severe dwarf A. thaliana lines [phenotype]'], ['Albino and dwarf phenotypes [phenotype]'], ['Dwarfism and late flowering phenotypes [phenotype]'], ['exaggerated dwarf phenotype [phenotype]'], ['early appearance of dwarf phenotype [phenotype]'], ['dwarf phenotype to cucumber [phenotype]'], ['dwarf phenotype of maize [phenotype]'], ['autoimmune dwarf phenotype [phenotype]'], ['**BR-dwarf phenotype [phenotype]**', 'BR-dwarfing [phenotype]'], ['**weak dwarf phenotype [phenotype]**', 'weak dwarf phenotypes [phenotype]'], ['mitigated dwarfism [phenotype]'], ['d3-2 defects in tiller growth and plant(s) height [phenotype]'], ['strongest dwarf phenotype [phenotype]'], ['**dwarf phenotypes of mutants [phenotype]**', 'mutant dwarfing phenotype [phenotype]'], ['dwarf phenotype of big5 [phenotype]'], ['intermediate dwarf phenotypes [phenotype]'], ['GA-deficient dwarf phenotype [phenotype]'], ['dwarf phenotype in rice seedlings [phenotype]'], ['strong dwarf phenotypic defect [phenotype]'], ['soybean dwarfism [phenotype]'], ['strong dwarfism in tomato [phenotype]'], ['more dwarf stature seedlings [phenotype]'], ['dwarfing of dmc [phenotype]'], ['dwarfing, smaller leaves, curling, and smaller tubers [phenotype]'], ['relatively lighter defects with slight dwarfism [phenotype]'], ['subtle dwarf phenotype [phenotype]'], ['AnPME phenotype [phenotype]'], ['dwarf transgenic plant(s) [phenotype]'], ['bk4 phenotype [phenotype]'], ['small dark-green dwarfs [phenotype]'], ['brassinosteroid deficient [phenotype]'], ['the dwarf phenotype of shi plant(s) [phenotype]'], ['stronger dwarf phenotype at 22°C [phenotype]'], ['dwarfing in the bal variant [phenotype]'], ['dwarf and compact plant(s) [phenotype]'], ['seedling-lethal dwarf [phenotype]'], ['whole-plant dwarfing [phenotype]'], ['partially dwarfed [phenotype]'], ['substantial dwarfing [phenotype]']]}\n",
      "-------------------\n",
      "{'127626': [['rdr6 mutation [gene]', '**Mutations in RDR6 [gene]**'], ['**rdr6 mutations [mutant]**', 'mutations in RNA-DEPENDENT RNA POLYMERASE6 [mutant]'], ['mutations in rdr2-2 [gene]'], ['Mutations in RDR2 [mutant]'], ['Impairment of RDR2 [gene]'], ['Mutations in RDR1 [gene]'], ['mutations in RNA dependent RNA polymerase (RdRP) [gene]'], ['Knock-out of RDR6 [treatment]'], ['No significant contribution of RDR2 [gene]'], ['rdr mutations [mutant]']]}\n",
      "-------------------\n",
      "{'58544': [['100 mg L-1 of Cr(VI) [treatment]'], ['2500 mg L-1 of Cr(VI) [treatment]'], ['internal pH of 6.0 [treatment]'], ['pH 6 [treatment]'], ['0.05 mM K<sup>+</sup> [treatment]'], ['Experimental manipulation of Ca(2+)(i) [treatment]'], ['4 C in a 10 mM Na2HPO4 pH 5.7 buffer [treatment]'], ['low Ca:Mg ratios [treatment]'], ['nominal absence of Ca2+ [treatment]'], ['presence of 20 mM extracellular Ca2+ [treatment]'], ['0.1:1 Ca:Mg treatment [treatment]'], ['1:1 Ca:Mg treatment [treatment]'], ['pH 6.5-7 [treatment]'], ['0.2 mM As(III) [treatment]'], ['four different Ca:Mg treatments [treatment]'], ['CO2:O2 ratios [treatment]']]}\n",
      "-------------------\n",
      "{'69542': [['elimination of disulfide bond in ALIS5 [phenotype]'], ['C53-C62 disulfide bond [phenotype]'], ['C57-C64 disulfide bond [phenotype]'], ['C58-C65 disulfide bond [phenotype]'], ['C205-C213 disulfide bond [phenotype]'], ['long distance between TrxR C153 and disulfide bond of Trx C387-C390 [phenotype]'], ['same disulfide bond connectivity as Br S32 [phenotype]'], ['same disulfide bond connectivity as Br S36 [phenotype]'], ['disulfide bridge connectivity C1-C3, C2-C4, C5-C6, and C7-C8 [phenotype]'], ['disulfide formation of Cys138 [process]'], ['Cys94-Cys131 disulfide [process]'], ['disulfide bond Cys71-Cys337 [process]'], ['Cys548-Cys606 disulfide [protein]'], ['143-161 disulfide bond [secondary structure]'], ['35-198 disulfide bond [secondary structure]'], ['69-178 disulfide bond [secondary structure]'], ['C-terminal disulfide [chemical structure]'], ['(Cys52)-S-S-(Cys85) [bond]'], ['Cu-S(Cys79) [bond]'], ['Cys71-Cys337 disulfide bond [protein structure]'], ['Cys222-Cys231 disulfide bond [protein structure]'], ['Cys375-Cys378 disulfide bond [protein structure]'], ['Cys122/125/127-Cys151 disulfide bond [protein structure]'], ['Cys122-Cys127 disulfide bond [protein structure]'], ['single disulfide bond at Cys-400 [protein motif]'], ['disulfide bonds (Cys186-Cys406 and Cys349-Cys364) [subcellular compartment]'], ['Cys186-Cys406 disulfide bond [subcellular compartment]']]}\n",
      "-------------------\n",
      "{'65725': [['mSin3A [protein]'], ['tpl [protein]'], ['rTGA2.1 [protein]'], ['OsTPR2 [protein]'], ['yeast Tup1 [protein]'], ['Tup1 [protein]'], ['RAP2.3 [protein]'], ['SPT5L (Suppressor of Ty insertion 5 - like) [protein]'], ['WUS motif [protein]'], ['Sin3a [protein]'], ['an essential co-regulator of eukaryotic transcription [process]'], ['**Groucho [protein]**', 'Groucho/Tup1-type transcriptional co-repressor [protein]'], ['tTA (tet-OFF) system [protein]'], ['**transcriptional coregulator [function]**', 'transcription coregulator [function]'], ['human transcription repressor NF-X1 [protein]']]}\n",
      "-------------------\n",
      "{'152027': [['**chromosome A06 in B. rapa [genome]**', 'sequence on chromosome A06 in B. rapa [genome]'], ['B. rapa chromosome A09 [chromosome]'], ['A01 [chromosome]'], ['**A06 [chromosome]**', 'ChrA06 [chromosome]'], ['TCP11_A02 [gene]'], ['SOC1_A05 [gene]'], ['SOC1_A03 [gene]'], ['chromosome A03 [organ]'], ['chromosome A02 [organ]'], ['chromosome A06 [organ]'], ['chromosome A07 [organ]'], ['chromosome A04 [organ]'], ['Chromosome A05 [organ]'], ['Chromosome A08 [organ]'], ['chromosomes of the A03 sub-genome [genomic location]'], ['end of A09 [genetic location]'], ['chrC02 [chromosome]'], ['chrA02 [chromosome]'], ['ChrA03 [chromosome]'], ['ChrA05 [chromosome]'], ['ChrA07 [chromosome]'], ['ChrA04 [chromosome]'], ['A09 [genomic region]'], ['A01 [genomic region]'], ['A03 [genomic region]'], ['A06 [genomic region]'], ['A05 [genomic region]'], ['A02 [genomic region]'], ['Chromosome A09 [location]'], ['chromosome A02 [location]'], ['chromosome A07 [location]'], ['Chromosome A01 [location]'], ['chromosome A03 [location]'], ['Chromosome A05 [location]'], ['chromosome A06 [location]'], ['Chromosome A08 [location]'], ['chromosome A04 [location]']]}\n",
      "-------------------\n",
      "{'74602': [['HypSys [peptide]'], ['SYSTEMIN [molecule]'], ['HypSys [molecule]'], ['**SYSTEMIN [protein]**', 'tomato wound signalling peptide systemin [protein]', 'Systemins [protein]', 'from prosystemin [protein]'], ['SYSTEMIN [peptide ligand]']]}\n",
      "-------------------\n",
      "{'51757': [['RNAi knock-down of the BR signaling components, OsBAK1 in rice and ZmBRI1 in maize [mutant]'], ['Preferential binding of brassinolide (BL) to BRI [gene]'], ['cell division control protein(s) 6 B [gene]'], ['bHLH transcriptional activators for cell elongation (ACEs) [gene]'], ['DNA-binding activity of bHLH60 [gene]'], ['transcriptional regulatory activities of bHLH60 [gene]'], ['elements related to light regulation and MYB transcription factor binding [gene structure]'], ['binding sites for DEHYDRATION-RESPONSIVE ELEMENT BINDING (DREB) protein(s) [gene]'], ['DNA binding activity of transcription factors Fos and Jun [process]'], ['DNA-binding transcription factor activity (GO:0003700) [phenotype]'], ['helix-turned-angle helix structure of MYB transcription factor [gene]'], ['protein(s) activity of phytochrome interacting factor 4 (PIF4) [gene]']]}\n",
      "-------------------\n",
      "{'95962': [['early arrest of chloroplast development [organ]'], ['impaired chloroplast development [organ]'], ['major changes in achene [organ]'], ['degeneration of the bulb [organ]'], ['development of cryptic bract [organ]'], ['growth defects in sepals [organ]'], ['no differentiation of ring meristem at stage 6 [organ]'], ['severe defects in hook development [organ]']]}\n",
      "-------------------\n",
      "{'23981': [['genetic regulation of bulbing process [phenotype]'], ['regulation of lateral main bud outgrowth [phenotype]'], ['**control of bud outgrowth [phenotype]**', 'Regulation of bud outgrowth [phenotype]', 'bud outgrowth control [phenotype]'], ['regulation of ligule-ear development [phenotype]'], ['regulation of polar growth [phenotype]'], ['regulation of petal growth [phenotype]'], ['regulation of cambium activity [phenotype]'], ['control of ice formation [phenotype]'], ['control of L. bicolor salt gland differentiation [phenotype]'], ['**regulation of axillary buds [phenotype]**', 'regulation of axillary bud development [phenotype]', 'controlling axillary bud development [phenotype]', 'regulating growth of axillary buds [phenotype]', 'regulating the development of axillary buds [phenotype]', 'control of axillary bud outgrowth [phenotype]'], ['growth control in poplar [phenotype]'], ['control of auxin biology [phenotype]'], ['regulating tiller bud growth [phenotype]'], ['post apical dominance mechanism [phenotype]']]}\n",
      "-------------------\n",
      "{'45446': [['S2Lb loss-of-function plant(s) [mutant]'], ['MtBS1 loss-of-function mutants [mutant]'], ['Loss of function mutants of WRKY2, WRKY40 and WRKY63 [mutant]'], ['Loss of PRE5 [mutant]'], ['TRS33 LOF mutant [mutant]'], ['EXO70A1 LOF [mutant]'], ['At3g47520 loss-of-function mutants [mutant]'], ['loss-of-function mutants of GLS5 [mutant]'], ['Loss-of-function mutations in SMXL2 [mutant]'], ['agb1-2 null mutations [mutant]'], ['SCAP1 loss-of-function mutants [mutant]'], ['loss of function spl combinatorial mutants [mutant]'], ['Loss-of-function mutations in SbPRR37 [mutant]'], ['Loss of function in AtSWEET2 [mutant]'], ['single loss-of-function mutants of MYB96 [mutant]'], ['Knock-out of RGTB2 gene(s) [mutant]'], ['Loss of function mutations in cellulose synthesis [mutant]'], ['loss-of-function of SlMLO1 [mutant]'], ['SlBZR1 loss-of-function mutants [mutant]'], ['mutants impaired in pPGI [mutant]'], ['loss-of-function mutations in HPY2 [mutant]'], ['MSH1 loss [mutant]'], ['Deletion of any of the six subunits [mutant]'], ['Loss-of-function mutations in AHA1 and 2 [mutant]'], ['BrMYB108 loss-of-function mutants [mutant]'], ['loss-of-function of BBX29 [mutant]'], ['Loss-of-function AtBAG5 mutant plant(s) [mutant]'], ['Loss-of-function in pepper plant(s) [mutant]'], ['H2Bub loss-of-function mutations [mutant]'], ['SPR1 loss-of-function mutants [mutant]'], ['loss-of-function mutant in RBR gene(s) [mutant]'], ['Loss of function of SWR1-C subunits [mutant]']]}\n",
      "-------------------\n",
      "{'19353': [['high-NUE genotypes [phenotype]'], ['traits for NUE [phenotype]'], ['higher NUE [phenotype]', '**high NUE [phenotype]**', 'greater NUE [phenotype]', 'elevated NUE [phenotype]'], ['NUE under high ammonium conditions [phenotype]'], ['high NUE cultivars [phenotype]'], ['high NUE rootstocks [phenotype]'], ['high and low NUE [phenotype]']]}\n",
      "-------------------\n",
      "{'76792': [['XXNRLD-type catalytic motif variants [protein motif]'], ['EXDXE conserved active-site motif [protein motif]'], ['EXDXE [sequence]'], ['**EXDXE motif [genomic feature]**', 'EXDXE conserved active-site motif [genomic feature]'], ['conserved HXHXD motif [protein motif]'], ['HExxH motif [protein motif]'], ['ExxER motif in TM1 [amino acid sequence]'], ['N-x7-R/K motif [metabolite]'], ['KxD and RxxxD [motif]'], ['conserved motif of active site [sequence]'], ['**Nudix box [protein motif]**', 'Nudix box motif [protein motif]', 'nudix motif [protein motif]'], ['**Nudix box [protein feature]**', 'nudix motif [protein feature]'], ['ExxER motif [subcellular compartment]'], ['ExxERF motif [gene]'], ['ExxER [structural element]'], ['GDXnERKX3D nuclease active site motif [protein domain]'], ['ExxER motif [gene]'], ['EXXEK motif [protein motif]'], ['EXXR motif [protein motif]'], ['REXXEE motif [protein motif]'], ['HxEx(n)CxxC motif [protein motif]'], ['Nudix box [sequence motif]'], ['HX(T/S)X motifs [protein motif]'], ['YXXXK motif [protein motif]'], ['NFYYxxED motif [phenotype]'], ['putative catalytic motif DXDXT [gene]'], ['RXS motif [protein]'], ['ExoI [sequence motif]'], ['ExoII [sequence motif]'], ['ExoIII [sequence motif]'], ['XXHQLD core motif [phenotype]'], ['conserved active-site motif EXDXE [protein domain]'], ['DnaQ family exonuclease fold [protein complex]'], ['HxPGDDPxxxExx [sequence]'], ['motif RE92LQEE [protein motif]'], ['canonical GDXnERKX3D [protein sequence]'], ['typical catalyze enzymatic reaction motif (HDEIDFEFLG) [gene]'], ['KxNG motif I of fungal tRNA ligases [protein motif]'], ['first HxT motif [protein motif]'], ['likely catalytic roles of the HxT motifs and other functional groups in the active site [protein]'], ['highly conserved ExDxEx [phenotype]'], ['SQPWPFPxS motif [protein motif]'], ['DIOX_N motif [protein motif]'], ['a conserved DXXDXDXE motif [structure]'], ['EXXEK/R motif [gene]'], ['E(x)6Y motif [protein domain]'], ['DxDxE motif [protein structure]'], ['HCX5R motif [protein motif]'], ['DxD motifs [subcellular compartment]'], ['GXGXXG fingerprint motif [gene]'], ['conserved HX(4)D motif [sequence motif]']]}\n",
      "-------------------\n",
      "{'55035': [['OsPHYB::YFPC fusion protein(s) [protein]'], ['OsPHYA::YFPC fusion protein(s) [protein]'], ['OsPHYC::YFPC fusion protein(s) [protein]']]}\n",
      "-------------------\n",
      "{'127419': [['microheterogeneity [phenotype]'], ['**n3PASs [phenotype]**', 'non-3UTR polyadenylation event [phenotype]'], ['non-conserved poly(A) sites [phenotype]'], ['There is no collection of poly(A) sites [phenomenon]'], ['potentially undesirable poly(A) sites [genomic feature]'], ['undesirable poly(A) sites [genomic feature]'], ['cryptic poly(A) sites [phenotype]'], ['**antisense poly(A) sites [phenotype]**', 'formation of opposite poly(A) sites [phenotype]'], ['novel n3PASs [phenotype]'], ['m6A-independent PAS [phenotype]'], ['poly(A) site evolution in mammalian gene(s) in humans and mouse [phenomenon]'], ['a rather low extent of evolutionary conservation of APA involving intronic poly(A) sites [phenomenon]'], ['arrhythmic poly(A) sites [sequence]'], ['poly(A) site usage within non-canonical sites [process]']]}\n",
      "-------------------\n",
      "{'25174': [['co-expression networks [phenotype]'], ['four attractors [phenotype]'], ['10 fixed point attractors [phenotype]'], ['opposite temporal expression patterns [phenotype]']]}\n",
      "-------------------\n",
      "{'96013': [['temperature at leaf level [organ]'], ['**leaf length [organ]**', 'length of leaf blades [organ]', 'length of the leaf [organ]', 'tip to base of leaf [organ]'], ['Hypocotyl length [organ]'], ['3.6 C in leaf [organ]'], ['lamina width [organ]', '**Leaf width [organ]**'], ['**Leaf blade dimensions [organ]**', 'leaf blade size [organ]'], ['overall thickness of leaf [organ]'], ['petiole length of light-grown plant(s) [organ]'], ['leaf thickness of 0.21 mm [organ]'], ['leaf thickness of 0.23 mm [organ]'], ['leaf thickness of 0.26 mm [organ]'], ['leaf thickness of 0.29 mm [organ]'], ['8.33 vegetative leaf [organ]'], ['Petiole length [organ]'], ['leaf and petiole sizes [organ]'], ['leaf sheath length [organ]'], ['length determination of petiole [organ]'], ['size reduction of leaf [organ]'], ['48 landmarks around the margin of each leaf [organ]'], ['size of true leaf [organ]'], ['leaf steady [organ]'], ['true leaf length [organ]'], ['leaf area basis [organ]'], ['tip to base in the oldest leaf [organ]'], ['the same leaf position [organ]']]}\n",
      "-------------------\n",
      "{'103231': [['**yeast strain 9.3 [organism]**', '9.3 strain [organism]', '9.3 yeast strain [organism]'], ['yeast strain AXT3 [organism]'], ['EBY.VW4000 yeast strain [organism]'], ['YK44 yeast [organism]', '**yeast strain YK44 [organism]**']]}\n",
      "-------------------\n",
      "{'22920': [['core structure pathway [phenotype]'], ['hydrolysis of glucose moiety [phenotype]'], ['**GIPC glycosylation [phenotype]**', 'degree of GIPC glycosylation [phenotype]', 'type of GIPC glycosylation [phenotype]'], ['studies of XyG function [phenotype]'], ['**GLS action [phenotype]**', 'Shift in allyl GLS hydrolysis in shoot tissue [phenotype]', 'structural and functional diversity of GLS [phenotype]', 'glucosinolate structural profile [phenotype]', 'GSL breakdown [phenotype]', 'the development of chemically separate GS types [phenotype]'], ['plant(s) specific modifications [phenotype]'], ['N-Glycosylation patterns [phenotype]'], ['**Glycome profiling [phenotype]**', 'glycome-profiling results [phenotype]', 'glycome profiling results [phenotype]'], ['changes in GX degree of polymerization [phenotype]'], ['structural diversity of product profiles [phenotype]'], ['SG production [phenotype]'], ['flexibility in glycosylation patterning [phenotype]'], ['modification of side chain [phenotype]'], ['altered glycosidic linkages [phenotype]'], ['glycolipid analysis [phenotype]'], ['partial reversion of glycome profile pattern [phenotype]'], ['attaching a sugar to flavonoids [phenotype]'], ['signal with antibody against N-glycans [phenotype]'], ['spontaneous rearrangement [phenotype]'], ['glycosylation in stigmas [phenotype]']]}\n",
      "-------------------\n",
      "{'55183': [['specialized and customized complexes [protein complex]'], ['specific complex [protein complex]'], ['smaller complexes [protein complex]'], ['**high molecular weight multiprotein complexes [protein complex]**', 'higher molecular complexes [protein complex]', 'large macromolecular complexes [protein complex]'], ['stable multiprotein complexes [protein complex]'], ['conglomerates [protein complex]'], ['biochemically stable NPC subcomplexes [protein complex]'], ['multi-subunit protein(s) complex [protein complex]'], ['various protein(s) complexes [protein complex]'], ['discrete bands [protein complex]'], ['polarity complex [protein]'], ['novel complex [protein complex]'], ['complexes containing TFs [protein complex]'], ['separate community clusters [protein complex]'], ['ten partners [protein complex]'], ['protein:protein networks [protein complex]'], ['multiple proteomes [protein complex]'], ['formation of protein-protein complexes [process]'], ['17 potential protein(s) interaction networks [protein complex]'], ['**1:1 non-covalent complex [protein complex]**', 'specific and stable stoichiometric 1:1 complex [protein complex]'], ['protein-DNA complexes called nucleoids [protein complex]']]}\n",
      "-------------------\n",
      "{'111095': [['hormone response elements [genetic element]', '**Hormone-responsive elements [genetic element]**'], ['hormone response elements (HREs) [process]', '**Hormone-responsive elements [process]**', 'hormonal responsive elements [process]'], ['hormone responsive elements [genomic element]', '**Hormone-responsive elements [genomic element]**'], ['Various hormone responsive elements [gene]'], ['hormone responsive elements [feature]', '**Hormone-responsive elements [feature]**', 'hormone response elements [feature]'], ['**Hormone-responsive elements [gene element]**', 'hormone response element [gene element]', 'hormonal responsive elements [gene element]'], ['hormone responsive elements [phenotype]'], ['phytohormone response elements [genomic feature]'], ['hormone-response-related regulatory elements [treatment]'], ['potential hormone-responsive elements [treatment]'], ['bHLH-responsive element [treatment]'], ['hormone response elements [regulatory element]', 'hormone-responsive motifs [regulatory element]', '**Hormone-responsive elements [regulatory element]**'], ['**Hormone-responsive elements [treatment]**', 'hormone response elements [treatment]'], ['hormone response elements [genetic feature]'], ['Hormone-responsive elements [sequence]'], ['hormone-response elements [gene]', '**Hormonal-responsive elements [gene]**', 'hormonal responsive elements (HREs) [gene]'], ['hormone response elements [gene feature]', '**hormone-response elements [gene feature]**'], ['hormone response elements [element]'], ['hormone-related responsive elements [gene structure]'], ['Hormone-responsive elements [gene structure]'], ['hormone response element [dna sequence]'], ['Hormone-responsive elements [genetic property]'], ['multiple hormone-responsive elements [phenotype]'], ['phytohormone responsive elements [protein domain]'], ['multiple hormone-responsive elements [sequence]']]}\n",
      "-------------------\n",
      "{'155976': [['SIP [gene]'], ['SIP1 [gene]'], ['**AtDIR1-like [gene]**', 'DIR1-like (At5g48490) [gene]'], ['Homo sapiens PDIA1 [gene]'], ['SiPTI1-5 [gene]'], ['SiPTI1-6 [gene]'], ['SiPTI1-7 [gene]'], ['SiPTI1-8 [gene]'], ['SiPTI1-9 [gene]'], ['SiPTI1-11 [gene]'], ['SiPTI1-12 [gene]'], ['SiPTI1-4 [gene]'], ['SiPTI1-10 [gene]'], ['SiPTI1-1 [gene]'], ['SiPTI1-3 [gene]'], ['IR form of SDIRIP1 [gene]'], ['SIP1 gene(s) [gene]'], ['GID1b1 [gene]'], ['GID1b2 [gene]'], ['GID1b3 [gene]'], ['hipI-SODC1b [gene]'], ['MdSIP1-2 [gene]'], ['SIP1;2 and SIP1;1 [gene]'], ['only SIP1 homologue [gene]'], ['SIP1 clade [gene]'], ['one SIP3 [gene]'], ['PslGID1b [protein]'], ['hipI-SODC1s [gene]']]}\n",
      "-------------------\n",
      "{'93457': [['**Early embryogenesis [tissue]**', 'very early embryos [tissue]', 'young embryos [tissue]', 'early stage of embryo [tissue]', 'initial cells of embryo [tissue]', 'Early embryos [tissue]', 'early embryo development [tissue]', '4-day embryos [tissue]', 'early embryo [tissue]'], ['**whole seed at heart stage [tissue]**', 'seeds containing heart stage embryos [tissue]'], ['early embryo and seed coats [tissue]'], ['late embryos [tissue]'], ['whole embryos [tissue]'], ['seed embryos [tissue]'], ['**early developmental seeds [tissue]**', 'very young seed [tissue]'], ['walnut embryo [tissue]'], ['developing embryos of oilseed crops [tissue]'], ['early development of leaf primordia [tissue]'], ['developing X. sorbifolia embryos [tissue]'], ['dry seed embryo [tissue]'], ['embryos at various stages of development [tissue]']]}\n",
      "-------------------\n",
      "{'122504': [['same role [phenotype]', 'similar roles [phenotype]', 'similar activity [phenotype]', 'similar functional properties [phenotype]', 'functionally very similar [phenotype]', 'similar biological roles [phenotype]', 'similar molecular activity [phenotype]', '**the same biological function [phenotype]**', 'similar role [phenotype]'], ['similar functions [function]'], ['similar roles [process]'], ['similar functions [phenomenon]']]}\n",
      "-------------------\n",
      "{'146950': [['HD-Zip IV gene(s) Glyma11g00570 [gene]'], ['HD-Zip IV gene(s) Glyma13g43350 [gene]'], ['HD-Zip IV gene(s) Glyma13g38430 [gene]']]}\n",
      "-------------------\n",
      "{'25896': [['**growth rate defect [phenotype]**', 'growth slower defects [phenotype]', 'severe defect in plant(s) growth [phenotype]'], ['more pronounced defect in phase IV growth [phenotype]']]}\n",
      "-------------------\n",
      "{'137085': [['reduced expression of RBE, HAN and JAG [gene]'], ['Reduced activity of SHINE transcription factors [gene]'], ['reduced expression of GGLO1 [gene]'], ['Reduction in the expression of GDEF2 [gene]', '**Reduced GDEF2 expression [gene]**'], ['**Reduced B class gene(s) expression [gene]**', 'decreased expression of B function gene(s) [gene]'], ['decreased transcription of PI [gene]'], ['reduced expression of B gene(s) CvAP3 [gene]']]}\n",
      "-------------------\n",
      "{'151274': [['PpMIP [gene]'], ['PpCHUP1 [gene]'], ['Pp1s63_162v6.1 [gene]'], ['Pp1s63_163v6.1 [gene]'], ['Pp1s63_165v6.1 [gene]'], ['PpIDD4 [gene]'], ['PpIDD6 [gene]'], ['PpHIP1;1 [gene]'], ['PpXIP1;1 [gene]'], ['PpXIP1;2 [gene]'], ['PpNIP5;3 [gene]'], ['PpNIP6;1 [gene]'], ['PpNIP5;1 [gene]'], ['PpNIP5;2 [gene]'], ['PpRALF1 [gene]'], ['PpAN2-1 [gene]'], ['PpNIP3 [gene]'], ['PpNIP6 [gene]']]}\n",
      "-------------------\n",
      "{'29536': [['**nonselective autophagy [process]**', 'Non-selective autophagy [process]', 'bulk autophagy [process]', 'bulk degradation of the cytoplasm [process]', 'bulk autophagic degradation [process]', 'autophagy-mediated non-selective degradation [process]', 'bulk degradation of cytoplasmic components [process]'], ['bulk autophagy [phenotype]'], ['bulk and selective cleavage of cellular components [process]']]}\n",
      "-------------------\n",
      "{'128175': [['increased loading of ribosomes [phenotype]', '**increase in ribosome loading [phenotype]**', 'consistent global increase in ribosome loading [phenotype]'], ['decrease in ribosome loading [phenotype]'], ['increase in mRNA ribosome loading [phenotype]'], ['increased ribosome occupancy [phenotype]', 'positive change in ribosome occupancy [phenotype]', '**increase in ribosome occupancy [phenotype]**', 'high frequency of ribosome occupancy [phenotype]'], ['80S ribosome occupancy [phenotype]'], ['**increase in 80S monosomes [phenotype]**', 'increase in 80S monoribosomes [phenotype]'], ['higher ribosome density [phenotype]'], ['increase in ribosomes [phenotype]'], ['increase in ribosomal subunits [phenotype]'], ['stronger 80S signals [phenotype]'], ['transcript-specific increases in ribosome association [phenotype]'], ['gain of ribosomes [process]']]}\n",
      "-------------------\n",
      "{'149112': [['cnu_m244a [gene]'], ['BnGPT1-A2 [gene]'], ['BnSIZ1-A2 [gene]'], ['BnBBM [gene]'], ['BnaC03g56870D [gene]'], ['BnC08.JAZ1-1 [gene]'], ['BnaA04g02480D [gene]'], ['MI7092 [gene]'], ['vector pWY86-1 [gene]'], ['BnaA09g15400D [gene]'], ['BnaA08g06910D [gene]'], ['BnaC06g17400D [gene]'], ['BnaA05g08910D [gene]'], ['J9709 allele [gene]'], ['BnEMS1125 [gene]'], ['BnaMYB21 [gene]'], ['BnaMYB150 [gene]'], ['BnSDJ1 [gene]'], ['128-kb interval on C02 [genomic region]']]}\n",
      "-------------------\n",
      "{'3641': [['**competitive strategies [phenotype]**', 'competitive behaviors [phenotype]'], ['Cooperative strategies in natural plant(s) communities [phenotype]'], ['**allocation of limited resources [phenotype]**', 'allocating resources [phenotype]', 'reallocation of energy and resources during defense responses [phenotype]'], ['tradeoffs between growth and defense [phenotype]', '**growth-defense trade-offs [phenotype]**', 'fine tune plant(s) growth and defense [phenotype]', 'regulating growth-defense trade-offs [phenotype]', 'accommodate growth and defense [phenotype]', 'optimization of growth and defense in nature [phenotype]', 'growth-defense continuum [phenotype]'], ['energy switch [phenotype]'], ['**tradeoffs between resistance and growth [phenotype]**', 'trade-off between growth and resistance [phenotype]'], ['efficient strategies for growth or defense [phenotype]'], ['bet-hedging strategy between defense and regeneration [phenotype]'], ['adaptive and low-fitness-cost defense mechanism [phenotype]'], ['energy to capture prey [phenotype]'], ['trade-offs between other defenses [phenotype]'], ['tradeoff between plant(s) growth and UV-B defense responses [phenotype]'], ['trade-offs relationship [phenotype]']]}\n",
      "-------------------\n",
      "{'75154': [['cognate effectors [protein]', '**cognate effector [protein]**', 'cognate effector protein(s) [protein]'], ['recognized effectors [protein]'], ['defeated effectors [protein]'], ['cognate pathogen effectors [protein]'], ['corresponding effectors [protein]'], ['effector receptors [protein]'], ['pathogen effector intracellular receptors [protein]']]}\n",
      "-------------------\n",
      "{'159628': [['FBP21/22 and AtSOC1 [gene]'], ['FBP28-SOC1 [gene]']]}\n",
      "-------------------\n",
      "{'62115': [['GFP-tagged ATG5 protein(s) [protein]'], ['TPLATE-GFP [protein]'], ['GFP-tagged TCPs [protein]'], ['P30-2XGFP [protein]'], ['**UBP22-GFP [protein]**', 'GFP-UBP22 [protein]'], ['PmoA-GFP [protein]'], ['PmoB-GFP [protein]'], ['EMP12-GFP [protein]'], ['GFP-tagged U1-70K [protein]'], ['GFP-TRB1/TRB2 [protein]'], ['GFP-tagged AtSTKLs [protein]'], ['green fluorescent protein(s) (GFP)-tagged atRSp31 [protein]'], ['GFP-ARPC2 [protein]'], ['GFP-tagged kinase subunits [protein]'], ['GFP-tagged POI [protein]'], ['GFP-fused TPLATE [gene]'], ['GFP-tagged ribosomal protein(s) RPL18 [protein]'], ['SPCH-GFP protein(s) [protein]'], ['GFP-tagged TPC subunit [protein]'], ['GFP-tagged TUA6 protein(s) [gene]']]}\n",
      "-------------------\n",
      "{'143526': [['TCP transcription factor TB1 [gene]'], ['**TCP transcription factor family [gene]**', 'TCP Transcription Factor gene(s) family [gene]'], ['TCP9 transcription factor [gene]'], ['**TCP transcription factor [protein]**', 'TCP transcription factors [protein]', 'TCP factors [protein]'], ['TCP transcription factors [gene]', 'TCP transcription factor gene(s) [gene]', '**TCP transcription factor [gene]**', 'TCP transcription factors (TFs) [gene]', 'TCP-type transcription factors [gene]', 'TCP transcriptional regulators [gene]'], ['TCP1 transcription factors [gene]'], ['TCP transcription factor [protein domain]'], ['**TCP transcription factor family [protein family]**', 'TCP transcription factors [protein family]'], ['TCP transcription factors [gene family]', '**TCP transcription factor family [gene family]**'], ['TCP/site2 [gene]'], ['TCP-domain transcription factors [protein]', '**TCP-domain transcription factor [protein]**']]}\n",
      "-------------------\n",
      "{'55001': [['35S::GFP + GSF fusion protein(s) [protein]'], ['GFP fused to native protein(s) [protein]'], ['LOS4-GFP fusion [protein]'], ['TSWV NSm:GFP fusion [protein]'], ['CMV MP:GFP fusion [protein]'], ['ZmTSA::GFP fusion protein(s) [protein]'], ['ZmTSAlike::GFP fusion protein(s) [protein]'], ['VAMP714:GFP fusion [protein]'], ['full-length GFP fusions of HRZ or BTS [protein]'], ['full-length GFP fusions [protein]'], ['GFP-fused MebHLH72 protein(s) [protein]'], ['GFP-fused MebHLH114 protein(s) [protein]'], ['GFP::AtRHM1 fusion protein(s) [protein]'], ['GFP::ARR22 fusion protein(s) [protein]'], ['GFP:MP fusion constructs [protein]'], ['GFP:MPwt fusion [protein]'], ['MP17:GFP fusion [protein]']]}\n",
      "-------------------\n",
      "{'145283': [['TAS3 transcript [gene]'], ['TAR2 expression [gene]'], ['LBD16/LBD18 transcriptional module [gene]'], ['transcriptional regulator (bdl) [gene]'], ['AUX1/2 [cis-element]'], ['SPARCs [plasmid]'], ['class III [gene]'], ['**transcriptional activator MONOPTEROS (MP) [gene]**', 'the MONOPTEROS (MP) transcription factor [gene]'], ['auxin-inducible transcript [gene]'], ['MADS-box transcription factor ANR1 [gene]'], ['DI (EAR) domain [protein domain]'], ['MP-dependent transcriptional activation [phenotype]'], ['Transcripts encoding protein(s) associated with auxin responses [gene]'], ['**TabZIP6D-20 [gene]**', 'TabZIP6D_20 [gene]'], ['Glyma.13G325200 [gene]'], ['Glyma.12G076200 [gene]'], ['HD-ZIPIII transcription factor REVOLUTA [gene]'], ['MAX4 transcripts [gene]'], ['TRX reductase (ntra ntrb) [gene]'], ['MP11ir isoform [gene]'], ['auxin signaling transcripts [phenotype]'], ['RIC effectors [gene]'], ['auxin transcriptional reporters [gene]'], ['TOPLESS corepressors [gene]', 'TOPLESS repressor [gene]', '**TPL (Topless) [gene]**', 'TOPLESS (TPL) co-repressors [gene]'], ['52 potentially interacting transcriptional regulators [gene]'], ['AUX/LAX transcription [gene]'], ['mRNAs for auxin-dependent transcription factors [gene]'], ['mRNAs of GH3-3 [gene]'], ['messenger RNAs for auxin receptors [gene]'], ['auxin transcriptional response marker [treatment]'], ['typical domain I [phenotype]'], ['auxin NAC [genomic element]'], ['auxin-regulated transcripts [phenotype]'], ['significant down-regulation of auxin-responsive factors [gene]'], ['critical functional domains of auxin receptors [gene]'], ['transcripts of several auxin receptors [gene]'], ['essential for rapid transcriptional responses [treatment]'], ['post-transcriptional silencing of auxin receptor transcripts [gene]'], ['auxin-inducible transcript [transcript]']]}\n",
      "-------------------\n",
      "{'63504': [['anti-PsaA antibodies [protein]'], ['PsaL protein(s) levels [protein]', '**level of PsaL [protein]**'], ['PsaB content [protein]'], ['protein(s) levels of PSI (PsaA, PsaO) [protein complex]'], ['PSI protein(s) PSAE [protein]'], ['PsaH-1 [protein]'], ['PSI assembly factor PSA3 [protein]'], ['PsaH level [protein]'], ['LOW PSII ACCUMULATION1 (LPA1) [protein]'], ['higher PsaE protein(s) abundance [protein]'], ['stabilization of PsaH [protein]'], ['stabilization of PsaO [protein]'], ['accumulation of PsaC subunit of photosystem I (PSI) [protein complex]'], ['PsaA level [protein]'], ['the content of photosystem protein(s) (PsaB, PsaA, and PsbO) [phenotype]'], ['maturation of PsaA [protein]'], ['maturation of PsaB [protein]'], ['maturation of PsaC [protein]'], ['wild-type levels of PsaF [protein]'], ['PsaB-PsaI-PsaH side of PSI [protein complex]']]}\n",
      "-------------------\n",
      "{'159448': [['**intricate genetic network [gene]**', 'an intricate network with various genetic regulators [gene]', 'a complex network of transcriptional activators and repressors [gene]'], ['deciphering the complex regulatory network [process]'], ['**a complex genetic regulatory network [process]**', 'complex and precise regulatory network [process]'], ['Flowering regulatory networks [process]', '**Flowering regulatory network [process]**'], ['complex network of floral regulators [gene network]'], ['Flowering regulatory network [genetic element]'], ['essential behaviour of the complex network [model]'], ['Flowering regulatory network [gene]']]}\n",
      "-------------------\n",
      "{'90048': [['Blocking CK perception or signaling pathways [process]'], ['negative regulation of cytokinin-activated signaling pathway [process]'], ['**reduced cytokinin signaling [process]**', 'reduced CK signalling [process]', 'suppressing cytokinin signaling [process]', 'inhibitors of CK signalling [process]'], ['negative regulator of cytokinin signaling [signaling pathway]'], ['inhibitory effect of CK-signaling pathway [treatment]'], ['negative regulators of cytokinin responses [process]']]}\n",
      "-------------------\n",
      "{'37023': [['**PIN7 vacuole localization [phenotype]**', 'vacuole localization of PIN7 [phenotype]'], ['Intracellular localization of PIN7 [phenotype]'], ['**PIN1 subcellular localization [phenotype]**', 'subcellular localization of PIN1 [phenotype]'], ['**PIN subcellular localization [phenotype]**', 'Subcellular localization of PIN protein(s) [phenotype]'], ['intracellular localisation of HvPIN1a [phenotype]']]}\n",
      "-------------------\n",
      "{'141304': [['CCOAOMT1 promoter [gene]'], ['COMT promoter [gene]'], ['CCoAOMT promoter [gene]']]}\n",
      "-------------------\n",
      "{'33859': [['subcellular localization [location]'], ['subcellular localization [structure]'], [\"protein's cellular localization [subcellular compartment]\", 'subcellular localization of protein(s) [subcellular compartment]', 'subcellular protein(s) localization [subcellular compartment]', '**Localization [subcellular compartment]**', 'their subcellular localization [subcellular compartment]', 'Cellular localization [subcellular compartment]', 'subcellular localisation of protein(s) [subcellular compartment]', 'localization in the cell [subcellular compartment]', 'localization within the cell [subcellular compartment]', 'localization in subcellular compartments [subcellular compartment]', 'intracellular protein(s) localization [subcellular compartment]', 'localization of protein(s) in cell compartments/structures [subcellular compartment]', 'localization in a given cellular compartment [subcellular compartment]', 'localization in cell [subcellular compartment]'], ['subcellular localization mechanism [subcellular compartment]'], ['**subcellular localization of protein(s) [process]**', 'Subcellular locations [process]', 'subcellular location of protein(s) [process]', 'subcellular protein(s) localization [process]'], ['subcellular localization of a protein(s) [protein]', '**localization of protein(s) [protein]**', 'localization of the protein(s) within the cell [protein]'], ['Subcellular deposition [subcellular compartment]'], ['Sub-cellular localization [description]'], ['membrane-localized protein(s) [subcellular compartment]'], ['Rho subcellular localization [subcellular compartment]'], ['retention of cargoes [subcellular compartment]'], ['multi-subcellular localization [process]'], ['cell type-specific subcellular localization [subcellular compartment]'], ['subcellular localization of alpha-tubulin protein(s) [subcellular compartment]'], ['subcellular localization of functional protein(s) [subcellular compartment]'], ['local concentrations of enzymes [protein]'], ['**dual localization [subcellular compartment]**', 'Dual targeting [subcellular compartment]', 'dual nature of sub-cellular localization [subcellular compartment]'], ['in different organelles [subcellular compartment]'], ['**sub-cellular distributions [subcellular compartment]**', 'localized subcellular distribution [subcellular compartment]'], ['**subcellular localization of target protein(s) [subcellular compartment]**', 'sub-cellular localization of target protein(s) [subcellular compartment]'], ['**determination of subcellular localization [subcellular compartment]**', 'determining subcellular localization of protein(s) [subcellular compartment]'], ['subcellular localization of polypeptides [location]'], ['polar subcellular localization [subcellular compartment]'], ['differential subcellular localization [subcellular compartment]'], ['subcellular localization and GO terms [subcellular compartment]'], ['subcellular localization of ion transporters [subcellular compartment]'], ['ubiquitous localization within the cell [subcellular compartment]'], ['subcellular localization [compartment]'], ['mostly in membrane systems [subcellular compartment]'], ['subcellular compartment expression [subcellular compartment]'], ['definite subcellular localization [subcellular compartment]'], ['annotations of cellular compartments [description]'], ['special subcellular localization [subcellular compartment]'], ['subcellular localization [protein complex]'], ['sub cellular localization [biological concept]'], ['dual localization [process]'], ['lumen compartment localization [subcellular compartment]'], ['subcellular compartmentalization of protein(s) [subcellular compartment]'], ['distribution of the protein(s) [subcellular compartment]'], ['subcellular localization properties [subcellular compartment]'], ['subcellular localization of membrane protein(s) [subcellular compartment]'], ['proper protein(s) localization [subcellular compartment]']]}\n",
      "-------------------\n",
      "{'77432': [['IQ domains [protein domain]'], ['Toprim domains [protein domain]'], ['homeodomains [protein domain]'], ['Per/Arnt/Sim domains [protein domain]'], ['Prion-like domains (PrDs) [protein domain]'], ['three CRM domains [phenotype]'], ['CRM domains [protein domain]'], ['candidate prion domains (PrDs) [protein domain]'], ['CATH-based protein(s) domain assignments [process]']]}\n",
      "-------------------\n",
      "{'112493': [['motif group III [gene]'], ['motif 3 [sequence motif]'], ['motif 11 [sequence]'], ['Motif 1 [gene region]'], ['Motif 2 [gene region]'], ['Motifs 6, 5, and 10 [gene region]'], ['Motifs 6 and 5 [gene region]'], ['Motif 7 [gene region]'], ['Motif 12 [gene region]'], ['motif 38 [gene region]'], ['Motif 4 [gene region]'], ['motif 8 [gene region]'], ['motif 9 [gene region]'], ['motif based on tested GCs [gene]'], ['motif region in NRPD1 alignment [gene]'], ['N-terminal of motif 1 [gene region]'], ['motif sequences [gene region]']]}\n",
      "-------------------\n",
      "{'90683': [['Co-expression with chain elongation pathway [treatment]'], ['chemical-inducible activation tagging system [treatment]', 'chemically inducible dimerization system [treatment]', '**chemically inducible system [treatment]**', 'limitations of chemically inducible systems [treatment]'], ['ectopic expression of transcription factors [treatment]'], ['removal of multiple growth suppressing transcriptional factors [treatment]'], ['HsPIP5K induction [treatment]'], ['ETI activation [treatment]'], ['transient transactivation assay [treatment]'], ['inducer concentration [treatment]'], ['concentration of inducer IPTG [treatment]'], ['Ectopic TCP activity [treatment]'], ['transient chemical induction [treatment]'], ['trans-differentiation assay [treatment]'], ['Ectopic expression of single transcription factors [treatment]'], ['induction of ANT-GR activity [treatment]'], ['Temporary induction of CUC2 [treatment]'], ['SCR induction [treatment]'], ['transfection [treatment]'], ['injection of lithium chloride [treatment]'], ['ITMV [treatment]'], ['Presence of inducer [treatment]'], ['tissue-specific inducible ablation [treatment]'], ['Rapid regulation [treatment]'], ['induction with IPTG [treatment]'], ['short-term ipt activation [treatment]'], ['application of detergent [treatment]'], ['Inducing the differentiation of specific cell type(s) [treatment]'], ['Chemically induced expression of the C27 domain [treatment]'], ['ProDH inducers (exogenous Pro treatment) [treatment]'], ['TOC1 chemical induction [treatment]'], ['induction by salt [treatment]'], ['Inducible knock-down or overexpression of transcription factors [treatment]'], ['exogenously supplied ammonia [treatment]'], ['inducible cell differentiation [treatment]'], ['direct transcriptional regulation of arsenite [treatment]'], ['inducible manner [treatment]'], ['copper induction concentration [treatment]'], ['copper induction [treatment]'], ['translationally induced [treatment]'], ['anoxic specific inducibility [treatment]'], ['alkaline stress inducible [treatment]'], ['co-activator [treatment]'], ['arsenate-induced transposon activation [treatment]'], ['biotin-limited conditions [treatment]'], ['targeted misexpression of diphtheria toxin [treatment]'], ['exogenous suppressors [treatment]'], ['omega-amino acid induction [treatment]'], ['CuSO<sub>4</sub> treatment [treatment]']]}\n",
      "-------------------\n",
      "{'156064': [['**Spo11 [gene]**', 'SPO-11 [gene]', 'SPO11 gene(s) [gene]', 'SPORULATION11 (SPO11) [gene]', 'meiotic SPO11 [gene]', 'SPO11 (SPORULATION 11) [gene]'], ['spo11-2 [gene]'], ['Ancestral duplication of SPO11 [gene]'], ['two meiosis-expressed SPO11 paralogues [gene]'], ['AtSPO11-3 [gene]'], ['Rec12 [gene]'], ['sporulation11-1 (spo11-1) [gene]', '**SPORULATION 11-1 (SPO11-1) [gene]**', 'gene(s) SPO11-1 [gene]'], ['Spp1 [gene]'], ['expression of Spo11beta gene(s) [gene]'], ['AtSPO11-1 and AtSPO11-2 [gene]'], ['MTOPIVB [gene]'], ['Function of SPO11 [gene]'], ['late detectable SPO11-2 [gene]'], ['SPO11 enzymes [gene]'], ['Spo11p [gene]'], ['SPO76 [gene]'], ['**SPO11 candidate(s) [gene]**', 'SPO11 candidates [gene]'], ['DmSPO11 [gene]'], ['S. cerevisiae Spo11 protein(s) [gene]'], ['topoisomerase-like SPO11 [gene]'], ['Osspo11-2 [gene]'], ['Osspo11-4 [gene]'], ['Spo11-3 (Top6A) [gene]'], ['AtSPO11-1 cDNAs [gene]'], ['presence of SPO11-2 [gene]'], ['SPO11-dependent DSBs [gene]'], ['functional SPO11 alleles [gene]'], ['native SPO11 gene(s) [gene]'], ['SPO11 in mice [gene]'], ['SPO11 in humans [gene]'], ['budding yeast SPO11-1 [gene]'], ['mouse SPO11-1 [gene]'], ['Spo11-dependent [gene]'], ['SPO22 [gene]'], ['T05E11.4 [gene]'], ['host SPO11 [gene]'], ['multiple functional SPO11 candidates [gene]'], ['**Spo11 in meiotic recombination [gene]**', 'Spo11 for meiotic recombination [gene]'], ['spo11 gene(s) expression [gene]'], ['OsP31 COMET [gene]'], ['**1 TopVIA/Spo11 homologue [gene]**', 'one homologue of TopVIA (Spo11) [gene]'], ['AtSPO11-1-dependent [gene]']]}\n",
      "-------------------\n",
      "{'122955': [['**conservation of gene(s) content and order [phenotype]**', 'conserved neighbourhood [phenotype]'], ['conservation of gene(s) content and order between species [phenotype]'], ['conserved gene(s) content and order [phenomenon]'], ['extensive conservation of gene(s) order [phenotype]'], ['**conservation of gene(s) content [phenotype]**', 'conservation regarding gene(s) content [phenotype]'], ['conservation regarding gene(s) order [phenotype]'], ['ortholog abundance [phenotype]']]}\n",
      "-------------------\n",
      "{'64205': [['AtClpP4 levels [protein]'], ['**chloroplastic Clp protein(s) [protein]**', 'other chloroplast-localized Clp protein(s) [protein]'], ['chloroplast stromal Clp proteases [protein]', 'stromal Clp protease [protein complex]', '**stromal Clp protease [protein]**'], ['**ClpB3 protein(s) levels [protein]**', 'higher levels of ClpB3 [protein]'], ['Availability of ClpT protein(s) [protein]'], ['Plastid ClpT2 [protein]'], ['**Stromal ClpC2 [protein]**', 'ClpC2 protein(s) accumulation [protein]'], ['ClpC protein(s) in plastids [protein]'], ['ClpC protein(s) in cyanobacteria [protein]'], ['potential substrates for stromal Clp protease [protein]', '**substrates for Clp protease [protein]**'], ['plastid-localized protein(s) HvClpS1 and HvThf1 [protein]'], ['mitochondrial CLP protein(s) [protein complex]'], ['stromal Ser Clps [protease]'], ['chloroplast-localized Clp protease subunits [protein]'], ['cytoplasmic ClpB protein(s) [protein]']]}\n",
      "-------------------\n",
      "{'101253': [['irradiated seeds of A. thaliana [organism]'], ['gamma-irradiated M1 A. thaliana leaf [organism]'], ['Exposure of control (non-hardened) A. thaliana leaf [organism]'], ['irradiated A. thaliana [organism]'], ['200 Gy-treated A. thaliana [organism]'], ['partial-shielded A. thaliana seeds [organism]'], ['chronically irradiated A. thaliana populations [organism]'], ['hormone-autonomous, radiation-induced tumor tissue line of A. thaliana [tissue]']]}\n",
      "-------------------\n",
      "{'31036': [['**body plan specification [process]**', 'body plan establishment [process]'], ['body plan establishment [developmental stage]'], ['body segmentation [process]'], ['**Boundary specification [process]**', 'boundary domain formation [process]'], ['body axis formation [process]'], ['organ differentiation in boundary regions [process]']]}\n",
      "-------------------\n",
      "{'154023': [['Functional mTERF15 [gene]'], ['Tetrahymena thermophila Pot2 [gene]'], ['TERT_Cs [gene]'], ['TERT_C [gene]'], ['TERT_12D locus [gene]'], ['TERT from Capsella rubella [gene]'], ['TERT from Eutrema salsugineum [gene]'], ['TERT from Schrenkiella parvula [gene]']]}\n",
      "-------------------\n",
      "{'44318': [['tri5 mutant of F. graminearum [mutant]', '**tri5 mutant [mutant]**', 'Tri5 mutant strain [mutant]'], ['crabca2-1 [mutant]'], ['crabca2-2 [mutant]'], ['crabca2-3 [mutant]'], ['CT3 [mutant]'], ['Δtri5 mutant strain [mutant]']]}\n",
      "-------------------\n",
      "{'62503': [['FP-CESA fusion protein(s) [protein]'], ['CFP [protein]'], ['Venus/cyan fluorescent protein(s) (CFP) fluorescence ratios [phenotype]'], ['**eCFP and eYFP [component]**', 'enhanced Cyan Fluorescent protein(s) (eCFP) and enhanced Yellow Fluorescent protein(s) (eYFP) [component]'], ['**eCFP [protein]**', 'enhanced cyan fluorescent protein(s) (ECFP) variant [protein]', 'enhanced cyan fluorescent protein(s) [protein]', 'ECFP (enhanced cyan fluorescent protein) [protein]'], ['NRIP1-Cerulean [protein]'], ['CFP-SKL [protein]'], ['interaction of cCFP-tagged protein(s) [protein]'], ['cCFP [protein]'], ['MP-CFP [protein]'], ['TMV MP-CFP [protein]'], ['ER-localized CFP [protein]'], ['CFP-LTi6a construct [genetic construct]'], ['LRE-cYFP molecules [gene]'], ['Plastid-targeted cyan fluorescent protein(s) (CFP) [protein]'], ['Dual detection of stroma-targeted CFP [treatment]', 'a plastid stroma-targeted cyan fluorescent protein(s) (CFP) [treatment]', '**stroma-targeted CFP [treatment]**'], ['CFP-DPB2 protein(s) [protein]'], ['Mito-cpYFP [protein]'], ['CFP-TAN1 [treatment]'], ['ST-CFP [protein]'], ['edeCFP [protein]'], ['CFP derivatives [protein]'], ['ICR2-YPet [protein]'], ['DCIP [tool]'], ['35S-cyan fluorescent protein(s) [protein]'], ['CFP-PTS1 [protein]'], ['CYAN FLUORESCENT protein(s) [protein]'], ['CFP marker [metabolite]'], ['CFP-tagged membrane receptor BRI1 [protein]'], ['YC [protein]'], ['Cryptochrome 2 nuclear localization signal-fused Cyan Fluorescent protein(s) gene(s) [gene]'], ['CFP [metabolite]'], ['CFP [treatment]'], ['CFP signal [protein]'], ['cYFP-TCP20 [gene]'], ['C-terminal portion of CFP [protein]'], ['N-terminal portions of Cerulean [protein]'], ['p6-cYFP [protein fragment]'], ['plasma membrane marker Calcineurin B-like calcium sensor 1-orange fluorescent protein(s) (CBL1-OFP) [protein]'], ['single circularly permuted fluorescent protein(s) (cpFPs) [protein]'], ['CFP [fluorescent protein]'], ['tCFP [protein]'], ['empty CYFP [protein]'], ['circularly permuted fluorescent protein(s) (cpFP) [protein]'], ['mitochondrial matrix-targeted pH-sensitive indicator cpYFP [gene]'], ['CFP-PTS1 [treatment]'], ['monomeric Cerulean-PTS1 (mCer-PTS1) [treatment]'], ['CDC48A-cerulean fluorescent protein(s) [treatment]'], ['phyB-CFP [protein]'], ['C-terminal half of SCFP3A [gene]'], ['FCA-CFP bodies [protein complex]'], ['FBN1a-CFP [protein]'], ['constructs of CFP-tagged BZR1 [protein]'], ['peroxisomal targeted EYFP [protein]'], ['enhanced cyan fluorescent protein(s) (CFP) derivatives [protein]'], ['phyB-CFP [protein]'], ['C-terminal half of SCFP3A [gene]'], ['FCA-CFP bodies [protein complex]'], ['FBN1a-CFP [protein]'], ['constructs of CFP-tagged BZR1 [protein]'], ['peroxisomal targeted EYFP [protein]'], ['circularly permuted fluorescent protein(s) (cpFP) [protein]'], ['mitochondrial matrix-targeted pH-sensitive indicator cpYFP [gene]'], ['CFP-PTS1 [treatment]'], ['monomeric Cerulean-PTS1 (mCer-PTS1) [treatment]'], ['CDC48A-cerulean fluorescent protein(s) [treatment]']]}\n",
      "-------------------\n",
      "{'37126': [['actin filaments aggregation [phenotype]', 'pronounced aggregation of actin [phenotype]', 'actin aggregates [phenotype]', '**actin aggregation [phenotype]**', 'massive bundling of actin microfilaments [phenotype]']]}\n",
      "-------------------\n",
      "{'161720': [['radioactively labeled acetyl-CoA [metabolite]'], ['radioisotope-labeled tetraethylammonium (TEA) [metabolite]'], ['titrated thymidine [metabolite]'], ['radiolabelled malate [metabolite]'], ['radio-labelled ASC [metabolite]'], ['unlabelled NADPH [metabolite]'], ['isotopic enrichment of ubiquinone-9 [metabolite]'], ['iodoTMT tags [metabolite]'], ['**radiolabeled ATP [metabolite]**', 'radiolabelled ATP [metabolite]'], ['labeled carnitine [metabolite]']]}\n",
      "-------------------\n",
      "{'19303': [['mite performance [phenotype]'], ['levels of herbivory [phenotype]'], ['Chewing rates [phenotype]'], ['aphid ability to reach the phloem [phenotype]'], ['effective feeding behavior [phenotype]'], ['number of sustained phloem sap ingestion [phenotype]'], ['aphid performances [phenotype]'], ['aphids to ingest sap [phenotype]'], ['feeding rates [phenotype]'], ['success rate of phloem ingestion [phenotype]'], [\"insect's ability to reach phloem [phenotype]\"], ['SE sap ingestion rate [phenotype]']]}\n",
      "-------------------\n",
      "{'73406': [['extracellular CIF (Casparian strip integrity factor) [protein]'], ['CIF1/2-GSO1/SGN3-PBL15/SGN1 module [protein]'], ['cif1 [protein]'], ['CIF2 [protein]'], ['**CIF1/2 [protein]**', 'CASPARIAN STRIP INTEGRITY FACTORS1/2 (CIF1/2) [protein]'], ['**cif1 [metabolite]**', 'Casparian Strip Integrity Factor 1 (CIF1) [metabolite]'], ['CIF1/2 [metabolite]']]}\n",
      "-------------------\n",
      "{'141728': [['Transformation with plasmids encoding AtElp1 [gene]'], ['pCambia1300/MIR390 plasmid [gene]'], ['plasmid pBEO210 [gene]'], ['pNT-AtNHX1 [genetic element]'], ['RT-based IPGT plasmid [tool]'], ['pBI-AtZIP60 [gene]']]}\n",
      "-------------------\n",
      "{'62513': [['Cation identified in Fluc [protein]'], ['mClover signals [protein]']]}\n",
      "-------------------\n",
      "{'117070': [['central population assembly [phenotype]'], ['contigs in the middle site [phenotype]'], ['similar amount of assembled centromere sequences [phenotype]']]}\n",
      "-------------------\n",
      "{'167375': [['**gallic acid [metabolite]**', '3,4,5-trihydroxybenzoic acid (gallic acid) [metabolite]'], ['Valic acid [metabolite]'], ['vanillic acid [metabolite]'], ['High concentration of vanillic acid [metabolite]'], ['Kojic acid [metabolite]'], ['Malvalic acid [chemical]'], ['Vanilic acid [metabolite]'], ['130 teak NAC protein(s) [protein]'], ['tartaric acid [metabolite]'], ['Tartaric acid derivatives [metabolite]'], ['Punicic acid [metabolite]'], ['Acid nature of FA [metabolite]'], ['Conjugated double bonds in punicic acid [metabolite]'], ['Decrease in glycosylated vanillic acid [metabolite]'], ['Citric acid [chemical]'], ['production of L-tartaric acid [metabolite]'], ['vanillic acid content [metabolite]'], ['esters of gallic acid [metabolite]'], ['natural tetramic acids [metabolite]'], ['geronic acid [metabolite]'], ['thioglycolic acid [metabolite]'], ['tannic acid [metabolite]'], ['tannic acid [compound]']]}\n",
      "-------------------\n",
      "{'88319': [['**ecotilling [treatment]**', 'Eco-TILLING [treatment]'], ['genetic improvement of crop salt tolerance [research field]'], ['ecotilling [gene]'], ['environmental and genotype-environment interaction [treatment]'], ['applied in field conditions [treatment]']]}\n",
      "-------------------\n",
      "{'173257': [['**blocking pathway [process]**', 'interference with metabolic pathways [process]'], ['**feedback inhibition by metabolites [process]**', 'end-product repression [process]', 'NADPH-mediated feedback inhibition [process]', 'feedback loops of key metabolic enzymes [process]'], ['down-regulation of anabolic metabolism [process]'], ['repression of upstream enzyme activities [process]'], ['Down-regulating expression of pathways [process]'], ['upstream inhibition [phenotype]'], ['identification of key enzymes [process]'], ['regulatory constraints [process]'], ['fine control [process]'], ['regulatory cross-talk [process]'], ['inhibition of enzyme [enzyme]'], ['**key rate-limiting factor [process]**', 'regulatory bottleneck [process]'], ['level of downstream processes [process]'], ['feedback inhibition of flux-determining enzymes [enzyme]'], ['enzyme activation/inactivation [process]', 'balancing of enzyme activities [process]', '**regulation of enzymatic activities [process]**', 'enzymatic regulation [process]'], ['inhibition of the catabolic pathway [process]'], ['negative regulation of cellular metabolic process [process]']]}\n",
      "-------------------\n",
      "{'150570': [['Node_3814 [gene]'], ['SlX7 [gene]'], ['SlX3 [gene]'], ['PpIRX10 [gene]'], ['SIX5 [gene]'], ['RSW1 locus [gene]'], ['RSW10 [gene]'], ['AoIRX14A [gene]'], ['**AoIRX14B [gene]**', 'AoIRX14_B [gene]'], ['G65 [genotype]'], ['PtSWEET16b [gene]'], ['**Sapur.15WG074900 [gene]**', 'Sapur.15WG074900 hypothetical protein(s) [gene]'], ['**Sapur.15WG075700 [gene]**', 'Sapur.15WG075700 hypothetical protein(s) [gene]'], ['BntWG10024039 [gene]'], ['BntWG10008444 [gene]'], ['Mp56 [gene]'], ['DUF231 members from rice [gene]'], ['RSW1(CESA1) product [gene]'], ['RSW3/PSL5 [enzyme]'], ['IQ67 DOMAIN 9 (IQD9) [gene]'], ['PtDXR [gene]'], ['GT43 IRX9 [gene]'], ['GT43 IRX14 [gene]'], ['RSW2 [gene]'], ['F644 [gene]'], ['Hevea brasiliensis DXR [gene]'], ['Populus trichocarpa DXR [gene]'], ['XXGG and XSGG subunits [phenotype]'], ['candidate FMOGS-OX [phenotype]'], ['Fol SIX8 [gene]']]}\n",
      "-------------------\n",
      "{'29577': [['ribosome export [process]', '**pre-ribosome export [process]**', 'export of pre-ribosomes from nucleus to cytoplasm [biochemical process]', 'preribosomal particle export [process]', 'transport of ribosome subunits to the cytoplasm [process]', 'export of pre-ribosomal particles to the cytoplasm [process]', 'preribosome export from the nucleus to the cytoplasm [process]'], ['impairment of nuclear export of pre-ribosomes [process]']]}\n",
      "-------------------\n",
      "{'130555': [['expression levels of A. thaliana gene(s) [gene]'], ['expression levels of A. thaliana (A. thaliana) transcripts [gene]']]}\n",
      "-------------------\n",
      "{'23129': [['**brassinosteroid binding [phenotype]**', 'BL binding [phenotype]', 'BRI1-BRs binding [phenotype]'], ['BR-induced cell physiological processes [phenotype]'], ['BZR1-mediated BR signals [phenotype]'], ['BR and light [phenotype]'], ['**Crosstalk between BR and sugar [phenotype]**', 'sugar inhibition of BR signaling [phenotype]'], ['BR-GA crosstalk mechanism [phenotype]', '**BR and GA signaling pathways [phenotype]**', 'genetic interaction between BR and GA pathways [phenotype]'], ['BRI1 and brassinosteroid-dependent pollen development [phenotype]', 'BR-induced BRI1 signaling [phenotype]', '**BRI1 function [phenotype]**', 'BRI1 pathway [phenotype]', 'BRI1-dependent signaling [phenotype]', 'BRI1-mediated brassinosteroid (BR) signalling [phenotype]', 'BRI1-mediated brassinosteroid perception [phenotype]'], ['signaling crosstalk of BR [phenotype]'], ['BIN2 signaling [phenotype]'], ['BR-mediated induction [phenotype]'], ['BR regulates phytochrome-mediated responses [phenotype]'], ['BR regulates cryptochrome-mediated responses [phenotype]'], ['pleiotropic effects of cell wall/BR signals [phenotype]'], ['pleiotropic effects of the BR pathway [phenotype]'], ['BR biosynthesis or signaling [phenotype]'], ['BR-induced leaf senescence [phenotype]'], ['inhibitory interaction between BRs and ethylene [phenotype]'], ['BR-induced changes in microtubule organization and cell patterning [phenotype]']]}\n",
      "-------------------\n",
      "{'103177': [['virulent fungi [organism]'], ['many fungi [organism]'], ['Some other fungi [organism]'], ['broad host range fungal pathogens [organism]'], ['fungal microbes [organism]'], ['broad-host-range fungi [organism]'], ['vascular fungi [organism]'], ['Fungus [phenotype]'], ['various fungi [organism]'], ['fungus in plant(s) root(s) [organism]'], ['diversity of antagonistic fungi [organism]'], ['diverse group of fungi [organism]']]}\n",
      "-------------------\n",
      "{'167332': [['**methyl SA (MeSA) [metabolite]**', 'SA conjugate methyl salicylate (MeSA) [metabolite]', 'emission of methylated SA (methyl salicylate, MeSA) [metabolite]', 'synthetic methyl salicylate [metabolite]', 'SA to MeSA [metabolite]', 'SA methyl ester (MeSA) [metabolite]']]}\n",
      "-------------------\n",
      "{'89734': [['repressive interaction [process]', 'synergistic inhibitory role [process]', '**inhibitory mechanism [process]**', 'inhibitory action [process]'], ['negative feedback system [process]'], ['negative growth control [process]'], ['Slowing down regulation [process]', '**signal downregulation [process]**'], ['Feedback inhibition [treatment]'], ['negative role in controlling turnover of NLRs [process]'], ['cholinergic blockade of inflammation [process]'], ['negative regulator [description]'], ['negative signal [process]'], ['antagonistic activities [process]'], ['partial antagonistic manner [process]'], ['dampening of immune signaling [process]']]}\n",
      "-------------------\n",
      "{'46669': [['npr1-5 [mutant]'], ['sid2 and npr1-1 mutants [mutant]'], ['**npr1-2 mutant [mutant]**', 'npr1-2 mutants [mutant]'], ['npr1-1 seeds [mutant]'], ['npr1-1 seedlings [mutant]'], ['NahG, pad4, and npr1 mutants [mutant]'], ['nahGxa0siz1 [mutant]'], ['coi1-16 npr1-1 [mutant]'], ['sid2 and npr1 mutants [mutant]'], ['gns1/Atelp2-5 mutants [mutant]'], ['osnpr1 [mutant]'], ['sid2-1 and npr1-1 mutants [mutant]'], ['ROS accumulation in npr1 mutant plant(s) [mutant]']]}\n",
      "-------------------\n",
      "{'94544': [['Different floral organ identities [phenotype]', 'floral organ determination [phenotype]', '**identity of floral organs [phenotype]**', 'specifying floral organ identity [phenotype]', 'central organ identity [phenotype]', 'flower organ identities [phenotype]', 'floral organs identities [phenotype]', 'floral organs identity [phenotype]', 'type of floral organs [phenotype]', 'organ identity in flowers [phenotype]'], ['five floral organs [phenotype]'], ['numbers of all floral organ types [phenotype]'], ['floral organ number or morphology [phenotype]'], ['floral organ number and identity [phenotype]'], ['**floral organ shape [phenotype]**', 'proper floral organ morphology [phenotype]', 'flower organ morphology [phenotype]', 'floral organ morphologies [phenotype]'], ['patterning of inner floral organs [phenotype]', '**floral body plan organization [phenotype]**', 'flower organ organization [phenotype]'], ['floral organ adhesion [phenotype]'], ['floral organ epidermal surface decoration [phenotype]'], ['determinate flower-like organs [phenotype]'], ['inner floral organs perfectly [phenotype]'], ['floral organs formation [phenotype]'], ['unique identities of different floral organs [organ]', 'organ identities in the flower [organ]', 'identities of floral organs [organ]', '**floral organ identity [organ]**', 'identity of floral organs [organ]', 'identity of individual floral organs [organ]', 'full organ identity [organ]', 'identities of different types of floral organs [organ]'], ['morphology of floral organs [organ]', 'flower shape and structure [organ]', '**floral organ shape [organ]**', 'appropriate floral organ shapes [organ]', 'distinct morphologies of floral organs [organ]', 'Flower morphology [organ]', 'flower morphologies [organ]'], ['**specification of floral organs [organ]**', 'the specification of floral organs [organ]'], ['specification of kiwifruit floral organs [organ]'], ['identification of floral organs [organ]'], ['morphology of a flowering plant(s) [organ]'], ['relatively standard floral organs [organ]']]}\n",
      "-------------------\n",
      "{'74276': [['SMXL8 [protein]'], ['OS04T0195000-01 [protein]'], ['OS04T0223000-01 [protein]'], ['**TRANSTHYRETIN-LIKE (TTL) [protein]**', 'Transthyretin-like (TTL) protein(s) [protein]'], ['F-box protein(s) MORE AXILLARY BRANCHES 2 (MAX2) [protein]'], ['EXORDIUM-like 4 protein(s) [protein]']]}\n",
      "-------------------\n",
      "{'134698': [['HEL gene(s) [gene]'], ['**STELLO1 [gene]**', 'STELLO (STL) 1 [gene]'], ['**STELLO2 [gene]**', 'STELLO (STL) 2 [gene]', 'STL2 [gene]'], ['she2 [gene]']]}\n",
      "-------------------\n",
      "{'88707': [['genetic resources of DSEs [gene]'], ['Transgenic expression of SDE15 in Duncan grapefruit (<i>Citrus × paradisi</i>) [treatment]'], ['salt-tolerant Cl- excluding rice line Pokkali [organism]'], ['HRT-mediated HR to TCV [phenotype]'], ['Pst-infected Su11 cDNA sample [treatment]'], ['tolerance to salinity and hydrogen peroxide treatments in transgenic rice [phenotype]'], ['treatment with fungal elicitor [treatment]']]}\n",
      "-------------------\n",
      "{'24195': [['interaction between multiple independently identified regulators [phenotype]'], ['factors and regulatory networks [phenotype]'], ['pleiotropic regulators of cell cycle, light signaling, circadian rhythms, flowering time, and hormone responses [phenotype]', '**pleiotropic regulator [phenotype]**'], ['multiple regulatory circuits [phenotype]']]}\n",
      "-------------------\n",
      "{'85032': [['**storage conditions [treatment]**', 'storage condition [treatment]'], ['high altitudes [treatment]'], ['hyperglycemic conditions [treatment]'], ['energy-deprivation conditions [condition]'], ['spring seasons [treatment]'], ['conditions during spring 2005 [treatment]'], ['fall seasons [treatment]'], ['operative conditions [treatment]'], ['hydrophobic environments [treatment]'], ['constant environments [environment]'], ['redox-conditions [treatment]'], ['low-energy conditions [treatment]'], ['low-glucose conditions [treatment]'], ['stromule-inducing conditions [treatment]'], ['excess conditions [treatment]'], ['ambient RH [treatment]'], ['low mortality environments [environment]'], ['eco-dormancy period [treatment]'], ['different seasonal environments [treatment]'], ['different environmentally relevant pH levels [treatment]'], ['winter and semi-winter ecological conditions [treatment]'], ['epitype-inducing conditions [treatment]']]}\n",
      "-------------------\n",
      "{'161220': [['Deficiency in Met biosynthesis [phenotype]'], ['absence of cellular IBA uptake [phenotype]'], ['Lack of PNC1 [protein]'], ['Lack of PNC2 [protein]'], ['complete loss of AMY3 [protein]'], ['Loss of AMY3 [gene]'], ['PHGDH deficiency [phenotype]'], ['missing re-import of vacuolar myo-inositol [metabolite]'], ['missing inositol transport via INT1 [metabolite]'], ['MSUD [phenotype]'], ['shortage of nucleoside monophosphates [phenotype]'], ['non-enzymatic degradation of triosephosphates [treatment]'], ['myo-inositol deficiency [phenotype]'], ['absence of Kdo biosynthesis [phenotype]'], ['Amino acid substitution in IPMDH1 [enzyme]'], ['Amino acid substitution in IPMDH2 [enzyme]'], ['HIBCH deficiency [phenotype]'], ['cAMP shortage [phenotype]'], ['lack of dietary AA [metabolite]'], ['absence of stimulatory action on downstream metabolic targets [phenotype]'], ['thiamine starved cells [cell type]'], ['**TDP deficiency [metabolite]**', 'limited TDP [metabolite]'], ['impaired Lys biosynthesis [metabolite]'], ['APRT-deficient tissue [tissue]'], ['Inactive oxidized form of AtAMY3 [gene]'], ['lack of PS [metabolite]'], ['Loss of MPC [gene]'], ['lack of P5CDH activity [phenotype]'], ['MCCase deficiency [phenotype]'], ['loss of FAH [enzyme]'], ['build up of phosphatidylmonomethylethanolamine [metabolite]'], ['Defects in TK activity [phenotype]'], ['Lack of adenine recycling [phenotype]'], ['absence of flavin prosthetic group [metabolite]'], ['no direct incorporation of ammonia into glutamate [phenotype]'], ['nucleotide limitation and imbalance [phenotype]'], ['loss of mtATPase [protein complex]'], ['lack of GS1-related glutamine biosynthesis [process]'], ['lack of FMN [metabolite]'], ['Gitelman syndrome [disease]'], ['complete absence of GLN2 [gene]'], ['incomplete formation of side-chain A [metabolite]'], ['mevalonate aciduria (MA) [phenotype]'], ['absent aspartate/glutamate general-acid [metabolite]'], ['**absence of T6P production [metabolite]**', 'failure to produce T6P [metabolite]'], ['presence of acetate [treatment]'], ['deficient splicing of the ndhA transcript [process]'], ['auxotrophies for nicotinic acid [metabolite]'], ['loss of Cu(2+) binding [metabolite]'], ['low level of mHS polypeptide [protein]'], ['metabolic decrease [phenotype]']]}\n",
      "-------------------\n",
      "{'670': [['three key growth periods [phenotype]'], ['stage 1 to stage 4 [phenotype]'], ['several developmental progressions [phenotype]'], ['several developmental stages throughout the plant(s) life cycle [phenotype]']]}\n",
      "-------------------\n",
      "{'65332': [['VIN3/VIL1-containing PRC2 [protein complex]'], ['FIGL1-FLIP complex [protein complex]'], ['Rca1/Emi1 F-box protein(s) [protein]'], ['Drosophila PRC1 E3 monoubiquitin ligase module [protein complex]'], ['**SCFFBL17 [protein complex]**', 'SCFFBL17 E3 ligase [protein complex]'], ['ARF3-IND complex [protein complex]'], ['these factors [protein complex]'], ['FLC expression through complex protein(s) networks [process]'], ['components of the ASAP complex [protein complex]']]}\n",
      "-------------------\n",
      "{'106753': [['CRISPR/Cas [gene]', '**CRISPR [gene]**', 'CRISPR/Cas tool [gene]', 'CRISPR technologies [gene]', 'CRISPR-based technologies [gene]', 'CRISPR/CRISPR-associated (Cas) system [gene]'], ['CRISPR technology [treatment]', 'CRISPR-based tools [treatment]', 'CRISPR/Cas technology [treatment]', 'CRISPR/Cas [treatment]', 'CRISPR/Cas-mediated GE [treatment]', 'CRISPR/CAS technologies [treatment]', 'CRISPR system [treatment]', '**CRISPR [treatment]**', 'CRISPR/Cas tools [treatment]', 'CRISPR/Cas-mediated editing [treatment]', 'CRISPR/CRISPR-associated protein(s) (Cas) system [treatment]', 'CRISPR/Cas editing [treatment]', 'CRISPR-Cas technology [treatment]', 'CRISPR-Cas [treatment]', 'CRISPR-based genome engineering [treatment]', 'CRISPR/Cas genome editing [treatment]', 'CRISPR gene(s) editing technology [treatment]'], ['**CRISPR-Cas system [technology]**', 'CRISPR-Cas-mediated genome editing [technology]', 'CRISPR/CAS system [technology]', 'CRISPR-Cas systems [technology]', 'CRISPR-based genome editing [technology]', 'CRISPR-Cas technology [technology]', 'CRISPR/Cas-mediated genome editing [technology]'], ['CRISPR-based technology [technique]', 'CRISPR-based gene(s) editing [technique]', 'CRISPR/Cas-based genome manipulations [technique]', 'CRISPR/Cas [technique]', '**CRISPR [technique]**', 'CRISPR based Genome Editing [technique]', 'CRISPR construct [technique]'], ['CRISPR/Cas [tool]', 'CRISPR-Cas systems [tool]', 'CRISPR-based genome editing tools [tool]', '**CRISPR [tool]**', 'CRISPR-Cas [tool]', 'CRISPR/CAS system [tool]', 'CRISPR system [tool]', 'CRISPR-based systems [tool]', 'CRISPR editing [tool]'], ['CRISPR/CRISPR-associated protein(s) (Cas) system [gene editing tool]'], ['CRISPR technologies [device]'], ['**CRISPR-editing [method]**', 'CRISPR-Cas system [method]'], ['CRISPR-Cas system [scientific method]'], ['**CRISPR [gene editing technology]**', 'CRISPR/Cas [gene editing technology]'], ['CRISPR/Cas [system]'], ['**CRISPR/CAS system [process]**', 'CRISPR-Cas technology [process]'], ['CRISPR/CAS system [protein]']]}\n",
      "-------------------\n",
      "{'92876': [['**biopesticides [treatment]**', 'biopesticide [treatment]', 'biological pesticides [treatment]'], ['host specificity [treatment]'], ['Natural enemies [treatment]'], ['a biological control agent [treatment]', '**biological control agent (BCA) [treatment]**', 'potent biological control agent [treatment]', 'potential biological control agent [treatment]'], ['biofumigation [treatment]'], ['potential means for biological control [treatment]'], ['effective biological control [treatment]'], ['biological nematode control agent [description]'], ['direct antagonism [treatment]'], ['agriculture as biological control agents [treatment]'], ['bioinsecticide [treatment]'], ['other biological agents [treatment]'], ['effectiveness of natural enemies [organism]'], ['last few decades [time period]'], ['biologic pest management [treatment]']]}\n",
      "-------------------\n",
      "{'74789': [['**non-race specific disease resistance (NDR1) [protein]**', 'Non-Race-specific Disease Resistance (NDR1) [protein]', 'NON-SPECIFIC DISEASE RESISTANCE 1 (NDR1) [protein]', 'nonrace-specific disease resistance 1 (NDR1) [protein]', 'NON-RACE SPECIFIC DISEASE RESISTANCE 1 (NDR1) [protein]'], ['Non-race specific disease resistance/harpin-induced (NDR1/HIN1)-like protein(s) [protein]'], ['functional protein(s) of non-race specific disease resistance1 (NDR1-1) and NPR1-2 [protein]']]}\n",
      "-------------------\n",
      "{'120616': [['Non-synonymous SNP [genetic variant]'], ['**Non-synonymous SNP [gene]**', 'Nonsynonymous single-nucleotide polymorphism [gene]', 'nonsynonymous SNP [gene]', 'Single nonsynonymous nucleotide substitution at GVS1 [gene]', 'one non-synonymous SNP [gene]'], ['One nonsynonymous SNP (G-T) [mutation]', '**nonsynonymous SNP [mutation]**', 'non-synonymous polymorphism [mutation]', 'non-synonymous single nucleotide polymorphism [mutation]'], ['Non-synonymous SNP [genetic marker]'], ['non-synonymous SNP mutation [genetic variation]', 'Low levels of nonsynonymous SNP [genetic variation]', '**Non-synonymous SNP [genetic variation]**', 'one unique non-synonymous SNP [genetic variation]', 'one non-synonymous single-nucleotide variant (SNV) in the exon [genetic variation]'], ['Non-synonymous SNP [genetic element]'], ['Non-synonymous nucleotide polymorphism [genomic variant]'], ['Non-synonymous SNP [genomic variation]']]}\n",
      "-------------------\n",
      "{'150152': [['TaSR7A [gene]'], ['TaSR7B [gene]'], ['TaSR4B [gene]'], ['TaSR4A [gene]'], ['TaSR4D [gene]'], ['TaSR2B [gene]'], ['TASR [gene]'], ['TaSR6B [gene]'], ['TaSR7D [gene]'], ['TaSR14D [gene]'], ['TaSRp30 [gene]'], ['wheat-germ tRNAArg [gene]'], ['TaSR2A [gene]'], ['TaSR2D [gene]']]}\n",
      "-------------------\n",
      "{'138671': [['RPP4 and At4g16950 expression levels [gene]'], ['Peak expression of At4g16260 [gene]', '**AT4G16260 expression [gene]**'], ['chitin-induced ATL12 expression [process]'], ['changes in expression level of AtS40-3 [gene]'], ['AtSIG1 expression [gene]'], ['Thermopriming-expressed ATG8s [gene]'], ['nitrogen limitation-inducible ATG8 [gene]'], ['Expression of AtbZIP60S [gene]'], ['AtABCG31 expression [gene]'], ['expression of RrATG18e [gene]'], ['expression of actA [gene]'], ['Expression of AtWKRY12 [gene]'], ['chitin-induced AtLEC expression [gene]'], ['Activation of AtFer1 expression [process]', '**AtFer1 gene(s) expression [process]**'], ['AtGSTF6 expression [gene]'], ['AtMIN7 level [gene]'], ['Expression of AtERF014 [gene]'], ['AtCHS expression [phenotype]'], ['expression of AtSHMT4 [phenotype]'], ['ATL9 induction [gene]'], ['AtPR1 gene(s) expression [gene]'], ['thermopriming-expressed ATG8 [phenotype]'], ['expression of At1g51890 [gene]'], ['AtWRKY22 expression [phenotype]'], ['AtPHI1 expression [phenotype]'], ['intermediate response in AtWRKY29 expression [phenotype]'], ['intermediate response in AtWRKY30 expression [phenotype]'], ['intermediate response in AtWRKY33 expression [phenotype]'], ['intermediate response in AtWRKY53 expression [phenotype]'], ['pathogen-induced ATG18a expression [process]'], ['Pathogen-induced expression of AtWRKY33 [gene]'], ['expression of AtPR1 and AtPR2 [gene]'], ['expression of up to six AtCDA gene(s) [gene]'], ['wound-responsive expression of AtFAD7 [gene]'], ['AtMYB44 production [phenotype]'], ['AtACT gene(s) expression [gene]'], ['pathogen-responsive expression of AtPDR12 [gene]'], ['mRNA expression of the AtDabb1 gene(s) [gene]']]}\n",
      "-------------------\n",
      "{'101864': [['A. lyrata subsp lyrata [organism]', '**A. lyrata subsp. lyrata [organism]**', 'A. l. lyrata [organism]', 'A. lyrata (subsp. lyrata) [organism]', 'ssp. lyrata [organism]'], ['A. lyrata PT [organism]', '**A. lyrata petraea [organism]**', 'A. lyrata (subsp. petraea) [organism]'], ['**A. lyrata pollen [organism]**', 'pollen of A. lyrata [organism]'], ['aly [organism]'], ['A. lyrata seeds [organism]'], ['plant(s) of A. lyrata [organism]'], ['A. lyrata S14 pollen [organism]'], ['A. lyrata subsp. lyrate [organism]'], ['nascent A. lyrata [organism]'], ['LYR [organism]'], ['origin of A. l. lyrata [organism]']]}\n",
      "-------------------\n",
      "{'22385': [['SNE accumulation [process]'], ['SSPP accumulation [process]']]}\n",
      "-------------------\n",
      "{'129106': [['A. thaliana R gene(s) T-DNA Insertion Collection (ARTIC) [gene]'], ['**A. thaliana R gene(s) T-DNA Insertion Collection (ARTIC) [resource]**', 'ARTIC [resource]'], ['raf17 T-DNA insertion [gene]'], ['T-DNA insertion (salk_018112) [treatment]'], ['Syngenta A. thaliana Insertion Library (SAIL) T-DNA insertions [treatment]'], ['SALK_059379 [gene]'], ['SAIL_232 [gene]'], ['GABI-Kat collection [resource]'], ['**T-DNA integration [gene]**', 'insertion events [gene]'], ['T-DNA insertion in the AtWAKL4 promoter [gene]'], ['Salk T-DNA insertion line Salk_001014 [gene]'], ['T-DNA insertion in the AtKinesin-13A gene(s) [gene]'], ['SALK_110396 [gene]'], ['phyA-t [gene]'], ['annotated T-DNA [gene]'], ['T-DNA insertion in IAR4 coding sequence [gene]'], ['T-DNA insertion in COG3 [gene]'], ['T-DNA insertion in COG8 [gene]'], ['T-DNA insertion in second intron of RRES1 [gene]'], ['SALK collection [genetic element]'], ['T-DNA insert in DUF239 [gene]'], ['T-DNA insert in the 2nd exon of AtNAP [gene]'], [\"T-DNA insertion in 5' untranslated region of A. thaliana Adenylate Kinase 6 (AAK6; AT5G60340) [gene]\"], ['Syngenta A. thaliana Insertion Library (SAIL) T-DNA lines [genetic tool]'], ['T-DNA insertions in BADC1, BADC2, and BADC3 [gene]'], ['T-DNA insertion in the AtHXK1 gene(s) [gene]']]}\n",
      "-------------------\n",
      "{'115627': [['Largest cluster [gene cluster]'], ['XTH gene(s) in cluster 1 [gene]'], ['XTH gene(s) in cluster 2 [gene]'], ['XTH gene(s) in cluster 4 [gene]'], ['sub-cluster 4 [process]'], ['cluster 25 [gene cluster]'], ['Cluster 37 [gene cluster]'], ['Cluster 28 [gene cluster]'], ['gene(s) of orthogroups in clusters 4 and 5 [gene]'], ['clade I [gene cluster]'], ['Clade II [gene cluster]'], ['cluster 74 [gene cluster]'], ['cluster 76 [gene cluster]'], ['cluster 46 [gene cluster]'], ['Cluster VI [gene cluster]'], ['clade containing syntenic clusters 2, 4, 5, and 14 [gene cluster]'], ['CDC16 clade [gene]'], ['CDC20 clade [gene]'], ['Clusters 6 [gene cluster]'], ['GO term analysis of TF Cluster 4 [treatment]'], ['GO terms associated with TF Cluster 1 and 5 [treatment]'], ['five gene(s) in paralogous cluster [gene cluster]'], ['Clique 4 [gene]'], ['PMEI cluster [clade]'], ['Module M6 [gene cluster]'], ['module M7 [gene cluster]'], ['six sub-clusters [cluster]'], ['some of the paralogs in Clade A5 [gene]'], ['gamma4 clade members [gene cluster]'], ['Clade IC [gene cluster]'], ['Clade III [gene cluster]'], ['Clade IV [gene cluster]'], ['Clade V [gene cluster]'], ['Clade VI [gene cluster]'], ['clade VII [gene cluster]'], ['clusters I-IV [gene cluster]'], ['C5 [phenotype]'], ['six gene(s) in paralogous cluster [gene cluster]'], ['cluster with VvHT24 [gene]']]}\n",
      "-------------------\n",
      "{'152158': [['Br181 to Br185 [gene]'], ['Br286 to Br287 [gene]'], ['Br233 to Br235 [gene]'], ['Br108 to Br109 [gene]'], ['Br074 to Br076 [gene]'], ['Br158 to Br160 [gene]'], ['Br207 to Br208 [gene]'], ['Br254 to Br256 [gene]'], ['Br274 to Br275 [gene]'], ['Br216 to Br218 [gene]'], ['Br163-ERF-B2 [gene]'], ['Br240-ERF-B2 [gene]'], ['Br112-ERF-B3 [gene]'], ['Br001-DREB-A3 [gene]'], ['Br008-ERF-B1 [gene]'], ['Br201-ERF-B6 [gene]'], ['Br074 [gene]'], ['Br075 [gene]'], ['Br076 [gene]'], ['Br171 [gene]'], ['Br147 [gene]'], ['Br222 [gene]'], ['Br002 [protein]'], ['Br030 [protein]'], ['Br200 [protein]'], ['Br214-ERF-B3 [gene]'], ['Br012-ERF-B1 [gene]'], ['Br047-AP2 [gene]']]}\n",
      "-------------------\n",
      "{'8071': [['Various cellular processes [phenotype]'], ['**multiple biochemical processes [phenotype]**', 'numerous biochemical processes [phenotype]'], ['multi-step biochemical reactions [phenotype]'], ['multiple molecular and physiological processes [phenotype]'], ['distinct biological consequences [phenotype]'], ['**molecular and biological processes [phenotype]**', 'various molecular and biological processes [phenotype]'], ['**multiple molecular processes [phenotype]**', 'diverse molecular processes [phenotype]'], ['various molecular and physiological events [phenotype]'], ['numerous cellular and molecular mechanisms [phenotype]'], ['complex biochemical activities [phenotype]']]}\n",
      "-------------------\n",
      "{'146433': [['TCX3 [gene]'], ['TCX2 [gene]'], ['Mutations in CTL1 [gene]']]}\n",
      "-------------------\n",
      "{'173695': [['stereospecific cyclization [function]'], ['Cyclization [treatment]'], ['cycloaddition [chemical reaction]'], ['cleavage of the cyclic ring of L-DOPA [process]'], ['Cyclization [process]'], ['Enzymatically driven cyclization [process]', '**Enzymatic cyclization [process]**'], ['sequential bicyclization and rearrangement [chemical process]'], ['Cyclization reactions [metabolite]'], ['intramolecular cyclization process [process]'], ['oxidative cyclization [process]'], ['Cyclization [phenotype]'], ['cyclized [process]'], ['Knorr cyclization reaction [reaction]'], ['intramolecular cyclization [reaction]'], ['cyclic condensation [reaction]'], ['cyclization of precursor [process]'], ['peptide cyclization [activity]'], ['macrocyclization reactions [process]', '**macrocyclization [process]**'], ['bicyclization [reaction]']]}\n",
      "-------------------\n",
      "{'120792': [['single copy of FN3K gene(s) [gene]'], ['single copy of either SERK1 or SERK2 [gene]']]}\n",
      "-------------------\n",
      "{'153959': [['Q135 [gene]'], ['D136 [gene]'], ['T693 [gene]'], ['T697 [gene]'], ['T701 [gene]'], ['F139 [gene]'], ['D85 [gene]'], ['Y186 [gene]'], ['E183 [gene]'], ['N64 [gene]'], ['Y67 [gene]']]}\n",
      "-------------------\n",
      "{'11488': [['**UV-B-induced DNA damage [phenotype]**', 'UVB-induced DNA damage [phenotype]', 'UV-B induced DNA damage [phenotype]'], ['**UV-induced DNA damage [phenotype]**', 'UV radiation-induced DNA damage [phenotype]', 'UV-damaged DNA [phenotype]', 'UV-induced DNA damages [phenotype]', 'UV damaged DNA [phenotype]'], ['**UV damage [phenotype]**', 'UV-induced damage [phenotype]', 'UV-light damage [phenotype]'], ['UVC [phenotype]'], ['UV-B-induced mutations [phenotype]'], ['UV-induced damage on single-stranded DNA [phenotype]'], ['UVC-induced DNA damage [phenotype]'], ['direct DNA damage [phenotype]'], ['increased frequency of DNA single-strand breaks [phenotype]'], ['UV-B-induced DNA damage recognition [phenotype]'], ['UV-induced damage products [phenotype]'], ['UV-B-induced photoproducts from DNA [phenotype]']]}\n",
      "-------------------\n",
      "{'116813': [['34.5%, 34.5%, and 27% C-to-T conversion at C4, C5, and C12 [metabolite]'], ['34.5%, 34%, and 14.25% C-to-T conversion at C4, C5, and C12 [metabolite]'], ['18.7% C-to-T conversion rate [phenotype]'], ['13.3% C-to-T conversion rate [phenotype]'], ['64.1% C-to-T conversion rate [phenotype]'], ['**C-to-T conversion rates > 50% [phenotype]**', 'over 50% conversion ratio [phenotype]', '> 50% conversion rates [phenotype]', '> 50% C-to-T conversion rates [phenotype]'], ['> 50% C-to-T conversion efficiency [phenotype]']]}\n",
      "-------------------\n",
      "{'174495': [['choline [metabolite]'], ['lipid biopolymers [metabolite]'], ['Microbial lipids [metabolite]'], ['lipid derivatives [metabolite]'], ['prostaglandins [chemical compound]'], ['lipid barriers [metabolite]'], ['phloem lipids [metabolite]'], ['P. brassicae egg lipids [metabolite]'], ['marker lipids [metabolite]'], ['bioactive lipid derivatives [metabolite]'], ['Polymer-bound lipids [metabolite]'], ['Plant-derived lipids [metabolite]'], ['ester-bound lipids [metabolite]'], ['certain lipids [metabolite]'], ['lipid-rich biomass feedstock for biofuels [metabolite]'], ['lipophilic signaling lipids [metabolite]', '**lipid signal [metabolite]**'], ['feedstock for biofuels [metabolite]'], ['a new class of lipid molecules [metabolite]'], ['effects of nitrolipids [metabolite]'], ['content of radish and onion polar and neutral lipids [metabolite]'], ['potent lipidic regulator [metabolite]'], ['lipid-derived hormone [metabolite]'], ['target lipids [metabolite]'], ['biologically active lipids [metabolite]'], ['authentic PBM lipid of plant(s) origin [metabolite]']]}\n",
      "-------------------\n",
      "{'15693': [['**Pi deficiency-induced inhibition of primary root(s) growth [phenotype]**', 'starvation-mediated repression of primary root(s) growth [phenotype]', 'inhibition of primary root(s) growth triggered by Pi starvation [phenotype]']]}\n",
      "-------------------\n",
      "{'173721': [['Production of ricinoleic acid [metabolite]', '**synthesis of ricinoleic acid [metabolite]**', 'ricinoleate biosynthesis [metabolite]', 'ricinoleate synthesis [metabolite]'], ['addition of linoleic acid [metabolite]'], ['production of EPA [metabolite]'], ['production of DHA [metabolite]'], ['**linoleic acid synthesis [metabolite]**', 'biosynthesis of linoleic acid [metabolite]', 'linoleic acid biosynthesis [metabolite]'], ['synthesis of GLA [metabolite]'], ['LA synthesis [phenotype]'], ['**synthesis of linolenic acid [metabolite]**', 'production of linolenic acids [metabolite]'], ['erucic acid synthesis [metabolite]'], ['seed alpha-linolenic acid biosynthesis [process]'], ['production of arachidonic acid [metabolite]', '**synthesis of arachidonic acid [metabolite]**'], ['ARA production [process]'], ['α-linolenic acid biosynthesis [metabolite]'], ['synthesis of nervonic acid [metabolite]']]}\n",
      "-------------------\n",
      "{'49211': [['T-DNA inserted in third intron of BIM1 [gene]'], ['T-DNA mutants of AtMMS21 [mutant]'], ['T-DNA insertion mutants affecting HO3 and HO4 [mutant]'], ['T-DNA insertional mutants in AtHMA1 [mutant]'], ['gammaH2AX levels in atm [mutant]']]}\n",
      "-------------------\n",
      "{'152916': [['GID1 homologs [gene]'], ['Omp85 homologs [gene]'], ['Fpg/Nei homologs [gene]'], ['Haspin candidate gene(s) [gene]'], ['SPT homologs [gene]'], ['**HEIP1 homologs [gene]**', 'homologs of HEIP1 [gene]'], ['OSIP108 homologues [gene]'], ['4 EHD orthologs [gene]'], ['orthologs of HSP21 [gene]'], ['MFS homologs [gene]'], ['Hsp70 homologs [gene]'], ['Thlaspi homologues of ClpP protease [gene]'], ['MIP homologues [gene]']]}\n",
      "-------------------\n",
      "{'171677': [['aldo-keto reductases (AKRs) [enzyme]'], ['aldo-keto reductase (AKR) superfamily [gene family]'], ['aldo-keto reductases (AKRs) [protein family]'], ['**aldo/keto reductase [gene]**', 'aldo/keto reductase (ChAkr) [gene]'], ['aldo-keto reductase [protein]']]}\n",
      "-------------------\n",
      "{'10272': [['**HR cell death [phenotype]**', 'HR cell death symptoms [phenotype]'], ['**HR cell death lesion [phenotype]**', 'formation of HR-like lesions [phenotype]', 'extended HR-like lesioning [phenotype]', 'HR-like spontaneous lesion formation [phenotype]'], ['constitutive HR-like cell death [phenotype]'], ['high HR cell death rate [phenotype]'], ['abrogated HR [phenotype]'], ['HR cell death area [phenotype]'], ['HR cell death domain [phenotype]'], ['microscopic HR [phenotype]'], ['HR cell death activity [phenotype]']]}\n",
      "-------------------\n",
      "{'63849': [['DCH model [phenotype]'], ['**manganese cluster [protein complex]**', 'Mn4O5Ca cluster [protein complex]', 'Mn4CaO5 cluster [protein complex]', 'Mn<sub>4</sub>CaO<sub>5</sub> cluster [protein complex]', 'Mn cluster of oxygen-evolving complex [protein complex]'], ['functional high-valence manganese cluster [protein complex]'], ['Cu2+-coordination site [protein complex]'], ['manganese-binding polypeptides [protein]'], ['trinuclear Cu cluster (T2/T3) [subcellular compartment]'], ['dinuclear Fe [metal complex]'], ['MnZn [metal complex]'], ['mononuclear Fe(II) site [metal complex]'], ['oxygen sites of the manganese cluster [protein complex]'], ['metal ion coordinator D-subunit [protein]'], ['Mn2+ supply for protein(s) glycosylation [function]'], ['catalytic manganese cluster [protein]'], ['four Mn-CDFs [protein family]'], ['manganese-stabilizing protein(s) [function]'], ['four thiolate ligands [chemical group]']]}\n",
      "-------------------\n",
      "{'25270': [['reduced cotyledon area [phenotype]', '**smaller cotyledons [phenotype]**', 'smaller cotyledon area [phenotype]', 'decreased size of cotyledons [phenotype]', 'smaller cotyledon [phenotype]', 'reduction of cotyledon size [phenotype]', 'reduction in cotyledon size [phenotype]', 'reduction in cotyledon area [phenotype]'], ['smaller leaf and cotyledons [organ]'], ['narrower lamina of cotyledons [phenotype]'], ['smaller cotyledons and leaf [phenotype]'], ['large reduction in cotyledon area [phenotype]']]}\n",
      "-------------------\n",
      "{'46999': [['ap2 mutants [mutant]'], ['AtENO2 mutation (eno2-) [mutant]'], ['ppd2-1 [mutant]'], ['ppd1-2 [mutant]'], ['aterg2-1 [mutant]'], ['aterg2 +/- [mutant]'], ['nrt2.7-2 mutant allele [mutant]'], ['hsi2-2 [mutant]'], ['Null adt2 [mutant]'], ['**HD2A single ko-mutant [mutant]**', 'hd2a mutant [mutant]'], ['HD2B knock-down [mutant]'], ['**hd2ahd2b line [mutant]**', 'loss of HD2A and HD2B function [mutant]'], ['**Atfer2 mutant seeds [mutant]**', 'Atfer2 seeds [mutant]'], ['HDA108 mutant [mutant]'], ['ldl2 [mutant]'], ['lec2 [mutant]'], ['gl2.8 [mutant]'], ['suvh5-2 [mutant]'], ['GL2 [mutant]'], ['lec2-1 abi3-5 [mutant]'], ['lacs2 mutants [mutant]'], ['hub2 mutants [mutant]'], ['seeds lacking HD2C [mutant]'], ['log2-1 mutant [mutant]'], ['CENH3-based HI [mutant]'], ['ntm2-1 [mutant]'], ['mutants in PEG2 [mutant]'], ['**da2-1 [mutant]**', 'da2-1 mutant [mutant]'], ['tfa2 [mutant]'], ['Ds-tagged hy2 mutant [mutant]'], ['lh-2 [mutant]'], ['hub1-2 [mutant]'], ['tfIIs-2 [mutant]'], ['hta3/hta3 hta5/+ seeds [mutant]'], ['hta5/hta5 hta3/ + seeds [mutant]'], ['athb2 seeds [mutant]'], ['thal-2 seeds [phenotype]'], ['hub1-5 mutants [mutant]'], ['emf2/+ [mutant]'], ['vrn2/- emf2/+ mutant [mutant]'], ['mucilage-modified 2 (mum2) [mutant]']]}\n",
      "-------------------\n",
      "{'115818': [['DEL-based NJ tree [phenotype]'], ['mitogenomes [gene]'], ['numt and mitogenome [gene]'], ['2984 unigenes [gene]'], ['MMGs [phenotype]'], ['presence of IM (PTOX) in their genomes [gene]']]}\n",
      "-------------------\n",
      "{'2178': [['**kernel set [phenotype]**', 'Kernel sets [phenotype]'], ['**two zygotes [phenotype]**', 'two separate double fertilization events [phenotype]', 'dizygotic twins [phenotype]'], ['post-fertilisation events [phenotype]'], ['occasional fertilization [phenotype]']]}\n",
      "-------------------\n",
      "{'80040': [['most information on spliceosome composition [knowledge]'], ['human spliceosomes [organism]']]}\n",
      "-------------------\n",
      "{'106176': [['first contig [metabolite]'], ['Sequencing data [metabolite]', 'The sequence data [metabolite]', '**sequence data [metabolite]**'], ['coding-sequence of bait [metabolite]'], ['oligo [metabolite]'], ['**Genome sequence data [metabolite]**', 'genomic sequence data [metabolite]'], ['accumulation of sequence data [metabolite]'], ['translations of crude genome [metabolite]'], ['sequence fragments [metabolite]'], ['samples and series [metabolite]'], ['geminivirus-like genomic sequence [metabolite]'], ['**genomic DNA sequence reads [metabolite]**', 'genome derived reads [metabolite]'], ['identified repeat sequences [metabolite]'], ['DNA sequence information [metabolite]', '**Sequence information [metabolite]**'], ['**consensus sequence [metabolite]**', 'Consensus sequences [metabolite]'], ['biological sequences [metabolite]'], ['mapped sequences [metabolite]'], ['non-redundant seed sequences [metabolite]'], ['sequencing results [metabolite]'], ['genome resequencing data [metabolite]'], ['**Large numbers of sequences [metabolite]**', 'total sequence numbers [metabolite]'], ['large-scale sequences [metabolite]'], ['DNA sequence data [metabolite]'], ['defined nucleic acid sequences [metabolite]'], ['sequence of the studied organism [metabolite]'], ['genomic templates [metabolite]'], ['information from several sequences [metabolite]'], ['lists of base positions [metabolite]'], ['strand [metabolite]'], ['sequenced clones [metabolite]'], ['Synthetic DNA sequences [metabolite]'], ['GC content of sequence [metabolite]'], ['query sequences [metabolite]'], ['grape sequences [metabolite]'], ['all sequences [metabolite]'], ['defined stretches of sequence [metabolite]'], ['accessibility of sequence information [metabolite]'], ['primer sequences of non-SNP markers [metabolite]'], ['set of reference sequences [metabolite]'], ['cost per Mb DNA sequence [metabolite]'], ['contigs sequences [metabolite]'], ['size and sequence files of chromosomes [metabolite]'], ['constructed repeats [metabolite]'], ['known sequences [metabolite]'], ['sequence scaffolds [metabolite]'], ['seed sequences [metabolite]'], ['a sequence barcode [metabolite]'], ['large quantities of genomic data [metabolite]'], ['sequence databases [metabolite]'], ['gene(s) sequence data [metabolite]'], ['number of molecules of target sequence [gene]']]}\n",
      "-------------------\n",
      "{'87763': [['**beginning of light period [treatment]**', 'beginning of the light period [treatment]'], ['middle of the light period [treatment]'], ['**end of light period [treatment]**', 'the end of the light period [treatment]', 'end of the light period [treatment]'], ['subsequent light phase [treatment]'], ['late in the dark period [treatment]'], ['onset of the dark period [treatment]'], ['single lights-off signals [treatment]']]}\n",
      "-------------------\n",
      "{'168604': [['elevated levels of 9-NAE-HOD [metabolite]', 'increase in 9-NAE-HOD [metabolite]', '**increased 9-NAE-HOD [metabolite]**'], ['increase in concentrations of 13-HOD [metabolite]', '**increased 13-HOD [metabolite]**'], ['increase in concentrations of 9-HOD [metabolite]'], ['elevated quantities of endogenous 9-NAE-HOD [metabolite]', '**elevated levels of endogenous 9-NAE-HOD [metabolite]**'], ['increased 13-NAE-HOD [metabolite]']]}\n",
      "-------------------\n",
      "{'96305': [['tip [organ]'], ['growing apex [organ]', 'apex of buds [organ]', '**plant(s) apex [organ]**', 'apex of plant(s) [organ]', 'tip of each plant(s) [organ]', 'the apex [organ]'], ['sepal tip [organ]'], ['apex to bottom [organ]'], ['tip of the stem(s) [organ]'], ['leaf tips of seedlings [organ]'], ['**seedling root(s) tip [organ]**', 'developing root(s) apex [organ]'], ['plant(s) apex [organism]'], ['tips of cotyledon [organ]'], ['border of the apex [organ]']]}\n",
      "-------------------\n",
      "{'31940': [['thylakoid fraction [subcellular compartment]'], ['**Thylakoid-attached lipid droplets [subcellular compartment]**', 'thylakoid-associated lipid droplets [subcellular compartment]'], ['Thylakoid lipids [subcellular compartment]'], ['thylakoid membrane fraction [subcellular compartment]'], ['substrates in both fractions of thylakoid membrane [thylakoid membrane]'], ['non-appressed thylakoid membrane [subcellular compartment]'], ['both membrane fractions [subcellular compartment]'], ['membrane fractions of chloroplast [subcellular compartment]'], ['bulk of structural lipids in photosynthetic membranes [subcellular compartment]'], ['all other membranes [subcellular compartment]'], ['thylakoid-associated monolayer lipid particles [subcellular compartment]']]}\n",
      "-------------------\n",
      "{'23986': [['**Natural age-dependent senescence [phenotype]**', 'aging pathway [phenotype]', 'a natural age-dependent process [phenotype]'], ['**Delay of senescence [process]**', 'delay the progress of senescence [process]', 'Delaying senescence [process]'], ['**senescence-retarding effect [phenotype]**', 'CK to delay senescence [phenotype]', 'bolsters delayed senescence [phenotype]', 'delay of senescence programs [phenotype]'], ['developmentally controlled senescence pathways [phenotype]', '**Senescence regulation [phenotype]**', 'Fine tuning senescence processes [phenotype]', 'Senescence control [phenotype]', 'regulation mechanism of senescence [phenotype]', 'senescence regulatory processes [phenotype]', 'regulating senescence [phenotype]', 'suppression of senescence [phenotype]', 'Regulation of senescence [phenotype]', 'different metabolic and regulatory aspects of senescence [phenotype]', 'central and important role in senescence [phenotype]', 'outstanding roles in senescence regulation [phenotype]', 'expression of key regulators in senescence [phenotype]', 'prevented senescence [phenotype]', 'an important role in the control of senescence [phenotype]', 'senescence regulatory pathway [phenotype]', 'coordination of the senescence program [phenotype]', 'an autoregulatory senescence inhibition system [phenotype]', 'stronger regulation in senescence [phenotype]', 'control of senescence [phenotype]', 'senescence controlling [phenotype]', 'significant role in senescence process [phenotype]', 'including those involved in senescence [phenotype]', 'controlling senescence [phenotype]'], ['control of barley senescence [phenotype]'], ['regulation of ethylene responses in seedling growth and flower senescence [phenotype]'], ['cell cycle control in senescence process [phenotype]'], ['regulation of senescence in A. thaliana [phenotype]'], ['regulation of flower organ senescence [phenotype]'], ['key components in control of plant(s) senescence and defense processes [phenotype]'], ['plant(s) senescence control [phenotype]', 'regulation of whole plant(s) senescence [phenotype]', '**regulation of plant(s) senescence [phenotype]**', 'plant(s) senescence regulation [phenotype]'], ['regulation of senescence and maturation [phenotype]'], ['**postharvest senescence regulation [phenotype]**', 'regulators of postharvest senescence [phenotype]', 'molecular regulation of postharvest senescence [phenotype]'], ['coordination of salt-triggered senescence [phenotype]'], ['programmed cell death and senescence [phenotype]'], ['yin-yang coordination [phenotype]'], ['flower senescence in ethylene-insensitive plant(s) [phenotype]'], ['senescence/dormancy processes [phenotype]'], ['mechanisms of perennial plant(s) maturation [phenotype]'], ['pre-senescence process [process]']]}\n",
      "-------------------\n",
      "{'80229': [['helicase domain of DCL1 [protein]'], ['processive DCL activity [phenotype]'], ['DCL2 functionality [phenotype]', '**DCL2 activity [phenotype]**'], ['HYL1-DCL1 interaction [protein complex]'], ['alternative DCL activities [process]'], ['DCL cleavage [gene]'], ['**DCL2 action [process]**', 'DCL2 activities [process]'], ['DCL activity [process]'], ['DCL1 involvement [process]'], ['DCL usage [process]'], ['CED-3 caspase-mediated cleavage of the Dicer protein(s) [phenotype]'], ['sequential DCL cleavage [process]'], ['high dicing activity of DCL3 and DCL4 [process]', 'dsRNA-cleaving activities of Dicer-like 3 (DCL3) and 4 (DCL4) [process]', '**dicing activities of DCL3 and DCL4 [process]**'], ['activity of Drosha [protein]'], ['DCL1 recruitment [process]'], ['two catalytic RNase III domains [subcellular compartment]'], ['DCL1 regions [subcellular compartment]'], ['enzymatic activity of DCL1 [phenotype]', '**DCL1 enzyme activity [phenotype]**'], ['DCL3 activity [protein]'], ['cleavage positioning of DCL1 [process]', 'the cleavage activity of DCL1 [process]', 'cleavage activity by DCL1 [process]', '**DCL1-mediated cleavage [process]**', 'DCL1 cleavage activity [process]'], ['mediating dsRNA- and DCL1-interactions [process]'], ['DCL1 recruitment [phenotype]'], ['substrate specificity of DCL3 [process]'], ['DCL-mediated dicing [treatment]'], ['activities of DCL1 [gene]'], ['DCL-dependent cleavage [process]'], ['activity of fungal DCLs [gene]'], ['activity of DCL1 [protein]', '**DCL1 activity [protein]**'], ['DCL3 activity [process]'], ['DCL1-dependent nuclear process [process]'], ['activity of DCL1 [process]'], ['activity of DCL1 complex [phenotype]'], ['assembly of DCL1 complex [phenotype]'], ['maternal activity of DCL3 [gene]'], ['paternal activity of DCL3 [gene]'], ['processing by DCL2 and DCL3 [process]'], [\"DCL1's processing activity [phenotype]\"], ['DCL1 in presence of RNA [protein]'], ['association of DCL1 [enzyme]'], ['cotranscriptional recruitment of DCL1 [protein]'], ['DCL1 in nucleoplasm [subcellular compartment]'], ['recognition activity of DCL1 [process]'], ['DCL step [process]'], ['same DCL-determined manner [process]'], ['DCL processing [protein]'], ['full length protein(s) activity by sequestering the DCL cofactors required for miRNA and siRNA production [phenotype]'], ['DCL2 expression [process]'], ['processing by DCL3 [process]'], ['secondary structure of DCL1 cleavage sites [gene]'], ['DCL processing [process]'], ['specialized DCL functions [process]'], ['compartmentalized DCL functions [process]']]}\n",
      "-------------------\n",
      "{'135188': [['**OEP80 [gene]**', 'AtOEP80/atTOC75-V [gene]', 'AtOEP80 [gene]'], ['OEP40 [gene]'], ['Oep16-2 [gene]'], ['OEP16-1 [gene]'], ['OEP16.4 [gene]'], ['OEP16-1.3 [gene]'], ['AtOEP37 [gene]'], ['OTP85 [gene]'], ['OTP970 [protein]'], ['ORGANELLE TRANSCRIPT PROCESSING82 (OTP82) [gene]']]}\n",
      "-------------------\n",
      "{'23916': [['flowering network [phenotype]', 'floral pathway integration [phenotype]', 'intricate network of flowering pathways [phenotype]', '**floral network [phenotype]**', 'floral integration pathways [phenotype]'], ['floral organ identity programs [phenotype]']]}\n",
      "-------------------\n",
      "{'101327': [['**endophyte-inoculated A. thaliana [organism]**', 'colonized A. thaliana [organism]', 'E+ A. thaliana [organism]', 'leaf endophytic colonization in A. thaliana [organism]', 'endophytism in A. thaliana [organism]', 'the endophytic compartment of A. thaliana [organism]'], ['Analyze the impact on rhizosphere indigenous bacterial community of A. thaliana [organism]', '**Wild-type A. thaliana rhizosphere [organism]**', 'rhizosphere of wild-type A. thaliana [organism]', 'Bacterial rhizosphere microflora of A. thaliana [organism]'], ['**A. thaliana microbiota [organism]**', 'microbial community of A. thaliana [organism]', 'bacteria in A. thaliana [organism]', 'A. thaliana-associated bacterial communities [organism]'], ['A. thaliana phyllosphere resident yeasts [organism]'], ['soil with A. thaliana [organism]'], ['**A. thaliana leaf bacteria [organism]**', 'maintaining normal bacterial microbiota in A. thaliana leaf [organism]', 'bacteria that frequently colonize A. thaliana leaf [organism]'], ['examination of potential connection between microbiota and previously reported autoimmune mutants in A. thaliana [treatment]'], ['air inoculum in A. thaliana [phenotype]'], ['endophytic A. thaliana microbiome assembly [phenotype]'], ['volatiles emitted by A. thaliana [organism]'], ['dispersal of other commensals [organism]'], ['their host A. thaliana plant(s) [organism]'], ['Endophytic colonization of A. thaliana by SA187 [phenotype]'], ['endogenous substance in A. thaliana [organism]'], ['in the presence of secretions from other A. thaliana plant(s) [organism]']]}\n",
      "-------------------\n",
      "{'60288': [['previous analysis [phenotype]'], ['completed [phenotype]'], ['tested [phenotype]']]}\n",
      "-------------------\n",
      "{'26457': [['reduced excess light availability [phenotype]'], ['**reduced photon flux [phenotype]**', 'fewer photons [phenotype]', 'low amount of photons [phenotype]'], ['Diminished stratospheric ozone [phenotype]'], ['Reduced light penetration [phenomenon]'], ['less than 45% of incident light [phenotype]'], ['much less heat [phenotype]'], ['less severe phenotypes with respect to photosynthesis [phenotype]'], ['lower light intensity for saturation of photosynthesis [phenotype]'], ['reduced carbon fixation and growth at low irradiance [phenotype]'], ['smaller effect of growth irradiance [phenotype]'], ['lower at low growth irradiance [phenotype]'], ['as a result of lower growth irradiance [phenotype]'], ['low at low growth irradiance and high growth temperature [phenotype]'], ['poor penetration depth [phenotype]'], ['reduced proportion of light-response and photosynthesis-related gene(s) [gene]'], ['lower percentage of light absorbance [phenotype]'], ['low spectral intensities [phenotype]'], ['lower amplitude photoconversion cross-sections [phenotype]'], ['decreased quantum yields [phenotype]'], ['reduction in photosynthetically active radiation [phenotype]'], ['lower irradiances [phenotype]'], ['reduced radiation use efficiency [phenotype]'], ['down-regulation of Rubisco activity at limiting light [phenotype]', '**Rubisco activity at limiting light [phenotype]**']]}\n",
      "-------------------\n",
      "{'145216': [['**cytokinin response regulator ARR1 [gene]**', 'Cytokinin module A. thaliana RESPONSE REGULATOR1 (ARR1) [gene]', 'CK signaling regulator A. thaliana RESPONSE REGULATOR1 (ARR1) [gene]'], ['ARR3 [gene]', '**cytokinin response regulator ARR3 [gene]**'], ['Cytokinin-inducible type-B A. thaliana RESPONSE REGULATOR (ARR)-WUSCHEL (WUS) module [gene]'], ['regulation of A. thaliana response regulators (ARRs) [gene]']]}\n",
      "-------------------\n",
      "{'141356': [['PME6 promoter [gene]'], ['**PAP1 promoter [gene]**', 'promoter of PAP1 [gene]', 'PAP1 gene(s) promoter [gene]'], ['**BrPYP promoter [gene]**', 'promoter region of BrPYP [gene]'], ['p85 [promoter]'], ['UP9C promoter [gene]'], ['TPK promoter [gene]'], ['PIP5K9 promoter [gene]'], ['PI4Kgamma5 promoter [gene]'], ['KNS4/UPEX1 promoter [gene]'], ['PAP8 promoter [gene]'], ['PYE promoter [gene]'], ['pyk10 promoter [gene]'], ['PKp1 promoter [gene]'], ['PKp2 promoter [gene]'], ['promoters of EXPA8 [gene]'], ['PEP promoter [description]'], ['PICKLE (PKL) promoter [gene]', '**PKL promoter [gene]**'], ['IkappaBa-promoter [gene]'], ['EXPA9 promoter [gene]'], ['Pipox promoter [gene]'], ['MAPK12 promoter [gene]'], ['PyANS promoter [gene]'], ['PKp-beta1 promoter [gene]']]}\n",
      "-------------------\n",
      "{'76699': [['G2 sequence motif [sequence motif]'], ['Consensus motif [feature]'], ['xExT motif [phenotype]'], ['euANT1 motif [gene]'], ['REP program [tool]'], ['His-Ans-His sequence motif [gene]'], ['CENPC motif [sequence]'], ['DESE sequence motif [protein motif]'], ['DDE motif [gene]'], ['similar N-terminal motif [gene]'], ['conserved core motif [phenotype]'], ['conserved MPS motif [protein motif]'], ['SCOOP-like motif [sequence motif]'], ['NSE/DTE motifs [protein motif]'], ['classic SAP motif [protein motif]'], ['STAESS motif [protein]'], ['four-Cys motif [protein motif]'], ['HST motif [phenotype]'], ['conserved motif (IVLEI) in the C-terminal region [protein]'], ['consensus sequence of HPR motif [sequence]'], ['R(S/T)S motif [protein]'], ['Q motifs [phenotype]'], ['N motif [protein motif]'], ['H motif [protein motif]'], ['P motif [protein motif]'], ['conserved (I/V)X(I/V) motif [sequence motif]'], ['highly conserved PI motif [protein]'], ['DPSIVITTYEGEHNH [conserved motif]'], ['canonical consensus of the TOS motif [protein motif]'], ['DUF548 sequence motif [sequence motif]']]}\n",
      "-------------------\n",
      "{'156169': [['bst4 [gene]'], ['standard BST1-3 [gene]'], ['CEBiP receptor [gene]'], ['DBCP [gene]'], ['FKBP15-2 immunophilin [gene]'], ['XBP1 [gene]'], ['FKBP12 [gene]'], ['FKBP39 [gene]'], ['GeBP8 [gene]'], ['GeBP9 [gene]'], ['Expression of WBC19 [gene]'], ['AtBiP-1 [gene]'], ['AtBiP-2 [gene]'], ['ZmBiP2 [gene]'], ['HsFKBP12 [gene]'], ['GbSBT1 [protein]'], ['StMYB44 protein(s) [protein]'], ['GTPBP4 [gene]'], ['PpHsp70-BiP1 [gene]'], ['PpHsp70-BiP2 [gene]'], ['gene(s) encoding PYK10-binding protein(s) PBP1 [gene]'], ['MSBP2 [gene]'], ['human TGFBIp [gene]'], ['BsTIP2;1 [gene]'], ['human CtBP [gene]'], ['HSD17B4 [gene]'], ['hupA [gene]'], ['C/EBP Homologous protein(s) (CHOP) gene(s) [gene]'], ['XBP1 in metazoans [gene]'], ['expression of BcBIP1 [gene]'], ['Efs2 [gene]'], ['BpWRKY17 protein(s) [gene]']]}\n",
      "-------------------\n",
      "{'35811': [['Central Cell nucleus [cell type]'], ['basal endosperm transfer layer (BETL) cells [cell type]', '**BETL cells [cell type]**', 'BETL [cell type]'], ['peripheral aleurone-like cell layer [cell type]'], ['Sub-aleurone Cells [cell type]'], ['endb layer [cell type]'], ['formation of nucellar projection and endosperm TCs [cell type]'], ['nucellar projection [cell type]'], ['**endocarp b [cell type]**', 'endocarp-b layer [cell type]'], ['transfer cells of the nucellar projection and modified aleurone [cell type]'], ['aleurone and underlying starchy endosperm cells [cell type]'], ['basal endosperm transfer cell layer of maize seed [tissue]', '**basal endosperm transfer cell layer (BETL) [tissue]**'], ['>40% of the cell volume in barley aleurone [cell type]'], ['more developed basal endosperm transfer cell layer (BETL) [tissue]'], ['basal endosperm transfer cell formation [process]'], ['chalazal region of developing seed [cell type]'], ['MCE [cell type]'], ['PEN [cell type]'], ['CZE [cell type]'], ['aleurone cell layer [cell type]'], ['enlarged endosperm nodules [cell type]'], ['aleurone and subaleurone [cell type]'], ['nucellar projection cells in wheat [organ]'], ['a homodiploid central cell [cell type]'], ['defects in basal endosperm transport layer (BETL) [tissue]'], ['two independent endosperms [cell type]'], ['two aleurone layers [cell type]'], ['alterations of micropylar endosperm cells [organ]']]}\n",
      "-------------------\n",
      "{'123227': [['Retained intron transcripts [transcript feature]'], ['i transcripts [transcript]'], ['IGN transcripts [transcript]'], ['transcripts that retain introns [gene]'], ['Transcripts with intron retention events [molecule]']]}\n",
      "-------------------\n",
      "{'42703': [['**mislocalization of PIN protein(s) [phenotype]**', 'mislocalized PIN transporters [phenotype]'], ['mistargeting of the auxin efflux carrier PIN FORMED1 [phenotype]'], ['mislocalization of PIN1 and PIN7 transporters [phenotype]'], ['mislocalization of PIN1 efflux carriers [gene]'], ['PIN1 mislocalization [subcellular compartment]'], ['mislocalization of PIN auxin transporters [protein]']]}\n",
      "-------------------\n",
      "{'136975': [['**decreased expression of XTH1 [gene]**', 'suppressed XTH1 expression [gene]'], ['Decreased expression of GL2 [gene]', '**suppressed GL2 [gene]**'], ['Inactivation of DGAT1 [gene]'], ['negative effect on expression of XTH20 [gene]'], ['repression of XTH17 and XTH20 expression [gene]'], ['downregulated homologs of GAUT4 [gene]'], ['downregulation of TGG1 [enzyme]'], ['downregulation of TGG2 [enzyme]'], ['reduced activity of YUC1 and YUC4 [gene]'], ['reduced expression of GALT5 [gene]']]}\n",
      "-------------------\n",
      "{'24942': [['WT crossovers [phenotype]'], ['WT networks [phenotype]'], ['WT and control [phenotype]', '**wildtype control [phenotype]**'], ['WT vein networks [phenotype]'], ['WT lines [phenotype]'], ['organ size of WT [phenotype]']]}\n",
      "-------------------\n",
      "{'108812': [['EWAS [process]'], ['**Novel epialleles [genetic element]**', 'new epialleles [genetic element]'], ['teM epialleles [metabolite]'], ['teM epialleles of MAF3 [metabolite]'], ['epiRILs [phenotype]'], ['Naturally occurring epiallele [gene]'], ['methylation state of spontaneous epialleles [gene]'], ['3MSP epiQTL on chromosome 3 [gene]'], ['metastable epialleles [phenotype]'], ['alleles of CMT2 [genetic variation]'], [\"CMT2' alleles [genetic variation]\"], ['gbM epialleles [epigenetic state]'], ['Epi-alleles of ncRNA locus At4g15242 [phenotype]'], ['activated epi-alleles [phenotype]'], ['activated epi-allele of At4g15242 [phenotype]'], ['activated epigenetic state of a demethylated E-TEL [phenotype]'], ['paternally contributed fie2 [gene]'], ['heritable FWA methylation [genomic locu]'], ['new epiallele [gene]'], ['emQTL [genetic variant]'], ['Pathway-mQTL [genetic variant]'], ['EWAS [genetic study]'], ['paternally inherited methylated Col alleles [gene]'], ['epi-alleles for Gypsy elements [phenotype]'], ['remethylation properties in epiRILs [phenotype]'], ['emergence of epialleles [phenotype]'], ['maternal allele demethylation [phenotype]'], ['five potential epialleles [gene]'], ['forward epimutation [phenotype]']]}\n",
      "-------------------\n",
      "{'100854': [['plant(s) Ontology (PO) terms [ontology]', '**plant(s) ontology [ontology]**', 'The plant(s) Ontology [ontology]'], ['The plant(s) anatomical entity branch [concept]'], ['plant(s) anatomical entity [concept]'], ['The root(s) term plant(s) anatomical entity [concept]'], ['a parent term [concept]'], ['plant(s) anatomical entity branch [organ]'], ['plant(s) anatomical entities [organ]'], ['plant(s) taxonomic classification feature [biological function]'], ['plant(s) anatomical entity branch [feature]']]}\n",
      "-------------------\n",
      "{'50885': [['MSH1-dr phenotype [phenotype]'], ['patrol1 mutant phenotype [mutant]'], ['detected in cat2-1 mutant [mutant]'], ['present in MET1 + segregants [mutant]'], ['max mutant phenotype [phenotype]'], ['plastid phenotype of hy1 [phenotype]']]}\n",
      "-------------------\n",
      "{'4510': [['salt-specific signals [phenotype]', 'salt signals [phenotype]', '**salt-stress signal [phenotype]**'], ['key indicator of salt damage symptom [phenotype]', 'salt-induced stress symptoms [phenotype]', '**salt stress symptoms [phenotype]**', 'symptoms of salt stress [phenotype]'], ['salt stress symptoms in tomato [phenotype]']]}\n",
      "-------------------\n",
      "{'100895': [['plant(s) with only chloronemata [organism]'], ['Asynchronously entrained plant(s) [organism]'], ['out-of-phase plant(s) [organism]'], ['gene(s) losses in nonAM plant(s) [organism]'], ['plant(s) with source leaf removed [organism]'], ['incompatible plant(s) [organism]'], ['none of the progeny plant(s) [organism]'], ['Unadapted plant(s) [plant type]'], ['multi-layered and opaque plant(s) specimens [organism]'], ['two lower plant(s) [organism]'], ['Most plant(s) [plant]'], ['plant(s) with immotile gametes [organism]'], ['plant(s) species without LHCB3 and LHCB6 [organism]'], ['out-of-phase N. attenuata plant(s) [organism]'], ['plant(s) that do not produce berberine [organism]'], ['plant(s) with non-canonical telomeres [organism]'], ['plant(s) lacking Casparian strips [organism]'], ['plant(s) lacking it [organism]']]}\n",
      "-------------------\n",
      "{'41948': [['transparent testa (tt) mutants [mutant]'], ['tds4 mutant [mutant]'], ['**tt4-1 mutant [mutant]**', 'tt4-1 seeds [mutant]'], ['G1 tt4-8 seeds [mutant]', '**tt4-8 mutant [mutant]**'], ['ans-4 mutant [mutant]'], ['chs null mutant tt4 [mutant]', '**transparent testa-4 (tt4) [mutant]**', 'flavonoid-deficient transparent testa4 (tt4) mutant [mutant]', 'tt4 mutant plant(s) [mutant]'], ['mutant tt12 [mutant]', '**TT12 [mutant]**'], ['tt3tt4 [mutant]'], ['Transparent testa4-2 [mutant]'], ['anthocyanin deficiency of tt19 mutants [mutant]'], ['anthocyanin without (aw) mutant [mutant]'], ['13 flavonoid-deficient transparent testa mutants [mutant]'], ['tt4(2YY6) [mutant]'], ['tds4-1 [mutant]']]}\n",
      "-------------------\n",
      "{'167231': [['Capsaicin [metabolite]'], ['Capsaicinoids [metabolite]'], ['Acidic lipopeptides [metabolite]'], ['Metabolism of pipecolic acid [metabolite]'], ['**PEN2-dependent metabolite [metabolite]**', 'PEN2-derived putative compounds [metabolite]'], ['endogenous Pip [metabolite]'], ['piceid and piceatannol [metabolite]'], ['pectic-derived signal [metabolite]'], ['Pip derivatives [metabolite]'], ['90% of the total alkaloids [metabolite]']]}\n",
      "-------------------\n",
      "{'31429': [['active mechanisms [process]'], ['molecular interactions [process]'], ['**Mitochondrion-nucleus cross-talk [process]**', 'Nuclear-mitochondrion interaction [process]'], ['**molecular regulation [process]**', 'understanding molecular regulation mechanisms [process]', 'regulation of molecular processes [process]'], ['Regulatory role [process]', '**regulatory function [process]**', 'key regulatory functions [process]', 'important regulatory functions [process]'], ['regulatory molecule [process]'], ['important regulators [process]'], ['central regulatory modules [process]'], ['multiple orchestrated regulatory processes [process]'], ['molecular mechanisms [system]'], ['central regulatory roles [function]', '**regulatory [function]**'], ['regulatory molecules [function]']]}\n",
      "-------------------\n",
      "{'174281': [['price of HMFS [metabolite]'], ['Content of membrane lipid peroxidation product MDA [metabolite]', '**content of MDA [metabolite]**', 'Malonaldehyde (MDA) content [metabolite]'], ['Levels of lipidated ATG8 [metabolite]'], ['Fish oil-like levels of DHA [metabolite]'], ['significant level of EPA [metabolite]'], ['highest accumulation of EPA [metabolite]'], ['levels of hexadecadienoic acid [metabolite]'], ['low erucic acid [metabolite]'], ['**ω-7 content [metabolite]**', 'omega-7 FA contents [metabolite]'], ['level of 2-arachidonoylglycerol [metabolite]'], ['adding 2.5, 5, and 10 μM of EDA [metabolite]'], ['adding 2.5, 5, and 10 μM of ETrA [metabolite]'], ['fecal NAPE levels [metabolite]'], ['NAPE levels in omental fat [metabolite]'], ['NAPE levels in omental fat depots [metabolite]'], ['total DHA levels [metabolite]'], ['global supplies of fish oil [metabolite]'], ['total DHA accumulation [metabolite]'], ['total and sn-2 DHA [metabolite]'], ['MDA and HAE levels [metabolite]'], ['cholesterol content of meat [metabolite]'], ['MUFA levels [metabolite]'], ['various concentrations of DHA [metabolite]']]}\n",
      "-------------------\n",
      "{'51356': [['Small cylindrical protrusions [phenotype]'], ['enlarged sterile lemma [organ]'], ['reduced gynoecia size [phenotype]'], ['a slightly wider petal claw [phenotype]'], ['slender and shorter siliques [phenotype]'], ['short and inflated ovaries [phenotype]'], ['loss of definition between the style and the ovary [phenotype]'], ['distorted ovule arrangement [phenotype]'], ['extraordinarily wide, heart-shaped ovaries [phenotype]'], ['small differences in flower size [phenotype]'], ['small differences in silique size [phenotype]']]}\n",
      "-------------------\n",
      "{'89653': [['**distribution of auxins [metabolite]**', 'Appropriate distribution of auxin [metabolite]', 'spatial distribution of auxin [metabolite]'], ['proper localization of auxin [metabolite]'], ['local application of auxin [metabolite]'], ['local action of auxin [metabolite]'], ['Diffuse sources of auxin [metabolite]'], ['Distribution of free auxin [metabolite]', '**free auxin distribution [metabolite]**'], ['local deregulation of auxin distribution [metabolite]'], ['distribution of exogenously applied auxin [metabolite]']]}\n",
      "-------------------\n",
      "{'141056': [['upstream AtU6 promoter [gene]'], ['AtSUC2 upstream sequence [gene]'], ['upstream region of ZmCP03 [gene]'], ['upstream promoter Prrn18-156 [gene]'], ['far-upstream region of atpB [gene]'], ['upstream region of AtGPX gene(s) [gene]'], ['**upstream region of AtPGR [gene]**', 'upstream region of AtPGR gene(s) [gene]'], ['upstream sequence in ERF1 [gene]'], ['upstream of HPR1 [gene]'], ['2500bp upstream of ZmPaps [gene]'], ['1000 bp upstream of ZmPaps [gene]'], ['E-box upstream of DYT1 [gene]'], ['AP2 upstream regions [gene]'], ['upstream promoter region of P. dulcis [organism]'], ['upstream region of AtKP1 gene(s) [dna element]'], ['upstream region of Acl1.4 gene(s) [gene]'], ['upstream sequence of AtPRX17 [gene]'], ['region upstream of Sb06g023280 gene(s) [gene]'], ['upstream region of atpC1 [gene]'], ['upstream region of ZmOCL1 [gene]'], ['DNA sequence upstream of SCR [gene]']]}\n",
      "-------------------\n",
      "{'93295': [['adaxial epidermal cell layer [cell type]', 'cells of the adaxial epidermis [cell type]', 'adaxial epidermal cells [cell type]', '**adaxial epidermis [cell type]**', 'Each adaxial epidermal leaf cell [cell type]', 'the adaxial epidermis [cell type]'], ['adaxial protoderm [cell type]'], ['adaxial epidermal cells [tissue]'], ['adaxial epidermal folding [tissue]'], ['division of leaf adaxial epidermal cells [cell type]'], ['division of adaxial epidermis cells [tissue]'], ['adaxial Allium cepa epidermis [tissue]'], ['considerable stomata on adaxial epidermis [tissue]'], ['adaxial and abaxial epidermal leaf cells [cell type]'], ['increased cell divisions in the adaxial epidermis [cell type]']]}\n",
      "-------------------\n",
      "{'69677': [['sequence differences in mitochondrial HPPK/DHPS [protein]'], ['50 amino acids of Tripsacum dactyloides mtDNA [genetic material]'], ['common peptides between Glu1 and Glu2 [protein]'], ['amino acid sequence homologues of RACK1 [protein]'], ['full 364-amino acid sequence of AtSDH [protein structure]'], ['coevolution of SWEET amino acids [gene]'], ['conserved peptide sequences in pGlcT2 [gene]'], ['C. reinhardtii mature GTR amino acid sequence [protein]'], ['plant(s) amino acid sequence [protein]'], ['high similarity was found between amino acid sequences of mabinlin II [protein]'], ['amino acid sequence substitutions in A. thaliana CaM isoforms [protein]'], ['deduced amino acid sequence of DWF1 [gene]'], ['deduced amino acid sequence of PFC1 [gene]'], ['partial amino-acid sequence [phenotype]'], ['amino acid sequence of several tryptic peptides of ESP [protein]'], ['Comparison of amino acid sequences of CAO [enzyme]'], ['protein(s) with a sequence similar to ABS [sequence]'], ['deduced amino-acid sequence of the potato mitochondrial citrate synthase [protein]'], ['deduced amino acid sequences of A. thaliana SMT [protein]'], ['high amino acids sequence similarity with yeast Swd2 [gene]'], ['different SSu peptide sequences [protein]'], ['aa78-93 identical to MtALT2 [gene]'], ['little homology at amino acid level [metabolite]'], ['amino acid sequence similarities [biological relationship]'], ['similarity between primary structures of scorpion toxins [treatment]'], ['high variability in amino acid sequence [feature]'], ['notable amino acid sequence similarity to yeast protein(s) [protein]'], ['significant similarity to AS2 in terms of the deduced amino acid sequence of the carboxy-terminal half [amino acid sequence]'], ['amino acid homology with animal counterparts [gene]'], ['sequence homology with AtVIT1 and ScCCC1 [protein]'], ['primary sequence homology to thioesterases [protein]'], ['same amino acid sequence in CLE motif as FCP1 [phenotype]'], ['high degree of sequence similarity with PSY [metabolite]', '**sequence similarity with PSY [metabolite]**'], ['cassava TIFY amino acid sequences [protein]'], ['amino acid sequence of the hobo transposase [protein]'], ['extension aa sequences in yeast [gene]'], ['high similarity with OTCase family [phenotype]'], ['PP1c sequences from animals [protein]'], ['microbial sequences [protein]'], ['several unique amino acids [phenotype]'], ['StYUC amino acid sequences [gene]'], ['yeast amino acid sequences [organism]'], ['major differences in amino acid sequence [process]'], ['amino acid sequence homology to Sec13 [gene]'], ['deduced amino acid sequences of R. communis SMT [protein]'], ['deduced amino acid sequences of Glycine max SMT [protein]']]}\n",
      "-------------------\n",
      "{'42848': [['silencing of HvRBK1 [mutant]'], ['NACbeta silencing [mutant]'], ['silencing of SlPL9 [gene]'], ['silencing of TAGL1 [mutant]'], ['silencing of SlSHN3 [mutant]'], ['silencing of EjCYP90A [mutant]'], ['Silencing of AtARP4 [mutant]'], ['Silencing of CDPK1 [mutant]'], ['PvBPS1 silencing in Phaseolus [mutant]'], ['SVL silencing [mutant]'], ['EBP1 silencing [mutant]'], ['HvMS1 silencing [gene]'], ['growth retardation of Atbt1 RNAi mutant plant(s) [mutant]'], ['SWC4 silencing [mutant]'], ['AhZIP6 silencing [mutant]'], ['Silencing of OscWRKY1 [mutant]'], ['Downregulation of SlbHLH068 [mutant]'], ['HIP2-silenced leaf [organ]'], ['SlRING1 silencing [mutant]']]}\n",
      "-------------------\n",
      "{'13588': [['**IFM size [phenotype]**', 'size of the inflorescence meristem [phenotype]'], ['SAM geometry [phenotype]'], ['change in meristem size [phenotype]'], ['division zone length in MZ [phenotype]'], ['**SAM size [phenotype]**', 'Height and width of SAM [phenotype]', 'subtle differences in the size of the SAM [phenotype]', 'shoot meristem size [phenotype]', 'vegetative meristem size [phenotype]', 'SAM area [phenotype]', 'SAM size maintenance [phenotype]', 'SAM normal size [phenotype]', 'SAM dimensions [phenotype]', 'overall size of SAM [phenotype]', 'shoot apical meristem size [phenotype]', 'constant SAM size [phenotype]', 'measurement of SAM size [phenotype]', 'size of the SAM [phenotype]', 'maintenance of shoot apical meristem size [phenotype]', 'SAM width [phenotype]', 'size of SAM [phenotype]'], ['finite meristem size [phenotype]'], ['meristem length and structure [phenotype]'], ['**overall meristem size [phenotype]**', 'meristem sizes [phenotype]'], ['sigma [phenotype]'], ['**SAM radius of curvature [phenotype]**', 'meristem radius of curvature [phenotype]'], ['computed for indented meristems [phenotype]'], ['significantly different from index for control meristems [measurement]'], ['intermediate cambial zone size [phenotype]'], ['**shoot and floral meristem size [phenotype]**', 'floral and apical meristem size [phenotype]'], ['**meristem lengths [phenotype]**', 'three classes for meristem length [phenotype]', 'apical meristem length [phenotype]', 'meristem length similar to wild-type [phenotype]', 'length of the meristem [phenotype]', 'varying meristem lengths [phenotype]', 'similar meristem length [phenotype]'], ['accessions with medium meristem size [phenotype]'], ['SAM structure [phenotype]'], ['basal meristem length [phenotype]'], ['meristem positioning/size [phenotype]'], ['size of the OC [phenotype]'], ['small difference in SAM-diameter [phenotype]'], ['JVM basin compared to WT [phenotype]'], ['**Meristematic Zone (MZ) size [phenotype]**', 'MZ size phenotype [phenotype]', 'size of Meristematic Zone (MZ) [phenotype]', 'MZ size to WT [phenotype]'], ['life span of the SAM [phenotype]'], ['nodule meristem size [phenotype]'], ['variation of meristem size [phenotype]'], ['shoot meristem proportion [phenotype]'], ['SAM phenotype [phenotype]'], ['flower meristem size [phenotype]'], ['meristem border size [phenotype]'], ['CMM size [phenotype]'], ['alterations in shoot meristem size [phenotype]'], ['adaptations in meristem size [phenotype]'], ['meristem stability [phenotype]'], ['appropriate size [phenotype]']]}\n",
      "-------------------\n",
      "{'55951': [['Malvalic acid [metabolite]'], ['Excess SeIV [metabolite]'], ['Nervonic acid (24:1) [metabolite]']]}\n",
      "-------------------\n",
      "{'123371': [['Transcript data [treatment]'], ['transcript information [genomic datum]'], ['transcription data [source]'], ['Transcript [genomic region]'], ['transcript information [gene]']]}\n",
      "-------------------\n",
      "{'121683': [['state 5 [phenotype]'], ['10 states [phenotype]'], ['larger scales [phenotype]'], ['smaller scales [phenotype]'], ['definition of chromatin states [phenotype]'], ['multiple traits by incorporating information from chromatin openness [phenotype]'], ['chromatin feature annotations [phenotype]']]}\n",
      "-------------------\n",
      "{'51109': [['autoimmune phenotypes in snc1 class [phenotype]', 'snc1-mediated autoimmunity [phenotype]', '**autoimmune phenotypes of snc1 [phenotype]**', 'snc1 auto-immune responses [phenotype]', 'auto-immunity in snc1 [phenotype]', 'autoimmune phenotypes of suppressor of npr1-1, constitutive1 (snc1) [phenotype]'], ['siz1 auto-immune phenotype [phenotype]', 'Auto-immune phenotype in SIZ1 mutant [phenotype]', 'Auto-immunity in SIZ1 mutant [phenotype]', '**SIZ1 auto-immunity [phenotype]**'], ['SNC1 autoimmune response in srfr1-4 mutant background [phenotype]'], ['**snc1-1 phenotype [phenotype]**', 'Auto-immunity in snc1-1 mutant [phenotype]'], ['autoimmunity of bal/snc1-2 [phenotype]'], ['high temperature-mediated recovery of snc1-1 auto-immunity [phenotype]'], ['autoimmunity in bon1 and snc1 [phenotype]'], ['autoimmune phenotypes of snc4-1D [mutant]'], ['snc4-1D autoimmune phenotypes [phenotype]'], ['snc1-like phenotypes [phenotype]'], ['additive effect on snc1 suppression [phenotype]']]}\n",
      "-------------------\n",
      "{'93067': [['shift in mating system toward selfing [treatment]'], ['Increase [process]'], ['sugar resources diverted toward growth [process]'], ['strategies counteracting inbreeding depression [process]']]}\n",
      "-------------------\n",
      "{'148116': [['ixr1-1 [gene]'], ['**BRASSINAZOLE-RESISTANT 1 [gene]**', 'brassinazole‐resistant 1 (BZR1) [gene]', 'BRASSINAZOLE RESISTANT1 [gene]', 'BRASSINAZOLE-RESISTANT-1 [gene]', 'brassinazole resistant 1 [gene]', 'BRZ1 (BRASSINAZOLE‐RESISTANT1) [gene]', 'Brassinazole resistant 1 (BRZ1) [gene]', 'PtBZR1 (BRASSINAZOLE RESISTANT1) [gene]', 'BrassinaZole-Resistant1(BZR1) [gene]', 'Brassinozole Resistant 1/bri1 EMS-Suppressor 1 [gene]'], ['**JASMONATE RESISTANT 1 (JAR1) [gene]**', 'Jasmonate resistance 1 (jar1) [gene]', 'JASMONATE RESISTANT1 (JAR1) [gene]', 'JASMONATE-RESISTANT1 [gene]', 'AtJAR1 [gene]', 'JAR1 (JASMONATE RESISTANT 1) [gene]', 'JASMONIC ACID RESISTANT1 (JAR1) [gene]'], ['**Coronatine Insensitive 1 (COI1) receptor protein(s) [gene]**', 'COI-1 (CORONATINE INSENSITIVE1) [gene]', 'JA receptor CORONATINE INSENSITIVE 1 (COI1) [gene]'], ['CORONATINE INSENSITIVE1 (COI1) [protein]'], ['**AUXIN RESISTANT1 [gene]**', 'AUXIN RESISTANT1 (AUX1) [gene]', 'AUXIN-RESISTANT1 [gene]'], ['**PYRABACTIN RESISTANCE1-Like protein(s) (PYLs) [gene]**', 'PYRABACTIN RESISTANCE1-LIKEs (PYLs) [gene]', 'PYRABACTIN RESISTANCE-LIKE1 PYL1 [gene]'], ['**Pyrabactin Resistance1 (PYR1) [gene]**', 'PYRABACTIN RESISTANCE 1 (PYR1) [gene]', 'PYRABACTIN RESISTANCE1 PYR1 [gene]', 'Pyrabactin Resistance1 [gene]'], ['JAR1 function [gene]'], ['AZI3 [gene]'], ['COI1 dosage [gene]'], ['**HOPZ-ACTIVATED RESISTANCE 1 (ZAR1) [gene]**', 'HOPZ-ACTIVATED RESISTANCE1 (ZAR1) [gene]'], ['JAX1 [gene]'], ['Auxin resistant 1 (AXR1) [gene]'], ['rib1 [gene]'], ['COPPER MODIFIED RESISTANCE1/PATRONUS1 (CMR1/PANS1) [gene]'], ['suppressor of auxin resistance1 (sar1) [gene]'], ['Expression of PRE1 [gene]'], ['Brassinazole‐resistant1 [protein]'], ['PYRABACTIN RESISTANCE 1 family [gene]'], ['BRASSINAZOLE RESISTANT 1 (BZR1)-mediated pollen formation [phenotype]'], ['jasmonic acid resistant (jar)1-1 [gene]'], ['JASMONATE RESISTANT 1 (JAR1) locus [gene]'], ['jasmonate resistant 6 (JAR6) [gene]'], ['jasmonate resistant 4 (JAR4) [gene]'], ['BRASSINAZOLE RESISTANT 1 (BZR1) expression [gene]'], ['Pyrabactin Resistant 1 (PYR) [gene]'], ['pyrabactin resistance 1/pyrabactin resistance 1-like (PYR1/PYLs) ABA receptors [gene]'], ['rapid nuclear localization of BRASSINAZOLE RESISTANT1/2 [gene]'], ['type III receptor PYRABACTIN RESISTANCE-LIKE 2 [gene]']]}\n",
      "-------------------\n",
      "{'46618': [['jar1 plant(s) [mutant]', 'Jasmonic acid resistant 1 (jar1) [mutant]', 'jar1 seedlings [mutant]', '**jar1 mutation [mutant]**', 'jar1 mutant plant(s) [mutant]', 'JA-deficient mutants (jar1) [mutant]', 'ISR in jar1 mutant [mutant]'], ['jar mutants [mutant]'], ['jar1-1 plant(s) [mutant]', 'jar1-1 mutants [mutant]', '**jar1-1 mutation [mutant]**', 'stomata of jar1-1 mutant plant(s) [mutant]', 'jasmonic acid resistant1 (jar1-1) mutant seedlings [mutant]'], ['jar1/mlo [mutant]'], ['osjar1 loss-of-function mutants [mutant]', '**osjar1 mutants [mutant]**'], ['JAR1-deficient leaf [mutant]'], ['jar1.1 [mutant]'], ['decreased sensitivity to MeJA inhibition of root(s) elongation [phenotype]']]}\n",
      "-------------------\n",
      "{'91142': [['Increasing CO2 from 200 to 300ppm [treatment]'], ['instantaneous CO<sub>2</sub> concentration [treatment]'], ['ambient CO2 (350 ppm) [treatment]'], ['both 400 and 800 ppm CO2 [treatment]'], ['CO2 shift from 0 to 700 ppm [treatment]'], ['about 0.15% CO2 [treatment]'], ['approximately 550 ppm [treatment]'], ['200 to 1000 ppm [treatment]']]}\n",
      "-------------------\n",
      "{'57999': [['proDc3:GUS reporter [gene]'], ['proXTH9::GUS reporter [gene]'], ['FUS3p:GUS reporter [gene]'], ['pAS2:GUS reporter line [gene]'], ['SEP3::GUS reporter gene(s) fusion [gene]'], ['AS2p:GUS reporter gene(s) [gene]'], ['ProPAS2:PAS2-GUS reporter gene(s) [gene]'], ['ProPDF1:GUS reporter [gene]'], ['**PRE6p:GUS reporter [gene]**', 'PRE6p:GUS reporter gene(s) [gene]'], ['pHEC:GUS reporters [gene]'], ['pPLL12:PLL12-GUS reporter [gene]'], ['TSD2:GUS reporter gene(s) [gene]'], ['pZIP4:GUS reporter [gene]'], ['GUS (uidA) reporter gene(s) [gene]'], ['Gal4:GUS reporter gene(s) expression [gene]'], ['Gal4:GUS reporter gene(s) [gene]'], ['OsCLE48p:GUS reporter gene(s) [gene]']]}\n",
      "-------------------\n",
      "{'76685': [['R-K-X-S-X-X-R motif [protein region]'], ['K-X-X-S-X-X-K/R-S/T/R/A motif [protein region]']]}\n",
      "-------------------\n",
      "{'141591': [['pPEM3 plus pPEG3 [gene]'], ['increased scFv-TM43-E10 RNA [gene]'], ['SsD14bDeltaN [gene]'], ['MCO Atcel1 [gene]'], ['DeltaU12 CBP20 gene(s) version [gene]'], ['Overexpression of truncated EMB2654-11M CDS [gene]'], ['CAT1 intron that included E2+E3 [gene]']]}\n",
      "-------------------\n",
      "{'132106': [['ZIP1/3/4/12 and IRT3 [gene]'], ['IRT3 and ZIP1 [gene]'], ['ZIP/IRTs [gene]'], ['Transcripts of IRT1, FRO2, and AHA2 [gene]'], ['IRT3 gene(s) from A. halleri [gene]'], ['All three IRT1 gene(s) [gene]'], ['**FRO2 and IRT1 [gene]**', 'iron-uptake gene(s) IRT1 and FRO2 [gene]']]}\n",
      "-------------------\n",
      "{'26910': [['**crop reduction [phenotype]**', 'decreased yield [phenotype]', 'loss of crop production [phenotype]', 'reduced agricultural production [phenotype]', 'reduction of output [phenotype]', 'reduced crop production [phenotype]', 'decrease in agricultural production [phenotype]', 'crop productivity reduction [phenotype]', 'reduced production of crops [phenotype]', 'reduction in crop production [phenotype]', 'reduced agriculture production [phenotype]', 'reduced production [phenotype]', 'abatement of food production [phenotype]'], ['low cereal crop production [phenotype]']]}\n",
      "-------------------\n",
      "{'81311': [['different extraction buffer compositions [protocol]'], ['standard reconstitution protocol [treatment]', '**reconstitution protocol [treatment]**'], ['evaluation and modification [treatment]'], ['universal protein(s) extraction protocol [protocol]'], ['sufficient quantity for purification [treatment]'], ['quality of histone extracts [modification]'], ['uniform and efficient protein(s) extraction [process]'], ['individually optimized protein(s) extraction protocols [protocol]'], ['individually optimized protein(s) extraction protocols to obtain samples of acceptable quality [treatment]']]}\n",
      "-------------------\n",
      "{'74671': [['TIR class [protein]', '**TIRs [protein]**', 'Toll-interleukin receptor domains [protein]', 'Toll/interleukin-1 like receptor (TIR) [protein]', 'Toll/Interleukin-1 receptor (TIR) [protein]', 'Toll and human interleukin-1 receptor (TIR) domain [protein]', 'Toll-Interleukin-1 receptor (TIR) domain [protein]', 'Toll/interleukin 1-receptor (TIR) [protein]', 'Toll-interleukin 1 receptor (TIR) domain [protein]', 'mammalian Interleukin-1 Receptor (TIR) [protein]', 'Toll-interleukin-1 receptor-like (TIR) domain [protein]', 'toll interleukin 1 receptor (TIR) [protein]'], ['**toll/interleukin-1 receptor (TIR)-domain containing protein(s) [protein]**', 'Toll/interleukin receptor (TIR) domain-containing protein(s) [protein]', 'Several novel types of TIR-domain-containing protein(s) [protein]', 'protein(s) with a Toll/interleukin-1 receptor (TIR) domain [protein]', 'protein(s) containing TIR domain [protein]'], ['**Toll/interleukin-1 receptor resistance (TIR) domain-containing NLRs (TNLs) [protein]**', 'Toll-Interleukin1-Receptor domain (TIR)-family NLRs (TNLs) [protein]', 'Toll/interleukin-1 receptor/resistance protein(s) (TIR) domains [protein]', 'Toll/interleukin-1 receptor/resistance protein(s) (TIR) domain [protein]', 'Toll/interleukin-1 receptor domain (TIR)-NLRs [protein]', 'Toll Interleukin 1 Receptor (TIR)-NB-LRR [protein]'], ['**TIR [protein domain]**', 'Toll/Interleukin-1 receptor (TIR) domains [protein domain]', 'Toll/Interleukin1 Receptor (TIR) domain [protein domain]', 'toll-interleukin receptor (TIR)-type [protein domain]', 'Toll/Interleukin-1-receptor-like domain [protein domain]', 'toll-interleukin-1 receptor (TIR)-like region [protein domain]'], ['**Toll/IL1 receptor/resistance protein(s) (TIR) domain [protein domain]**', 'Toll, Interleukin-1 Receptor, Resistance protein(s) (TIR) domain [protein domain]'], ['Toll/Interleukin-1 receptor (TIR) domains [domain]'], ['Toll-interleukin 1 receptor (TIR) domain [phenotype]'], ['**mammalian Interleukin-1 Receptor (TIR) [subcellular compartment]**', 'Toll/interleukin-1 receptor domains [subcellular compartment]'], ['Toll/Interleukin-1 receptor (TIR) [protein feature]'], ['TIR domain a group of R protein(s) [gene]']]}\n",
      "-------------------\n",
      "{'165554': [['C chain [metabolite]'], ['glucose at reducing end of acarbose [metabolite]'], ['free reducing end [phenotype]'], ['unbranched Glc residue at reducing ends [metabolite]', '**reducing end of glucan chain [metabolite]**'], ['non-reducing end of a pre-existing glucan chain [metabolite]', '**non-reducing end of glucan chain [metabolite]**']]}\n",
      "-------------------\n",
      "{'55077': [['260 protein(s) [protein]'], ['338 protein(s) [protein]'], ['302 protein(s) [protein]'], ['182 protein(s) [protein]'], ['168 protein(s) [protein]'], ['188 protein(s) [protein]'], ['452 protein(s) [protein]'], ['620 protein(s) [protein]'], ['198 protein(s) [protein]'], ['322 protein(s) [protein]'], ['392F [protein]'], ['258T [protein structure]'], ['592 protein(s) [protein]'], ['280 protein(s) [protein]'], ['132 protein(s) [protein]'], ['212 protein(s) [protein]'], ['232 other protein(s) [protein]'], ['250 [protein]'], ['238 protein(s) [protein]']]}\n",
      "-------------------\n",
      "{'15643': [['proper tip growth in root(s) hair cells [phenotype]'], ['proper termination of root(s) hair growth [phenotype]'], ['fast root(s) hair growth [phenotype]'], ['**normal root(s) hair elongation [phenotype]**', 'proper root(s) hair formation and elongation [phenotype]', 'appropriate root(s) hair elongation [phenotype]', 'normal growth of root(s) hairs [phenotype]', 'normal hair elongation [phenotype]'], ['elongation ceased [phenotype]']]}\n",
      "-------------------\n",
      "{'107925': [['H3K4me3 enrichment [metabolite]', 'H3K4 hyper-methylation [metabolite]', 'high levels of histone H3K4me3 [metabolite]', '**increased H3K4me3 levels [metabolite]**', 'elevated H3K4me3 levels [metabolite]', 'sustained enrichment of H3K4 trimethylation [metabolite]', 'sustained H3K4me3 enrichment [metabolite]', 'increased levels of histone 3 lysine 4 trimethylation (H3K4me3) [metabolite]'], ['high levels of H3K4me1 [metabolite]'], ['increased H3K4 trimethylation in MAF5 [metabolite]'], ['increased association with H3K4me3 [metabolite]'], ['enrichment of H3K4me3 [phenotype]', 'increased trimethylation of histone H3 at Lys-4 (H3K4me3) [phenotype]', 'Increased levels of H3K4 trimethylation [phenotype]', 'elevated H3K4me3 [phenotype]', 'increase of H3K4me3 signals [phenotype]', 'Increase of permissive mark H3K4me3 [phenotype]', 'Elevated level of H3K4me3 [phenotype]', '**increased H3K4me3 [phenotype]**', 'high H3K4me3 levels [phenotype]', 'increased histone H3 lysine 4 tri-methylation (H3K4me3) levels [phenotype]', 'high levels of histone H3 lysine 4 methylation (H3K4me3) [phenotype]', 'increased H3K4me3 levels [phenotype]', 'increase in H3K4me3 marks [phenotype]', 'increased H3K4me3 marks [phenotype]', 'increased H3K4 trimethylation [phenotype]'], ['high levels of histone H4K203Me [phenotype]'], ['stronger effects on H3K4me1 [phenotype]'], ['increased signals of H3K4me0 [phenotype]'], ['increased H3K4me3 at 59 loci [phenotype]'], ['atypically high H3K4me3 [phenotype]'], ['**Increased levels of H3K4 trimethylation [process]**', 'enrichment of H3K4 trimethylation [process]'], ['increased H3K4me3 levels at endogenous loci [process]'], ['high levels of H3K4me1 [process]'], ['increasing H3K4me3 at the TSS [epigenetic mark]'], ['overaccumulation of H3K4me3 [epigenetic mark]', '**increased H3K4me3 [epigenetic mark]**', 'increased H3K4me3 modifications [epigenetic mark]', 'enrichment of H3K4me3 [epigenetic mark]', 'abundance of H3K4me3 [epigenetic mark]', 'increase in H3K4me3 [epigenetic mark]', 'high levels of H3K4me3 [epigenetic mark]'], ['high levels of H3K4 methylation [epigenetic mark]'], ['elevated H3K4me1/2 [epigenetic mark]'], ['**H3K4me3 hyper-methylation [epigenetic modification]**', 'HS-induced H3K4 trimethylation (H3K4me3) [epigenetic modification]'], ['**increase in active H3K4 marks [epigenetic modification]**', 'increase in H3K4 marks [epigenetic modification]'], ['increased H3K4me3 marks [subcellular compartment]', 'increased H3K4me3 levels [subcellular compartment]', 'higher levels of H3K4me3 [subcellular compartment]', '**increased H3K4me3 [subcellular compartment]**'], ['increased enrichment of H3K4me2 [subcellular compartment]'], ['increased levels of H3K4me3 marks [modification]', 'Increased levels of H3K4 trimethylation [modification]', 'enrichment of H3K4me3 [modification]', 'elevated H3K4 trimethylation [modification]', 'increase in histone H3 lysine 4 (H3K4) trimethylation [modification]', '**increased H3K4me3 levels [modification]**'], ['elevated H3K4 dimethylation [modification]'], ['increased H4K20me3 in promoter region [modification]'], ['elevated H3K4me3 [marker]'], ['hyper-methylation of H3K4 [marker]'], ['enrichment of H3K4me2 [histone modification]'], ['increased H3K4me3 [histone modification]'], ['**increased H3K4me3 [treatment]**', 'increased histone H3K4 methylation level [treatment]', 'strong enrichment of H3K4me3 [treatment]', 'strong signals for H3K4me3 [treatment]'], ['**increased H3K4me3 [gene]**', 'H3K4 trimethylation at promoter region of ACT7 [gene]'], ['high transcription of H3K4 methyltransferases [gene]'], ['higher level of H3K4me2 [gene]'], ['higher levels of H3K4me3 [chromatin mark]'], ['higher levels of H3K4me2 [epigenetic feature]'], ['H3K4me3 enrichment [protein]'], ['peak of H3K4me3 accumulation [post-translational modification]']]}\n",
      "-------------------\n",
      "{'112506': [['PPR motifs [gene]'], ['five in-tandem PPR motifs [phenotype]'], ['P and S motifs [gene]'], ['2 to 30 PPR motifs [phenotype]']]}\n",
      "-------------------\n",
      "{'5769': [['defence response-related pathways [phenotype]', '**defence-related pathways [phenotype]**', 'defense-related pathways [phenotype]', 'defense mediated pathways [phenotype]'], ['antagonistic pathway [phenotype]'], ['defense-related branch [phenotype]'], ['biotic defense pathways [phenotype]'], ['defence regulatory pathways [phenotype]']]}\n",
      "-------------------\n",
      "{'102196': [['A. thaliana root(s) endodermis [tissue]'], ['A. thaliana root(s) callus [organ]'], ['A. thaliana root(s) endodermis [organ]'], ['A. thaliana root(s) suberin [organ]'], ['A. thaliana LRP margins [tissue]'], ['A. thaliana root(s) cambium [tissue]'], ['vascular bundles of A. thaliana root(s) [tissue]'], ['A. thaliana root(s) protophloem [tissue]'], ['A. thaliana root(s) cap-columella [tissue]'], ['surface of A. thaliana root(s) and shoots [tissue]'], ['inner tissues of A. thaliana root(s) and shoots [tissue]'], ['A. thaliana root(s) membranes [organ]'], ['A. thaliana root(s) endodermis [organism]'], ['endogenous compound in root(s) and basal parts of A. thaliana shoot tissues [tissue]'], ['A. thaliana columella [organ]'], ['root(s) hairs and epidermis of A. thaliana [tissue]'], ['cortex of A. thaliana root(s) [tissue]'], ['**A. thaliana root(s) plasma membranes [tissue]**', 'PM of A. thaliana root(s) [tissue]'], ['vacuolar-enriched membranes of A. thaliana root(s) [tissue]'], ['tissues in the A. thaliana shoot and root(s) [organ]']]}\n",
      "-------------------\n",
      "{'28390': [['**Endomembrane trafficking [process]**', 'endomembrane trafficking route [process]'], ['Endomembrane routes [process]'], ['endomembrane system [process]']]}\n",
      "-------------------\n",
      "{'140092': [['**ERF gene(s) expression [gene]**', 'AP2/ERF transcription factor expression levels [gene]', 'Expression level of most AP2/ERFs [gene]'], ['ER stress marker gene(s) expression [phenotype]'], ['ERF109 expression [gene expression]'], ['**expression of ERF7 [gene]**', 'expressions of ERF7 [gene]'], ['Expression patterns of some ERF subfamily gene(s) [gene]'], ['kinetics of expression of ERF1 [gene]'], ['expression of ERF1 in WS [gene]'], ['expression of ERF114 in WS [gene]'], ['expression of ETHYLENE RESPONSIVE FACTOR53 (ERF53) [gene]'], ['expression of ERF54 [gene]'], ['expression of ERF6 [gene]'], ['expression of ERF106 [gene]'], ['ERF1A expression [gene]'], ['stability of ERFVII transcription factors [gene]'], ['ERF1 expression [process]'], ['expression of ER-stress-related transcripts [gene]'], ['**ERF2 expression [gene]**', 'transcript levels of ERF2 [gene]'], ['ERF3 expression [gene]']]}\n",
      "-------------------\n",
      "{'19453': [['water-use efficiency [phenotype]'], ['**intrinsic water use efficiency [phenotype]**', 'intrinsic water-use efficiency [phenotype]'], ['Water productivity (WP) [phenotype]', '**Water productivity [phenotype]**', 'total biomass produced per unit of water transpired [phenotype]', 'biomass produced per unit of water transpired [phenotype]', 'WP [phenotype]', 'biomass accumulation side of WUE [phenotype]'], ['**amount of carbon gained per unit water consumed [phenotype]**', 'carbon capture per unit of water consumed [phenotype]'], ['carbon-water trade-off [phenotype]'], ['yield, chlorophyll content, and biomass under waterlogging [phenotype]'], ['whole-plant water use efficiency [phenotype]'], ['favorable instantaneous water use efficiency [phenotype]']]}\n",
      "-------------------\n",
      "{'62638': [['His-GroEL [protein]'], ['secreted GRFT [protein]', '**GRFT [protein]**', 'Purified GRFT [protein]', 'GRFT production [protein]'], ['human gamma-interferon [protein]'], ['human glutamate carboxypeptidase II (HsGCPII) [protein]'], ['secreted glycoprotein HSA/GM-CSF [protein]']]}\n",
      "-------------------\n",
      "{'396': [['in vitro bulb formation [phenotype]'], ['spontaneously in tissue culture [phenotype]'], ['**sprouting of in vitro microtubers [phenotype]**', 'induce sprouting [phenotype]']]}\n",
      "-------------------\n",
      "{'170979': [['transcription co-repression Ctbp dehydrogenase [enzyme]'], ['**Ferredoxin-NADP reductase [enzyme]**', 'Ferredoxin-NADP+ reductase [enzyme]', 'ferredoxin NADP+ oxidoreductase [enzyme]'], ['**cytochrome P450 reductase [enzyme]**', 'NADPH-cytochrome P450 reductase [enzyme]', 'cytochrome P450 reductase (CPR) [enzyme]', 'P450 cytochrome reductase [enzyme]', 'NADPH cytochrome P450 reductase (CPR) [enzyme]', 'CPR [enzyme]', 'CYP450 reductase [enzyme]', 'NADPH-cytochrome P450 reductase (CPR) [enzyme]', 'NADPH-dependent CPR [enzyme]', 'NADPH-dependent CPRs [enzyme]', 'NADPH-dependent cytochrome P450 reductase [enzyme]', 'P450 reductase [enzyme]', 'NADPH-CYP reductase [enzyme]', 'NADPH-P-450 reductases [enzyme]', 'NADPH-cytochrome P-450 reductase [enzyme]', 'cytochrome P450 reductases (CPRs) [enzyme]'], ['Microsomal NADPH-cytochrome P-450 reductase [enzyme]'], ['NADPH-CPR (ATR2) [enzyme]'], ['Cytochrome P450 reductase 1 [enzyme]'], ['yeast NADPH-P450 reductase [enzyme]'], ['Cytochrome p450 reductase enzymes [gene]'], ['enzyme activities of CPRs [gene]'], ['NADPH-P450 reductases (ZoRED1 and ZoRED2) [protein]'], ['NADPH-Cyt P450 reductase [protein]'], ['cytochrome P450 reductase [protein]'], ['NADPH-dependent cytochrome P450 reductase [phenotype]'], ['coupling between P450 enzymes and CPRs [enzyme]'], ['**NADPH cytochrome C reductase [enzyme]**', 'NADPH-cytochrome-c reductase [enzyme]'], ['cytochrome c reductase [enzyme]'], ['cytochrome c reductase (complex III) [enzyme]'], ['dimeric cytochrome c reductase [enzyme]'], ['**cytochrome b<sub>5</sub> reductase [enzyme]**', 'NADH-cytochrome b5 reductase (CBR) [enzyme]', 'NADH-dependent CBR [enzyme]', 'CBRs [enzyme]', 'NADH-cytochrome b5 reductase [enzyme]', 'cytochrome b<sub>5</sub> [enzyme]', 'b5R class [enzyme]'], ['yeast b5R [enzyme]'], ['NADH-cytochrome b5 reductase [metabolite]'], ['human NADH cytochrome b5 reductase [gene]'], ['NADH-Cyt b5 reductase [protein]'], ['NADH-Cyt b5 reductases of yeast and mammals [protein]'], ['reducing partners (CYRs) [enzyme]'], ['membrane-bound cytochrome b5 enzymes [enzyme]'], ['cytosolic short-chain dehydrogenase/reductase [protein]'], ['CBR-mediated electron transfer chains [reaction]'], ['CPR-mediated electron transfer chains [reaction]'], ['partial reconstruction of an archaeal F420-reducing NiFe hydrogenase [enzyme]']]}\n",
      "-------------------\n",
      "{'151814': [['orf79 [gene]'], ['SLG1indica [gene]'], ['rice ssg4 locus [gene]'], ['SMG4 locus [gene]'], ['fertility restorer (RF; Os10t0497366) [gene]'], ['SHI/STY/SRS gene(s) Lks2 [gene]'], ['**asp-lsl [gene]**', 'ASP-LSL locus [gene]'], ['**tms5 [gene]**', 'locus tms5 [gene]', 'tms5 locus [gene]'], ['tms5 (ZH11) [gene]'], ['**P/TGMS gene(s) in rice [gene]**', 'P/TGMS genetic loci in rice [gene]'], ['MEL2-like locus [gene]'], ['DMC1s [gene]'], ['OsMSH4/5 [gene]'], ['DPL2 [gene]'], ['SA locus [gene]'], ['S24 [gene]'], ['hsa1 [gene]'], ['**rice gamete eliminator S5 [gene]**', 'S5 locus [gene]', 'wide compatibility gene(s) S5 [gene]'], ['Rfk1 of Kosena radish [gene]'], ['**Long sterile lemma (G1) [gene]**', 'LONG STERILE LEMMA1 (G1) [gene]'], ['SHATTERINGABORTION1 [gene]'], ['Suppression of anther-specific single-copy gene(s) [gene]'], ['SSC [gene]'], ['maize ortholog of rice GS5 [gene]'], ['S3574 gene(s) [gene]'], ['identification of GMS gene(s) in rice [gene]'], ['causal gene(s) of LC5 [gene]'], ['BoroII rice orf79 [gene]'], ['SGS3 of rice [gene]'], ['TGMS locus [gene]'], ['MEL2 locus [gene]'], ['glabrous rice 1 locus [gene]'], ['gene(s) that suppress fon2 mutation [gene]'], ['sl gene(s) from slender glume mutant of rice [gene]'], ['TS6-ref [gene]'], ['one SAG [gene]'], ['global characterization of NBS-LRR gene(s) [gene]']]}\n",
      "-------------------\n",
      "{'166903': [['**porphobilinogen [metabolite]**', 'PBG [metabolite]', 'monopyrrole PBG [metabolite]', 'porphobilinogen (PBG) [metabolite]', 'monopyrrole porphobilinogen (PBG) [metabolite]'], ['planar porphyrins [metabolite]']]}\n",
      "-------------------\n",
      "{'9347': [['distribution plots [phenotype]'], ['**allelic frequency [phenotype]**', 'overall allele frequencies [phenotype]'], ['frequency spectrum [phenotype]'], ['blunt end distribution [phenotype]'], ['stationary distribution of attractor probabilities [phenotype]'], ['present Tubingen frequency [phenotype]'], ['frequency distributions [phenotype]'], ['frequency distribution more uniform [phenotype]'], ['**Total frequency [phenotype]**', 'frequency summation [phenotype]']]}\n",
      "-------------------\n",
      "{'45093': [['plant(s) lacking functional Galpha [mutant]'], ['plant(s) lacking functional Ggamma1Ggamma2 [mutant]'], ['plant(s) lacking functional SAV3 [mutant]'], ['plant(s) with loss of COL5 function [mutant]'], ['plant(s) lacking three, four, five, and six PYR/PYLs [mutant]'], ['plant(s) lacking a functional ARP2/3 complex [mutant]'], ['plant(s) deficient in the production of small RNAs [mutant]', '**plant(s) deficient in the biogenesis of small RNAs [mutant]**'], ['plant(s) lacking functional CO protein(s) [mutant]'], ['plant(s) lacking subunits of RNA Polymerase IV [mutant]'], ['plant(s) lacking AGO4 [mutant]'], ['plant(s) lacking AGO7 [mutant]'], ['plant(s) lacking AGO5 [mutant]'], ['plant(s) lacking AGO8 [mutant]'], ['plant(s) lacking SDE5 [mutant]'], ['plant(s) lacking complete AG activity [mutant]']]}\n",
      "-------------------\n",
      "{'52957': [['HDG3/HDG2 orthologs [gene]'], ['FWA/HDG7 orthologs [gene]'], ['C2C2-Dof gene(s) orthologs [gene]'], ['C2C2-CO-like gene(s) orthologs [gene]'], ['C3H gene(s) orthologs [gene]'], ['no prevalent expression of HK4 orthologs [gene]'], ['AtFANCC orthologs [gene]'], ['P1 orthologues [gene]'], ['CLE17 orthologous gene(s) group [gene group]'], ['CLE19 orthologous gene(s) group [gene group]'], ['TDIF orthologous gene(s) [gene]'], ['orthologues of AtCLV3 [gene]', '**AtCLV3 orthologues [gene]**'], ['J2 orthologues [gene]'], ['TDF1 orthologs [gene]'], ['Class D orthologs [gene]'], ['GmERD15 orthologs [gene]'], ['GmIAA27 orthologs [gene]'], ['DUR3-orthologs [gene]'], ['crr4 gene(s) orthologues [gene]'], ['SEP1/2/3 orthologs [gene]'], ['Xrn1p orthologs [gene]'], ['orthologs to THH1 [gene]'], ['other UNC-93 orthologs [gene]'], ['CBF ortholog [gene]'], ['IAA16 orthology group [gene]']]}\n",
      "-------------------\n",
      "{'46416': [['fewer root(s) (fwr) [mutant]', 'fwr mutation [mutant]', '**fwr mutant [mutant]**', 'FWR [mutant]', 'fewer root(s) (fwr) mutation [mutant]'], ['reduced lateral root(s) formation in gpa1 mutants [mutant]', 'new formation of root(s) in mutants severely compromised in LR formation [mutant]', 'LR-less mutants [mutant]', '**mutants incapable of lateral root(s) initiation [mutant]**'], ['lrt1 primary root(s) [organ]'], ['**absent lateral root(s) cap [organ]**', 'no lateral root(s) cap formation in late stages [organ]'], ['lateral root(s) in mutant rum1 [organ]'], ['**lars mutants [mutant]**', 'less adventitious root(s) (lars) mutants [mutant]'], ['fewer root(s) suppressor1 (fsp1) [mutant]']]}\n",
      "-------------------\n",
      "{'19214': [['Zn excess soil [phenotype]', '**Zn-polluted soils [phenotype]**'], ['**Zn toxicity symptoms [phenotype]**', 'Zn toxicity phenotypes [phenotype]', 'plant(s) Zn toxicity [phenotype]'], ['Zn detoxifier [phenotype]', '**Zn detoxification [phenotype]**', 'detoxification of zinc [phenotype]'], ['Zn- and Cu-polluted soils [phenotype]'], ['zinc/cadmium/lead-rich soils [phenotype]']]}\n",
      "-------------------\n",
      "{'114365': [['CBF3 phase and waveform [phenotype]'], ['3 s every 3 min [phenotype]'], ['3 h of darkness [time]'], ['three hours of each other [phenotype]']]}\n",
      "-------------------\n",
      "{'130281': [['A. thaliana PRORP2 [gene]'], ['ten putative A. thaliana nucleotidyl transferases [gene]'], ['A. thaliana t-unit pre-tRNA gene(s) array [gene]'], ['A. thaliana pre-tRNA(Tyr) [gene]'], ['A. thaliana nuclear tRNAHis gene(s) [gene]'], ['A. thaliana pre-tRNAs [gene]'], ['A. thaliana gene(s) involved in pre-mRNA splicing [gene]'], ['A. thaliana nuclear pre-tRNAPhe [gene]'], ['A. thaliana mitochondrial pre-tRNACys [rna]'], ['A. thaliana functional homolog of TRM7 gene(s) [gene]'], ['A. thaliana nuclear tRNAs [gene]']]}\n",
      "-------------------\n",
      "{'168492': [['Significant increase in protein-bound Met [metabolite]'], ['high protein(s) [metabolite]'], ['higher free Cys [metabolite]'], ['**higher Met [metabolite]**', 'higher Met content [metabolite]', 'increase in Met content [metabolite]'], ['higher Cys content [metabolite]']]}\n",
      "-------------------\n",
      "{'123167': [['multiple ESTs/mRNAs [metabolite]'], ['embryo proper mRNA populations [metabolite]'], ['mRNA libraries [metabolite]'], ['mRNA samples [metabolite]'], ['**mRNA microarray data [metabolite]**', 'microarray profiling data of mRNAs [metabolite]'], ['mRNA data [metabolite]'], ['reduced levels of transgene mRNA [metabolite]'], ['mRNA populations [metabolite]'], ['large polysome mRNA levels [metabolite]'], ['comparison of two mRNA populations [metabolite]'], ['Small sets of transcripts [metabolite]'], ['single species of mRNA [metabolite]'], ['mRNA fingerprints [metabolite]'], ['labeled mRNA [metabolite]'], ['unique human mRNA sequences [metabolite]'], ['Cleaved mRNA fragments [metabolite]'], ['intact transgene mRNA [metabolite]'], ['dicistronic messages [metabolite]'], ['mRNA fragment [metabolite]'], ['tagging multiple mRNA species [metabolite]'], ['broad range of mRNAs [metabolite]'], ['CB mRNAs [metabolite]'], ['mRNA makeup [metabolite]'], ['small portion of the mRNAs [metabolite]'], ['thousands of mRNAs [metabolite]'], ['available mRNA [metabolite]'], ['highly expressed transcripts [metabolite]'], ['profiling of mRNAs [metabolite]'], ['equal amounts of transcripts [metabolite]'], ['mRNA fragment size [metabolite]'], ['sequenced mRNA [metabolite]'], ['more than 12,000 stored mRNA species [metabolite]'], ['specific subset of mRNA [metabolite]'], ['changes in abundance of mRNA species [metabolite]']]}\n",
      "-------------------\n",
      "{'20160': [['Transformation of crop plant(s) [phenotype]'], ['**Genetic increase in crop resistance [phenotype]**', 'crop insect-resistant genetic engineering [phenotype]', 'development of transgenic crops with resistance to pests and pathogens [phenotype]', 'higher crop safety [phenotype]'], ['genetic enhancement of crop allelopathy [phenomenon]'], ['screening of herbicide-resistant accessions [phenotype]', '**genetic manipulation of herbicide resistance [phenotype]**', 'increasing herbicide tolerance [phenotype]'], ['**commercial glyphosate-resistant crops [phenotype]**', 'target herbicide for engineering herbicide-resistant crops [phenotype]', 'breeding of glyphosate resistant crops [phenotype]']]}\n",
      "-------------------\n",
      "{'60479': [['eight defense-related modules [phenotype]'], ['Four functional domains [phenotype]'], ['Four recent studies [phenotype]'], ['four sequential phases [phenotype]'], ['Four compounds [phenotype]'], ['six groupings [phenotype]'], ['seven protein(s) domains [phenotype]'], ['5 gene(s) [phenotype]'], ['four types [phenotype]'], ['approximately 500 species [phenotype]'], ['correlation with angle of the fourth branch [phenotype]'], ['four major classes [phenotype]'], ['six branches [phenotype]'], ['five domains [phenotype]'], ['disease categories 5, 4 [phenotype]'], [\"novel 5'SS [phenotype]\"], ['5 classes of families [phenotype]'], ['five types of colorful samples [phenotype]'], ['several GO categories [phenotype]'], ['five pathways [phenotype]'], ['all 10 traits [phenotype]'], ['nine clans [phenotype]'], ['fifteen gene(s) [phenotype]'], ['2, 4, and 5 [phenotype]'], ['at least five accessions contributing to expression variation [phenotype]'], ['fourth in China [phenotype]'], ['several distinct types [phenotype]'], ['first four issues [phenotype]'], ['eight functional categories [phenotype]'], ['nine branches [phenotype]'], ['nine lines with low or high NPMI [phenotype]'], ['broad spectrum of functional categories [phenotype]'], ['four key domains [phenotype]'], ['all four key domains [phenotype]'], ['multiple AprE based on four key domains [phenotype]'], ['categorized into 6 super-clusters [phenotype]'], ['core, softcore, shell, and cloud categories [phenotype]'], ['other categories [phenotype]'], ['six major clusters [phenotype]'], ['twelve gene(s) [phenotype]'], ['six chemical families [phenotype]'], ['59 different tissues or cell types [phenotype]'], ['20 gene(s) in GO:0012502 category [phenotype]'], ['6 gene(s) in GO:0012502 category [phenotype]'], ['2 gene(s) in GO:0012502 category [phenotype]'], ['12 gene(s) in GO:0012502 category [phenotype]']]}\n",
      "-------------------\n",
      "{'109937': [['sites binding to transcription factors [genomic datum]', '**transcription factor binding regions [genomic datum]**'], ['HD-binding cis-elements [subcellular compartment]']]}\n",
      "-------------------\n",
      "{'41256': [['CSE-deficient A. thaliana [mutant]', '**A. thaliana cse mutants [mutant]**'], ['4CL1-deficient A. thaliana [mutant]'], ['A. thaliana alx8 mutant [mutant]'], ['Cell adhesion-defective A. thaliana mutants [mutant]'], ['A. thaliana cer3 mutant [mutant]'], ['A. thaliana laccase mutants (atlacs) [mutant]', '**A. thaliana atlac mutants [mutant]**'], ['A. thaliana ref4 mutants [mutant]'], ['lignin-deficient A. thaliana mutants [mutant]', '**A. thaliana lignin mutants [mutant]**', 'A. thaliana lignin biosynthesis mutants [mutant]'], ['lipid-related A. thaliana mutants [mutant]', '**A. thaliana lipid mutants [mutant]**'], ['A. thaliana anisotropy1 mutant [mutant]'], ['A. thaliana cer8 mutant [mutant]'], ['A. thaliana lacs9-1 mutant [mutant]'], ['A. thaliana loss-of-function CSE-2 mutant [mutant]'], ['A. thaliana cts-1 lines [mutant]'], ['A. thaliana kat2 mutants [mutant]'], ['A. thaliana cad2-1 mutants [mutant]'], ['A. thaliana comt mutants [mutant]'], ['A. thaliana tra2 mutant [mutant]'], ['A. thaliana eceriferum (cer) mutants [mutant]'], ['A. thaliana LCB Delta4-desaturase mutant [mutant]'], ['A. thaliana cer mutants (e.g., cer1, cer3, cer6) [mutant]'], ['A. thaliana csld mutants [mutant]'], ['A. thaliana CesA8 mutants [mutant]'], ['A. thaliana suberin biosynthesis mutant seeds [mutant]'], ['A. thaliana myb46 mutant [mutant]'], ['vtc2 A. thaliana [mutant]'], ['A. thaliana cer9 mutant [mutant]'], ['A. thaliana AtCesA7 mutant [mutant]'], ['A. thaliana aact1 mutants [mutant]'], ['A. thaliana mucilage mutants [mutant]'], ['A. thaliana QsuB plant(s) [mutant]'], ['A. thaliana SALK mutant 063792 lignin (ASL) [metabolite]'], ['A. thaliana mutant 063792 [mutant]'], ['A. thaliana carbon metabolism mutant [mutant]'], ['A. thaliana mur8-1 mutants [mutant]'], ['A. thaliana Reduced Wall Acetylation2 mutant [mutant]'], ['A. thaliana pectin methylesterase 3 mutants [mutant]'], ['A. thaliana T-DNA insertion mutants of REDUCED WALL ACETYLATION 2 (rwa2) [mutant]', '**A. thaliana rwa2 [mutant]**'], ['A. thaliana lignin mutant ref8 [mutant]'], ['A. thaliana fatty acid biosynthesis 1-1 fatty acid desaturase 2 fatty acid elongase 1 mutant [mutant]'], ['**A. thaliana cer1 mutant [mutant]**', 'A. thaliana eceriferum1 (cer1) mutants [mutant]'], ['A. thaliana procuste mutant [mutant]'], ['A. thaliana cer17 mutant [mutant]'], ['**A. thaliana cer4 mutant [mutant]**', 'A. thaliana cer4 mutants [mutant]'], ['A. thaliana geranylgeranyl reductase1 mutants [mutant]'], ['A. thaliana ncer mutants [mutant]'], ['A. thaliana ap2-7 mutant [mutant]'], ['A. thaliana cer7 mutant [mutant]'], ['A. thaliana kcs9 mutant [mutant]'], ['A. thaliana callose synthase mutants [mutant]'], ['A. thaliana lac17 mutant [mutant]'], ['A. thaliana icx1 mutant [mutant]'], ['fah1 mutant of A. thaliana [mutant]'], ['A. thaliana cellulose synthase (CesA) mutants [mutant]', '**A. thaliana cesa mutants [mutant]**'], ['A. thaliana cyt1 mutants [mutant]'], ['A. thaliana Wax Mutant leaf [mutant]'], ['A. thaliana 1a3b Rubisco mutant [mutant]'], ['A. thaliana fatty acid elongase1 mutant [mutant]'], ['A. thaliana fla16 mutant [mutant]'], ['ccr1 mutant A. thaliana [mutant]'], ['**A. thaliana cer2 mutant [mutant]**', 'A. thaliana cer2 mutation [mutant]'], ['A. thaliana lac4-2 lac17 mutant [mutant]'], ['A. thaliana lac4 mutant [mutant]'], ['A. thaliana cer mutants [mutant]'], ['A. thaliana pmr4-1 [mutant]'], ['A. thaliana vtc-5-1 and vtc-5-2 mutants [mutant]'], ['inability of A. thaliana fah1-2 mutant [mutant]'], ['A. thaliana cadc cadd [mutant]']]}\n",
      "-------------------\n",
      "{'44975': [['**IBA-response mutants [mutant]**', 'iba response mutants [mutant]'], ['IBA-response defects in acx mutants [mutant]'], ['IBA RESPONSE5 (IBR5) [mutant]'], ['IBA-resistant, IAA-sensitive mutants [mutant]'], ['IBR1 [mutant]'], ['IBA-response mutant (ibr3) [mutant]'], ['IBA responsiveness in pex5-2 [mutant]'], ['IBA responsiveness in pex6-1 root(s) [mutant]'], ['cad2 hyposensitivity to IBA [treatment]']]}\n",
      "-------------------\n",
      "{'89781': [['regulation of carbon allocation [process]'], ['understanding the regulation of photosynthesis [process]', '**photosynthesis regulation [process]**', 'regulating photosynthesis [process]'], ['regulating photosynthetic capacity [process]'], ['regulation of photosynthesis efficiency [process]'], ['regulation of chlorophyll accumulation [process]']]}\n",
      "-------------------\n",
      "{'86746': [['X-ray absorption [description]'], ['low X-ray absorptions [description]'], ['partial absorption of X-Rays [description]'], ['X-ray absorption of animal and plant(s) tissues [treatment]']]}\n",
      "-------------------\n",
      "{'1327': [['anthesis time of mutant hermaphrodite flower [phenotype]'], ['time required for flowers to reach anthesis [phenotype]'], ['anthesis time of male flower [phenotype]', '**male flowering time (MFLW) [phenotype]**'], ['female flowering time (FFLW) [phenotype]'], ['9 to 16 months [phenotype]']]}\n",
      "-------------------\n",
      "{'26380': [['**decrease in maximum quantum efficiency of PSII [phenotype]**', 'decrease in potential efficiency of PSII [phenotype]', 'reduction of maximum quantum efficiency of PSII [phenotype]', 'reduced maximum quantum yield of photosystem 2 [phenotype]', 'lower value of maximum quantum efficiency of PSII [phenotype]', 'decrease in PSII maximum photochemical efficiency [phenotype]'], ['reducing PSII efficiency [phenotype]'], ['reduction of effective and maximum quantum yield of photosystem II [phenotype]']]}\n",
      "-------------------\n",
      "{'107206': [['**peak calling [process]**', 'detection of enriched regions [process]']]}\n",
      "-------------------\n",
      "{'5119': [['long-term responses [process]'], ['**Response to stresses [process]**', 'reaction to stress conditions [process]', 'response to adversity [process]', 'response to environmental stress conditions [process]'], ['primary stress responses [process]'], [\"plants' reaction to unfavorable conditions [process]\"], ['Responses to environmental stresses [condition]'], ['Stress response [metabolic process]'], ['Stress response [description]']]}\n",
      "-------------------\n",
      "{'81739': [['sticky protein(s) [protein]'], ['Sieve element protein(s) [protein]'], ['Rhomboid-like protein(s) [protein]'], ['cystatin-like protein(s) [protein]'], ['PFDs [protein]'], ['Collagen-related structural motif (CSM) patterns [metabolite]', '**CSMs [metabolite]**'], ['AAA+ protein(s) [protein]'], ['spectrin-like protein(s) [protein]'], ['protein(s) with SLiMs [protein]'], ['Functional human PDIs [protein]'], ['MS2 coating protein(s) [protein]'], ['protein(s) disulfide isomerase-like protein(s) [protein]'], ['isoprenylated protein(s) [protein]'], ['Specific LysM-type protein(s) [protein]'], ['4/1-like protein(s) [protein]'], ['Oscillatoria protein(s) [protein]'], ['prion-like protein(s) [protein]'], ['soap [protein]'], ['SIMIBI protein(s) [protein clas]'], ['starch protein(s) [protein]'], ['ciliary profile protein(s) [protein]'], ['SUMOylated SUMO [protein]'], ['novel protein(s) scaffolds [protein]'], ['new synaptic protein(s) [protein]'], ['Disulfide-bonded protein(s) [protein]'], ['Rapidly evolving surface protein(s) [protein]'], ['Polyisoprenylated protein(s) [protein]'], ['level of auxiliary protein(s) such as Hlips [protein]'], ['ancillary factors [protein family]'], ['protein(s) with OCD activity [protein]'], ['housekeeping preproteins [protein]'], ['multivalent LLPS-driving scaffold molecules [protein]'], ['collagen-like protein(s) [protein]'], ['CSM-containing protein(s) [protein]'], ['fibrinogen-like clotting factors [function]'], ['specific client protein(s) [protein]'], ['amide II protein(s) [protein]'], ['protein(s) involved in production of beta-mannans [protein]'], ['actin-like protein(s) [protein]'], ['WD40-like protein(s) [protein]'], ['iron containing protein(s) [protein]'], ['other IDPs [protein]'], ['deltaproteobacterial protein(s) [protein]'], ['chaperone-bound but refoldable protein(s) [protein]'], ['a large number of auxiliary protein(s) [protein]'], ['additional wall protein(s) [protein]'], ['gene(s) for scaffold protein(s) [protein]'], ['nascent client protein(s) [protein]'], ['diverse client protein(s) [protein]'], ['other LLPS-responsive protein(s) [protein]'], ['core set of ciliary function protein(s) [protein]'], ['proline based protein(s) [function]'], ['habitat-specific adhesins [protein]'], ['putative scaffold protein(s) [protein]'], ['disulfide protein(s) [protein]'], ['those linked by the PsMTa linker [protein]'], ['protein(s) necessary for development of phototropic curvatures [protein]'], ['non-membranous protein(s) [protein]'], ['CWP structure [protein]']]}\n",
      "-------------------\n",
      "{'77546': [['an N-terminal transit peptide-like sequence [protein domain]', '**N-terminal transit sequences [protein domain]**']]}\n",
      "-------------------\n",
      "{'147190': [[\"'stress related' gene(s) [gene]\", 'stress-associated gene(s) [gene]', '**stress related gene(s) [gene]**'], ['hormones involved in stress [gene]'], ['**Expression of plant(s) stress-response gene(s) [gene]**', 'Expression of response to stress related gene(s) [gene]', 'stress response-related gene(s) expression [gene]', 'stress-related gene(s) expressions [gene]'], ['Signalling-related gene(s) [gene]'], ['stress-related gene(s) paralogues [gene]'], ['SSTF gene(s) [gene]'], ['**heat-stress-related gene(s) [gene]**', 'heat stress related gene(s) [gene]'], ['stress-related basic secretory protein(s) [gene]'], ['stress-related glycoproteins [protein]'], ['stress response gene(s) GSTF14 [gene]'], ['**plant(s) Core Environmental Stress Response gene(s) [gene]**', 'plant(s) core environmental stress response (PCESR) gene(s) [gene]', 'core environmental stress response gene(s) [gene]'], ['RHS gene(s) [gene]'], ['stress response factors [gene]'], ['plant(s) Stress gene(s) Database [database]'], ['stress effective gene(s) [gene]'], ['**Stress-related DEGs [gene]**', 'stress response-associated DEGs [gene]'], ['seagrass gene(s) [gene]'], ['stress response gene(s) in plant(s) [gene]'], ['stress sensors [gene]'], ['developmental priming gene(s) [gene]'], ['Stress associated protein(s) (SAP) gene(s) [gene]'], ['Se-responsive gene(s) [gene]'], ['functional relations with other gene(s) [gene]'], ['putative stress-responsive gene(s) [gene]'], ['GSR gene(s) [gene]'], ['STREs [gene]'], ['stress-related metabolic gene(s) [gene]'], ['stress signalling gene(s) [gene]'], ['Group II gene(s) associated with stress responses [gene]'], ['stress mechanism gene(s) [gene]'], ['increased protein(s) involved in stress responses [gene]'], ['key stress response gene(s) [gene]'], ['**stress response-related gene(s) [gene]**', 'stress responses-related gene(s) [gene]'], ['stress responses in shoot-specific DE gene(s) [tissue]'], ['stress-related protein(s) gene(s) [gene]'], ['stress pathways gene(s) [gene]', 'stress pathway gene(s) [gene]', '**stress-related pathways [gene]**'], ['stress gene(s) in A. thaliana [process]'], ['stress gene(s) in A. thaliana root(s) [gene]'], ['stress gene(s) in A. thaliana shoots [gene]'], ['Stress response gene(s) [gene family]'], ['Stress-related gene(s) [process]'], ['stress-associated protein(s) 7 [gene]'], ['stress-sensing gene(s) [gene]'], ['stress involved gene(s) [gene]'], ['stress-related gene(s) in Solanum pennellii [gene]'], ['Stress-related gene(s) [gene group]'], ['stress-specific gene(s) in A. thaliana [gene]'], ['stress responses [gene]'], ['plant(s) growth/development/stress responses related gene(s) [gene]'], ['protein-coding gene(s) involved in stress response [gene]'], ['transcription factor gene(s) related to stress response [gene]'], ['stress response functions [gene]'], ['stress- and pathogenesis-related gene(s) [gene]'], ['UGTs involved in stress-inducible responses [gene]']]}\n",
      "-------------------\n",
      "{'172310': [['Carbon interaction [process]'], ['carbohydrate flux [process]'], ['**Carbon flow [process]**', 'Carbon flux [process]'], ['carbon flow in proline [metabolite]'], ['carbon flow in serine [metabolite]'], ['carbon flow in tyrosine [metabolite]'], ['carbon flow analysis [treatment]'], ['**Carbon flux [metabolite]**', 'Carbon Flow [metabolite]', 'carbon (C) flow [metabolite]'], ['phosphorus/carbon exchange [process]'], ['**Carbon transition [process]**', 'Carbon transitions [process]'], ['One-carbon flow [process]'], ['carbon mobilization [process]'], ['carbon exchange [process]'], ['**CO<sub>2</sub> fluxes [process]**', 'carbon dioxide transfer [process]'], ['flux of carbon through lactate [metabolic pathway]'], ['Carbon flow [metabolic process]'], ['carbon metabolic flow [process]'], ['carbon flow into sugars via gluconeogenesis and respiration [metabolite]'], ['glucose-to-acetate carbon source transition [process]'], ['carbon mobilization [pathway]'], ['transfer of mitochondrial CO2 [process]'], ['carbon flow from primary metabolism to secondary metabolism [process]'], ['carbon flux through PPP [process]'], ['direction of carbon flow [process]'], ['entry of carbon from sucrose [process]'], ['carbon flow into sugars via gluconeogenesis and respiration [metabolic process]'], ['carbon transfer [process]'], ['inorganic carbon flow [process]'], ['carbon flow in the cell wall synthesis network [process]']]}\n",
      "-------------------\n",
      "{'147191': [['contigs related to response to stress [gene]'], [\"probe-sets related to 'response to stress' [gene]\"]]}\n",
      "-------------------\n",
      "{'170441': [['Mn cofactor specificity [phenotype]'], ['positive potential [phenotype]'], ['substrate binding to AOS [phenotype]'], ['preferred substrate [phenotype]'], ['(S)-preference [phenotype]'], ['(R)-preference [phenotype]'], ['**specificity for sterol substrate [phenotype]**', 'overlapping sterol substrate preferences [phenotype]'], ['interaction with the substrate in the catalytic site [phenotype]'], ['**chain-length and oxidation state preferences [phenotype]**', 'detailed understanding of chain length and oxidation state preferences [phenotype]'], ['plane of -1 sugar [phenotype]'], ['intrinsic preference for di-uridylation activity [phenotype]'], ['substrate preference for sterol precursors [phenotype]'], ['preferential specificity for acyl-CoAs [phenotype]'], ['hydroxylation abilities [phenotype]'], ['tail-first substrate binding [phenotype]'], ['preference for trans-activity [phenotype]'], ['identical substrate dependency [phenotype]'], ['excellent enantioselectivity [phenotype]'], ['enzymatic stability [phenotype]']]}\n",
      "-------------------\n",
      "{'169252': [['changes in foliar ABA level [metabolite]'], ['variation tendency of malic acid [metabolite]'], ['variation tendency of glycolic acid [metabolite]'], ['Changes in phytohormone profiles [metabolite]', '**Phytohormone signatures [metabolite]**'], ['pH changes in AZ cells [metabolite]'], ['changes in phytohormone levels [metabolite]'], ['Changes in hormone levels [metabolite]'], ['variation in temperature-dependent total SA [metabolite]'], ['Differences in pool sizes of ABA and catabolites [metabolite]', 'variability in ABA concentrations [metabolite]', '**fluctuating ABA levels [metabolite]**', 'differential ABA accumulation [metabolite]'], ['variation in utilization of malic acid [metabolite]'], ['changes in stress hormones [metabolite]'], ['changes in oxylipin levels [metabolite]'], ['deviations in metabolite profiles in shoots [organ]'], ['significant changes in jasmonate levels during light/dark transition [metabolite]'], ['significant changes in jasmonate levels throughout daily light period [metabolite]'], ['cold regulation of flavonoids [metabolite]'], ['climate/elevation cline in SA [metabolite]'], ['remodelling of metabolome in blackcurrant [organism]'], ['differences in PA levels [metabolite]'], ['different MgATP2- concentrations [metabolite]'], ['importance of metabolite composition under non-stress conditions [metabolite]'], ['**different types and amounts of metabolites under salt stress [metabolite]**', 'different types and abundance of metabolites under salt stress [metabolite]'], ['different types and amounts of metabolites under drought [metabolite]'], ['total SA change [metabolite]'], ['different MDA content under salt stress [metabolite]'], ['large variety of compounds with wide range of chemical diversity and content [metabolite]'], ['changes in foliar H2O2 content [metabolite]'], ['global changes in metabolite content [metabolite]'], ['metabolomic level mechanisms [phenotype]'], ['changes with PL [metabolite]'], ['isolation of minor key plant(s) biomarkers [metabolite]']]}\n",
      "-------------------\n",
      "{'123050': [[\"polyadenylated RNA 3' ends [genomic region]\", '**RNA poly(A) tails [genomic region]**', 'polyadenylated [genomic region]'], ['**poly(A) tails [region]**', 'polyadenylated [region]'], ['poly(A)-tailed 5.8S pre-rRNAs [gene]'], ['SOC1 mRNA poly(A) tail lengths [rna]'], ['poly-A RNA [treatment]'], ['**proximally polyadenylated COOLAIR [rna]**', 'proximally polyadenylated COOLAIR transcript [rna]'], ['proximally polyadenylated class I COOLAIR [gene]'], ['Polyadenylated RNA [genomic sequence]'], ['poly(A)-ending (pa) SINE RNAs [rna]'], ['specific polyadenylation signals [gene]'], ['tNos polyadenylation sites [gene]'], ['tRbcS polyadenylation sites [gene]'], ['a specific subclass of polyadenylated viral RNAs with A-rich tails [region]'], ['polyadenylated phytoviral RNAs [region]'], ['proximally polyadenylated COOLAIR transcript [transcript]'], ['polyA tails [subcellular compartment]'], [\"3' poly(A) tails [genetic element]\"], ['new polyadenylation site [gene]'], ['polyadenylated pre-tRNA [genomic feature]'], ['two polyadenylation sites [gene]'], ['main population [rna]'], ['poly(A)-ending species [rna]']]}\n",
      "-------------------\n",
      "{'9338': [['14°C [phenotype]'], ['Mean oscillation frequency [phenotype]'], ['41°C [phenotype]'], ['39°C [phenotype]'], ['0.86 microm/sec [phenotype]'], ['0.94 microm/sec [phenotype]'], ['0.91 microm/s [phenotype]'], ['1.43 microm/s [phenotype]'], ['8.6 to 10.5 [phenotype]'], ['30 to 40 degree slope [phenotype]'], ['~35 C [phenotype]'], ['30 mM step [phenotype]'], ['peak at 35 microm-45 microm [phenotype]'], ['peak at 40 microm-50 microm [phenotype]'], ['1.2 [phenotype]'], ['~31 s [phenotype]'], ['N1 = 4 - 28 [phenotype]'], ['mean angle of 47 +- 26 [phenotype]'], ['2.71-times control levels [phenotype]'], ['at 34 C [phenotype]'], ['3.29 at 17 C [phenotype]'], ['+-1 C of target temperature [phenotype]'], ['1 C of target temperature [phenotype]'], ['floating layer on top of 100 000 g supernatant [phenotype]']]}\n",
      "-------------------\n",
      "{'145919': [['Defensin-like (AT1G13609) [gene]'], ['**Capsicum annum defensin CADEF1 [gene]**', 'Defense-related CaDEF1 (defensin) gene(s) expression [gene]'], ['DEFL0770 group [gene family]'], ['DEFL0770 gene(s) [gene]'], ['NaDLP1 [gene]'], ['DEFL [gene]'], ['Defensin-like protein(s) 13 [gene]'], ['At5g05410 [gene]'], ['Def1 [gene]'], ['Def2 [gene]'], ['DEFL206 [gene]'], ['DEFL208 [gene]'], ['DEFL202 [gene]'], ['DEFL207 [gene]'], ['At DEFL203 [gene]'], ['AhDEF1s [gene]'], ['NbDef1.5 [gene]'], ['NbDef2.1 [gene]'], ['NbDef2.2 [gene]'], ['SD-1 receptor kinase and defensin-like gene(s) family members [gene]'], ['DEFENSIN-LIKE1 [gene]']]}\n",
      "-------------------\n",
      "{'4926': [['Interplay between ontogeny and abiotic stresses [phenotype]'], ['growth, development and several biotic or abiotic stress-related responses in plant(s) [phenotype]'], [\"plant's response to biotic and abiotic stress [phenotype]\"], ['**adaptation to biotic and abiotic stress [phenotype]**', 'adaptation to biotic or abiotic stress [phenotype]'], ['abiotic stress responses during seed germination [phenotype]'], ['abiotic stress responses during seedling development [phenotype]'], ['development, gravitropism, pathogen responses, and abiotic stress responses [process]'], ['growth, development, and abiotic stress responses in various plant(s) [phenotype]'], ['induction of responses to abiotic and biotic stresses [phenotype]'], ['different tissues under different abiotic stresses [phenotype]'], ['early seedling responses to biotic and abiotic stresses [phenotype]'], ['cell division and abiotic stress responses [phenotype]'], ['trade-offs between development, abiotic stress, and biotic stress responses [phenotype]'], ['various developmental and biotic and abiotic stress response-related pathways [phenotype]'], ['response to biotic and abiotic stresses in rosettes [phenotype]'], ['response to biotic and abiotic stresses in root(s) [phenotype]']]}\n",
      "-------------------\n",
      "{'75945': [['EBD [protein domain]'], ['ES domain [protein domain]'], ['Dt [protein domain]'], ['A. thaliana EDS1 alpha-helical EP-domain [protein]'], ['**ETR1 receiver domain [protein domain]**', \"ethylene receptor's receiver domain [protein domain]\", 'Receiver domains of ethylene receptors [protein domain]'], ['ETT-specific domain [protein]'], ['Dimeric ethylene response sensor 1 DHp domain [protein domain]'], ['C-terminal EP domain (EPD) [subcellular compartment]'], ['AUX/IXX family domain [protein domain]'], ['AUX/IXX family domain [protein]'], ['destruction box, DII [protein domain]', '**DII (Degron) domain [protein domain]**', 'functional DII domain [protein domain]'], ['auxin_inducible domain [protein domain]'], ['fluorescently tagged auxin-interaction domain [protein]'], ['InterPro auxin carrier domain [protein domain]'], ['domain for ethylene binding and signaling [phenotype]'], ['auxin response super family domain [subcellular compartment]'], ['signal input domain [protein domain]'], ['domain I of IAA17 [protein domain]'], ['Aux/IAA auxin-interaction domain [protein domain]']]}\n",
      "-------------------\n",
      "{'22536': [['flavonol branch-specific activators [phenotype]'], ['structural specifiers [phenotype]']]}\n",
      "-------------------\n",
      "{'121491': [['GPCR classifications [phenotype]'], ['group pair comparisons [phenotype]'], ['extended features [phenotype]'], ['redundancy among JAZs [phenotype]'], ['classes B and C [phenotype]'], ['small, independent interacting groups [phenotype]'], ['most dissimilar group [phenotype]'], ['small isolated groups [phenotype]'], ['first PERK member to be characterized [phenotype]'], ['up to four imperfect repeats [phenotype]'], ['MDHC-I [phenotype]'], ['subtype B [phenotype]'], ['g [phenotype]'], ['high-velocity group [phenotype]'], ['10 types [phenotype]'], ['most dynamic group [phenotype]'], ['eight canonical or modified types [phenotype]'], ['small sizes in ancestors [phenotype]'], ['not clear [phenotype]']]}\n",
      "-------------------\n",
      "{'131563': [['Cation/Chloride Cotransporters (CCCs) [gene]', 'cation chloride co-transporter (CCC) [gene]', '**CCCs [gene]**', 'CCC gene(s) [gene]', 'plant(s) CCC protein(s) [gene]', 'plant(s) cation-chloride cotransporters (CCCs) [gene]', 'gene(s) encoding cation-chloride cotransporters (CCC) [gene]'], ['9 CLCs [gene]'], ['MdCLC-a/b [gene]'], ['**CLC [gene]**', 'CLC family gene(s) [gene]', 'CHLORIDE CHANNEL (CLC) [gene]', 'chloride channels [gene]', 'CLC homologues [gene]'], ['Cl- channel/transporter CLCe [gene]'], ['Clath [gene]'], ['bacterial CLCs [gene]'], ['ChR2 gate [gene]'], ['CHLORIDE CHANNEL B [gene]'], ['Cl- transporters [gene]', '**chloride transporters [gene]**', 'CHLORIDE CHANNEL FAMILY (CLC) transporters [gene]'], ['CHLORIDE CHANNEL C (CLC-C) [gene]'], ['GmCLC1 protein(s) [gene]'], ['choline transporter-like 1 (CHER1) [gene]'], ['algal CLC [gene]']]}\n",
      "-------------------\n",
      "{'18849': [['high level of storage oil mobilization [phenotype]'], ['increase in seed TAG production [phenotype]'], ['**enhance seed lipid accumulation [phenotype]**', 'increase in accumulation of storage lipids in seeds [phenotype]'], ['increased oil production in seeds [metabolite]']]}\n",
      "-------------------\n",
      "{'13290': [['**Adaxial-abaxial polarity [phenotype]**', 'abaxial-adaxial polarity [phenotype]', 'Abaxial/adaxial polarity [phenotype]', 'AD/AB polarity [phenotype]', 'leaf abaxial-adaxial polarity [phenotype]', 'wild-type ab- and adaxial polarity [phenotype]', 'adaxial/abaxial polarity [phenotype]', 'adaxial-abaxial polarity of leaf [phenotype]', 'proper adaxial-abaxial polarity [phenotype]', 'adaxial and abaxial polarity [phenotype]', 'ad-/abaxial polarity [phenotype]', 'ad/abaxial leaf polarity [phenotype]'], ['**adaxial-abaxial polarity specification [phenotype]**', 'AD/AB polarity specification [phenotype]', 'polarity specification of adaxial/abaxial axis [phenotype]', 'polarity specification of the adaxial/abaxial axis [phenotype]', 'ad/abaxial polarity specification [phenotype]', 'proper specification of adaxial-abaxial polarity [phenotype]', 'Leaf adaxial-abaxial polarity specification [phenotype]', 'adaxial/abaxial polarity specification [phenotype]'], ['Axis development and polarity specification of A. thaliana leaf [phenotype]'], ['leaf adaxial-abaxial polarity determination [phenotype]', 'determination of adaxial-abaxial polarity [phenotype]', '**Adaxial-abaxial polarity determination [phenotype]**', 'abaxial/adaxial polarity determination [phenotype]'], ['AD/AB polarity establishment [phenotype]', 'establishing adaxial-abaxial polarity [phenotype]', '**adaxial-abaxial polarity establishment [phenotype]**'], ['establishment of adaxial-abaxial polarity in integuments [phenotype]'], ['abaxial polarity establishment [phenotype]'], ['polarity development of lateral organs [phenotype]'], ['abaxial polarity development [phenotype]'], ['**Adaxial-abaxial axis [phenotype]**', 'adaxial-abaxial axis of polarity [phenotype]', 'polarity of adaxial-abaxial axis [phenotype]'], ['polar distribution along adaxial-abaxial axis [phenotype]', '**adaxial-abaxial polarity distribution [phenotype]**'], ['Adaxial-abaxial polarity identity [phenotype]'], ['adaxial polarity identities [phenotype]'], ['abaxial polarity identities [phenotype]'], ['Adaxial polarity determinants [phenotype]'], ['major abaxial polarity determinant [phenotype]'], ['abaxial/adaxial polarity program [phenotype]'], ['adaxial and medial/marginal positional identities [phenotype]'], ['outer whorl abaxial/adaxial polarity [phenotype]'], ['paraxial abaxial polarity of lateral organs [phenotype]'], ['anther adaxial-abaxial polarity [phenotype]'], ['dynamic and competing ab-/adaxial polarity [phenotype]'], ['adaxial and abaxial fates [phenotype]'], ['adaxial/abaxial polarity of carpel development [phenotype]'], ['abaxial polarity specification [phenotype]'], ['leaf adaxial polarity specification [phenotype]'], ['abaxial polarity in leaf [phenotype]'], ['adaxial/abaxial tissue polarity and associated WOX activities [phenotype]'], ['early abaxial/adaxial polarity [phenotype]'], ['polar determination of abaxial identity in the ovule [phenotype]'], ['adaxial polarity marker [phenotype]'], ['abaxial polarity marker [phenotype]']]}\n",
      "-------------------\n",
      "{'98830': [['Transiently transformed tobacco [organism]'], ['transformed Nicotiana benthamiana pavement cells [cell type]'], ['**Agro-infiltrated tobacco leaf [tissue]**', 'Agrobacteria-infused Nicotiana tobacum leaf tissue [tissue]'], ['leaf extracts from transplastomic tobacco [organ]'], ['constructs transiently expressed in tobacco leaf [organ]'], ['transgenic tobacco BY-2 cells [cell type]'], ['protein(s) purified from leaf of WT and transgenic tobacco plant(s) [organism]'], ['Transformed callus cultures of Nicotiana tabacum [organism]'], ['sgRNA in tobacco protoplasts [organism]'], ['tobacco protoplast system [cell type]'], ['tobacco protoplast transient system [organism]'], ['tobacco agro-infiltrated leaf [organ]'], ['F31 leaf [tissue]'], ['transgenic tobacco leaf protoplasts [cell type]'], ['transformed tobacco cells [cell type]'], ['tobacco BY2 suspension cells [cell type]'], ['Agrobacterium-treated tobacco leaf explants [treatment]'], ['**transiently transformed tobacco leaf [organ]**', 'transiently transformed Nicotiana tabacum leaf [organ]'], ['root(s) of transformed tobacco [organ]'], ['plant(s) protoplasts of tobacco [organism]']]}\n",
      "-------------------\n",
      "{'104369': [['salivary glands [organ]'], ['**Insect midgut [organ]**', 'mid-gut [organ]', 'midgut of insects [organ]'], ['mouthparts [organ]'], ['**insect gut [organ]**', 'insect guts [organ]', 'insect intestine [organ]'], ['**insect digestive tract [organism]**', 'guts of insects [organism]', 'digestive system of insects [organism]'], ['alkaline midguts [organ]'], ['slightly acidic midguts [organ]'], ['peritrophic matrix [organ]'], ['gut of wood-decaying beetles [organ]'], ['digestion and absorption of food in the insect gut [organ]'], ['insect gut barrier [organ]'], ['entire gut [organ]']]}\n",
      "-------------------\n",
      "{'47170': [['sel1 mutation [genetic element]'], ['disruption of SYT1 [mutant]'], ['sax1 mutation [mutant]'], ['syn1 mutation [mutant]'], ['knocking out both TSO1 and SOL2 [treatment]']]}\n",
      "-------------------\n",
      "{'165937': [['AGP-pectin-xylan proteoglycan [metabolite]'], ['unique proteoglycan [metabolite]'], ['apoplastic heteroglycans [metabolite]'], ['protein(s) and proteoglycans [metabolite]']]}\n",
      "-------------------\n",
      "{'150418': [['PIO000603 [gene]'], ['PIO000689 [gene]'], ['SORBI_3001G248500 (GR2) [gene]'], ['Sobic.003g402600 [gene]'], ['Sobic.004G230000 [gene]'], ['Sobic.004G236000 [gene]'], ['Sobic.007G183200 [gene]'], ['Sobic.002G231600 [gene]'], ['Sobic.001G328500 [gene]'], ['Sobic.008G171600 [gene]'], ['SORBI_3008G116500 [gene]'], ['Sobic.004G281800.1 (SbRVE6) [gene]'], ['Sobic.001G352400.1 [gene]'], ['Sobic.001G352400.2 [gene]'], ['Sobic.001G352400.3 [gene]'], ['Sobic.001G352400.4 [gene]'], ['Sobic.002g193000.1 [gene]'], ['Sobic.001G480400.1 [gene]'], ['Sobic.001G141300.1 [gene]'], ['Sobic.008G096200 [gene]'], ['PROG homolog (Sobic.002G036600.2) in sorghum [gene]'], ['Sobic.002G260800 [gene]'], ['Sobic.004G237300 [gene]'], ['Sobic.003G148600 [gene]'], ['Sobic.010G080400 [gene]'], ['Sobic.003G035100 [gene]'], ['Sobic.001G110900 [gene]'], ['Sobic.003G131600 [gene]'], ['Sobic.001G469300 [gene]'], ['Sobic.003G429800.1 [gene]'], ['Sobic.009G122600.1 [gene]'], ['Sobic.010G261700.1 [gene]'], ['Sobic.002G274500.1 [gene]'], ['Sobic.007G179000.1 [gene]'], ['Sobic.007G047300 [gene]'], ['Sobic.004G062500 [gene]']]}\n",
      "-------------------\n",
      "{'8926': [['**morphological differences [phenotype]**', 'minor morphological variation [phenotype]'], ['**color difference [phenotype]**', 'obvious differences in leaf color [phenotype]', 'color differences [phenotype]', 'color variation [phenotype]'], ['Similar variability [phenotype]', '**Differences in phenotype [phenotype]**', 'variance [phenotype]', 'plant(s) phenotypic variation [phenotype]', 'variation for this trait [phenotype]', 'Natural variations [phenotype]', 'phenotypic differentiation [phenotype]', 'Variation for traits [phenotype]', 'differentiation difference [phenotype]', 'natural variance of traits [phenotype]', 'contrasting phenes [phenotype]', 'observed variation [phenotype]', 'phenotypical variations [phenotype]', 'variation in phenotypical traits [phenotype]', 'variation in multiple traits [phenotype]', 'phenotypic trait differences [phenotype]', 'distinct differences [phenotype]', 'phenotypical differences [phenotype]', 'obvious phenotypic variance [phenotype]', 'clear differences [phenotype]', 'observed phenotypic differences [phenotype]', 'different degrees [phenotype]', 'apparent differences [phenotype]', 'measurable phenotypic differences [phenotype]', 'observed trait variation [phenotype]'], ['natural variation among genotypes [phenotype]', 'variation among plant(s) genotypes [phenotype]', '**Differences between genotypes [phenotype]**', 'genotypically determined variances [phenotype]', 'plant(s) model parameters varying among genotypes [phenotype]'], ['differences between the three genotypes [phenotype]'], ['variation between parental genotypes [phenotype]'], ['**variability among accessions [phenotype]**', 'variations among plant(s) accessions [phenotype]', 'among-accession variation [phenotype]', 'variation among accessions [phenotype]'], ['**population-level differences [phenotype]**', 'differences between populations [phenotype]'], ['**ecotype-dependent differences [phenotype]**', 'ecotype difference [phenotype]'], ['variation in seedling phenotypes [phenotype]'], ['species-associated differences [phenotype]', 'different characteristics in different species [phenotype]', '**species-to-species differences [phenotype]**', 'differences across diverse lineages [phenotype]', 'phenotypic divergence among species [phenotype]', 'species-specific differences in plant(s) [phenotype]', 'interspecific phenotypical variability [phenotype]', 'consistent differences between species [phenotype]'], ['**metabolic differences [phenotype]**', 'biochemical differences [phenotype]'], ['**phenotypic differences from WT [phenotype]**', 'phenotypic differences compared to WT plant(s) [phenotype]', 'phenotypic differences to WT [phenotype]'], ['phenotypic differences under standard growth conditions [phenotype]'], ['differences in disease phenotype manifestation [phenotype]'], ['different degrees of mutation [phenotype]'], ['phenotypic differences compared to control plant(s) [phenotype]'], ['phenotypic differences in agronomic traits [phenotype]'], ['**Plant-to-plant variability [phenotype]**', 'Variability between plant(s) [phenotype]', 'individual variability between plant(s) [phenotype]'], ['separation between treatments [phenotype]'], ['differences of means [phenotype]'], ['phenotypic variation in sorghum [phenotype]'], ['difference between predicted and observed phenotype [phenotype]'], ['**varietal differences [phenotype]**', 'varies between varieties [phenotype]'], ['distinct variation in physiological traits [phenotype]'], ['wild-type variance [phenotype]'], ['variation among species [organism]'], ['variation among individual plant(s) [organism]'], ['differences between the cultivars [organism]']]}\n",
      "-------------------\n",
      "{'105086': [['plant(s) cDNAs libraries [gene]'], ['constructing a cDNA library [phenotype]', '**cDNA-libraries [phenotype]**'], ['making libraries of full length cDNAs from A. thaliana [phenotype]'], ['making libraries of full length cDNAs from maize [phenotype]'], ['making libraries of full length cDNAs from soybean (Glycine max) [phenotype]']]}\n",
      "-------------------\n",
      "{'77227': [['NTRC domains [protein domain]'], ['interactions between NTR domains [protein domain]'], ['RTM protein(s) domains [protein domain]'], ['Three-dimensional structure of RTX domain [protein domain]'], ['reverse transcriptase domains [feature]'], ['three glutaredoxin (GRX) domains [protein domain]'], ['three TRX domains [protein domain]'], ['three thioredoxin fold domains [protein feature]'], ['thioredoxin domains [protein]'], ['four thioredoxin-like domains [protein]'], ['three distinct cupredoxin domains [protein domain]'], ['different regions for cupredoxin domains [protein domain]'], ['three WD40 repeat domains [protein domain]'], ['core functional domain in the ROT4 protein(s) [protein]'], ['**two TRX domains (TRX-B) [protein]**', 'two domains (TRX domain 1 and domain 2) [protein]'], ['two TRX domains [protein domain]'], ['five key domains of TOR [protein domain]'], ['tandem BRX domains [protein domain]']]}\n",
      "-------------------\n",
      "{'81729': [['SCOOP14 [peptide]'], ['active forms of certain Rhos [protein]'], ['PII [protein]'], ['x/psi/kappa 14-3-3 [protein]'], ['14-3-3omega [protein]'], ['gas-phase intact protein(s) ions [protein]'], ['octameric apoSyn-CSK protein(s) [protein]'], ['solid form of SEC14-like [protein]'], ['14-3-3 phi [protein]'], ['14-3-3 upsilon [protein]'], ['\\nα-casein [protein]'], ['14-3-3ο [protein]'], ['levels of p24 protein(s) [protein family]'], ['Ma3 [protein]'], ['LBP/BPI subfamily [protein]'], ['SQM> [protein]'], ['GF14/lamda [protein]'], ['GF14/omicron [protein]'], ['14-3-3omega-YN [protein]'], ['protein(s) forms with a GPA peptide [protein]'], ['hp14.5 [protein]'], ['a J-protein type III [protein]'], ['UNKNOWN 2, FBN7B [protein]'], ['mu [protein]'], ['Iota [protein]'], ['stable alkaline protein(s) [protein]'], ['homotypic PB [protein complex]'], ['GPA1-dependent [protein]'], ['co-purified peptide [protein]']]}\n",
      "-------------------\n",
      "{'42228': [['**tgd3 [mutant]**', 'tgd3 mutant [mutant]'], ['tgd mutants [mutant]'], ['tgd1-1 mutant [mutant]'], ['trigalactosyldiacylglycerol2 (tgd2) mutant [mutant]'], ['tgd1 mutant [mutant]'], ['tsd2 [mutant]'], ['**tsd1 [mutant]**', 'tsd1 mutant [mutant]'], ['tds4-2 [mutant]'], ['Soil-grown tsd2 mutants [mutant]'], ['Deltasuc2 mutant [mutant]'], ['Deltasuc3 mutant [mutant]']]}\n",
      "-------------------\n",
      "{'75904': [['CHASE domain [protein domain]'], ['CD domain (LH) DxxDE (P) xC [genetic element]']]}\n",
      "-------------------\n",
      "{'107662': [['**Polycomb silencing [process]**', 'Polycomb-mediated chromatin regulation [process]', 'transcriptional silencing by Polycomb [process]'], ['**Polycomb chromatin [subcellular compartment]**', 'Polycomb-repressed chromatin [subcellular compartment]', 'Polycomb-silenced chromatin regions [subcellular compartment]'], ['Polycomb silencing [mechanism]'], ['loss of multiple Polycomb subunits [protein complex]'], ['Polycomb silencing [protein complex]'], ['loss of Polycomb protein-mediated chromatin silencing [phenotype]'], ['higher proportion of FLC loci switched to Polycomb silenced state [phenotype]'], ['somatic Polycomb domains [epigenetic mark]'], ['accumulation of specific Polycomb complexes [gene]']]}\n",
      "-------------------\n",
      "{'33899': [['Subcellular localization of AGS1 [enzyme]'], ['subcellular localization of AGG3 [phenotype]'], ['**subcellular localization of AtRGS1-YFP [subcellular compartment]**', 'subcellular localization of EGL-4 translation [subcellular compartment]', 'initial evidence of targeting to endoplasmic reticulum and/or Golgi network [subcellular compartment]', 'initial evidence of targeting to cytosol [subcellular compartment]'], ['subcellular targeting of AtGALT29A [gene]']]}\n",
      "-------------------\n",
      "{'55696': [['Redox state of metabolites [process]'], ['PQ totals [metabolite]'], ['UQ totals [metabolite]'], ['singlet state (1O2) [metabolite]'], ['Reduced NPQ [metabolite]'], ['PQ-9 content [metabolite]', '**PQ-9 levels [metabolite]**'], ['low PQ-9 levels [metabolite]'], ['non-oxidized SQD [metabolite]'], ['Zerovalent selenium [metabolite]'], ['no reactive oxygen species production [metabolite]'], ['oxidation steps [metabolite]'], ['reduction potential [metabolite]']]}\n",
      "-------------------\n",
      "{'136924': [['downregulation of CsTFL1 [gene]'], ['Down-regulated transcripts of CaRDR1 [gene]'], ['Down-regulation of NLR [gene]'], ['downregulation of NCR gene(s) [gene]'], ['downregulation of TCP13 [gene]'], ['downregulation of TCP17 [gene]'], ['downregulation of RICE1 and RICE2 [process]'], ['downregulation of PtEG16 [gene]'], ['Downregulation of MigPSY gene(s) expression [gene]'], ['down-regulation of SPP gene(s) expression [process]'], ['down-regulated NatA complex [gene]'], ['down-regulation of VAC14 [gene]'], ['knockdown of Nod factor receptor gene(s) [treatment]'], ['down-regulation of RAV2/TEM2 gene(s) [gene]'], ['downregulation of TbAIR9 [gene]'], ['down-regulation of KN1 [gene]'], ['downregulated TOR activity [gene]'], ['Downregulation of hNAA10 [process]'], ['down-regulation of TaSTP13 [gene]'], ['down-regulation of the H-origin copy [gene]'], ['downregulation of EXT3 [gene]'], ['down-regulation of EXLA2 [gene]'], ['downregulation of TUB1 expression [gene]'], ['Down-regulation of Ca-DOX [gene]'], ['Down-regulation of EsCBL10a [gene]'], ['Down-regulation of OsWAKL21.2 [gene]'], ['Downregulation of TPLATE [gene]'], ['down-regulation of ThtAG2 [gene]'], ['downregulation of TPPA expression [gene expression]'], ['downregulation of TPPD expression [gene expression]'], ['downregulation of TPPG expression [gene expression]'], ['downregulation of TPPH expression [gene expression]'], ['downregulation of TPPI expression [gene expression]'], ['downregulation of UBQ11 [gene]'], ['down-regulation of LRR-RLP transcripts [gene]'], ['down-regulation of NB-LRR transcripts [gene]'], ['additional downregulation of ROP4 or ROP6 [gene]'], ['led to the down-regulation of RLP23 [gene]'], ['led to the down-regulation of BDA1 [gene]'], ['Downregulation of CpFT1 [gene]'], ['down-regulation of TCP3 [gene]'], ['down-regulation of TCP4 [gene]'], ['down-regulation of TCP10 [gene]'], ['down-regulation of TCP24 [gene]'], ['**down-regulated TCP gene(s) [gene]**', 'down-regulation of TCP family gene(s) [gene]'], ['downregulation of AT3G26140 [gene]'], ['down-regulation of TPS [gene]'], ['down-regulated MLP-like 43 [gene]'], ['transcriptional downregulation of CPD and ROT3 [gene]'], ['downregulation of RbohB [gene]'], ['downregulation of TaARF6 and 9 [gene]'], ['downregulation of flagellar gene(s) [gene]'], ['down-regulation of TP gene(s) [gene]'], ['down-regulation of T/RT gene(s) [gene]'], ['downregulation of A-ARR [gene]'], ['downregulation of gene(s) related to motility [gene]'], ['downregulation of gene(s) related to chemotaxis [gene]'], ['downregulation of RAP2.6 [gene]'], ['downregulation of CscalS5 and CscalS8 expression [gene]'], ['downregulation of gene(s) related to actin dynamics [gene]'], ['down-regulation of yqxM-sipW-tasA operon [gene]'], ['down-regulation of epsA-O operon [gene]'], ['downregulation of TUB1 [gene]']]}\n",
      "-------------------\n",
      "{'66190': [['RS2Z members [protein]'], ['group G [protein group]'], ['subfamilies BZ1-BZ8 [protein group]'], ['subfamilies BZ9-BZ39 [protein group]'], ['BZ24-BZ26 [subfamily]'], ['eight out of 12 SlJAZs [gene]'], ['group 3 B-box protein(s) family [protein family]'], ['one or two additional TFIIB-related protein(s) subfamilies [gene]'], ['6 RS2Z members [protein feature]'], ['3 RSZ members [protein feature]'], ['3 RS2Z members [protein feature]'], ['Subgroup 1 [protein subgroup]'], ['subgroup B1 [category]'], ['subgroup B2 [category]'], ['subgroup D [category]'], ['subgroups 1 and 2 [protein]'], ['44 subfamilies [protein group]'], ['BZ43 [group]'], ['BZ40-BZ42 [group]'], ['groups D and E [group]'], ['BZ37-BZ39 [group]'], ['eight sub-families [group]'], ['I to IV [protein family]'], ['subfamily VI protein(s) [protein]'], ['phi, tau, theta, zeta and DHAR sub-sections [protein family]']]}\n",
      "-------------------\n",
      "{'100855': [['structural integrity of plant(s) [organism]'], ['**plant(s) architecture [organism]**', 'plant(s) architectures [organism]', 'The architecture of sessile plant(s) [organism]', 'plant(s) with complex 3D architecture [organism]'], ['dominant distribution form in plant(s) [organism]'], ['morphologically in plant(s) [organism]'], ['naturalists [organism]'], ['plant(s) analysis [organism]'], ['plant(s) anatomical space [concept]'], ['plant(s) Structure [concept]'], ['wonder at the magnificence of plant(s) [concept]'], ['functional assemblage of heterogeneous plant(s) [process]'], ['plant(s) architecture [process]'], ['**plant(s) size [organ]**', 'plant(s) stature [organ]']]}\n",
      "-------------------\n",
      "{'75918': [['Hpt proteins/domains [protein]', '**HPt domains [protein]**', 'intermediate protein(s) with an Hpt domain [protein]', 'HPt domain [protein]'], ['Dimerization and Histidine phosphotransfer domain (DHp) [protein]'], ['histidine-containing phototransfer domain [protein]'], ['**DHp domains [protein domain]**', 'DHp domain [protein domain]'], ['His-containing phosphotransfer (HPt) domains [protein domain]']]}\n",
      "-------------------\n",
      "{'51024': [['daysleeper phenotype [phenotype]'], ['cob-6 mutant phenotypes [phenotype]'], ['mlks2 phenotypes [mutant]'], ['npq6 mutation [phenotype]'], ['glcat14 mutant phenotypes [phenotype]'], ['Phenotypes of the mutants [phenotype]'], ['clv2 phenotypes [phenotype]'], ['G protein(s) phenotypes of insertion mutations [phenotype]'], ['nop9 mutant phenotype [phenotype]'], ['gcn5 phenotypes [phenotype]'], ['pgm1 phenotype [phenotype]'], ['phenotypic changes in hon5 [mutant]'], ['cmu1cmu2 phenotypes [phenotype]'], ['galt8 mutant phenotypes [phenotype]'], ['ttg1 phenotypes [mutant]'], ['tmn1 mutant phenotypes [phenotype]'], ['phenotype of double glk mutant [phenotype]'], ['procuste1 mutant phenotype [phenotype]'], ['mutant phenotypes of pgr7 [phenotype]'], ['gl1 phenotypes [phenotype]'], ['mpk6 phenotypes [phenotype]'], ['mrp5-1 mutant phenotype [mutant]'], ['gns2/Atelp3 mutant phenotypes [phenotype]'], ['phenotype of CI mutants [phenotype]'], ['glabra1 mutant phenotype [phenotype]'], ['dek1 mutant phenotypes [phenotype]']]}\n",
      "-------------------\n",
      "{'12202': [['twisted inflorescence, root, and silique phenotypes [phenotype]'], ['needle-like inflorescence stem(s) [phenotype]'], ['simple inflorescences [phenotype]'], ['inflorescences in place of tendrils [phenotype]'], ['slender inflorescences [phenotype]'], ['helical twisting of inflorescence stem(s) [phenotype]'], ['similar inflorescence stem(s) [phenotype]'], ['**inflorescence twisting [phenotype]**', 'twisting of inflorescences [phenotype]'], ['fasciated primary inflorescence stem(s) [phenotype]'], ['white upper inflorescence stem(s) [phenotype]'], ['pendent inflorescence stem(s) phenotype [phenotype]'], ['WT inflorescence stem(s) wax [phenotype]'], ['inflorescences that form only a few flowers and grow as a pin-like structure [phenotype]'], ['determinate zigzag-shaped inflorescence [phenotype]'], ['stiff inflorescence stem(s) [phenotype]']]}\n",
      "-------------------\n",
      "{'139501': [['expression of tryptophan [phenotype]'], ['asparagine synthetase gene(s) expression in wheat (Triticum aestivum) embryo [phenotype]', '**asparagine synthetase gene(s) expression in wheat embryo [phenotype]**'], ['constitutive expression of gene(s) involved in biosynthesis of tryptophan-derived metabolites [phenotype]'], ['induction of mRNAs and protein(s) for tryptophan biosynthetic enzymes [gene]'], ['asparagine synthetase gene(s) expression [gene]'], ['expression of TRP biosynthetic gene(s) ASA1 [gene]'], ['strong transcriptional activation of tryptophan and camalexin biosynthetic gene(s) [gene]'], ['expression of OSBS [gene]']]}\n",
      "-------------------\n",
      "{'83317': [['post translated modification [phenotype]', '**post-translation modifications [phenotype]**']]}\n",
      "-------------------\n",
      "{'170129': [['reduction in overall CDK activities [gene]'], ['**repression of PAL [enzyme]**', 'repression of PAL activity [enzyme]', 'repression of PAL enzyme activity [enzyme]'], ['inhibition of PM-associated PP2C.D phosphatases [phenotype]'], ['inhibition of ceramide synthase [phenotype]'], ['PAH inactivation [phenotype]'], ['CDK-specific suppression of PAH1 catalytic function [gene function]'], ['Suppression of PAH activity [enzyme]'], ['Inhibition of PC-PLC activity [process]'], ['inhibition of PLC [process]'], ['inhibition of PLC activity [phenotype]'], ['inhibition of PLC activity [enzyme]'], ['**Downregulation of SPT [enzyme]**', 'decrease in serine palmitoyltransferase activity [enzyme]'], ['Reduction in catalytic activity of PAH1 [gene]'], ['Reduction in catalytic activity of PAH2 [gene]'], ['Cantharidin promoted inhibition of PPPs [process]'], ['inhibition of ceramide synthase [enzyme]'], ['Disruption of PAL [gene]'], ['degradation of PaWRI3 [phenotype]'], ['repression of PAL enzyme activity [phenotype]'], ['phosphatase activity inhibition [process]'], ['PP2C phosphatase inhibition [phenotype]'], ['SPT inhibition [phenotype]'], ['chemical inhibition of PLD activity [phenotype]'], ['lower phospholipase A1-IIdelta abundance [enzyme]'], ['inhibitory effects on ABI1-dependent phosphatase activity in vitro [process]'], ['decline in activity of P5CS [gene]'], ['Galpha inhibition of PLDalpha1 activity [enzyme activity]'], ['PA-mediated inhibition of OsAKT2 [phenotype]']]}\n",
      "-------------------\n",
      "{'97953': [['Artemisia argyi [organism]'], ['Isatis indigotica [organism]'], ['Isatis indigotica Fortune [organism]', '**Isatis indigotica Fort. [organism]**'], ['Melia azedarach [organism]'], ['wild A. lancea [organism]'], ['medicinal uses of Euryale ferox [organism]'], ['Azadirachta [organism]'], ['Caesalpinia gilliesii [organism]'], ['**Azadirachta indica [organism]**', 'A. indica [organism]'], ['Eucalyptus maculata [organism]'], ['Azadirachta indica (Neem) [organism]']]}\n",
      "-------------------\n",
      "{'115562': [['zebrafish-human orthologs [gene]'], ['**D14 orthologs [gene]**', 'D14 ortholog [gene]'], ['52 BHI orthologs of mTERF27 [gene]'], ['One Nse4 homolog [gene]'], ['orthologs of three other new hubs [gene]'], ['sequence alignment of HEN1 homologs [gene]'], ['BEH3 orthologs [gene]'], ['BRCA1 orthologs [gene]'], ['KIN17 orthologs [gene]'], ['RKS1 orthologs [gene]'], ['Lal2 sequence orthologs [gene]'], ['less distant MAR1 orthologs [gene]'], ['Number of orthologous events of StDREB gene(s) [gene]'], ['JAZ orthologs [gene]'], ['Rec114 orthologs [gene]'], ['17 CNL orthologs [gene]'], ['other DYRKP orthologs [gene]'], ['4 homologs [gene]'], ['numerous formin paralogs [gene]'], ['close orthologs in 10 other species [organism]'], ['5 homologous ESTs [gene]'], ['DUR3 orthologs [gene]'], ['gene(s) that are likely orthologs of the ORFs p61, p32, and p24 found in typical cileviruses [gene]'], ['FN3K orthologs [gene]'], ['orthologues from other organisms [gene]']]}\n",
      "-------------------\n",
      "{'21468': [['**water-deficit stress tolerance [phenotype]**', 'water deficit stress tolerance [phenotype]', 'water-deficit stress tolerant [phenotype]', 'stress tolerance against water deficit stress [phenotype]', 'tolerance of water-deficit stress [phenotype]'], ['water deficit-tolerance in Medicago truncatula [phenotype]'], ['water deficit stress tolerance in tomato [phenotype]'], ['**water deficit stress resistance [phenotype]**', 'water-deficit stress resistance [phenotype]', 'resistance to water-deficit stress [phenotype]', 'water deficit resistance [phenotype]'], ['water deficit sensitivity [phenotype]']]}\n",
      "-------------------\n",
      "{'111598': [['unknown regulatory sequences [phenotype]'], ['conserved stretches in regulatory regions [phenotype]'], ['**non-coding DNA [gene]**', 'Non-coding regions [gene]', 'Noncoding DNA [gene]', 'its noncoding sequences [gene]'], ['specific noncoding sequences [genomic region]'], ['CNSs [gene region]'], ['noncoding regulatory regions [genomic region]'], ['**noncoding sequences [phenotype]**', 'non-coding DNA sequence [phenotype]'], ['Noncoding DNA [phenomenon]'], ['cryptic regulatory sequences [gene]'], ['non-coding regulatory region of Dt2 [gene]'], ['TCO non-coding sequence [gene]'], ['non-transcribed region [sequence]'], ['Intronic regions [regulatory region]'], ['DNA-sequence independent mechanisms [process]']]}\n",
      "-------------------\n",
      "{'79623': [['Ubiquitin-dependent proteolysis [process]', 'Ubiquitin-mediated degradation [process]', 'Proteolysis of ubiquitin-labeled protein(s) [process]', '**Ubiquitin-mediated proteolysis [process]**', 'ubiquitin-proteasome mediated protein(s) degradation [process]'], ['ubiquitin-mediated proteasome activities [process]'], ['ubiquitin-mediated cleavage [process]'], ['Ubiquitin-mediated endocytosis [process]'], ['ubiquitin-mediated proteasome activities [treatment]'], ['Ubiquitin-mediated proteolysis [treatment]', 'Ubiquitin protein(s) degradation [treatment]', '**Ubiquitin-mediated degradation [treatment]**', 'ubiquitin/proteasome-mediated degradation [treatment]'], ['Ub-driven autophagy [treatment]'], ['regulated ubiquitin-mediated proteolysis [phenotype]'], ['ubiquitin-mediated destruction [phenotype]'], ['Ubiquitin-mediated proteolysis [function]']]}\n",
      "-------------------\n",
      "{'144748': [['JAZ 1, 3, 9 and 10 [gene]'], ['COI1 and JAZ protein(s) [protein]'], ['binding of COI1-JA-Ile and JAZ protein(s) [process]'], ['JAZ-COI1 interaction [process]'], ['COI-JAZ interaction [phenotype]'], ['COI1-JAZ complex formation [phenotype]'], ['COI1-JAZ9 interaction [phenotype]'], ['COI1-mediated JAZ degradation [process]'], ['JAZ3-COI1 coprecipitation [phenotype]'], ['JAZ3deltaN-COI1 interaction [phenotype]'], ['interaction with OsJAZs [gene]'], ['COI1 interaction with JAZ protein(s) [gene]', '**interaction between COI1 and JAZ repressor [gene]**'], ['JA-Ile/COR-dependent COI1-JAZ interaction [phenotype]'], ['COI1-JAZ9 interaction [protein interaction]'], ['COI1 to degrade JAZ repressors [gene]'], ['interaction between JA receptor COI1 and JAZ1 [protein]'], ['triggering COI1-JAZ1 interaction [process]']]}\n",
      "-------------------\n",
      "{'75489': [['PrLD [domain]'], ['**PrLDs [protein domain]**', 'Prion-like domains (PrLDs) [protein domain]', 'PrLD [protein domain]', 'Prion-like domain (PrLD) [protein domain]'], ['ARF19 PLD [protein domain]'], ['ARF PLDs [protein domain]'], ['PrLDs-containing protein(s) [protein]'], ['CARs-DB [database]'], ['pCARs [protein domain]'], ['human PrLDs [protein domain]'], ['**PrLDs [protein]**', 'PrLD protein(s) [protein]', 'PrLD [protein]'], ['Amyloid regions in PrLDs [protein domain]'], ['Amyloid sequence space in plant(s) PrLDs [protein domain]'], ['PrLDs containing CARs [protein domain]'], ['PrLD [protein region]'], ['ELF3 PrLD [protein region]'], ['Luminidependens PrD (LDPrD) [protein domain]'], ['two predicted PrLDs [protein domain]'], ['tyrosines in AtPARCL PLD [protein region]'], ['large putative PLD [protein domain]'], ['similar sequence composition to ARF6-8 PLDs [protein domain]'], ['similar sequence composition to TDP43 PLD [protein domain]'], ['similar sequence composition to hnRNPA1 PLD [protein domain]'], ['similar sequence composition to EWS PLD [protein domain]'], ['other PLD-containing protein(s) [protein]'], ['PrLD of 180 amino acids [subcellular compartment]'], ['single hit per PrLD [protein domain]'], ['bacteria PrLDs [protein domain]'], ['viruses PrLDs [protein domain]'], ['S. cerevisiae PrLDs [protein domain]'], ['known PrLDs in yeast [protein]']]}\n",
      "-------------------\n",
      "{'14473': [['**leaf maturity [phenotype]**', 'maturing leaf [phenotype]', 'maturing and immature leaf [phenotype]'], ['mature leaf in RGN [phenotype]'], ['Erected leaf until maturity [phenotype]'], ['leaf kept throughout life-cycle [phenotype]', '**lifespan of leaf [phenotype]**', 'Older leaf [phenotype]', 'long-lived leaf [phenotype]', 'longer leaf lifespan [phenotype]', 'leaf life-span [phenotype]', 'lifespan of older leaf [phenotype]'], ['leaf states [phenotype]'], ['final size and shape of the leaf [phenotype]', '**mature leaf shape [phenotype]**'], ['leaf of 4- and 6-week-old plant(s) [phenotype]'], ['leaf life history [phenotype]']]}\n",
      "-------------------\n",
      "{'151910': [['GmUC1 [gene]'], ['GmUC2 [gene]'], ['GsPUB8 [gene]']]}\n",
      "-------------------\n",
      "{'170087': [['GPAT4 and GPAT8 [enzyme]'], ['glycerol-3-phosphate acyltransferase (GPAT) [enzyme]', '**GPAT [enzyme]**', 'glycerol-3-phosphate acyltransferases (GPATs) [enzyme]', 'glycerol-3-phosphate acyltransferases [enzyme]', 'glycerol phosphate acyltransferases (GPAT) [enzyme]', 'glycerol-3-phosphate acyltransferase [enzyme]', 'glycerol 3-phosphate acyltransferase [enzyme]', 'Gly3P acyltransferase (GPAT) [enzyme]', 'glycerol 3-phosphate acyltransferase (GPAT) [enzyme]', 'glycerol phosphate acyltransferase [enzyme]', 'G3P acyltransferase (GPAT) [enzyme]'], ['lysophosphatidylglycerol acyltransferase 1 (LPGAT1) [enzyme]'], ['glycerol 3-phosphate acyltransferases (GPAT5, GPAT7) [gene]'], ['GPAT9-type GPAT [enzyme]'], ['Glycerol-3-phosphate acyltransferase (EC 2.3.1.15) [enzyme]'], ['glycerol-3-phosphate acyltransferases [gene]'], ['**Glycerol-3-phosphate O-acyltransferases (GPAT) [enzyme]**', 'glycerol-3-phosphate O-acyltransferase (GPAT) [enzyme]'], ['glycerol-3-phosphate acyltransferase 4 (GPAT4) [enzyme]', '**GPAT4 [enzyme]**', 'A. thaliana GPAT4 acyltransferase [enzyme]'], ['GPCAT [enzyme]'], ['GPCAT assays [process]'], ['**GPCAT activity [enzyme]**', 'high GPCAT activity [enzyme]', 'GPCAT activities [enzyme]'], ['stepwise acylation by G3P acyltransferase [enzyme]'], ['plastid glycerol-3-phosphate acyltransferase [enzyme]'], ['**glycerol-3-phosphate acyltransferase (GPAT) [protein]**', 'GPATs [protein]', 'glycerol-3-phosphate acyl transferase (GPAT) [protein]'], ['endogenous GPAT activity [gene]'], ['membrane-bound, eukaryotic G3P acyltransferases (GPATs) [enzyme]'], ['GPAT4 and GPAT6 [enzyme]'], ['gpat5 [enzyme]'], ['plant(s) GPATs [enzyme]'], ['GPAT8 [enzyme]'], ['**chloroplast glycerol-3-phosphate acyltransferase (GPAT) [enzyme]**', 'chloroplast GPAT [enzyme]'], ['GRETCHEN HAGEN 3 (GH3) acyl acid amido synthetases [enzyme]'], ['GPAT activity [process]'], ['microsomal GPAT [enzyme]'], ['E. coli GPAT enzyme activity [process]'], ['GPAT activity [enzyme]'], ['tgl4p [enzyme]'], ['plastid glycerol-3-phosphate acyltransferase activity [enzyme]'], ['functional GPAT enzymes [gene]']]}\n",
      "-------------------\n",
      "{'130937': [['AtEDEM1/AtEDEM2 [gene]', 'two EDEM homologs [gene]', '**both AtEDEMs [gene]**', 'AtEDEM homolog [gene]', 'AtEDEM homologs [gene]'], ['AtEDEM1 homologous gene(s) [gene]', 'AtEDEM1 homolog [gene]', '**EDEM1 [gene]**'], ['AtEDEM2 homologs from land plant(s) [gene]', 'EDEM2s [gene]', '**EDEM2 [gene]**'], ['ELMOD_A [gene]'], ['ELMOD_E [gene]'], ['ELMOD_D [gene]'], ['ELMOD_C [gene]'], ['ELMOD_F [gene]'], ['ELMOD_B [gene]']]}\n",
      "-------------------\n",
      "{'42267': [['Abstinence by mutual consent (amc) [mutant]'], ['Atrad16 null mutant [mutant]'], ['Atrad16 single null mutant [mutant]'], ['at14a null mutant cells [mutant]'], ['atfolt1 null mutant [mutant]']]}\n",
      "-------------------\n",
      "{'112262': [['dsDNA sequences [gene]'], ['**long strands of DNA [metabolite]**', 'long thread of DNA [metabolite]'], ['different strands [genomic feature]'], ['Mitochondrial genome [nucleic acid structure]']]}\n",
      "-------------------\n",
      "{'18335': [['highest rate of embryogenic callus induction [phenotype]', '**high efficiency of EC production [phenotype]**']]}\n",
      "-------------------\n",
      "{'71717': [['intracellular CLCs [protein]'], ['CLCs [protein family]'], ['CCC protein(s) [protein]'], ['chloride ion-binding protein(s) [protein]']]}\n",
      "-------------------\n",
      "{'79400': [['tRNA-dependent amidotransferase [enzyme]'], ['Pretransfer editing [process]'], ['Posttransfer editing [process]'], ['tRNA-SPECIFIC METHYLTRANSFERASE 2A [enzyme]'], ['tRNA-SPECIFIC METHYLTRANSFERASE 2B [enzyme]'], ['tRNA nucleotidyltransferase [protein]'], [\"functional 2'-phosphotransferase [enzyme]\"], ['mRNA triphosphatase [enzyme]'], ['**tRNA methyltransferase [enzyme]**', 'a tRNA methyltransferase [enzyme]'], ['nucleotidyltransferase (MUT68) [enzyme]'], ['Trm11p catalytic activity [gene]'], ['tRNA maturase [enzyme]'], ['trans-editing activity [process]'], [\"active site that catalyzes 5' pre-tRNA cleavage [process]\"], ['catalyzing the peptidyl transferase reaction [process]'], ['tRNA 5-methylcytosine methyltransferase [enzyme]'], ['tRNA m1A methyltransferase [enzyme]'], ['tRNA(m(2)(2)G(26))dimethyltransferase [protein]'], ['psi35 synthase [enzyme]']]}\n",
      "-------------------\n",
      "{'42070': [['cml9-a [mutant]'], ['cml9-b [mutant]'], ['HTR4-HTR5-double-knockout-HTR8-RNAi lines [mutant]'], ['hy5-ks50 mutant lines [mutant]'], ['s8h mutant line [mutant]'], ['proscoop10 mutants [mutant]'], ['EHH9 [mutant]'], ['EHH69 [mutant]'], ['EQO124 [mutant]'], ['EVM265 [mutant]'], ['hei10-3 [mutant]'], ['hei10-4 [mutant]'], ['hei10-5 [mutant]'], ['HTA9-1 [mutant]'], ['NCED9 [mutant]'], ['nhx6 single-knockout line [mutant]'], ['EH14-6 [mutant]'], ['AR-CHI-9 [mutant]'], ['hta8-1 [mutant]']]}\n",
      "-------------------\n",
      "{'5210': [['Priming [phenotype]'], ['Priming phase [phenotype]'], ['Priming memory [phenotype]'], ['**priming phenomenon [phenotype]**', 'priming phenomena [phenotype]'], ['**stress-induced plant(s) priming [phenotype]**', 'plant(s) stress priming [phenotype]', 'facilitating plant(s) stress priming [phenotype]'], ['**primed plant(s) [phenotype]**', 'a primed state in plant(s) [phenotype]'], ['physiological priming [phenotype]'], ['priming process [phenotype]'], ['anticipation of stress [phenotype]'], ['priming resistance [phenotype]'], ['priming effect hypothesis [phenotype]'], ['metabolic priming [phenotype]'], ['**primable [phenotype]**', 'primability of plant(s) [phenotype]'], ['strongest primable [phenotype]'], ['priming response [phenotype]'], ['Priming of stress response [phenotype]'], ['**stress-primed state [phenotype]**', 'state of stress-priming [phenotype]'], ['stress-priming compound [phenotype]'], ['favourable pre-alert state [phenotype]'], ['plant(s) pre-conditioning [phenotype]'], ['pre-sensitized state [phenotype]'], ['plant(s) for future stresses [phenotype]'], ['non-priming phenotype [phenotype]'], ['primed + triggered plant(s) [phenotype]'], ['triggered-only plant(s) [phenotype]'], ['long-term priming [phenotype]'], ['priming in plant(s) [phenotype]'], ['post-challenge primed state [phenotype]'], ['constitutively primed [phenotype]'], ['priming and resistance to stresses [phenotype]'], ['prestress state [phenotype]'], ['early priming and preconditioning stage [phenotype]']]}\n",
      "-------------------\n",
      "{'104675': [['prokaryotic defense system [organism]'], ['rRNA in archeae [organism]'], ['genetic system of eubacteria [organism]'], ['eubacterial chromosomes [organism]']]}\n",
      "-------------------\n",
      "{'101085': [['divergent accessions [organism]'], ['patterns of sequence diversity within and among A. lyrata populations [organism]'], ['**divergent A. thaliana accession [organism]**', 'divergent A. thaliana accessions [organism]'], ['most divergent A. thaliana strains [organism]']]}\n",
      "-------------------\n",
      "{'85478': [['**nitrate medium [treatment]**', 'NO3- medium [treatment]', 'nitrate-containing medium [treatment]', 'medium containing nitrate [treatment]', 'Nitrate-sufficient growth medium [treatment]', 'N medium [treatment]', 'medium supplied with NO(3)(-) [treatment]'], ['pre-culture in nitrate medium [treatment]'], ['**High NO3- medium [treatment]**', 'NO3--rich medium [treatment]'], ['medium NO3- concentration [treatment]'], ['**agar plate supplemented with nitrate [treatment]**', 'agar plate containing nitrate [treatment]'], ['media containing different NO3- concentrations [treatment]'], ['medium with salt and high NO3- content [treatment]'], ['NO2-Tyr-containing medium [treatment]'], ['NO3--free medium [treatment]'], ['low nitrate MS [treatment]'], ['low nitrate plates [treatment]'], ['low nitrate medium [treatment]'], ['5 mM NO3- agar medium [treatment]'], ['medium containing 10 mM, 3 mM or 1 mM NO3- [treatment]'], ['nutrient solution containing NO3- and reduced N [treatment]'], ['media with nitrate as the sole nitrogen source [treatment]'], ['NO3(-)-free, N-limited medium [treatment]']]}\n",
      "-------------------\n",
      "{'29126': [['retromer function [process]'], ['**membrane recruitment of core retromer complex [process]**', 'membrane recruitment of retromer core complex [process]'], ['**endosomal recruitment of retromer core complex [process]**', 'endosomal recruitments of core retromer complex [process]', 'endosomal recruitment of retromer complex [process]', 'endosomal recruitment of core retromer [process]', 'recruitment of the plant(s) retromer to endosomes [process]'], ['**retromer-dependent endosomal trafficking pathways [process]**', 'endosomal trafficking pathways mediated by retromer [process]']]}\n",
      "-------------------\n",
      "{'3539': [['pro-survival strategy [phenotype]', 'survival strategy of plant(s) [phenotype]', 'effective survival strategies [phenotype]', 'powerful survival strategy [phenotype]', '**survival strategy [phenotype]**', 'essential survival strategy [phenotype]', 'survival strategy against environmental changes [phenotype]'], ['a selective strategy for species reproduction and transmission [phenotype]'], ['recently derived adaptation program of land plant(s) [phenotype]'], ['novel solutions to challenges in the environment [phenotype]'], ['diverse environmental adaptation strategies [phenotype]']]}\n",
      "-------------------\n",
      "{'173797': [['reduction of unsaturated fatty acids [metabolite]'], ['fatty acid desaturation reactions [metabolite]', '**fatty acid desaturation [metabolite]**', 'desaturation reactions [metabolite]', 'Desaturation of fatty acids [metabolite]', 'FA desaturation [metabolite]'], ['First step in the synthesis of unsaturated fatty acids [metabolite]'], ['sphingolipid fatty acid desaturation [metabolite]'], ['desaturation of palmitoyl-ACP to generate palmitoleic acid [metabolite]'], ['desaturation of monounsaturated FAs [metabolite]'], ['**introduction of a double bond [metabolite]**', 'double bonds into fatty acids [metabolite]'], ['degradation of cis-unsaturated fatty acids [metabolite]'], ['desaturation of acyl-CoA fatty acids (FAs) [metabolite]'], ['desaturation and hydroxylation [metabolite]'], ['desaturation activity [metabolite]'], ['desaturation of lipid-linked dienoic fatty acids [metabolite]'], ['desaturation of dienoic fatty acids [metabolite]'], ['desaturation of dienoic fatty acids to trienoic fatty acids [metabolite]'], ['desaturate VLCFAs [metabolite]'], ['**Desaturation of fatty acids [process]**', 'Fatty acid (FA) desaturation [process]', 'FA desaturation [process]', 'desaturation of FAs [process]', 'Desaturation reaction [process]', 'desaturation [process]'], ['desaturation of acyl groups [process]'], ['desaturation of FAs [phenotype]', '**desaturase reaction [phenotype]**', 'desaturase reactions [phenotype]'], ['insertion of double bonds into pre-formed fatty acid chains [reaction]'], ['introducing double bonds [chemical reaction]'], ['desaturation [metabolic process]'], ['lipid desaturation [metabolic pathway]'], ['desaturation [subpathway]']]}\n",
      "-------------------\n",
      "{'138140': [['**BIK1 expression [gene]**', 'expression level of BIK1 [gene]'], ['basal expression levels of CYP83B1 [gene]'], ['BAM1 expression [gene expression]'], ['protein(s) level of BES1 [gene]'], ['**protein(s) level of BZR1 [gene]**', 'AtBZR1 abundance [phenotype]', 'BZR1 levels [phenotype]'], ['BES1 expressions [gene]', '**BES1 expression [gene]**'], ['Expression of BZR1 [gene]'], ['Expression of bzr1-1D [gene]'], ['expression level of BcGR1.1 [gene]'], ['expressions of BAS1 [gene]'], ['BEN1 expression [gene]'], ['Strong BnTTG1 expression [gene]'], ['Manipulation of BnTTG1 transcript levels [gene]'], ['spatial expression profiles of BES1s [gene]', '**expression pattern of BES1 [gene]**'], ['BFN1 expression [phenotype]'], ['Early expression of BAC1 [gene]'], ['expression patterns of BUD1 [gene]'], ['BnSCT1 expression [gene expression]'], ['expression of BZIP9 [gene]'], ['**BZS1 protein(s) level [gene]**', 'BZS1 level [gene]'], ['BZS1 expression [phenotype]'], ['expression of BFN1 [gene]'], ['expression of BI-1 [gene]'], ['tissue-specific degradation of BZR1 [phenotype]'], ['wild-type level of BSL1 and BSU1 transcripts [phenotype]'], ['expression of BP/KNAT1 [gene]'], ['effect of BZR1 [gene]'], ['stability of BZR1 protein(s) [phenotype]'], ['expression of EBF1 gene(s) [phenotype]'], ['locations of OsBON1 [gene]'], ['BAM1 expression [gene]'], ['CBE1 expression [phenotype]'], ['BZR1 in vivo [gene]'], ['abundance of BAK1 [gene]'], ['BAK1 gene(s) expression [phenotype]'], ['importance of BES1-L [phenotype]'], ['expression of BcIRT1 [gene]'], ['expression of BjNPR1 [gene]']]}\n",
      "-------------------\n",
      "{'63463': [['**photosystem I PSI-K protein(s) [protein]**', 'nuclear-encoded photosystem I (PS I) protein(s) PSI-K [protein]']]}\n",
      "-------------------\n",
      "{'42619': [['loss of ALA3 [gene]'], ['null allele of AtCSLD5 [gene]'], ['Loss of function of A. thaliana microRNA-machinery gene(s) [mutant]'], ['Loss of function of AtAAH [gene]'], ['Depletion of AtPAO3 [mutant]'], ['loss of AtPAO5 function [mutant]'], ['Loss of A. thaliana SC35/SCL protein(s) [protein]'], ['Null alleles of AtBMI1A [gene]'], ['Loss of function mutations in A. thaliana STT3a [mutant]'], ['Loss-of-function mutations to ATROPUS, GFA1/CLOTHO/VAJRA-1/MEE5, and LACHESIS [protein]'], ['Loss of function of AtNOA1 [mutant]'], ['Loss of AtWRI1 [mutant]'], ['Abolition of AtAAP6 function [gene]'], ['AtPAO4 loss-of-function mutants [mutant]'], ['AtCYP18-3 partial loss-of-function alleles [mutant]'], ['deficiency of AtVKOR/LTO1 [gene]'], ['Loss of AtCGLD11 function [mutant]'], ['Deletion of AtRBOHD [mutant]'], ['abrogation of MAPDA [mutant]'], ['loss of function of A. thaliana RP subunits RPN10 [gene]'], ['deficiency of AtDJ1C protein(s) [protein]']]}\n",
      "-------------------\n",
      "{'106019': [['soybean meta-information file [dataset]'], ['Ensembl plant(s) reference set [dataset]'], ['OneKP dataset [treatment]']]}\n",
      "-------------------\n",
      "{'89306': [['Pollen viability analysis [treatment]', '**pollen viability assays [treatment]**', 'pollen viability screening [treatment]', 'pollen viability testing [treatment]', 'resistant to I2-KI staining [treatment]', \"Alexander's stain [treatment]\"], ['in vitro germinated pollen [treatment]', '**in vitro pollen germination assays [treatment]**', 'in vitro pollen germination protocol [treatment]', 'pollen germination assays [treatment]', 'in vitro germination assays [treatment]'], ['inhibition of pollen germination [treatment]'], ['Treatment of pollen grains [treatment]'], ['controlled temperatures [treatment]'], ['Freeze-storing pollen [treatment]'], ['Pollen abortion experiments [treatment]']]}\n",
      "-------------------\n",
      "{'102593': [['**Phytophthora sojae infection [organism]**', 'P. sojae infection [organism]'], ['**P. viticola populations in Japan [organism]**', 'Japanese P. viticola populations [organism]'], ['Vector psyllids [organism]'], ['P. sojae zoospores [organism]'], ['CaPM phytoplasma [organism]'], ['Phytophthora culture [organism]'], ['sequence from Phytophthora capsici [organism]'], ['P. sojae arsenal [organism]'], ['four species of Phytophthora [organism]'], ['phytoplasma level [organism]'], ['**Phytoplasma infection [organism]**', 'Infection of phytoplasma [organism]'], ['severe human malaria [disease]'], ['New races of Phytophthora pathogens [organism]'], ['Candidatus Phytoplasma aurantifolia [organism]'], ['Typical insect vectors [organism]'], ['No infection by Phytophthora species [organism]'], ['New hosts for HLVd [pathogen]'], ['Phytophthora genome [organism]'], ['stolbur phytoplasma (isolate PO) [organism]'], ['Infection by non-pathogenic Phytophthora sojae [organism]'], ['Phytoplasmas [bacterium]'], ['phytoplasma presence [organism]'], ['compatible P. capsici [organism]'], ['incompatible P. capsici [organism]'], ['P. sojae invasion [organism]'], ['all tested Phytophthora species [organism]'], ['most species in the genus Phytophthora [organism]'], ['all P. sojae pathotypes [organism]'], ['grapevine pathogen [organism]'], ['full aggressive P. capsici [organism]'], ['susceptibility to infection by Phytophthora capsici [organism]'], ['JWB phytoplasma infection [organism]'], ['phytoplasma invasion [organism]'], ['other phytoplasma strains [organism]'], ['phytoplasma transmission [process]'], ['potential vector for P. falciparum [organism]'], ['Ppn race 0 [organism]'], ['virulence of Phytophthora capsici [organism]'], [\"plants' susceptibility to Phytophthora pathogens [organism]\"]]}\n",
      "-------------------\n",
      "{'4955': [['**regulation of plant(s) responses to abiotic stresses [phenotype]**', 'regulating response to multiple abiotic stresses [phenotype]', 'regulating abiotic stress responses [phenotype]', 'regulating responses to several abiotic stresses [phenotype]', 'balancing responses to abiotic stresses [phenotype]', 'regulating plant(s) abiotic stress [phenotype]'], ['regulating plant(s) growth and response to abiotic stress [phenotype]', '**regulating plant(s) growth, development, and response to abiotic stress [phenotype]**', 'regulating plant(s) growth, development, and response to abiotic stresses [phenotype]', 'various biological processes of plant(s) growth, development, and abiotic stress response [phenotype]'], ['metabolic regulation, plant(s) growth, and abiotic stress response [phenotype]'], ['regulating seed germination, root(s) and shoot development, and abiotic stress responses [phenotype]'], ['crucial role in multiple abiotic stress responses [phenotype]'], ['control of light response, hormone response, and abiotic stress [phenotype]'], ['a function in sensing/transducing abiotic stress [phenotype]'], ['mediating biotic stress responses [phenotype]', '**regulation of biotic stress responses [phenotype]**', 'regulation of biotic-stress responses [phenotype]', 'regulation of plant(s) response to biotic stresses [phenotype]', 'regulating responses to biotic stresses [phenotype]', 'regulation of biotic stresses [phenotype]', 'diverse functions in biotic stress responses [phenotype]', 'regulation of plant(s) biotic stress response [phenotype]'], ['regulating biotic stress processes [phenotype]'], ['an important role in response to biotic stresses [phenotype]'], ['modulating biotic stresses [phenotype]'], ['functions in biotic stress situations [phenotype]'], ['biotic stress regulation [phenotype]'], ['regulation of biotic stress [phenotype]'], ['Cl- homeostasis under biotic stress [phenotype]'], ['mediation of plant(s) responses to biotic and abiotic stresses [phenotype]', 'regulating abiotic and biotic stress responses [phenotype]', 'regulating plant(s) response to biotic and abiotic stresses [phenotype]', '**regulation of biotic and abiotic stress responses [phenotype]**', 'regulating the responses of plant(s) to biotic and abiotic stresses [phenotype]', 'regulation of plant(s) biotic and abiotic stress response [phenotype]', 'role against abiotic and biotic stresses [phenotype]', 'biological functions of plant(s) in response to abiotic and biotic stresses [phenotype]', 'regulation of abiotic and biotic responses [phenotype]', 'control of various abiotic and biotic stress responses [phenotype]', 'biological functions related to biotic and abiotic stresses [phenotype]', 'regulation of abiotic and biotic stress [phenotype]', 'mediating plant(s) responses to biotic and abiotic stresses [phenotype]', 'regulation of plant(s) biotic and abiotic stress responses [phenotype]', 'regulation of plant(s) responses to biotic and abiotic stresses [phenotype]'], ['prioritization between biotic and abiotic stress responses [phenotype]'], ['mediating responses to biotic or abiotic stresses [phenotype]'], ['development, metabolism and responses to biotic and abiotic stresses [phenotype]', 'essential roles in plant(s) growth/development and biotic/abiotic stress responses [phenotype]', '**regulation of plant(s) growth, development, and abiotic and biotic stress responses [phenotype]**', 'important roles in biological processes such as plant(s) development, metabolism and abiotic, and biotic stress responses [phenotype]'], ['**regulation of plant(s) tolerance to biotic and abiotic stresses [phenotype]**', 'regulating plant(s) tolerance to biotic and abiotic stresses [phenotype]'], ['**crucial role in response to abiotic and biotic stresses [phenotype]**', 'crucial role in response to biotic and abiotic stressors [phenotype]'], ['transcriptional integration between biotic and abiotic stress [phenotype]'], ['molecular mechanisms responsible for abiotic and biotic stress responses [phenotype]'], ['regulating plant(s) biotic and abiotic stress resistance [phenotype]'], ['cell-death control under biotic and abiotic stresses [phenotype]'], ['production of compounds used in biotic and abiotic stress responses [phenotype]'], ['basal metabolism and plant(s) adaptation to biotic or abiotic stress [phenotype]'], ['balancing leaf age-dependent biotic and abiotic stress responses [phenotype]'], ['regulating variety of abiotic and biotic stresses [phenotype]'], ['dual roles in both abiotic and biotic stress responses in plant(s) [phenotype]'], ['regulating crosstalk between abiotic and biotic stress responses [phenotype]'], ['regulating different biotic and abiotic stress stimuli [phenotype]'], ['governing various defense responses [phenotype]'], ['photosynthetic biology and stress responses [phenotype]']]}\n",
      "-------------------\n",
      "{'93865': [['Root-primordia [cell type]'], ['four-cell-layered primordium [cell type]'], ['two out of three successive merophytes [cell type]'], ['primordials of lateral root(s) [cell type]'], ['primary meristemoids [cell type]'], ['overlying primordia [cell type]'], ['Stages I and II LR primordial initials [cell type]'], ['lateral root(s) primordium endodermal cells [cell type]'], ['maize lateral root(s) primordia [cell type]'], ['L1 cells of all apical meristems and young organ primordia [cell type]']]}\n",
      "-------------------\n",
      "{'40959': [['che-2 cml23-2 cml24-4 mutant [mutant]'], ['figl1 mus81 brca2a-2/brca2a-1 [mutant]'], ['figl1 brca2a-2 brca2b [mutant]'], ['**Far1-2 far4-1 far5-1 [mutant]**', 'far1-2 far4-1 far5-1 mutant [mutant]']]}\n",
      "-------------------\n",
      "{'143138': [['VDOP1 [protein]'], ['VDOF2 [protein]'], ['DOF3.4/OBP1 [gene]']]}\n",
      "-------------------\n",
      "{'114599': [['PIN1 expression patterns [gene]'], ['**expression pattern of MYB gene(s) [gene]**', 'expressions of MYB gene(s) [gene]'], ['Expression patterns of MYB duplicates [gene]']]}\n",
      "-------------------\n",
      "{'65206': [['group A Hsfs [category]'], ['Group C Hsfs [category]'], ['Group B Hsfs [category]'], ['C-I subfamily [gene]'], ['C-II subfamily [gene]'], ['Group Ia members [gene]'], ['Group IIa members [gene]'], ['**Class B Hsfs [protein]**', 'class B Hsf protein(s) [protein]', 'class B heat shock factor protein(s) [protein]'], ['**class A and class C Hsfs [protein]**', 'class A and C Hsfs [protein]'], ['**Class C HSFs [protein]**', 'class C Hsf protein(s) [protein]'], ['class B and C Hsfs [protein]'], ['subclass A2 Hsfs [gene]'], ['HSF classes A and B [gene]'], ['C-type Hsfs [protein clas]'], ['subclasses A9, B3, and B5 [protein]'], ['class I small Hsps [protein]'], ['class II small Hsps [protein]'], ['CI(r) subfamily [phenotype]'], ['C1 and C2 [phenotype]'], ['class A and B cotton Hsfs [protein clas]']]}\n",
      "-------------------\n",
      "{'6596': [['stochastic forces [phenotype]'], ['Hill-Robertson effects [phenomenon]']]}\n",
      "-------------------\n",
      "{'27993': [['reduced quantity of mucilage [phenotype]', 'mucilage recession [phenotype]', 'reduced amount of mucilage [phenotype]', 'further reductions in mucilage [phenotype]', 'very little mucilage [phenotype]', 'reduced levels of mucilage [phenotype]', 'reduced amounts of mucilage [phenotype]', '**mucilage reduction [phenotype]**', 'mucilage reductions [phenotype]'], ['significant reduction of overall mucilage production [phenotype]', '**reduced mucilage production [phenotype]**', 'further reduction in mucilage production [phenotype]'], ['reduction in mucilage biosynthesis [phenotype]'], ['decrease in seed mucilage accumulation [phenotype]', '**reduction in seed mucilage production [phenotype]**', 'reduced seed coat mucilage production [phenotype]'], ['28% reduction in seed mucilage volume [phenotype]'], ['reduction in mucilage extrusion [phenotype]', '**reduced mucilage extrusion [phenotype]**'], ['reduced expression of mucilage-related gene(s) [phenotype]'], ['reduction in total mucilage content [phenotype]'], ['57.5% reduction in mucilage area [phenotype]'], ['2.7% reduction in mucilage area [phenotype]'], ['24% reduction in mucilage area [phenotype]'], ['6% reduction in mucilage area [phenotype]'], ['reduction in mucilage content [phenotype]'], ['release of reduced amount of mucilage [phenotype]', '**reduced mucilage release [phenotype]**']]}\n",
      "-------------------\n",
      "{'97374': [['male sterile florets in wheat [organism]'], ['female-fertile alfalfa plant(s) [organism]'], ['male-fertile soybean plant(s) [organism]'], ['male sterility in maize [organism]'], ['male sterility in rice [organism]'], ['male sterility in sorghum [organism]'], ['male sterility in wheat [organism]'], ['female-sterile alfalfa plant(s) [organism]']]}\n",
      "-------------------\n",
      "{'98092': [['white phenotypic fruit of pomegranates [organism]'], [\"pollen of olive tree cultivar 'Picual' [organism]\"], [\"'Piel de Sapo' [organism]\"], ['Spanish Piel de Sapo (PS) cultivar [organism]'], ['Persea americana Mill [organism]'], ['**Capsicum frutescens AC08-045 [organism]**', 'AC08-045 [organism]'], ['**Capsicum frutescens AC08-201 [organism]**', 'White-Fruited Accession AC08-201 [organism]', 'AC08-201 [organism]'], [\"**Capsicum annuum 'Micropep Yellow' [organism]**\", \"C. annuum 'MY' [organism]\"], ['White pepper cultivars [organism]'], ['Habanero White [organism]'], ['Chupetino White [organism]'], ['White Moruga [organism]'], ['Piperaki Long [organism]'], ['Sorghums with a white, lemon-yellow, red, brown, or black pericarp [organism]'], ['wax gourd [organism]'], ['**white-skinned wax gourd [organism]**', 'white-skinned wax gourds [organism]'], ['C. arabica var. Typica [organism]'], ['P. edulis f. flavicarpa [organism]'], ['**red peppers [organism]**', 'Red pepper fruits [organism]'], ['Purple-skinned eggplants [organism]'], ['Green peel eggplant cultivars [organism]'], ['White eggplants [organism]'], ['Gujarat [organism]'], ['orange cultivated [organism]'], ['tubers of potato cv. Desiree [organism]'], ['Bhut jolokia [organism]'], ['pumpkin plant(s) [organism]'], ['purple eggplants [organism]'], ['Blue fluorescence in banana [organism]'], ['Nendran [cultivar]'], ['Malpighia glabra [organism]'], ['green bell fruits of Capsicum annuum var. grossum [organism]'], [\"'Maor' [organism]\"], ['Picual olive cultivars [organism]'], ['cv. Mytilini Round [organism]'], ['carioca beans [organism]'], ['orange cultivars [organ]'], ['purple granadilla [organism]'], ['yellow granadilla [organism]'], ['poplar, tomato, and mango [organism]'], ['Solanum nigrum var. americanum [organism]'], [\"wild-type 'Navelate' fruit [organism]\"], [\"Corylus avellana L. 'Montebello' [organism]\"], ['heart-shaped fruits [phenotype]'], ['virescens fruit [organ]'], [\"A. eriantha 'White' [organism]\"], ['high β-carotene melon variety [organism]'], ['Ribes album L. fruits [organ]']]}\n",
      "-------------------\n",
      "{'61755': [['anti-AtCEP2 peptide antibodies [protein]'], ['anti-AtCEP antibodies [protein]'], ['anti-ETEC antibodies [protein]'], ['**tuberculosis (TB) antigen ESAT-6 [protein]**', '6 kDa tuberculosis antigen [protein]'], ['E2 antibodies [protein]'], ['AtExo70E2 antibodies [antibody]'], ['animal TGase antibodies [protein]'], ['carcinoembryonic antigen (CEA) [antigen]'], ['animal TGase antibodies [antibody]']]}\n",
      "-------------------\n",
      "{'153123': [['NLP7 orthologs [gene]'], ['homologous gene(s) of NRT1.1 [gene]'], ['two NiR orthologues [gene]']]}\n",
      "-------------------\n",
      "{'68999': [['nitrated Y545 [post-translational modification]'], ['aminated Y771 [post-translational modification]']]}\n",
      "-------------------\n",
      "{'166222': [['Starch breakdown [metabolite]'], ['**Starch degradation rate [metabolite]**', 'rate of starch breakdown [metabolite]'], ['increased rate of starch degradation [process]'], ['amylolysis [process]'], ['starch hydrolysis to maltose [metabolic process]'], ['starch degradation rates [process]'], ['starch or carbon degradation into osmolytes [phenotype]'], ['amylolytic degradation [phenotype]'], ['Breakdown of granular starch [metabolite]'], ['regulation of starch degradation [metabolite]']]}\n",
      "-------------------\n",
      "{'135617': [['coordinately regulated gene(s) for enzymes of the mtETC bypass pathway [gene]', '**gene(s) involved in the alternative respiratory pathway [gene]**', 'gene(s) of the alternative respiratory pathway [gene]'], ['Transcription of some soluble electron carriers [gene]'], ['expression of gene(s) involved in electron transport [gene]'], ['expression of mitochondrial genome-encoded gene(s) [gene]'], ['mitochondrial electron transport gene(s) expression [gene]'], ['gene(s) involved in cellular respiration [gene]', 'gene(s) for cellular energy production [gene]', '**gene(s) involved in respiration [gene]**'], ['**gene(s) for mitochondrial electron transport [gene]**', 'mitochondrial electron transport pathway gene(s) [gene]', 'mitochondrial electron transport gene(s) [gene]'], ['gene(s) related to respiratory chain [gene]', '**gene(s) involved in the electron transport chain [gene]**', 'gene(s) related to mitochondrial respiratory chain [gene]', 'gene(s) related to respiratory electron transfer [gene]', 'mitochondria respiratory chain gene(s) [gene]', 'several gene(s) encoding components of the mitochondrial respiratory chain [gene]', 'specific components of respiratory electron transport chains [gene]'], ['**mt-encoded gene(s) [gene]**', 'mitochondrial-encoded gene(s) [gene]'], ['gene(s) involved in electron transport and energy pathways [gene]'], ['gene(s) involved in glycolysis, TCA cycle and the electron transport chain [gene]'], ['mitochondrial electron transport associated gene(s) [process]'], ['cytochrome C biogenesis 382 [gene]']]}\n",
      "-------------------\n",
      "{'91023': [['2 hpi till 2 dpi of SA treatment [treatment]'], ['up to 2 dpi [treatment]'], ['2 dpi or 4 dpi after Foc treatment [time point]'], ['6 dpi after Foc treatment [time point]'], ['2 dpi after Foc treatment [time point]']]}\n",
      "-------------------\n",
      "{'33794': [['**ESCRT-III [subcellular compartment]**', 'ESCRT-III complex [subcellular compartment]', 'ESCRT-III components [subcellular compartment]'], ['VO subcomplex [subcellular compartment]'], ['Vesicular Tubular Complex [organelle]'], ['ERGIC/VTC [subcellular compartment]'], ['tip of a MOP [subcellular compartment]']]}\n",
      "-------------------\n",
      "{'69854': [['AS2-specific amino acid sequences in ZF motif [gene]'], ['conserved Asp-X-X-His (DXXH) motif [gene]'], ['TD-motif [protein motif]'], ['DD-motif [protein motif]'], ['Asn-Tyr-Polar motif [protein motif]'], ['no conserved TD-motif [protein motif]'], ['ND-motif [protein motif]'], ['His/Asn motif [metabolite]'], ['Asn-X-Ser/Thr sequon [metabolite]'], ['His-Phe-Asp (HFD) motif [metabolite]'], ['a conserved sequence containing an aspartic acid-rich consensus motif [metabolite]'], ['Asp-Pro-Ala motif [genetic element]'], ['Pro-Asp-Gly-Asn-Ser-Phe-Asp [motif]'], ['Asp-Pro-Ser-Asn motif [motif]'], ['NPA (Asn-Pro-Ala) motifs [structure]'], ['ND-motif [structure]'], ['DD-motif [structure]'], ['two asparagine-proline-alanine (NPA) amino acid motifs [protein feature]'], ['Glu/Asp acid triad [protein feature]'], ['two Asn-Pro-Ala amino acid motif [phenotype]'], ['Asn-Xaa-Ser/Thr motif [phenotype]'], ['Asn-Gly motif [phenotype]'], ['characteristic aspartate motif [phenotype]'], ['two highly conserved asparagine-proline-alanine (NPA) motifs [protein]'], ['**NPA (Asn-Pro-Ala) sequence [sequence]**', 'signature sequence Asn-Pro-Ala (NPA) [sequence]'], ['NPS (Asn-Pro-Ser) sequence [sequence]'], ['amino acid consensus sequence NxS/T [sequence]'], ['conserved Asn-Pro-Ala motif [amino acid sequence]']]}\n",
      "-------------------\n",
      "{'160824': [['**succinate [metabolite]**', 'Succinic acid [metabolite]'], ['**succinate semialdehyde [metabolite]**', 'SSA intermediate [metabolite]'], ['**Level of succinate [metabolite]**', 'levels of succinate [metabolite]', 'succinate abundance [metabolite]'], ['**Elevation of succinate [metabolite]**', 'upregulated TCA cycle succinate [metabolite]'], ['TCAC intermediate succinate [metabolite]'], ['succinate and alpha-ketoglutarate [metabolite]'], ['intermediate production of argininosuccinate [metabolite]'], ['addition of succinate [metabolite]'], ['succinate derivative of 9 [metabolite]'], ['succinyl-CoA to succinate [metabolite]'], ['20 mM succinate [metabolite]'], ['succinate [treatment]'], ['low succinic acid [metabolite]'], ['GABA to succinate [metabolite]']]}\n",
      "-------------------\n",
      "{'117728': [['Major QTLs [genomic region]'], ['two QTL [genomic region]'], ['BMC159-TBR17 QTLs [gene]'], ['QTL cluster 7 [genomic region]'], ['QTL cluster 8 [genomic region]'], ['second QTL [genetic element]'], ['QTL for fitness and freezing tolerance [genetic locu]'], ['pleiotropic cluster of six QTLs [genomic region]'], ['QTL data from SoyBase [treatment]'], ['Gramene QTL module [database]'], ['region of eam10 QTL [genetic locu]'], ['QTL regions from the Cvi or Kashmir-2 accession [gene]'], ['six pairs of significant epistatic QTL [qtl]'], ['QTL clusters [genomic region]'], ['QTLs on A08 and C07 [genomic region]'], ['quantitative loci [genetic marker]'], ['PRA QTLs [genomic region]'], ['Putative QTL [gene]'], ['regions carrying QTL for oil content in oilseed rape [genomic region]'], ['quantitative trait loci [treatment]'], ['LP-specific QTL regions [genomic region]'], ['OP-specific QTL regions [genomic region]'], ['LP-specific QTL region [genomic region]'], ['OP-specific QTL region [genomic region]'], ['QTL for induced trichome density on chromosomes II, III, IV, and V [genetic marker]'], ['QTL for induced trichome density on chromosomes I, III, and V [genetic marker]'], ['QTL region between KSC138-10 and KSC138-9 [phenotype]'], ['identified QTL regions [genetic region]'], ['other QTL for light traits [genetic locu]'], ['QTL from other studies [genomic region]'], ['QTLs for root(s) system size [gene]'], ['four UV-B-specific QTL [genomic region]'], ['QTL over FLC genomic region [gene]'], ['two QTL regions [genomic region]'], ['QTL identified by Tian et al. [genomic region]'], ['QTL identified by Jia et al. [genomic region]'], ['genomic regions underlying four QTL for branch angle [phenotype]'], ['pigQTLdb [database]'], ['QTLs for leaf color [genetic marker]'], ['QTL clusters for maize kernel oil content [gene]']]}\n",
      "-------------------\n",
      "{'91765': [['**ABA-treated plant(s) [treatment]**', 'plant(s) subjected to ABA [treatment]'], ['ABA-treated Col WT [treatment]'], ['**ABA-treated seedlings [treatment]**', 'ABA treated seedlings [treatment]'], ['ABA-treated conditions [treatment]'], ['AA1-treated plant(s) [treatment]'], ['**ABA-treated group [treatment]**', 'ABA treated groups [treatment]', 'ABA-treated groups [treatment]'], ['ABA-treated fruit [treatment]'], ['Application of (-)-ABA [treatment]'], ['ABA-treated cells [treatment]'], ['**ABA-treated sample [treatment]**', 'ABA-treated samples [treatment]'], ['ABA-treated gametophytes [treatment]'], ['ABAs in WT [treatment]'], ['WT after ABA [treatment]'], ['**ABA-treated leaf [treatment]**', 'leaf treated with ABA [treatment]'], ['wild-type control under ABA [treatment]'], ['leaf of ABA-treated plant(s) [treatment]'], ['tobacco under ABA [treatment]'], ['callus treated with abscisic acid [treatment]'], ['ABA-treated plant(s) [organism]'], ['ABA-treated WT [organism]'], ['ABA-treated pineapples [organism]'], ['ABA-treated pepper leaf [tissue]'], ['ABA-treated leaf [tissue]'], ['ABA-treated seedlings [tissue]'], ['ABA-treated leaf [organ]']]}\n",
      "-------------------\n",
      "{'123962': [['257 candidate target gene(s) [gene]'], ['13,334 gene(s) [gene]'], ['12,130 DE target gene(s) [gene]'], ['206 target gene(s) [gene]'], ['47 potential gene(s) targets [gene]'], ['167 target gene(s) [gene]'], ['338 mRNAs [gene]'], ['173 non-redundant mRNAs [gene]'], ['267 non-redundant mRNAs [gene]'], ['300 target gene(s) [gene]']]}\n",
      "-------------------\n",
      "{'78010': [['N-terminal MYB-interacting region [gene]'], ['**MYB29 non-MYB region [protein domain]**', 'the non-MYB region of MYB29 [protein domain]'], ['the non-MYB region of MYB95 [protein domain]'], ['MYB interaction region (MIR) [protein domain]', 'MYB-interacting region [protein domain]', '**MYB interaction region [protein domain]**'], ['MYB interaction site [protein domain]'], ['MIR motif [protein]'], ['partial MIR domains [protein]']]}\n",
      "-------------------\n",
      "{'133452': [['Thalianol gene(s) cluster [genomic feature]'], ['THAA2 [gene]'], ['Thalianol gene(s) cluster [genome]'], ['Compact version of thalianol gene(s) cluster [gene cluster]'], ['thalianol cluster [gene cluster]'], ['tirucalladienol cluster [gene cluster]'], ['The gliotoxin cluster [gene]'], ['TAN2 [gene]'], ['At5g47970 gene(s) [gene]'], ['Xantha-l [gene]'], ['TT10 gene(s) [gene]'], ['TaLOG gene(s) family members [gene]'], ['tds5 [gene]'], ['Bronze-2 gene(s) [gene]'], ['tannin deficient seed 4 [gene]'], ['THAA1 and THAH gene(s) [gene]'], ['THAA1 and THAS gene(s) [gene]'], ['gene(s) THAA2 and THAH [gene]'], ['THAD [gene]'], ['TAN1 locus [gene]'], ['TAN2 locus [gene]'], ['THR1 [gene]'], ['transcription of HEMA1 [gene]']]}\n",
      "-------------------\n",
      "{'161439': [['**phosphorylated intermediates [metabolite]**', 'phosphorylated metabolic intermediates [metabolite]'], ['**phosphoryl groups [metabolite]**', 'phosphoryl residues [metabolite]'], ['phosphorylated metabolites [metabolite]']]}\n",
      "-------------------\n",
      "{'166404': [['**Dihydroquercetin [metabolite]**', 'Dihydroquercetin (DHQ) [metabolite]', 'DHQ [metabolite]'], ['(+)-dihydroquercetin [metabolite]'], ['DPBA-bound quercetin [metabolite]'], ['Q-DPBA fluorescence [phenotype]'], ['D3-quercetin [metabolite]'], ['(+/-)-dihydroquercetin [metabolite]'], ['flavonoids quercetin [compound]'], ['two molecules of dihydroquercetin [metabolite]']]}\n",
      "-------------------\n",
      "{'26539': [['**Biomass reduction [phenotype]**', 'lower biomass reduction rates [phenotype]', 'lower biomass production rates [phenotype]', 'decreased biomass accumulation [phenotype]', 'reductions in biomass production [phenotype]', 'reductions in plant(s) biomass [phenotype]'], ['**decreased leaf biomass [phenotype]**', 'enhanced downregulation of leaf biomass [phenotype]'], ['reduction in plant(s) growth and biomass [phenotype]'], ['reduced productivity [phenotype]'], ['Reduced metabolic rate [phenotype]'], ['low de novo carbohydrate production [phenotype]']]}\n",
      "-------------------\n",
      "{'90992': [['6 h of PEG6000 stress [phenotype]'], ['12 h after PEG 6000 treatment [treatment]'], ['6 h after PEG6000 treatment [treatment]'], ['3 h after PEG treatment [treatment]'], ['12 h after PEG treatment [treatment]'], ['6 h after PEG treatment [treatment]'], ['24 h after PEG treatment [process]'], ['6 h pga [treatment]'], ['FD 6 h [time]'], ['35 s after PEG solution reached cells [phenotype]'], ['16th hpt in self [treatment]']]}\n",
      "-------------------\n",
      "{'59529': [['defense response to Gram-positive bacteria [phenotype]'], ['defense responses against necrotrophic fungi [phenotype]'], ['defense responses to M. oryzae [phenotype]']]}\n",
      "-------------------\n",
      "{'148751': [['about 30% of homoeolog triads [gene]'], ['94 trihelix gene(s) [gene]'], ['Ten wheat trihelix gene(s) [gene]'], ['32 new DREB subfamily members [gene]'], ['323 circRNAs [rna]']]}\n",
      "-------------------\n",
      "{'165549': [['D-glucose conjugate [metabolite]', '**Glucose conjugates [metabolite]**', 'conjugated with glucose [metabolite]', 'glucoconjugates [metabolite]'], ['O-glucosides [metabolite]'], ['Glucosyl ester [metabolite]', 'anthranilate-glucose ester [metabolite]', '**glucose ester [metabolite]**', 'glucose esters [metabolite]', 'O-glucose esters [metabolite]', '1-O-beta-glucose esters [metabolite]', '1-O-glucose esters [metabolite]'], ['Acyl-Glucose [metabolite]'], ['vanillylglucose [metabolite]'], ['16.25 398 383 glucoside [metabolite]'], ['mass of glucose conjugated cyanidin [metabolite]'], ['inactive glucose conjugate [metabolite]'], ['glucan chain association [metabolite]'], ['**glucoside [metabolite]**', 'glucoside species [metabolite]', 'glucoside product [metabolite]'], ['formation of anthocyanin 3,7-diglucosides [metabolite]'], ['malonylglucosides [metabolite]'], ['malonyl glucose [metabolite]'], ['glucoside-conjugated forms [metabolite]'], ['acyl-glucoses as donor molecule [metabolite]'], ['glucosylated form [metabolite]'], ['cinnamoyl- and p-coumaroyl-glucose synthesis [metabolite]'], ['3-malylglucoside [metabolite]'], ['instable aglucones [metabolite]']]}\n",
      "-------------------\n",
      "{'153190': [['OHP subfamily [gene]'], ['DFR subgroup [gene]'], ['PH1704 sub-group [gene]']]}\n",
      "-------------------\n",
      "{'1512': [['Peloric phenotype [phenotype]'], ['plant(s) with defects in flowering [phenotype]'], ['**abnormal flower generation [phenotype]**', 'formation of defective flowers [phenotype]'], ['flowers with many abnormal phenotypes [phenotype]'], ['severe flower phenotype [phenotype]'], ['**transgenic plant(s) with conspicuous flower defects [phenotype]**', 'transgenic flowers with severe phenotype [phenotype]'], ['abnormal flower, fruit and seed development [phenotype]'], ['abnormal development of flower organs [phenotype]'], ['abnormal flower and seed development [phenotype]'], ['abnormal flower bud emergence [phenotype]'], ['reiterative flower-in-flower phenotype [phenotype]'], ['disrupted petal development [phenotype]'], ['reversed order of first flowers and axillary buds [phenotype]'], ['abnormal spacing of flowers in the inflorescence [phenotype]'], ['thousands of years [time]']]}\n",
      "-------------------\n",
      "{'124484': [['Exogenous control RNA [metabolite]'], ['Luciferase control RNA [metabolite]'], ['cRNA [metabolite]'], ['Biotin-labeled copy RNA [metabolite]'], ['substrate variants [metabolite]'], ['wild-type substrates [metabolite]']]}\n",
      "-------------------\n",
      "{'94510': [['initial stage of euphyllas [tissue]'], ['ES [tissue]'], ['**11D euphyllas [tissue]**', '11 day euphyllas [tissue]'], ['S-Dev1 [tissue]'], ['EG stage [tissue]'], ['early tuber development [tissue]'], ['developing shoot apices [tissue]']]}\n",
      "-------------------\n",
      "{'16960': [['induction of P700 oxidation [phenotype]'], ['EBC during germination [phenotype]'], ['YLG-based competition assay [phenotype]'], ['photoconversion of D2 [phenotype]'], ['Cd-induced phytosiderophore release [phenotype]'], ['stimulation of SMT wave [phenotype]'], ['induction levels of luciferase activity [phenotype]'], ['polarization following TVCV infection [phenotype]'], ['effect on luciferase activity [phenotype]'], ['positive and negative cotton effects [phenotype]'], ['luciferase response to Cd [phenotype]'], ['luciferase response to copper [phenotype]'], ['luciferase induction under -Fe conditions [phenotype]'], ['4OH-ICN biosynthesis at last step [phenotype]'], ['changes in Pst-LUX growth [phenotype]'], ['5-hydroxylation activity [phenotype]'], ['systemic luciferase activity [phenotype]'], ['sAPX expression [phenotype]'], ['light-shift-mediated production of H\\\\u2082O\\\\u2082 [phenotype]']]}\n",
      "-------------------\n",
      "{'153122': [['BRIZ orthologs [gene]'], ['Orthologs of MtSERF1 [gene]'], ['AXR4 homologs [gene]'], ['RBV orthologs [gene]'], ['AREB/ABF orthologs [gene]'], ['IRI-like paralogs [gene]'], ['ASR orthologs [gene]'], ['BZR1 orthologous gene(s) [gene]'], ['collinear BZR1 orthologous gene(s) pairs [gene]'], ['homologs of TRX [gene]'], ['two IRE1 orthologs [gene]'], ['orthologs of ARH1 and YAH1 [gene]'], ['orthologs of RECQ4ab [gene]'], ['DWF4 orthologs [gene]'], ['homologous counterparts of BrARFs [gene]'], ['other RecQ-like gene(s) [gene]'], ['LRX homologs [gene]']]}\n",
      "-------------------\n",
      "{'147230': [['**Tolerance-associated gene(s) [gene]**', 'gene(s) implicated in plant(s) stress tolerance [gene]', 'gene(s) related to tolerance [gene]', 'tolerant gene(s) [gene]', 'functional gene(s) associated with stress tolerance [gene]']]}\n",
      "-------------------\n",
      "{'40194': [['OEs seedlings [mutant]'], ['vip mutant seedlings [mutant]'], ['ILL6-OE seedlings [mutant]'], ['hy5 seedlings [mutant]'], ['FLT sku5 seeds [organism]'], ['pkl seeds [mutant]'], ['Seeds from rs 5-1 and 5-2 plant(s) [mutant]'], ['pepr1 pepr2 seedlings [mutant]'], ['keg seedlings [mutant]'], ['PCG/aabb seedlings [mutant]'], ['Seeds of ref6 mutant [mutant]'], ['gcs mutant seedlings [mutant]'], ['U-PEST seedlings [mutant]'], ['HY5ox/pifQein3eil1 seedlings [mutant]'], ['PDH-S seeds [mutant]'], ['anac060 seedlings [organism]'], ['pgr5 mutant seedlings [mutant]'], ['Cvi seedlings [mutant]'], ['rootstocks of A. thaliana and pea SL biosynthetic mutants [mutant]'], ['**seeds of RS5 mutants [mutant]**', 'seeds from RS5 mutants [mutant]'], ['seedlings, mature rosette leaves, flowers, and siliques of ProBrAOP2.1:GUS transgenic plant(s) [mutant]'], ['seeds of the cgl (C5) mutant [mutant]']]}\n",
      "-------------------\n",
      "{'124186': [['miRNA regulated downstream gene(s) [gene]'], ['RNA-mediated gene(s) expression [process]'], ['Post-transcriptional regulators [phenotype]'], ['Post-transcriptional regulators [function]'], ['Post-transcriptional regulators [process]'], ['post-transcriptional regulation of gene(s) expression [gene]'], ['post-transcriptional regulation by microRNAs [gene expression]'], ['post-transcriptional regulators of gene(s) expression [process]'], ['transcriptional control of miRNA-regulated HD-ZIP III protein(s) [gene]'], ['Post-transcriptional regulators [role]'], ['expression of complementary RNAs [gene]']]}\n",
      "-------------------\n",
      "{'99387': [['RP75/59 [organism]'], ['PI 126435 [organism]'], ['17S28 F2 population [organism]'], ['Yellow Pear [organism]'], ['TO-937 [organism]'], ['cv. Aichi First and cv. Ueleie 106WP [organism]'], ['cv. Ueleie 106 WP [organism]'], ['TA503 [organism]']]}\n",
      "-------------------\n",
      "{'146530': [['Sweet potato IbNAC1 [gene]'], ['**StNAC053 [gene]**', 'StNAC053 transcription factor [gene]'], ['IpNAC8 [gene]'], ['IpNAC12 [gene]'], ['IpNAC6 [gene]'], ['IpNAC11 [gene]'], ['IpNAC3 [gene]'], ['IpNAC9 [gene]'], ['IpNAC4 [gene]'], ['IpNAC7 [gene]'], ['IpNAC1 [gene]'], ['StNAC280 [gene]'], ['StNTP1 and StNTP2 [gene]', '**StNTP1/2 [gene]**', 'potato transcription factors StNTP1 and StNTP2 [gene]'], ['StNTP1 [gene]'], ['StNTP2 [gene]'], ['TpNAC gene(s) [gene]'], ['21 TpNACs [gene]'], ['110 NAC gene(s) in Solanum tuberosum [gene]'], ['CDS sequence of potato NAC TFs [gene]'], ['**potato NAC [gene]**', 'potato NACs [gene]']]}\n",
      "-------------------\n",
      "{'151388': [['JcSPL3 [gene]'], ['JcSPL13 [gene]'], ['JcSPL6 [gene]'], ['JcSPL1 [gene]'], ['JcSPL11 [gene]'], ['JcSPL5 [gene]'], ['JcSPL4 [gene]'], ['JcSPL16 [gene]'], ['JcSPL2 [gene]'], ['JcSPL9 [gene]'], ['JcSPL8 [gene]'], ['JcSPL10 [gene]'], ['JcSPL12 [gene]'], ['JcSPL14 [gene]'], ['JcSP12 [gene]'], ['JcSPL7 [gene]']]}\n",
      "-------------------\n",
      "{'61033': [['XA21 protein(s) abundance [phenotype]'], ['1140 protein(s) spots in dry embryos [phenotype]'], ['63 nonadditive protein(s) spots in dry embryos [phenotype]'], ['22% transcription & translation protein(s) in dry embryos [phenotype]'], ['4 protein(s) spots with same expression patterns in dry and 24 h imbibed embryos [phenotype]'], ['68 ERF protein(s) [phenotype]'], ['differentially accumulated protein(s) spots [phenotype]'], ['20 unique autophosphorylation sites [phenotype]']]}\n",
      "-------------------\n",
      "{'129608': [['gene(s) encoding mTERF protein(s) from A. thaliana [gene]'], ['AtERF4 and AtERF7 [gene]'], ['AtERF11 transcription [gene]'], ['58% amino acid identity with AtERF3 [gene]'], ['50% amino acid identity with AtERF7 [gene]'], ['AtERF123~AtERF126 [gene]']]}\n",
      "-------------------\n",
      "{'15366': [['positive gravitropic response [phenotype]', '**positive gravitropism [phenotype]**', 'positively geotropic response [phenotype]', '(positive gravitropism) [phenotype]', 'positively gravitropic [phenotype]', 'positive geotropism [phenotype]'], ['Gravitropism [phenomenon]'], ['**negative gravitropism [phenotype]**', '(negative gravitropism) [phenotype]', 'negative gravitropic reaction [phenotype]'], ['shoot gravity responses [phenotype]', 'shoot gravity response [phenotype]', '**shoot gravitropic response [phenotype]**'], ['positive shoot gravity responses [phenotype]'], ['signs of agravitropism [phenotype]'], ['altered gravitropic shoot bending response [phenotype]', '**altered shoot gravitropism [phenotype]**'], ['negative gravitropism of the shoot [phenotype]', 'negative-gravitropic response of the shoot [phenotype]', '**shoot negative gravitropism [phenotype]**'], ['mild agravitropic response [phenotype]'], ['plant(s) gravitropic response [phenotype]', '**regular gravitropism [phenotype]**'], ['orthogravitropic growth [phenotype]'], ['gravitropic response in shoots and root(s) [phenotype]'], ['positive ortho-gravitropic growth [phenotype]'], ['shoot gravitropic growth [phenotype]'], ['aberrant gravitropic responses [phenotype]'], ['functional rescue of shoot gravitropism phenotypes [phenotype]'], ['positively gravitropic manner [phenotype]'], ['negative root(s) gravitropism [phenotype]'], ['faster and greater gravitropism [phenotype]', '**accelerated gravitropism [phenotype]**'], ['reversed gravitropism [phenotype]'], ['normal positive gravitropism [phenotype]'], ['gravitropic growth responses [phenotype]'], ['negative taproot gravitropism [phenotype]']]}\n",
      "-------------------\n",
      "{'160722': [['D-Lactate [metabolite]'], ['D-chiro-inositol [metabolite]'], ['cis (2R,3S) [metabolite]'], ['DMA [treatment]'], [\"2'2-dithiobisethanol [metabolite]\"], ['D-(-)-quinic acid [metabolite]'], ['**butyl iodide [metabolite]**', 'BuI [metabolite]'], ['2-mercaptethanol [treatment]'], ['3-(3,4-dichlorophenyl)-1,1-dimethylurea [metabolite]'], ['DMPI [compound]'], ['dme [metabolite]'], ['demethoxyubiquinone [metabolite]'], ['1,3-propanediole [metabolite]'], ['1,3-dibromopropane [metabolite]'], ['biotin analog (e.g., HPDP-biotin) [metabolite]'], ['trisubstituted glycerol [metabolite]'], ['decylubiquinone [metabolite]'], ['ethyl iodide [metabolite]'], ['propyl iodide [metabolite]'], ['allyl iodide [metabolite]'], ['2-mercaptoethanol [metabolite]'], ['1,2-bis(N,N-dimethylcarbamoyl)hydrazine [metabolite]'], ['reductant supplied by PMDH [metabolite]'], ['MEHP [metabolite]'], ['peracetate adduct [metabolite]'], ['phenethyl alcohol [metabolite]'], ['biotin-cadaverine [metabolite]'], ['dopachrome methyl ester [metabolite]']]}\n",
      "-------------------\n",
      "{'52510': [['21 gene(s) [gene]'], ['21,912 gene(s) [gene]'], ['29,788 gene(s) [gene]'], ['23,764 gene(s) [gene]'], ['21,506 gene(s) [gene]'], ['21,168 gene(s) [gene]'], ['21,288 gene(s) [gene]'], ['21,299 gene(s) [gene]'], ['21,868 gene(s) [gene]'], ['21,447 gene(s) [gene]'], ['21,333 gene(s) [gene]'], ['22,800 gene(s) [gene]'], ['23,156 gene(s) [gene]'], ['21,715 gene(s) [gene]'], ['27,206 gene(s) [gene]'], ['21,798 gene(s) [gene]'], ['21,570 gene(s) [gene]'], ['21,359 gene(s) [gene]'], ['27,420 gene(s) [gene]'], ['twenty-one gene(s) [gene]'], ['23,895 gene(s) [gene]']]}\n",
      "-------------------\n",
      "{'172622': [['TCA Cycle [gene]'], ['TCA cycle flow [metabolic pathway]'], ['TCA Cycle [phenomenon]'], ['mitochondrial tricarboxylic acid (TCA) cycle [process]']]}\n",
      "-------------------\n",
      "{'124639': [['Guide RNAs [treatment]', '**CRISPR guide RNAs [treatment]**', 'guide RNA sequences [treatment]', 'gRNA sequences [treatment]', 'guides [treatment]', 'CRISPR guides [treatment]', 'crRNA guides [treatment]', 'same set of guide RNAs [treatment]'], ['two guide RNAs [treatment]'], ['guide RNA targeting rules [treatment]'], ['single guide RNAs [gene]'], ['**crRNA [gene]**', 'Guide RNAs [gene]', 'crRNA guides [gene]', 'guide crRNA [gene]', 'guides [gene]', 'CRISPR RNAs (crRNAs) [gene]'], ['Chimeric guide RNA (gRNA) [gene]'], ['CasPhi guide RNAs [gene]'], ['multi-guide crRNA [gene]'], ['perfectly complementary multi-guide crRNA [gene]'], ['precursor CRISPR RNAs (crRNA) [gene]'], ['one CRISPR RNA transcript [gene]'], ['crRNA target sequence [gene]'], ['crRNA sequence [gene]'], ['RNA-guided enzymes [gene]'], ['**short guide RNA [rna]**', 'short CRISPR RNA (crRNA) [rna]'], ['uniquely structured CRISPR RNA (crRNA) [rna]'], ['Guide RNAs [rna]', 'SgRNAs [rna]', 'CRISPR RNA (crRNA) [rna]', 'RNA guide for NiR1 [rna]', '**crRNA [rna]**', 'CRISPR RNA [rna]', 'single guide RNA components [rna]'], ['nucleic acid guided nucleases [protein]'], ['RNA-guided nuclease [protein]'], ['CRISPR guide RNA (gRNA) [rna sequence]'], ['crRNA [molecule]'], ['sgRNA primer sequences [genetic material]'], ['spacer [genetic material]'], ['natural occurring single guide RNA [genetic material]'], ['CRISPR RNA (crRNA) sequence [genetic element]'], ['gRNA sequences [metabolite]'], ['RNA-guided endonuclease [enzyme]'], ['RNA-guided DNA endonuclease [tool]']]}\n",
      "-------------------\n",
      "{'25469': [['one-third of that in wild-type seeds [phenotype]', '**one third as many seeds as Col-0 [phenotype]**'], ['lowest expression levels in WT seedlings [phenotype]']]}\n",
      "-------------------\n",
      "{'63985': [['VIPP1 is required for vesicle formation [phenotype]'], ['lower level of TAP38/PPH1 [protein]']]}\n",
      "-------------------\n",
      "{'125670': [['additional genome equivalent [process]'], ['modifying rDNA loci number [process]'], ['unintended amplifications of other genomic regions or sequences from different organisms [process]'], ['additional sites in the genome [process]'], ['mutant mapping [process]'], ['microsatellite origin [genetic process]']]}\n",
      "-------------------\n",
      "{'79230': [['FBA6-GapC complexes [protein complex]'], ['GA proteome [protein complex]'], ['other AGO complexes [protein]'], ['hOGT/UDP-5S-GlcNAc/peptide complex [protein]'], ['O-GlcNAc target overlapping proteomes [protein complex]']]}\n",
      "-------------------\n",
      "{'46941': [['**hes seeds [mutant]**', 'hes seed [mutant]'], ['essp8 seedlings [mutant]'], ['**emb2750 seeds [mutant]**', 'emb2750 mutant seeds [mutant]']]}\n",
      "-------------------\n",
      "{'3910': [['HS signal [phenotype]'], ['Heat stress (HS)-induced oxidative stress [phenotype]'], ['understanding heat stress response (HSR) mechanisms [phenotype]', '**HSR mechanism [phenotype]**', 'studying HSR mechanism [phenotype]', 'multiple molecular mechanisms to activate HSR [phenotype]'], ['**HSR [phenotype]**', 'heat stress response (HSR) system [phenotype]', 'heat-shock responses (HSRs) [phenotype]'], ['cyclopentenone-induced HSR [phenotype]'], ['expression of HS markers [phenotype]'], ['HSR-independent metabolic response [phenotype]'], ['HSR in plant(s) [phenotype]', '**plant(s) HSR [phenotype]**'], ['responses during last 1 h HS period [phenotype]'], ['HsfA1-independent pathway [phenotype]'], ['key role in heat stress response (HSR) [phenotype]'], ['networks within the plant(s) heat stress response (HSR) [phenotype]'], ['amplification of HSRs [phenotype]'], ['regulation of HS memory [phenotype]'], ['HS by reprogramming their transcriptome, proteome, metabolome and lipidome [phenotype]'], ['activators in the HSR [phenotype]'], ['**regulation of HSR [phenotype]**', 'regulating HSR [phenotype]'], ['HSR establishment [phenotype]'], ['genetic HSR responses [phenotype]'], [\"different functions in sepal's HSR [phenotype]\"], ['HSR to mild HS [phenotype]'], ['processes of HSR [phenotype]'], ['HSR in tomato [phenotype]'], ['conserved HS response [phenotype]'], ['HS and combined HS+WS stress [phenotype]'], ['early HSRs [phenotype]'], ['evoke the HSR [phenotype]', '**HSR-inducing activities [phenotype]**'], ['HSR-like response [phenotype]'], ['hs response in vivo [phenotype]']]}\n",
      "-------------------\n",
      "{'72583': [['endocytosis of A. thaliana FLS2 [protein]'], ['Inhibitor-treated suspension-cultured A. thaliana cells [cell type]'], ['A. thaliana PTS2 receptor [protein]'], ['internalization of the A. thaliana PIN2 protein(s) [phenotype]'], ['A. thaliana VIP2 [protein]']]}\n",
      "-------------------\n",
      "{'71933': [['RABA4b [protein]'], ['RABA1d [protein]'], ['RabD2c GTPase [protein]'], ['RAB-A1c GTPase [protein]'], ['RAB-D [protein]'], ['RAB-D2a/ARA5 GTPase [protein]', '**RABD2a [protein]**'], ['RabE1 GTPases [protein]', '**RabE1 [protein]**'], ['**RAB-A2a [protein]**', 'RabA2a GTPase [protein]'], ['RabC2a [protein]'], ['RABE [protein]'], ['Rab GTPases RabA4d [protein]'], ['**RABA2A [protein]**', 'RAB GTPase A2A [protein]'], ['RAB-A2 [protein]'], ['RAB-A2 [subclas]']]}\n",
      "-------------------\n",
      "{'81435': [['FALCON [algorithm]'], ['Consurf analysis [treatment]'], ['Critical Assessment of protein(s) Function Annotation algorithms (CAFA) [event]'], ['CAFA [method]']]}\n",
      "-------------------\n",
      "{'66965': [['MAPK activation [process]'], ['**DFPM-mediated immune MAPK activation [phenotype]**', 'enhanced activation of immune MAP kinase-signaling pathways [phenotype]'], ['protein(s) kinase signalling cascades [phenotype]'], ['PRR-activated MAPKs [protein]'], ['PRR-activated MPK6 [protein]'], ['**enhanced activation of MPK kinases [phenotype]**', 'elevated MPKs activation [phenotype]', 'stronger activation of MAP kinases [phenotype]'], ['activation of mitogen-activated protein(s) (MAP) kinase activity [protein]', 'strong MAPK activation [protein]', '**mitogen-activated protein(s) kinase activation [protein]**'], ['reduced MPK phosphorylation [phenotype]'], ['mitogen-activated protein(s) kinase (MAPK) activity [phenotype]'], ['strong MPK3 activation [phenotype]']]}\n",
      "-------------------\n",
      "{'70356': [['Modules with additional RVDs [protein sequence]'], ['residues in other positions [protein structure]'], ['amino acid residues [protein]'], ['Residues [protein component]'], ['Residues [protein structure]'], ['polymorphic residues [protein]'], ['residues maintaining ordered structure [protein feature]'], ['invariant Arg residue [residue]'], ['movement of residues [protein segment]'], ['flexible residues [protein structure]'], ['Residues [gene structure]'], ['subtle residue motion [protein region]'], ['protein(s) residues [protein]'], ['amino acid residue [gene]'], ['head and tail residues [protein structure]'], ['residues nitrated [modification]'], ['Residues [residue]'], ['Multiple residues [protein domain]'], ['intrinsically disordered residues [protein feature]'], ['amino acid residue side chains [general]'], ['Residues [structure]'], ['amino acid residues [chemical group]'], ['variable-position residues [residue]'], ['Nt residues [position]'], ['natural residues of protein(s) [protein]'], ['Multiple residues [protein complex]'], ['assign residues [method]'], ['dissimilar amino acid residues [protein]'], ['variable residues [structure]'], ['variable residues in positions 7 and 9 [amino acid]'], ['interchanged residues [protein feature]'], ['putative residues [genetic element]'], ['subfamily- or subgroup-specific residues [protein domain]']]}\n",
      "-------------------\n",
      "{'174512': [['36-C SQDG [metabolite]'], ['sulfoquinovosyldiacylglycerols [metabolite]', '**sulfoquinovosyldiacylglycerol [metabolite]**', 'SQDG lipid species [metabolite]', 'SQDGs [metabolite]', 'anionic lipids sulfoquinovosyldiacylglycerol (SQDG) [metabolite]', 'SQDG class [metabolite]', 'sulphoquinovosyldiacylglycerol [metabolite]', 'sulfoquinovosyldiacylglycerol (SODG) [metabolite]'], ['unsaturated lipids in SQDG [metabolite]'], ['Four lipid classes [metabolite]', '**MGDG, DGDG, SQDG, PG [metabolite]**'], ['sulfolipid [metabolite]'], ['a major anionic lipid in chloroplast [organ]']]}\n",
      "-------------------\n",
      "{'77940': [['primase module [protein domain]', '**primase domain [protein domain]**'], ['RNA polymerization domain (RPD) [protein domain]', 'RNA polymerase (RPD) domains [protein domain]', '**RPD [protein domain]**', 'RNAP domain [protein domain]', 'RNAP domain of A. thaliana primase [protein domain]', 'RNA polymerase domain of bacteriophage T7 primase [protein domain]'], ['RNAP and ZFD in RNA primases [protein domain]', '**ZFD and RPD [protein domain]**'], ['ZFD important for template recognition [protein domain]'], ['ZFD important for transfer of the initial diribonucleotide from the RPD [protein domain]'], ['ZFDs subdomain of T7 primase [protein]'], ['ZFD and RPD domains of AtPrimase [protein]'], ['RPD domain [protein]'], ['activity of RNAP domain of AtPrimase [phenotype]']]}\n",
      "-------------------\n",
      "{'14077': [['Cotyledon petiole fusion at seedling stage [phenotype]'], ['**cotyledon petiole fusion [phenotype]**', 'fused cotyledon petiole phenotype [phenotype]', 'fused cotyledon petiole [phenotype]'], ['**cotyledon fusions [phenotype]**', 'dramatically fused cotyledons [phenotype]', 'fused cotyledon phenotype [phenotype]'], ['development of seedlings with fused cotyledons [phenotype]']]}\n",
      "-------------------\n",
      "{'103516': [['cyanobacterial model species [organism]'], ['S. elongatus PCC 7942 [organism]'], ['**S. elongatus PCC 6301 [organism]**', 'freshwater cyanobacterium S. elongatus PCC 6301 [organism]'], ['cyanobacterial PSII models [organism]']]}\n",
      "-------------------\n",
      "{'41778': [['isoprene-emitting A. thaliana [mutant]'], ['Mutant ga20ox1-3 A. thaliana [mutant]'], ['**ga5 mutant of A. thaliana [mutant]**', 'ga5 plant(s) of A. thaliana [mutant]'], ['**ga4 mutant of A. thaliana [mutant]**', 'ga4 plant(s) of A. thaliana [mutant]']]}\n",
      "-------------------\n",
      "{'172446': [['phosphorylation of starch granule surface [process]'], ['phosphorylation of starch [metabolite]'], ['**starch phosphorylation/de-phosphorylation cycle [process]**', 'simultaneous phosphorylation and dephosphorylation of starch [process]']]}\n",
      "-------------------\n",
      "{'48228': [['xlg2 [mutant]'], ['scyl2 mutant [mutant]'], ['lux-2 mutants [mutant]', '**lux-2 [mutant]**'], ['smxl2 single mutant [mutant]', '**smxl2 [mutant]**', 'smxl2 mutant [mutant]'], ['cce2 allele [mutant]'], ['**clsy2 mutant [mutant]**', 'clsy2 single mutant [mutant]'], ['CRY2P532L [gene]'], ['cia2-1 [mutant]'], ['cil-2 [mutant]'], ['lox2 [mutant]'], ['cia2 level [mutant]']]}\n",
      "-------------------\n",
      "{'62742': [['GST-WDL3 [treatment]'], ['GST-GhIAR3a [protein]'], ['GST-GhIAR3b [protein]'], ['GST-AtSUMO1-3HA [protein]'], ['GST-AtSUMO2-3HA [protein]'], ['GST-GmSUMO-3HA [protein]'], ['GST-PvSUMO-3HA [protein]'], ['GST-AtSUMO3-3HA [protein]'], ['GST-AtSUMO5-3HA [protein]'], ['GST-HuSUMO1-3HA [protein]'], ['GST-HuSUMO2-3HA [protein]'], ['GST-HuSUMO4-3HA [protein]'], ['GST-Smt3-3HA [protein]']]}\n",
      "-------------------\n",
      "{'93012': [['three drugs [treatment]'], ['combinatorial treatment [treatment]'], ['combination of the two drugs [treatment]'], ['multicomponent therapeutics [method]', '**drug-drug combinations [method]**']]}\n",
      "-------------------\n",
      "{'2712': [['**Valve separation [phenotype]**', 'complete valve separation [phenotype]', 'valve dehiscence [phenotype]', 'separation of the valves [phenotype]', 'dehiscence of valves [phenotype]'], ['valves [phenotype]'], ['proper valve margins development [phenotype]']]}\n",
      "-------------------\n",
      "{'21671': [['accessions from warmer climates [phenotype]'], ['**high latitude [phenotype]**', 'adaptation to higher latitudes [phenotype]', 'when planted at higher latitudes [phenotype]', 'growth in high-latitude environments [phenotype]', 'northernmost area [phenotype]'], ['selection of warm-adapted species [phenotype]'], ['warm-season turfgrass [phenotype]'], ['warmer climates [phenotype]'], ['**warm-growth plant(s) [phenotype]**', 'grow in relatively warm environments [phenotype]', 'plant(s) growing under warm temperature [phenotype]'], ['plant(s) are returned to warm [phenotype]'], ['Warm-acclimated plant(s) [phenotype]'], ['value of plant(s) in warm conditions [phenotype]', '**thermal benefit to plant(s) [phenotype]**', 'positive effects in warmer climates [phenotype]'], ['thermophilic vegetable [phenotype]'], ['extending the growing season [phenotype]'], ['temperature-adapted phenotype [phenotype]'], ['equator to Arctic Circle [phenotype]'], ['tropical/sub-tropical crop [phenotype]'], ['wheat accessions from warmer climates [phenotype]'], ['constitutive warm temperature phenotypes [phenotype]'], ['adaptation to warmer climates [phenotype]'], ['warmer southern latitudes [location]'], ['cool season turfgrass [phenotype]']]}\n",
      "-------------------\n",
      "{'135336': [['AP endonuclease activity of ScAPN2 [gene]'], ['CcKTN [gene]'], ['Microtubule-severing enzyme ATKATANIN1 [gene]'], ['apyrase MIG-23 from C. elegans [gene]'], ['C. sinensis PAT1 [gene]'], ['PDE166 [gene]'], ['enzyme encoded by LOC606473 [enzyme]'], ['KTI family [gene]'], ['capping enzyme [gene]'], ['PtKCS21-283 [gene]'], ['PIMT2 isoforms [gene]'], ['PtdKTI5 [gene]'], ['One subclade of monocot catalases [gene]'], ['small subunit of Colocasia esculenta AGPase [gene]'], ['AGPase small subunit of Theobroma cacao [gene]'], ['PtKCS11-260 [gene]']]}\n",
      "-------------------\n",
      "{'150457': [['Morus001839 [gene]'], ['Morus002012 [gene]'], ['Morus002795 [gene]'], ['Morus016120 [gene]'], ['Morus016684 [gene]'], ['Morus018169 [gene]'], ['Morus020771 [gene]'], ['Morus023144 [gene]'], ['Morus025559 [gene]'], ['MnMADS1 [gene]'], ['c79593_g1 [gene]'], ['c68277_g1 [gene]'], ['c70333_g2 [gene]'], ['c78979_g1 [gene]'], ['c79142_g3 [gene]']]}\n",
      "-------------------\n",
      "{'66974': [['mitogen-activated protein(s) kinase (MAPK) signaling [phenotype]'], ['accumulation of dormant mitogen-activated protein(s) kinases 3 and 6 [process]'], ['MAPK modules [phenotype]']]}\n",
      "-------------------\n",
      "{'9377': [['The F2 population [phenotype]'], ['Majority of loci [phenotype]'], ['tail of the distribution in northern Sweden [phenotype]'], ['10 events [phenotype]'], ['fourth allele [phenotype]'], ['Eight colonies [phenotype]'], ['1891 features [phenotype]'], ['twelve members [phenotype]'], ['sixteen members [phenotype]'], ['35 cases [phenotype]'], ['five times [phenotype]'], ['system size of 100 [phenotype]'], ['system size of 1000 [phenotype]'], ['tenth [phenotype]'], ['six to seven [phenotype]'], ['five or six [phenotype]'], ['eight members [phenotype]'], ['192 [phenotype]'], ['84 members [phenotype]'], ['71 members [phenotype]'], ['29 members [phenotype]'], ['66 members [phenotype]'], ['56 members [phenotype]'], ['86 members in Z. mays [phenotype]'], ['25 members [phenotype]'], ['second most abundant [phenotype]'], ['populations at 5 mM step [phenotype]'], ['populations at 10 mM step [phenotype]'], ['0 or 1 [phenotype]'], ['an output of 1 [phenotype]'], ['8 billion [phenotype]'], ['x=7 [phenotype]'], ['178 SNPs [phenotype]'], ['~10,000 mutations [phenotype]'], ['close to 10,000 mutations [phenotype]'], ['first time proportion reached 50% [phenotype]'], ['three of four lines [phenotype]'], ['majority [phenotype]'], ['twelve [phenotype]'], ['57 in number [phenotype]'], ['crown age of 26 kya [phenotype]'], ['four [phenotype]'], ['10 mM [phenotype]'], ['about 19 000 years old [phenotype]'], ['four of five groups [phenotype]'], ['107 continuous and dichotomous phenotypes [phenotype]'], ['seven to 10 [phenotype]'], ['eight samples [phenotype]'], ['3 out of 75 meiocytes [phenotype]'], ['100% homozygous [phenotype]'], ['76 members [phenotype]'], ['41 members [phenotype]'], ['39 members [phenotype]'], ['9 billions [phenotype]'], ['seven [phenotype]'], ['Tk = 10 data points [phenotype]'], ['more than two billion people worldwide [phenotype]'], ['arabic numeral suffix [phenotype]'], ['31st generation and ancestor generation [phenotype]'], ['epiRIL population [phenotype]'], ['>20,000 experimental individuals [phenotype]'], ['120 accessions [phenotype]'], ['100% population in a single state [phenotype]'], ['50% within the next 35 years [phenotype]'], ['chartreuse bulk [phenotype]'], ['yellow bulk [phenotype]']]}\n",
      "-------------------\n",
      "{'166178': [['presence of Lugol-stained starch granules [metabolite]'], ['multiple starch grains [metabolite]'], ['visible starch grains [metabolite]'], ['loosely packed starch granules [metabolite]']]}\n",
      "-------------------\n",
      "{'95782': [['**Floral organ primordia [organ]**', 'floral primordia [organ]'], ['development of inflorescence primordia [organ]'], ['first set of primordia [organ]'], ['central organ primordia [organ]'], ['functional floral primordia [organ]'], ['floral primordia and developing flowers [organ]'], ['specification of floral organ primordia [organ]'], ['all floral organ primordia [organ]'], ['gynoecium primordia developing on floral meristem platform [organ]'], ['correct differentiation of organs from common primordia [organ]'], ['floral primordia [cell type]', '**Floral organ primordia [cell type]**'], ['lateral (floral) primordia [cell type]'], ['more floral organ primordia [cell type]'], ['young floral organ primordia [cell type]'], ['**Floral organ primordia formation [phenotype]**', 'floral organ primordial formation [phenotype]', 'formation of floral organ primordia [phenotype]'], ['**Floral organ primordia [phenotype]**', 'floral primordia [phenotype]'], ['further differentiation of inflorescence organ primordia [phenotype]'], ['all the floral organ primordia [phenotype]'], ['formation of floral organ primordia after floral initiation [phenotype]'], ['original floral structure [tissue]'], ['Floral organ primordia [subcellular compartment]'], ['production of floral organ primordia [process]']]}\n",
      "-------------------\n",
      "{'2213': [['FDR-type 2n gametes [phenotype]'], ['**2n-gamete production [phenotype]**', 'controlling diploid gamete formation [phenotype]'], ['the number of DEM2 alleles in the parent plant(s) that undergoes meiosis [phenotype]'], ['the number of functional wild-type DEM2 alleles in both the megagametophyte and in the parent plant(s) that undergoes meiosis to produce the megagametophyte [phenotype]']]}\n",
      "-------------------\n",
      "{'157848': [['Ethylene-dependent Gravitropism-deficient and Yellow-green 1 (EGY1) [gene]'], ['Elong locus [gene]'], ['Arabis EGL3 [gene]'], ['EIN6 ENHANCER [gene]'], ['MHZ6/OsEIL1 [gene]'], ['OsEIL2 [gene]'], ['one classical EXT (MA_74039g0010) [gene]'], ['ENE4 [gene]'], ['ENE3 [gene]'], ['EOL [gene]'], ['GSL.ELF3 [gene]'], ['ELG [gene]'], ['ETHYLENE-INSENSITIVE 3-LIKE 2 [gene]'], ['VR-EIL1 [gene]'], ['interaction of FEI [gene]'], ['ELONGATA3 [gene]'], ['EE3 [gene]'], ['EIN3 LIKE 1 [gene]']]}\n",
      "-------------------\n",
      "{'87401': [['formal mentoring programs [treatment]'], ['mental health initiatives [treatment]'], ['diversity, equity, and inclusion efforts [treatment]'], ['Next Generation Science Standards (NGSS) [treatment]'], ['individual development plans [resource]'], ['design of future programs [treatment]'], ['health-related majors [funding]']]}\n",
      "-------------------\n",
      "{'14243': [['short and curled siliques [phenotype]'], ['abnormally flattened siliques [phenotype]'], ['abnormally shaped siliques [phenotype]']]}\n",
      "-------------------\n",
      "{'7573': [['swelling population [phenotype]', '**population explosion [phenotype]**', 'recent explosion of population [phenotype]', 'Increase in population [phenotype]'], ['Increasing human population [phenotype]', 'Increased human population [phenotype]', '**increase in human population [phenotype]**', 'Rapid increase in human population [phenotype]'], ['increasing due to anthropogenic activities [phenotype]'], ['human demand for settlement [phenotype]'], ['**Increasing human population [phenomenon]**', 'Increases in human population [phenomenon]']]}\n",
      "-------------------\n",
      "{'48479': [['**bak1-5 mutant [mutant]**', 'bak1-5 allele [mutant]', 'bak1-5 mutant allele [mutant]'], ['bzr1-D [mutant]'], ['bos1-1 allele [mutant]'], ['Complementation of Atserk3/bak1-4 mutant [mutant]', '**bak1-4 homozygous mutant [mutant]**'], ['bak1-4 allele [gene]'], ['BAF1 F-box decoy [mutant]'], ['bak1-3 and bak1-5 alleles [mutant]'], ['bzr1-2 [mutant]'], ['bzr1 [mutant]'], ['**bzr1-1D mutation [mutant]**', 'hyperactive bzr1-1D mutation [mutant]', 'bzr1-1D protein(s) [mutant]', 'bzr1-1D gain-of-function mutation [mutant]', 'bzr1-1D allele [mutant]', 'gain-of-function mutant of BZR1 [mutant]'], ['bak1 mutant allele [mutant]'], ['bir1-1 [mutant]'], ['bzr1-1D allele [gene]'], ['gain-of-function mutant of BRS1 [mutant]'], ['bil1-1D/bzr1-1D [mutant]'], ['bak1 bkk1 allele combinations [mutant]'], ['bak1-1 [mutant]'], ['BOS1 [mutant]'], ['gain-of-function allele of bda1 [mutant]'], ['functional mutant allele of BZR1 [gene]'], ['sobir1-13 [mutant]'], ['BAK1 in fls2 mutant [gene]']]}\n",
      "-------------------\n",
      "{'174386': [['knockout of acyl-CoA synthetase [enzyme]'], ['**Long-chain acyl-CoA synthetase [enzyme]**', 'long chain acyl-CoA synthetase [enzyme]'], ['reduced LONG-CHAIN ACYL-COA SYNTHETASE activity [gene]'], ['long-chain acyl-CoA synthetase 1 [protein]'], ['long chain acyl-CoA synthetase [protein]'], ['long-chain fatty acyl-CoA synthetase [enzyme]'], ['an acyl-CoA synthetase [protein]'], ['long-chain acyl-CoA synthetase activity [gene]'], ['one isoform of long-chain acyl CoA synthase [protein]'], ['long-chain acyl-CoA synthetase activity [process]'], ['long-chain acyl-CoA synthase [protein]'], ['**very-long-chain acyl-CoA synthetase activity [enzyme activity]**', 'very-long-chain acyl-coenzyme A (CoA) synthetase activity [enzyme activity]'], ['very long-chain acyl-CoA synthetase [enzyme]'], ['long-chain acyl-CoA synthetase activity [activity]'], ['fatty acyl-CoA synthetase activity [enzyme function]'], ['fatty acyl-CoA synthetase activity [enzyme]']]}\n",
      "-------------------\n",
      "{'150304': [['ZmBx10-ZmBx12 locus [gene]', '**maize ZmBx10/11/12/14 [gene]**', 'maize ZmBx10-ZmBx12 or ZmBx14 [gene]', 'ZmBx10/11/12/14 [gene]'], ['**ZmC1 [gene]**', 'ZmMYB134/C1 [gene]', 'ZmC1 (colorless1) [gene]'], ['**ZmABCB1 [gene]**', 'ZmABCB1/BR2 [gene]'], ['ZmGL1 [gene]'], ['**ZmGL2 [gene]**', 'GLOSSY2 (ZmGL2) [gene]', 'ZmGLOSSY2 (ZmGL2) [gene]'], ['ZmDWF4 [gene]'], ['ZmB [gene]'], ['maize BRONZE2 (BZ2) [gene]', 'ZmBZ2 [gene]', 'BZ2 in maize [gene]', 'BZ2 ortholog in maize [gene]', '**BRONZE2 (BZ2) [gene]**'], ['ZmWRI1a and ZmWRI1b [gene]'], ['Maize ZmWRI1a [gene]'], ['ZjBAM5, ZjBAM6, ZjBAM3, ZjBAM2, and ZjBAM8 [gene]'], ['ZjBAM1, ZjBAM2, ZjBAM5, and ZjBAM6 [gene]'], ['BZU2 [gene]'], ['ZmBt2 [gene]'], ['ZmCER1 [gene]'], ['ZmCER4 [gene]'], ['ZmWSD11 [gene]'], ['CHITINASE B1 (ZmCTB1; Zm00001d025753) [gene]'], ['ZmABA3 [gene]'], ['ZmABCB2, ZmABCB10_1, and ZmABCB10_2 [gene]'], ['BZR1-BAMs [gene]'], ['bx2 [gene]'], ['bx3 [gene]'], ['bx4 [gene]'], ['bx6 [gene]'], ['bx9 [gene]'], ['bz1 [gene]'], ['maize homoeologs b1 (Booster 1; Zm00001d000236) [gene]'], ['ZmMAX1b [gene]'], ['ZmMAX1c [gene]'], ['ZmMAX1a [gene]'], ['Zmvt2 [gene]'], ['ZmABCB10-1 [gene]'], ['ZmABCB2-1 [gene]'], ['ZmABCB10-2 [gene]'], ['ZmABCC4 [gene]'], ['ZmMYB94 gene(s) [gene]'], ['ZmMYB-IF35 [gene]'], ['ZmOPR2 and ZmLOX10 [gene]'], ['BA1 (ZmbHLH85) [gene]'], ['ZmKS (ZmbHLH84) [gene]'], ['Maize ZmLOX1 [gene]', '**ZmLox1 [gene]**'], ['ZmBRCA2 [gene]'], ['ZmSLAH2 [gene]'], ['ZmMYB81 [gene]'], ['ZmBx7 [gene]'], ['ZmKCAS-4 [gene]'], ['ZmCER8 [gene]'], ['ZmABC2 [gene]'], ['1228-bp ZmCYP97C19 DNA sequence [gene]'], ['SbCAD2 and ZmCAD2 [gene]'], ['BZR-BAMs [gene]'], ['ZmALS1 [gene]'], ['ZmALS2 [gene]'], ['ZmCER3.2 [gene]'], ['ZmOPR1-3 [gene]'], ['ZmABI4 gene(s) [gene]'], ['ZmABCB10 [gene]'], ['ZmphyB [gene]']]}\n",
      "-------------------\n",
      "{'64017': [['**cyanobacterial protein(s) [protein]**', 'protein(s) found in cyanobacteria [protein]'], ['protein(s) products of PhANGs [protein]'], ['chlorophyte protein(s) sequences [gene]'], ['heterologous photosynthetic protein(s) [protein]'], ['One-helix protein(s) (OHPs) [protein]'], ['cyanobacterial orthologues [protein]'], ['GreenCut2 protein(s) [protein]'], ['Micromonas protein(s) [protein]'], ['leaf microsomal protein(s) [protein]'], ['GreenCut [protein grouping]'], ['protein(s) in glaucophyte alga Cyanophora paradoxa [organism]'], ['protein(s) associated with cyanobacteria [organism]'], ['protein(s) from cyanobacteria [organism]', '**cyanobacterial protein(s) [organism]**', 'cyanobacterial origin of the protein(s) [organism]'], ['a protein(s) that is conserved in photoautotrophic organisms [organism]'], ['conserved protein(s) in photoautotrophic plant(s) [protein]'], ['cyanobacterial division protein(s) [protein]'], ['half of leaf protein(s) [organ]'], ['chlorophyte specific surfaces [characteristic]'], ['chlorophyte protein(s) [protein]'], ['streptophyte protein(s) [protein]'], ['the most abundant plant(s) leaf protein(s) [protein]', '**the most abundant leaf protein(s) [protein]**'], ['protein(s) of a green alga (Chlorophyta) [organism]'], ['protein(s) of cyanobacterium Microcoleus sp PCC 7113 [organism]'], ['protein(s) from three cyanobacterial genomes [organism]']]}\n",
      "-------------------\n",
      "{'166780': [['**Luteolin-6-C-glucoside [metabolite]**', 'luteolin-6-C-glucoside (Isoorientin) [metabolite]'], ['luteolin 7-O-ss-d-glucoside [metabolite]'], ['metabolite c (glucoraphanin) [metabolite]'], ['p-coumaroyl-glucoside [metabolite]'], ['(R)-Glabridin [metabolite]'], ['D3-sakuranetin [metabolite]'], ['formononetin 7-O-glucoside [metabolite]'], ['luteolin 7-glucuronide [metabolite]'], ['glucosylated flavonoids [metabolite]'], ['C-Glycosylflavones [metabolite]'], ['chrysophanol-1-O-beta-d-glucopyranoside [metabolite]'], ['extracted Kiwi fruit glucomannan [metabolite]'], [\"( - )-maackiain-3-O-glucosyl-6''-O-malonate [metabolite]\"], ['8-C-glucosylnaringenin [metabolite]'], ['diosmetin-7-O-glucoside [metabolite]'], [\"luteolin 5', 7-O-glucoside [metabolite]\"], ['isoflavone-7-glucoside [metabolite]'], ['C-glucosyl quinochalcone [metabolite]'], ['C-glucosyl-2-hydroxyflavanones [metabolite]'], ['6C-glucosylflavones [metabolite]'], ['C-glycosylluteolin [metabolite]'], ['C-glycosylchrysoeriol [metabolite]'], ['curcumin glucosides [metabolite]'], ['luteolin glycosylation [metabolite]'], ['luteolin\\u200b-7\\u200b-O\\u200b-β\\u200b-d\\u200b-glucoside [metabolite]']]}\n",
      "-------------------\n",
      "{'121095': [['modifying FIL [gene]'], ['In planta analysis of modified gene(s) [gene]'], ['transformation with NLPPya or NLPPp [gene]'], ['gene(s) modification in higher plant(s) [process]'], ['cisgenically [process]'], ['in planta gene(s) modification studies [genetic process]']]}\n",
      "-------------------\n",
      "{'2505': [['unique evolutionary position between bryophytes and seed plant(s) [phenotype]'], ['Berrie suggestions [phenotype]'], ['model bryophyte [phenotype]']]}\n",
      "-------------------\n",
      "{'126507': [['147 base pairs of DNA [metabolite]'], ['146-147 bp of DNA [metabolite]'], ['40 bp hairpin construct [metabolite]'], ['488 bp hairpin construct [metabolite]']]}\n",
      "-------------------\n",
      "{'2402': [['The allopolyploid process [phenotype]', '**Allopolyploidization [phenotype]**', 'allopolyploid hybridization [phenotype]'], ['allopolyploid wheat evolution [phenotype]'], ['Allopolyploidization [phenomenon]'], ['alloploidization [phenomenon]'], ['Polyploidization of B. napus [phenomenon]']]}\n",
      "-------------------\n",
      "{'1520': [['**biennial habit [phenotype]**', 'biennial flowering habit [phenotype]', 'biennially [phenotype]', 'biannual [phenotype]'], ['annual-biennial regulation [phenotype]']]}\n",
      "-------------------\n",
      "{'9555': [['**oxidative cell death [phenotype]**', 'apoptosis induced by oxidative stress [phenotype]', 'reactive oxygen-induced cell death [phenotype]', 'oxidative stress-mediated cell death [phenotype]', 'cell death by oxidative stress [phenotype]'], ['oxidative cell death pathway [phenotype]']]}\n",
      "-------------------\n",
      "{'98995': [['generation 5 [organism]'], ['200-Gy irradiated plant(s) [organism]'], ['ion-beam-mutagenized Bu-5 population [organism]'], ['**gamma radiation-treated plant(s) [organism]**', 'γ-irradiated plant(s) [organism]'], ['irradiated seeds [organism]'], ['**Chronically irradiated natural accessions [organism]**', 'natural accessions from radioactively contaminated areas [organism]'], ['Irradiated plant(s) [organism]'], ['irradiated maize [organism]'], ['all additionally irradiated accessions [organism]']]}\n",
      "-------------------\n",
      "{'147170': [['HKT1;1HLS allele [gene]'], ['HKT1;1LLS allele [gene]'], ['HLS-1 allele [gene]'], ['HKT1;1HLS plant(s) [gene]'], ['broadly distributed high leaf Na allele [gene]']]}\n",
      "-------------------\n",
      "{'81158': [['PRR7 proteolysis [process]'], ['PRR degradation [process]'], ['AvrPphB-mediated degradation [process]'], ['**PRR5 degradation [process]**', 'PRR5 proteolysis [process]'], ['increased proteolysis of PHBs [process]'], ['AvrRpt2-induced RIN4 degradation [process]'], ['AvrPtoB-dependent endocytic degradation of CERK1 [process]']]}\n",
      "-------------------\n",
      "{'48286': [['rar1-21 [mutant]'], ['NPR1-3 [mutant]'], ['nsp1-1 allele [mutant]'], ['cue1-1 allele [mutant]'], ['crn-1 allele [mutant]'], ['strong alleles cuc1-1 [mutant]'], ['cas-c1 mutation [mutant]'], ['**rsw1 allele [mutant]**', 'radially swollen 1 (rsw1) allele [mutant]'], ['cas1-1 mutant [mutant]'], ['sunn-1 allele [mutant]'], ['destabilized cpr1 allele [gene]'], ['Kas-1 allele [mutant]'], ['vha-a1-1 (Cas9+) [mutant]'], ['casp1-1/casp3-1 mutant [mutant]'], ['cas1-2 [mutant]'], ['cas1-3 [mutant]'], ['nonconditional mutant allele of RSW1 gene(s) (rsw1-2) [gene]'], ['hypomorphic cas1-1 allele [mutant]'], ['unique gain-of-function mutant suppressor of npr1-1 constitutive 1 (snc1) [mutant]'], ['reported spontaneous origin of the nec1 Parkland allele [mutant]']]}\n",
      "-------------------\n",
      "{'64516': [['Atp11p [protein]'], ['Atp12p [protein]'], ['absence of Atp11 [protein]'], ['AtOTC [protein]'], ['ATPA peptide [protein]'], ['AtpA [mitochondrial targeting peptide]'], ['atwhy2 [protein]'], ['AtMia40 [protein]'], ['AtRSH1 protein(s) [protein]'], ['AtTOM40 protein(s) [protein]'], ['ATR2 [protein]'], ['chimeric protein(s) containing amino-terminal part of AtCGL160 and Synechocystis sp. PCC6803 Atp1 [protein]'], ['dimers of AtSATm trimers [enzyme]'], ['Monomeric state of AtPP16-1 [protein]'], ['6 kDa protein(s) (At3g46430) [protein]'], ['identical protein(s) levels of atpB [protein]'], ['biogenesis of atToc75-III [protein]'], ['catalytic efficiency of 15.6 +- 1.2 x 103 M-1sec-1 [phenotype]'], ['1MJH [protein]']]}\n",
      "-------------------\n",
      "{'64471': [['**RuBisCO SSU [protein]**', 'RuBisCO SSU [protein subunit]', 'RuBisCO small subunit protein(s) [protein]'], ['A. thaliana RuBisCO small subunit [protein]'], ['RuBisCO LS [protein subunit]', '**RuBisCO LS [protein]**'], ['15N-labeled RuBisCO large subunit [protein]'], ['RAF2 (RuBisCO assembly factor 2) [protein]'], ['bifunctional RibA protein(s) [protein]'], ['RuBisCO_small domain protein(s) [gene]'], ['RubisCO protein(s) [enzyme]'], ['16 protein(s) subunits [phenotype]'], ['RuBisCO small subunit 1B/3B [protein]'], ['RuBisCo small subunit 2B [protein]'], ['eight small subunits [structure]'], ['RBSC-1 [protein]'], ['RBSC-2A [protein]'], ['RBSC-3 [protein]'], ['RBSC-4 [protein]'], ['three small RuBisCO subunits of A. thaliana (RBCS1B, 2B, 3B) [protein]'], ['54.24 h [phenotype]'], ['activation state of RuBisCO [protein]'], ['the structure of the activated form of spinach RuBisCO [protein]']]}\n",
      "-------------------\n",
      "{'141136': [['pAsr11779 [promoter]'], ['pAsr1491 [gene]'], ['pAsr1755 [gene]'], ['pAsr1994 [gene]'], ['pAsr11779 [gene]'], ['full-length promoter pAsr11875 [gene]'], ['promoters from HCF107 [gene]'], ['promoters from HCF173 [gene]'], ['3405 gene(s) promoters [gene]']]}\n",
      "-------------------\n",
      "{'78873': [['chaperonin 60 subunit beta [protein]'], ['chaperonin 60 subunit beta 2 [protein]'], ['co-chaperonin subunits [protein]']]}\n",
      "-------------------\n",
      "{'172928': [['PA biosynthesis [metabolic process]', 'Polyamine synthesis [process]', '**Polyamine biosynthesis [process]**', 'Polyamine biosynthesis [metabolic process]', 'polyamine (PA) biosynthesis [process]', 'polyamine biosynthetic pathway [process]', 'biosynthesis of polyamines [process]', 'polyamine biosynthesis pathway [process]', 'biosynthesis of polyamine [process]', 'polyamine biosynthetic process [process]'], ['biosynthetic pathways of amines and polyamines [process]'], ['defects of polyamine biosynthesis pathway [process]', '**Dysfunctions of polyamine biosynthesis pathway [phenotype]**', 'polyamine biosynthetic pathway [phenotype]'], ['modification of polyamine synthesis [process]'], ['**Polyamine biosynthesis [metabolic pathway]**', 'Polyamine synthesis [metabolic pathway]', 'polyamine anabolic pathway [metabolic pathway]', 'polyamine biosynthetic pathway [metabolic pathway]'], ['polyamines biosynthesis [pathway]', '**Polyamine biosynthesis [pathway]**'], ['polyamine (PA) biosynthetic pathway [metabolite]', '**polyamine (PA) biosynthesis [metabolite]**', 'polyamine biosynthetic pathway [metabolite]', 'polyamine synthesis pathway [metabolite]', 'polyamine biosynthesis pathway [metabolite]', 'polyamine pathways [metabolite]', 'synthesis side of polyamine metabolism [metabolite]', 'polyamine pathway [metabolite]'], ['polyamine pathway activation [phenotype]'], ['Polyamines and GABA biosynthesis [process]']]}\n",
      "-------------------\n",
      "{'91229': [['40 mM CaCl2 [treatment]'], ['CaCl2 addition [treatment]', '**calcium chloride [treatment]**', 'Addition of CaCl2 [treatment]', 'Incubation with CaCl(2) [treatment]'], ['1 mM KHCO3 [treatment]'], ['addition of 0.2 M CaCl2 [treatment]'], ['2 mM Ca2+ [treatment]'], ['**10 mM CaCl2 [treatment]**', '10 mM CaCl2 treatment [treatment]', 'Superfusion with 10 mM CaCl2 [treatment]'], ['5 mM Ca2+ [treatment]'], ['CaSO4 [treatment]'], ['low (0.5 mM) extracellular Ca(2+) [treatment]'], ['5 mM extracellular Ca(2+) [treatment]'], ['3 mm Ca(2+) [treatment]'], ['Pretreatment with cellulase [treatment]'], ['calcium treatment [treatment]'], ['1 mM Co2+ [treatment]'], ['3-h post-treatment with 0.2mM CaCl2 [treatment]'], ['30 mM CaCl2 [treatment]'], ['Calcium gradient [treatment]', '**Ca gradient [treatment]**'], ['1 mM Ca2+ under drought conditions [treatment]'], ['Ca2+-alginate mixture [treatment]'], ['three different Ca to Mg ratios [treatment]'], ['2mM CaCl2 [treatment]'], ['5 mM CaSO4 [treatment]'], ['200 mM calcium chloride [treatment]'], ['each hour of CaCl2 treatment [treatment]'], ['Ca(2+) of 0.3-3.0 mM [treatment]'], ['1 mM Ca(2+) [treatment]'], ['5 mM Ca(NO3)2 [treatment]'], ['Ca(NO(3))(2) [treatment]'], ['100 microM CaCl(2) [treatment]'], ['with Ca2+ [treatment]'], ['calcium alginate layers [treatment]'], ['transient calcium depletion [treatment]']]}\n",
      "-------------------\n",
      "{'72373': [['**Ammonium transporters [protein]**', 'ammonium transporter protein(s) [protein]', 'ammonium transporters (AMT) [protein]', 'plant(s) ammonium transporters [protein]', 'functional ammonium transporters [protein]', 'ammonium/methylammonium permeases [protein]'], ['NH4+ exporters [protein]'], ['Ammonium transporter-1-like protein(s) [protein]'], ['UMAMIT uniporters [protein]'], ['Ammonium transporters [process]']]}\n",
      "-------------------\n",
      "{'84835': [['**Al stress [treatment]**', 'Al stress conditions [treatment]', 'In the presence of Al3+ [treatment]', 'Al stresses [treatment]', 'aluminum (Al) stress [treatment]', 'aluminium stress [treatment]', 'Al(3+) stress [treatment]', 'Al-stress [treatment]', 'Al stress treatment [treatment]', 'under Al stress [treatment]', 'Aluminum stress conditions [treatment]'], ['**High levels of aluminum [treatment]**', 'High aluminium levels [treatment]'], ['Tolerance assay [treatment]'], ['root(s) treatments with Al [treatment]'], ['exposure to excessive amounts of Al [treatment]'], ['**AlCl3 treatment [treatment]**', 'AlCl3 stress [treatment]', 'AlCl3 treatments [treatment]', 'AlCl<sub>3</sub> [treatment]'], ['**+Al treatment [treatment]**', '+Al stress [treatment]'], ['Al tolerance assessment [treatment]', '**Al tolerance [treatment]**', 'Al stress tolerance [treatment]', 'Al-tolerance [treatment]', 'plant(s) tolerance to Al [treatment]'], ['Al responses [treatment]', 'plant(s) responses to Al [treatment]', 'response to aluminium [treatment]', 'in response to Al stress [treatment]', '**Al stress response [treatment]**', 'response to Al toxicity [treatment]', 'Al response [treatment]', 'Al-stress response [treatment]'], ['Al-induced oxidative stress [treatment]'], ['acidic aluminum stress [treatment]'], ['all tissues with Al treatment [treatment]'], ['**Al and H+ stress [treatment]**', 'H+ and Al stresses [treatment]'], ['low-pH and Al stresses [treatment]'], ['**aluminum stress in root(s) [treatment]**', 'root(s) under Al3+ stress [treatment]'], ['1.5-h exposure to Al [treatment]'], ['5 min exposure to Al [treatment]'], ['12 h after Al treatment [treatment]'], ['2 h under Al stress [treatment]'], ['24 h under Al stress [treatment]'], ['without Aluminum exposure [treatment]'], ['60 muM AlCl3 [treatment]'], ['Treatment with aluminium [process]'], ['root(s) growth during Al stress [process]'], ['**Al3+ stress [process]**', 'Al-stress [process]'], ['Aluminum toxic stress response [phenotype]'], ['individual accessions under Al and proton toxic conditions [phenotype]'], ['mechanisms to cope with Al stress [phenotype]'], ['Al and H+ stress-inducing conditions [phenotype]'], ['aluminum (Al) stress [metabolite]'], ['Aluminum stress [stres]'], ['breeding transgenic crops tolerant to Al stress conditions [organism]'], ['Al tolerance to alfalfa plant(s) [organism]'], ['hybrid Liriodendron tolerance to Al stress [organism]'], ['aluminium-exposed cell suspensions of tobacco [cell type]'], ['**root(s) tips under Al stress [organ]**', 'Al-treated root(s) tips [organ]']]}\n",
      "-------------------\n",
      "{'160119': [['FBH1, 2, 3, 4 [gene]'], ['LRB1 members [gene]']]}\n",
      "-------------------\n",
      "{'40359': [['genetic mutants in model systems [treatment]'], ['**large collection of mutants [mutant]**', 'Thousands of mutant lines [mutant]', 'large number of mutants [mutant]', 'extensive collection of mutants [mutant]'], ['mutant 1 [mutant]'], ['mutant 2 [mutant]'], ['mutant 3 [mutant]'], ['8 mutants [mutant]'], ['Several of these mutants [mutant]'], ['Many more mutants [mutant]'], ['microbial mutants [mutant]'], ['mutants available [mutant]'], ['60,000 mutants [mutant]'], ['one particular mutant [mutant]'], ['stable mutant [mutant]'], ['a range of mutants [mutant]'], ['mutant levels [mutant]'], ['all mutant lines [organism]'], ['plentiful mutant resources [resource]']]}\n",
      "-------------------\n",
      "{'118117': [['Bhaga genome assembly [genomic resource]'], ['GR contigs [genetic element]'], ['**SRR1174256 [genome]**', 'CLC assembly of SRR1174256 [genome]', 'ordered scaffold of barley genome [genome]', 'improved reference genome assembly [genome]', '4.13 Gb genome assembled [genome]', 'ZS11_PB [genome]', 'current barley genome [genome]'], ['Early version genome assembly [treatment]'], ['low sequence alignment against cabbage genome [genomic datum]'], ['GDFQ00000000 [accession number]'], ['comparing the GR contigs to A. thaliana and Vitis vinifera chromosomes [genetic resource]'], ['genome assembled by Buell and colleagues [gene]'], ['**contig_40275 [contig]**', 'contig_1558010 [contig]']]}\n",
      "-------------------\n",
      "{'41639': [['20 A. thaliana lines [mutant]'], ['eleven M<sub>3</sub> A. thaliana lines [organism]']]}\n",
      "-------------------\n",
      "{'63143': [['Holophytochrome [protein]'], ['dichroic phytochrome [protein]'], ['main phytochrome at 16 C [protein]'], ['**light-activated phytochrome [protein]**', 'red-light-activated form of phytochrome [protein]', 'photoactive phytochrome [protein]'], ['active forms of phytochrome receptors [protein]'], ['photoactive holophytochrome [protein]'], ['presence of active phytochrome [protein]'], ['functional phytochrome A [protein]']]}\n",
      "-------------------\n",
      "{'99179': [['Radish genome [genomic region]'], ['**Radish genome [organism]**', 'radish genome assembly [organism]', 'Radish genomes [organism]', 'R. sativus genome [organism]', 'whole radish genome [organism]', 'genome of Raphanus sativus [organism]'], ['Radish genome [gene]'], ['Radish genome [genome]']]}\n",
      "-------------------\n",
      "{'29800': [['**Symmetric division [phenotype]**', 'Symmetric cell divisions [phenotype]', 'symmetric divisions [phenotype]', 'Symmetric cell division [phenotype]', 'symmetry of cell division [phenotype]'], ['symmetric division of the germ cell [cell type]'], ['symmetrical amitoses [phenotype]'], ['correct symmetric division [phenotype]'], ['default symmetric division [phenotype]'], ['symmetric zygotic division [phenotype]'], ['symmetric vertical division [phenotype]'], ['balanced segregation of chromosomes [phenotype]'], ['symmetric proliferative divisions [phenotype]'], ['symmetric cell division of MC [phenotype]'], ['inferred symmetric division [phenotype]'], ['regular, symmetric divisions [phenotype]']]}\n",
      "-------------------\n",
      "{'108622': [['HAT activity [phenotype]'], ['HAC [phenotype]'], ['Hac-regulated candidates [phenotype]'], ['**acetylated state [phenotype]**', 'adequate level of acetylation [phenotype]'], ['histone acetylation at pRD29A-LUC promoter [phenotype]'], ['**histone acetylation levels in coding regions of defense gene(s) [phenotype]**', 'basal histone acetylation levels in coding regions of defense gene(s) [phenotype]'], ['histone acetylation levels in coding/promoter regions of ACS2 [phenotype]'], ['histone acetylation levels in coding/promoter regions of ACS6 [phenotype]'], ['HAC requirement [phenotype]']]}\n",
      "-------------------\n",
      "{'90709': [['SFM [compound]'], ['Imprimatins B [chemical]'], ['Imprimatins B3 [treatment]'], ['Imprimatins B4 [treatment]'], ['Dha-A [treatment]'], ['rEcSAG1m intradermal boost [treatment]'], ['N1-MPP AddaVax group [treatment]'], ['N1-MPP plus AddaVax group [treatment]'], ['Administration of NA vaccine separately [treatment]'], ['Combination of XB and flg22 [treatment]'], ['intranasal administration with combination of HA2-AtCYN and M2eP vaccines [treatment]'], ['immune deficiency (Imd) pathway [treatment]'], ['immunomodulatory effects [treatment]'], ['AddaVax [phenotype]'], ['admixing NA with inactivated monovalent vaccines enhanced responses to NA [phenotype]'], ['Salmonella typhimurium mitogen adjuvant [treatment]'], ['immunoreactive component of ZNC [treatment]']]}\n",
      "-------------------\n",
      "{'38807': [['IbSUT4 transgenic A. thaliana lines [mutant]'], ['CSLD4 transgenic csld4 mutant lines [mutant]'], ['Col0-SB1.7(18) SINE expressing line [mutant]'], ['SB1 expressing lines [mutant]', '**SB1 transgenic A. thaliana [mutant]**'], ['transgenic mips1/GhMIPS1D lines [mutant]'], ['tt12/PvMATE8 lines [mutant]'], ['35S-ipiO1 lines [mutant]'], ['GmSYP24hx lines [mutant]'], ['CkSNAP33 transgenic A. thaliana lines [mutant]'], ['HA-UBC22 lines [mutant]'], ['TH lines [mutant]'], ['VqSTS36 transgenic A. thaliana and tomato lines [mutant]'], ['GmSAP16 transgenic A. thaliana lines [mutant]'], ['IbSAP16-transgenic A. thaliana lines [mutant]'], ['Six transgenic stable A. thaliana lines [mutant]'], ['line CS84812 [mutant]'], ['pip5k-GmPIP5K-2 [mutant]'], ['pip5k-GmPIP5K-6 [mutant]'], ['ipt-161 lines [mutant]']]}\n",
      "-------------------\n",
      "{'160646': [['keto group of isoquinolin-1(2H)-one [metabolite]'], ['acetyl-substituent [metabolite]'], ['C29 homologue [metabolite]'], ['C31 homologue [metabolite]'], ['Methylated acetaminophen [metabolite]'], ['The ethynyl group of EU [metabolite]'], ['Farnesyl group [metabolite]'], ['single methyl group [metabolite]'], ['Acetate groups [metabolite]'], ['mono substituent of chlorine atom at meta position of phenyl ring [metabolite]'], ['tert-butyl group [metabolite]'], ['side-chain R [metabolite]'], ['enyne group [metabolite]'], ['Type of hydroquinone ring substituents [metabolite]'], ['**electron-withdrawing substituent [metabolite]**', 'electron withdrawing group [metabolite]'], ['phenyl substituent [metabolite]'], ['propenyl substituent [metabolite]'], ['reducing end of the tetrasaccharide [metabolite]'], ['C13(2)-carboxymethyl group [metabolite]'], ['additional methoxy group of sinapyl moiety [metabolite]'], ['aryl sulfonyl group [metabolite]'], ['C12 propionate group of BV [metabolite]'], ['ethylidene group [metabolite]'], ['Ethanolamine (Etn) moieties [metabolite]'], ['alkyne group [metabolite]'], ['methyl Cgamma groups [metabolite]'], ['alkyne group of propargylcholine [metabolite]'], ['different number of methyl groups on the D-ring [metabolite]'], ['same sulfonamide functional group [metabolite]'], ['sulfinamide group [metabolite]'], ['four methyl groups [metabolite]'], ['one methylene [metabolite]'], ['three methines [metabolite]'], ['additional tertiary alcohol group [metabolite]'], ['nine methylene groups [metabolite]'], ['t-butyldimethylsilyl (TBS) group [metabolite]'], ['R group length [metabolite]'], ['exocyclic ethano groups [metabolite]'], ['exocyclic etheno groups [metabolite]'], ['N-acetyl group [metabolite]'], ['trifluorinated methyl group [metabolite]'], ['esterified groups [metabolite]'], ['methylene group [metabolite]'], ['methylthio moiety of MTA [metabolite]'], ['keto group of pregnanes [metabolite]'], ['the keto group of pregn-5-anes [metabolite]'], ['N-methyl group of (S)-reticuline [metabolite]'], ['type of modifications of R group [metabolite]'], ['terminal alkyne group of alkynyl carboxamide [metabolite]'], ['methyl group donor [metabolite]'], ['aminopropyl groups for Spd [metabolite]'], ['formyl group of the NCCs [metabolite]'], ['pyrrole-bound ethyl group [metabolite]'], ['diastereotopic protons of methylene group [metabolite]'], ['methyl donor [phenotype]'], ['leaving group of indole-3-acetaldoxime [metabolite]'], ['9-fluorenylmethoxycarbonyl (FMOC) [metabolite]'], ['Cbz protecting groups [metabolite]'], ['azido groups [metabolite]'], ['4-O-tetrahydropyranyl (THP) group [metabolite]'], ['beta-lactone reactive group [metabolite]']]}\n",
      "-------------------\n",
      "{'103854': [['Arxula adeninivorans [organism]'], ['Mrakia gelida [organism]']]}\n",
      "-------------------\n",
      "{'158312': [['LBDs [gene]'], ['LOB DOMAIN-CONTAINING protein(s) 3 (LBD3) [gene]'], ['PtaLBD1 [gene]'], ['Class I LBD gene(s) [gene]'], ['class II LBD gene(s) [gene]'], ['three LBDs [gene]'], ['PpLBDs gene(s) [gene]'], ['LBD factors AtLBD37 [gene]'], ['LBD factors AtLBD39 [gene]'], ['LBD16, LBD17, LBD18, and LBD29 [gene]'], ['LBD18/ASL20 [gene]'], ['Downregulated LBD gene(s) [gene]'], ['LBDs [gene family]'], ['LBD/ASLs [gene]'], ['LBDs expression [gene]'], ['42 LBD/ASL gene(s) [gene]'], ['expression of LBD1 [gene]'], ['functions of LBD gene(s) [gene]'], ['other LBD/ASLs [gene]']]}\n",
      "-------------------\n",
      "{'143774': [['OsbZIP58 [protein]'], ['OsbZIP52/RISBZ5 [gene]'], ['OsbZIP50 [protein]'], ['OsbZIP81 [gene]'], ['RF2a/OsbZIP75 [gene]'], ['Three bZIP TFs OsbZIP71, OsbZIP52, and ZAT10 [gene]'], ['ROS-bZIP1 [gene]'], ['ROS-responsive bZIP [gene]'], ['OsbZIP46 (OsABF2) [protein]', '**OsbZIP46 [protein]**'], ['5 OsbZIPs [gene]'], ['OsbZIP48 from rice [gene]'], ['OsABF2/OsbZIP46 [gene]'], ['OSBZ8 [protein]'], ['OsbZIP48 protein(s) [protein]'], ['**OsbZIP50 transcription factor [gene]**', 'transcription factor OsbZIP50 [gene]'], ['OsbZIP50 mRNA [gene]'], ['ER stress sensor protein(s) OsbZIP39 [gene]'], ['OsbZIP74 protein(s) [protein]'], ['OsbZIP74 expression [gene]'], ['17 OsbZIP protein(s) [protein]'], ['OsbZIP23 [protein]'], ['OsbZIP [gene]'], ['rice OsbZIP [gene]'], ['rice bZIP transcriptional activator RF2b [protein]'], ['responsiveness by OsbZIP [gene]']]}\n",
      "-------------------\n",
      "{'148245': [['OsSIK1 [gene]'], ['OsJAC1 overaccumulation [phenotype]'], ['Anti-apoptosis gene(s) OsSAP and AtBI-1 [gene]'], ['anti-apoptosis gene(s) OsSAP [gene]'], ['OST1 phosphorylation [process]'], ['OSH1 gene(s) (At5g01580) [gene]'], ['OsARAG1 [gene]'], ['OSR1 [protein]'], ['OsAPO2 [gene]'], ['mutation of OsCASP1 gene(s) [gene]'], ['OsAPN [gene]'], ['OsSAP7 [gene]'], ['AtSOAR1 [gene]'], ['OSM1/SYP61 [gene]'], ['MpAsr gene(s) [gene]'], ['OsSAP11 [protein]'], ['S-RBP11 [protein]'], ['OsSARD1 protein(s) [gene]'], ['AtOSA1 protein(s) [protein]'], ['Overexpression of AtOSR2 [gene]'], ['oxidative stress-related ABC1-like protein(s) 1 (Osa1) [protein]'], ['94.8% identity to OsSAPK8 [gene]'], ['OTS1 (OsOTS1) [gene]'], ['significant role of StSOS1s [gene]'], ['senescence-associated gene(s) OsSAP [gene]'], ['endogenous HOS15 protein(s) [protein]']]}\n",
      "-------------------\n",
      "{'107561': [['**RNA-seq analysis [process]**', 'RNAseq profiling [process]', 'RNA-sequencing analysis [process]', 'RNA-Seq data analysis [process]'], ['**RNA-Seq data [process]**', 'RNA-sequencing (RNA-seq) data [process]', 'examining RNA-Seq data [process]', 'RNA-seq expression data [process]', 'RNA-Seq reads [process]', 'RNA-seq samples [process]'], ['**RNA-seq study [process]**', 'RNA-Seq-based study [process]', 'in RNA-Seq-based study [process]'], ['RNA-Seq sequencing [process]'], ['RNAseq analyses [experiment]'], ['mRNA-Seq analysis [metabolite]', 'RNA-Sequencing [metabolite]', '**RNA-seq analysis [metabolite]**'], ['RNA sequencing (RNA-Seq) data [gene]'], ['RNA-Seq reads [genomic element]']]}\n",
      "-------------------\n",
      "{'154282': [['VNIs [gene]'], ['DEGs in HD [gene]'], ['HD-ZIPIII factors [gene]'], ['fused lobes (fdl) gene(s) [gene]'], ['fdl hexosaminidase gene(s) [gene]'], ['GhKNAT7 subgroup members [gene]'], ['Class III HD-ZIP Ginkgo gene(s) [gene]'], ['Scare Family protein(s) 4 (SCAR4) [gene]'], ['Class III HD-ZIP gene(s) [dna sequence]'], ['Hub gene(s) ALG10 [gene]'], ['Drosophila TRGs [gene]'], ['D3/MAX2 gene(s) [gene]'], ['DHN interactions [gene]'], ['DHN gene(s) containing a Y-segment [gene]'], ['Deletions of three zinc finger gene(s) [gene]'], ['HD-ZIPIII adaxially localized protein(s) [gene]'], ['potential interaction network of GH_D02G1759 [gene]'], ['single domain major allergens (SDMA) [gene]'], ['HAM gene(s) function [gene family]'], ['GERMINS [gene family]'], ['HD-Zip III gene(s) [gene clas]'], ['Different HDZ IV gene(s) [gene family]'], ['BIG ARF-GEFs [gene]'], ['HD-Zip II and HD-Zip III gene(s) [gene]'], ['Characterization of the NGA gene(s) family [gene]'], ['HDG protein(s) [gene]'], ['The five IDL gene(s) [gene]'], ['Expression domains of HD-ZIP III gene(s) [gene]'], ['Downstream targets of VND gene(s) [gene family]'], ['first HDG gene(s) [gene]'], ['six-member family [gene family]'], ['functionally unidentified DIR-a gene(s) [gene]'], ['HD-ZIPIII, KANADI, and YABBY gene(s) [gene]'], ['conserved motifs found in BAHD acyltransferases [gene]'], ['VND-family gene(s) [gene]'], ['functional HAE and HSL2 [gene]'], ['GADD gene(s) family members [gene]'], ['degree of disorder of protein(s) encoded by D. grimshawi gene(s) [gene]'], ['HDG4 and -5 [gene]'], ['class III and IV gene(s) [gene]'], ['two other homeologous NAM1 gene(s) [gene]']]}\n",
      "-------------------\n",
      "{'75046': [['ClPIP [protein]'], ['Topology and functional content of MePPI-In [protein complex]'], ['75 kDa MIP protein(s) [protein]']]}\n",
      "-------------------\n",
      "{'89308': [['**gravity vector [treatment]**', 'direction of the gravity vector [treatment]', 'direction of gravity [treatment]', 'vector direction of the g-force [treatment]', 'Gravitational vector [treatment]'], ['**Gravitropism [treatment]**', 'A gravitropic stimulus [treatment]', 'gravi-stimulation [treatment]', 'Re-orientation of A. thaliana seedlings [treatment]', 'initial gravitational cue [treatment]', 'reoriented to induce a tropistic response [treatment]'], ['growth axis orientation relative to gravitational direction [treatment]', '**upright orientation defined by gravity [treatment]**'], ['longitudinal auxin gradient [treatment]'], ['90° gravistimulation [treatment]'], ['stimuli provided by in-flight centrifuge [treatment]'], ['support [treatment]'], ['isotropic growth [treatment]'], ['any orientation [treatment]'], ['180° gravistimulation [treatment]'], ['growth determinant [treatment]'], ['tilt the model [device]'], ['Anisotropy [treatment]']]}\n",
      "-------------------\n",
      "{'112165': [['**Helitron elements [gene]**', 'Helitron family of transposable elements (TEs) [gene]', 'helitron TEs [gene]', 'Helitron conserved signals [gene]'], ['Helitron family DHH00002 [gene]'], ['RC/Helitron TEs [gene]', '**RC/Helitrons [gene]**', 'RC/Helitron class [gene]', 'RC/Helitron [gene]'], ['DNA/Helitron [gene]'], ['**Helitron elements [genomic element]**', 'RC/Helitrons superfamily [genomic element]', 'rolling circle/helitron (RC/Helitron) sequences [genomic element]', 'helitron [genomic element]'], ['insect hAT element Hermes [genomic element]'], ['maize helitrons [dna transposable element]'], ['rice helitrons [dna transposable element]'], ['maize helitrons [genetic element]'], ['rice helitrons [genetic element]'], ['Helitron-type TEs [genetic element]'], ['RC/Helitron class of TEs [genetic element]'], ['**RC/Helitrons [genetic element]**', 'RC/Helitron elements [genetic element]', 'RC/Helitron TEs [genetic element]'], ['Helitron reference sequences [genetic element]'], ['Helitron elements [genetic element]'], ['Helitron insertion [genetic element]'], ['Helitron ATREP15 [genetic element]'], ['full-length Helitron [genetic element]'], ['Helitron and MuDR high-recombination transposons [genetic element]'], ['**RC/Helitron superfamily [genetic element]**', 'RC Helitron superfamily [genetic element]'], ['Helitron fragment (ATREP10D) [genetic element]'], [\"5' RC/Helitron fragment (ATREP1) [genetic element]\"], ['putative transposable elements (Helitron) [genetic element]'], ['Helitrons [chromosomal region]'], ['non-TIR Helitron Tns [genetic entity]'], ['RC/Helitrons [dna sequence]'], ['RC/Helitron superfamily [dna sequence]'], ['**Helitron elements [transposable element]**', 'helitron [transposable element]', 'helitron TE [transposable element]'], ['helitron [element]'], ['Autonomous Helitrons [element]'], ['RC/Helitron elements [phenotype]'], ['HELITRON superfamily [genomic region]'], ['helitron transposable element (TE) [subcellular compartment]'], ['RC/Helitron class [sequence]']]}\n",
      "-------------------\n",
      "{'49728': [['AtNRX1M1 [mutant]'], ['AtNRX1M3 [mutant]'], ['E. coli Trx_C32S [mutant]'], ['single Cys-to-Ser variants of TRX [mutant]'], ['TRXo1 C37S variant [mutant]'], ['TRXo1 C40S variant [mutant]'], ['C99S mutant [mutant]'], ['C106S mutation [mutant]'], ['C109S substitution [mutant]'], ['monocysteinic mutant of TRX y [mutant]'], ['C100S variant [mutant]'], ['C140S variant [mutant]'], ['PrxA C119S mutant [mutant]'], ['replacements (C304S, C307S, and C304S/C307S) [mutant]']]}\n",
      "-------------------\n",
      "{'70628': [['intracellular transporters PHT5 (VPT1) [protein]'], ['**PHT5 family [protein]**', 'PHOSPHATE TRANSPORTER 5 family (PHT5) [protein]'], ['Phosphate Transporter 4; 6 (PHT4;6) [protein]'], ['PHOSPHATE TRANSPORTER 5 family (PHT5) [protein family]'], ['VACUOLAR PHOSPHATE TRANSPORTER (VPT) [protein family]'], ['VACUOLAR PHOSPHATE TRANSPORTER (VPT) [protein]'], ['phosphate transporter5;1-2 (pht5;1-2) [protein]']]}\n",
      "-------------------\n",
      "{'72847': [['inhibitor of apoptosis (IAP) protein(s) family [protein family]', '**IAP protein(s) family [protein family]**'], ['protein(s) with IAA as a prosthetic group [protein]']]}\n",
      "-------------------\n",
      "{'132121': [['**beta amylase [gene]**', 'Beta-amylase [gene]', 'Beta-amylases [gene]', 'beta-AMY [gene]', 'beta-amylase (BMY) [gene]', 'Atbeta-Amy [gene]', 'β-amylases [gene]'], ['**beta-amylase 1 [gene]**', 'beta-amylase1 [gene]', 'beta-amylases 1 [gene]'], ['beta-amylase2 [gene]'], ['**beta-amylase3 [gene]**', 'beta-amylase 3 [gene]', 'beta-amylase 3 (BMY3) [gene]', 'beta-AMYLASE 3 (At4g17090) [gene]', 'beta-amylase (At4g17090) [gene]', 'beta-amylases 3 [gene]', 'betaAMY3 [gene]'], ['Beta-amylase 5 [gene]'], ['beta-amylase 9 [gene]'], ['**thioredoxin-regulated beta-amylase TR-BAMY [gene]**', 'TR-BAMY [gene]'], ['beta-amylase encoding gene(s) [gene]'], ['**beta-amylase 6 [gene]**', 'b-AMY6 [gene]'], ['beta-amylase (BnaC02g12830D) [gene]'], ['beta-AMYLASE (At4g15210) [gene]'], ['barley alpha-amylase mRNA [molecule]'], ['Chloroplast-targeted beta-amylase [gene]'], ['expression of beta-amylase [gene]'], ['beta-amylases (BMY7 and BMY8) [gene]'], ['beta-amylase, BMY7 [gene]'], ['gene(s) associated with beta-amylase activity [phenotype]'], ['expression of the beta-amylase gene, AT-beta-Amy [gene]']]}\n",
      "-------------------\n",
      "{'8072': [['disassembly into smaller foci [phenotype]'], ['Boundary specification [phenotype]'], ['tissue-scale dynamics [phenotype]'], ['tissue formation [phenotype]'], ['inter-tissue organization [phenotype]'], ['engineered tissues [phenotype]'], ['somatic sector formation [phenotype]'], ['spatial restrictions [phenotype]'], ['anatomy [phenotype]'], ['at the surface [phenotype]'], ['tumor biology [phenotype]'], ['the target [phenotype]'], ['regional scale [phenotype]'], ['pattern specification [phenotype]'], ['physical restrictions [phenotype]'], ['organ size homeostasis [phenotype]'], ['whole-body imaging [phenotype]'], ['tumour origin [phenotype]'], ['novel perspective of tissue organization [phenotype]'], ['tissue morphology [phenotype]'], ['tissue dimensions and morphology [phenotype]'], ['boundary maintenance [phenotype]'], ['tissue transformation [phenotype]'], ['highly localized [phenotype]'], ['broad domain boundary specification [phenotype]'], ['refining the boundary region [phenotype]'], ['boundary restriction [phenotype]'], ['organ localization [phenotype]'], ['tissue-scale patterns [phenotype]'], ['similar domains [phenotype]'], ['organ-specific and developmental stage-specific structure [phenotype]'], ['basic body structure [phenotype]'], ['one presumed organ [phenotype]'], ['tissue and organ patterning [phenotype]'], ['formation of discrete domains [phenotype]'], ['remain intact [phenotype]'], ['simple internal structure [phenotype]'], ['tissue specific features [phenotype]'], ['normal development of specialized tissues [phenotype]'], ['tissue context [phenotype]'], ['modification in tissues [phenotype]'], ['mechanical constraints on surrounding tissues [phenotype]'], ['asymmetries in specific organs [phenotype]'], ['boundary to ensure organ separation [phenotype]'], ['organ scale analysis [phenotype]'], ['break down large communities into smaller communities [phenotype]'], ['a large variety of tissues, developmental stages, and environmental conditions [phenotype]'], ['site of origin [phenotype]'], ['anatomical spatial location [phenotype]'], ['three different regions [phenotype]'], ['domain traits [phenotype]'], ['somatically [phenotype]'], ['histology [phenotype]'], ['establishing tissue identity [phenotype]'], ['wild-type tissues [phenotype]'], ['tissue-specific way [phenotype]'], ['organ transplantation [phenotype]'], ['establishing polarity [phenotype]'], ['specification of tissue types in animals [phenotype]']]}\n",
      "-------------------\n",
      "{'75726': [['Helix 8 [protein domain]'], ['**helix 1 [protein domain]**', 'h1 [protein domain]'], ['helix S4 [protein domain]'], ['helix 1 of F-box domain from TIR1 [protein]']]}\n",
      "-------------------\n",
      "{'421': [['bolting plant(s) [phenotype]'], ['speed to reach bolting stage [phenotype]'], ['**post-bolting phase [phenotype]**', 'postbolting plant(s) [phenotype]'], ['third phenotypic class for bolting time [phenotype]'], ['date of bolting [phenotype]']]}\n",
      "-------------------\n",
      "{'136198': [['overexpression of OsASR5 [gene]'], ['Overexpression of TaASR1 [gene]'], ['OsSAP over-expression [treatment]'], ['Overexpression of OsiSAP8 [gene]'], ['Overexpression of SiABI5.12 [gene]'], ['Overexpression of OsPRX38 [gene]'], ['Overexpression of OsCYP18-2 [gene]'], ['Overexpression of OsAIR3 [gene]'], ['OsCTR1 overexpression [gene]']]}\n",
      "-------------------\n",
      "{'103253': [['number of B. graminis colonies [organism]'], ['E. amylovora aggregates [phenotype]'], ['Integrity of bacula [structure]'], ['P. indica network [structure]'], ['growth-retarded small mycelia of Bgt [organism]'], ['normal infection threads [phenotype]'], ['Two early stage P. patens GMCs [cell type]'], ['smaller number of Bgh colonies [organism]'], ['Bgh haustoria establishment [phenotype]'], ['thirteen anastomosis groups [phenotype]'], ['formation of biofilms of A. tumefaciens [process]'], ['embryological studies of B. holboellii [phenotype]'], ['Botrytis hyphal morphology [cell type]'], ['heterotrophic +Pi B. napus suspension cells [cell type]'], ['heterotrophic -Pi B. napus suspension cells [cell type]'], ['Bgt microcolonies [phenotype]'], ['M. truncatula and B. distachyon [organism]'], ['early interaction between D. officinale and Mycena sp. [phenotype]'], ['young fruitlets and sink leaf [organ]']]}\n",
      "-------------------\n",
      "{'71874': [['**Adaptins [protein]**', 'adaptin [protein]'], ['Monomeric adaptors (GGAs) [protein]'], ['**alpha-adaptin [protein]**', 'alpha adaptins [protein]'], ['μ1-Adaptin [protein]'], ['**gamma-adaptin [protein]**', 'gamma-ADR [protein]'], ['Adaptin heterotetramers [protein complex]'], ['adaptor protein-3 delta-adaptin [protein]'], ['adaptor protein-2 alpha-adaptin [protein]'], ['delta-adaptin [protein]']]}\n",
      "-------------------\n",
      "{'145329': [['CYTOKININ RESPONSE FACTOR (CRF) gene(s) [gene]', 'CYTOKININ RESPONSE FACTORS (CRFs) [gene]', '**CYTOKININ RESPONSE FACTOR (CRF) [gene]**', 'CRF group [gene]'], ['**CRFs in Group A [gene]**', 'A clade CRF sequences [gene]'], ['CRF1-CRF8 [gene]'], ['CRF1-6 [gene]'], ['12 members (CRF1-CRF12) [gene]'], ['CRF clade III [gene]'], ['CRF Clade V [gene]'], ['CRF1 [gene]'], ['SlCRF1 [gene]'], ['**CRF3 [gene]**', 'CYTOKININ RESPONSE FACTORS (CRFs) 3 [gene]'], ['BnaA3.CRF3 [gene]'], ['SlCRF3 [gene]'], ['Cytokinin response factors (CRF5) [gene]'], ['CYTOKININ RESPONSE FACTOR 8 (CRF8) [gene]'], ['SlCRF8 [gene]'], ['Cytokinin response factor 4 (CRF4) [gene]', '**CYTOKININ RESPONSE FACTOR (CRF4) [gene]**', 'cytokinin response factor 4 [gene]'], ['root(s) CRF4 [gene]'], ['**AP2 domain transcription factor CYTOKININ RESPONSE FACTOR2 (CRF2) [gene]**', 'Cytokinin Response Factors CRF2/TMO3 (AT4G23750) [gene]'], ['BnaA3.CRF2 [gene]'], ['SlCRF2 [gene]'], ['CRF2a [gene]'], ['cytokinin response factor (CRF6) [gene]'], ['Avr9-Cf9 interaction [phenotype]'], ['CYTOKININ RESPONSE FACTOR 4a (GmCRF4a) [gene]'], ['CYTOKININ RESPONSE FACTORS (CRFs) 7 [gene]'], ['CRF2 protein(s) [protein]']]}\n",
      "-------------------\n",
      "{'82563': [['predicted 13,175,414 PPIs [process]'], ['24,726 PPIs [protein interaction network]'], ['90,370 possible PPIs [process]'], ['343 pPPIs between 179 protein(s) [protein]'], ['386 pPPIs [interaction]'], ['more than 345,000 predicted PPIs [process]'], ['more than 316,000 PPIs [process]'], ['300k novel PPIs [phenotype]']]}\n",
      "-------------------\n",
      "{'44763': [['pgp1pgp19 double mutant root(s) [mutant]'], ['**sqd2 pgp1-1 double mutant [mutant]**', 'sqd2pgp1-1 double mutant [mutant]'], ['**sqd2-2 pgp1-1 double mutant [mutant]**', 'sqd2-2 pgp1-1 [mutant]'], ['**sqd2-2 pgp1-2 double mutant [mutant]**', 'sqd2-2 pgp1-2 [mutant]'], ['sqd1 pgp1-2 [mutant]'], ['sqd1 pgp1-1 [mutant]'], ['double knockout mutant of PGP1 and PGP2 (pgp1-2 pgp2) [mutant]'], ['fpgs1 and fpgs3 double mutant [mutant]'], ['**fpgs1ccoaomt1 [mutant]**', 'double-mutant fpgs1ccoaomt1 [mutant]', 'fpgs1ccoaomt1 mutant [mutant]'], ['pgk1.1 pgk3.2 double mutant [mutant]']]}\n",
      "-------------------\n",
      "{'129486': [['AtLIL3.2 [protein]'], ['AtLLG1 [gene]'], ['AtLLG2 [gene]'], ['AtLLG3 [gene]'], ['AtLOG3 [gene]'], ['AtLOG1 [gene]'], ['**AtLOG gene(s) [gene]**', 'AtLOGs [gene]'], ['structures of AtLOG3 and AtLOG8 [gene]'], ['AtZML1 [gene]'], ['AtZML2 [gene]'], ['AtMLKLs [gene]'], ['three LLG gene(s) (AtLLG1, AtLLG2, and AtLLG3) [gene]'], ['three MLKLs (AtMLKLs) [gene]'], ['102-kD protein, AtLOX2 [protein]']]}\n",
      "-------------------\n",
      "{'92084': [['CK [treatment]', '**Cytokinins [treatment]**'], ['kin [treatment]', '**kinetin (KT) hormone [treatment]**', 'exogenous applications of kinetin [treatment]'], ['applications of kinetin and thidiazuron [treatment]'], ['cytokinin treatment [metabolite]', '**Cytokinin treatments [metabolite]**'], ['thidiazuron (TDZ) [metabolite]']]}\n",
      "-------------------\n",
      "{'37233': [['changes in culture cells [cell type]'], ['Specific cellular environment [cell type]'], ['**changes in cell number [cell type]**', 'effect on cell number [cell type]'], ['cell-specific manner [condition]'], ['Inhibitor-treated cells [cell type]'], ['Cell viability [cell type]'], ['Homogeneity of cell types [cell type]'], ['Functional changes of cells [cell type]'], ['**cell dependent [cell type]**', 'depending on cell type [cell type]'], ['time dependent [cell type]'], ['a tissue-specific manner [tissue]', '**tissue-specific manner [tissue]**'], ['response of a population of cells [cell type]'], ['behavior of cells [cell type]'], ['creation of a specific cellular environment [cell type]'], ['changes at cell level [cell type]'], ['at least one tissue under different stress conditions [tissue]'], ['different cell state [cell type]', '**cell states [cell type]**'], ['outputs depending on respective cell types [cell type]'], ['cell conditions [cell type]'], ['changes in cellular identity [cell type]'], ['cell-type specific level [cell type]'], ['proficiency of cells [cell type]']]}\n",
      "-------------------\n",
      "{'133369': [['plant(s) TPP aptamers [gene]'], ['TPK [gene]'], ['Chlamydomonas TPP riboswitches [phenotype]'], ['**TPP riboswitches [gene]**', 'TPP riboswitch sequence [gene]', 'thiamine-pyrophosphate (TPP) riboswitch [gene]', 'E. coli TPP riboswitch [gene]'], ['thiamine pyrophosphate enzyme [gene]'], ['**TPP biosynthesis gene(s) [gene]**', 'expression of TDP biosynthesis gene(s) [gene]'], ['Thr49 [gene]'], ['bacterial TPP aptamers [gene]'], ['presence of TPP in TPPswec [gene]'], ['thiamine-phosphate synthase gene(s) [gene]'], ['TPK and TH1 [gene]'], ['dTph1 insertion in BR biosynthesis gene(s) CD9 [gene]'], ['thiamine pyrophosphate (TPP)-specific riboswitch [rna]']]}\n",
      "-------------------\n",
      "{'112662': [['unigenes [phenotype]'], ['maternal-specific transcripts [phenotype]'], ['E1I1p transcripts [phenotype]'], ['multiple spots per nucleus [phenotype]'], ['first transcript [phenotype]'], ['non-normalized library [phenotype]'], ['Single transcript variant [phenotype]'], ['polytype transcripts [phenotype]'], ['constitutively spliced exons [phenotype]'], ['high translation efficiency group [phenotype]'], ['thousands of novel transcription units [phenotype]'], ['multitude of minor transcription start sites [phenotype]'], ['background or noisy transcripts [phenotype]'], ['major transcript ends [phenotype]'], ['**functional form of its transcript [phenotype]**', 'functional forms of its transcript [phenotype]'], ['three transcripts [phenotype]'], ['polycistronic translation [phenotype]'], ['both strands [phenotype]'], ['number of transcript isoforms [phenotype]'], ['complementary transcripts [phenotype]'], ['predominant expression of one transcript [phenotype]', '**one major form [phenotype]**', 'favored one dominant form [phenotype]'], ['full-length cDNA or MPSS expression evidence [phenotype]'], ['dicistronic manner [phenotype]'], ['one transcript per gene(s) [phenotype]'], ['transcripts of other gene(s) [phenotype]'], ['polycistronic [phenotype]'], ['three major transcripts [phenotype]'], ['DNA-templated transcripts [phenotype]'], ['in the same direction [phenotype]'], ['prediction of their encoded ORFs [phenotype]'], ['seven different transcripts [phenotype]'], ['translation of thousands of transcripts [phenotype]'], ['coding sequence of mRNA transcriptomes [phenotype]'], ['more than 18,000 gene(s) transcripts [phenotype]'], ['Additional exons [phenotype]'], ['17.8% antisense transcripts [phenotype]'], ['unfiltered list [phenotype]'], ['complexity and diversity of the coding transcriptome [phenotype]'], ['protein-coding capacity [phenotype]'], ['proximally polyadenylated class I [phenotype]'], [\"5' end and middle peaks [phenotype]\"], ['multiple functional transcripts [phenotype]'], ['high number of 34 transcripts [phenotype]'], ['**protein-coding transcripts [phenotype]**', 'protein(s) coding [phenotype]'], ['intermediate transcript levels [phenotype]'], ['translated ORFs [phenotype]'], ['transcriptome comprising UGT gene(s) [phenotype]'], ['distinct transcript [phenotype]'], ['either form [phenotype]'], ['more than 90 transcripts [phenotype]'], ['polycistronic accumulation [phenotype]'], ['similar levels of antisense transcripts [phenotype]'], ['five transcripts [phenotype]'], ['all nine gene(s) are transcribed [phenotype]'], ['adequate levels of functional transcripts [phenotype]'], ['same transcriptional unit [phenotype]'], ['thousands of transcripts [phenotype]'], ['abundance of functional transcripts [phenotype]'], ['large transcriptome complexity [phenotype]'], ['novel r-protein transcript population [phenotype]'], ['High number of transcripts [gene]'], ['1/4 transcripts [gene]'], ['two-thirds of total transcript values for entire family [gene]'], ['dominant transcripts [gene]'], ['significant portion of predicted sense transcripts [gene]'], ['the combined expression of all transcripts from one gene(s) [genomic feature]'], ['abundance of exonized transcripts [genetic element]'], ['transcripts from 55% of zebrafish gene(s) [organism]']]}\n",
      "-------------------\n",
      "{'113447': [['expression patterns of DEGs [phenotype]'], ['differential numbers in DEGs [gene]'], ['number of DEGs [phenotype]'], ['DESeq2 output [phenotype]'], ['correlation between DEGs [phenotype]'], ['Expression of DEGs [gene]', 'The expression level of these DEGs [gene]', '**expression level of DEGs [gene]**'], ['Overview of DEGs [phenotype]'], ['DE Score distributions of DE gene(s) unique for organs [phenotype]'], ['**increased DEG number [phenotype]**', 'increased number of differentially expressed gene(s) (DEGs) [phenotype]'], ['number of DEPs [phenotype]'], ['Proportion of DEGs [phenotype]'], ['differences in DEG fractions [gene]'], ['comparative level of DEG [phenotype]', '**Expression of DEGs [phenotype]**', 'relative expression of DEGs [phenotype]'], ['functional information of DEGs [phenotype]'], ['DEG status [phenotype]'], ['threshold to judge DEGs [phenotype]'], ['DE Score distribution [phenotype]'], ['differences in significant DEG [phenotype]']]}\n",
      "-------------------\n",
      "{'169713': [['lower levels of Z-3-HAL [metabolite]'], ['lower tZ levels [metabolite]'], ['**lower cZ [metabolite]**', 'lowered cZs [metabolite]']]}\n",
      "-------------------\n",
      "{'30382': [['BrdU-labelled leptotene nuclei [structure]'], ['pachytene-like nuclei [structure]'], ['haploid nuclei [cell type]'], ['Germ cell nuclei (GN) [cell type]'], ['Meiotic nucleus [organ]'], ['nuclei of the female gametophyte [cell type]'], ['male nuclei [cell type]'], ['Female nuclei [cell type]'], ['**nuclei of meiocytes [cell type]**', 'nuclei of meiotic cells [cell type]', 'meiocyte nuclei [cell type]', 'meiotic nuclei [cell type]'], ['**meiotic nuclei [subcellular compartment]**', 'meiocyte nuclei [subcellular compartment]'], ['meiotic tetrad nuclei [cell type]'], ['nucleus of synergids [cell type]'], ['diploid pachytene nuclei [organism]'], ['Meiotic nucleus [organism]'], ['nuclei of four meiotic products [subcellular compartment]'], ['nuclei of male meiocytes [cell type]'], ['cytoplasm of pre-meiotic sporocytes [cell type]'], ['meiotically stable nuclei [cell type]'], ['nucleus of a specific developmental form in the meiotic stage in Trypanosoma [organism]'], ['egg nucleus [cell type]'], ['spherical organelles [organ]']]}\n",
      "-------------------\n",
      "{'171678': [['acyl CoA oxidase [enzyme]', '**acyl-CoA-oxidase [enzyme]**', 'each ACX enzyme [enzyme]', 'peroxisomal acyl-CoA oxidase 4 (ACX4) [enzyme]', 'acyl-CoA oxidase isozymes [enzyme]', 'Acyl-CoA-oxydase1 (ACX1) [enzyme]', 'acyl-CoA oxidase 1 [enzyme]', 'putative peroxisomal acyl-CoA oxidase/dehydrogenase [enzyme]'], ['Glyoxylate cycle enzymes [enzyme]'], ['**peroxisomal beta-oxidation enzymes [enzyme]**', 'enzymes of peroxisomal beta-oxidation [enzyme]'], ['key enzymes in beta-oxidation [protein]', 'Fatty acid beta-oxidation enzymes [protein]', 'core beta-oxidation enzymes [protein]', '**beta-oxidation enzymes [protein]**'], ['cytochrome b 5-fusion enzymes FAH1 and FAH2 [enzyme]'], ['enzymes of beta-oxidation [gene]', '**beta-oxidation enzymes [gene]**', 'ACX enzymes [gene]'], ['Fatty acid beta-oxidation enzymes [enzyme]', '**beta-oxidation enzyme [enzyme]**'], ['**Acyl-coenzyme A oxidase [protein]**', 'long-chain acyl-CoA oxidase [protein]', 'PTS1-targeted short-chain acyl-CoA oxidase [protein]', 'short chain acyl-CoA oxidase activity [protein]'], ['Chy1 [enzyme]'], ['Peroxisomal acyl-CoA synthetase activity [phenotype]'], ['Acyl-activating enzyme isoform1 [protein]'], ['Acyl-activating enzyme isoform5 [protein]'], ['short chain-specific oxidase [enzyme]'], ['peroxisomal CSY [enzyme]', '**glyoxysomal citrate synthase [enzyme]**'], ['**cytochrome P450 pathways [enzyme]**', 'P450 fatty acid oxidases [enzyme]'], ['Auxiliary enzymes [enzyme]'], ['P450-type metabolizing FA enzymes [protein]'], ['Group 1 ALTs [enzyme]'], ['glyoxysomal enzymes [protein]'], ['enzymes essential for glyoxylate cycle [process]'], ['PTS1-containing peroxisomal fatty acid beta-oxidation enzymes [enzyme]'], ['endogenous peroxisomal acyl-CoA synthetases [protein]'], ['fatty acyl coenzyme A dioxygenase [protein]'], ['peroxisomal co-enzyme synthetase [protein]'], ['peroxisome-localized enzyme [protein]'], ['enzymes for fatty acid oxidation [enzyme]'], ['fatty acyl-CoA oxidase activity [phenotype]'], ['enoyl-CoA hydratases/isomerases [protein]'], ['carboxyltransferase (alpha-CT and beta-CT) [enzyme]'], ['catalytic activity of cytochrome P450 enzymes [protein complex]'], ['β-oxidation pathway enzyme [enzyme]']]}\n",
      "-------------------\n",
      "{'82170': [['repressed synthesis of most normal protein(s) [protein]'], ['**BiP protein(s) accumulation [phenotype]**', 'over-expression of BiP [phenotype]'], ['**upregulated SUMOylated protein(s) [protein]**', 'accumulation of SUMO1/2-conjugated protein(s) [protein]', 'increase in SUMO conjugated protein(s) [protein]'], ['increased chaperone expression [phenotype]', '**chaperone accumulation [phenotype]**', 'common accumulation of several stress marker chaperones [phenotype]'], ['formation of HMM species [protein complex]'], ['higher levels of chaperones [protein]'], ['elevated expressions of PDIs [gene]'], ['**SUMO conjugates accumulation [phenotype]**', 'elevated levels of conjugated SUMO [phenotype]', 'SUMO-protein conjugate levels [phenotype]', 'SUMO conjugate increase [phenotype]'], ['FBN accumulation [process]'], ['Increase in proteolipid mRNA [gene]'], ['**Increased BiP [protein]**', 'elevated levels of BiP [protein]'], ['increased unordered structure [phenotype]'], ['accumulation of DNA damage repair protein(s) [phenotype]'], ['accumulation of Hlips [protein]'], ['secretion of a high number of LSPs [protein]'], ['increase in FTSH4 [protein]'], ['excessive accumulation of EIN3 [protein]'], ['level of SGS3 protein(s) [protein]', '**SGS3 protein(s) levels [protein]**'], ['transient accumulation of D1 [protein complex]'], ['transient accumulation of D2 [protein complex]'], ['higher overlap of the cold-accumulated protein(s) [protein]'], ['elevated synthesis of factors [process]'], ['inferior proline accumulation [process]'], ['down-accumulation of protein(s) [protein]'], ['ProDH accumulation [protein]'], ['increased abundance of insoluble aggregated mitochondrial protein(s) [protein]'], ['accumulation of RPS10 [protein]'], ['accumulation of poly(ADP-ribosyl)ated protein(s) [phenotype]'], ['increase in total soluble protein(s) [biochemical change]'], ['accumulation of several LEA protein(s) [phenotype]'], ['increased accumulation of catalase protein(s) aggregates [protein]'], ['increased ubiquitination of insoluble protein(s) aggregates [phenotype]'], ['FtsH6 protein(s) accumulation [protein]'], ['resetting of HSP21 abundance [protein]'], ['accumulation of ubiquitinated products [phenotype]'], ['prestressed levels [phenomenon]'], ['accumulation of other protein(s) [protein]'], ['reduced cytosolic Hsp70 expression [phenotype]'], ['disappearance of Lhcb1 [protein]'], ['disappearance of Lhcb2 [protein]'], ['dehydration-induced insoluble ubiquitinated protein(s) accumulation [phenotype]'], ['decrease of related protein(s) [protein]'], ['significant increase in protein(s) tyrosine nitration [phenotype]']]}\n",
      "-------------------\n",
      "{'108894': [['regions with lower methylation levels [process]'], ['**regions depleted of methylation [phenotype]**', 'regions with decreased DNA methylation [phenotype]']]}\n",
      "-------------------\n",
      "{'85713': [['Positive effects of elevated CO2 [treatment]'], ['Negative effects of elevated CO2 [treatment]']]}\n",
      "-------------------\n",
      "{'36585': [['absence of degeneration and collapse of septum cells [phenotype]'], ['abnormal cellular structures in embryo sacs [phenotype]'], ['loss of central cell fate [phenotype]'], ['**loss of micropylar cell identity [phenotype]**', 'identity defects in cells at the micropylar pole [phenotype]'], ['loss of SMC polarization [cell type]'], ['abnormal SMC formation and polarization [phenotype]'], ['embryo sacs showing incomplete cellularization [phenotype]', '**failed cellularization of embryo sac [phenotype]**'], ['abnormal tapetum programmed cell death (PCD) [phenotype]'], ['disrupted cell specification of basal cell [phenotype]'], ['disrupted cell specification of apical cell [phenotype]'], ['loss of CSC trajectory re-orientation [subcellular compartment]'], ['distortions in suspensor-cell specification [phenotype]'], ['aborted transition of the tapetum to a secretory type [phenotype]'], ['failure in the fusion of the polar cells in embryo sac development [phenotype]']]}\n",
      "-------------------\n",
      "{'81731': [['G-protein subunits [protein]'], ['GH3 protein(s) from group I [protein]'], ['GST subunits [protein]'], ['**epsilon group [protein]**', 'epsilon members [protein]', 'epsilon group members [protein]', 'epsilon subgroup [protein]', 'Epsilon [protein]'], ['epsilon group [protein group]'], ['**Non-epsilon group [protein]**', 'non-epsilon members [protein]', 'non-epsilon group members [protein]', 'non-epsilon subgroup [protein]', 'members of non-epsilon group [protein]'], ['free subunits [protein]'], ['epsilon group members [phenotype]'], ['**Type II Ggamma protein(s) [protein]**', 'group II Ggamma protein(s) [protein]'], ['extra-large Gα (XLG) subunit [protein]'], ['G-protein subunits [subunit]'], ['putative gamma-type subunit [protein]'], ['a type B gamma-subunit [protein]'], ['Galpha protein(s) in the soluble state [protein]'], ['14-3-3 non-epsilon group [protein]'], ['Greek letters [descriptor]'], ['**prototypical G protein(s) α subunit [protein]**', 'all known G protein(s) alpha subunits [protein]'], ['ε cluster [protein complex]']]}\n",
      "-------------------\n",
      "{'152381': [['Expression patterns of segmental duplicated gene(s) of SlAAT superfamily [gene]'], ['**SPL gene(s) clusters from S. lycopersicum [gene]**', 'SPL gene(s) family members in tomato [gene]'], ['SPX family members of Solanum lycopersicum [gene]'], ['SPX-EXS family gene(s) of Solanum lycopersicum [gene]'], ['**SpenPAL gene(s) [gene]**', 'SlPAL and SpenPAL gene(s) [gene]', 'SlPALs and SpenPALs [gene]', 'SpenPALs [gene]'], ['capsaicin biosynthetic gene(s) [gene]'], ['SpenPAL1-SpenPAL11 [gene]'], ['two copies of squalene synthase gene(s) [gene]']]}\n",
      "-------------------\n",
      "{'11848': [['**amount of pectin [phenotype]**', 'pectin abundance [phenotype]'], ['pectin precipitation [phenotype]'], ['RR stained mucilage [phenotype]'], ['pectin status [phenotype]'], ['peaks for stretching vibration of methyl-esterified carboxylic groups [phenotype]'], ['Presence of non-released mucilage polysaccharides [phenotype]'], ['hydrated gels [phenotype]'], ['extrusion of mucilage upon hydration [phenotype]'], ['rapid bursting of mucilage secretory cells [phenotype]']]}\n",
      "-------------------\n",
      "{'32695': [['**Cell wall protein(s) [protein]**', 'Wall protein(s) [protein]', 'Cell wall protein(s) (CWPs) [protein]', 'protein(s) in the cell wall [protein]'], ['**Structural cell wall protein(s) [protein]**', 'Cell wall structural protein(s) [protein]', 'structural cell-wall protein(s) [protein]', 'structural CWP [protein]', 'cell wall structure protein(s) [protein]', 'structural CWPs [protein]', 'structural protein(s) of cell walls [protein]'], ['**Cell-wall-related protein(s) [protein]**', 'Cell wall-related protein(s) [protein]', 'cell wall related protein(s) [protein]'], ['**plant(s) cell wall protein(s) [protein]**', 'plant(s) cell wall protein(s) (CWP) [protein]'], ['over 1000 cell wall protein(s) [protein]'], ['Cell wall-plasma membrane linker protein(s) (CWLP) [protein]'], ['Cell wall-specific protein(s) [protein]'], ['cell wall protein(s) moiety [protein]'], ['Most CWPs [protein]'], ['plant(s) cell-wall-related protein(s) [protein]'], ['cell wall attached protein(s) [protein]'], ['**cell wall associated protein(s) [protein]**', 'cell-wall-associated protein(s) [protein]'], ['cell wall reinforcement protein(s) [protein]'], ['cell wall component-related protein(s) [protein]'], ['cell wall-associated kinases (WAKs) [protein]'], ['cell wall protein(s) inventory [protein]'], ['enrichment of cell wall protein(s) [protein]'], ['components of cell wall [protein]'], ['wall protein(s) components [protein]'], ['protein(s) with Cell Wall Related Processes [protein]'], ['major group of plant(s) cell wall protein(s) [protein]'], ['**structural cell wall-related protein(s) [protein]**', 'cell wall structure-related protein(s) [protein]'], ['cell wall proteoglycan [protein]'], ['cell wall glycine-rich protein(s) [protein]'], ['many cell wall protein(s) [protein]'], ['cell-wall-bound protein(s) [protein]'], ['two cell wall-related protein(s) [protein]'], ['cell wall adhesion protein(s) [protein]'], ['cell wall-targeted protein(s) [protein]'], ['Cell wall protein(s) [protein group]'], ['cell wall structure protein(s) [protein family]'], ['structural constituent of cell wall [molecular function]'], ['cell wall protein(s) groups [material]'], ['major protein(s) component of cell walls [organ]'], ['Cell wall protein(s) [subcellular compartment]'], ['Cell wall structural protein(s) [gene]'], ['cell wall protein-based defense [process]'], ['Cell wall protein(s) [tissue]'], ['Wall protein(s) [metabolite]'], ['structural constituent of cell wall [function]'], ['cell wall localization [tissue]'], ['linkers of the cell wall to the plasma membrane [function]']]}\n",
      "-------------------\n",
      "{'155406': [['RNAi knock-down of OsCYCA2;1 [gene]'], ['RNAi-mediated knock-down of ASCC3 or ALKBH3 [gene]'], ['RNAi of cathepsin L (Ov-cpl) and cathepsin Z (Ov-cpz) [gene]'], ['RNAi analysis of OsSPO11-1 [gene]'], ['LPCAT single-gene RNAi [treatment]'], ['OsSPX1 RNA interference [gene]'], ['RNAi of OsPDK1 [gene]']]}\n",
      "-------------------\n",
      "{'87406': [['favorite textbook [education]'], ['Education [concept]'], ['female education attainment [treatment]'], ['little to no training [treatment]'], ['undergraduate education [activity]'], ['medical school [education]'], ['introductory level [education]'], ['Education [discipline]'], ['Education [activity]']]}\n",
      "-------------------\n",
      "{'58568': [['3-d submergence [treatment]'], ['wet methodologies [treatment]'], ['two microvalves [treatment]'], ['15 analytical technologies [treatment]'], ['easily leached by rainfall [treatment]'], ['various environmental challenges [treatment]'], ['12 microbial biomarkers in soil samples (control) [treatment]'], ['water environment [treatment]'], ['sludge from wastewater treatment plant(s) [treatment]'], ['microbial collections [treatment]'], ['isolating pure strains [treatment]'], ['characterizing bacterial communities [treatment]'], ['terrible environmental conditions [treatment]'], ['diluent water [treatment]'], ['water tank [treatment]']]}\n",
      "-------------------\n",
      "{'168286': [['overproduction of anthocyanins [metabolite]', 'excessive production of anthocyanins [metabolite]', '**anthocyanin overproduction [metabolite]**'], ['**anthocyanin overaccumulation [metabolite]**', 'Over-accumulation of anthocyanins [metabolite]', 'excessive accumulation of anthocyanins [metabolite]', 'excessive accumulation of anthocyanin [metabolite]', 'over-accumulation of anthocyanin [metabolite]', 'hyperaccumulation of anthocyanins [metabolite]', 'abnormal accumulation of anthocyanins [metabolite]', 'anthocyanin hyper-accumulation [metabolite]', 'hyperaccumulation of anthocyanin [metabolite]'], ['over-accumulation of anthocyanin in emerging rosette leaf [metabolite]']]}\n",
      "-------------------\n",
      "{'76172': [['**CAP domain [protein]**', 'a cap domain [protein]'], ['MID1(136-170)BD [gene]'], ['SZ [protein domain]'], ['NICP [protein domain]'], ['P4 [protein domain]'], ['MADS domains [protein domain]', '**MADS-box domains [protein domain]**'], ['I (intervening) domain [protein domain]', 'intervening (I) domain [protein domain]', 'Intervening (I domain) [protein domain]', '**I-domain [protein domain]**', 'intervening I domain [protein domain]'], ['J-PI I-domain [protein domain]'], ['DIX domains [protein domain]'], ['IRPGC domain [protein]'], ['Domains III (DIII) and IV (DIV) [protein domain]'], ['**SEPALLATA3 MI domain [protein domain]**', 'SEP3MI [protein domain]', 'MI domain of SEP3 [protein domain]'], ['K-domain [protein]'], ['M domain [metabolite]'], ['DJ-1 domain [protein domain]'], ['LisH [protein domain]'], ['central D1M domains [protein domain]'], ['blade 4 [protein domain]'], ['PB1 DOMAIN [protein structure]'], ['GCT box [protein domain]']]}\n",
      "-------------------\n",
      "{'99230': [['hybrid breeding [technique]'], ['hybrids [treatment]'], ['**Hybrid combinations [organism]**', 'crop species hybrids [organism]'], ['Hybridization [event]'], ['cross-breeding [process]'], ['controlled cross-breeding [process]'], ['hybrid breeding programs [process]'], ['interspecific hybridization of Brassica species [process]'], ['grafting or crossing [process]'], ['**hybridisation [process]**', 'hybrid breeding [process]'], ['**inter-specific cross [process]**', 'inter-specific crosses [process]']]}\n",
      "-------------------\n",
      "{'37228': [['metazoan developmental processes [phenotype]'], ['cellularisation [phenotype]'], ['**cell remodeling [phenotype]**', 'cellular reorganization events [phenotype]'], ['gathering cell cycle information [phenotype]'], ['observation of cellularization [phenotype]'], ['cell totipotency [phenotype]'], ['processes related to multicellularity [phenotype]'], ['morphogenesis of cell components [phenotype]'], ['cell reversion [phenotype]'], ['margin cell file formation [phenotype]'], ['cellular origin [phenotype]'], ['establishment of multicellularity [phenotype]'], ['breaking cellular symmetry [phenotype]'], ['cellular biogenesis [phenotype]']]}\n",
      "-------------------\n",
      "{'131897': [['3-TM (semiSWEET) gene(s) [gene]'], ['seven SWEET gene(s) [gene]'], ['7-TM SWEET gene(s) [gene]'], ['140 semiSWEET gene(s) [gene]']]}\n",
      "-------------------\n",
      "{'104911': [['**Arbuscular Mycorrhizal fungi [organism]**', 'Arbuscular mycorrhiza [organism]', 'arbuscular mycorrhizal [organism]', 'Arburscular mycorrhizal fungi [organism]', 'Arbuscular mycorrhizae [organism]', 'arbuscular mycorrhizal fungus [organism]', 'arbuscular mycorrhiza fungi [organism]', 'arbuscular mycorrhizae fungi [organism]'], ['plant-arbuscular mycorrhizal symbioses [organism]'], ['**arbuscular mycorrhizal symbiosis [organism]**', 'symbiotic associations with arbuscular mycorrhizal fungi [organism]', 'symbioses with arbuscular mycorrhizal fungi [organism]', 'symbiosis of arbuscular mycorrhizal fungi [organism]', 'symbiosis with arbuscular mycorrhizal fungi [organism]', 'symbiosis with arbuscular-mycorrhizal fungi [organism]'], ['**arbuscular mycorrhizal symbiosis [process]**', 'arbuscular mycorrhizal symbioses [process]', 'arbuscular mycorrhizal fungi symbiosis [process]', 'arbuscular mycorrhizal fungal symbiosis [process]'], ['Arbuscular mycorrhiza [organ]'], ['Arbuscules [organism]'], ['plant(s) interactions with arbuscular mycorrhizal fungi [organism]', '**symbiotic interactions with arbuscular mycorrhizal fungi [organism]**'], ['initiation of arbuscular mycorrhizae [process]'], ['plant-arbuscular mycorrhizal interactions [process]'], ['initiation of colonization events by arbuscular mycorrhizal fungi [organism]'], ['symbiosis with arbuscular mycorrhizal fungi in rice [organism]'], ['>80% of plant(s) [organism]'], ['branching of arbuscular mycorrhizal fungi [organism]'], ['attracting mutualistic arbuscular mycorrhizal fungi [organism]'], ['arbuscular mycorrhizal fungi colonization [organism]'], ['arbuscular mycorrhizal root(s) [organ]'], ['400 million years ago [time]']]}\n",
      "-------------------\n",
      "{'92627': [['keg extracts [treatment]'], ['Zoospore-free fluid [treatment]'], ['treatment of wild-type protein(s) extracts [treatment]'], ['Bacillus crude extract [treatment]'], ['Salmonella-infected leaf homogenates [treatment]'], ['less contaminated agars [treatment]'], ['dilute chelators such as EDTA or HCL-NaOH [treatment]'], ['pre-treatment with bacterial extracts [treatment]'], ['pre-treatment with flagellin-derived peptide [treatment]'], ['Crushed egg extracts [treatment]'], ['Ethanol-treated scaffolds [treatment]'], ['gut content extracts [treatment]'], ['**bacterial EVs [treatment]**', 'EVs from cultured bacteria [treatment]'], ['**Chlorella supernatants [microbe]**', 'cell-free supernatants from Chlorella species [microbe]'], ['detergent extracts [treatment]'], ['Crude bacterial extracts [treatment]'], ['egg extracts [treatment]'], ['other supernatants [treatment]'], ['enriched enzymatic extracts [treatment]'], ['boiled samples [treatment]'], ['large quantities of starting material [treatment]'], ['lysate supernatant [treatment]'], ['intercellular washing fluids [tissue]'], ['partially purified extracts [treatment]'], ['Escherichia coli extracts [treatment]']]}\n",
      "-------------------\n",
      "{'79792': [['UIM Sequences [protein]'], ['Ufo1 [protein]'], ['UIM of Ubx5 [protein domain]'], ['receptors containing UIM [protein]'], ['C53 IDR-UFM1 interaction [interaction]'], ['UIM protein(s) [protein]'], ['UIM2 [protein]']]}\n",
      "-------------------\n",
      "{'112158': [['helitron [dna sequence]'], ['Helitron2 [te family]'], ['Helitron family [gene]'], ['Helitron-like transposons [element]'], ['helitron [order]'], ['Helitron elements [phenotype]'], ['HELITRON super-family [genetic element]']]}\n",
      "-------------------\n",
      "{'46614': [['coi1-1 mutation [mutant]'], ['**coi1-21 [mutant]**', 'coi1-21 mutations [mutant]'], ['coi1-22 [mutant]'], ['coi1-2 hypomorph [mutant]'], ['coi1-2 mutants [mutant]'], ['plant(s) mutated for COI1 [mutant]'], ['coi1 mutant allele [mutant]'], ['coi1 genetic background [mutant]']]}\n",
      "-------------------\n",
      "{'86233': [['N acquisition [process]'], ['Enhanced ammonium acquisition [process]'], ['sensing of external N [process]'], ['**N acquisition capacity [process]**', 'N acquisition ability [process]'], ['sink strength for N-remobilization [process]']]}\n",
      "-------------------\n",
      "{'148520': [['gene(s) induced in response to Pi deficiency [gene]'], ['gene(s) responsive to Pi or Fe deficiency [gene]']]}\n",
      "-------------------\n",
      "{'127494': [['target site in chloroplast [organ]'], ['editing of rpoA-78691 and clpP1-69942 sites [process]'], ['editing of particular sets of Cs in mitochondria and/or chloroplast [process]'], ['editing of 14 sites in chloroplast [process]'], ['**editing at multiple sites of plastid RNAs [process]**', 'editing of multiple chloroplast RNA sites [process]', 'editing of multiple plastid transcripts [process]'], ['editing of seven sites [process]'], ['editing of six plastid RNA editing sites [process]'], ['editing of chloroplast-encoded RNA transcripts [gene]'], ['editing of most sites in chloroplast [organ]']]}\n",
      "-------------------\n",
      "{'142496': [['branched 1 (BRC1) [gene]', '**BRANCHED1 (BRC1) [gene]**', 'transcription factor BRC1 [gene]', 'transcription factor BRANCHED1 (BRC1) [gene]', 'Bud BRC1 transcripts [gene]', 'branching integrators BRC1 [gene]', 'expression of branching regulators BRC1 [gene]', 'transcription of BRC1 [gene]'], ['CEP RECEPTOR 1 (CEPR1) [protein]'], ['TB1/FC1/BRC1 pathway [gene]'], ['TEOSINTE BRANCHED 1 (TB1) [gene]'], ['BoBAK1 CT domain [protein domain]'], ['Boundary of ROP domain1 (BDR1) [gene]'], ['BRIC-A-BRAC/TRAMTRACK/BROAD-COMPLEX 1/2 (BT1/BT2) [gene]'], ['BIR1 regulation [phenotype]'], ['BPS gene(s) [gene]'], ['BPS1 signal [gene]'], ['RBR1-controlled gene(s) [gene]'], ['BRC1/ATHB21/40/53 regulatory module [protein complex]'], ['BrpREV [gene]'], ['BAM1 [receptor]'], ['transcriptional activation of BRC1/2 [gene]'], ['yeast BRF1 [gene]'], ['TCP protein(s) BRC1 [protein]'], ['BIR1 homeostasis [gene]'], ['specific physiological function to certain BOR1 homologs [phenotype]'], ['**expression of TB1/BRC1 [gene]**', 'transcription of TB1/BRC1 [gene]'], ['developmental regulation through BRI1 [protein]']]}\n",
      "-------------------\n",
      "{'157694': [['MdNCED6/9 [gene]'], ['DAB4/COI1 [gene]'], ['AUTOPHAGY-RELATED 6 [gene]'], ['INFLORESECENCE DEFICIENT IN ABSCISSION (IDA) [gene]'], ['TOADSTOOL 2 [gene]'], ['2-OHM-induced senescence gene(s) [gene]'], ['**SA INDUCTION-DEFICIENT 2 (SID2) [gene]**', 'SA INDUCTION DEFICIENT2 (SID2) [gene]', 'SA regulator SID2 [gene]'], ['FAF2 [gene]'], ['FAF4 [gene]'], ['**FANTASTIC FOUR (FAF) [gene]**', 'FAF gene(s) [gene]'], ['FANTASTIC FOUR 3 [gene]'], ['SXS domain [protein domain]'], ['Short-day Sensitive Cell Death 1 gene(s) [gene]'], ['JA marker gene(s) VEGETATIVE STORAGE PROTEIN2 (VSP2) [gene]'], ['OsFAMA expression [gene]'], ['FAF-like [gene]'], ['partial FWA activation [process]'], ['TOADSTOOL [gene]'], ['SA INDUCTION DEFICIENT 2 (SID2) gene(s) promoter [gene]'], ['FAS2 gene(s) [gene]']]}\n",
      "-------------------\n",
      "{'15596': [['**ratio of atrichoblast/trichoblast cell length [phenotype]**', 'lower atrichoblast/trichoblast ratio [phenotype]'], ['shorter epidermal cells and greater root(s) hair production [phenotype]'], ['increase in number of cells at trichoblast position [phenotype]'], ['increased circumferential epidermis cell quantities [phenotype]'], ['more non-hair cells [phenotype]'], ['increased trichoblast cell area [phenotype]']]}\n",
      "-------------------\n",
      "{'150015': [['**maize homolog [gene]**', 'Its maize ortholog [gene]', 'maize ortholog [gene]', 'orthologous gene(s) of maize [gene]', 'one maize homolog [gene]', 'orthologues from Zea mays [gene]'], ['homologs from Zea mays [organism]'], ['maize ortholog of GIGANTEA [gene]']]}\n",
      "-------------------\n",
      "{'86889': [['**MIL [treatment]**', 'MIL-based lysis and extraction [treatment]', 'a simple, rapid, miniaturized ionic liquid (IL)-based extraction method [treatment]', 'MIL-based cell disruption [treatment]'], ['one-step plant(s) cell lysis and DNA extraction [treatment]'], ['extraction from larger volumes of plasma [treatment]'], ['silica-based methods [treatment]'], ['10% Chelex [treatment]'], ['Cold formic acid/SPE extraction [treatment]'], ['TiO2 extraction protocol [treatment]'], ['New extraction protocol [treatment]'], ['column-based methods [treatment]'], ['BAC DNA isolation [treatment]'], ['ease of DNA extraction [treatment]'], ['Hirt lysis procedure [treatment]'], ['ER isolation protocol [treatment]'], ['mass isolation of male gametes [treatment]'], ['**Enriching the sample for viral nucleic acids [treatment]**', 'Using virions extracts [treatment]'], ['**ExoQuick system [treatment]**', 'ExoQuick method [treatment]'], ['combination method [treatment]'], ['Ultracentrifugation method [treatment]'], ['**combined ultracentrifugation-ExoQuick method [treatment]**', 'combining ultracentrifugation with ExoQuick system [treatment]'], ['NaOH extraction method [treatment]'], ['Conventional DNA extraction methods [treatment]'], ['mild extraction conditions [treatment]'], ['spiking plasmid-derived material [treatment]'], ['narrow bare open capillaries [treatment]'], ['alkaline lysis [technique]'], ['DNA extraction methods [treatment]'], ['improving DNA recovery [treatment]'], ['isolation of circulating nucleic acids [treatment]'], ['increasing plasma volume [tissue]'], ['new DNA extraction [treatment]'], ['bead beating [treatment]'], ['standard isolation techniques [treatment]']]}\n",
      "-------------------\n",
      "{'162083': [['Nrf2 signaling pathway [signaling pathway]'], ['upregulation of cellular antioxidant pathways [process]'], ['recycling of antioxidants [process]']]}\n",
      "-------------------\n",
      "{'133703': [['CAROTENOID CLEAVAGE DIOXYGENASE 1 (CCD1) [gene]'], ['CAROTENOID CLEAVAGE DIOXYGENASE8 [gene]'], ['**Carotenoid cleavage dioxygenase (CCD) gene(s) [gene]**', 'carotenoid cleavage dioxygenases (CCDs) [gene]'], ['CAROTENOID CLEAVAGE DIOXYGENASE enzymes (CCD1 and CCD4) [gene]'], ['Nine carotenoid cleavage dioxygenase (CCD) gene(s) [gene]']]}\n",
      "-------------------\n",
      "{'40475': [['transgenic ACD line [mutant]'], ['ACD tomato line [mutant]'], ['MM tomato lines [mutant]']]}\n",
      "-------------------\n",
      "{'27345': [['attenuation of R gene-dependent defense responses [phenotype]', 'attenuation of R-gene signaling pathways [phenotype]', '**compromised R-gene mediated resistance [phenotype]**', 'are negatively affected in R gene-mediated resistance [phenotype]', 'impaired R gene-mediated resistance [phenotype]'], ['weak R gene(s) response [process]']]}\n",
      "-------------------\n",
      "{'161793': [['2-deoxy-glucose (2-DG) [treatment]', '2-deoxy-d-glucose (2D) [treatment]', '**2-DOG [treatment]**'], ['(-)5DS [metabolite]'], ['CN-PMF [metabolite]'], ['**6-deoxyglucose [metabolite]**', '6-deoxy-d-glucose [metabolite]', '6-deoxyglucose (6DOG) [metabolite]'], ['F-gluconic acid [metabolite]'], ['isophtalate [metabolite]'], ['2F-Fuc [metabolite]'], ['3OMG [metabolite]', '**3-O-methyl-glucose [metabolite]**', '3-O-methylglucose (3OMG) [metabolite]'], ['2DOG [metabolite]', '**2-DOG [metabolite]**', '2-deoxy-glucose [metabolite]', 'glucose analog 2-deoxyglucose [metabolite]'], ['Nonsulfatable CIF2Y64F [metabolite]'], ['Turanose, a sucrose analogue [metabolite]'], ['unincorporated UDP-D-(14)CGalpA [metabolite]'], ['6-azido-6-deoxy-beta-D-glucose [metabolite]'], ['gelator without an ester bond (NAA-GFFY) [metabolite]'], ['1-deoxynojirimycin [metabolite]'], ['dUpNHpp [metabolite]'], ['FDM-6-P [metabolite]'], ['FDM-1-P [metabolite]'], ['3-O-methyl-glucose [treatment]'], ['dynorphin A (YGGFLRRIRPKLK) [metabolite]']]}\n",
      "-------------------\n",
      "{'103680': [['Klebsiella pneumoniae 342 [organism]'], ['**Klebsiella [organism]**', 'Klebsiella spp. [organism]'], ['Klebsiella aerogenes [organism]'], ['Klebsiella oxytoca [organism]'], ['Kribbella [organism]'], ['Klebsiella michiganensis [organism]'], ['Lelliottia jeotgali [organism]']]}\n",
      "-------------------\n",
      "{'130199': [['A. thaliana Ara6 [gene]'], ['RAB5 group of A. thaliana [gene]']]}\n",
      "-------------------\n",
      "{'122720': [['**identical sequences [gene]**', 'gave the same sequence [gene]', 'identical [gene]'], ['Two copies of the same DNA sequence [gene]'], ['same mechanisms [gene]'], ['the same genotypes [gene]'], ['same regions [gene]'], ['same markers [gene]'], ['same vector architecture [gene]'], ['same constructs [gene]'], ['same number and distribution of gene(s) [gene]'], ['Same protein(s) [gene]'], ['**nearly identical gene(s) duplicates [gene]**', 'highly identical gene(s) CNV regions [gene]'], ['identical Ks distribution [gene]'], ['same position [gene]'], ['same clone [gene]'], ['same genomic organisation [gene]'], ['same genetic background [genetic element]'], ['identical sequence environments [sequence]'], ['same reaction [phenotype]'], ['same position [phenotype]'], ['no difference in their epigenetic patterns [phenotype]'], ['100% identical [phenotype]'], ['same marker gene(s) [phenotype]'], ['same biochemical pathway [phenotype]'], ['same gene(s) function [phenotype]'], ['same gene(s) ID in genome versions [phenotype]'], ['same number of introns [phenotype]'], ['same ESTs align to all gene(s) [phenotype]'], ['almost identical annotations [phenotype]'], ['antisense and sense copies of the target [phenotype]'], ['same individuals [organism]'], ['the same regions [organism]'], ['same metabolic pathway [genomic feature]'], ['identical coding sequences in parents [genomic feature]'], ['same haplotype [genetic factor]'], ['same construct [construct]'], ['identical protein(s) sequences from distinct gene(s) [record]'], ['same gene(s) in different pathways [process]'], ['same genomic location [genome]'], ['same genomic locus in both species [genome]']]}\n",
      "-------------------\n",
      "{'2353': [['**female fertility [phenotype]**', 'female reproductive competence [phenotype]', 'normal female fertility [phenotype]', 'maternal fertility [phenotype]', 'plant(s) female fertility [phenotype]'], ['mgt5 fertility [phenotype]'], ['**capacity of F1 plant(s) to fertilize ovules [phenotype]**', 'ability to fertilize [phenotype]']]}\n",
      "-------------------\n",
      "{'77847': [['**mutations in DBD [protein domain]**', 'changes in DBD domain [protein domain]'], ['DBD-HD [gene]'], ['DBD-HD inputs [protein]'], ['54 different DBD classes [protein domain]'], ['DBD fusions [protein]'], ['mutations on DBD and PB1 domains [protein domain]'], ['isolated DBD of GT-1 [protein domain]'], ['mDBD [protein variant]'], ['absence of DBD [domain]'], ['a DNA-binding domain (DBD) fused to one HD [phenotype]'], ['all other reported amino acid substitutions in DBD of eukaryotic heat shock factors are dominant negative mutations [mutation]'], ['BIND database [database]'], ['ubiquitin-binding domain (UBD)-containing repair factors [phenotype]'], ['two TMDs and two NBDs [gene]'], ['one-third single domain DWDs [protein]'], ['equal number of single and multi domain DWDs [protein]'], ['three MBD domains [protein domain]'], ['pH-dependent binding of N-terminal DBD to DNA [phenotype]']]}\n",
      "-------------------\n",
      "{'90823': [['1 mM CysNO [treatment]'], ['**S-nitrosoglutathione (GSNO) [treatment]**', 'NO donor GSNO [treatment]'], ['1.0 µM CysNO [treatment]'], ['Exogenous NO donor, S-nitroso-L-cysteine [treatment]', 'NO-donor S-nitrosocysteine [treatment]', '**S-nitroso-L-cysteine [treatment]**'], ['enhanced tolerance to NO donor CysNO [treatment]']]}\n",
      "-------------------\n",
      "{'68083': [['HXK1-independent glucose signaling [treatment]'], ['HXK1-sensing independent pathway [process]'], ['Non-canonical Wnt signaling [treatment]'], ['HXK-independent pathways [phenotype]'], ['HXK1-independent glucose signaling [process]']]}\n",
      "-------------------\n",
      "{'27928': [['thinner and weaker cell walls [phenotype]', 'weakened wall strength [phenotype]', 'Weak wall [phenotype]', 'Weak cell wall [phenotype]', '**impaired cell wall integrity [phenotype]**', 'Compromising plant(s) CW integrity [phenotype]', 'compromised cell wall integrity [phenotype]', 'weaker cell walls [phenotype]', 'reduced mechanical strength of cell wall [phenotype]', 'mechanically weakened CW [phenotype]', 'loss of structural integrity [phenotype]', 'weakened cell walls [phenotype]'], ['**secondary cell wall deficiency [phenotype]**', 'mild impact on secondary wall strength [phenotype]', 'weakened secondary cell walls [phenotype]'], ['weakening of endosperm cell walls [phenotype]']]}\n",
      "-------------------\n",
      "{'23032': [['actin reorganization during flg22-induced stomatal closure [phenotype]'], ['**flg22-regulation of stomatal closure [phenotype]**', 'flg22 regulation of stomatal movements [phenotype]']]}\n",
      "-------------------\n",
      "{'158375': [['Phototropins phot1 and phot2 [gene]', 'Phototropins [gene]', 'phototropin [gene]', 'phot1phot2 [gene]', '**PHOT1/PHOT2 [gene]**', 'phototropins (phot1 and phot2) [gene]', 'phototropin family of blue-light photoreceptors [gene]', 'blue light receptors phot1 and phot2 [gene]', 'Phototropin family photoreceptors [gene]', 'PHOTOTROPINS 1 and 2 [gene]', 'Phototrophins (PHOT1 and PHOT2) [gene]', 'phototropin family of photoreceptors [gene]', 'activities of phototropin blue-light receptors [gene]', 'both blue light receptors (phot1/phot2) [gene]', 'PHOT1/PHOT2 blue light photoreceptors [gene]'], ['phototropin (Phot) 1 or 2 blue light photoreceptors [gene]'], ['AGC kinases phototropin 1 and 2 (phot1 and phot2) [protein]'], ['phototropin1 (phot1) blue light photoreceptors [gene]', '**PHOT 1 [gene]**', 'functional PHOT1 gene(s) [gene]']]}\n",
      "-------------------\n",
      "{'31597': [['PIN polarity patterns [process]'], ['**PIN polarity network [process]**', 'complex PIN polarization network [process]'], ['dynamic PIN positioning [process]'], ['Control of PIN protein(s) distribution [process]'], ['PIN subcellular dynamics [process]'], ['PIN recycling [process]'], ['**PIN localization [process]**', 'subcellular distribution of PIN [process]'], ['distribution of PINs [process]'], ['**PIN repolarization process [process]**', 'dynamic directional changes in PIN plasma membrane localisation [process]'], ['PIN1 distribution [process]'], ['reduced PIN trafficking [cellular process]'], ['binary, apical-vs.-basal switches in PIN polarity [process]'], ['PIN distribution in neighboring cells [process]'], ['polar distribution of PIN1 [process]']]}\n",
      "-------------------\n",
      "{'77116': [['SsCO protein(s) [protein]'], ['COL subfamily [protein family]']]}\n",
      "-------------------\n",
      "{'111196': [['CTCC cis-element [nucleotide sequence]'], ['TGGACGG element [genomic element]'], ['Site II elements (TGGGCY) [genomic element]'], ['CANNTG [genetic element]'], ['TGGGC(C/T) motif [cis-regulatory element]'], ['CCGCGT CRE [motif]'], ['GC-Motif [gene]'], ['ACGT cis regulatory element [genetic element]'], ['GCN4 motif [cis-element]'], ['Cis-elements with ACGT core sequence [genomic region]'], ['TGACG-motif [treatment]'], ['AAGAA-motif [gene structure]'], ['region 2 motif [gene]'], ['fifth motif [gene]'], ['TAGGGTTT [ci element]'], ['CGTCGT [regulatory element]'], ['**GC-Motif [regulatory element]**', 'GC-motif element [regulatory element]'], ['AACA motif [genomic feature]'], ['TCT-motif [genetic element]'], ['Skn-1_motif and GCN4_motifs [genomic location]'], ['TGA-elements [phenotype]'], ['TCT-motif [cre]'], [\"'CTCC' cis-element [dna sequence]\"], ['CArG motif [genetic element]'], ['GCN4-motif [regulatory element]'], ['GC-Motif [motif]'], ['TGA elements [genomic feature]'], ['CArG motif [dna sequence]'], ['TCT-motif [cis-acting element]'], ['CACGCG in CE3 [genomic region]'], ['DNA carrying TGA cis-element as-1 [dna]'], ['ACGT-containing cis-element [dna element]'], ['GCN4 motif [development element]'], ['GCN4 [gene region]'], ['TGTC-containing cis-regulatory elements [gene]'], ['GCCCORE [cis-element]'], ['GTAC cis-acting element of downstream regulatory gene(s) promoter region [gene]']]}\n",
      "-------------------\n",
      "{'92610': [['**biostimulants [treatment]**', 'plant(s) biostimulants [treatment]', 'plant(s) bio-stimulants (BSs) [treatment]', 'plant(s) based biostimulants [treatment]', 'plant(s) biostimulant [treatment]'], ['growth factors [treatment]'], ['plant-based life support systems in space [treatment]'], ['bacterial plant(s) growth promoting diffusible or volatile compounds [treatment]'], ['Use of bioeffectors [treatment]'], ['plant-based biosensors [treatment]', '**phytosensors [treatment]**'], ['Biological factors [treatment]'], ['Bacillus-containing biological products [treatment]'], ['plant-based inducers [treatment]'], ['silicon-based biostimulant [treatment]'], ['Growth substances [treatment]'], ['Rhizobacterial compounds [treatment]'], ['Mitogens [treatment]'], ['new formulations [treatment]'], ['natural growth-promoting compounds [treatment]'], ['plant-based life support facilities [treatment]'], ['potential plant(s) biostimulant [treatment]'], ['additives [treatment]'], ['concentrated bio-stimulant [treatment]'], ['quality of bio-stimulants [treatment]'], ['better energy source [treatment]'], ['treatment with other biostimulants [treatment]'], ['use of plant(s) biostimulants [treatment]'], ['stress protectants [treatment]'], ['growth promoters in animals [treatment]'], ['natural plant(s) growth regulator [treatment]'], ['plant(s) growth-promoting natural products [treatment]']]}\n",
      "-------------------\n",
      "{'106386': [['NTC control [gene]'], ['technical advantage of involving no PCR [process]']]}\n",
      "-------------------\n",
      "{'120210': [['Genomes [genomic locus]', '**GENOME [genomic locus]**'], ['**genetic locus [genomic region]**', 'a locus [genomic region]', 'one genetic locus [genomic region]'], ['genome location [organism]'], ['single locus [genomic location]', 'single genomic locus [genomic location]', 'a specific locus within the genome [genomic location]', 'specific locus [genomic location]', '**genomic locus [genomic location]**', 'a locus [genomic location]'], ['One locus [genetic element]'], ['single specific locus [genome]', '**single locus [genome]**'], ['**one genomic locus [gene]**', 'major genomic locus [gene]'], ['single genomic locus [metabolite]'], ['single locus [genetic location]']]}\n",
      "-------------------\n",
      "{'160076': [['**FPA [gene]**', 'FLOWERING LOCUS PA (FPA) [gene]'], ['FLOWERING WAGENINGEN (FWA) [gene]', 'flowering wa-geningen (FWA) [gene]', '**FWA (FLOWERING WAGENINGEN) [gene]**', 'FWA (flowering wa-geningen) [gene]'], ['CCT and FPA [gene]'], ['binding of FPA [gene]'], ['expression of FWA gene(s) [gene]', '**FWA RNA expression [gene]**'], ['Unigene0068304 FPA [gene]'], ['Unigene0068303/Unigene0031805 FPA [gene]'], [\"5' region of FWA [gene]\"], ['FPA transcript [gene]'], ['**FPA functions [phenotype]**', 'function of FPA [phenotype]'], [\"FPA's function [gene function]\"]]}\n",
      "-------------------\n",
      "{'122763': [['**orthologous regions [phenotype]**', 'orthologous positions [phenotype]'], ['**Homologous regions [phenotype]**', 'Homology regions [phenotype]', 'homologous region [phenotype]'], ['**homeologous chromosome groups [phenotype]**', 'homoeologous chromosomes [phenotype]'], ['putative homologous chromosomal regions [phenotype]'], ['co-localization of gene(s) [phenotype]'], ['reduction in homologous loci [phenomenon]'], ['pairs of regions with similar characteristics [genetic concept]'], ['two or more closely related chromosomes [gene]'], ['seven homeologous groups [phenotype]'], ['close proximity of several gene(s) [genomic feature]'], ['associated condensed homologs [phenotype]', '**homologous chromosomal pairs [phenotype]**'], ['same chromosomes [gene]'], ['same chromosome [chromosome]'], ['functional and physical commonalities with animal sex chromosomes [gene]'], ['orthologous genomic positions [genomic region]'], ['similar allelic combination [genomic feature]'], ['same chromosome [organism]'], ['homoeologous regions [genetic location]'], ['**same chromosome [gene structure]**', 'chromosomes of the same origin [gene structure]'], ['similar positions on three corresponding chromosomes [genetic location]'], ['similar chromosome constitution as intermediate ancestor [phenotype]'], ['**same syntenic cluster [phenotype]**', 'counterparts in same cluster [phenotype]'], ['overlapping gene(s) in apple chromosomes [gene]'], ['same chromosomes [genomic region]'], ['homologous blocks [phenotype]'], ['similar genomic organization [phenotype]'], ['orthologous clusters in common within regions [treatment]'], ['homologous gene(s) on different wheat chromosomes [phenotype]'], ['similar chromosomal distribution of the gene(s) [phenotype]'], ['a close relative in the corresponding regions [phenotype]']]}\n",
      "-------------------\n",
      "{'70621': [['AtHak5 [protein]'], ['WT AtHAK5 [protein]'], ['BnKAT2 and AtKAT2 protein(s) [protein]', '**KAT2 (K+ transporter 2) [protein]**'], ['In Arabidopsis, AKT2 [protein]', '**potassium transporter AKT2 [protein]**'], ['Akr2a-dependent transport [process]'], ['bestrophin-like protein(s) 1-3 (BST1-3) [protein]'], ['TrkH/KtrB group of cation transporters [protein]'], ['TaHAK13 activity [phenotype]'], ['TaHAK13 protein(s) [protein]'], ['TaHAK13 interaction [gene]'], ['HAK protein(s) of cluster II [gene family]'], ['Trk1p-Trk2p potassium transporter system [protein]'], ['Trk1p [protein]'], ['**Trk1p-Trk2p potassium transport system [protein complex]**', 'Trk1p-Trk2p transport system [protein complex]', 'Trk1p-Trk2p function [protein complex]'], ['ATHKT1 [protein]'], ['SODIUM POTASSIUM root(s) DEFECTIVE 1 (NaKR1) [protein]', '**NaKR1 [protein]**'], ['**Subclass2 transporters [protein]**', 'subclass2 [protein]'], ['**AKT3 [protein]**', 'AKT3 polypeptide [protein]'], ['KAT-like protein(s) [protein]'], ['202 PTG protein(s) sequences [gene]'], ['PTG protein(s) sequences [gene]'], ['K+ carrier TRH1 [protein]'], ['K+ transporters AtKCO1 [protein]'], ['HvHKT2;1 [protein]'], ['potassium transporter [protein complex]'], ['TPK1 peptide [protein]'], ['NaKR1-FT complex [protein complex]'], ['NrHAK1 protein(s) [protein]'], ['TRH1 subunit [protein]'], ['wheat TaHKT1-mediated K(+)/Na(+) symporter [protein]'], ['**potassium transporter AKT1 [protein]**', 'K+ transporter AKT1 [protein]', 'AKT1 (K+ transporter) [protein]'], ['fungal and bacterial K+ transporters from Trk/Ktr families [protein family]'], ['Endomembrane K+ Transporters [protein]'], ['K+ transporter family ATANNs [protein]'], ['Trk/Ktr/HKT K+ transporters [protein family]'], ['ZMK1 [protein domain]'], ['K+ transporter [phenotype]'], ['TPK3 isoform [protein]'], ['Sat4p [gene]'], ['yeast K+ transporter ScTRK1 [protein]'], ['TrkH cation transporter protein(s) conserved domain [protein domain]'], ['ANK, Ion_trans_2, and KHA domains [gene]'], ['KAT1 (K+ transporter 1) [protein]'], ['KAT2 and KAT1 [protein]'], ['EcKefC-like protein(s) from bacteria [gene]'], ['both Trk and Kdp K+ transporters [protein]'], ['14-3-3 protein(s) GRF6 [protein]'], ['ion transporters for potassium (ATkT3/KUP4/TRH1) [protein]'], ['bumetanide-sensitive Na+-K+-2Cl--cotransporter [protein]'], ['protein(s) of 777 amino acids [protein]'], ['**prokaryotic KtrB [protein]**', 'K(+) transport activity of KtrB [protein]'], ['TrkH [protein]'], ['KdpA transporter subunits [protein]'], ['fungal Trk protein(s) [protein]'], ['founding member of novel group of eukaryote K(+)/H(+) antiporters [protein]']]}\n",
      "-------------------\n",
      "{'160504': [['**SSA [metabolite]**', 'succinic semialdehyde (SSA) [metabolite]', 'Succinic semialdehyde [metabolite]', 'succinic semi-aldehyde (SSA) [metabolite]', 'succinate semialdehyde (SSA) [metabolite]'], ['Accumulation of SSA [metabolite]'], ['**GABA to SSA [metabolite]**', 'conversion of GABA to SSA [metabolite]', 'GABA conversion into succinic semialdehyde [metabolite]'], ['reduction of succinic semialdehyde (SSA) [metabolite]']]}\n",
      "-------------------\n",
      "{'72250': [['**STP transporters [protein]**', 'STPs [protein]', 'Sugar transport protein(s) (STPs) [protein]', 'Sugar transport protein(s) (STP) [protein]'], ['TST-type carriers [protein]'], ['phloem-loading protein(s) [protein]'], ['Carbohydrate transporter [protein]'], ['Sugar transport protein(s) (STP1) [protein]'], ['multiple sugar transporters [protein]'], ['glycocarrier protein(s) [protein]'], ['sugar-loading transporters [protein]'], ['other sugar Major facilitators [protein]'], ['phloem loaders [function]'], ['novel sugar transporters [phenomenon]'], ['SUGAR TRANSPORT protein(s) (STP) family [protein family]']]}\n",
      "-------------------\n",
      "{'139809': [['SlERF2 transcript abundance [gene]'], ['SlPROPEP transcript levels [gene]'], ['CsPYR1 transcript level [gene]'], ['**CsPYL8 transcript level [gene]**', 'CsPYL8 transcript levels [gene]'], ['**CsPYL9 transcript level [gene]**', 'CsPYL9 transcript levels [gene]'], ['SLY1 transcript levels [gene]'], ['oleosin transcript levels [gene]'], ['Transcript levels of SlSR2 and SlSR3L [gene]'], ['MtSEP1-5 transcript levels [gene]']]}\n",
      "-------------------\n",
      "{'35567': [['Louvain component 8 [cell type]'], ['cells in Louvain component 1 [cell type]'], ['Louvain component 10 [cell type]'], ['36 root(s) cell clusters [cell type]'], ['24 putative cell clusters [cell type]'], ['clusters 18-21 [cell type]']]}\n",
      "-------------------\n",
      "{'115768': [['core and shell OGs [gene]'], ['cloud to core OGs [gene]']]}\n",
      "-------------------\n",
      "{'25420': [['reduced final plant(s) stature [phenotype]'], ['not the highest plant(s) height [phenotype]'], ['slightly reduced height [phenotype]', '**slightly reduced plant(s) height [phenotype]**', 'slight height reduction [phenotype]', 'slight shorter plant(s) height [phenotype]', 'slightly shorter plant(s) height [phenotype]'], ['**shorter overall plant(s) height [phenotype]**', 'plant(s) with reduced height [phenotype]', 'reduced overall plant(s) height [phenotype]', 'less plant(s) height [phenotype]', 'smaller in overall stature [phenotype]', 'smaller plant(s) height [phenotype]', 'shorter in height [phenotype]', 'plant(s) of shorter stature [phenotype]', 'smaller and shorter plant(s) [phenotype]', 'shorter plant(s) statures [phenotype]', 'less height [phenotype]'], ['**significantly shorter overall plant(s) height [phenotype]**', 'significantly shorter plant(s) height [phenotype]'], ['moderately reduced plant(s) stature [phenotype]', 'moderately reduced height [phenotype]', '**moderately reduced plant(s) height [phenotype]**'], ['25% shorter plant(s) [phenotype]'], ['much lower plant(s) height [phenotype]'], ['extremely shorter heights [phenotype]'], ['**shorter seedling height [phenotype]**', 'smaller and shorter seedlings [phenotype]'], ['significantly shortened seedling height [phenotype]'], ['shorter plant(s) in shade [phenotype]']]}\n",
      "-------------------\n",
      "{'114401': [['Markers of Non-Transcriptional Oscillator (NTO) [gene]'], ['removing TOC1 [gene]'], ['high non-oscillating levels of CsTOC1 mRNAs [gene]'], ['**oscillations of CAB2 promoter activity [phenotype]**', 'circadian oscillations of CAB2 promoter activity [phenotype]'], ['JMJD5 clock function [phenotype]'], ['response of circadian luciferase reporter [phenotype]'], ['weak changes in phase of TOC1 rhythms [phenotype]'], ['luciferase activity being rhythmic [phenotype]'], ['oscillating promoter-driven luciferase activity [phenomenon]'], ['TOC1 circadian oscillation [phenotype]'], ['importance of TOC1 transcriptional regulation [phenotype]'], ['circadian pattern similar to Pn FT1 [phenotype]'], ['two TOC1/LHY oscillators [phenotype]'], ['shorter period of TOC1 oscillations [phenotype]'], ['circadian oscillations of LHCB1.1 [rna]'], ['requirement for normal ELF3 function [phenotype]'], ['TOC1 circadian waveform [phenotype]']]}\n",
      "-------------------\n",
      "{'173630': [['acquisition of sulfur [process]'], ['desulphuration of cysteine [process]', '**cysteine desulfuration [process]**', 'desulfuration of cysteine [process]', 'extraction of sulfur from cysteine [process]', 'L-Cys desulfuration [process]', 'release of sulfur from cysteine [process]', 'H2 S production [process]', 'sulfur mobilization from cysteine [process]', 'cysteine desulfurization [process]', 'mobilization of sulphur from cysteine [process]'], ['hydrolysis of glutathione S-bimane [process]'], ['desulfuration of L-Cysteine [phenotype]'], ['converting H2S to cysteine [process]'], ['conversion of Cys to Cys-sulphonic acid [process]'], ['desulfuration of L-Cysteine [metabolite]'], ['cysteine degradation [process]'], ['AtSufS cysteine desulfurization [process]']]}\n",
      "-------------------\n",
      "{'60811': [['combinations with the highest MCC and ACC [phenotype]'], ['MCC of 0.659 [phenotype]'], ['MCC of 0.614 [phenotype]'], ['CM value of 26.7 +- 0.4 [phenotype]'], ['CM value of 26.8 +- 0.4 [phenotype]'], ['CM value of 27.2 +- 0.6 [phenotype]'], ['CM value of 27.1 +- 0.7 [phenotype]']]}\n",
      "-------------------\n",
      "{'109139': [['119 bp minimal promoter region [gene]'], ['less than 173 bp promoter [gene]'], ['fragment D [metabolite]'], ['994 bp distal fragment [sequence]'], ['pTRY-B fragment [genomic region]', '**minimal TRY promoter fragment [genomic region]**'], ['316 bp long [genetic element]'], ['truncated, -394tCUP promoter fragment [gene]'], [\"3' promoter boundary [sequence]\"], ['score for TATA-less promoters [method]'], ['15 kb from their nearest promoter [genome feature]'], ['satC core promoter for (-)-strand synthesis [phenotype]'], ['region as short as 271 bp upstream of ATG can act as minimal promoter [phenotype]'], ['minimal upstream promoter segment of 19 bp [dna sequence]'], ['minimum promoter region [gene region]']]}\n",
      "-------------------\n",
      "{'72695': [['Geisler-Lee set [dataset]'], ['De Bodt set [dataset]'], ['ArathMitoP set [dataset]'], ['Krahmer et al. (2019) dataset [study]']]}\n",
      "-------------------\n",
      "{'68689': [['Parker et al. [author]'], ['unPAK research [program]'], ['Chen et al. and Ju et al. studies [study]'], ['Park et al. [source]'], ['unPAK students [organism]'], ['Muhammad Asaf Khan [author]'], ['PDK1 in yeast two-hybrid assay [treatment]'], ['identification of PD-Associated protein(s) Kinase (PAPK) [protein]'], ['E3 ubiquitin ligase activity of Parkin [gene]']]}\n",
      "-------------------\n",
      "{'170670': [['enzymes DIT2.1 and GLU1 [enzyme]'], ['Gluconacetobacter diazotrophicus (GAB) [organism]'], ['Glycolate dehydrogenases (GlcD) [enzyme]']]}\n",
      "-------------------\n",
      "{'29340': [['**phloem fiber elongation [process]**', 'fibre outgrowth [process]', 'elongation stage of fiber development [process]'], ['early stages of fiber development [process]'], ['xylan extension [process]'], ['stimulating cellular elongation [cell type]'], ['thickening in interfascicular fiber walls [cell type]']]}\n",
      "-------------------\n",
      "{'80761': [['**Nob1 [protein]**', 'Nob1 protein(s) [protein]'], ['Nop2 in Saccharomyces cerevisiae [gene]'], ['NOP9 [gene]'], ['**Nop16 [protein]**', 'Nop16 protein(s) [protein]'], ['A. thaliana Nop16 protein(s) [protein]'], ['NMD3 [protein]'], ['Mammalian NS [protein]'], ['NOP10 [protein]'], ['MPP10 [gene]'], ['Pescadillo (PES) [protein]'], ['NOP10 [gene]'], ['Nip7p [protein]'], ['TCOF [protein]'], ['EBNA1 binding protein(s) 2 (EBP2) [protein]'], ['Nop7 (PES) [gene]'], ['nucleolin-2 variant [gene]'], ['MA16 [protein]'], ['pescadillo-like protein(s) [protein]'], ['nucleolar protein(s) NOP56 [protein]'], ['RIBOPHORIN II (RPN2) family protein(s) [protein]']]}\n",
      "-------------------\n",
      "{'129304': [['**AtLTPI-4 [gene]**', 'AtLtpI.4 [gene]'], ['AtLTP4.5 [gene]'], ['AtLtpII.1 [gene]'], ['AtLtpII.2 [gene]'], ['AtLtpII.3 [gene]'], ['AtLtpI.5 [gene]'], ['AtLtpI.7 [gene]'], ['AtLtpI.8 [gene]'], ['AtLTPIV.3 [gene]'], ['AtLTPI.1 [gene]'], ['AtLTPII.10 [gene]'], ['AtLTPII.6 [gene]'], ['AtLTPII.8 [gene]'], ['AtLTPIV.1 [gene]'], ['AtLTPVI.1 [gene]'], ['AtLtp [gene]'], ['AtLtpI [gene]'], ['OsLTPV.1 and AtLTPV.1 [gene]'], ['AtLTPI.4, AtLTPI.5, AtLTPI.6, AtLTPI.7, AtLTPI.8, AtLTPI.10, AtLTPI.11, and AtLTPI.12 [gene]'], ['AtLTPIV.4 and AtLTPIV.5 [gene]'], ['AtLTPVI.3 and AtLTPVI.4 [gene]']]}\n",
      "-------------------\n",
      "{'101483': [['**A. thaliana dataset [organism]**', 'A. thaliana data set [organism]', 'one A. thaliana dataset [organism]', 'real A. thaliana data [organism]', 'the A. thaliana dataset [organism]'], ['a multitude of data for A. thaliana [organism]'], ['other A. thaliana methylome datasets [organism]'], ['analysis of A. thaliana datasets [organism]'], ['coexpression data for A. thaliana [organism]'], ['valuable resource for evolutionary genetic and functional studies in A. thaliana [organism]'], ['existing information from A. thaliana (A. thaliana) [organism]']]}\n",
      "-------------------\n",
      "{'15478': [['**enhanced lateral root(s) formation [phenotype]**', 'increase in lateral root(s) number [phenotype]', 'increase of lateral root(s) numbers [phenotype]'], ['**lateral root(s) elongation [phenotype]**', 'increase in length of lateral root(s) [phenotype]'], ['increased production of secondary root(s) [phenotype]'], ['increased formation of lateral root(s) and root(s) hairs [phenotype]'], ['stimulation of lateral root(s) growth in nutrient patches [phenotype]']]}\n",
      "-------------------\n",
      "{'160703': [['hydroxyl group of mandelonitrile [chemical group]'], ['sulfonamide compounds [chemical group]', '**sulfonamide functional group [chemical group]**'], ['S=O [phenotype]'], ['cyano group [chemical group]'], ['Hydroxyl group of 2,4,6-TCP [chemical]'], ['nitrogen atom in amino group [compound]'], ['vinyl thioethyl group [chemical group]'], ['amine group (-NH3) [material]'], ['monophosphonate [chemical group]'], ['Cis-diol groups [chemical group]'], ['sulfonamide group [chemical moiety]'], ['epoxide group [chemical group]'], ['amine group [chemical group]'], ['Electronegative sulfonyl groups [compound]'], ['cyanosulfilimine group [compound]'], ['amide groups [chemical group]'], ['alkynyl group [chemical group]'], ['alkyne group [chemical group]'], ['sulfonyl moiety [chemical structure]'], ['ester group [chemical group]'], ['methoxy group [chemical group]'], ['guanidine group [chemical group]'], ['nitrogen of the cyano group [chemical group]'], ['N3-H group [phenotype]'], ['exocyclic double bond [chemical group]'], ['S atom [chemical group]'], ['N atom [chemical group]'], ['bisphosphonic acid group [chemical group]'], ['thiol group [chemical group]'], ['another free thiol [chemical group]'], ['-N=C=S reactive group [chemical group]'], ['sulfilimine group [chemical group]'], ['sulfhydryl group [chemical]'], ['sulfonamide [group]'], ['sulfonyl moiety [chemical group]'], ['ethynyl group [chemical group]'], ['nitrogen-containing functional group [functional group]'], ['nitrile group [chemical group]'], ['chiral sulfoxide group [chemical structure]']]}\n",
      "-------------------\n",
      "{'10963': [['MF response [phenotype]'], ['metal oxide nanoparticles [phenotype]'], ['ZnO morphology [phenotype]'], ['lamina inclination response [phenotype]'], ['cantilever tip displacement [phenotype]'], ['silver nanoparticles production [phenotype]'], ['mechanical properties of the MNs [phenotype]'], ['imaging the site of the puncture [phenotype]'], ['second localized surface plasmon resonance [phenotype]'], ['magnetic behavior [phenotype]'], ['poro-elastic effects [phenotype]'], ['ice thickness near 100 nm [phenotype]'], ['external magnetic fields [phenotype]']]}\n",
      "-------------------\n",
      "{'38965': [['amiRXCT-54 [mutant]'], ['amiRXCT-58 [mutant]'], ['amiR-beta1/beta2-cop-10 mutant [mutant]'], ['amiRBR1 line [mutant]'], ['amiRNA-DEK1 lines [mutant]'], ['amiRNA-DEK1 [mutant]'], ['amiRNA14-10 [mutant]'], ['amiRNA lines of AtSRP4 [mutant]'], ['amiRNA lines of AtSRP5 [mutant]'], ['artificial microRNA-mediated silenced line amiR-ACER-1 [mutant]'], ['AtFlot1 amiRNA 15-5 lines [mutant]'], ['amiRNA-DEK1 stem(s) [mutant]']]}\n",
      "-------------------\n",
      "{'107921': [['H3K4m2/m3 [epigenetic mark]', '**H3K4 di- or trimethylation [epigenetic mark]**'], ['H3K4 di- and trimethylation [metabolite]'], ['H3K4me1/2 [epigenetic modification]'], ['de novo deposition of H3K4me3 marks [epigenetic mark]'], ['**H3K4 methyltransferases [gene]**', 'H3K4 methyltransferase gene(s) [gene]'], ['H3K4 di- and trimethylation [epigenetic modification]'], ['H3K4me3 histone modification [epigenetic modification]'], ['H3K4me2/3 [process]'], ['H3K4me3 deposition [modification]'], ['deposition of H3K4me [modification]']]}\n",
      "-------------------\n",
      "{'158817': [['gene(s) for primexine formation [gene]'], ['gene(s) regulating pollen wall and coat formation processes [gene]'], ['23 gene(s) associated with pollen wall formation [gene]'], ['**gene(s) involved in pollen exine formation [phenotype]**', 'gene(s) related to pollen exine formation [phenotype]'], ['splicing of gene(s) involved in pollen wall formation [process]'], ['gene(s) involved in outer pollen wall formation [gene]'], ['pre-mRNA splicing of gene(s) for pollen tube development [process]'], ['splicing of a pollen cell wall gene(s) [gene]'], ['**pollen exine formation gene(s) [gene]**', 'gene(s) related to exine formation [gene]'], ['regulator of primexine formation [process]'], ['expression of several gene(s) involved in pollen wall deposition and pollen mitotic division [phenotype]']]}\n",
      "-------------------\n",
      "{'89325': [['embryogenic callus initiation [treatment]'], ['callus induction rate [process]'], ['callus induction [treatment]'], ['Vascular cell induction culture system [treatment]'], ['Callus etiolation [treatment]'], ['callus-induction step [treatment]'], ['ectopic ground cell divisions [treatment]'], ['Knowledge of callus generation [treatment]'], ['indirect regeneration [treatment]', '**regeneration from callus [treatment]**'], ['Efficient in vitro callus generation [process]'], ['PFFA [treatment]'], ['Callus production [treatment]'], ['callus induction in vitro [treatment]'], ['NEC [process]'], ['SDC formation [treatment]'], ['transformation from non-embryogenic callus to embryogenic callus [process]'], ['callus induction [process]'], ['PLU treatment [treatment]'], ['long term callus propagation [treatment]'], ['callus propagation [treatment]']]}\n",
      "-------------------\n",
      "{'140128': [['CAMTA1 dependent positively regulated gene(s) [gene]'], ['**upregulated CKX gene(s) [gene]**', 'up-regulation of CKX gene(s) [gene]'], ['increase in CWINV2 [gene]']]}\n",
      "-------------------\n",
      "{'47996': [['**ARP2/3 mutants [mutant]**', 'lack of Arp2/3 subunits [mutant]', 'loss of ARP2/3 complex subunits [mutant]', 'mutations affecting ARP2/3 complex subunits [mutant]', 'loss of ARP2/3 complex [mutant]', 'impairment of ARP2/3 function [mutant]', 'disruption of ARP2/3 complex function [mutant]', 'mutations affecting ARP2/3 complex [mutant]', 'Mutations disrupting the ARP2/3 complex [mutant]', 'mutants lacking functional ARP2/3 complex [mutant]', 'true ARP2/3 mutant [mutant]', 'ARP2/3 complex subunit mutants [mutant]'], ['**arp2 mutant [mutant]**', 'arp2-G302Y mutants [mutant]', 'arp2 (actin-related protein(s) 2) mutant [mutant]', 'arp2 mutants [mutant]'], ['mutations in ARP2 [gene]'], ['arp2/arp2 mutant cells [cell type]'], ['**arpc1a/b-c1 mutants [mutant]**', 'arpc1a/b-c1 mutant [mutant]'], ['arpc1 mutant in moss [mutant]'], ['arpc2 mutant [mutant]'], ['arpc3-c1 mutants [mutant]'], ['arpc3 mutants [mutant]'], ['ARP3 mutant [mutant]'], ['arc3 single mutants [mutant]'], ['**ARPC4 [mutant]**', 'arpc4 mutants [mutant]'], ['arpc4 pavement cells [mutant]'], ['**arpc5 [mutant]**', 'arpc5 mutants [mutant]'], ['ARPC5/p16 mutant [mutant]'], ['scar1234, brk1, or arpc5 mutants [mutant]'], ['Arp6 [mutant]'], ['ARP5-defective animal cells [mutant]'], ['mutants of ARP2/3-SCAR complex subunits [mutant]'], ['mutants of the Arp2/3 complex pathway [mutant]'], ['mutants in the actin polymerization-regulating ARP2/3 pathway [mutant]'], ['**Mutations in APC/C subunits [mutant]**', 'mutations in APC subunits [mutant]'], ['APC/C gain-of-function simulations [mutant]'], ['APC11 mutant [mutant]'], ['APC1 mutant [mutant]'], ['APC4 mutant [mutant]'], ['APC8 disruption [mutant]'], ['APC8 mutant [mutant]'], ['APC13 disruption [mutant]'], ['APC13 mutant [mutant]'], ['defects in the structure of actin filaments [cell structure]']]}\n",
      "-------------------\n",
      "{'173945': [['post-translational activation of MGDG synthesis [treatment]'], ['post-translational regulation of MGDG synthesis [treatment]'], ['MGD activity [phenotype]'], ['**MGDG synthase activity [metabolite]**', 'MGDG synthesis activity [metabolite]'], ['lipid class synthesized by GGGT [enzyme]'], ['MGD2/3-mediated galactolipid biosynthesis [metabolite]'], ['first step of galactolipid synthesis [process]']]}\n",
      "-------------------\n",
      "{'4789': [['response of plant(s) cells to environmental stimuli [process]'], ['signaling responses [process]'], ['changes in response to environmental and developmental signals [process]'], [\"plant's cellular responses [cell type]\"]]}\n",
      "-------------------\n",
      "{'99646': [['90% of accessions from Relict groups [organism]'], ['Swedish genotypes of L. campestre [organism]'], ['P8 (Est-1 x RRS7) [organism]'], ['P129 (C24 x RRS10) [organism]'], ['Bor-4 x NFA-8 (P20) [organism]'], ['Col and Ler populations [organism]'], ['Catalonian HKT1;1LLS-harboring samples [organism]'], ['loblolly pine trees from two seed sources [organism]'], ['Ler RNA-pool [organism]'], ['allergic sera [phenotype]'], ['UFRGS 8 x Pc68/5*Starter population [organism]']]}\n",
      "-------------------\n",
      "{'21683': [['**cold tolerance of rice [phenotype]**', 'cold tolerance of rice plant(s) [phenotype]', 'rice growth at low temperature [phenotype]', 'cold tolerance in rice (Oryza sativa) [phenotype]', 'tolerance of rice to low temperature stress [phenotype]', 'cold tolerance in rice [phenotype]', 'cold tolerance on rice plant(s) [phenotype]', 'cold resistance of rice [phenotype]', 'tolerance of rice to cold [phenotype]'], ['cold tolerance of japonica rice [phenotype]'], ['stress tolerance at low temperature [phenotype]'], ['cold tolerance of rice varieties [phenotype]'], ['cold tolerance in rice and sugarcane [phenotype]'], ['cold tolerance of rice seedlings [phenotype]']]}\n",
      "-------------------\n",
      "{'89238': [['genetic transformation of RCBr [process]', '**RCBr transformation [process]**', 'transformation of RCBr [process]']]}\n",
      "-------------------\n",
      "{'59223': [['Cd uptake by Peltigera horizontalis [organism]'], ['Increasing the lipid content of C. ellipsoidea [organism]']]}\n",
      "-------------------\n",
      "{'32571': [['**cell membrane lipids [metabolite]**', 'plant(s) membrane lipids [metabolite]', 'plant(s) plasma membrane lipids [metabolite]'], ['galactolipids [compound]'], ['sulfolipids [compound]'], ['Lipid bilayer matrix [subcellular compartment]'], ['**Lipid membranes [organ]**', 'lipids of the plant(s) cell membrane [organ]'], ['Membrane lipids [cell component]'], ['various membrane, storage, and extracellular lipids [metabolite]'], ['surface sterols [metabolite]'], ['lipid order in membranes of living plant(s) cells [study]'], ['major class of lipids in plant(s) [organism]'], ['sphingolipids in plant(s) [molecule]'], ['cell lipids [tissue]'], ['sterols in the plant(s) cell membranes [metabolite]'], ['major constituents in plant(s) membrane lipids [metabolite]'], ['leaf membrane lipids [metabolite]'], ['cuticular membrane [metabolite]'], ['storage and membrane lipids of seeds [organ]'], ['membrane lipids in plant(s) chloroplast [organ]'], ['lipid-based barrier [tissue]'], ['most profuse lipid of photosynthetic membranes [organ]']]}\n",
      "-------------------\n",
      "{'9744': [['**defects in development [phenotype]**', 'defects in developmental processes [phenotype]'], ['defective [phenotype]'], ['defects at any stage of reproductive development [phenotype]'], ['**defects in growth and development [phenotype]**', 'defects of growth and development [phenotype]'], ['defects in neural development [phenotype]'], ['defects in host development [process]'], ['defects in adult stage [phenotype]']]}\n",
      "-------------------\n",
      "{'93167': [['biofilms developed in response to xylan [treatment]'], ['bacterial exopolysaccharides (EPS) production [phenomenon]']]}\n",
      "-------------------\n",
      "{'108297': [['assembly on methylated DNA [metabolite]'], ['methylated cytosines directly [process]', '**methylate DNA [process]**', 'methylated [process]', 'Addition of a methyl group [process]', 'methylate target DNA [process]', 'efficiently methylated [process]'], ['cDNA catalyzed three sequential methylations [reaction]']]}\n",
      "-------------------\n",
      "{'42453': [['phyB Pfr inactivation [phenotype]'], ['extremely low levels of BRP [phenotype]'], ['near-complete loss of editing at psbF-C77 [phenotype]'], ['complete disruption of B1L expression [phenotype]'], ['complete FEX inactivation [phenotype]'], ['loss of coenzyme binding [phenotype]'], ['inactivity of the enzyme [phenotype]'], ['BON1 inactive [phenotype]'], ['a different intron [phenotype]']]}\n",
      "-------------------\n",
      "{'173614': [['ascorbate metabolism [metabolite]'], ['reticuline metabolism [process]'], ['Citrate metabolism [process]'], ['oxalate metabolism [process]'], ['oleate metabolism [process]'], ['Metabolism of ΔL [metabolite]'], ['fumarate metabolism [metabolite]'], ['ferulic acid metabolism [metabolite]'], ['**isocitrate metabolism [process]**', 'isocitrate metabolic process [process]'], ['metabolism of propionate [process]'], ['metabolism of isobutyrate [process]'], ['propionate metabolism [metabolite]'], ['metabolism of D-Asp [metabolite]']]}\n",
      "-------------------\n",
      "{'87710': [['UV-C [phenotype]'], ['UV [light]']]}\n",
      "-------------------\n",
      "{'55327': [['RanBP1 [protein domain]'], ['RanBP2 [protein domain]'], ['RanBP2 nucleoporin [protein]'], ['major RBP in REP [protein]'], ['REP binding [protein]'], ['many RPs and RBFs [protein]'], ['Many RBPs [protein]'], ['RBP37 [protein]'], ['13 selected RBPs [protein]'], ['three RBPs [protein]'], ['11 RBPs [protein]'], ['RBP [protein region]']]}\n",
      "-------------------\n",
      "{'3739': [['barley immune response [phenotype]'], ['tobacco immune-related responses [phenotype]', 'defense responses in tobacco [phenotype]', 'tobacco defense response [phenotype]', '**immune responses in tobacco [phenotype]**'], ['immune response in N. benthamiana [phenotype]'], ['immune responses in cotton [phenotype]', '**cotton immune response [phenotype]**'], ['wheat immune response [phenotype]'], ['immune response of rice (Oryza sativa) [phenotype]', '**immune response in rice [phenotype]**', 'rice immune responses [phenotype]'], ['immune response of tomato (Solanum lycopersicum) [phenotype]'], ['immune response of crops [phenotype]'], ['cassava immune responses [phenotype]'], ['immune response in tobacco suspension culture cells [phenotype]'], ['immune responses in soybean [phenotype]']]}\n",
      "-------------------\n",
      "{'136317': [['Guard cell-targeted over-expression of betaCA1 [gene]'], ['Guard cell-targeted over-expression of betaCA4 [gene]'], ['**Guard-cell-targeted overexpression of A. thaliana Hexokinase 1 [gene]**', 'Guard-cell-targeted overexpression of hexokinase [gene]', 'guard-cell-targeted overexpression of AtHXK1 [gene]'], ['**Guard-cell-specific expression of AtHXK1 [gene]**', 'Guard-cell-targeted expression of AtHXK1 [gene]']]}\n",
      "-------------------\n",
      "{'75426': [['MTI receptors [protein]'], ['unidentified proteinaceous MAMPs or DAMPs [metabolite]'], ['MAMP motif of type 1 NLPs [protein]'], ['NAMPs [molecule]'], ['PAMPs/MAMPs [molecule]'], ['**DAMP Pep1 [metabolite]**', 'damage-associated molecular pattern peptide 1 (Pep1) [metabolite]'], ['MAMP epitopes [molecule]'], ['peptide MAMPs [metabolite]'], ['DAMP AtPep1 [treatment]'], ['DAMP Pep1 [peptide]'], ['PAMP signals [molecule]'], ['MAMP elicitors elf18 [molecule]'], ['separate PAMP elicitors [protein]'], ['inducibly secrete affinity-tagged proteinaneous MAMPs [molecule]']]}\n",
      "-------------------\n",
      "{'88040': [['blue light sensors [treatment]'], ['blue-light receptor [treatment]'], ['blue light component [treatment]'], ['blue light-specific components [treatment]']]}\n",
      "-------------------\n",
      "{'81258': [['D-SCRIPT [method]'], ['Data Mining Prediction (DMP) method [method]'], ['DBP prediction models [method]']]}\n",
      "-------------------\n",
      "{'103010': [['Fusarium moniliforme [organism]'], ['F. sporotrichioides [organism]'], ['F. anthophilum [organism]'], ['F. dlamini [organism]'], ['F. napiforme [organism]'], ['**R. fascians [organism]**', 'R.fascians [organism]'], ['F. culmorum [organism]'], ['Fusarium eumartii [organism]'], ['Fusarium equiseti [organism]'], ['Fusarium semitectum [organism]'], ['Fusarium dactylidis [organism]'], ['Fusarium tricinctum [organism]'], ['Fusarium heterosporum [organism]'], ['Fusarium proliferatum [organism]'], ['Fusarium fujikuroi [organism]'], ['C. fioriniae [organism]'], ['Corynespora cassiicola [organism]'], ['Fusarium venenatum [organism]'], ['na2 [organism]'], ['F. circinatum [organism]'], ['pathogenic fungus Alternaria brassicicola [organism]'], ['C. foetidum [organism]'], ['**Fusarium culmorum 891 [organism]**', 'F. culmorum 891 [organism]'], ['rust fungus Cromartium quercuum f. sp. fusiforme [organism]'], ['Sphaerotheca fusca [organism]'], ['C. fulcum race 4 [organism]'], ['fungus Cladosporium Fulvum [organism]'], ['Fusarium culmorum fungi [organism]'], ['Fusarium culmorum 334 [organism]'], ['Fusarium delphinoides [organism]'], ['fungus Alternaria alternata [organism]'], ['Fusarium solani var. coeruleum [organism]'], ['closely related Fusarium species [organism]']]}\n",
      "-------------------\n",
      "{'39284': [['AtERF014-OE plant(s) [mutant]'], ['AtPDF1.1 OE plant(s) [mutant]'], ['similarities with WS induced responses in OE-AhERF-VII plant(s) [mutant]'], ['OP of 35S-TaABC1 plant(s) [mutant]'], ['Enhanced ABA sensitivity of COP1 cyt-1 [mutant]'], ['**AtAVP1 overexpression [mutant]**', 'OE-AVP1 plant(s) [mutant]'], ['AtSAG-overexpressing plant(s) [mutant]'], ['ERF96 overexpression [mutant]'], ['AtNPR1-overexpressing sweet orange lines [mutant]'], ['AtTZF1 overexpression plant(s) [mutant]'], ['AtTZF2/AtC3H20/AtOZF1 overexpression [mutant]'], ['NAC103 over-expression plant(s) [mutant]'], ['AHL-overexpressing transgenic plant(s) [mutant]'], ['atozf2 antisense lines [mutant]'], ['AtOZF2-overexpressing plant(s) [mutant]'], ['ASR1-OE plant(s) [mutant]'], ['OST1 overexpressing lines [mutant]'], ['AtADH1 overexpressing plant(s) [mutant]'], ['Enhanced ethylene response 1 (eer1) [mutant]'], ['enhanced ethylene-response (eer) mutants [mutant]'], ['BSabi bundle sheath cells [cell type]'], ['AtPP2CF1oe plant(s) [mutant]'], ['AGL21-overexpressing plant(s) [mutant]'], ['Insensitivity of ATL31-overexpressing plant(s) [mutant]'], ['radiation-resistant transcription of atm [phenotype]'], ['enhanced susceptibility of AtPIIN_08944 plant(s) to Hpa [phenotype]'], [\"increased tolerance of AtF6'H1-transgenic plant(s) to SDS [phenotype]\"], ['heat-hypersensitive phenotypes of AtMYBS1-OE and atmybs1max1 plant(s) [phenotype]'], ['AtPDI1 overexpression plant(s) [mutant]'], ['enhanced disease resistance in AtSAG101-overexpressing plant(s) [mutant]'], ['enhanced tolerance to B toxicity [phenotype]'], ['AtCSP2-overexpressing seeds [mutant]'], ['agb1-3 plant(s) [mutant]']]}\n",
      "-------------------\n",
      "{'91177': [['EC50 value of lincomycin [treatment]'], ['IC50 values of lincomycin [treatment]'], ['7dSh concentration of 5 microg mL-1 [phenotype]'], ['optimal labeled LPS E. coli concentration [treatment]'], ['50 µg/mL LPS [treatment]']]}\n",
      "-------------------\n",
      "{'88364': [['low-cost methodology [treatment]', '**cost-effective method [treatment]**'], ['rapid selection methods [treatment]'], ['CAM Biodesign [approach]'], ['CAM Biodesign [treatment]'], ['ALSV vector technology [treatment]'], ['screening approaches and protocols [treatment]'], ['High-efficiency technologies [treatment]'], ['improved algorithms [tool]'], ['improved algorithms [treatment]'], ['Deep-learning-based approach [treatment]'], ['Non-invasive, high-throughput screening methods [treatment]'], ['ALSV vector technology even more useful [technique]'], ['attractive alternative [treatment]'], ['development of HTPP systems [treatment]'], ['time-efficient screening [research method]'], ['more efficient screening [treatment]'], ['aggressive quality control [treatment]'], ['developing new screening techniques [treatment]'], ['sophisticated strategies [treatment]']]}\n",
      "-------------------\n",
      "{'31465': [['**stomatal patterning [process]**', 'Stomata patterning [process]'], ['stomatal differentiation programs [process]'], ['regulation of stomatal complex patterning [activity]'], ['**Regulation of stomatal development [process]**', 'control of stomatal development [process]', 'stomatal development regulation [process]'], ['Genetic and cell biological mechanisms [process]'], ['Stomatal pore formation [process]'], ['Key players for stomatal development [process]'], ['leaf stomatal development pathways [process]'], ['**Stomatal Development Pathways [process]**', 'stomatal development pathway [process]'], ['development and patterning of stomata [process]'], ['role of BASL in controlling stomatal ACD [process]'], ['stomatal spatial patterning [process]'], ['stomatal-related processes [process]'], ['regulator of stomatal patterning [process]'], ['stomatal development and differentiation [process]'], ['stomatal development and/or movement [function]'], ['stomatal patterning process [process]'], ['in stomatal development [process]'], ['SPCH in early stage of stomatal development [process]'], ['stomatal lineage decisions [process]'], ['understanding of stomatal development [process]'], ['mechanisms and gene(s) underlying stomatal development [process]'], ['stomatal number establishment [process]'], ['cellular processes of stomata [cell type]'], ['MAPK-dependent processes [process]'], ['control of stomatal patterning and differentiation [process]'], ['proper stomatal patterning [process]'], ['elucidation of underlying molecular mechanisms [process]'], ['stomatal development signal regulation network [process]']]}\n",
      "-------------------\n",
      "{'45197': [['Knock-out mutations in RBAM017410 [mutant]'], ['bam4 knockout mutants [mutant]'], ['Knockout mutants lacking BAM1 [mutant]'], ['Marchantia polymorpha HSFB1 knockout mutants [mutant]'], ['SAMBA knock-out mutants [mutant]']]}\n",
      "-------------------\n",
      "{'57719': [['zma-pre-MIR167h [gene]'], ['zma-pre-MIR167c [gene]'], ['zma-pre-MIR167e [gene]'], ['zma-pre-MIR167j [gene]'], ['zma-pre-MIR167f [gene]'], ['zma-pre-MIR167i [gene]']]}\n",
      "-------------------\n",
      "{'12113': [['chartreuse color [phenotype]'], ['white bulbs [phenotype]'], ['chartreuse onion bulb color [phenotype]'], ['chartreuse bulb color [phenotype]'], ['chartreuse onion [phenotype]']]}\n",
      "-------------------\n",
      "{'136929': [['down-regulation of CCA1 [gene]'], ['Transcriptional downregulation of ANS [gene]'], ['downregulation of SOC1 [gene]'], ['decreased expression of NIGT1 transcriptional repressor gene(s) [gene]'], ['down-regulation of WUS expression [gene expression]', '**down-regulated expression of WUS [gene]**'], ['down-regulated ORE1 [gene]'], ['down-regulated SGR1/NYE1 [gene]'], ['down-regulated expression of ZIF1 [gene]'], ['down-regulated expression of IRT1 [gene]'], ['down-regulation of Ade_DRM1_1G transcript [gene]'], ['downregulation of DOG1 expression [gene]'], ['down-regulation of beta-amylase 1 (BAM1) [gene]'], ['down-regulation of ALS3 [gene]']]}\n",
      "-------------------\n",
      "{'89624': [['Auxin sensing [treatment]', '**auxin perception [treatment]**', 'Auxin perception/signaling [treatment]', 'binding auxin [treatment]', 'normal auxin perception and signaling [treatment]', 'transduction of the auxin signal [treatment]'], ['auxin sensing via TMKs [treatment]'], ['**temporal integration of the auxin signal [treatment]**', 'temporal integration of auxin [treatment]'], ['auxin-induced TCS signal [phenotype]'], ['nuclear auxin perception [treatment]']]}\n",
      "-------------------\n",
      "{'165747': [['AcGGM [metabolite]'], ['A. thaliana AcGGM [metabolite]'], ['Galactose (Gal) monosaccharides [metabolite]'], ['MeGlcA-Gal [carbohydrate]'], ['**GalA and Rha [metabolite]**', 'galacturonic acid and rhamnose [metabolite]'], ['Gal and/or Man [metabolite]', 'Gal and Man sugars [metabolite]', '**Gal and Man [metabolite]**'], ['Cya 3-(Xyl-Glc-Gal) [metabolite]'], ['GalA, Gal, and Xyl as major non-cellulosic monosaccharides in A. thaliana leaf [metabolite]'], ['GalA, Ara, Rha, and Gal [metabolite]'], ['Glc as well as Gal, mannose, and xylose [metabolite]'], ['α-Gal [metabolite]'], ['dominant monosaccharide in pectic polysaccharides [metabolite]']]}\n",
      "-------------------\n",
      "{'162510': [['**increased accumulation of ROS [metabolite]**', 'excess accumulation of ROS [metabolite]', 'Stress-triggered ROS accumulation [metabolite]', 'enhanced accumulation of ROS [metabolite]', 'Continued accumulation of ROS [metabolite]', 'Early accumulation of ROS [metabolite]', 'enhanced ROS accumulation [metabolite]', 'greater accumulation of ROS [metabolite]', 'accelerated ROS accumulation [metabolite]', 'high accumulation of reactive oxygen species (ROS) [metabolite]', 'Rapid accumulation of ROS [metabolite]', 'exaggerated production and accumulation of ROS [metabolite]', 'elicited accumulation of ROS [metabolite]', 'increase in ROS accumulation [metabolite]', 'rapid generation and accumulation of ROS [metabolite]', 'higher ROS accumulation [metabolite]', 'strong ROS accumulation [metabolite]', 'higher accumulation of reactive oxygen species (ROS) [metabolite]', 'high levels of ROS accumulation [metabolite]', 'macroscopic accumulation of ROS [metabolite]', 'greater accumulation of reactive oxygen species (ROS) [metabolite]', 'accelerated generation/accumulation of reactive oxygen species (ROS) [metabolite]', 'elevated ROS accumulation [metabolite]', 'altered accumulation of ROS [metabolite]', 'stimulation of ROS accumulation [metabolite]', 'strong accumulation of ROS [metabolite]', 'enhanced reactive oxygen species (ROS) accumulation [metabolite]', 'high-level accumulation of ROS [metabolite]'], ['Rapid intracellular accumulation of ROS [metabolite]', 'Increased accumulation of cytoplasmic ROS [metabolite]', '**accumulation of cellular ROS [metabolite]**'], ['accumulation of cytosolic and lipid ROS [metabolite]'], ['**ROS accumulation in the chloroplast [metabolite]**', 'elevated accumulation of chloroplast-derived ROS [metabolite]', 'higher levels of reactive oxygen species (ROS) in chloroplast [metabolite]'], ['apoplastic accumulation of ROS [metabolite]'], ['HS-induced ROS accumulation [metabolite]'], ['Al-induced increase in ROS [metabolite]'], ['salt-elicited excessive ROS [metabolite]', '**accumulation of ROS under salt stress [metabolite]**', 'salt stress-induced ROS levels [metabolite]', 'salinity-induced ROS accumulation [metabolite]'], ['increase of ROS accumulation under drought and salinity stresses [metabolite]'], ['heat-induced ROS accumulation [metabolite]'], ['cold-induced ROS accumulation [metabolite]'], ['accumulation of O2- [metabolite]'], ['accumulation of internal oxidative stress [metabolite]'], ['heat-induced ROS accumulation [process]'], ['risk of further ROS accumulation [process]'], ['accumulation of ROS during stress [process]']]}\n",
      "-------------------\n",
      "{'46706': [['**ga1-3 germination [mutant]**', 'ga1-3 mutant seeds [mutant]', 'gibberellic acid-requiring1-3 [mutant]'], ['GA auxotroph ga1 [mutant]'], ['**ga1-3 germination [phenotype]**', 'germination of GA biosynthesis mutant ga1-3 [phenotype]'], ['non-germinating phenotype of ga1 [phenotype]']]}\n",
      "-------------------\n",
      "{'168303': [['PA and FA accumulation [metabolite]'], ['accumulation of raphanusamic acid [metabolite]', '**raphanusamic acid accumulation [metabolite]**'], ['accumulation patterns of CP [metabolite]'], ['polyprenol accumulation [metabolite]'], ['accumulation of iP [metabolite]'], ['accumulation of allantoin [process]'], ['overaccumulation of higher polyamines [process]'], ['Accumulation of propanedioic acid [metabolite]'], ['Polyamine accumulation [process]'], ['pathogen-induced putrescine accumulation [metabolite]'], ['accumulation of punicic acid [metabolite]', '**PAA accumulation [metabolite]**', 'accumulates PAA [metabolite]'], ['accumulation of shikimate [phenotype]'], ['accumulation of phenylpropanoids [metabolite]'], ['accumulation of caffeoylputrescine [metabolite]'], ['increased accumulation of artemisinic acids [metabolite]'], ['accumulation of tetrapyrroles [metabolite]'], ['initial decrease followed by substantial increase in diamine putrescine levels [metabolite]'], ['increase in pCA [metabolite]'], ['accumulation of 12-carboxy-JA-Ile [metabolite]'], ['accumulation tendency of capsaicinoid [metabolite]'], ['capsaicin accumulation [metabolite]'], ['HCAA-accumulation [phenotype]'], ['accumulation of HGA [metabolite]'], ['accumulation of 8-epixanthatin [metabolite]'], ['puerarin accumulation [metabolite]'], ['accumulation of protocatechuate [metabolite]'], ['accumulation of large amount of Trp [metabolite]'], ['accumulation of oxidosqualene [metabolite]'], ['accumulation of transcripts for plantacyanin [process]'], ['accumulation of sinapate [chemical compound]'], ['peak accumulation of camptothecin [metabolite]']]}\n",
      "-------------------\n",
      "{'97961': [['**Ricinus communis (castor bean) [organism]**', 'Ricinus comunis [organism]', 'castor plant(s) [organism]', 'Castor bean plant(s) [organism]', 'Ricinus communis L. [organism]', 'castor oil plant(s) [organism]', 'eukaryota (Ricinus communis) [organism]'], ['Ricinus [organism]'], ['**tigernut (Cyperus esculentus) [organism]**', 'tiger nut [organism]', 'oil-rich tuber of Cyperus esculentus [organism]'], ['cashew plant(s) [organism]'], ['**Physic nut or Barbados nut (Jatropha curcas) [organism]**', 'Adult J. curcas [organism]', 'Barbados nut (Jatropha) [organism]', 'physic nuts [organism]'], ['Coconut (Cocos nucifera L.) [organism]'], ['Ricinus communis seed cakes [organism]'], ['burning bush [organism]'], ['from Cocos nucifera [organism]'], ['Ceratonia siliqua [organism]'], ['Ricinus communis (RcLPCAT) [organism]']]}\n",
      "-------------------\n",
      "{'16321': [['longer hypocotyls and root(s) [phenotype]'], ['maximum increase in length of Solanum lycopersicum root(s) and shoots [phenotype]'], ['slightly longer root(s) [phenotype]'], ['slightly longer root(s) and hypocotyls [phenotype]'], ['slightly longer primary root(s) [phenotype]'], ['root(s) about 20% longer [phenotype]']]}\n",
      "-------------------\n",
      "{'54693': [['stress specific transcription factors [protein]'], ['transcriptional enhancers [protein]'], ['Forkhead (FHL1) transcription factor [protein]'], ['Transcription factor WRINKLED1 (WRI1) [protein]'], ['3 TFs [protein]'], ['SCW-regulating MYB protein(s) [protein]'], ['mRNA binding factors [protein]'], ['transcription factor FIT [protein]'], ['STAT transcription factors [protein]'], ['Dormant transcription factors [protein]'], ['Membrane-associated transcription factors (MTFs) [protein]'], ['SPT23/MGA2 transcription factor [protein]'], ['Transcriptionally active form (p90) [protein]'], ['126 R2R3-MYB transcription factors [protein]'], ['UVB-responsive split transcription factor [protein]'], ['tissue-specific TFs [protein]'], ['MYB-like transcription factor [protein]'], ['TtNAC transcription factors (TFs) [protein]'], ['trihelix-GT transcription factors [protein]'], ['14 membrane-bound TFs [protein]'], ['SQUAMOSA promoter-binding protein-like (SPL) transcription factor [protein]', '**SPL transcription factor [protein]**'], ['Transcription factor SPEECHLESS (SPCH) [protein]'], ['trans-regulating factors [protein]'], ['regulating CnNAC TFs [protein]'], ['MYBS1 transcription factor [protein]'], ['Mammalian PAS transcription factors [protein]'], ['C4HDZ transcription factors [protein]'], ['binding site of Forkhead (FHL1) transcription factor [protein]'], ['TFIIA large subunit (TFIIA-L1) [protein]'], ['light and MYB transcription factors [protein]'], ['yeast transcription elongation factor Spt5 [protein]'], ['transcriptional repressor protein(s) VNI2 [protein]'], ['SCW-associated TFs [protein]'], ['100 transcription factors [protein]'], ['three dimethylase/mtTFB-like protein(s) [protein]'], ['initiation factor 3 [protein]'], ['translation initiation factor 4A [protein]'], ['cold-specific transcription factors (TFs) [protein]'], ['AGP1 transcription factor [protein]'], ['SRE-binding protein(s) (SREBP) transcription factor [protein]'], ['five common transcription factors [protein]'], ['Five transcription factors [protein]'], ['SND1-mediated transcription network [protein]'], ['SND1 transcription factor [protein]'], ['ABA-responsive bZIP-type transcription factors [protein]'], ['mammalian TRFs [protein]'], ['nuclear transcription factor B-type response regulators [protein]'], ['DNA-binding transcription factor [protein]'], ['protein(s) involved in transcription (AC2) [protein]'], ['regulatory factors PSY2 [protein]'], ['labile transcription factors [protein]'], ['binding sites of transcription factors [protein]'], ['21 known transcription factors [protein]'], ['REM transcription factors [protein]'], ['alpha-mating factor signal [protein]'], ['putative transcription factors (TFs) [protein]'], ['the action of specific TFs [protein]'], ['transcription factor ATAF1 [protein]'], ['diverse zinc finger transcription factors [protein]'], ['protein:protein interactions with TGA transcription factor family [protein]'], ['different ABCD transcription factors [protein]'], ['maize HD-Zip IV transcription factor [protein]'], ['essential cofactor of MYBs and WDRs [protein]'], ['several MYB TFs [protein]'], ['intracellular localization of GmMYB176 transcription factor [protein]'], ['TRF1 and TRF2 binding [protein]'], ['G-box regulatory factor [protein]'], ['general regulatory factor [protein]'], ['G-box factor 14-3-3 homolog [protein]']]}\n",
      "-------------------\n",
      "{'98533': [['**Ler wild-type background [organism]**', 'wild-type background Ler [organism]', 'wild-type background (Ler) [organism]'], ['**WT Ler-0 plant(s) [organism]**', 'wild-type Ler-0 plant(s) [organism]', 'L er-0 WT [organism]'], ['LR density of WT [organism]'], ['Wild-type TCLs [organism]'], ['wild-type RtSS plant(s) [organism]'], ['untransformed parental WT L. japonica [organism]'], ['WT L. japonica [organism]'], ['wild-type root(s) tips (Ler ecotype) [organism]'], ['L. japonicus WT [organism]'], ['WT Ler-0 early seedling development [organism]'], ['wild-type rhizospheres [organism]'], ['L. erecta wild-type (WT) plant(s) [organism]'], ['larvae from wild-type leaf [organism]'], ['wild-type (Ler) developing seeds [organism]'], ['parental Ler seedlings [genetic background]'], ['WT LRK-516 [organism]'], ['wild-type Ler pollen [organism]'], ['L1 WT rootstocks [organ]'], ['wild-type Ler plant(s) [organism]'], ['Landsberg erecta WT [organism]'], ['callus of WT L. japonica [organism]'], ['wild and LC population [organism]'], ['induced wild-type leaf [organ]'], ['WT LRP [organism]'], ['noninoculated Ler-0 [organism]'], ['SAR-induced wild-type leaf [organ]'], ['wild-type LRs [organism]'], ['VLD wild-type R108 plant(s) [organism]'], ['wild-type (CENH3 +/+) Landsberg erecta [organism]'], ['Col-wt parent [organism]'], ['lbo/WT graft [organism]'], ['Ler WT accessions [organism]']]}\n",
      "-------------------\n",
      "{'36476': [['MT bending segments [phenotype]'], ['MT growth [phenotype]'], ['MT band formation [phenotype]', '**MT bands [phenotype]**', 'all bands [phenotype]', 'localizing MT bands [phenotype]'], ['Number of MT filaments [phenotype]', 'MT filament number [phenotype]', '**MT number [phenotype]**'], ['MT clustering [phenotype]'], ['MT bundles [phenotype]'], ['MTs in trichoblasts of differentiation zone [phenotype]'], ['MT amplification [phenotype]'], ['**MT length [phenotype]**', 'length of the MT [phenotype]'], ['no visible difference in amount of microtubules bound to plasma membrane [phenotype]']]}\n",
      "-------------------\n",
      "{'48138': [['Semi-dominant or dominant mutants of tubulin gene(s) [mutant]'], ['pol V single mutants [mutant]'], ['G-protein single mutants [mutant]'], ['initiation mutants [mutant]'], ['profilin mutants [mutant]'], ['ABC transporter mutants [mutant]'], ['Acetyltransferase mutants [mutant]'], ['CHLH missense mutants [mutant]'], ['dnt mutants [mutant]'], ['**actin mutants [mutant]**', 'a variety of actin-affected mutants [mutant]', 'key actin mutants [mutant]'], ['conformation change mutants [mutant]'], ['CRAC mutants [mutant]'], ['Disruption of LDAPs [mutant]'], ['dme mutants [mutant]'], ['mutants of small HSPs [mutant]'], ['slow-cycling variants [mutant]'], ['PLUTO point mutants [mutant]'], ['AMPA mutants [mutant]'], ['Sec22 mutants [mutant]'], ['prolyl hydroxylase genetic mutants [mutant]'], ['signal-transfer mutants [mutant]'], ['disruption of interaction in sup lines [mutant]'], ['proline mutants [mutant]'], ['dimerization-deficient CRY2 mutants [mutant]'], ['trm-like mutants [mutant]'], ['CRI-1 and CRI-2 mutants [mutant]'], ['MP30 deletion mutants [mutant]'], ['gain-of-function DELLA-mutants [mutant]'], ['PslDELLA-overexpression [mutant]'], ['DELLA mutant protein(s) [mutant]'], ['Mutants stabilizing DELLA protein(s) [mutant]'], ['DELLA-Q mutants [mutant]'], ['mutants with altered DELLA activity [mutant]'], ['mutants of the TF [mutant]'], ['oligomerization mutants [mutant]'], ['mutant desaturase protein(s) [protein]'], ['dC series [mutant]'], ['Heat-shocked TBs [mutant]'], ['DSM mutants [mutant]'], ['MRP mutants [mutant]'], ['Mpmpk1 mutants [mutant]'], ['cam alleles [mutant]'], ['Six independent dmr1 mutants [mutant]'], ['various mutant transit peptides [mutant]'], ['mutant ER microsomes [mutant]'], ['Overexpression of \\tα-helix breaking mutant [mutant]'], ['Helical mutants [mutant]'], ['CRIB point mutants [mutant]'], ['Coexpressing mutants [mutant]'], ['cop / det / fus mutants [mutant]'], ['Partially functional alleles of TPC subunits [mutant]'], ['μD (μ3)-adaptin mutant [mutant]'], ['mutant pumps [mutant]'], ['Higher-order mutants of subtilases [mutant]'], ['mutated desaturases [mutant]'], ['dilysine mutants [mutant]'], ['ETT point mutants [mutant]'], ['HSR impaired mutants [mutant]'], ['CA and DN mutations [mutant]'], ['mutants of phosphatase [mutant]'], ['receptor recycling mutants [mutant]'], ['mutants deficient in chromophore incorporation [phenotype]'], ['mutants in polyproline-rich protein(s) formin [mutant]'], ['helicase mutants [mutant]'], ['over-expression of GTP-restricted mutants [mutant]'], ['functional impact of photoreceptor point mutations [mutant]'], ['mutants in blue-light photoreceptors [mutant]']]}\n",
      "-------------------\n",
      "{'43924': [['PLDzeta2 knockout mutant [mutant]'], ['knockout mutant of LPAT2 [mutant]'], ['GPAT1 knockout [mutant]'], ['Physcomitrella patens PIP2;1 knockouts [mutant]'], ['Physcomitrella patens PIP2;2 knockouts [mutant]'], ['**GPT2 knock out mutant [mutant]**', 'gpt2 knock out mutants [mutant]'], ['Z1 [mutant]'], ['Z2 [mutant]'], ['pplaIIIdelta-KO seeds [mutant]'], ['aapt2 KO [mutant]'], ['diacylglycerol kinase knockouts [mutant]'], ['PMAT1 knockouts (pmat1) [mutant]']]}\n",
      "-------------------\n",
      "{'76485': [['integrity of Ser-909 [protein domain]'], ['MSL8N [protein region]'], ['MSL9N [protein region]'], ['disordered MSL10N [protein]'], ['MSL9N [protein]'], ['MSL10N [protein]'], ['27 kDa engineered form of ES domain [mutant]'], ['closed state of Get3d [protein domain]'], ['DeltaCTD variant [protein domain]'], ['HA-CA-RACB-DeltaCSIL [protein]'], ['IDR of Mpo DFE [protein domain]'], ['disorder in bridge domain [subcellular compartment]'], ['BAK1-5 CD [protein]'], ['Mutation in Sec7 domain [protein domain]'], ['Mg(2+)-free form of ETRRD [protein domain]'], ['full-length of SlVKOR with Trx-like domain without the transit peptide [protein]'], ['H2 interface [protein domain]'], ['8-10% [protein domain]'], ['MBD6CDelta [protein]'], ['disorder of MSL10N [protein region]'], ['mOD [protein variant]'], ['mNES/EAR [protein variant]'], ['76% of ES domain to be disordered [phenotype]'], ['cleavage of MLD from SYMRK [phenotype]'], ['intra-molecular disulfide of monomeric STN7 [protein]'], ['mPHD [protein domain]'], ['ORTH-mPHD [protein domain]'], ['RMSD of 0.17 nm in FAD upper region [protein domain]'], ['RMSD of 0.29 nm in PD [protein domain]'], ['RMSD of 0.25 nm in FAD lower region [protein domain]'], ['truncated match to manganese/iron superoxide dismutase C-terminal profile [protein domain]'], ['less stable than CMT-BAH [protein domain]'], ['amino acid exchange in CERK1 ectodomain [protein]'], ['activity of DeltaMORN [protein]']]}\n",
      "-------------------\n",
      "{'91774': [['EPS production by rhizobacterial strains [organism]'], ['Application of SAGs [treatment]'], ['**EPS treatment [treatment]**', 'EPS foliar sprays [treatment]', 'Exopolysaccharide-type PB [treatment]', 'EPS-treated plant(s) [treatment]'], ['EPS pretreated leaf [organism]'], ['ATP addition [treatment]'], ['Broccoli pre-treating with SA and chitosan [organism]'], ['auxin-producing endophytes [treatment]'], ['SID [treatment]'], ['serine protease inhibitor [treatment]'], ['silicon [compound]'], ['Biostimulant products [treatment]'], ['Amino acid-based biostimulants [treatment]'], ['PBS [treatment]'], ['6S-2 spore fertilizer [treatment]'], ['t-piceid [treatment]'], ['Enzymatic antioxidant system [treatment]'], ['SiO2-NPs [treatment]'], ['**S1T13 [treatment]**', 'Application of S1T13 [treatment]'], ['**SlPep [treatment]**', 'Exogenous SlPep application [treatment]'], ['anti-oxidative system (AOS) [treatment]'], ['exogenous fructose, glucose, sucrose and glycerol pre-treatments [treatment]'], ['Sodium silicate [treatment]'], ['ornithine supplementation [treatment]'], ['activation of scavenging enzymes [treatment]'], ['exogenous treatment with Xyl [treatment]'], ['Priming by Put [metabolite]'], ['GSH supplementation [metabolite]'], ['Ectopic expression of defensins [treatment]'], ['Seven newly reported HDIs [treatment]'], ['EBL pre-treatment [treatment]'], ['Exogenous supply of xyloglucan [treatment]'], ['Exogenous treatment with recombinant AsES [treatment]'], ['PSP1 [treatment]'], ['exogenous supplementation of A. thaliana root(s) exudates with RA [organ]'], ['Exogenous supplementation of Spm [treatment]'], ['Exogenous addition of protein(s) amino acids [treatment]'], ['active PEX [treatment]'], ['ascorbate supplementation [treatment]']]}\n",
      "-------------------\n",
      "{'160180': [['class E [gene]'], ['DEF-like MADS-box gene(s) [gene]'], ['class D [phenotype]'], ['class E [phenotype]']]}\n",
      "-------------------\n",
      "{'164658': [['proposed molecules [metabolite]', '**candidate compounds [metabolite]**', 'candidate molecules [metabolite]', 'candidate compound [metabolite]', 'identifying strong candidate compounds [metabolite]', 'possible candidates [metabolite]'], ['**Hits [metabolite]**', 'top scoring compounds [metabolite]', 'hit compound [metabolite]', 'hit compounds [metabolite]'], ['**lead compounds [metabolite]**', 'leads [metabolite]'], ['**target molecules [metabolite]**', 'all target compounds [metabolite]'], ['compounds of interest [metabolite]']]}\n",
      "-------------------\n",
      "{'108085': [['genic CG methylation patterns [epigenetic feature]'], ['CG-context methylation levels [epigenetic modification]'], ['CpG methylation [process]', '**CpG DNA methylation [process]**', 'DNA methylation at CG sites [process]'], ['Methylation at CG sites [sequence]'], ['DNA methylation on CG sites [phenomenon]'], ['methylation in the context of CpG [treatment]', '**DNA methylation of CpG sites [treatment]**'], ['CG-SMP [epigenetic modification]'], ['Promoter-specific CpG methylation [treatment]'], ['targeted CG methylation [modification]'], ['CG epigenetic patterning [phenotype]'], ['global CpG DNA methylation [metabolite]'], ['bacterial CG-specific DNA methyltransferase [gene]'], ['CRISPR-based CG-specific targeted DNA methylation systems [treatment]'], ['bacterial CG-specific DNA methyltransferase MQ1 [gene]'], ['Methylation changes in CG-DMRs [process]'], ['genic CG methylation levels [epigenetic modification]'], ['stochastic methylation mechanism in CG context [epigenetic modification]'], ['CG DNA methylation [epigenetic mechanism]'], ['Methylation of CCG sites [epigenetic mark]'], ['three cytosine methylation types [process]'], ['methylation at CNG sites [process]'], ['CNN methylation [process]'], ['three types of cytosine methylation [epigenetic modification]'], ['CpG methylation of DNA [epigenetic process]'], ['single CG epimutations [epigenetic modification]'], ['DNA methylated in their coding sequence at CG sites [process]'], ['DNA methylation at the CaMV35S enhancer [epigenetic modification]'], ['CNG methylation in pericentric heterochromatin [process]'], ['both methylation and demethylation in CpGs of coding regions of various gene(s) [gene]'], ['genic CpG methylation [gene feature]'], ['a CRISPR-based CG-specific targeted DNA methylation system for plant(s) [tool]'], ['trigger de novo DNA methylation in the CG context [process]', '**de novo CpG methylation [process]**'], ['CpNpG DNA methylation [modification]'], ['de novo CpNpG methylation [process]'], ['CpXpG methylation [epigenetic process]']]}\n",
      "-------------------\n",
      "{'14586': [['opposite direction of true leaf [phenotype]', 'switch from spiral to decussate phyllotaxy [phenotype]', 'opposite leaf [phenotype]', '**opposite phyllotaxy [phenotype]**', 'opposite and distichous phyllotaxis [phenotype]'], ['**left-handed twisting of cotyledon petioles [phenotype]**', 'left hand skewing of cotyledon petioles [phenotype]'], ['horizontally oriented, bipinnately compound leaf [phenotype]'], ['first trifoliate leaf growing opposite to the second [phenotype]'], ['opposite or verticillate leaf arrangement [phenotype]'], ['opposite cauline leaf [phenotype]'], ['handedness of spiral of leaf placement [phenotype]']]}\n",
      "-------------------\n",
      "{'102136': [['A. thaliana cell culture system [cell type]'], ['Vascular Cell Induction Culture System Using A. thaliana leaf (VISUAL) [treatment]'], ['A. thaliana cell culture line [cell type]'], ['A. thaliana liquid culture system [model system]'], ['undifferentiated A. thaliana cell culture line [cell type]']]}\n",
      "-------------------\n",
      "{'46329': [['nes1-6 [mutant]'], ['nes1-3 [mutant]']]}\n",
      "-------------------\n",
      "{'67918': [['LHCII phosphorylation by STN7 [protein]'], ['phosphorylation-induced degradation of PIFs [process]'], ['reversible phosphorylation of LHCII [phenotype]'], ['**ABA-dependent protein(s) phosphorylation [process]**', 'ABA-triggered protein(s) phosphorylation [process]'], ['high increase in LHCII phosphorylation [process]'], ['high-light-dependent phosphorylation of CP29 variants [protein]'], ['Phot1 autophosphorylation [gene]'], ['SPA1-mediated phosphorylation [gene]'], ['PPK-dependent phosphorylation [phenotype]'], ['Phot kinase activity [enzyme activity]'], ['Differential phosphorylation of TSP9 protein(s) [treatment]'], ['rapid phosphorylation of PIF1/3/4/5 [protein]'], ['phosphorylation of PSII protein(s) D1 [process]'], ['light-induced phosphorylation of H(+)-ATPase [protein]'], ['blue-light-induced autophosphorylation [treatment]'], ['CRY2 phosphorylation [process]'], ['The rapid reversibility of these acclimatory responses [process]'], ['protein(s) phosphorylation by STN8 [process]'], ['phot1 kinase activation [process]'], ['parallel increase in abundance and phosphorylation at the same transition [process]'], ['decreased phosphorylation in Response to biotic stimulus protein(s) [protein]'], ['activator of LHCII phosphorylation [process]'], ['increase in LHCII phosphorylation [process]'], ['pattern of LHCII phosphorylation [process]'], ['reversible phosphorylation of the LHC protein(s) [protein]'], ['Light-induced phosphorylation of PIF3 [process]'], ['transphosphorylation between phot1 molecules [process]'], ['transphosphorylation between phot1 and phot2 [process]'], ['Light-dependent receptor autophosphorylation [process]'], ['reduced phosphorylation of PIF3 [process]'], ['phosphorylation followed by degradation of components associated with phytochromes [protein]'], ['light-induced phosphorylation of PIFs [process]'], ['light/dark-regulated phosphorylation change of BZR1 [protein]'], ['successive phosphorylation reactions [process]'], ['STN7-kinase-dependent phosphorylation of LHCII [process]'], ['phyA-mediated PIF3 phosphorylation [protein modification]'], ['phot1 kinase activity [enzyme activity]']]}\n",
      "-------------------\n",
      "{'108001': [['methylation of H3 lysine 27 residues [metabolite]', 'histone 3 lysine 27 methylation levels [metabolite]', '**Histone H3 Lysine 27 methylation [metabolite]**', 'histone H3K27-methylation [metabolite]'], ['lysine 27 methylation of histone 3 [treatment]'], ['trimethylation of histone H3 Lys-27 (H3K27me3) [modification]'], ['histone H3 lysine 27 trimethylation (H3K27me3) accumulation [metabolite]', 'trimethylation on lysine 27 of histone3 (H3K27me3) [metabolite]', 'lysine 27 on histone H3 (H3K27me3) [metabolite]', 'trimethylation of lysine 27 of the histone H3 tail (H3K27me3) [metabolite]', '**histone 3 lysine27 trimethylation (H3K27me3) [metabolite]**', 'epigenetic modification H3K27me3 [metabolite]'], ['asymmetric methylation at Arg-17 of histone H3 [metabolite]']]}\n",
      "-------------------\n",
      "{'4439': [['an experimental system to identify mechanisms of cold tolerance [phenotype]', '**mechanisms for handling low-temperature stress [phenotype]**', 'CS response and tolerance [phenotype]'], ['better management systems [phenotype]']]}\n",
      "-------------------\n",
      "{'114153': [['P1-DR construct [construct]'], ['Reporter gene(s) assays [treatment]', '**Reporter assays [treatment]**'], ['COLORFUL-PR1pro reporter [treatment]'], ['COLORFUL-PDF1.2apro reporter [treatment]'], ['expression, greening, and reporter analyses [technique]'], ['Reporter activity [process]'], ['Reporters [treatment]'], ['Reporter gene(s) analyses [experiment]'], ['**dual reporter system [treatment]**', 'dual-reporter system [treatment]'], ['output reporters [treatment]'], ['pSHP22kb reporter [treatment]'], ['pSHP21kb reporter [treatment]'], ['pSHP22kb-3XmCArG reporter [treatment]'], ['Reporter assays [method]'], ['reporter interactions [treatment]'], ['DR5 levels [gene expression]'], ['**dual expression anatomy lines [treatment]**', 'dual expression and anatomical marker lines [treatment]'], ['DEAL marker [treatment]'], ['Marker line [treatment]'], ['Reporter gene(s) technology [method]'], ['Functional reporter analysis [treatment]'], ['transcriptional reporters [resource]'], ['transcriptional reporters [tool]'], ['scoring of blue sectors [process]'], ['in vitro estrogen response element reporter assays [treatment]'], ['reporter enzyme assays [treatment]'], ['Promoter analyses [treatment]'], ['reporter expression studies [treatment]'], ['enhancer trap reporter lines [treatment]'], ['dual reporter strategy [treatment]'], ['WUS-reporter expression [process]'], ['reporter of expression [process]'], ['tracking agents [treatment]'], ['mRNA reporter for expression pattern analysis [process]'], ['resolution provided by current reporter systems [method]'], ['WT reporter constructs [gene]'], ['transcription monitoring reporter [process]'], ['several different reporter systems [treatment]'], ['LexA operator-dependent reporter system [treatment]'], ['Reporter assays [technique]'], ['reporter-based studies [treatment]']]}\n",
      "-------------------\n",
      "{'91325': [['dimethylation of histone 3 lysine 9 (H3K9) [treatment]', '**H3K9m2 hypermethylation [treatment]**', 'dimethylated histone tails at lysine 9 (H3K9m2) [treatment]', 'acquiring H3K9m2 methylation [treatment]'], ['H3K9m2 demethylase activity [treatment]'], ['methylated Lys9 of histone H3 [treatment]'], ['H3K4me2 [treatment]'], ['H3K4me3 demethylation [treatment]'], ['**unmodified H3K4 [treatment]**', 'unmethylated H3K4 [treatment]'], ['trimethylating lysine-3 on histone 4 (H3K4me3) [treatment]'], ['trimethylation of lysine 36 of histone H3 (H3K36me3) [treatment]'], ['trimethylated H3K36 and H3K4 [treatment]'], ['methylated Lys27 of histone H3 [treatment]', '**methylation of H3K27 [treatment]**'], ['histone demethylation [treatment]'], ['diurnal histone demethylation [treatment]'], ['methylated-Lys at histone tails [treatment]'], ['Reprogramming of H3 Lysine Methylation [treatment]']]}\n",
      "-------------------\n",
      "{'156140': [['**Swi3p [gene]**', 'yeast SWI3 subunit [gene]', 'yeast SWI/SNF complex subunit SWI3 [gene]']]}\n",
      "-------------------\n",
      "{'110718': [['tRNA-leaders [gene]'], ['r-protein regulon [gene]'], ['r-protein coding sequences [gene]'], ['**gene(s) coding for translational machinery [gene]**', 'gene(s) involved in protein(s) translation mechanisms [gene]', 'gene(s) encoding subunits of the translational machinery [gene]'], ['18S ribosomal DNA gene(s) [gene]'], ['distinctions in r-protein coding sequences [gene]'], ['tRNA-coding regions [gene]'], ['pre-rRNA-coding sequences [gene]'], ['fraction of tRNA gene(s) [gene]'], ['one cluster of tRNA-Ser and tRNA-Tyr [gene]'], ['overlapping fragments coding for human seryl-tRNA synthetase [gene]']]}\n",
      "-------------------\n",
      "{'6443': [['unchanged temperature profile [phenotype]'], ['different temperature stability [phenotype]'], ['alterations in thermal dependence [phenotype]', '**relatively temperature-sensitive [phenotype]**', 'temperature-dependent activities [phenotype]', 'different temperature sensitivities [phenotype]']]}\n",
      "-------------------\n",
      "{'60118': [['1-3-bp deletions [phenotype]'], ['Downstream response to BQ exposures [phenotype]'], ['ideal BS [phenotype]'], ['Excessive B accumulation [phenotype]'], ['M:BS ratio [phenotype]'], ['PIB:v o ratio [phenotype]'], ['beta-mixture model [phenotype]'], ['CO breakpoint predictions [phenotype]'], ['limitations in BR levels [phenotype]'], ['single base substitutions (SBSs) [phenotype]'], ['beam-film (BF) model [phenotype]'], ['BF sprinkling (BFS) model [phenotype]'], ['SPAD value in BOP period [phenotype]'], ['SPAD value in FBP period [phenotype]'], ['BOP_blup and D_SPAD_blup values of Hap1 [phenotype]'], ['gap filling during SP-BER [phenotype]'], ['~22% of BP term annotations [phenotype]'], ['no previous BP GO term annotation [phenotype]'], ['positional preference of BSs [phenotype]'], ['prediction of BSs [phenotype]'], ['rate of DMBQ cycling [phenotype]'], ['sufficient BQ exposure [phenotype]'], ['0.827 for population A [phenotype]'], ['0.864 for population A [phenotype]'], ['0.887 for population B [phenotype]'], ['0.911 for population B [phenotype]'], ['0.952 for population B [phenotype]'], ['0.955 for population B [phenotype]'], ['0.109 for population A [phenotype]'], ['0.092 for population B [phenotype]'], ['0.083 for population B [phenotype]'], ['0.058 for population B [phenotype]'], ['0.055 for population B [phenotype]'], ['0.385 for population B [phenotype]'], ['0.212 for population B [phenotype]'], ['0.170 for population B [phenotype]'], ['0.101 for population B [phenotype]'], ['0.077 for population B [phenotype]'], ['0.054 for population B [phenotype]'], ['0.045 for population B [phenotype]'], ['BIN2 activity [phenotype]'], ['0.1-0.2 SFB [phenotype]'], ['limiting boron conditions [phenotype]'], ['**excess B tolerance [phenotype]**', 'tolerance of high B [phenotype]'], ['banana CI [phenotype]'], ['bud transformation [phenotype]'], ['similar fractions of peaks with BaMM BSs [phenotype]'], ['bootstrap values of 90% [phenotype]'], ['bud break capacity [phenotype]'], ['until ~6 dpb [phenotype]'], ['D35S::BnLATE results [phenotype]'], ['AtBIN2 activity [phenotype]'], ['higher average BPI [phenotype]'], ['berry-specific expression [phenotype]'], ['Hull colour b* value (HCb*) [phenotype]'], ['one B-BOX domain [phenotype]'], ['smallest bZIP [phenotype]'], ['BAC library construction [phenotype]'], ['larger BP-AG distance [phenotype]'], ['A1 and A3 with bootstrap value >=80% [phenotype]'], ['measurements on bivalents [phenotype]'], ['higher yellow (b*) [phenotype]'], ['increased B sensitivity [phenotype]'], ['high levels of B [phenotype]'], ['preferential distribution of B [phenotype]'], ['B sensitivity [phenotype]'], ['beta-folds [phenotype]'], ['Brevipalpus vector performance [phenotype]']]}\n",
      "-------------------\n",
      "{'77335': [['Domain II [protein domain]'], ['VWA domains [protein domain]'], ['presence of one or more PPI domains [protein domain]'], ['IDS [protein domain]'], ['connector domain [protein domain]'], ['oligomerization domains [protein]'], ['Armadillo domains [protein domain]'], ['additional protein(s) domain [protein]'], ['heterologous homo- and heterodimerization protein(s) domains [protein]'], ['**interaction domains [protein domain]**', 'interacting domains [protein domain]'], ['**protein-protein interaction domains [protein domain]**', 'protein-protein-interaction domain [protein domain]', 'domains specialized in protein-protein interactions [protein domain]'], ['Integrated domains [protein complex]'], ['promiscuous domains [protein domain]'], [\"g e' pairs [protein domain]\"], [\"a a' pairs [protein domain]\"], ['inserted domain [protein domain]'], ['**protein-protein interaction domains [protein]**', 'protein(s) interaction domains [protein]'], ['single MoRF region [protein domain]'], ['intracellular loops [protein domain]'], ['protein-protein interaction module [function]'], ['protein-protein interaction domains [feature]'], ['two PPLPP domains [protein domain]'], ['PPLPP domains [protein domain]'], ['Salt bridges [protein domain]'], ['large surface-to-volume ratio protein(s) domain [protein domain]'], ['continuous surface patches [protein domain]'], ['multiple interfaces [protein domain]'], ['specialized amino acid modules [protein domain]'], ['protein(s) containing coil-coil domain [protein]'], ['putative protein/protein interaction domain [protein domain]'], ['protein(s) containing two protein-protein interactive domains [protein]'], ['large interaction interfaces [protein domain]'], ['two intermolecular disulfide bonds [protein domain]'], ['an extended interaction surface [protein domain]'], ['seven WD domains [protein domain]'], ['hinge domains [protein domain]'], ['protein-protein interface [protein domain]'], ['oligomerization domains (ODs, HR-A/B regions) [phenotype]'], ['peculiarities of their oligomerization domains [protein domain]'], ['multiple protein-protein interaction domains [protein]'], ['mutually independent VirE2 domains [protein]'], ['a protein(s) with a MATH domain and a coiled-coil domain [protein]'], ['protein-protein interaction domains [protein feature]'], ['common docking domains (CD-domains) [gene]'], ['protein(s) domain specific binding [process]'], ['protein(s) interaction domains [protein domain]'], ['single or multiple PPI domains [protein]'], ['dimerization region [protein domain]'], ['Heterodimers [protein domain]'], ['two predicted protein-protein interaction domains [protein domain]'], ['protein-protein interaction domains [protein complex]'], ['molecular recognition features (MoRFs) [protein domain]'], ['one ZZ domain [protein region]'], ['protein(s) interaction domains [process]'], ['protein-protein interaction domains [function]'], ['internal repeat structure associated with protein-protein interactions [protein domain]'], ['protein(s) interaction modules [protein domain]'], ['potential protein-protein interaction domain [protein domain]'], ['protein(s) domain junctions [protein]'], ['domains involved in protein(s) degradation [protein domain]']]}\n",
      "-------------------\n",
      "{'172480': [['sugar-specific pathways [process]'], ['newly synthesized sugars [metabolite]'], ['sugar metabolism-related metabolic pathways [metabolic pathway]'], ['sugar and ethylene metabolisms [metabolite]'], ['sugar biosynthetic processes [metabolite]'], ['central sugar metabolism [process]'], ['sugar [process]'], ['fructose metabolism [metabolic pathway]'], ['selective sugar pathways [phenotype]'], ['pathways that affect sugar metabolism [metabolite]'], ['sugar metabolism in root(s) [metabolic process]'], ['sugar metabolism through cyFBP [metabolite]'], ['sugar phosphorylation [process]'], ['sugar metabolic processes [process]'], ['key node of sugar metabolism [process]'], ['plant(s) sugar metabolism [process]'], ['sugar interconversion pathway [process]'], ['synthesis and degradation of sugars [metabolite]'], ['induction of sugar metabolism [metabolite]'], ['sugar metabolic pathways [process]'], ['carbohydrates for male gametophyte development [process]'], ['sugar skeleton rearrangement [process]'], ['similar glycometabolic activities [process]'], ['metabolic incorporation of modified sugars [treatment]'], ['lipid into sugars [process]']]}\n",
      "-------------------\n",
      "{'11989': [['irregular or cubic shape [phenotype]'], ['irregular in shape and intermediate in size [phenotype]'], ['**atypical starch granule morphology [phenotype]**', 'deformed and curved starch granules [phenotype]', 'irregular shape starch granules [phenotype]']]}\n",
      "-------------------\n",
      "{'160793': [['malonate [compound]'], ['**l-malate [metabolite]**', 'carboxylic acids malate [metabolite]', 'four-carbon compound malate [metabolite]'], ['malate decarboxylation [metabolite]'], ['C2 units [metabolite]'], ['(2S)-ethylmalonyl-CoA [metabolite]'], ['C3-C5 [metabolite]'], [\"3-(2'-methylthio)-ethylmalate [metabolite]\"], ['subunit beta of methylcrotonyl-CoA carboxylase [protein]'], ['Mal2- for import to mitochondria [metabolite]'], ['substrates 1-O-sinapoyl-beta-glucose and L-malate [metabolite]'], ['hyper-accumulation of 3-methylcrotonyl CoA [metabolite]'], ['S-3-hydroxy-3-methylglutaryl-CoA (HMG-CoA) [metabolite]']]}\n",
      "-------------------\n",
      "{'138085': [['content of endogenous NF-YC2 [gene]'], ['expression of any one NF-YC [gene]'], ['expression of NFYA1 [gene]']]}\n",
      "-------------------\n",
      "{'153343': [['HHP family [gene]'], ['LHC gene(s) family members [gene]'], ['lhca gene(s) [gene family]'], ['lhcb4-6 gene(s) [gene family]'], ['OHP subfamily [protein subfamily]'], ['Lhca1-4 [gene family]'], ['CCCH gene(s) family in PlnTFDB [database]'], ['CCCH gene(s) family in DPTF [database]']]}\n",
      "-------------------\n",
      "{'31740': [['Signaling mechanisms [phenotype]', '**Signaling mechanism [phenotype]**'], ['cell-signalling information [phenotype]'], ['**Cell signaling [phenotype]**', 'signalling functions [phenotype]', 'cell signalling [phenotype]', 'cellular signaling events [phenotype]', 'cell signaling functions [phenotype]', 'biological signalling [phenotype]'], ['Signaling from wound [phenotype]'], ['**Positional signaling mechanism [phenotype]**', 'positional signaling mechanisms [phenotype]'], ['global positional signaling mechanism [phenotype]'], ['**rapid signal transduction [phenotype]**', 'rapid and dynamic signaling [phenotype]'], ['Extracellular signaling systems [phenotype]'], ['Guard cell signal transduction [phenotype]'], ['**Cell-cell signaling [phenotype]**', 'signal shift ways by cell-to-cell traveling [phenotype]', 'cell-cell signal transduction [phenotype]'], ['Signaling through pathway [phenotype]'], ['Rapid communication [phenotype]'], ['Intra-cellular signaling processes [phenotype]'], ['cell signaling and growth [phenotype]'], ['signaling and cell division processes [phenotype]'], ['**adhesion-dependent signaling [phenotype]**', 'adhesion and signaling [phenotype]'], ['surface molecular-mediated signalling transduction [phenotype]'], ['maturation of signaling components [phenotype]'], ['signaling during biological processes [phenotype]'], ['cytoplasmic signaling [phenotype]'], ['cell signaling and differentiation [phenotype]'], ['eukaryotic cellular signaling [phenotype]'], ['**cellular communication/signal transduction [phenotype]**', 'cellular communication and signal transduction mechanism [phenotype]', 'cell communication/transduction [phenotype]'], ['signalling and regulatory responses [phenotype]'], ['flagellin transduction [phenotype]'], ['ECM signaling in animals [phenotype]'], ['molecular understanding of signaling processes [phenotype]'], ['**TCS signaling [phenotype]**', 'TCS information flow [phenotype]'], ['**signaling in animals [phenotype]**', 'signaling in animal systems [phenotype]'], ['elaborate signalling mechanisms [phenotype]'], ['communications between cell surface and extracellular environment [phenotype]'], ['important signal transduction mechanisms [phenotype]'], ['physical and signaling continuum [phenotype]']]}\n",
      "-------------------\n",
      "{'122506': [['similar biological function in embryogenesis [phenotype]'], ['**common functions [phenotype]**', 'shared function [phenotype]', 'shared functions [phenotype]'], ['common gene(s) activity [phenotype]'], ['overlapping functions in development [phenotype]'], ['common biochemical function [phenotype]'], ['partly similar function [phenotype]']]}\n",
      "-------------------\n",
      "{'31361': [['metacaspase activation [phenotype]'], ['regulation of immunoglobulin production [process]'], ['interleukin production [process]'], ['cytokine activation [process]'], ['increasing cAMP [process]'], ['Activation of toxins [gene]'], ['Induction of autophagy [pathway]'], ['release of IL-2 [gene]'], ['activation of lymphocytes [cell type]'], ['anti-retroviral response [process]'], ['leukocyte activation [cell type]'], ['platelet activation [cell type]'], ['surveillance cell release of interleukin-1beta [process]'], ['modulation of the immune response [process]'], ['massive activation of cell expansion pathway [cell process]'], ['immune function [cellular process]'], ['constitutive defense activation [process]'], ['developmentally regulated defense responses [process]'], ['an intracellular inducible enzyme [enzyme]'], ['immune cell activation [process]'], ['unwanted activation of cellular processes [phenomenon]'], ['T-cell priming [cell type]'], ['conditional activation [process]']]}\n",
      "-------------------\n",
      "{'20951': [['Growth of WT-Ntc [cell type]'], ['Growth of WT-Syc [cell type]'], ['growth of liquid culture-grown mutants [phenotype]'], ['rosette size in control and 300mM NaCl [phenotype]'], ['increased resistance to Pto DC3000 at 17 C [phenotype]'], ['DC3000 (avrRpm1) growth [phenotype]'], ['moderate growth rate [phenotype]']]}\n",
      "-------------------\n",
      "{'159175': [['VERNALIZATION INSENSITIVE 3 (VIN3) family gene(s) [gene]'], ['MeV [gene]'], ['three Variant In Methylation gene(s) (VIM1-3) [gene]'], ['five VIM gene(s) [gene]']]}\n",
      "-------------------\n",
      "{'59286': [['T:Zmetr2b line L9 root(s) [organ]'], ['T:Zmers1b line L11 root(s) growth [organ]'], ['relationship between internode length [organ]'], ['mean internode length [organ]'], ['reduced stem(s) length [organ]'], ['Glenlea root(s) samples [organ]'], ['radial pattern of chicory root(s) [organ]'], ['soluble fractions from root(s) [organ]'], ['DR5::GUS activity in root(s) tips [organ]'], ['P. indica-exposed ein3 root(s) [organ]'], ['stem(s) height [organ]'], ['reduced number of internodes [organ]'], ['alternative splicing responses in root(s) [organ]'], ['lower parts of DR5::GFP root(s) tips [organ]'], ['upper parts of DR5::GFP root(s) tips [organ]'], ['lower parts of auxin-treated DR5::GFP root(s) tips [organ]'], ['upper parts of auxin-treated DR5::GFP root(s) tips [organ]'], ['DR5::GFP root(s) tips [organ]'], ['lower parts of 2-NOA-treated DR5::GFP root(s) tips [organ]'], ['upper parts of 2-NOA-treated DR5::GFP root(s) tips [organ]'], ['lower parts of 2-NOA-treated DR5::GFP/35S::AtAAF root(s) tips [organ]'], ['transcript abundance in root(s) [organ]'], ['root-stem comparison [organ]'], ['root-leaf comparison [organ]'], ['root-stem and stem-leaf comparisons [organ]'], ['root-stem and root-leaf comparisons [organ]'], ['shoot apices and root(s) tips [organ]'], ['expression pattern in root, stem(s) and leaf tissues [organ]'], ['donor rootstocks [organ]'], ['rootstock samples [organ]'], ['xylem loading in root(s) [organ]'], ['LP in white lupin root(s) [organ]'], ['accumulation of DR5:GFP signal in root(s) tips [organ]'], ['GSH in root(s) [organ]']]}\n",
      "-------------------\n",
      "{'84317': [['flg22-induced proteome remodeling [phenotype]'], ['NatA-mediated proteome imprinting [phenotype]'], ['NatB-mediated imprinting of the proteome [phenotype]'], ['**direct evidence of myristoylation [phenotype]**', 'myristoylated in vitro [phenotype]'], ['modification of several candidates upon immune elicitation [phenotype]']]}\n",
      "-------------------\n",
      "{'11177': [['near-constitutively narrow PD aperture [phenotype]'], ['(TVW/DMAX)2 [phenotype]'], ['hypotonic environment [phenotype]'], ['transient depolarization [phenotype]'], ['action potential [phenotype]'], ['Unknown factor [phenotype]'], ['free‐running voltage [phenotype]'], ['membrane potential of -240 mV [phenotype]'], ['least negative E m values [phenotype]'], ['depolarization difference [phenotype]'], ['immediate electrical depolarization [phenotype]'], ['depolarization response [phenotype]'], ['**hyperpolarized [phenotype]**', 'hyperpolarization state [phenotype]', 'under hyperpolarization [phenotype]'], ['-337 mV [phenotype]'], ['degree of depolarization [phenotype]'], ['more than -70 mV [phenotype]'], ['**basal Ca2+ concentrations [phenotype]**', 'resting Ca2+ concentrations [phenotype]', 'free Ca2+ levels [phenotype]'], ['early depolarization [phenotype]'], ['depolarizing voltage passed K+ equilibrium potential [phenotype]'], ['positive voltage [phenotype]'], ['lower depolarized membrane potential [phenotype]'], ['maximum depolarization [phenotype]'], ['minimum depolarization [phenotype]'], ['smaller initial depolarization [phenotype]'], ['initial depolarization [phenotype]'], ['initial osmotic volume [phenotype]'], ['final osmotic volume [phenotype]'], ['fractional nonosmotic volume [phenotype]'], ['relative nonosmotic volume beta [phenotype]'], ['gating charge undiminished when P0 near 10-4 [phenotype]'], ['dilated state [phenotype]'], ['electrical depolarization [phenotype]'], ['contractile tone [phenotype]'], ['threshold opening pressure of around -15 mm Hg [phenotype]'], ['mild depolarization [phenotype]'], ['more negative TMP [phenotype]'], ['spike-like depolarization [phenotype]'], ['wild-type DeltaVm [phenotype]'], ['midpoint potential [phenotype]'], ['rheobase of 2 V [phenotype]'], ['presence or absence of a membrane potential [phenotype]'], ['voltage more negative than -100 mV [phenotype]'], ['15 A shorter membrane arm [phenotype]'], ['potentials negative to the peak [phenotype]']]}\n",
      "-------------------\n",
      "{'59093': [['**E. coli O157:H7 [organism]**', 'O157:H7 [organism]'], ['O157:H7 population [organism]', '**population of Escherichia coli O157:H7 [organism]**', 'population of E. coli O157:H7 [organism]'], ['O157:H7 strains [organism]', '**O157:H7 strain [organism]**'], ['O157:H7 and STm 14028s [organism]'], ['Larger population of O157:H7 and STm 14028s [organism]'], ['E. coli O157:H7 strain 7386 [organism]'], ['wild-type strain of Escherichia coli O157:H7 [organism]'], ['population of E. coli O157:H7 ATCC 43895 wild-type strain [organism]'], ['E. coli O157:H7 43895 wild-type strain [organism]'], ['decreased numbers of E. coli O157:H7 ATCC 43895 wild-type strain [organism]'], ['E. coli O157:H7 genome [organism]'], ['E. coli O157:H7::MRE103 [organism]'], ['attachment of O157:H7 [organism]']]}\n",
      "-------------------\n",
      "{'31879': [['Vacuolar metal stores [biochemical process]'], ['an important organelle for the regulation of metal homeostasis [process]']]}\n",
      "-------------------\n",
      "{'152530': [['lrrpk mRNA [gene]', '**light-repressible receptor protein(s) kinase (lrrpk) [gene]**']]}\n",
      "-------------------\n",
      "{'116062': [['10,559 unigenes [gene]'], ['8,016 unigenes [gene]'], ['18,854 unigenes [gene]'], ['1,167 unigenes [datum]'], ['68,906 unigenes [gene]'], ['21,213 unigenes [gene]'], ['18,172 unigenes [gene]'], ['22,167 unigenes [gene]'], ['52,270 unigenes [gene]'], ['20,803 unigenes [gene]'], ['17,554 unigenes [gene]'], ['62,912 unigenes [gene]'], ['73,807 unigenes [gene]'], ['19,707 unigenes [gene]'], ['17,171 unigenes [gene]'], ['13,000 unigenes [gene]'], ['148,046 unigenes [gene]'], ['66,798 unigenes [gene]'], ['72,914 unigenes [gene]'], ['108,926 unigenes [gene]'], ['70,682 unigenes [gene]'], ['100,882 unigenes [gene]'], ['134,900 unigenes [gene]']]}\n",
      "-------------------\n",
      "{'54130': [['PLD-PA signaling complexes [gene]'], ['overexpression of RRL [gene]'], ['pyramiding of AGLU1 and RCH10 [gene]'], ['increased levels of GLX1 and GLX2 [gene]'], ['more than half of the LTPs [gene]'], ['Overexpression of LSC [gene]'], ['**overexpression of LEC2 [gene]**', 'expression of castor bean LEC2 [gene]', 'over-expression of castor LEC2 [gene]', 'overexpression of LEC2 from castor bean [gene]', 'LEC2 transcripts [gene]', 'abundances of transcripts that encode LEC2 from castor bean [gene]'], ['ELF3-ELF4-LUX [gene]'], ['Lox transcript levels [gene]'], ['Loss of LEA4 protein(s) [gene]'], ['abundance of LEA protein(s) [gene]'], ['overexpression of wheat LEA3 gene(s) (WZY3-1) [gene]'], ['overexpression of Lily (Lilium longiflorum) LiHsfA1 gene(s) [gene]'], ['leucine rich repeat (LRR) domain [gene]'], ['light harvesting complex 1 (LHCA1) [gene]'], ['overexpression of LBD37/LBD38/LBD39 gene(s) [gene]'], ['overexpression of CsLBD39 gene(s) [gene]'], ['overexpression of HhbHLH2 [gene]'], ['transcripts of LHCB1.2 marker [gene]'], ['Activation of the LD pathway [gene]'], ['low 18:1-mediated induction of R gene(s) expression [gene expression]'], ['expression of LACS2 [gene]'], ['LP-induced expression of Chalcone Synthase (CHS) [gene]'], ['disruption of mouse Lsh gene(s) [gene]'], ['mRNA levels of lycopene beta- and epsilon-cyclases [gene]'], ['Overexpression of a homolog of LEC1 [gene]'], ['full-length LjPLP-IV transcription [gene]'], ['induced expression of LEAs: RD29B, RAB18 and KIN2 [gene]'], ['expression of NUCLEOTIDE BINDING SITE-LEUCINE RICH REPEAT receptors [gene]'], ['decreased expression of LEB [gene]'], ['decreased expression of LUT1 [gene]'], ['8th intron of ICME-LIKE2 [gene]'], ['10th intron of ICME-LIKE2 [gene]'], ['expression of L1L [gene]'], ['lowered basal level of BHLH105/ILR3 expression [gene]'], ['14 fold induction of RAB18 [gene]'], ['expression of LlC3H18 [gene]'], ['dual localization of bHLH039 [gene]'], ['full-scale nuclear bHLH039 accumulation [gene]'], ['stable accumulation of prolamins [gene]'], ['delayed transcriptional activation of RAP2.6L [gene]'], ['**increased number of LRR-XII [gene]**', 'expansion of LRR-XII [gene]'], ['exon-intron structure dissimilar to other EXLB gene(s) [gene structure]'], ['reduced degree of oscillation of LHY [gene]'], ['ubiquitination of LHY [gene]'], ['ubiquitinylation of RLKs [gene]'], ['down-accumulation of major latex protein(s) (MLP) [gene]'], ['higher rate of LSGs [gene]'], ['lateral root(s) negative regulator GaCASP27 [gene]'], ['accumulation of PLETHORA1 (PLT1) [gene]'], ['accumulation of PLETHORA2 (PLT2) [gene]'], ['LOX2 transcript [gene]'], ['activities of BELL [gene]'], ['ability of ATXR2 to promote expression of LBD gene(s) [gene]'], ['accumulation of RBOHD [gene]'], ['altered levels of ELONGATED HYPOCOTYL5 (HY5) [gene]'], ['increase in LFY expression [gene]']]}\n",
      "-------------------\n",
      "{'169792': [['suppression of stress hormones [metabolite]', '**lower content of stress-related hormones [metabolite]**'], ['Reduced PRO, SS, and SP content [metabolite]'], ['decrease in proline content [metabolite]', '**reduced proline content [metabolite]**', 'low proline content [metabolite]', 'decreased proline content [metabolite]'], ['reduced level of accumulated free proline [metabolite]', '**lower free proline [metabolite]**', 'lower free proline content [metabolite]'], ['**lower proline [metabolite]**', 'lower levels of proline [metabolite]', 'low contents of proline (Pro) [metabolite]', 'lower level of proline [metabolite]'], ['**decreased proline accumulation [metabolite]**', 'lower levels of proline accumulation [metabolite]'], ['lower proline concentration levels [metabolite]', '**lower proline concentration [metabolite]**'], ['down-regulation of proline biosynthesis [metabolite]'], ['lower levels of 3-Cyanoalanine [metabolite]']]}\n",
      "-------------------\n",
      "{'44413': [['fzl-2 [mutant]'], ['fzl-4 [mutant]'], ['**FZL [mutant]**', 'Lack of FZL [mutant]'], ['depletion of CURT [mutant]', '**CurT deficiency [mutant]**'], ['CURVATURE THYLAKOID 1A (CURT1A) [mutant]', '**curt1 mutants [mutant]**'], ['loss of LTO1 [mutant]'], ['Trx z [mutant]'], ['loss of WHIRLY1 [mutant]'], ['tcr mutant [mutant]', '**tcr [mutant]**'], ['TROL knockout mutant [mutant]'], ['tobacco MGD1 mutant [mutant]'], ['**LTD null mutant [mutant]**', 'null LTD mutants [mutant]'], ['Lack of Trx f [mutant]'], ['combined deficiency of Trx f and PGR5 [mutant]'], ['trxm1m2 and ntrc mutants [mutant]'], ['kowhy1 mutant [mutant]'], ['ko1/3 line [mutant]'], ['pgr5tcr mutant [mutant]']]}\n",
      "-------------------\n",
      "{'73419': [['**PhI network [database]**', 'PlaPID [database]'], ['Root-cap-specific PPI networks [protein]'], ['PPI information of plant(s) [process]'], ['PPI networks in plant(s) [process]'], ['first plant(s) PPI network [process]'], ['**PPI prediction in plant(s) genomes [process]**', 'big-PI plant(s) Predictor [process]'], ['screening of PPIs among Golgi-localizing protein(s) in planta [process]'], ['plant(s) PPI prediction [task]'], ['PPI context [attribute]'], ['protein-protein interaction (PPI) and co-expression networks in J. curcas [phenotype]'], ['PPIs in planta [phenotype]'], ['store and analyze plant(s) PPI networks [interaction]']]}\n",
      "-------------------\n",
      "{'49793': [['**pas2-1 mRNA variants [metabolite]**', 'production of pas2-1 mRNA variants [metabolite]'], ['spliced variants of betaCA2 [phenotype]'], ['mutant splice variant [phenotype]']]}\n",
      "-------------------\n",
      "{'134504': [['E. coli GLP-F [gene]'], ['glxK [gene]'], ['bacterial GLP-F [gene]'], ['two gene(s) in the L. gmelinii genome [organism]']]}\n",
      "-------------------\n",
      "{'171396': [['Jar1 [enzyme]', 'JA-amino synthetase [enzyme]', 'jasmonate-amido synthetase 1 (JAR1) [enzyme]', 'jasmonate amino acid synthetase [enzyme]', 'jasmonoyl-isoleucine synthetase (JAR1) [enzyme]', 'Jasmonoyl-L-amino acid synthetase [enzyme]', 'Jasmonic acid-amido synthetase JAR1 [enzyme]', 'JASMONATE RESISTANT1 (JAR1) [enzyme]', 'jasmonic acid-amino acid synthetase [enzyme]', '**JAR1 (JASMONATE RESISTANT 1) [enzyme]**', 'JA amino acid synthetase [enzyme]', 'jasmonate amino acid synthetase 1 [enzyme]', 'JA-Ile synthase [enzyme]', 'JA-amino acid synthetase (JAR1) [enzyme]', 'activity of JA-Ile synthetase JAR1 [enzyme]', 'jasmonoyl-L-isoleucine synthetase [enzyme]', 'JA conjugate synthase (JAR1) [enzyme]', 'JA-conjugating enzyme JAR1 [enzyme]', 'the activity of JA-Ile synthetase JAR1 [enzyme]', 'JA-amido synthetase [enzyme]'], ['JA biosynthetic enzyme 12-oxophytodienoate reductase [enzyme]'], ['jasmonic acid biosynthetic enzyme [enzyme]'], ['**jasmonic acid carboxyl methyltransferase [enzyme]**', 'jasmonic acid carboxyl methyltransferase (JMT) [enzyme]', 'JA carboxyl methyltransferase [enzyme]'], ['**auxin-amido synthetases [enzyme]**', 'IAA-LEU RESISTENT-like6 (ILL6) [enzyme]', 'GH3 acyl acid amido synthetases [enzyme]', 'IAA-amino acid conjugate synthase [enzyme]', 'auxin conjugate synthetases [enzyme]'], ['**allene oxide synthases [enzyme]**', 'allene-oxide synthase [enzyme]', 'allene-oxide synthase (AOS) [enzyme]'], ['JAR1/FIN219 [enzyme]'], ['phytohormone-amino acid synthetase [enzyme]'], ['ornithine N-delta-acetyltransferase [enzyme]'], ['**JAR1 enzyme [protein]**', 'jasmonate-amido synthetase [protein]', 'Jasmonoyl-l-amino acid synthetase JAR1 [protein]', 'JA-amino-synthetase [protein]', 'JA-amino synthetase [protein]', 'JA-amidosynthetase (JAR1) [protein]', 'enzyme involved in JA-Ile biosynthesis [protein]', 'JA-conjugating enzyme [protein]', 'jasmonate-conjugating enzyme [protein]', 'jasmonate-amino synthetase [protein]'], ['**IAA amino acid synthase [protein]**', 'IAA-amido synthetases [protein]', 'IAA-amide synthetase [protein]', 'enzyme responsible for auxin conjugation [protein]', 'IAA-amide synthetases [protein]', 'participation of IAA-amido synthetases [protein]'], ['LOX [protein]'], ['Allene Oxide Synthase (AOS) [protein]'], ['**JA carboxyl methyl transferase [protein]**', 'jasmonate O-methyltransferase [protein]', 'orthologous JA carboxyl methyltransferase [protein]'], ['JA-Ile catabolic pathway [phenotype]'], ['jasmonic acids-amido synthetases [phenotype]'], ['**JA-isoleucine biosynthesis enzyme [phenotype]**', 'JA-isoleucine biosynthesis enzymes [phenotype]'], ['IAA-amido synthetase activity [activity]'], ['13-AOS activity [enzyme activity]'], ['C12-hydroxylation of JA-Ile [reaction]'], ['cleavage of JA-Ile [reaction]']]}\n",
      "-------------------\n",
      "{'36974': [['direction of gradient [phenotype]'], ['rapid and transient cytoplasmic Ca²⁺ elevation in root(s) [phenotype]'], ['Asymmetric calcium spikes [metabolite]'], ['asymmetry of Ca2+i [metabolite]']]}\n",
      "-------------------\n",
      "{'130268': [['higher dS of A. thaliana IL gene(s) [gene]'], ['Transcripts of A. thaliana and loblolly pine (Pinus taeda) CslA gene(s) [gene]']]}\n",
      "-------------------\n",
      "{'154497': [['37 F-box gene(s) on chromosome 5 [gene]'], ['36 F-box gene(s) on chromosome 11 [gene]'], ['34 F-box gene(s) on chromosome 18 [gene]'], ['34 F-box gene(s) on chromosome 21 [gene]'], ['6 F-box gene(s) on chromosome 4 [gene]'], ['all 11 CBF gene(s) [gene]']]}\n",
      "-------------------\n",
      "{'103242': [['DSEs [organism]'], ['10 rhizobacteria [organism]'], ['root(s) endophyte Enterobacter sp. SA187 [organism]'], ['CJAN1179 [root endophyte]'], ['Direct contact of root(s) with Enterobacter sp.L7 [organism]'], ['128 root-endophytic strains [organism]'], ['PLR [organism]'], ['1.38 in Ru3.1-31 [organism]'], ['SA187-treated root(s) endosphere samples [tissue]'], ['**root(s) colonization by SA187 [organism]**', 'colonization levels by SA187 [organism]'], ['RGI-inducing Arthrobacter CL28 [organism]'], ['root(s) endophytic bacterium [organism]'], ['primary root(s) response to M. luteus LS570 [process]']]}\n",
      "-------------------\n",
      "{'115840': [['2908 DEGs [gene]'], ['6859 DEGs [gene]'], ['3479 DEGs [gene]'], ['3535 DEGs [gene]'], ['3111 DEGs [gene]'], ['3413 DEGs [gene]'], ['3418 DEGs [gene]']]}\n",
      "-------------------\n",
      "{'103935': [['A281 [organism]'], ['Agrobacterium strains [organism]'], ['K599 [organism]'], ['Agrobacterium strain [organism]'], ['Agrobacterium strain [bacterium]'], ['**Agrobacterium strain VOT [organism]**', 'strain VOT [organism]', 'VOT [organism]'], ['Agrobacterium wild-type strain C58 [organism]'], ['**Agrobacterium tumefaciens C58C1 [organism]**', 'Agrobacterium tumefaciens strain C58C1 [organism]'], ['LBA4404 strain [organism]', '**Agrobacterium LBA4404 [organism]**', 'LBA4404 [organism]', 'strain LBA4404 [organism]', 'LBA4404 Agrobacterium tumefaciens strain [organism]'], ['**Agrobacterium AGL1 [organism]**', 'Agrobacterium strain AGL1 [organism]'], ['agrobacterial strains [organism]'], ['GAANTRY ArPORT1 strain [organism]'], ['T-DNA transfer-competent Agrobacterium strain [organism]'], ['LMG201 [organism]'], ['Agrobacterium timefaciens hypervirulent strain A281 [organism]'], ['hypervirulent strain of Agrobacterium [organism]'], ['roles of Agrobacterium strain [organism]'], ['K599-HGCI [organism]'], ['Agrobacterium ABI strains [organism]'], ['Agrobacterium C58 sequence [organism]'], ['oncogenic Agrobacterium strain [organism]'], ['oncogenic strain C58 [organism]'], ['**Agrobacterium C58 strain [organism]**', 'C58 strain [organism]'], ['Agrobacterium tumefaciens C58C2 [organism]'], ['GV2260 [organism]'], ['conventional Agrobacterium strain [organism]']]}\n",
      "-------------------\n",
      "{'12873': [['alterations in stomatal size [phenotype]', 'changes to stomatal size [phenotype]', 'largest stomate [phenotype]', 'increasing width/length of stoma [phenotype]', 'stomata to enlarge [phenotype]', 'plant(s) with increased stomatal size [phenotype]', '**increase in stomatal size [phenotype]**', 'more swollen stomata [phenotype]', 'significant change in stomatal length [phenotype]', 'longer stomata length [phenotype]', 'wider stomata width [phenotype]', 'swell up [phenotype]', 'enlarged stomata [phenotype]'], ['opening degree of stomatal pores [phenotype]', 'aperture change [phenotype]', 'Larger stomatal openings of the double mutant [phenotype]', 'maximum size of stomatal pores [phenotype]', '**increases in stomatal pore aperture [phenotype]**', 'larger openings [phenotype]', 'larger stomatal pores [phenotype]', 'pore width changes [phenotype]'], ['**Increase in guard cell size [phenotype]**', 'guard-cell swelling [phenotype]', 'swelling of the guard cells [phenotype]'], ['elongated guard cell conformation [phenotype]'], ['**stomatal complex extension [phenotype]**', 'increase of stomatal complex length during opening [phenotype]'], ['radial stiffening of guard cells is not required for stomatal opening [phenotype]'], ['enlargement of pavement cells [phenotype]'], ['larger SS, SP, and SW [phenotype]'], ['extreme swelling of protoplasts [phenotype]'], ['increase in small unfused vacuoles [phenotype]'], ['opening of the adjacent stoma [phenotype]'], ['extra adjacent stomata [phenotype]'], ['better quality images [phenotype]']]}\n",
      "-------------------\n",
      "{'75202': [['pathogenesis-related protein(s) 4 [protein]'], ['PATHOGENESIS-RELATED protein(s) [protein]'], ['**PATHOGENESIS-RELATED protein(s) 1 [protein]**', 'PATHOGENESIS-RELATED1 [protein]', 'Pathogenesis related protein(s) 1 [protein]'], ['pathogenesis-related protein(s) 1 C [protein]'], ['pathogenesis-related protein(s) 1-like protein(s) [protein]'], ['putative germin-like protein(s) [protein]'], ['pathogenesis-related protein(s) families [protein family]'], ['pathogenesis-related thaumatin protein(s) [protein]'], ['pathogenesis-related protein(s) (At2g14610) [protein]']]}\n",
      "-------------------\n",
      "{'45677': [['Mis-expression of FLOWERING LOCUS T [gene]'], ['bHLH60 overexpression [mutant]'], ['overexpression of SAPK10 [mutant]'], ['Overexpression of OsmiR393 [mutant]'], ['ectopic overexpression of MAF5 [gene]'], ['ULT1 overexpression [mutant]'], ['Overexpression of ABI5 [mutant]'], ['Altered TaGI expression [phenotype]'], ['plant(s) overexpressing MAF2 [mutant]'], ['Overexpression of MAF3 [mutant]'], ['Overexpression of MAF5 [mutant]'], ['overexpression of an aberrant AtAHL20 protein(s) [treatment]'], ['overexpression of MAF2var2 [isoform]'], ['Overexpression of CIB1 [mutant]'], ['excessive AGL24 [gene]'], ['Ectopic expression of GmLUXb [gene]'], ['ELF3-overexpressing (ELF3-OX) plant(s) [mutant]'], ['Ectopic expression of MiFT1 [gene]'], ['Ectopic expression of MiFT2 [gene]'], ['altered expression levels in top1α-7 seedlings [mutant]'], ['Ectopic expression of BOP1 and BOP2 [gene]'], ['Overexpression of AtGRP7 [mutant]'], ['Over-expression of ZTL [mutant]'], ['Over-expressing LKP2 plant(s) [mutant]'], ['florigen overexpression [mutant]'], ['overproduction of AKIN10 [gene]'], ['DELLA overexpressing plant(s) [mutant]']]}\n",
      "-------------------\n",
      "{'92052': [['Auxin-derived compounds [treatment]'], ['**Synthetic auxins [treatment]**', 'Synthetic auxin [treatment]'], ['artificial auxin [treatment]'], ['artificial auxin production [treatment]'], ['natural/synthetic auxins [treatment]'], ['**Auxin analogues treatment [treatment]**', 'auxin analogue treatment [treatment]', 'exogenous auxin analogue [treatment]', 'exogenous auxin analogues [treatment]', 'Exogenous auxin analogs [treatment]', 'application of auxin analogs [treatment]', 'exogenously applied auxin analog [treatment]'], ['Xylem-inducing effect of auxin analogs [treatment]'], ['other chemically distinct synthetic auxins [metabolite]'], ['competing auxin source [metabolite]'], ['halogenated auxins [metabolite]']]}\n",
      "-------------------\n",
      "{'74871': [['**TIP (TCV-interacting protein) [protein]**', 'TCV-INTERACTING protein(s) (TIP) [protein]'], ['Turnip crinkle virus p38 [protein]', '**TCV p38 [protein]**', 'Turnip crinkle virus capsid protein(s) [protein]', '38 kDa capsid protein(s) [protein]'], ['Virus replication protein(s) p28 [protein]'], ['P88 [protein]'], ['expression of functional P8 and P9 protein(s) of TCV [protein]']]}\n",
      "-------------------\n",
      "{'123789': [['nucleolar rRNA gene(s) [subcellular compartment]', '**NOR regions [subcellular compartment]**'], ['NORs in nucleolus [gene]']]}\n",
      "-------------------\n",
      "{'44595': [['waxy lines [mutant]', '**waxy (wx) [mutant]**'], ['three starch-deficient mutants [mutant]']]}\n",
      "-------------------\n",
      "{'132503': [['LAC10 [gene]'], ['**L-gulono-1,4-lactone oxidase [gene]**', 'GLOase [gene]', 'GUL oxidase [gene]', 'ORF encoding l-gulono-1,4-lactone oxidase [protein]', 'cDNA encoding L-gulono-1,4-lactone oxidase [gene]', 'rat L-gulono-1,4-lactone oxidase [gene]'], ['Mammalian GulLOs [gene]'], ['**TT10 [gene]**', 'TT10/LAC15 [gene]', 'laccase-like polyphenol oxidase (TT10) [gene]'], ['LACCASEs [gene]'], ['C. neoformans laccase [gene]'], ['Rice MAX1 homolog carlactone oxidase [gene]'], ['lactate oxidase [gene]'], ['Rc(l)-2-HAOX1 [gene]'], ['Rc(l)-2-HAOX2 [gene]'], ['Rc(l)-2-HAOX3 [gene]'], ['Ricinus (l)-2-HAOX [gene]'], ['3 homologs of L-ascorbate oxidase [gene]'], ['lactoylglutathione lyase glyoxalase I-like family [gene]'], ['27 laccases [gene]'], ['4 L-ascorbate oxidases [gene]'], ['lupeol C-28 oxidase [gene]'], ['**lupeol oxidase [gene]**', 'lupeol oxidase gene(s) [gene]'], ['BnLODX [gene]'], ['CotA-type laccases [gene]'], ['peroxidase 72-like [gene]'], ['larreatricin hydroxylase [gene]'], ['l-gulono-lactone oxidase [gene]'], ['PEROXIDASE 64 [gene]'], ['multicopper oxidases [gene]'], ['Laccase/LAC gene(s) [gene]'], ['LACCASE gene(s) [gene family]'], ['Glycine max L-gulonolactone oxidase-4 [gene]'], ['functional L-gulono-1,4-lactone oxidase [gene]'], ['laccase/diphenol oxidase [gene]'], ['laccase (VviLAC17, VIT_13s0019g02160) [gene]'], ['Aerococcus viridians LOX [gene]'], ['four l-2-hydroxy acid oxidases [gene]'], ['streptophyte lHAOX [gene]'], ['D-arabinono-1.4-lactone oxidase gene(s) [gene]'], ['L-gulono-g-lactone oxidase (GLO) gene(s) [gene]'], ['gulono-1,4-lactone dehydrogenase/oxidase gene(s) [gene]'], ['laccases (LAC) [gene]'], ['laccase encoding transcript [gene]'], ['CELLOX (cellodextrin oxidase) [gene]']]}\n",
      "-------------------\n",
      "{'60775': [['32% to 23% [phenotype]'], ['23% to 11% [phenotype]'], ['32% to 19% [phenotype]'], ['23% to 6% [phenotype]'], ['0% to 12.79% [phenotype]'], ['12 to 20% [phenotype]'], ['13 to 33% range [phenotype]'], ['13.5% of sites [phenotype]'], ['13.4% to 23% [phenotype]'], ['12.5% rate [phenotype]'], ['19.6% and 20.6% of cases [phenotype]'], ['averaged 22.5% [phenotype]'], ['7.8 to 18.2% [phenotype]']]}\n",
      "-------------------\n",
      "{'50728': [['**adaxialized phenotype of ago1-12 [phenotype]**', 'adaxialized phenotype of the ago1-12 mutant [phenotype]'], ['curly leaf phenotype in abs5-1D [phenotype]'], ['leaf abaxialization in as2-1 [phenotype]'], ['agb1-1 leaf rounder [phenotype]'], ['agb1-2 leaf rounder [phenotype]'], ['leaf senescence more effectively than aif2-1 plant(s) [phenotype]'], ['**leaf angle of lpa1 [phenotype]**', 'maize mutant lpa1 leaf angle [phenotype]'], ['leaf-lamina phenotypes similar to as2 plant(s) [phenotype]'], ['leaf angle of mutant bhlh112 [phenotype]'], ['rounder leaf in agb1 mutants [phenotype]'], ['margin in abcb19 [mutant]'], ['adaxial/abaxial differentiation [phenotype]']]}\n",
      "-------------------\n",
      "{'32886': [['site of cell wall fusion [subcellular compartment]', 'future sites of cell plate insertion [subcellular compartment]', 'future position of the new cell wall [subcellular compartment]', '**the division plane [subcellular compartment]**'], ['site of cell plate fusion [process]']]}\n",
      "-------------------\n",
      "{'163974': [['Release of Ca2+ [metabolite]'], ['**inward flux of Ca2+ ions [metabolite]**', 'Influx of extracellular Ca(2+) [metabolite]', 'triggering Ca2+ influxes [metabolite]'], ['rapid efflux of Ca2+ [metabolite]'], ['flow of Ca2+ [metabolite]', '**Ca(2+) fluxes [metabolite]**', 'Ca2+ ion fluxes [metabolite]', 'calcium ion fluxes [metabolite]'], ['**Propagating calcium waves [metabolite]**', 'systemic propagation of Ca+2 [metabolite]']]}\n",
      "-------------------\n",
      "{'47189': [['12645 mutation [mutant]'], ['lld mutation [mutant]'], ['kan mutations [mutant]']]}\n",
      "-------------------\n",
      "{'43411': [['No ANAC060 mRNA [gene product]'], ['no detectable transcripts from jaz7-1 locus [gene]'], ['No transcript [description]'], ['no CHIL transcripts [phenotype]'], ['no WRKY43 transcript [phenotype]'], ['no GRXC9 induction [phenotype]'], ['no signal for ggpPS mRNA [phenotype]'], ['no full-length transcript of AHK5 [gene]'], ['no authentic transcripts for their respective disrupted FRK gene(s) [gene]'], ['no NRPD2 transcript amplification [phenotype]'], ['no detectable transcript levels of PRX62 and PRX69 [gene]'], ['no transcript levels [phenotype]'], ['no detectable AtDjC17 mRNA [phenotype]'], ['no signal in null lax2 mutants [mutant]'], ['no detectable A1494 mRNA [mrna]'], ['were not detectable in ABI3 knockout lines [mutant]']]}\n",
      "-------------------\n",
      "{'138584': [['Expression of LTP3 [gene]', '**LTP3 expression [gene]**'], ['LSU4 expression [gene]'], ['HLS1 gene(s) expression [gene]'], ['LTP1pro-positive nuclei [cell type]'], ['reconfiguration of LjPLP-IV gene(s) transcription [gene]'], ['Expression of LACs [gene]'], ['Transient expression of LsHPT [gene]'], ['Expression of YSL1 [gene]'], ['endogenous expression of RPN1b [gene]'], ['**expression of hrpL [gene]**', 'hrpL expression [gene expression]'], ['transcription of LchMATE2 [gene]'], ['lipase expression [gene]']]}\n",
      "-------------------\n",
      "{'10196': [['D. dadantii-induced necrosis [phenotype]'], ['cell death induced by nematode feeding [phenotype]', '**nematode-induced cell death [phenotype]**'], ['cell death induced by pathogen presence [phenotype]', '**pathogen-mediated cell death [phenotype]**', 'infection-induced cell death [phenotype]', 'pathogen-associated cell death [phenotype]', 'cell death due to pathogen infection [phenotype]'], ['egg-induced cell death [phenotype]'], ['mycotoxin-induced cell death [phenotype]', 'mycotoxin stress-induced cell death [phenotype]', '**mycotoxin-induced programmed cell death [phenotype]**'], ['Pseudomonas syringae-induced cell death [phenotype]'], ['spreading accelerated P. syringae pv. tomato DC3000 disease-associated cell death [phenotype]'], ['avirulent pathogenic Pseudomonas-induced cell death [phenotype]'], ['Pto-induced necrosis [phenotype]'], ['Pst-induced cell death [phenotype]'], ['sugar-induced cell death [phenotype]'], ['mildew-induced cell death [phenotype]'], ['**T3SS-dependent cell death [phenotype]**', 'type III secretion system (T3SS)-dependent cell death [phenotype]'], ['cell death of lily [phenotype]', '**lily cell death [phenotype]**'], ['B. elliptica-induced cell death [phenotype]'], ['Induction of cell death [phenotype]'], ['nonhost pathogen-induced cell death [phenotype]'], ['enhanced disease-associated cell death [phenotype]'], ['disease-associated cell death [phenotype]'], ['bacterial cell death [phenotype]'], ['toxicity-triggered cell death [phenotype]'], ['necrosis induced by oomycete pathogens [phenotype]'], ['fungal infection-induced cell death [phenotype]', '**cell death induced by fungal infection [phenotype]**'], ['further spread of the fungus by triggering plant(s) cell death [phenotype]'], ['mycotoxin-induced AL-PCD [phenotype]'], ['E. amylovora-induced cell death [phenotype]'], ['powdery mildew-induced cell death [phenotype]'], ['pathogen-induced epidermal cell death [phenotype]'], ['autoimmune response-inducing necrosis [phenotype]'], ['CMV-induced necrosis [phenotype]'], ['wound-induced cell-death [phenotype]', '**wound induced cell death [phenotype]**'], ['invading pathogens in dead cell clusters [phenotype]'], ['host CD [phenotype]'], ['Alternaria brassicicola-induced host cell death [phenotype]', '**A. brassicicola-induced cell death [phenotype]**'], ['phytotoxin-triggered programmed cell death [phenotype]'], ['cell death in response to fungal pathogen B. cinerea [phenotype]', '**B. cinerea-induced cell death [phenotype]**'], ['pathogen in death cell zone [phenotype]'], ['elicitor-triggered cell death [phenotype]', '**elicitor-induced cell death [phenotype]**'], ['pathogen-derived toxin-induced cell death [phenotype]'], ['cell death after M. oryzae inoculation [phenotype]'], ['necrosis in host A. thaliana [phenotype]'], ['more cellular death post-infection [phenotype]'], ['SAMT-induced cell death [phenotype]'], ['DON-induced cell death rate [phenotype]'], ['susceptible-induced cell death [phenotype]'], ['infected plant(s) cell death [phenotype]'], ['dramatic mildew-induced cell death [phenotype]']]}\n",
      "-------------------\n",
      "{'20497': [['CFU/plant [phenotype]'], ['3988 CFU [phenotype]'], ['2192 CFU [phenotype]'], ['1454 CFU [phenotype]'], ['2.14 Log (CFU/cm2) concentration of pathogen [phenotype]'], ['Xcc colony-forming units [phenotype]']]}\n",
      "-------------------\n",
      "{'31924': [['highly organized complex of protein(s) and pigment molecules [protein complex]'], ['CP complexes organization [phenotype]'], ['**organization of pigment-protein complexes [protein complex]**', \"CP complexes' organization [protein complex]\", 'organization of PPCs [protein complex]'], ['**thylakoid PPC composition [protein complex]**', 'CP complexes and lipid matrix [protein complex]'], ['thylakoid localized photosynthetic apparatus [protein complex]'], ['disparities in arrangement of Chlorophyll-Protein complexes [protein complex]'], ['**organization of CP complexes [structure]**', 'changes in supramolecular organization of CP complexes [structure]'], ['distribution of LHCs [protein complex]'], ['macroorganization of light-harvesting complexes [protein complex]'], ['accumulation of thylakoid multiprotein complexes [protein complex]']]}\n",
      "-------------------\n",
      "{'6974': [['sequential manner [phenotype]'], ['Symmetry-breaking event [phenotype]'], ['**synchronization [phenotype]**', 'developmental synchrony [phenotype]', 'synchronous pattern [phenotype]', 'tightly synchronized [phenotype]'], ['**developmental asynchrony [phenotype]**', 'asynchronous way [phenotype]'], ['**Coordinated development [phenotype]**', 'coordinated during developmental stages [phenotype]', 'coordinated development of multicellular organisms [phenotype]'], ['two or more developmental processes [phenotype]']]}\n",
      "-------------------\n",
      "{'96162': [['**leaf margins [tissue]**', 'margin [tissue]', 'edge of leaf [tissue]', 'leaf edge tissues [tissue]', 'marginal tissue [tissue]', 'marginal part of the leaf [tissue]'], ['leaf margin of L. chinense [tissue]'], ['margin of the 5th or 6th leaf [tissue]'], ['distal leaf margin [tissue]'], ['proximal leaf margin [tissue]'], ['leaf margin formation [tissue]'], ['**Midrib [tissue]**', 'midrib of leaf [tissue]', 'leaf midrib [tissue]', 'mid-rib [tissue]', 'midrib region [tissue]'], ['pepper leaf midrib [tissue]'], ['midrib of cauline leaf [tissue]'], ['**Leaf primordium [tissue]**', 'leaf primordial [tissue]', 'leaf primordial tissues [tissue]'], ['**developing leaf primordia [tissue]**', 'developing leaf primordium [tissue]'], ['**young leaf primordium [tissue]**', 'young leaf primordia [tissue]'], ['incipient leaf primordium [tissue]'], ['central region of leaf primordium [tissue]'], ['subtending leaf primordium [tissue]'], ['2 cm leaf primordia [tissue]'], ['entire leaf primordia [tissue]'], ['margin of young leaf primordia [tissue]'], ['two leaf/paraclade primordia per plant(s) [tissue]'], ['junction between terminal leaflet and lateral leaflet primordia [tissue]'], ['bases of leaflet primordia [tissue]'], ['leaf initiating sites [tissue]'], ['leaf middle of L. chinense [tissue]'], ['areas of the leaf where veins are no longer forming [tissue]'], ['phyllosphere of growing plant(s) [tissue]'], ['distal region of each leaf [tissue]'], ['leaf axis [tissue]'], ['developing ligule [tissue]'], ['distal green region [tissue]'], ['marginal domain [tissue]'], ['**primordial leaf margins [tissue]**', 'margins of leaf primordia [tissue]', 'primordial leaf margins [tissue]'], ['cortex of growing leaf margins [tissue]'], ['middle zone of leaf [tissue]'], ['young leaf margins [tissue]'], ['leaf epidermis of the sheath elongation zone [tissue]'], ['medial region [tissue]'], ['proximal part of the leaf [tissue]'], ['proximal region of the blade [tissue]'], ['leaf occipital part [tissue]'], ['first section of leaf [tissue]'], ['margins of old leaf [tissue]'], ['sinus [tissue]'], ['distal part of expanding leaf [tissue]'], ['growing regions of leaf blade [organ]'], ['central tissues [organ]'], ['future primary vein region [tissue]'], ['emerging leaf primordia [tissue]'], ['proximal part of the leaf primordium [tissue]'], ['distal part of young developing leaf [tissue]'], ['leaf marginal meristem (marginal blastozone) [tissue]']]}\n",
      "-------------------\n",
      "{'86937': [['AP-MS strategy [treatment]', '**AP-MS experiments [treatment]**', 'AP/MS experiment [treatment]', 'AP/MS experiments [treatment]', 'AP-MS protocol [treatment]'], ['mass spectrometry-based in vitro phosphorylation strategy [treatment]'], ['TEO-MS/MS [treatment]'], ['TAP-MS protocol [treatment]'], ['capillary electrophoresis mass spectrometry [treatment]', '**CZE-MS/MS [treatment]**', 'CE-MS using a triple quadrupole mass spectrometer [treatment]', 'CZE-MS/MS analysis [treatment]'], ['**Top-down Mass Spectrometry [treatment]**', 'top-down MS/MS [treatment]', 'top-down FTMS approach [treatment]'], ['MS2 experiments [treatment]'], ['Moving from UV to MS detection [treatment]'], ['muSPE method [treatment]'], ['The TMS method [treatment]', '**TMS [treatment]**'], ['Recent instrument developments [treatment]'], ['TbPL-mass spectrometry (TbPL-MS) [treatment]'], ['BIN2 PL-MS experiments [treatment]'], ['MS-based methods [treatment]'], ['MS results [treatment]'], ['Standard MS-based approaches [treatment]'], ['higher order MS/MS [treatment]'], ['technology development in mass spectrometry [treatment]', '**technological developments in mass spectrometry (MS) [treatment]**', 'advancements in MS technologies [treatment]'], ['SWATH-MS analysis [treatment]'], ['MS and MS/MS [treatment]'], ['methods for chromatography mass spectrometry-based optimization [concept]'], ['MS and chromatography experiments [treatment]'], ['**Peptidomics [treatment]**', 'mass spectrometry peptidomics [treatment]'], ['Integration of MS [treatment]'], ['proton-transfer-reaction mass spectrometry (PTR-MS) [treatment]'], ['molecular docking and dynamics simulations (MDS) [treatment]'], ['MS compatible mobile phase [treatment]'], ['detectable levels on 2xMS [treatment]'], ['MS-validated data [treatment]'], ['power of MSn [treatment]'], ['manual analyses of MS/MS raw data [treatment]'], ['matrix application [treatment]'], ['2D-BN/SDS-PAGE followed by MS analysis [treatment]'], ['conventional bottom-up MS [treatment]'], ['multi-element analysis [treatment]'], ['MSI experiments [treatment]'], ['combining PL-MS and phosphoproteomics [treatment]'], ['future developments in mass spectrometry [treatment]'], ['full MS<sup>1</sup> scans [treatment]'], ['development of core technologies in separations and mass spectrometry [treatment]'], ['pseudo-MS/MS spectra [treatment]'], ['automatic MS3 [treatment]'], ['development of novel mass analyzers [treatment]'], ['poor detectability by mass spectrometry (MS) [treatment]']]}\n",
      "-------------------\n",
      "{'152378': [['Seventeen SlSWEETs [gene]'], ['Six SlSWEETs in Clade I [gene]'], ['Five SlSWEETs in Clade II [gene]'], ['Salinas-Cornejo and colleagues [researcher]'], ['17 SlGASAs [gene]'], ['50 SlAAE gene(s) [gene]'], ['53 SlAAE gene(s) [gene]'], ['five SlZHD gene(s) [gene]'], ['three SlZHD gene(s) [gene]'], ['two SlZHD gene(s) [gene]'], ['16 SlZHD gene(s) [gene]'], ['Tomato genomes [genome]'], ['42% overall identity with tomato SWEETs [gene]'], ['five SlGASAs [gene]'], ['seven SlGASAs [gene]'], ['10 out of 12 tomato chromosomes [gene]']]}\n",
      "-------------------\n",
      "{'55154': [['**LlBTPC:Ub-LlPTPC:LlPTPC complex [protein complex]**', 'Ub-LlPTPC:LlPTPC [protein complex]', 'LlBTPC:Ub-LlPTPC:LlPTPC [protein complex]'], ['BTPC:Ub-PTPC:PTPC complex [protein complex]', '**PTPC:BTPC heterooctamer [protein complex]**', 'Ub-PTPC:PTPC complex [protein complex]'], ['most plant(s) PPs [protein complex]'], ['conserved PP4 complex [protein complex]'], ['conserved PP4 heterotrimer [protein complex]'], ['Protein-protein interaction (PPi) networks [protein complex]', '**PPI networks [protein complex]**'], ['Pr:Pr homodimer [protein complex]'], ['Pfr:Pr heterodimer [protein complex]'], ['stability of Pfr:Pfr dimers [protein complex]', 'stability of Pfr:Pfr [protein complex]', 'Pfr:Pfr homodimer [protein complex]', '**Pfr:Pfr [protein complex]**', 'Pfr:Pfr form [protein complex]', 'Pfr:Pfr indirectly [protein complex]'], ['60S:P-Stalk [protein complex]'], [\"interaction of PRP8 with 5'SS [protein complex]\"], [\"interaction of PRP8 with 3'SS [protein complex]\"], ['interaction of PRP8 with BPS [protein complex]'], ['processing-bodies (P-bodies) [protein complex]'], ['Ser/Thr PPs [protein complex]'], ['Ser/Thr as substrates for PKs and PPs [protein complex]'], ['T-pilus biogenesis [protein complex]'], ['Pfr:Pfr to Pfr:Pr [protein complex]'], ['Pfr:Pr to Pr:Pr [protein complex]']]}\n",
      "-------------------\n",
      "{'83676': [['**PPIs [protein-protein interaction]**', 'PPI-partners [protein-protein interaction]'], ['Higher-order PPIs [process]'], ['**PPIs [interaction]**', 'PPI inter-relationships [interaction]'], ['PPIs [protein function]']]}\n",
      "-------------------\n",
      "{'126195': [['small amounts of cohesin [phenotype]'], ['NOR condensation phenotypes [gene]'], ['decondensed FISH signals [phenotype]']]}\n",
      "-------------------\n",
      "{'76278': [['**L80 domain [protein domain]**', 'L80 domain [domain]'], ['GT-35-Glycogen-Phosphorylase [domain]']]}\n",
      "-------------------\n",
      "{'20565': [['susceptibility to DC3000(avrRps4) [phenotype]'], ['susceptibility to Pst-avrRps4 [phenotype]']]}\n",
      "-------------------\n",
      "{'28357': [['The molecular mechanisms of Na+/H+ transport [phenotype]', 'proper function of membrane attached Na+/H+ exchangers [phenotype]', \"Na+/H+ exchangers' activity [phenotype]\", 'function of Na+/H+ antiporters [phenotype]', 'Na+/H+ antiport activities [phenotype]', 'Na(+)/H(+)-exchange activity [phenotype]', '**Na+/H+ antiport [phenotype]**', 'Na+/H+ antiporter activity [phenotype]', 'Na(+) /H(+) antiport [phenotype]', 'Na(+)/H(+) transporting activity [phenotype]', 'Na(+)/H(+) antiport [phenotype]'], ['**plasma membrane Na+/H+ antiport activity [phenotype]**', 'PM Na+/H+ antiport activity [phenotype]', 'plasma membrane-bound Na+/H+ antiports [phenotype]', 'plasma membrane Na(+)/H(+)-exchange activity [phenotype]'], ['vacuolar Na+/H+ exchange activities [phenotype]'], ['**Na+/Pi transport activity [phenotype]**', 'Na+-dependent active transport of inorganic phosphate (Pi) [phenotype]', 'Na+/Pi co-transport activity [phenotype]'], ['Ca(2+)/H(+) antiport activity [phenotype]'], ['Na+ uptake activity [phenotype]'], ['Na+ transporter [phenotype]'], ['high selectivity for Na+ over K+ [feature]'], ['Na+-selectivity [phenotype]'], ['Na+-selective uniporter [phenotype]'], ['Na+ selective transporter [phenotype]'], ['Na(+)-selective transport [phenotype]'], ['**Na+-K+ cotransport [phenotype]**', 'Na+/K+ co-transport [phenotype]', 'Na(+)-K(+) cotransport [phenotype]', 'Na+-K+ co-transport [phenotype]', 'Na(+)-coupled K(+) transport [phenotype]'], ['Na+, K+, and Mn2+ transport activity [phenotype]'], ['Na+-coupled co-transporters [phenotype]'], ['molecules with Na+ ions [phenotype]'], ['molecules with Na+ or H+ ions [phenotype]'], ['Na+-dependent transport activity [phenotype]'], ['Na⁺-transporting activity [phenotype]'], ['Ni²⁺ transport activity [phenotype]'], ['NCX-like activity [phenotype]'], ['Na+ dependency of transport [phenotype]']]}\n",
      "-------------------\n",
      "{'168785': [['**Chloride (Cl-) [metabolite]**', 'chloride ions [metabolite]', 'Chloride ion (Cl) [metabolite]', 'Chloride (Cl(-)) [metabolite]'], ['K+, NO3-, and Cl- [metabolite]'], ['Cl+ [metabolite]'], ['Cl- and H+ [metabolite]'], ['sodium and chloride ions [metabolite]'], ['Low concentrations of Cl- [metabolite]']]}\n",
      "-------------------\n",
      "{'174379': [['adiponectin [hormone]'], ['Antiadipogenic functions of WDTC1 [gene]'], ['FAAH-silenced tissues [tissue]'], ['BCAAs in mouse fat cells [phenotype]'], ['dysfunctions of adipose tissue [organ]']]}\n",
      "-------------------\n",
      "{'151225': [['ChEC7 [gene]'], ['ChEC12 [gene]'], ['ChEC12a [gene]'], ['ChEC24 [gene]'], ['ChMob2 [gene]'], ['ChMob3 [gene]'], ['ChACE2 [gene]'], ['ChMob1 [gene]'], ['C. higginsianum Mob2 [gene]'], ['ChEC74 [gene]'], ['ChEC98 [gene]'], ['ChEC104 [gene]'], ['ChEC106 [gene]'], ['ChEC108 [gene]'], ['ChEC111 [gene]'], ['ChEC36 [gene]'], ['ChEC39 [gene]'], ['ChEC51a [gene]'], ['ChEC89 [gene]'], ['ChEC96 [gene]'], ['ChEC9 [gene]'], ['ChEC17 [gene]'], ['ChEC113 [gene]'], ['Ch063_04363 [gene]'], ['CH063_126029 [gene]'], ['CH063_07779 [gene]'], ['gene(s) CH63R_14477 [gene]'], ['effector candidate ChEC7 [gene]']]}\n",
      "-------------------\n",
      "{'167425': [['**Isothiocyanates [metabolite]**', 'isothiocyanate [metabolite]', 'Naturally-occurring isothiocyanates (ITCs) [metabolite]', 'ITCs [metabolite]', 'Isothiocyanates (ITCs) [metabolite]', 'Natural isothiocyanates [metabolite]'], ['**Glucosinolate-derived isothiocyanates (ITCs) [metabolite]**', 'isothiocyanates derived from glucosinolates [metabolite]'], ['**Aliphatic glucosinolate-derived isothiocyanates [metabolite]**', 'aGS-ITCs [metabolite]'], ['Aliphatic ITCs [metabolite]'], ['**Antimicrobial activity of aliphatic glucosinolate-derived isothiocyanates [metabolite]**', 'anti-microbial activity of AGS-derived isothiocyanates [metabolite]'], ['allyl isothiocyanate (AITC) [metabolite]'], ['**thiocyanates, isothiocyanates, and nitriles [metabolite]**', 'isothiocyanate, nitrile or thiocyanate [metabolite]'], ['isothiocyanates and nitriles [metabolite]'], ['isocyanates [metabolite]'], ['contents of isothiocyanates (ITCs) [metabolite]'], ['SCN [metabolite]'], ['aromatic isothiocyanates [metabolite]']]}\n",
      "-------------------\n",
      "{'65231': [['**DREAM complex [protein complex]**', 'DRM [protein complex]', 'DREAM complexes [protein complex]', 'DREAM-complex [protein complex]'], ['**DREAM complex [protein]**', 'DREAM complexes [protein]'], ['**DREAM complex components [protein complex]**', 'subunits of the DREAM complex [protein complex]', 'components of DREAM [protein complex]'], ['DREAM-like [protein complex]'], ['animal DREAM complexes [protein complex]'], ['dREAM/MMB complex [protein complex]'], ['DREAM/dREAM-like complex [protein complex]'], ['ALY3 [protein]']]}\n",
      "-------------------\n",
      "{'79544': [['NatA [gene]'], ['N-terminal acetyltransferase 1 (NAT1) [gene]'], ['NATB activity [gene]'], ['Nat [protein]'], ['NatF [protein]'], ['Nt-acetyltransferases (NATs) [protein]', '**N-terminal acetyltransferases (NATs) [protein]**'], ['five co-translationally acting Nats (NatA-E) [protein]'], ['posttranslational Nalpha-acetyltransferase (Nat) machinery [protein]', '**posttranslationally acting Nats [protein]**'], ['plastidic Nats [protein]'], ['NatA [enzyme]', '**N-terminal acetyltransferase A (NatA) [enzyme]**'], ['NATE [enzyme]'], ['NatF [enzyme]'], ['NATH [enzyme]'], ['**NatB [enzyme]**', 'Nt-acetyltransferase B (NatB) [enzyme]'], ['N-terminal acetyltransferases (NATs) [enzyme]'], ['five ribosome-associated N-acetyltransferases (NatA-E) [enzyme]'], ['human NatA activity [enzyme]'], ['**NATB-mediated NTA [treatment]**', 'NatB-dependent NTA [treatment]'], ['cotranslational NTA [treatment]'], ['Posttranslational and cotranslational NTA [treatment]'], ['**NTA of ACOs [treatment]**', 'NTA modification of ACOs [treatment]'], ['NatA-mediated acetylation [treatment]'], ['NatA [treatment]'], ['NATB-mediated NTA [protein modification]'], ['NATA activity [process]'], ['N-terminal acetylation by NatB [process]'], ['NatA substrates [metabolite]'], ['NATA activity [phenotype]'], ['NatA, NatB, and NatC/E/F [phenotype]'], ['**N-terminal acetyltransferases (Nat) complexes [protein complex]**', 'N-terminal acetyltransferase (Nat) complexes [protein complex]'], ['NATE [protein complex]'], ['NatF [protein complex]'], ['Nalpha-acetyltransferase A (NatA) complex [protein complex]', '**N-terminal acetyltransferase A (NatA) complex [protein complex]**', 'Nalpha-acetyltransferase A complex [protein complex]', 'N-terminal acetyltransferase complex A (NatA) [protein complex]'], ['**N-terminal acetyltransferase complex E [protein complex]**', 'N-terminal acetyltransferase E complex [protein complex]'], ['seven N<sup>\\nα</sup>-acetyltransferases (NatA-F and NatH) [protein complex]'], ['N-terminal acetyltransferase (Nat) complexes NatA and NatB [protein complex]'], ['human NatA complex [protein complex]'], ['Nalpha-terminal acetyltransferase B (NatB) [protein complex]'], ['ribosome-associated N-terminal acetyltransferases (Nats) [protein complex]']]}\n",
      "-------------------\n",
      "{'152992': [['KCS gene(s) [gene]'], ['Human gene(s) encoding Elongator subunits [gene]'], ['archetype SUMO gene(s) from T. hassleriana [gene]'], ['EcSRS-L gene(s) [gene]'], ['next gene(s) upstream of SPT [gene]'], ['symbiosis-associated gene(s) [gene]'], ['GSK sequence in Porphyra [gene]'], ['T. cruzi KMP11 gene(s) [gene]'], ['5 flagellin (FliC) paralogs [gene]'], ['animal SC35 gene(s) [gene]'], ['conserved symbiosis gene(s) [gene]'], ['4 Skp1-like gene(s) [gene]'], ['evolutionarily conserved Spo11 [gene]'], ['conserved function of SsYABBY gene(s) [gene]'], ['well-conserved biochemical functions of CRC [gene]'], ['Chlamydomona FKBP gene(s) families [gene]']]}\n",
      "-------------------\n",
      "{'60554': [['NW-type zygomorphy [phenotype]'], ['divided into ten subfamilies based on phylogenetic relationship [phenotype]'], ['four subclades [phenotype]'], ['actinomorphy [phenotype]'], ['transition to complete actinomorphy [phenotype]'], ['seven subclades [phenotype]']]}\n",
      "-------------------\n",
      "{'125654': [['Genome reduction [process]'], ['incomplete genome annotation [process]'], ['imprecise genome annotation [process]'], ['patients with these syndromes [organism]'], ['Genomic DNA loss [genomic element]'], ['suppression of genetic modification [genomic event]'], ['Loss of integrity of genomic DNA [genomic dna]'], ['genome degradation [process]'], ['organellar genome degradation [process]'], ['genome abandonment [process]'], ['genomic damage [genome]'], ['damage to DNA [genetic material]'], ['genome contraction [event]'], ['genome size reduction [genome]'], ['potential gene(s) loss during coconut genome evolution [organism]'], ['elimination from the genome [process]'], ['intrinsic stresses [treatment]'], ['loss of genome [genome]']]}\n",
      "-------------------\n",
      "{'15562': [['root(s) hair and trichome development under salt treatment [phenotype]'], ['root(s) hair growth under NaCl treatment [phenotype]'], ['proper root(s) hair growth [phenotype]'], ['primary root(s) and root(s) hairs development [phenotype]'], ['**root(s) hair response to abiotic stresses [phenotype]**', 'root(s) hair stress response [phenotype]'], ['root(s) hair development under stress conditions [phenotype]'], ['root(s) hair auxin uptake capacity [phenotype]'], ['root(s) hair growth stimulation [phenotype]'], ['re-establishment of root(s) hair development [phenotype]'], ['active root(s) hair growth [phenotype]'], ['root(s) hair growth response [phenotype]'], ['conditional root(s) hair development [phenotype]']]}\n",
      "-------------------\n",
      "{'21958': [['Engineering strategy [phenotype]'], ['improving plant(s) architecture [phenotype]'], ['**Transformation methodology [phenotype]**', 'Transformation system [phenotype]', 'stable transformation methods [phenotype]', 'technology for transformation [phenotype]', 'simple genetic transformation and cultivation protocols [phenotype]', 'transformation protocols [phenotype]'], ['**Engineering complex traits [phenotype]**', 'the engineering of morphologies of agronomic interest [phenotype]', 'methods for complex trait engineering [phenotype]'], ['glycoengineering strategy [phenotype]'], ['genetic engineering approaches [phenotype]', '**Genetic engineering strategies [phenotype]**'], ['artificial methods [phenotype]'], ['use of transgenic approaches [phenotype]'], ['Genetic approaches [process]'], ['Stable methods [process]'], ['engineering efforts [process]'], ['attractive modality for trait engineering in plant(s) [process]'], ['engineering a crop [technique]'], ['molecular breeding strategies [study]']]}\n",
      "-------------------\n",
      "{'65518': [['K9-methylated histone H3 [protein]'], ['K9 dimethylated H3 tails [molecule]']]}\n",
      "-------------------\n",
      "{'106938': [['RELAX program [treatment]'], ['MITOFY [treatment]'], ['PAML software package [treatment]'], ['codeml program [treatment]']]}\n",
      "-------------------\n",
      "{'31479': [['crosstalk between cell wall [tissue]'], ['cell wall-related pathways [process]'], ['cell wall modification and remodeling, protein(s) metabolism and signal transduction [function]'], ['cell wall interactions [process]']]}\n",
      "-------------------\n",
      "{'8680': [['distinct expression patterns [phenotype]'], ['two lines of different biological functions [phenotype]'], ['organism specific [phenotype]'], ['unique biological roles [phenotype]'], ['ecotype-specific expression [phenotype]'], ['distinct diversification [phenotype]'], ['distinct developmental roles [phenotype]']]}\n",
      "-------------------\n",
      "{'84113': [['15N-1H dipolar couplings [phenomenon]'], ['high TH activity [phenotype]'], ['s020,w and D20,w [phenotype]'], ['bundling cutoff angle [phenotype]'], ['MD barriers [phenotype]'], ['PM6-D3 barriers [phenotype]'], ['elevated R2 rates [measurement]'], ['R1 rates [measurement]'], ['R2 15N relaxation rates [measurement]'], ['tau1 [phenotype]'], ['thermal reversion rates [parameter]'], ['Enthalpy of interaction [phenotype]'], ['EX2 behavior [phenotype]'], ['T(m)s of archaeal TBPs [phenotype]'], ['WTCg [phenotype]'], ['high deuterium exchange rate [phenotype]'], ['slow nucleotide exchange property [phenotype]'], ['motionally averaged C6 2H quadrupolar coupling [phenotype]'], ['large rigid-limit CQ values [phenomenon]', '**rigid-limit values of 22.7 kHz [phenomenon]**'], ['dipolar couplings [phenomenon]'], ['CQ [phenotype]'], ['rigid-limit quadrupolar coupling [phenotype]'], ['TH activity [phenotype]'], ['folding free energies of -27.5 kcal mol-1 [phenotype]'], ['Tm by 6-8 C [phenotype]'], ['carbonylation rate constant [phenotype]'], ['tilt angle [phenotype]'], ['DeltaGtransf [phenotype]'], ['DeltaGfold~-17 kcal/mol [phenotype]'], ['factor ranging from 1.5 to 3.9 in polymer micelles [phenotype]'], ['lowest conjugation level [phenotype]'], ['E and alpha are independent of temperature [parameter]'], ['9.7 Hz [phenotype]'], ['temperature-induced unfolding midpoint (T m) [phenotype]'], ['highest level of hormone binding at 0°C [phenotype]'], ['optimal conditions for photoconversion [phenotype]'], ['deuteron-hydrogen exchange rates [phenotype]'], ['degree ranges from 6 up to 14 [phenotype]'], ['dissociation half-life of 4.2 min [phenotype]'], ['dissociation half-life of 25 s [phenotype]'], ['-TDeltaS = -32.7 kJ mol-1 [phenotype]'], ['thetaI [coefficient]'], ['50.88°C and 66.51°C [phenotype]'], ['diffusion coefficient of 8.8×10^-3 mum2*s^-1 [phenotype]'], ['average first-order rate constant of turnover [phenotype]'], ['LLPS transition temperature [phenotype]'], ['dissociation rate of NPH3 [phenotype]'], ['melting point of 62.8°C [phenotype]'], ['-0.31 cal mol-1 K-1 [phenotype]'], ['5-fold increase in the maximal reaction rate [phenotype]'], ['77 K [condition]'], ['significant temperature dependence [phenotype]'], ['midpoint transition temperature of 44 degrees C [phenotype]'], ['transition temperature of 49 degrees C [phenotype]'], ['transition temperature of 58 degrees C [phenotype]'], ['duty ratio of ~90% [phenotype]'], ['lifetime of 0.56 s [phenotype]'], ['half-times ranging from 52 to 120s [phenotype]']]}\n",
      "-------------------\n",
      "{'73108': [['NPH3 [protein]'], ['**PHOTOTROPIN1 (PHOT1/NPH1) [protein]**', 'Nonphototropic hypocotyl 1 (NPH1) protein(s) [protein]'], ['NPH3/RPT2-Like (NRL) protein(s) [protein]'], ['NPH4 [protein]'], ['NON-PHOTOTROPIC HYPOCOTYL 3 (NPH3)/RPT2-like (NRL) family [protein family]'], ['NPH3 function [protein]']]}\n",
      "-------------------\n",
      "{'55344': [['Important putative catalytic residues [protein motif]'], ['Four characteristic cysteine residues for zinc docking [protein motif]'], ['cysteine-histidine-asparagine catalytic triad [protein motif]'], ['VAIK catalytic motif [protein motif]']]}\n",
      "-------------------\n",
      "{'113817': [['diverse transcription factors (TF) [gene]', 'multiple TFs [gene]', 'different sets of TFs [gene]', '**TF(s) [gene]**', 'various TFs [gene]', 'Numerous transcription factors (TFs) [gene]', 'some TFs [gene]', 'specific transcription factors (TFs) [gene]', 'multiples TFs [gene]', 'at least 2 classes of TFs [gene]', 'Many valuable transcription factors (TFs) [gene]', 'key transcription factors (TFs) [gene]', 'several transcription factors (TFs) [gene]', 'specific sets of key TFs [gene]', 'master transcription factors (TFs) [gene]', 'few selected transcriptional factors (TFs) [gene]'], ['regulation of transcription factors (TFs) [gene]'], ['**transcription factor (TFs) families [gene]**', 'Transcription factor (TF) families [gene]'], ['transcription factor (TF) and their targets [gene]'], ['novel TF modules [gene]'], ['**individual TFs [protein]**', 'one TF [protein]', 'certain TFs [protein]', 'Specific TFs [protein]', 'several TFs [protein]', 'Different classes of TFs [protein]', 'many transcription factors (TFs) [protein]', 'many different TFs [protein]', 'regulating TFs [protein]', 'various transcription factors (TFs) [protein]', 'Numerous transcription factors (TFs) [protein]', 'specific TF [protein]'], ['TFs [genetic variation]'], ['regulation via TF monomers [protein complex]']]}\n",
      "-------------------\n",
      "{'115283': [['**high competence of the transgene loci [phenotype]**', 'ability of transgenes to function as masters or targets of epigenetically heritable trans-silencing interactions [phenotype]']]}\n",
      "-------------------\n",
      "{'152739': [['At1g48490 [gene]'], ['T7-SIK1KD [gene]'], ['HsPKR [gene]'], ['Srb10 [gene]'], ['**Prp4 kinase [gene]**', 'Prp4k [gene]'], ['PGSC0003DMG400004655 [gene]'], ['Pka1 [gene]'], ['SpWee1 kinase [gene]'], ['SKY1 [gene]'], ['Membrane-associated IRAK/Pelle kinase SSP [gene]'], ['PpARK/PpCTR1 [gene]'], ['At2G01210 [gene]'], ['**ZmSIRK1 activation segment [gene]**', 'Activation segment of ZmSIRK1 [gene]'], ['E805ZmSIRK1 [gene]'], ['ABA-box of ZmOST1 [gene]'], ['AT2G30940 [gene]'], ['R.stricta.7380 [gene]'], ['R.stricta.23118 [gene]'], ['R.stricta.23205 [gene]'], ['MpNEK1 [gene]'], ['p70s6k [gene]'], ['QSK1 [gene]'], ['ZEUS1 [gene]'], ['gene(s) At1g73080 [gene]'], ['PKS6 coding sequence [gene]'], ['**ZRK7 [gene]**', 'ZRK 7 [gene]'], ['transcription of SHAGGY-like kinase 11/12 (SK11/12) [gene]'], ['yeast Ypk3 [gene]'], ['ZmOST1-interactive polypeptide ZmKS [gene]'], ['spliceosome-associated kinase [gene]'], ['NDR/LATS kinase Dbf2 [gene]'], ['gene(s) encoding SAK1/PAK1 [gene]'], ['ZRK 1 [gene]'], ['PSKR1-KD [gene]'], ['ZmOST1 phosphorylation [gene]'], ['Sterile 20 (STE20)/Hippo orthologue [gene]'], ['known mammalian RBSK [gene]']]}\n",
      "-------------------\n",
      "{'37690': [['zet-4 and zeth-1 double mutant [mutant]'], ['zet-1 (Ler) /zet-4 (Col) [mutant]'], ['zet-1 ZETH (Ler) /zet-4 zeth-1 (Col) [mutant]']]}\n",
      "-------------------\n",
      "{'31898': [['**thylakoid formation [subcellular compartment]**', 'developing thylakoid membrane [subcellular compartment]', 'assembly of photosynthetic membranes [subcellular compartment]'], ['**thylakoid formation [organ]**', 'assembly of thylakoid membranes [organ]', 'assembly of thylakoids [organ]'], ['assembly of thylakoid pigment-protein complexes [process]']]}\n",
      "-------------------\n",
      "{'120002': [['introgression of particular alleles [gene]', '**introgression of allele [gene]**', 'repeated exchanges of alleles [gene]'], ['occasional introgression of highly distinct genotypes [genotype]'], ['introgression for alleles [genetic element]']]}\n",
      "-------------------\n",
      "{'38784': [['Argah lines [mutant]'], ['COLDAIR_Mut lines [mutant]'], ['IAA13-overexpressing lines [mutant]'], ['CCA1-inducible lines [mutant]'], ['wrky40 mutant lines [mutant]'], ['AtZTL-ox lines [mutant]'], ['TAR1 disturbed expression lines [mutant]'], ['CuAO8 insertional mutant lines [mutant]'], ['tno1 lines [mutant]'], ['STGalTX-DeltaXF lines [mutant]'], ['STGalT-DeltaXF lines [mutant]'], ['**ACD line [mutant]**', 'ACD lines [mutant]'], ['WRKY23-SRDX lines [mutant]'], ['wrky23-1 lines [mutant]'], ['wrky23-2 lines [mutant]'], ['CHUP1-R4A&S12A&R20Aox lines [mutant]'], ['OXCCSD lines [mutant]'], ['AlaAT-expressing lines [mutant]'], ['ABA-insensitive lines [mutant]'], ['SoIAA2Delta27-expressing lines [mutant]'], ['GLR3.7-S860A overexpression lines 10-2 and 15-6 [mutant]'], ['K-TTG1 positive lines [mutant]'], ['NF-YA repressor lines [mutant]'], ['Gmnmhc5 lines [mutant]'], ['bHLH059 lines [mutant]'], ['ext18 insert lines [mutant]'], ['B. distachyon HCT down-regulated lines [mutant]'], ['PeuLAC2-OE lines [mutant]'], ['35S-TCP5 lines [mutant]'], ['aux1rcr1ox lines [mutant]'], ['SAE2UFDCt expressing lines [mutant]'], ['AS-IAA3 lines [mutant]'], ['FLC + NOS-T line [mutant]'], ['control and mXyl lines [mutant]'], ['Constitutively-deficient phytochrome lines [mutant]'], ['iaaMox lines [mutant]'], ['JA-insensitive (jai1) lines [mutant]'], ['Ta CYP20-2 lines [mutant]'], ['ZmOXS2a expression lines [mutant]'], ['StBMI1-1-AS lines [mutant]'], ['Lines altered in AtGRXS17 expression [mutant]'], ['AOX2 antisense lines [mutant]'], ['CoAOS1/2 lines [mutant]'], ['CRTI-expressing lines [mutant]'], ['hd2a mutant lines [mutant]'], ['cytoplasmic COP1 lines [mutant]'], ['Col FLC lines [mutant]'], ['PHToxYCF1ox lines [mutant]'], ['previous TOR suppression lines [mutant]'], ['NF-YA2 repressor line [mutant]'], ['SYP121R20AR21A lines [gene]'], ['TsBAHD-expressing lines [mutant]'], ['QuoA expression lines [mutant]'], ['ToxA sensitive line [phenotype]'], ['gain-of-function aha2 mutant lines [mutant]'], ['cax4 lines [mutant]'], ['Cd-treated lines [mutant]'], ['TaAIP2A complemented lines [mutant]'], ['TaAIP2B complemented lines [mutant]']]}\n",
      "-------------------\n",
      "{'165081': [['ion balance [metabolite]'], ['luminal Ba2+ [metabolite]'], ['Increased ion level [metabolite]'], ['Trapped ions [metabolite]'], ['Cation levels [metabolite]'], ['Cytosolic anion concentration [metabolite]'], ['ion osmoregulation [metabolite]'], ['concentration of anions [metabolite]'], ['homeostasis of cations [metabolite]'], ['ion concentrations outside the cell [metabolite]'], ['ion concentrations of their cellular environments [metabolite]'], ['concentrations of ions [metabolite]'], ['cation exchange [metabolite]']]}\n",
      "-------------------\n",
      "{'70479': [['**PTR family [protein family]**', 'POT/PTR family of transporters [protein family]'], ['Structures for Atm family members [protein family]'], ['PTR2/POT transporter family [protein family]'], ['**ProT family [gene family]**', 'proline transporters (ProTs) [gene family]'], ['ProT family [gene]'], ['large plasma membrane-localized transporter family [protein family]']]}\n",
      "-------------------\n",
      "{'126834': [['**DNA replication stress response [phenotype]**', 'preventing replicative stress [phenotype]', 'DNA replication to continue [phenotype]'], ['DDT [process]', '**DNA damage tolerance [process]**'], ['**DDT [phenotype]**', 'two-branch DNA-damage tolerance mechanism [phenotype]', 'DNA damage tolerance pathways [phenotype]', 'error-free DNA damage tolerance pathway [phenotype]', 'different DNA damage tolerance pathways [phenotype]', 'error-free DNA-damage tolerance [phenotype]', 'error-free DNA damage tolerance [phenotype]'], ['efficient TCR [phenotype]'], ['enhanced DNA synthesis-dependent endoreplication [phenotype]'], ['effective DNA damage responses [phenotype]']]}\n",
      "-------------------\n",
      "{'83479': [['Accumulation of protein(s) assemblies [phenotype]'], ['protein(s) folding and assembly [phenotype]'], ['nucleotide-dependent self-assembly [phenotype]', '**GTP-dependent self-assembly [phenotype]**'], ['complex assembly of PIN [phenotype]'], ['assembly of complexes [phenotype]', 'formation of the protein(s) complex [phenotype]', '**protein(s) complex assembly [phenotype]**', 'protein(s) assembly formation [phenotype]', 'formation of protein(s) assemblies [phenotype]', 'formation of protein(s) complexes [phenotype]', 'formation of a protein(s) complex [phenotype]', \"'cellular protein-containing complex assembly' [phenotype]\"], ['**purification of in vivo complexes [phenotype]**', 'copurification of in vivo complexes [phenotype]'], ['PRR complex formation [phenotype]'], ['**protein(s) assembly [phenotype]**', 'assembly of protein(s) [phenotype]'], ['receptor complex formation [phenotype]'], ['formation or retention of megacomplexes [phenotype]'], ['stable signaling complexes [phenotype]'], ['short-lived signaling complexes [phenotype]'], ['Oligomerization [phenotype]'], ['large signaling complexes [phenotype]'], ['newly assembled complexes [phenotype]'], ['reconstructing assembly pathways [phenotype]'], ['protein(s) assembly and processing [phenotype]'], ['maturation of iron-sulfur cluster assembly [phenotype]'], ['**iron-sulfur cluster assembly [phenotype]**', 'iron/sulfur cluster formation [phenotype]'], ['rapid assembly [phenotype]'], ['rate of assembly [phenotype]'], ['inflammasome assembly [phenotype]'], ['compaction of nascent peptide chain [phenotype]'], ['signaling complex [phenotype]'], ['assembly of protein(s) in ER [phenotype]'], ['production of inclusion bodies [phenotype]'], ['active complex formation [phenotype]'], ['TET web formation [phenotype]'], ['producing active monomers [phenotype]'], ['assembly yield [phenotype]'], ['pattern spontaneously by diffusive protein(s) movement and protein(s) complex formation [phenotype]'], ['assembly with BPMs [phenotype]'], ['dynamic head-to-tail polymerization [phenotype]', '**head-to-tail polymerization [phenotype]**'], ['assembly of higher-order complexes [phenotype]'], ['synthesis and assembly of mitochondrial protein(s) complexes [phenotype]'], ['shifted complexes [phenotype]'], ['early steps of formation [phenotype]'], ['complex activation [phenotype]'], ['aggregation of newly synthesized protein(s) [phenotype]'], ['production of conjugated vaccines [phenotype]'], ['stabilization of import complex [phenotype]'], ['subunit assembly [phenotype]'], ['in vivo complex formation [phenotype]'], ['clusters containing protein(s) associated with protein(s) assembly and degradation [phenotype]'], ['multiprotein complex formation [phenotype]'], ['formation of specific protein(s) complexes [phenotype]'], ['**a critical concentration in the very low micromolar range [phenotype]**', 'critical transition concentration [phenotype]'], ['assembly into VLPs [phenotype]'], ['recruitment of class I protein(s) [phenotype]']]}\n",
      "-------------------\n",
      "{'123875': [['RNA Pol III-dependent U3 small nuclear RNA (snRNA) promoter [gene]', 'RNA polymerase III -type 3- U3 or U6 promoters [gene]', 'RNA Polymerase III (U6 or U3 promoters) [gene]', '**U6 or U3 small nuclear RNA (snRNA) promoters [gene]**'], ['type 1 promoters of RNA polymerase III [gene]'], ['**type 3 promoter of RNA polymerase III [gene]**', 'type 3 of pol III gene(s) [gene]'], ['RNA polymerase-III (Pol III) promoters [gene]', 'Pol III RNA gene(s) promoters [gene]', '**RNA polymerase III promoter [gene]**'], ['Pol III promoter [treatment]'], ['RNA polymerase III (Pol III) promoter [molecule]']]}\n",
      "-------------------\n",
      "{'107511': [['AGO1 RIP-seq [treatment]'], ['small RNA cloning and sequencing projects [treatment]'], ['transcriptome analysis on recombinant inbred lines (RILs) [treatment]'], ['Bookend Floral Bud annotation [treatment]'], ['Laser capture RNAseq of developing ligules [method]'], ['mining of maize expressed sequence tag and RNA-profiling databases [method]'], ['Filichkin et al. [researcher]'], ['Raherison et al., 2012 [researcher]'], ['deep sequencing of AGO1-small RNA complexes [technique]'], ['miRNA sequencing, degradome, and physiological analyses [technique]'], ['IGP [tissue]'], ['tissue analyzed [tissue]'], ['mining the repository of available ESTs [process]'], ['Deep sequencing of AGO1-bound small RNAs [process]'], ['dissect A. thaliana smRNAome [process]'], ['genome-wide surveys of uncapped RNA molecules [process]'], ['SERS bioimaging of microRNA [process]'], ['Laser-capture microdissection RNA-seq [phenotype]'], ['Genome-wide analyses of RNA Helicase gene(s) family [phenotype]'], ['unwounded TRAP-Seq comparisons [phenotype]'], ['wounded TRAP-Seq comparisons [phenotype]'], ['RNA-Seq analysis of pollen isolated from single anther [phenotype]'], ['analysis of pooled rice RNA [phenotype]'], ['RNA-seq analysis of cpSG [protein complex]'], ['high throughput sequencing of a wheat small RNA library [gene]'], ['Analysis of expressed EST [gene]'], ['RNA and protein(s) sequences [gene]'], ['tRNA sequencings in various plant(s) [organism]'], ['RNA from whole seedlings [genetic material]'], ['Bazin et al. [study]'], ['173 studies on other species [study]'], ['ANT-GR RNA-Seq experiment [experiment]'], ['large-scale analysis on rice poly(A) signals [sequence]']]}\n",
      "-------------------\n",
      "{'162516': [['**Overproduction of reactive oxygen species [metabolite]**', 'excess ROS [metabolite]', 'excessive production of ROS [metabolite]', 'over-production of ROS [metabolite]', 'Over-accumulation of ROS [metabolite]', 'over-accumulated ROS levels [metabolite]', 'Overwhelming production of ROS [metabolite]', 'stress-stimulated overproduction of ROS [metabolite]', 'ROS overproduction [metabolite]', 'Over-accumulation of reactive oxygen species (ROS) [metabolite]', 'Over accumulation of ROS [metabolite]', 'Overaccumulation of ROS [metabolite]', 'uncontrolled overproduction of ROS [metabolite]', 'Overproduction of reactive oxygen species (ROS) [metabolite]', 'formation of excessive ROS species [metabolite]', 'production of excessive reactive oxygen species (ROS) [metabolite]', 'uncontrolled production of reactive oxygen species (ROS) [metabolite]', 'redundant ROS accumulation [metabolite]', 'excess generation of ROS [metabolite]', 'excessive generation of reactive oxygen species (ROS) [metabolite]', 'ROS excesses [metabolite]', 'increasing overproduction of reactive oxygen species [metabolite]', 'ROS over-generation [metabolite]', 'formation of excess ROS [metabolite]', 'excess level of ROS species [metabolite]', 'overflow of ROS [metabolite]', 'stress-induced excess ROS [metabolite]', 'excessive generation of ROS [metabolite]', 'overaccumulated ROS [metabolite]', 'production of excessive ROS [metabolite]', 'excessively produced ROS [metabolite]', 'excessive formation of ROS [metabolite]', 'hyperaccumulation of reactive oxygen species [metabolite]', 'over accumulated ROS [metabolite]', 'generation of excess metabolic ROS [metabolite]'], ['overaccumulation of mitochondrial originated ROS [metabolite]', '**overproduction of mitochondrial ROS [metabolite]**', 'production of excess ROS in mitochondria [metabolite]', 'mitochondrial ROS overproduction [metabolite]', 'excess of mitochondrial ROS load [metabolite]'], ['**imbalance of ROS production [metabolite]**', 'an imbalance between the production of reactive oxygen species (ROS) [metabolite]', 'imbalance in the generation and removal of reactive oxygen species (ROS) [metabolite]'], ['oxidative stress [gene]'], ['age induced excess ROS [metabolite]'], ['excess release of reactive oxygen species (ROS) [metabolite]'], ['ROS overproduction [process]'], ['overproduction of reactive oxygen/nitrogen species [metabolite]'], ['production of excess ROS in the cytoplasm [metabolite]'], ['prolonged ROS production [metabolite]'], ['excess-initiation of ROS [metabolite]'], ['high risk for generation of reactive oxygen species (ROS) [metabolite]'], ['excessive nuclear ROS levels [metabolite]']]}\n",
      "-------------------\n",
      "{'135491': [['HvTFIIS [gene]'], ['FtsH11 [gene]'], ['CaFtsH06 [gene]'], ['CaFtsH05 [gene]'], ['CaFtsH01 [gene]'], ['CaFtsH04 [gene]'], ['CaFtsH09 [gene]'], ['CaFtsH10 [gene]'], ['CaFtsH11 [gene]'], ['CaFtsH02 [gene]'], ['CaFtsH07 [gene]'], ['CaFtsH08 [gene]'], [\"F6'H1 WT [gene]\"], ['FtsHi3 [gene]'], ['AtFtsH3/10 [gene]'], ['FtsHi [gene]'], ['HTD1 protein(s) [protein]'], ['His-FPG [gene]'], ['MpHSFA1 [gene]'], ['Mphsfb1 [gene]'], ['Mphsfa1V [gene]'], ['HSFA1U [gene]'], ['HSFA1V [gene]'], ['HaFT-1 [gene]'], ['CtHSR1 [gene]'], ['Filamentation temperature-sensitive H (FtsH) [gene]'], ['FtsH1-FtsH4 [gene]'], ['FtsH family [gene family]'], ['12 ftsH gene(s) [gene family]']]}\n",
      "-------------------\n",
      "{'167352': [['SM [metabolite]', 'sinapoylmalate [metabolite]', '**Sinapoyl malate [metabolite]**', 'Sinapoyl malate (SM) [metabolite]', 'Sinapoyl-malate [metabolite]', 'AA or sinapoyl malate [metabolite]', 'sinapoylmalate (SM) [metabolite]', 'synapoyl malate [metabolite]', 'sinapoylmalates [metabolite]'], ['**Sinapoyl malate (SM) [compound]**', 'sinapolymalate [compound]'], ['**cis-isomer of SM [metabolite]**', 'cis-SM [metabolite]'], ['trans-SM [metabolite]'], ['sinapoyl-malate precursors [metabolite]'], ['Role of sinapoyl malate [metabolite]'], ['soluble SM [metabolite]'], ['SM/OH-FM ratio [metabolite]'], ['new correlation between malate and sinapate [phenotype]'], ['reduced sinapoyl malate content [metabolite]'], ['dicarboxylate anion of sinapic acid (S)-malate ester [metabolite]'], ['accumulation of sinapoylmalate [metabolite]'], ['monomeric precursors sinapoyldimalate (SMM) [metabolite]'], ['Si-HM [metabolite]'], ['SM-HM [metabolite]'], ['**sinapoylmalate production [metabolite]**', 'production of sinapoylmalate [metabolite]'], ['greater sinapoylmalate production [metabolite]'], ['**synthesis of sinapoyl malate [metabolite]**', 'synthesis of sinapoylmalate [metabolite]'], ['loss in sinapoyl malate [metabolite]'], ['novel mechanism involving sinapoyl malate [process]'], ['2-O-sinapoyl-L-malate [metabolite]']]}\n",
      "-------------------\n",
      "{'165067': [['**Potassium ion [metabolite]**', 'potassium ions [metabolite]', 'a potassium cation [metabolite]'], ['Cytosolic potassium (K+) [metabolite]', '**Cytosolic K+ [metabolite]**', 'cytoplasmic K+ [metabolite]'], ['K+ and H+ [metabolite]'], ['**sodium or potassium ion [metabolite]**', 'K+ or Na+ [metabolite]'], ['presence of potassium ions [metabolite]']]}\n",
      "-------------------\n",
      "{'66664': [['**Zinc finger protein(s) [gene]**', 'Zinc-finger protein(s) [gene]', 'Zn-finger protein(s) [gene]'], ['Zn-finger protein(s) [protein]', '**Zinc finger protein(s) [protein]**', 'Zinc-finger protein(s) [protein]', 'zinc-finger-proteins [protein]', 'a zinc finger protein(s) [protein]'], ['ZINC FINGER protein(s) 8 [gene]'], ['three zinc finger protein(s) [protein]'], ['zinc finger binding protein(s) [protein]'], ['small zinc finger protein(s) [protein]'], ['MYC-associated zinc finger (MAZ) protein(s) [protein]'], ['tandem zinc finger C-x8-C-x5-C-x3-H type family protein(s) [gene]'], ['zinc finger C-x8-C-x5-C-x3-H type family protein(s) [gene]'], ['zinc finger protein(s) (ZmZF-5) [protein]'], ['type-IV zinc-finger protein(s) [protein]'], ['Animal zinc finger protein(s) [protein]'], ['bi-zinc finger polypeptides [protein]'], ['Tandem zinc finger protein(s) (TZFs) [protein]'], ['synthetic zinc finger protein(s) [gene]'], ['zinc finger structure protein(s) [protein]'], ['artificial zinc finger protein(s) [protein]'], ['1-4 zinc finger structures [protein]'], ['a zinc-finger-domain protein(s) [protein]'], ['zinc-finger and MSI1 protein(s) [protein]'], ['ZINC FINGER protein(s) 4 [gene]'], ['mRNA encoding zinc finger protein(s) [metabolite]'], ['DnaJ-like zinc finger protein(s) [protein]'], ['zinc finger and VEFS domain-containing protein(s) [phenotype]'], ['four zinc finger protein(s) [protein]'], ['pox virus zinc finger protein(s) [protein]'], ['zinc finger protein(s) AZF2 [protein]'], ['several zinc finger protein(s) [protein]'], ['tandem zinc finger protein(s) [protein]'], ['DNA-binding Zn-finger protein(s) [protein]'], ['custom-made DNA-binding protein(s) [protein]'], ['WIP-type zinc finger protein(s) [protein]'], ['zinc-finger protein(s) 2 [protein]']]}\n",
      "-------------------\n",
      "{'61923': [['AGB1-mCherry [protein]'], ['**mCherry-sfGFP [protein]**', 'mCherry-sfGFP reporter [protein]'], ['cis alpha-GolgiManI-mCherry [protein]'], ['Got1p homologue-mCherry [protein]'], ['nuclear marker AtWWP1-mCherry [protein]']]}\n",
      "-------------------\n",
      "{'162359': [['ROS production by RBOHs [metabolite]', '**RBOH-dependent ROS production [metabolite]**'], ['RBOHD-generated ROS [metabolite]'], ['RBOH activity [process]'], ['RBOH-dependent ROS production [process]'], ['activation of RBOHs [process]'], ['**ROS-producing RBOHs [enzyme]**', 'RbOH [enzyme]'], ['RBOH activation [phenotype]'], ['RBOHF-dependent ROS accumulation [phenotype]']]}\n",
      "-------------------\n",
      "{'102013': [['A. thaliana floral tissues [organ]'], ['A. thaliana tissues and organs [tissue/organ]'], ['A. thaliana graft junction [tissue]'], ['A. thaliana leaf vascular tissue [tissue]'], ['flavonol-rich tissues of A. thaliana [organism]']]}\n",
      "-------------------\n",
      "{'153636': [['full-length KOR1 [gene]'], ['full-length TAF1 [gene]'], ['full-length LAZ5 [gene]'], ['full-length CLV1 [gene]'], ['Full-length MSH1 [gene]'], ['Full-length EARLI1 [gene]'], ['full-length MiIDL1 gene(s) [gene]'], ['full-length CLV2 [gene]'], ['full-length LOV1 [gene]'], ['full-length LMADS9 [gene]']]}\n",
      "-------------------\n",
      "{'50605': [['fugu5-1 leaf [organ]'], ['ftsh4-5, lgo-2 ftsh4-5, and LGOoe ftsh4-5 mutants [mutant]'], ['Marker T33 [gene]'], ['Disease development caused by WT strain Foc [organism]'], ['Dof5.1-D [mutant]'], ['leaf domains of tcp mutants [organ]'], ['yellow first true leaf [phenotype]'], ['cs26 leaf [organ]'], ['ago1-12 leaf [tissue]'], ['mutant expression pattern of PHT1;4:GUS in fry1-7 root(s) stock [organ]'], ['stl-vs.-ftl [mutant]'], ['altered phenotype of drl1-101 leaf cells [phenotype]'], ['largest lamina compared to Col-0 and pif4pif5 [phenotype]'], ['reticulate leaf phenotype in cue1 single mutant and cue1-1/ppdk-1 double mutant [phenotype]'], ['fkd1 leaf vein pattern phenotype [phenotype]'], ['exaggerated leaf cell phenotype of hfr1/sics1 [phenotype]'], ['apical fate in tpl-1 [phenotype]'], ['ectopic GUS activity in leaf [phenotype]'], ['clf-50 ag leaf [mutant]'], ['responses opposite to the1-4 stomata [organ]'], ['second leaf of slasat1 mutants compared to WT leaf [organ]'], ['phyB apices and leaf [organ]'], ['virescent phenotype of svr11-1 [phenotype]'], ['leaf and apices of mutants [phenotype]'], ['kan1 kan2 leaf vasculature [phenotype]'], ['robustness of phyllotaxis in clasp-1 [mutant]'], ['shoots of cpr5 [mutant]'], ['corresponding areas of Xu142fl [mutant]'], ['dmbn leaf tissues [tissue]'], ['leaflet proliferation in the palm1 mutant [phenotype]'], ['distinct differences in floral tissues of tt3, tt4, tt5, tt6 and ttg [phenotype]'], ['young ffc leaf [tissue]'], ['leaf variegation of ftsh2 [phenotype]'], ['proliferative, stem(s) cell identity [phenotype]']]}\n",
      "-------------------\n",
      "{'26979': [['cold conditions [phenotype]'], ['Increasing frequency of extremely low temperature [phenotype]'], ['**extremely low temperatures [phenotype]**', 'extremely cold temperatures [phenotype]'], ['coldes [phenomenon]'], ['Chilling event [phenotype]'], ['Extreme cold climatic events [phenotype]'], ['Unpredictable cold snaps [phenotype]'], ['cold spells [phenotype]'], ['winter temperatures less than -30 C [phenotype]'], ['severe cold [phenotype]'], ['temperatures as low as -19 C [phenotype]'], ['very low winter temperatures [phenotype]'], ['temperatures as low as -9°C to -11°C [phenotype]'], ['no survival at -8°C [phenotype]'], ['extreme cold [phenotype]'], ['more extreme cold [phenotype]'], ['-2.8 C [phenotype]'], ['-1.7 C [phenotype]'], ['extreme winter temperatures [phenotype]'], ['minimum winter temperature [phenotype]'], ['(-)22 C [phenotype]'], ['half of the plant(s) dying at 4 C [phenotype]'], ['severe hypothermia [phenotype]'], ['poor survival in Finland [phenotype]'], ['-1.0°C [phenotype]'], ['severe cold temperatures [phenotype]'], ['-6 C [phenotype]'], ['cold snaps [phenomenon]'], ['subzero temperatures [phenotype]'], ['temperatures as low as -18 C [phenotype]'], ['cold waves [phenotype]'], ['frost days [phenotype]'], ['near freezing temperature [phenotype]'], ['severe sub-zero temperatures [phenotype]'], ['cold and freezing [phenotype]'], ['long below -15°C [phenotype]'], ['absolute minimum winter temperature [phenotype]'], ['sporadic cold peaks [phenotype]'], ['very cold and dry air [phenotype]'], ['very low temperatures [phenotype]'], ['traits incompatible with long-term growth in cooler climates [phenotype]'], ['cold snaps during leaf senescence [phenotype]']]}\n",
      "-------------------\n",
      "{'2212': [['increase in diploid gametes [phenotype]', '**diploid gamete production [phenotype]**']]}\n",
      "-------------------\n",
      "{'32062': [['primary plastids [organ]'], ['two daughter plastids [organelle]'], ['protodermal plastids [organelle]'], ['protoplasm [organ]'], ['mid-plastid [organelle]'], ['pre-existing organelles [organelle]'], ['prokaryotic origin [origin]'], ['proto-chloroplast [organelle]']]}\n",
      "-------------------\n",
      "{'100596': [['**woody flowering plant(s) [organism]**', 'flowering woody plant(s) [organism]']]}\n",
      "-------------------\n",
      "{'16570': [['**increased electron transport rate [phenotype]**', 'stimulation of photosynthetic linear electron flow [phenotype]', 'extra electron transport [phenotype]', 'improved electron transport [phenotype]', 'enhanced photosynthetic electron flow [phenotype]', 'stimulation of linear electron flow of photosynthesis [phenotype]', 'optimal linear electron transport [phenotype]', 'more efficient photosynthetic electron transport [phenotype]', 'higher photosynthetic electron transport [phenotype]', 'efficient photosynthetic electron transport [phenotype]', 'increased electron transfer activity [phenotype]', 'more electron transport [phenotype]', 'efficient electron transport [phenotype]', 'high rates of linear electron transport [phenotype]'], ['Alternative sinks for excess electrons [phenotype]'], ['High redox potentials [phenotype]'], ['higher acceptor-side limitation [phenotype]'], ['prominent role in ST [phenotype]'], ['energy in excess of that used by the photosynthetic electron-transport chain [phenotype]'], ['partially maintained photosynthetic electron transport activity [phenotype]'], ['rapid saturation of photosynthetic electron transport [phenotype]']]}\n",
      "-------------------\n",
      "{'151681': [['two homologs of Su(z)12 [gene]'], ['same branches as rice [gene]'], ['two gene(s) in same sub-group as rice gene(s) [gene]'], ['at least two RAD23-like gene(s) [gene]']]}\n",
      "-------------------\n",
      "{'50986': [['bdr1-1 mutant [mutant]'], ['bir1 mutant phenotype [mutant]'], ['bir1 phenotype [phenotype]'], ['br1 allele [phenotype]'], ['albino and dwarf phenotype of bdr1 [phenotype]'], ['phenotype of bdr1-1 [phenotype]'], ['phenotype of bdr1-2 [phenotype]'], ['b1 paramutation [phenotype]'], ['abs1-1D BR-deficient-related phenotypes [phenotype]'], ['suppression of all mutant phenotypes of bir1-1 [phenotype]'], ['phenotype of bzr1-D [phenotype]'], ['bor1-1 mutant phenotype [phenotype]']]}\n",
      "-------------------\n",
      "{'55214': [['rhomboid-like protein(s) in plant(s) [protein]'], ['RHO of plants:ROP GTPases [protein]'], ['tomato GRF protein(s) [protein]']]}\n",
      "-------------------\n",
      "{'2112': [['synchronous development between locules [phenotype]'], ['palea morphogenesis in spikelets [phenotype]']]}\n",
      "-------------------\n",
      "{'45179': [['CK2mut plant(s) [mutant]', '**CK2 mutants [mutant]**', 'CK2-defective plant(s) [mutant]']]}\n",
      "-------------------\n",
      "{'60494': [['reactions in trade-offs [phenotype]'], ['adaptive plasticity [phenotype]'], ['Cyto-Nuclear Coadaptation [phenotype]'], ['adaptive cline [phenotype]'], ['efficient adaption to environment [phenotype]'], ['trade-off between optimality and minimal adjustments [phenotype]'], ['trade-off between growth rate and adaptability [phenotype]'], ['fluxes in an absolute trade-off [phenotype]'], ['evolved to optimise sensing of stimuli [phenotype]'], ['adaptations to complex environmental conditions [phenotype]'], ['adaptation to various environmental conditions [phenotype]'], ['diverse physiological adaptations [phenotype]'], ['spaceflight physiological adaptation [phenotype]', '**adaptation to spaceflight [phenotype]**'], ['flexibility of the organism [phenotype]', '**physiological flexibility [phenotype]**'], ['adaptation to coldes [phenotype]'], ['database of p-adaptations and e-adaptations [phenotype]'], ['some process of adaptation in evolution [phenotype]'], ['plasticity in SA:growth tradeoffs [phenotype]'], ['dynamic homeostasis [phenotype]'], ['a very large metabolic adjustment [phenotype]'], ['adaptations to external cues [phenotype]'], ['natural variation in adaptive traits [phenotype]'], ['**insights into adaptation mechanisms [phenotype]**', 'insight into processes of adaptation [phenotype]'], ['physiological adjustments in distinct organs [phenotype]'], ['adaptation of both partners [phenotype]'], ['unique adaptive environmental regulation mechanisms [phenotype]'], ['adaptation influences on life-history trait co-variation [phenotype]'], ['better adaptation to cold [phenotype]'], ['long-term adaptation to the environment [phenotype]'], ['changes in physiological traits [phenotype]'], ['broader adaptive trait [phenotype]'], ['adaptation and survival in phenotype-based screening strategy [phenotype]']]}\n",
      "-------------------\n",
      "{'11832': [['Broad scattering feature [phenotype]'], ['evenly spaced substitution [phenotype]'], ['Clustered positioning of alpha-(1 6) branch linkages [phenotype]'], ['broad peak [phenotype]'], ['(200) or (110) planes [phenotype]'], ['branching level of 4-5% [phenotype]'], ['non-random branching frequency and pattern [phenotype]'], ['**one or more branches [phenotype]**', 'some branches [phenotype]'], ['branches separated by short distances [phenotype]'], ['tail-to-tail fashion [phenotype]'], ['one branch point [phenotype]'], ['two or more branch points [phenotype]'], ['trimming ISA activity [phenotype]'], ['many short branches [phenotype]'], ['distinct CLD [phenotype]'], ['broader peak at DP14 [phenotype]'], ['branching points [phenotype]'], ['evenly spaced pattern [phenotype]'], ['wild-type levels of cellulose and normal birefringence [phenotype]'], ['degree of order [phenotype]']]}\n",
      "-------------------\n",
      "{'130161': [['**A. thaliana type I AtMC1 [gene]**', 'A. thaliana type I metacaspase, AtMC1 [gene]'], ['A. thaliana type I AtMC2 [gene]'], ['A. thaliana metacaspases [gene]'], ['A. thaliana full-length AtMC1 [gene]'], ['A. thaliana AtMC4 [gene]'], ['A. thaliana type II metacaspase, AtMC4 [gene]'], ['A. thaliana type I metacaspase, AtMC3 [gene]'], ['A. thaliana type II metacaspase, AtMC9 [gene]'], ['A. thaliana AtCHC1 [gene]'], ['A. thaliana AtCHC2 [gene]'], ['A. thaliana ACBP3 [gene]'], ['A. thaliana AtACBP6 [gene]'], ['A. thaliana AtACBP2 [gene]'], ['A. thaliana AtACBP3 [gene]'], ['A. thaliana AtACBP4 [gene]'], ['A. thaliana AtACBP5 [gene]']]}\n",
      "-------------------\n",
      "{'19069': [['**Toxicity [phenotype]**', 'toxicity in plant(s) [phenotype]', 'its toxicity [phenotype]'], ['cesium treatment [phenotype]'], ['above higher concentration [metabolite]'], ['toxicity thresholds for iodide [phenotype]'], ['high toxin A expression [phenotype]'], ['three toxicity indicators [phenotype]'], ['Cs phytotoxicity [phenotype]'], ['dose-dependent accumulation patterns [phenotype]'], ['Herbicide detoxification [phenotype]'], ['Higher plant(s) genotoxicity assays [phenotype]'], ['contamination of soil [phenotype]'], ['toxic substance efflux [phenotype]'], ['highly toxic to plant(s) [phenotype]', '**highly toxic [phenotype]**'], ['significant damage [phenotype]'], ['severe toxicity to plant(s) [phenotype]'], ['P-toxicity phenotypes [phenotype]'], ['top toxic substances [phenotype]'], ['maximum toxic effect [phenotype]'], ['genotoxic in plant(s) systems [phenotype]'], ['higher toxicity in plant(s) [phenotype]'], ['high-dosage-mediated toxic effect [phenotype]'], ['severe phytotoxicity [phenotype]'], ['dangerous levels of industrial contamination [phenotype]'], ['the second most phytotoxic element [phenotype]'], ['the most phytotoxic element [phenotype]'], ['ZEN toxicity [phenotype]'], ['noticeable phytotoxicity [phenotype]'], ['development of atrazine injury [phenotype]'], ['strong phytotoxicity [phenotype]'], ['toxic air pollutant [phenotype]'], ['strong toxicity [phenotype]'], ['toxic levels [phenotype]'], ['toxic levels in vegetables, grains, and fruits [phenotype]'], ['risk of overdosage [phenotype]'], ['potential toxicity to plant(s) [phenotype]'], ['residual toxicity [phenotype]'], ['extreme difficulty of phytotransformation by plant(s) [phenotype]'], ['Radionuclide contamination [phenotype]'], ['moderate phytotoxic effect [phenotype]'], ['chemical residues [phenotype]'], ['perennial poisonous plant(s) [phenotype]'], ['Eupatorium adenophorum toxicity [phenotype]'], ['level of air pollution [phenotype]'], ['toxicity for A. thaliana growth [phenotype]'], ['toxicity issues [phenotype]'], ['high toxicity to almost all plant(s) [phenotype]'], ['high residual toxicity [phenotype]'], ['phytotoxic at high concentrations [phenotype]'], ['knowledge on toxicity [phenotype]'], ['strong phytotoxic potential [phenotype]'], ['moderately contaminated soil [phenotype]'], ['plant(s) at high concentrations [phenotype]'], ['significant effect in MN assays with Tradescantia [phenotype]'], ['much higher photosynthetic toxicity [phenotype]'], ['major phytotoxic pollutant [phenotype]'], ['toxicity of organic contaminants [phenotype]'], ['genotoxicity of herbicide-contaminated water [phenotype]'], ['highly phytotoxic [phenotype]'], ['toxic symptom [phenotype]'], ['exposure conditions [phenotype]'], ['distinct bleaching symptoms [phenotype]'], ['toxic at high tissue concentrations [phenotype]'], ['excellent indicators of cytotoxic, cytogenetic, and mutagenic effects [phenotype]'], ['increased levels of pollutants [phenotype]']]}\n",
      "-------------------\n",
      "{'70036': [['Arg416 [amino acid]'], ['Arg443 [amino acid]'], ['Arg156 [amino acid]'], ['Lys/Arg537 [amino acid]'], ['Lys/Arg548 [amino acid]'], ['Arg-259 [amino acid]'], ['**Arg-267 [amino acid]**', 'Arg 267 [amino acid]'], ['Arg403 [amino acid]'], ['R345 [amino acid]'], ['R402 [amino acid]'], ['Arg1008 [amino acid]'], ['Arg1032 [amino acid]'], ['Arg453 [amino acid]'], ['AtHCT Arg356 [amino acid]'], ['Arg455 [amino acid]'], ['Arg461 [amino acid]'], ['Arg507 [amino acid]'], ['Arg683 [amino acid]'], ['Arg551 [amino acid]'], ['Arg547 [amino acid]'], ['**Arg436 [amino acid]**', 'Arg 436 [amino acid]'], ['Arg295 [amino acid]'], ['Arg366 [amino acid]'], ['Arg501 [amino acid]'], ['Arg-415 [amino acid]'], ['Arg377 [amino acid]'], ['Arg-496 to Leu-647 [amino acid sequence]'], ['Arg285 [protein]']]}\n",
      "-------------------\n",
      "{'56183': [['ItbNAC110 [gene]'], ['ItbNAC114 [gene]'], ['ItfNAC15 [gene]'], ['ItfNAC28 [gene]'], ['ItfNAC62 [gene]'], ['ItfNAC44 [gene]'], ['ItbNAC32 [gene]'], ['ItbNAC51 [gene]'], ['ItbNAC106 [gene]'], ['ItbNAC107 [gene]'], ['ItbNAC108 [gene]'], ['ItfNAC [gene]'], ['ItfNAC target gene(s) [gene]'], ['ItbNAC109 [gene]']]}\n",
      "-------------------\n",
      "{'75941': [['EXS domain [protein domain]'], ['PIP cytosolic domains [protein]'], ['insertion domain of A. thaliana PID kinase [protein complex]'], ['Domain D-II [structure]'], ['BAR domain of SH3P2 [gene]'], ['two PEST domains [protein domain]'], ['SH3 DOMAIN-CONTAINING PROTEIN2 (SH3P2) [protein]'], ['predicted PTS1 domains [protein]'], ['oxysterol-binding domain of PiORP1 [protein]'], ['canonical PTS1 [protein domain]'], ['exostosin-specific domains [protein]'], ['PTS1 domain [protein]'], ['functional PTS1 domain [protein]'], ['cytosolically exposed RING domain of PEX2 [protein domain]'], ['class IV homeodomain HD-ZIP family START domain containing protein(s) [protein]'], ['Sir2 domain [phenotype]'], ['a protein(s) with an atypical PIPT domain [protein domain]'], ['SOK1 polarity domain [protein domain]'], ['inhibitor of ectopic domain formation [phenotype]'], ['protein(s) with PCI/PINT-associated module domain [protein]'], ['PTS1-binding region [protein domain]']]}\n",
      "-------------------\n",
      "{'102933': [['Cultivar Remus [organism]', '**Wheat cultivar Remus [organism]**'], ['Trojan [organism]']]}\n",
      "-------------------\n",
      "{'160016': [['functional differences in Heading date 1 (Hd1) [gene]'], ['Variations of CO and Hd1 [gene]']]}\n",
      "-------------------\n",
      "{'30687': [['the oscillatory network of the circadian clock [process]'], ['central oscillator network [organ]']]}\n",
      "-------------------\n",
      "{'31772': [['**ligand-mediated receptor internalization [process]**', 'ligand-induced endocytosis [process]', 'ligand-induced receptor endocytosis [process]', 'Internalization of receptors upon ligand-binding [process]', 'receptor-mediated entry [process]', 'ligand-induced CME [process]'], ['GPCR internalization [process]'], ['Following RK internalization [process]'], ['internalization of other receptor-like kinases [process]'], ['endocytosis and signaling of LeEix2 [process]'], ['extracellular ligand binding [process]'], ['endocytosis independently of exogenous application of ligand [process]'], ['ligand entry [phenotype]']]}\n",
      "-------------------\n",
      "{'19386': [['biomass accumulation [phenotype]'], ['plasticity in above-ground biomass [phenotype]'], ['total carbon (Ctot) [phenotype]'], ['lowest biomass accumulation rate [phenotype]'], ['biomass accumulator [phenotype]'], ['aboveground biomass accumulation [phenotype]'], [\"plant's capacity to accumulate biomass [phenotype]\", '**biomass production capacity [phenotype]**'], ['above ground biomass under non-stressed conditions [phenotype]'], ['above-ground biomass yield [phenotype]'], ['efficient biomass production [phenotype]'], ['aboveground biomass in grassland [phenotype]'], ['accumulated biomass [phenotype]'], ['net carbon gain [phenotype]'], ['natural productivity [phenotype]'], ['total above ground biomass [phenotype]'], ['assimilate productivity [phenotype]']]}\n",
      "-------------------\n",
      "{'172583': [['**Methionine chain elongation pathway [metabolic pathway]**', 'methionine chain-elongation pathway [metabolic pathway]'], ['Methionine chain elongation pathway [process]'], ['Met chain elongation cycle [pathway]'], ['Met chain elongation pathway [pathway]'], ['Met chain elongation pathway [process]'], ['**methionine chain elongation [metabolic pathway]**', 'Met chain elongation [metabolic pathway]', 'Methionine chain-elongation [metabolic pathway]'], ['methionine chain-elongation pathway [biochemical pathway]'], ['Cycles of methionine chain elongation [process]'], ['Close relationship between the two pathways [process]'], ['methionine chain-elongation pathway [process]'], ['**methionine chain-elongation pathway [pathway]**', 'methionine-chain elongation pathway [pathway]'], ['Met chain-elongation pathway [process]'], ['methionine chain elongation cycle [metabolic pathway]'], ['Met elongation [metabolic pathway]'], ['first two cycles of methionine chain elongation [metabolic pathway]'], ['first two rounds of methionine chain elongation [metabolic pathway]'], ['substrates and/or products of methionine chain elongation [metabolic pathway]'], ['Met chain elongation [pathway]'], ['Met chain elongation pathways [process]'], ['chain elongation pathway of Met [process]'], ['**Met chain elongation pathway [metabolic pathway]**', 'Met chain-elongation pathway [metabolic pathway]'], ['Methionine chain elongation pathway [metabolite]'], ['elongation cycle [metabolic pathway]'], ['recruitment of the methionine chain-elongation pathway [biochemical pathway]'], ['divergence of the two pathways [biochemical pathway]'], ['Both pathways [pathway]'], ['recruitment of the methionine chain-elongation pathway [pathway]'], ['evolutionary and functional relationship between Leu biosynthesis and Met chain elongation pathway [process]'], ['initial step of Met chain elongation pathway [metabolic pathway]'], ['**early cycles of Met chain elongation [metabolic pathway]**', 'early cycles of Met chain elongation pathway [metabolic pathway]'], ['first cycle of Met chain elongation [metabolic pathway]'], ['first round of Met chain elongation cycle [metabolic pathway]'], ['first cycle of Met chain elongation occurs in other tissues [metabolic pathway]'], ['chain elongation pathway of methionine-derived glucosinolates [metabolite]'], ['initial step of Met chain elongation [process]']]}\n",
      "-------------------\n",
      "{'162392': [['**ROS producers [metabolite]**', 'ROS-producing agent [metabolite]', 'important producer of ROS [metabolite]'], ['non-enzymatic sources [phenotype]'], ['**enzymes related to ROS synthesis and catabolism [protein]**', 'ROS-metabolism-related enzymes [protein]'], ['ROS production by oxidases [protein]'], ['ROS-producing reagents [treatment]'], ['**ROS-producing enzymes [protein]**', 'ROS-generating enzymes [protein]', 'ROS producing enzymes [protein]'], ['**enzymatic ROS production [metabolite]**', 'direct enzymatic conversion of molecular oxygen (O2) to reactive oxygen species (ROS) [metabolite]'], ['protein(s) that generate ROS [metabolite]'], ['ROS-related enzymes [enzyme]'], ['ROS-generating machinery [metabolite]'], ['**ROS produced by NADPH-oxidases [enzyme]**', 'ROS production enzyme NADPH oxidase [enzyme]'], ['ROS-producing systems [protein]']]}\n",
      "-------------------\n",
      "{'120406': [['Identified mutations [phenotype]'], ['tumour-specific aberrations [phenotype]'], ['**germline mutations [phenotype]**', 'germline-transmitted mutations [phenotype]', 'mutations in the germline [phenotype]', 'germline transmissible [phenotype]'], ['mutational signatures [mutation]'], ['inherited disease mutation databases [database]'], ['More than half of human cancers [phenotype]', '**oncological diseases [phenotype]**', 'many human cancers [phenotype]'], ['**mutation types [phenotype]**', 'base substitutions, deletions, insertions, and translocations [phenotype]', 'various types of mutation [phenotype]', 'several types of mutation [phenotype]', 'other types of mutation [phenotype]', 'certain mutation types [phenotype]', 'specific mutational classes [phenotype]'], ['unique molecular signature [phenotype]'], ['segregating mutations [phenotype]'], ['alternative classifications [phenotype]'], ['Well-characterized mutations [phenotype]'], ['somatic and germline mutations [phenotype]'], ['unmasking of deleterious mutations [phenotype]'], ['independent mutation data [phenotype]'], ['systematic asymmetries in heritable mutations [genetic feature]'], ['difficulties to differentiate mutations [phenotype]', '**difficulty in detecting mutations [phenotype]**'], ['lung cancer patients [phenotype]'], ['list of likely candidate mutations [phenotype]'], ['mutational status of parental tumor cells [phenotype]'], ['fixed mutations in ancestral line [phenotype]'], ['targeted heritable mutations [phenotype]'], ['screening of these exons [phenotype]'], ['sporadic breast cancers [phenotype]'], ['inherited breast cancer cases [phenotype]', '**hereditary breast cancers [phenotype]**'], ['breast/ovarian cancers [phenotype]'], ['non-small-cell lung carcinoma [phenotype]']]}\n",
      "-------------------\n",
      "{'103786': [['S-PS population [organism]', '**S-PS [organism]**'], ['**NS-JB [organism]**', 'NS-JB population [organism]'], ['NS-PP and S-RL populations [organism]'], ['**S-PS and S-SD populations [organism]**', 'S-SD and S-PS [organism]', 'S-SD and S-PS populations [organism]'], ['S-SD population [organism]', '**S-SD [organism]**'], ['NS-PP and S-SD [organism]'], ['NS-JB and NS-PP [organism]'], ['NS-JB, S-RL, and S-SD [organism]'], ['NS-PP population [organism]'], ['serpentine populations S-SD and S-PS [organism]'], ['S-RL population [organism]'], ['NS-JB, NS-PP, and S-PS populations [organism]'], ['Plech population [organism]'], ['sample from Czech Republic [organism]'], ['small populations in eastern cluster [organism]'], ['Rs-C population [organism]'], ['parental taxa A. lyrata complex [organism]'], ['commonality of A. lyrata [organism]'], ['SC populations of L. alabamica [organism]'], ['reports in A. lyrata [organism]'], ['comparing Norwegian populations to GER [organism]'], ['two European populations of A. lyrata ssp. petraea [organism]'], ['Plech population [population]'], ['Southern Germany [location]'], ['serpentine population from Sankt Marien-Feistritz [location]']]}\n",
      "-------------------\n",
      "{'61853': [['SCOOP10B [peptide]'], ['At3g59930 peptide [protein]'], ['hBD-2 [peptide]']]}\n",
      "-------------------\n",
      "{'97819': [['Purple sweet potato [organism]', '**Ipomoea batatas [organism]**', 'Ipomoea batatas (sweet potato) [organism]', 'Ipomoea batatas (L.) Lam [organism]'], ['Purple Peruvian potato [organism]'], ['Impoea batatas [organism]']]}\n",
      "-------------------\n",
      "{'62892': [['first human MIF receptor [protein]'], ['Affinity for LPS-binding sites [phenotype]'], ['QFC algorithm [method]'], ['GBP recruitment [phenotype]'], ['Recognition of PGN [phenotype]'], ['UFM1-binding [phenotype]'], ['CCRC-M35 epitope binding [phenotype]'], ['LM20 antibody binding [gene]'], ['**JIM20 labelling [phenotype]**', 'JIM20 signal [phenotype]'], ['MAB4/MEL recruitment [phenotype]'], ['ccmFN2 translation [phenotype]'], ['specificity of FaeG variants [protein]'], ['Type III AFP [protein]'], ['presence of binding protein(s) [protein]'], ['interaction of Pol lambda with PCNA2 [phenotype]'], ['DRP1A immunofluorescence at cell plate [phenotype]'], ['affinity resin [phenotype]'], ['potential to rescue secretion of misfolded but active glycoproteins [phenotype]'], ['higher RFFIT activity [phenotype]'], ['LPS binding site detection [phenotype]'], [\"BTS' capacity to interact with PYEL protein(s) [phenotype]\"], [\"GRFT's ability to bind oligomannose glycosylated protein(s) [phenotype]\"], [\"GRFT's viral neutralization activity [phenotype]\"], ['in vivo functionality of GnTI [protein]'], ['biopharmaceutics via binding with mannose receptor (MR) expressed on the cell surface [phenotype]'], ['efficacy of MR-mediated delivery [phenotype]'], ['block Ebola virus trafficking and fusion [phenotype]'], ['GFPP accommodation [phenotype]'], ['affinity of TCTPs [protein]'], ['role in CLE sensing [phenotype]'], ['ceramide sensor [phenotype]'], ['HIV-Tat interaction [protein]'], ['baseline GAI, RGL1 and RGL3 affinity [phenotype]'], ['enhanced GAI affinity [phenotype]'], ['binding to chitin beads [phenotype]'], ['folding of elf18 responsive EF-Tu receptor (EFR) [protein]'], ['RPP7 coding potential [phenotype]'], ['affinity for GT-1 [protein]'], ['P. falciparum Rab (PfRab) nomenclature [phenotype]'], ['higher fluorescence intensity with JIM5 antibody [phenotype]'], ['binding potential with recombinant RACK1A protein(s) [protein]'], ['JIM7 immune intensity [phenotype]'], ['in vivo specificity of RPF2 [phenotype]'], ['FaeG [molecular surface]'], ['AprA-mediated cleavage [phenotype]'], ['recognition of flagellin by TLR5 [phenotype]'], ['functional AtEFR receptor [gene]'], ['UDP-glucose recognition [phenotype]'], ['early detection of epitope-tagged protein(s) in Agrobacterium-infected plant(s) cells [phenotype]'], ['normal flagellin binding [phenotype]'], ['cell-binding affinity for fungal cells [phenotype]'], ['eIF4E recruitment [phenotype]'], ['EFR function [phenotype]'], ['plasma membrane RGD-binding activity [phenotype]'], ['f-actin binding capacity [function]']]}\n",
      "-------------------\n",
      "{'24749': [['Increased expression of KRP6 or KRP1 [phenotype]'], ['CVP2 hyperactivity [phenotype]'], ['KNOX accumulation [phenotype]']]}\n",
      "-------------------\n",
      "{'45547': [['**lfy and puchi mutants [mutant]**', 'puchi and lfy mutants [mutant]']]}\n",
      "-------------------\n",
      "{'10580': [['compromising autophagy [phenotype]'], ['Lipophagy [phenotype]'], ['**Autophagic breakdown [phenotype]**', 'degradation of autophagic bodies [phenotype]'], ['nutrient stress-induced autophagy [phenotype]'], ['degradation by selective autophagy [phenotype]'], ['carbon starvation-induced bulk and selective autophagic degradation [phenotype]']]}\n",
      "-------------------\n",
      "{'161408': [['PHT1;4-mediated phosphate uptake [process]'], ['**PHOSTIN (PSN) [metabolite]**', 'specific induction to Pi deficiency [metabolite]', 'rapid and transient formation of PHS-P [metabolite]', 'discovery of the first chemical [metabolite]'], ['PSN11 [chemical]'], ['autophagy induction by phosphate (Pi) limitation [treatment]'], ['Pi-regulated small peptide synthesis [phenotype]'], ['Phosphate-dependent control of SL synthesis [phenotype]'], ['induced and secreted under Pi starvation [phenotype]'], ['PSR hormone [phenotype]']]}\n",
      "-------------------\n",
      "{'75734': [['Carboxy-terminal alpha-helix [subcellular compartment]', '**C-terminal alpha-helices [subcellular compartment]**'], ['α-helical linkers [subcellular compartment]'], ['C-terminal alpha helical sub-domain [subcellular compartment]']]}\n",
      "-------------------\n",
      "{'32689': [['Cell-wall-processing protein(s) [protein]'], ['**Cell wall remodeling protein(s) [protein]**', 'wall modeling protein(s) [protein]', 'cell wall modifiers [protein]'], ['biosynthesis of wall protein(s) [protein]'], ['cell wall protein(s) pectin-methylesterase [protein]'], ['production of cell wall modification protein(s) [protein]'], ['cell wall synthesis protein(s) [protein]'], ['wall-bound XTHs [protein]'], ['cell wall activities [protein complex]'], ['Engineering cell wall protein(s) partners [process]'], ['protein(s) cross-linking in the cell wall [process]']]}\n",
      "-------------------\n",
      "{'99908': [['China 1039 [organism]'], ['Wuyunjing 8 [organism]'], ['Taipei-309 [organism]', '**TP309 [organism]**', 'Taipei 309 [organism]', 'Taipei 309 (TP309) [organism]'], [\"Pei'ai64 [organism]\"], ['Nanjing11 [organism]'], ['Taipei 309 control plant(s) [organism]'], ['Jia64 [organism]'], ['**Lemont [organism]**', \"'Lemont' [organism]\"], ['Deng Pao Zhai [organism]'], ['Xiaowei [organism]'], ['Hainan [organism]'], ['Jia33 [organism]'], ['PB1 rice cultivar [organism]'], ['Chunyou84 (CY84) [organism]'], ['Zhonghua 11 calli [organism]'], [\"Pei'ai 64S rice seedlings [organism]\"], ['Zhonghui 8015 [organism]'], ['Pusa-169 [organism]']]}\n",
      "-------------------\n",
      "{'32498': [['Mechanical stresses [process]'], ['structural support of tissues [tissue]'], ['strength of tissues [tissue]'], ['mechanical support [process]'], ['mechanical protection for inner living cells [tissue]'], ['gall tissues [tissue]'], ['**tissue integrity [tissue]**', 'structural integrity to plant(s) tissues [tissue]'], ['**maintenance of tissues [tissue]**', 'Maintenance [tissue]'], ['tissue maintenance [process]'], ['**structural support [function]**', 'structural and physical support [function]'], ['structural and functional integrity of the SAM [organ]'], ['mechanical protection [process]'], ['mechanical conflicts in tissues [tissue]'], ['physical continuity [process]']]}\n",
      "-------------------\n",
      "{'20785': [['SNPs or indels [phenotype]'], ['Bn [phenotype]'], ['bW [genotype]'], ['S24 haplotype [phenotype]'], ['Alleles of heading accessions [phenotype]'], ['black seed coat genotypes [phenotype]'], ['major changes between sensitive and resistant varieties of Brassica rapa [phenotype]'], ['**Weedy type [phenotype]**', 'Weedy type plant(s) [phenotype]', 'weedy type of plant(s) [phenotype]'], ['complexity of Brassica genome [phenotype]'], ['Hap-TAA [phenotype]'], ['swedes [phenotype]'], ['Winter lines [phenotype]'], ['Br class II [phenotype]'], ['Genetic diversity of G blocks [phenotype]'], ['Low genetic diversity within B. napus [phenotype]'], ['winter type accessions [genotype]'], ['semi-winter types [genotype]'], ['AON [phenotype]'], ['B481-6 line [phenotype]'], ['Hap_ATT [genotype]'], ['bolting time of C634 [phenotype]'], ['RBr genetic repertoire [phenotype]'], ['chartreuse individuals [phenotype]'], ['crucifer evolution [phenotype]'], ['RC-Brassica experiments [phenotype]'], ['complex traits in B. napus [organism]'], ['remarkable results [phenotype]'], ['phenotype of B. napus [organism]'], ['a more dynamic structure compared with other cruciferous mitotypes [phenotype]'], ['208 kb [phenotype]'], ['Winter accessions [phenotype]'], ['plant(s) height of 121 to 167 cm [phenotype]'], ['different S-haplotype backgrounds [phenotype]'], ['seeds of HOCL and LOCL [phenotype]'], ['diverse functions in pepper [phenotype]'], ['APEX01-2B [phenotype]'], ['APEX01-1B [phenotype]'], ['good taste [phenotype]'], ['complex glucosinolate traits [phenotype]'], ['network conservation in Brassicaceae [phenotype]'], ['leaf-heading accessions [phenotype]'], ['heading and nonheading accessions [phenotype]'], ['qualitative phenotypic variation in response to A. brassicicola [organism]'], ['R18500 [genotype]'], ['phenotypes of 228 Brassica accession seedlings under drought [phenotype]'], ['three genotypes [phenotype]'], ['ComA [phenotype]'], ['three vegetables grown under shade condition [phenotype]'], ['clustered according to their lineages within Brassicaceae [phenotype]'], ['a new model system for studies of SI in the Brassicaceae [phenotype]'], ['yellow color in B. rapa [phenotype]'], ['cool-season legume model [phenotype]'], ['weedy type of soybean [phenotype]'], ['Westar [phenotype]'], ['Eps-3Am in T. monococcum [phenotype]'], ['insights into Brassicaceae genomes [phenotype]'], ['Yangtze River basin [phenotype]'], ['annual oil morphotype [phenotype]'], ['semi-winter type [phenotype]'], ['requirements of functional genomics research [phenotype]'], ['last B. rapa common ancestor [phenotype]'], ['high-HI B. napus [phenotype]'], ['high- and low-HI B. napus lines [phenotype]'], ['B. rapa growth [phenotype]'], ['C03 [phenotype]'], ['NH-Br and H-Br groups [phenotype]'], ['H-Br group [phenotype]'], ['S20b [phenotype]'], ['three rapeseed ecotype groups [phenotype]'], ['found in B. rapa [phenotype]'], ['high-HSW accessions [phenotype]'], ['four lineages [phenotype]'], ['identifying tolerant lines of Chinese cabbage [phenotype]'], ['natural hybridization between B. rapa and B. oleracea [phenotype]'], ['in the last 10,000 years [phenotype]'], ['PA deposition profile in rapeseed [phenotype]'], ['hybridization breadth of interspecific/intergeneric crosses in Brassicaceae [phenotype]'], ['216 species [phenotype]']]}\n",
      "-------------------\n",
      "{'89746': [['Wax accumulation [process]', '**production of cuticular waxes [process]**', 'cuticle wax development [process]', 'surface wax production [process]', 'epicuticular wax biosynthesis [process]', 'regulating cuticular wax deposition [process]', 'cuticular wax production in response to changes in ambient humidity [process]', 'synthesis of a wax cuticle [process]'], ['Cutinase disruption [process]'], ['wounding-induced production [process]'], ['cuticle integrity [process]'], ['cutin analysis [process]'], ['defense compound synthesis [process]'], ['study of cuticle synthesis [process]']]}\n",
      "-------------------\n",
      "{'75854': [['NERD1 beta-propeller [protein structure]'], ['**beta-propeller [protein domain]**', 'Beta-propeller structure [protein domain]', 'beta-propeller group [protein domain]'], ['N-terminal beta propellers [protein]'], ['beta-propellers [protein structure]'], ['**seven-bladed beta-propeller structure [protein structure]**', 'seven-bladed beta-propeller configuration [protein]', 'seven-bladed beta-propeller core [protein structure]'], ['Beta Propeller A-C (BPA-BPC) [protein structure]'], ['beta-propeller 4 of GTS1 [protein region]'], ['six kelch repeats predicted to fold into a beta propeller [protein]']]}\n",
      "-------------------\n",
      "{'71512': [['Dynactin [protein]'], ['**intermediate chain of dynein [protein]**', 'intermediate chains of cytoplasmic dynein [protein]'], ['components of the cytoplasmic dynein-dynactin complex [protein complex]', '**dynein/dynactin motor [protein complex]**'], ['the dynactin complex [protein complex]'], ['assembly of an intricate molecular complex involving cytoplasmic dynein [protein complex]']]}\n",
      "-------------------\n",
      "{'99067': [['C. rumelica [organism]'], ['C. laxa [organism]'], ['C. hispida [organism]'], ['**Oryza sativa L. ssp. japonica [genome]**', 'Oryza sativa L. ssp. Japonica genome [genome]'], ['genome of two subspecies [organism]'], ['progenitor of subgenome 1 of C. sativa [organism]'], ['relationship with subgenome 1 of C. sativa [organism]'], ['C. arabica subgenomes [organism]'], ['C. eugenioides subgenome [organism]'], ['C. canephora subgenome [organism]'], ['E. indica (AA genome) [organism]'], ['C. grandiflora-derived subgenome [genome]'], ['**Oryza sativa L. ssp. Indica genome [genome]**', 'Oryza sativa L. ssp. indica [genome]'], ['three homoeologous subgenomes [organism]']]}\n",
      "-------------------\n",
      "{'113143': [['**native promoter-driven constructs [treatment]**', 'native promoter of each gene(s) [treatment]'], ['artificial inducible promoter [gene]'], ['complemented lines with native promoter [mutant]'], ['native and synthetic promoter constructs [gene]'], ['large tags and expression by endogenous promoters [gene]'], ['artificial promoter [gene]'], ['endogenous promoter [gene expression]'], ['native promoters [technology]']]}\n",
      "-------------------\n",
      "{'150525': [['GhSNF-YB22 [gene]'], ['GhANF-YB21 [gene]']]}\n",
      "-------------------\n",
      "{'19977': [['**Enhanced resistance [phenotype]**', 'higher level of resistance [phenotype]', 'stronger resistance [phenotype]', 'stronger resistance phenotype [phenotype]', 'more resistance [phenotype]', 'further increased resistance [phenotype]', 'higher plant(s) resistance [phenotype]', 'elevated level of resistance [phenotype]', 'increased resistance phenotype [phenotype]', 'increased resistance capability [phenotype]', 'more resistant plant(s) [phenotype]', 'elevated resistance [phenotype]', 'enhanced level of resistance [phenotype]'], ['long-lasting enhanced resistance [phenotype]'], ['increased systemic resistance [phenotype]'], ['stronger basal resistance [phenotype]'], ['elevated resistance during subsequent infections [phenotype]']]}\n",
      "-------------------\n",
      "{'163081': [['antimicrobial secondary metabolite accumulation [metabolite]'], ['accumulation of antimicrobial secondary metabolites [molecule]']]}\n",
      "-------------------\n",
      "{'10847': [['mitochondrial density [phenotype]'], ['**mitochondrial iron levels [phenotype]**', 'iron in mitochondria [phenotype]'], ['status of mitochondrial Fe-S cluster assembly [phenotype]'], ['oxygen carrier [phenotype]']]}\n",
      "-------------------\n",
      "{'86850': [['We have deconstructed the traditional ISH protocol [treatment]'], ['RapiGest [treatment]'], ['optimization of In-Fusion [treatment]'], ['PEA-Clarity [treatment]'], ['Rosenfeld et al. [author]'], ['Sequential processing with PNGase F [treatment]'], ['Sequential processing with legumain [enzyme]'], ['relatively high efficiency of this procedure [process]'], ['Manual removal of zona pellucida [treatment]'], ['bacterial PI-PLC treatment [process]'], ['high-throughput purified granule methods [treatment]'], ['solution-based targeted genomic enrichment methodology [treatment]'], ['tandem StrepII tags [treatment]'], ['distinct tailoring rationale [treatment]'], ['Newly developed procedure [treatment]'], ['Peptidomic analysis [treatment]'], ['Integrated approach to proteomics and peptidomics [treatment]'], ['N-glycan processing [treatment]'], ['mPEG-MAL treatment [treatment]'], ['EPG digestion [treatment]'], ['SFI [treatment]'], ['tagmented products [process]'], ['enhanced stringency [treatment]'], ['and may become the preferred method for plant(s) proteomics in the future [treatment]'], ['E3-DART system [treatment]'], ['solid-phase enzymatic reactions [treatment]'], ['multi-enzyme digestion approach [treatment]']]}\n",
      "-------------------\n",
      "{'23162': [['signal transduction cross-talk [phenotype]'], ['cross-communication [phenotype]'], ['combinatorial cross-talk [phenotype]'], ['**Cross talk [phenotype]**', 'cross-talks [phenotype]'], ['molecular cross-talk [phenotype]'], ['interactions between signaling pathways [phenotype]', 'cross talk between multiple signaling pathways [phenotype]', '**cross-talk between signalling pathways [phenotype]**', 'cross-talk of different signaling pathways [phenotype]'], ['cross-talk in plant(s) [phenotype]'], ['bidirectional cross-talk [phenotype]'], ['cross talk between temperature and H2O2 signaling [phenotype]'], ['cross-talk with other signaling molecules [phenotype]'], ['complex cross-talk [phenotype]'], ['cross-talk and signal integration [phenotype]'], ['cross talk between defense signaling pathways [phenotype]'], ['node for cross talk [phenotype]', '**cross-talk node [phenotype]**'], ['considerable cross talk [phenotype]'], ['metabolic cross-talk [phenotype]']]}\n",
      "-------------------\n",
      "{'17727': [['wavelength dependent photosystem excitation energy distribution [phenotype]', '**photosystem excitation-energy distribution [phenotype]**', 'excitation energy distribution [phenotype]'], ['excitation pressure of PSII [phenotype]', '**photosystem II excitation pressure [phenotype]**'], ['excitation trapping time [phenotype]'], ['Increase in free-energy gap [phenotype]'], ['560 nm light [phenotype]'], ['590 nm light [phenotype]'], ['relative excitation level of photosystems [phenotype]'], ['pH-dependent de-excitation of singlet excited chlorophyll molecules [phenotype]']]}\n",
      "-------------------\n",
      "{'169878': [['plant(s) DHS activities [enzyme]', '**overall activity of DHS [enzyme]**'], ['Expression levels of AthDHS2 [enzyme]'], ['plant(s) DHS activity [enzyme activity]', '**total DHS activity [enzyme activity]**'], ['XDH protein(s) levels [enzyme]'], ['carrot DHS activity [enzyme]'], ['maize shoots DHS activity [enzyme]'], ['overall DHS activity [process]']]}\n",
      "-------------------\n",
      "{'76439': [['Efh [gene feature]'], ['N-glycosylation within FAS1 domain of FLA11 [protein domain]'], ['PHD domain [phenotype]'], ['GST-fused SE Middle (SE-Mid) domain [protein]'], ['SGF11 and ENY2 amino or carboxy-terminal extra-domains [protein]'], ['GH1 domains [protein domain]'], ['GH1 domain of g34113.t1 [protein domain]'], [\"Domain IV and the G' subdomain of domain I [protein complex]\"], [\"SPY's self-fucosylation region [protein region]\"], ['presence of FH2 domain [protein domain]'], ['EF1G [domain]'], ['HA-F+Kelch peptide [gene]'], ['**GYF-domain-containing protein(s) [protein]**', 'GYF domain protein(s) [protein]', 'GYF domain-containing protein(s) [protein]'], ['GYF domains [protein domain]'], ['PF02213 [protein family]'], ['GAF domain cleft [structure]'], ['C2 part of GbMYB25 [protein]'], ['GK domains of NGK-1 [protein]'], ['crystal structure of the ENT domain of EMSY [protein domain]'], ['Crystal structures of Geph-G [protein domain]'], ['Crystal packing of the DDL FHA domain [protein domain]'], ['predicted 46 kDa amino-terminal GI domain [phenotype]'], ['binding epitope of the GYF domain from the yeast SMY2 protein(s) [protein domain]'], ['GhIQD21 [protein]'], ['N-glycosylation in the FAS1 domain [protein domain]'], ['5KXF [identifier]'], ['GroEL protein(s) domain [protein]'], ['domains III, V, and VI [protein domain]'], ['H/G domain [protein domain]'], ['single GH1 domain [protein domain]'], ['WG-1 [subdomain]'], ['TFIIB-like domain architecture [protein domain]'], ['core GGF region [protein domain]'], ['**peptides spanning a portion of the HEAT1/MIFG domain [protein domain]**', 'peptides covering the HEAT1/MIFG domain [protein domain]'], ['protein(s) with GST domains and EF1 gamma domain [protein]'], ['protein(s) with GST domains but not EF1 gamma domain [protein]'], ['EF1Bgamma domain [phenotype]'], ['trans auto-fucosylation sites [protein domain]'], ['structural constraints between Fc N-glycan and IgG1 CH2 domains [protein]'], ['GhWDR-C [protein domain]'], ['cysteine-rich, EGF-like, and three sushi domains [subcellular compartment]'], ['Thr-138 [protein domain]'], ['GYGSTQTAxxxSxLxA repeats [protein domain]'], ['TIFY and Jas domains [domain]'], ['PHD-SWIB-Plus3-GYF domain [protein domain]'], ['glutamate-rich regions [protein domain]'], ['GIY-YIG nuclease domain [protein domain]'], ['polyproline (Poly-P) of formin AtFH1 [protein region]']]}\n",
      "-------------------\n",
      "{'172634': [['**OPP pathway [gene]**', 'the oxidative pentose phosphate (OPP) pathway [gene]'], ['Oxidative pentose phosphoric pathway (OPPP) [process]', 'OPPP [process]', 'Oxidative pentose phosphate (OPP) [process]', '**OPP pathway [process]**', 'oxidative pentose phosphate pathway (OPPP) [process]', 'OPPP pathway [process]', 'OPP [process]'], ['OPPP [metabolic pathway]', '**OPP pathway [metabolic pathway]**'], ['OPPP flux [pathway]'], ['OPPP [pathway]'], ['cytosolic OPPP flux [process]'], ['cytosolic OPPP [process]'], ['OPP pathway [phenotype]'], ['OPPP activation [process]'], ['OPPP pathway [metabolite]'], ['**oxidative pentose phosphate pathway (OPPP) [metabolite]**', 'oxidative pentose phosphate pathway (OPPP) [metabolite]'], ['OPP [metabolite]'], ['**oxidative part of OPPP [metabolic pathway]**', 'oxidative part of the OPPP [metabolic pathway]'], ['NADPH production by the OPPP [metabolite]'], ['incomplete oxidative pentose-phosphate pathway (OPPP) [metabolic pathway]'], ['**oxidative pentose-phosphate pathway (OPPP) [metabolic pathway]**', 'oxidative pentose phosphate pathway (OPPP) [metabolic pathway]'], ['the OPP pathway for nucleotide biosynthesis [gene]'], ['carbon skeletons from a shared resource pool in the OPP pathway [gene]'], ['carbon flow through OPPP [metabolic pathway]'], ['**first committed step of the oxidative pentose phosphate pathway (OPPP) [pathway]**', 'first committed step of the oxidative pentose-phosphate pathway (OPPP) [pathway]'], ['G6P flux through cytosolic OPPP [phenotype]'], ['accumulation of OPPP intermediates [metabolite]'], ['first OPPP step [process]'], ['NADPH-producing OPPP reactions [metabolic pathway]'], ['**third OPPP step [process]**', 'third step of the oxidative pentose-phosphate pathway (OPPP) [process]'], ['transport of OPPP metabolites [process]'], ['glycolysis and OPPP [metabolic pathway]'], ['separation of Calvin-Benson cycle from OPPP [phenotype]'], ['first step of the oxidative pentose-phosphate pathway (OPPP) [pathway]'], ['regeneration of glucose-6-P molecules in the oxidative pentose phosphate pathway [metabolic pathway]'], ['production of OPPP/glycolysis intermediates [process]'], ['OPPP in the dark [process]'], ['activation of the OPPP [metabolic pathway]'], ['OPP pathway [metabolic process]'], ['step of OPPP beyond PGL [metabolite]'], ['complete oxPPP [process]'], ['OPPP reactions [process]'], ['first step of OPPP [metabolic pathway]']]}\n",
      "-------------------\n",
      "{'109900': [['**Dof motif [gene]**', 'presence of conserved DNA binding with one finger (Dof) binding elements [gene]', 'three conserved Dof elements [gene]', 'cognate sites for Dof transcription factors [gene]', '(A/T)AAAG sequence [gene]'], ['IDD motif [gene]'], ['CBF DNA binding motif [gene]'], ['Dof-box (-341/-338) [dna sequence]', '**Dof TFBS [dna sequence]**'], ['IDEF2 motif [dna sequence]'], ['ERF115 motif [phenotype]'], ['Dof factor binding motifs [phenotype]'], ['Dof-binding consensus sequences in their promoter regions [protein]'], ['CORE sites [gene region]'], ['MEF2-type [dna motif]'], ['Dof-binding motifs [dna motif]'], ['AAAG sequence [sequence]'], ['CTTT binding site [sequence]']]}\n",
      "-------------------\n",
      "{'14502': [['**leaf color [phenotype]**', 'trait leaf color [phenotype]', 'leaf colour [phenotype]', 'leaf color phenotype [phenotype]', 'color of leaf [phenotype]', 'leaf color information [phenotype]', 'leaf color traits [phenotype]', 'color trait [phenotype]'], ['leaf colour in lettuce [phenotype]'], ['Red colour of lettuce leaf [phenotype]'], ['Red leaf [phenotype]'], ['red leaf and petioles [phenotype]'], ['light red [phenotype]'], ['light red and dark red leaf [phenotype]'], ['orange leaf [phenotype]'], ['colorful autumn foliage [phenotype]'], ['head leaf color [phenotype]']]}\n",
      "-------------------\n",
      "{'167068': [['natural product statins [metabolite]'], ['unnatural cytochalasan derivatives [metabolite]'], ['sulfoxides [metabolite]'], ['sulfated product 11 [metabolite]'], ['sulfonated natural products [metabolite]'], ['pharmaceutical formulations [metabolite]'], ['tablets [metabolite]'], ['N-hydroxylated compounds [metabolite]'], ['drug-like scaffolds [metabolite]']]}\n",
      "-------------------\n",
      "{'88886': [['30-day high altitude balloon mission [treatment]'], ['1.5 years of exposure to space vacuum [treatment]'], ['10 days on STS-9 [treatment]']]}\n",
      "-------------------\n",
      "{'65805': [['E0-4-2, E3-1-2, and E5-3-1 fragments [gene]'], ['AFP2 recruitment of TPL/HDACs [protein]'], ['activity of binding partners like HDA6 [protein]'], ['CMM2 interactions [protein interaction]'], ['FANCM direct DNA-binding cofactors [protein]'], ['two proteins, PCF1 and PCF2 [protein]'], ['protein-DNA interaction network [signaling pathway]'], ['E2F binding to RB [protein]'], ['ABF-ABRE associations [protein-gene interaction]'], ['EmBP-1 [protein]'], ['binding of CCA1 [protein]'], ['EIN3/EIL protein(s) [protein]'], ['SA-bound NPR1 dimer [protein]'], ['several portions of HBP-1a(17) [protein]'], ['LHP1 and ABAP1 [protein]'], ['DNA binding of ARF6 and ARF8 [process]'], ['ZRF1 binding to chromatin [protein]'], ['CDK8 binding [protein]'], ['interaction with ARF6 and ARF8 [protein]'], ['CRYPTOCHROME-INTERACTING BASIC-HELIX-LOOP-HELIX 1 (CIB1) [protein]'], ['bind to CmMYB6 [protein]'], ['correct binding of regulators [process]'], ['specific KNAT homeodomain protein(s) [protein]'], ['ABF3 binding [interaction]'], [\"SOB3/AHL29's physical interaction with transcription factors [protein]\"], ['dimerization protein(s) DPb [gene]'], ['Binding of CG-1 [protein]'], ['ABF4-AtCPK32 interaction [interaction]']]}\n",
      "-------------------\n",
      "{'7940': [['Viral diseases [phenotype]'], ['viral process [phenotype]'], ['Viruses [phenotype]'], ['how viruses tune the relationship between multiplication and virulence [phenotype]', '**link between virus multiplication and virulence [phenotype]**'], ['many other aspects of the viral lifecycle [phenotype]'], ['**virus behavior [phenotype]**', 'behaviour of viruses [phenotype]'], ['vector behaviors [phenotype]'], ['viral dynamics [phenotype]'], ['virus research [phenotype]'], ['virome [phenotype]'], ['category viral process [phenotype]'], [\"viruses' activities [phenotype]\", '**viral functions [phenotype]**']]}\n",
      "-------------------\n",
      "{'90345': [['plant(s) hormones and abiotic stress signals [regulatory factor]'], ['AtBI-1 function [process]'], ['Biotic and abiotic interactions [process]'], ['regulation of defense response to biotic and abiotic stresses [process]', '**response to biotic and abiotic stresses [process]**', 'induction of responses to biotic and abiotic stresses [process]', 'vital regulators in plant(s) responses to biotic and abiotic stresses [process]', 'abiotic/biotic responses [process]', 'plant(s) immunity and abiotic stress responses [process]', 'different abiotic and biotic stress responses [process]', 'mediating plant(s) response to abiotic and biotic stresses [process]', \"regulation of plants' response to abiotic and biotic stresses [process]\"], ['developmental and abiotic stress responses [process]'], ['mediators during biotic and abiotic stresses [function]'], ['pathogen attack and development to abiotic stresses [process]'], ['(a)biotic stress tolerance and hormone signalling [process]'], [\"plant's signaling response to abiotic stresses [process]\"], ['phytohormone and abiotic stresses [process]'], ['factors sensitive to biotic and abiotic stimuli [protein complex]'], ['A. thaliana responses to multiple abiotic stresses [process]'], ['both plant(s) growth and development and abiotic stress [process]'], ['important role in abiotic stresses [process]'], ['key components in response to a wide range of abiotic stresses [process]'], ['abiotic stress, light and phytohormone responses [process]']]}\n",
      "-------------------\n",
      "{'31615': [['auxin-mediated modulation of the mitotic-to-endocycle switch [phenotype]'], ['auxin-dependent inhibition of endocytosis [process]', '**auxin-regulated endocytosis [process]**', 'auxin regulation of endocytosis [process]']]}\n",
      "-------------------\n",
      "{'157227': [['GmSAUR [gene]'], ['StSAUR [gene]'], ['ZmSAUR1 (Zea mays SAURs) [gene]', '**ZmSAUR1 [gene]**'], ['SlSAUR1-like [gene]']]}\n",
      "-------------------\n",
      "{'131559': [['BtuB gene(s) [gene]'], ['BtuC gene(s) [gene]'], ['BtuN gene(s) [gene]'], ['cytochrome b(5) [gene]']]}\n",
      "-------------------\n",
      "{'99091': [[\"Genome sequencing of O. fragrans 'Rixianggui' (OFR) [organism]\"], [\"**O. fragrans 'Liuyejingui' (OFL) [organism]**\", 'OFL [organism]'], ['genome-wide sequence and transcriptome of Osmanthus fragrans [organism]'], ['D. oleifera genome [organism]']]}\n",
      "-------------------\n",
      "{'65306': [['**CO/NF-Y complex [protein complex]**', 'CO/NF-Y heterotrimers [protein complex]', 'CO/NF-Y structure [protein complex]'], ['Hd1/NF-Y complexes [protein complex]'], ['NF-YB/NF-YC dimer [protein complex]'], ['NF-YA-YB-CO trimer [protein complex]'], ['**CO/NF-YB/NF-YC trimer complex [protein complex]**', 'trimer structure of NF-YB-YC-CO [protein complex]'], ['PIF4-NF-YB9 complex [protein complex]'], ['NF-TOC1 [protein complex]'], ['NF-YCs [protein complex]']]}\n",
      "-------------------\n"
     ]
    }
   ],
   "source": [
    "for chunk in chunks:\n",
    "    print(chunk)\n",
    "    print('-------------------')\n",
    "    "
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Submit this file to batchapi in the playground"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 132,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "done writing to jsonl file\n"
     ]
    }
   ],
   "source": [
    "import jsonlines\n",
    "import json\n",
    "output_path = '/'\n",
    "out_file = 'Users/manojitharajula/Documents/PhD/Connectome/Entity_disambiguation/kmeans/o3mini_val_ins/o3minihigh_1k_outs.jsonl'\n",
    "with jsonlines.open(output_path + out_file, mode='w') as file:\n",
    "    for i, chunk in enumerate(chunks):\n",
    "        \n",
    "        line = {\n",
    "            \"custom_id\": str(i),\n",
    "            \"method\": \"POST\",\n",
    "            \"url\": \"/v1/chat/completions\",\n",
    "            \"body\": {\n",
    "                \"model\": \"o3-mini\",\n",
    "                #\"temperature\": 0, #Enable this for non-reasoning model\n",
    "                #\"top_p\": 0, #Enable this for non-reasoning model\n",
    "                #\"frequency_penalty\": 0, #Enable this for non-reasoning model\n",
    "                #\"presence_penalty\": 0, #Enable this for non-reasoning model\n",
    "                \"response_format\": {\"type\": \"json_object\"},\n",
    "                \"messages\": [\n",
    "                    {\n",
    "                        \"role\": \"system\",\n",
    "                        \"content\": constant_system_message\n",
    "                    },\n",
    "                    {\n",
    "                        \"role\": \"user\",\n",
    "                        \"content\": constant_user_message\n",
    "                    },\n",
    "                    {\n",
    "                        \"role\": \"user\",\n",
    "                        \"content\": json.dumps(chunk, separators=(',', ':'))\n",
    "                    }\n",
    "                ],\n",
    "                \"reasoning_effort\": \"high\"  # Options: \"low\", \"medium\", \"high\" (only for reasoning models)\n",
    "                #\"max_tokens\": 16384 #Enable this for non-reasoning model\n",
    "            }\n",
    "        }\n",
    "        file.write(line)\n",
    "        #print(f'Wrote chunk {i+1} to file')\n",
    "\n",
    "\n",
    "\n",
    "\n",
    "print('done writing to jsonl file')"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Read the batchapi output file and convert it to csv"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Skipped 0 lines\n"
     ]
    }
   ],
   "source": [
    "import json\n",
    "import jsonlines\n",
    "import csv\n",
    "# output CSV file from .json file\n",
    "skip_lines = 0\n",
    "input_file = \"./o3mini_val_outs/batch_67c1822ff1e081908a02fc175e749f0b_output.jsonl\" # input batch jsonl output file from GPT\n",
    "output_csv = \"./o3mini_val_outs/o3mini_1k.csv\"\n",
    "with jsonlines.open(input_file) as reader, open(output_csv, mode=\"w\", newline=\"\", encoding=\"utf-8\") as f:\n",
    "    writer = csv.writer(f)\n",
    "\n",
    "    writer.writerow([\"cluster_id\",\"is_correct\", \"user_prompt\", \"clusters\"])\n",
    "    for line in reader:\n",
    "        # Navigate into the response body\n",
    "        try:\n",
    "            body = line[\"response\"][\"body\"]\n",
    "            # Get the JSON string from the assistant's \"content\"\n",
    "            content_str = body[\"choices\"][0][\"message\"][\"content\"]\n",
    "            data = json.loads(content_str)\n",
    "            for cluster_id, cluster_data in data.items():\n",
    "                # Ensure cluster_data is a dictionary before accessing keys\n",
    "                if not isinstance(cluster_data, dict):\n",
    "                    print(f\"Skipping: cluster_data for {cluster_id} is not a dictionary ->\", cluster_data)\n",
    "                    continue\n",
    "\n",
    "                # Extract values safely\n",
    "                is_correct = cluster_data.get(\"is_correct\", \"N/A\")  # Default to \"N/A\" if missing\n",
    "                user_prompt = cluster_data.get(\"user_prompt\", \"N/A\")\n",
    "                clusters = json.dumps(cluster_data.get(\"clusters\", []), ensure_ascii=False)\n",
    "\n",
    "                writer.writerow([cluster_id, is_correct, user_prompt, clusters])\n",
    "\n",
    "\n",
    "            \n",
    "            \n",
    "\n",
    "        except (KeyError, IndexError, json.JSONDecodeError):\n",
    "            # Skip lines that don't match the expected structure\n",
    "            print(\"Skipping line:\", line)\n",
    "            skip_lines += 1\n",
    "            print(line[\"response\"][\"body\"][\"choices\"][0][\"message\"][\"content\"])\n",
    "            pass\n",
    "print(f\"Skipped {skip_lines} lines\")\n",
    "              "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Creating jsonl file for O3mini validation (V5)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "#This is a script that loads a pandas dataframe from csv, converts the string formatted as a list into a list of lists. The list of lists is then flattened into a single list.\n",
    "\n",
    "import pandas as pd\n",
    "import random\n",
    "import json\n",
    "df = pd.read_csv('./gpt_final_outs/o3mini_maxclust_80_subset_random_1000_nocutoff.csv', index_col=None)\n",
    "\n",
    "#rename the column \"Entity Name\" to \"Entity\" and \"Group Items\" to \"output\"\n",
    "df = df.rename(columns={\"Group Items\": \"Output\"})\n",
    "\n",
    "# Convert the string formatted as a list into a list of lists\n",
    "df['Output'] = df['Output'].apply(lambda x: eval(x))\n",
    "\n",
    "# Flatten the list of lists into a single list\n",
    "df['Input'] = df['Output'].apply(lambda x: [item.strip(\"*\") for sublist in x for item in sublist])\n",
    "\n",
    "#randomize the order of the single list\n",
    "df['Input'] = df['Input'].apply(lambda x: random.sample(x, len(x)))\n",
    "#df = df[df['Input'].apply(len) < 30]\n",
    "# make a new column, where internal lists of length 1 are removed \n",
    "df['Output>1'] = df['Output'].apply(lambda x: [item for item in x if len(item) > 1])\n",
    "#create a flag column by name one_item. For each row in Output column, If length of the list in output column is less than 2, put True otherwise False.\n",
    "df['one_item'] = df['Output>1'].apply(lambda x: True if len(x) == 0 else False)\n",
    "\n",
    "#zip the Group_Items_no_singles with the Entity Name \n",
    "df[\"Output\"] = df.apply(lambda row: {row[\"Entity Name\"]: row['Output']}, axis=1)\n",
    "df[\"Input\"] = df.apply(lambda row: {row[\"Entity Name\"]: row['Input']}, axis=1)\n",
    "df['Output>1'] = df.apply(lambda row: {row[\"Entity Name\"]: row['Output>1']}, axis=1)\n",
    "\n",
    "\n",
    "\n",
    "\n",
    "\n",
    "#convert the columns to json strings\n",
    "df[\"Output\"] = df[\"Output\"].apply(lambda x: json.dumps(x) if isinstance(x, dict) else x)\n",
    "df[\"Input\"] = df[\"Input\"].apply(lambda x: json.dumps(x) if isinstance(x, dict) else x)\n",
    "df[\"Output>1\"] = df[\"Output>1\"].apply(lambda x: json.dumps(x) if isinstance(x, dict) else x)\n",
    "\n",
    "#Change order of columns to have input, then output, then output>1, then entity name\n",
    "df = df[['Entity Name', 'Input', 'Output', 'Output>1', 'one_item']]\n",
    "\n",
    "#filter by Output>1 length of the list > 1\n",
    "#use column Output, but use only the ones where it clusteres multiple items together"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "#read output csv file\n",
    "#load the  csv file\n",
    "output_csv = pd.read_csv(\"./o3mini_val_outs/o3mini_1k.csv\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "#create a dictionary with key as cluster_id and value as the clusters\n",
    "o3_correct_cluster_dict = {}\n",
    "for index, row in output_csv.iterrows():\n",
    "    o3_correct_cluster_dict[row['cluster_id']] = row['clusters']\n",
    "\n",
    "#create a new column named o3_corrected_cluster in df\n",
    "df['o3_corrected_cluster'] = df['Entity Name'].map(o3_correct_cluster_dict)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "variable_o3mini_corrected_cluster_enitity_dict =  df.set_index('Entity Name')['o3_corrected_cluster'].to_dict()\n",
    "chunks_variable_o3mini_corrected_cluster = split_dict(variable_o3mini_corrected_cluster_enitity_dict,1)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "variable_o3mini_corrected_cluster_dict = {}\n",
    "for d in chunks_variable_o3mini_corrected_cluster:\n",
    "    variable_o3mini_corrected_cluster_dict.update(d)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "detailed_description = \"\"\"\n",
    "Input-1\n",
    "\n",
    "{\n",
    "  \"0\": [\n",
    "    \"meiotic arrest [phenotype]\",\n",
    "    \"delayed/arrested meiosis [phenotype]\",\n",
    "    \"absence of meiotic arrest [phenotype]\",\n",
    "    \"meiotic prophase arrest [phenotype]\",\n",
    "    \"males arresting in the middle of prophase I [phenotype]\",\n",
    "    \"early arrest [phenotype]\",\n",
    "    \"leptotene arrest [phenotype]\",\n",
    "    \"pachytene arrest [phenotype]\",\n",
    "    \"arrest in late prophase I [phenotype]\",\n",
    "    \"meiosis I arrest [phenotype]\",\n",
    "    \"meiotic arrest at telophase I [phenotype]\",\n",
    "    \"male meiosis until the end of the first division [phenotype]\",\n",
    "    \"termination of meiosis after anaphase I [phenotype]\",\n",
    "    \"meiotic arrest in anaphase II [phenotype]\",\n",
    "    \"arrested zygotic divisions [phenotype]\",\n",
    "    \"arrested endosperm nuclear divisions [phenotype]\",\n",
    "    \"arrest of the first mitotic division in gametogenesis [phenotype]\",\n",
    "    \"arresting prior to the first mitotic division [phenotype]\",\n",
    "    \"mitotic arrest during female gametogenesis [phenotype]\"\n",
    "  ]\n",
    "}\n",
    "\n",
    "Clustered entities given input-1:\n",
    "\n",
    "{\n",
    "  \"0\": [\n",
    "    [\n",
    "      \"Meiotic block [phenotype]\",\n",
    "      \"**meiotic arrest [phenotype]**\",\n",
    "      \"meiotic arrest phenotype [phenotype]\",\n",
    "      \"meiotic division stop [phenotype]\"\n",
    "    ],\n",
    "    [\n",
    "      \"block in meiosis prophase I [phenotype]\",\n",
    "      \"**meiotic prophase arrest [phenotype]**\"\n",
    "    ],\n",
    "    [\n",
    "      \"**pachytene arrest [phenotype]**\",\n",
    "      \"meiotic arrest at pachytene [phenotype]\"\n",
    "    ],\n",
    "    [\n",
    "      \"arrest at the end of meiosis I [phenotype]\",\n",
    "      \"arrest after meiosis I [phenotype]\",\n",
    "      \"**meiotic arrest at telophase I [phenotype]**\"\n",
    "    ],\n",
    "    [\n",
    "      \"**termination of meiosis after anaphase I [phenotype]**\",\n",
    "      \"premature termination of meiosis after anaphase I [phenotype]\"\n",
    "    ],\n",
    "    [\n",
    "      \"arrest of meiotic progression in anaphase II [phenotype]\",\n",
    "      \"**meiotic arrest in anaphase II [phenotype]**\"\n",
    "    ],\n",
    "    [\n",
    "      \"arresting the first mitosis during gametogenesis [phenotype]\",\n",
    "      \"**arrest of the first mitotic division in gametogenesis [phenotype]**\",\n",
    "      \"FNM half-stop [phenotype]\"\n",
    "    ]\n",
    "  ]\n",
    "}\n",
    "\n",
    "Output-1:\n",
    "{\n",
    "  \"0\": {\n",
    "    \"reasoning\": \"All synonyms describing a complete, generic meiotic halt were clustered under **'meiotic arrest [phenotype]'**, chosen as the representative because it is concise and universally recognized. Broader prophase I arrests merged into **'meiotic prophase arrest [phenotype]'**, selected for its succinct coverage of any block in prophase I. Pachytene-specific arrests were grouped under **'pachytene arrest [phenotype]'**, chosen for clarity and direct reference to the fully synapsed stage. Arrests at the end of meiosis I appear under **'meiotic arrest at telophase I [phenotype]'**, precisely indicating telophase I. Post-anaphase I failures were combined into **'termination of meiosis after anaphase I [phenotype]'**, spotlighting the exact point of cessation. Anaphase II stalls were labeled **'meiotic arrest in anaphase II [phenotype]'**, pinpointing the second division stage. Finally, blocks in the first post-meiotic mitosis formed the cluster **'arrest of the first mitotic division in gametogenesis [phenotype]'**, chosen for its explicit naming of that specific cell-cycle event. Single-term categories (e.g., 'delayed/arrested meiosis [phenotype]', 'absence of meiotic arrest [phenotype]') remain excluded from the final output.\"\n",
    "  }\n",
    "}\n",
    "\n",
    "\n",
    "Input-2\n",
    "{\n",
    "  \"2\": '[\n",
    "    \"Salt-stress severity [treatment]\",\n",
    "    \"high NaCl stress [treatment]\",\n",
    "    \"potassium deprivation stress [treatment]\",\n",
    "    \"salt-stress response [treatment]\",\n",
    "    \"salt stress tolerance [treatment]\",\n",
    "    \"heat and salt stress conditions [treatment]\",\n",
    "    \"prolonged levels of salt stress [treatment]\",\n",
    "    \"recovery from salt stress [treatment]\",\n",
    "    \"salt stress assay [treatment]\",\n",
    "    \"salt stress signaling pathways [treatment]\",\n",
    "    \"gradual salt stress treatments [treatment]\",\n",
    "    \"salt and low temperature stresses [treatment]\",\n",
    "    \"salt and silicon stresses [treatment]\"\n",
    "  ]'\n",
    "}\n",
    "\n",
    "clustered entities for given input-2\n",
    "{\n",
    "  \"2\": [\n",
    "    [\n",
    "      \"Salt-stress severity [treatment]\",\n",
    "      \"**high NaCl stress [treatment]**\",\n",
    "      \"salt-stress response [treatment]\",\n",
    "      \"salt stress tolerance [treatment]\",\n",
    "      \"prolonged levels of salt stress [treatment]\",\n",
    "      \"recovery from salt stress [treatment]\",\n",
    "      \"salt stress assay [treatment]\",\n",
    "      \"salt stress signaling pathways [treatment]\",\n",
    "      \"gradual salt stress treatments [treatment]\"\n",
    "    ]\n",
    "  ]\n",
    "}\n",
    "\n",
    "Output-2:\n",
    "{'2':{'reasoning':'All treatments that involve only salt (NaCl)––such as “Salt-stress severity,” “salt-stress response,” “salt stress tolerance,” “prolonged levels of salt stress,” “recovery from salt stress,” “salt stress assay,” “salt stress signaling pathways,” “gradual salt stress treatments,” and “high NaCl stress”––cluster together, because each focuses on the same abiotic stress factor (NaCl). They appear in the final output under the representative “**high NaCl stress [treatment]**”. In contrast, single-term items that mix salt with another factor (e.g., heat, low temperature, or silicon) or that involve a completely different stress (e.g., potassium deprivation) remain separate clusters of length one and thus are omitted from the output.'}}\n",
    "\n",
    "\"\"\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "import jsonlines\n",
    "import json\n",
    "output_path = '/Users/manojitharajula/Documents/PhD/Connectome/Entity_disambiguation/kmeans/o3mini_val_ins/o3minihigh_v2_1k_outs.jsonl'\n",
    "with jsonlines.open(output_path, mode='w') as file:\n",
    "    for i in range(len(chunks_variable_input_entity)):\n",
    "    \n",
    "        line = {\n",
    "            \"custom_id\": str(i),\n",
    "            \"method\": \"POST\",\n",
    "            \"url\": \"/v1/chat/completions\",\n",
    "            \"body\": {\n",
    "                \"model\": \"o3-mini\",\n",
    "                #\"temperature\": 0, #Enable this for non-reasoning model\n",
    "                #\"top_p\": 0, #Enable this for non-reasoning model\n",
    "                #\"frequency_penalty\": 0, #Enable this for non-reasoning model\n",
    "                #\"presence_penalty\": 0, #Enable this for non-reasoning model\n",
    "                \"response_format\": {\"type\": \"json_object\"},\n",
    "                \"messages\": [\n",
    "                    {\n",
    "                        \"role\": \"system\",\n",
    "                        \"content\": system_message\n",
    "                    },\n",
    "                    {\n",
    "                        \"role\": \"user\",\n",
    "                        \"content\": detailed_description\n",
    "                    },\n",
    "                    {\n",
    "                        \"role\": \"user\",\n",
    "                        \"content\": json.dumps(chunks_variable_input_entity[i], indent=2)\n",
    "                    }\n",
    "                ],\n",
    "                \"reasoning_effort\": \"high\"  # Options: \"low\", \"medium\", \"high\" (only for reasoning models)\n",
    "                #\"max_tokens\": 16384 #Enable this for non-reasoning model\n",
    "            }\n",
    "        }\n",
    "        file.write(line)\n",
    "        #print(f'Wrote chunk {i+1} to file')\n",
    "\n",
    "\n",
    "\n",
    "\n",
    "print('done writing to {} file'.format(output_path))"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Read the output jsonl file and convert it to csv file"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "import json\n",
    "import jsonlines\n",
    "import csv\n",
    "# output CSV file from .json file\n",
    "skip_lines = 0\n",
    "input_file = \"./o3mini_val_outs/batch_67c58e5895ec81909dfeaea04841b738_output.jsonl\" # input batch jsonl output file from GPT\n",
    "output_csv = \"./o3mini_val_outs/o3mini_1k_v2_again.csv\"\n",
    "with jsonlines.open(input_file) as reader, open(output_csv, mode=\"w\", newline=\"\", encoding=\"utf-8\") as f:\n",
    "    writer = csv.writer(f)\n",
    "\n",
    "    writer.writerow([\"cluster_id\",\"reasoning\"])\n",
    "    for line in reader:\n",
    "        # Navigate into the response body\n",
    "        try:\n",
    "            body = line[\"response\"][\"body\"]\n",
    "            # Get the JSON string from the assistant's \"content\"\n",
    "            content_str = body[\"choices\"][0][\"message\"][\"content\"]\n",
    "            data = json.loads(content_str)\n",
    "            for cluster_id, cluster_data in data.items():\n",
    "                #if int(cluster_id) not in empty_keys:\n",
    "                reasoning = cluster_data.get(\"reasoning\")\n",
    "                writer.writerow([cluster_id, reasoning])\n",
    "\n",
    "\n",
    "            \n",
    "            \n",
    "\n",
    "        except (KeyError, IndexError, json.JSONDecodeError):\n",
    "            # Skip lines that don't match the expected structure\n",
    "            print(\"Skipping line:\", line)\n",
    "            skip_lines += 1\n",
    "            print(line[\"response\"][\"body\"][\"choices\"][0][\"message\"][\"content\"])\n",
    "            pass\n",
    "print(f\"Skipped {skip_lines} lines\")\n",
    "              "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "connectome",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.12.4"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
